{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Modelado de los residuos restantes de los cristales\n",
    "### Usamos Modeller\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/joel/anaconda3/envs/mds/lib/python3.6/importlib/_bootstrap.py:219: RuntimeWarning: numpy.ufunc size changed, may indicate binary incompatibility. Expected 192 from C header, got 216 from PyObject\n",
      "  return f(*args, **kwds)\n"
     ]
    }
   ],
   "source": [
    "# Librerías\n",
    "from modeller import * # licencia modeller necesaria\n",
    "from prody import *\n",
    "import csv\n",
    "from modeller import * # licencia modeller necesaria\n",
    "from modeller.automodel import *\n",
    "from prody import * \n",
    "from Bio import pairwise2, SeqIO\n",
    "import os\n",
    "import glob"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Obtener la secuencia de UniProt\n",
    "Utilizamos la secuencia UNIPROT correspondiente a la PROTEÍNA de interés."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Nombre de la proteina de interes\n",
    "prot_name = 'erk2'\n",
    "# Secuencia P28482 (ERK2_HUMAN)\n",
    "uniprot_id = \"P28482\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "MAAAAAAGAGPEMVRGQVFDVGPRYTNLSYIGEGAYGMVCSAYDNVNKVRVAIKKISPFEHQTYCQRTLREIKILLRFRHENIIGINDIIRAPTIEQMKDVYIVQDLMETDLYKLLKTQHLSNDHICYFLYQILRGLKYIHSANVLHRDLKPSNLLLNTTCDLKICDFGLARVADPDHDHTGFLTEYVATRWYRAPEIMLNSKGYTKSIDIWSVGCILAEMLSNRPIFPGKHYLDQLNHILGILGSPSQEDLNCIINLKARNYLLSLPHKNKVPWNRLFPNADSKALDLLDKMLTFNPHKRIEVEQALAHPYLEQYYDPSDEPIAEAPFKFDMELDDLPKEKLKELIFEETARFQPGYRS\n"
     ]
    }
   ],
   "source": [
    "# Secuencia de la CDK2 de UniProt\n",
    "fasta = SeqIO.read(open(F'./data/{uniprot_id}.fasta'),'fasta')\n",
    "seq_prot = str(fasta.seq)\n",
    "print(seq_prot)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Función find gaps\n",
    "Cargamos la función que permite permite identificar los gaps en una secuencia y devolver el rango del gap."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [],
   "source": [
    "import sys\n",
    "sys.path.append(r'../..')\n",
    "from modulos_python.find_gaps import find_gaps\n",
    "# Recibe una secuencia (con gaps como \"-\") y un valor r para valorar el gap en +/- r posiciones "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'num_gaps': 2,\n",
       " 'gap_lengths': [3, 1],\n",
       " 'gap_list': [[1, 3], [5, 5]],\n",
       " 'gap_window': [[1, 3], [5, 5]]}"
      ]
     },
     "execution_count": 5,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# EJEMPLO:\n",
    "find_gaps(\"---3-\", r=0)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Obtenemos los PDB IDs de los cristales\n",
    "Los ids los tomamos de la tabla de metadatos generada cen la libreta *Get_Metadatos..*.  "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Total de PDB IDs a modelar: 165\n"
     ]
    }
   ],
   "source": [
    "# Los IDs de los cristales están en el archivo\n",
    "import pandas as pd\n",
    "import glob\n",
    "file_path = glob.glob(F\"./data/TABLA_MTDATA_{prot_name.upper()}_*.csv\")[0]\n",
    "df_prot = pd.read_csv(file_path, index_col=1)\n",
    "pdbids_list = df_prot.index\n",
    "print(\"Total de PDB IDs a modelar:\", len(pdbids_list))"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Se ejecuta Modeller para refinamiento por loops\n",
    "#### Descripción de los pasos a ejecutar:\n",
    "- **1)** Se definen los directorios de entradas y salidas.\n",
    "- **2)** Se carga la estructura cristalográfica a modelar (*cristal*), correspondiente a cada pdb id.\n",
    "    - Si la proteína ya fue modelada se omite volver a modelarla.\n",
    "- **3)** Se hace un alineamiento de la secuencia del cristal y la secuecnia **P24941**.\n",
    "- **4)** Se identifican los gaps en la secuencia del cristal, si los hay. Se define una ventana de $\\pm$ *r* residuos que se considerarán dentro del gap.\n",
    "- **5)** Se modifica el método *select_atoms* de la clase *MyModel* de MODELLER para indicar que únicamente modele los residuos que pertenecen al gap del cristal, dejando intacto el resto de los átomos de la proteína.\n",
    "- **6)** Ejecuta MODELLER con los parámetros indicados, generando únicamente un modelo, el cual es guardado como _PDBID_**_MODLL.pdb**."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Directiros y parámetros base\n",
    "struct_dir = F'../ARCHIVOS/CRISTALES/PROT_{prot_name.upper()}_CHAINS/'\n",
    "tail_pdb = '_A.pdb' # extensión del archivo pdb del cristal\n",
    "tail_model = '_MODLL' # epiteto del archivo del modelo final, la extensión .pdb la agrega modeller\n",
    "pdbs_model_dir = F\"../ARCHIVOS/CRISTALES/PROT_{prot_name.upper()}_MODELOS_modeller_NoPrep/\" # Carpeta de salida para los modelos\n",
    "num_res_window = 2 # Número de residuos adyacentes al gap que podrán modelarse\n",
    "\n",
    "# Creamos el directorio de salida si es que no existe\n",
    "import os\n",
    "if not os.path.exists(pdbs_model_dir):\n",
    "    os.makedirs(pdbs_model_dir)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2756 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "El modelo ya existe en el directorio\n",
      "El modelo ya existe en el directorio\n",
      "El modelo ya existe en el directorio\n",
      "El modelo ya existe en el directorio\n",
      "El modelo ya existe en el directorio\n",
      "El modelo ya existe en el directorio\n",
      "El modelo ya existe en el directorio\n",
      "El modelo ya existe en el directorio\n",
      "Modelando proteína 2ojg\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    37821    35175\n",
      "226 (of 2915 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2808 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:      -91.0292\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     226\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    35175    1555\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      310\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         187.0739\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     232       0      0   0.003   0.003     0.68954       1.000\n",
      " 2 Bond angle potential               :     314       0      0   1.907   1.907      21.323       1.000\n",
      " 3 Stereochemical cosine torsion poten:     137       0      4  48.190  48.190      48.636       1.000\n",
      " 4 Stereochemical improper torsion pot:      91       1      1   1.737   1.737      8.4359       1.000\n",
      " 5 Soft-sphere overlap restraints     :     310       0      0   0.002   0.002     0.11591       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     210       0      0   0.151   0.151      3.7728       1.000\n",
      "10 Distance restraints 2 (N-O)        :     190       0      0   0.381   0.381      18.740       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      32       0      0   3.983   3.983      5.9861       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      20       0      1  80.600  80.600      11.355       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      17       0      0  70.766  70.766      7.3552       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:      11       0      0 101.824 101.824      6.9609       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       3       0      0 101.448 101.448      1.3098       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     146       0      0   0.667   0.667      1.5132       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      33      20      5  27.218 131.345      39.517       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     119       0      0   0.956   0.956      11.364       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 2ojg_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    5874.0537\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  4                           : Stereochemical improper torsion potentia\n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10117 329F 329F C   CA   2648 2640   12.79    0.00   12.79    4.50    0.00   12.79    4.50\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9  -69.01  -68.20    6.40    0.54  -62.50  179.97   29.18\n",
      "    1          2A   2A N   CA      9   10  138.95  145.30                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14  -78.17  -68.20   13.36    0.85  -62.50  165.65   27.94\n",
      "    2          3A   3A N   CA     14   15  154.20  145.30                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19  -64.22  -68.20    9.91    0.67  -62.50  177.13   28.94\n",
      "    3          4A   4A N   CA     19   20  136.23  145.30                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24 -121.54 -134.00   12.46    0.41  -62.50 -178.11   32.96\n",
      "    4          5A   5A N   CA     24   25  147.06  147.00                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29  -68.07  -68.20    4.56    0.36  -62.50  178.45   29.52\n",
      "    5          6A   6A N   CA     29   30  140.74  145.30                  -40.90\n",
      "    6  10235   6A   7A C   N      32   34 -160.05 -134.00   33.46    0.89  -62.50  179.86   34.65\n",
      "    6          7A   7A N   CA     34   35  167.99  147.00                  -40.90\n",
      "    7  10237   8G   9A C   N      41   43  -64.72  -68.20    4.61    0.29  -62.50  176.84   29.09\n",
      "    7          9A   9A N   CA     43   44  142.28  145.30                  -40.90\n",
      "    8  10238   9A  10G C   N      46   48 -143.47 -167.20   54.04    1.67   82.20  178.49   13.83\n",
      "    8         10G  10G N   CA     48   49  126.05  174.60                    8.50\n",
      "    9  10239  10G  11P C   N      50   52  -58.39  -58.70    2.71    0.20  -64.50  179.71   13.74\n",
      "    9         11P  11P N   CA     52   53  -33.19  -30.50                  147.20\n",
      "   10  10240  11P  12E C   N      57   59 -131.00 -117.80   50.30    2.24  -63.60  150.32   23.80\n",
      "   10         12E  12E N   CA     59   60 -174.66  136.80                  -40.30\n",
      "   11  10241  12E  13M C   N      66   68  -67.61  -73.00   31.87    2.25  -63.40  145.15   22.00\n",
      "   11         13M  13M N   CA     68   69  174.41  143.00                  -40.50\n",
      "   12  10244  15R  16G C   N      92   94  179.16 -167.20   21.99    0.86   82.20  177.64   12.86\n",
      "   12         16G  16G N   CA     94   95  157.35  174.60                    8.50\n",
      "   13  10245  16G  17Q C   N      96   98  -91.15  -73.00   18.96    1.24  -63.80  175.64   27.29\n",
      "   13         17Q  17Q N   CA     98   99  146.20  140.70                  -40.30\n",
      "   14  10249  20D  21V C   N     131  133  -92.54  -62.40   64.33    7.04  -62.40   64.33    7.04\n",
      "   14         21V  21V N   CA    133  134   14.44  -42.40                  -42.40\n",
      "   15  10559 330K 331F C   N    2657 2659 -176.61 -124.20   63.24    1.65   58.10 -167.77   27.61\n",
      "   15        331F 331F N   CA   2659 2660  178.70  143.30                   32.90\n",
      "   16  10560 331F 332D C   N    2668 2670  -80.66  -70.90   10.20    0.65  -63.30  167.66   21.78\n",
      "   16        332D 332D N   CA   2670 2671  153.25  150.30                  -40.00\n",
      "   17  10561 332D 333M C   N    2676 2678  -81.49  -73.00   15.84    0.99  -63.40  164.12   25.94\n",
      "   17        333M 333M N   CA   2678 2679  156.38  143.00                  -40.50\n",
      "   18  10585 356P 357G C   N    2880 2882  169.85 -167.20   24.51    0.36  -80.20  110.32    7.06\n",
      "   18        357G 357G N   CA   2882 2883 -176.79  174.60                  174.10\n",
      "   19  10586 357G 358Y C   N    2884 2886 -113.86 -124.30   15.30    1.22  -63.50  177.31   30.69\n",
      "   19        358Y 358Y N   CA   2886 2887  146.59  135.40                  -43.40\n",
      "   20  10587 358Y 359R C   N    2896 2898  -66.57  -72.10    9.58    0.83  -63.00  169.22   23.10\n",
      "   20        359R 359R N   CA   2898 2899  149.72  141.90                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    1    9   32   49  123  183  235  306  385  429  495\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "2ojg_MODLL.B99990001.pdb       187.07388\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 2oji\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    38110    35464\n",
      "131 (of 2915 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2815 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     131\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    35464     683\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      180\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          78.0711\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     133       0      0   0.004   0.004     0.58111       1.000\n",
      " 2 Bond angle potential               :     179       0      0   1.817   1.817      11.353       1.000\n",
      " 3 Stereochemical cosine torsion poten:      72       0      1  48.198  48.198      24.822       1.000\n",
      " 4 Stereochemical improper torsion pot:      49       0      0   1.138   1.138      1.4615       1.000\n",
      " 5 Soft-sphere overlap restraints     :     180       0      0   0.001   0.001     0.22348E-01   1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      68       0      0   0.187   0.187      1.3182       1.000\n",
      "10 Distance restraints 2 (N-O)        :      68       0      1   0.442   0.442      6.9449       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      20       0      0   3.211   3.211      2.4311       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       9       0      0  77.186  77.186      2.2461       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       7       0      0  74.551  74.551      2.9690       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       6       0      0  81.757  81.757      2.9766       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       2       0      0  82.294  82.294      1.9381       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      34       0      0   0.344   0.344     0.11738       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      20      11      1  26.446 137.650      16.718       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      16       0      0   1.309   1.309      2.1722       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 2oji_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    3134.2930\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9  -95.37  -68.20   33.95    2.20  -62.50  156.94   27.44\n",
      "    1          2A   2A N   CA      9   10  165.65  145.30                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14  -79.52  -68.20   13.32    1.30  -62.50  179.99   28.71\n",
      "    2          3A   3A N   CA     14   15  138.29  145.30                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19  -57.03  -68.20   11.17    0.94  -62.50  174.07   28.26\n",
      "    3          4A   4A N   CA     19   20  145.11  145.30                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24  -76.09  -68.20   19.49    1.31  -62.50  156.57   26.35\n",
      "    4          5A   5A N   CA     24   25  163.12  145.30                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29  -62.23  -68.20    8.55    0.84  -62.50  167.68   27.47\n",
      "    5          6A   6A N   CA     29   30  151.42  145.30                  -40.90\n",
      "    6  10235   6A   7A C   N      32   34  -60.03  -68.20   24.90    1.72  -62.50  162.70   26.78\n",
      "    6          7A   7A N   CA     34   35  121.78  145.30                  -40.90\n",
      "    7  10237   8G   9A C   N      41   43  -76.98  -68.20   15.89    1.54  -62.50  173.57   27.77\n",
      "    7          9A   9A N   CA     43   44  132.06  145.30                  -40.90\n",
      "    8  10238   9A  10G C   N      46   48 -143.28 -167.20   25.78    0.38   82.20 -153.65   15.57\n",
      "    8         10G  10G N   CA     48   49  164.98  174.60                    8.50\n",
      "    9  10239  10G  11P C   N      50   52  -53.42  -58.70    5.36    0.63  -64.50  178.99   12.78\n",
      "    9         11P  11P N   CA     52   53  -31.45  -30.50                  147.20\n",
      "   10  10240  11P  12E C   N      57   59  -82.23  -69.30   24.72    2.24  -63.60  162.81   20.74\n",
      "   10         12E  12E N   CA     59   60  121.44  142.50                  -40.30\n",
      "   11  10587 358Y 359R C   N    2896 2898  -55.19  -72.10   17.01    1.26  -63.00  178.92   23.65\n",
      "   11        359R 359R N   CA   2898 2899  140.15  141.90                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:   249  256   2.173\n",
      "  serious non-bonded atom clash:   250  413   2.113\n",
      "  serious non-bonded atom clash:   250  414   2.063\n",
      "  serious non-bonded atom clash:   251  405   1.456\n",
      "  serious non-bonded atom clash:   252  413   1.373\n",
      "  serious non-bonded atom clash:   252  414   2.083\n",
      "  serious non-bonded atom clash:   252  415   0.982\n",
      "  serious non-bonded atom clash:   252  416   1.666\n",
      "  serious non-bonded atom clash:   252  417   2.078\n",
      "  serious non-bonded atom clash:   253  415   1.710\n",
      "  serious non-bonded atom clash:   253  417   1.593\n",
      "  serious non-bonded atom clash:   253  418   2.205\n",
      "  serious non-bonded atom clash:   253  419   2.093\n",
      "  serious non-bonded atom clash:   254  415   2.118\n",
      "  serious non-bonded atom clash:   254  417   0.853\n",
      "  serious non-bonded atom clash:   254  419   1.566\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:    12    3    1    5    3   12   24   65  103  194  229  322  366  425  490\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "2oji_MODLL.B99990001.pdb        78.07114\n",
      "\n",
      "Modelando proteína 2ojj\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    38509    35863\n",
      "131 (of 2915 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2835 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     131\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    35863     720\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      147\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          74.0802\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     133       0      0   0.002   0.002     0.14208       1.000\n",
      " 2 Bond angle potential               :     179       0      0   1.417   1.417      6.9930       1.000\n",
      " 3 Stereochemical cosine torsion poten:      72       0      1  52.956  52.956      30.079       1.000\n",
      " 4 Stereochemical improper torsion pot:      49       0      0   0.864   0.864     0.82045       1.000\n",
      " 5 Soft-sphere overlap restraints     :     147       0      0   0.001   0.001     0.37742E-01   1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      72       0      0   0.099   0.099     0.56077       1.000\n",
      "10 Distance restraints 2 (N-O)        :      68       0      0   0.428   0.428      4.0983       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      20       0      0   1.773   1.773     0.74114       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       9       0      0 103.495 103.495      2.9030       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       7       0      0  42.618  42.618     0.65566       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       6       0      0 114.067 114.067      4.0948       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       2       0      0 131.161 131.161      1.6557       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      61       0      0   0.272   0.272     0.97204E-01   1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      20      13      2  30.411 151.957      19.687       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      22       0      0   0.772   0.772      1.5138       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 2ojj_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    3675.9419\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9  -70.20  -68.20    5.59    0.52  -62.50  179.18   29.75\n",
      "    1          2A   2A N   CA      9   10  140.08  145.30                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14  -75.36  -68.20   13.92    0.90  -62.50  162.37   27.26\n",
      "    2          3A   3A N   CA     14   15  157.24  145.30                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19 -110.30 -134.00   24.15    0.98  -62.50  174.17   31.08\n",
      "    3          4A   4A N   CA     19   20  151.62  147.00                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24 -129.23 -134.00   12.29    0.76  -62.50  174.07   32.10\n",
      "    4          5A   5A N   CA     24   25  158.33  147.00                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29 -135.80 -134.00    6.57    0.32  -62.50 -178.73   33.63\n",
      "    5          6A   6A N   CA     29   30  153.31  147.00                  -40.90\n",
      "    6  10235   6A   7A C   N      32   34 -157.14 -134.00   27.93    0.68  -62.50 -177.14   35.01\n",
      "    6          7A   7A N   CA     34   35  162.63  147.00                  -40.90\n",
      "    7  10237   8G   9A C   N      41   43  -70.88  -68.20   16.98    1.27  -62.50  157.26   26.19\n",
      "    7          9A   9A N   CA     43   44  162.07  145.30                  -40.90\n",
      "    8  10238   9A  10G C   N      46   48  -65.83  -80.20   30.10    0.70   82.20 -156.84    6.58\n",
      "    8         10G  10G N   CA     48   49  147.65  174.10                    8.50\n",
      "    9  10239  10G  11P C   N      50   52  -55.59  -58.70    4.24    0.31  -64.50  179.65   13.88\n",
      "    9         11P  11P N   CA     52   53  -33.37  -30.50                  147.20\n",
      "   10  10240  11P  12E C   N      57   59 -129.11 -117.80   20.57    0.75  -63.60  178.19   27.45\n",
      "   10         12E  12E N   CA     59   60  153.99  136.80                  -40.30\n",
      "   11  10241  12E  13M C   N      66   68 -121.11 -125.60   17.12    0.90  -63.40  172.42   30.34\n",
      "   11         13M  13M N   CA     68   69  157.02  140.50                  -40.50\n",
      "   12  10243  14V  15R C   N      81   83  -67.86  -63.00    4.91    0.85 -125.20 -173.39   10.01\n",
      "   12         15R  15R N   CA     83   84  -41.82  -41.10                  140.60\n",
      "   13  10587 358Y 359R C   N    2896 2898 -104.73 -125.20   27.19    0.81  -63.00  169.04   20.36\n",
      "   13        359R 359R N   CA   2898 2899  122.71  140.60                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    1    0   12   27   56  106  201  231  273  399  399  508\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "2ojj_MODLL.B99990001.pdb        74.08018\n",
      "\n",
      "Modelando proteína 2y9q\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    38458    35812\n",
      "121 (of 2915 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2690 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     121\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    35812     955\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      182\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         101.3576\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     123       0      0   0.008   0.008      2.2718       1.000\n",
      " 2 Bond angle potential               :     170       0      0   1.880   1.880      11.137       1.000\n",
      " 3 Stereochemical cosine torsion poten:      69       0      3  53.528  53.528      33.109       1.000\n",
      " 4 Stereochemical improper torsion pot:      51       0      0   0.769   0.769     0.99070       1.000\n",
      " 5 Soft-sphere overlap restraints     :     182       0      0   0.005   0.005     0.61893       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     175       0      3   0.378   0.378      20.342       1.000\n",
      "10 Distance restraints 2 (N-O)        :     154       0      1   0.443   0.443      12.800       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      20       0      0   2.023   2.023     0.21522       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       9       0      0  73.174  73.174    -0.93804       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       5       0      0  53.418  53.418      1.5525       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       2       0      0  83.739  83.739     0.90647       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       1       0      0 105.304 105.304     0.72366       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      90       0      0   0.744   0.744     0.72350       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      21      10      2  28.159 118.205      15.471       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      65       0      0   1.135   1.135      1.4344       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 2y9q_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    3002.4297\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9  -72.94  -68.20    8.45    0.54  -62.50  167.13   27.91\n",
      "    1          2A   2A N   CA      9   10  152.30  145.30                  -40.90\n",
      "    2  10232   3A   4A C   N      17   19 -136.83 -134.00   15.55    0.81  -62.50  173.54   32.42\n",
      "    2          4A   4A N   CA     19   20  162.29  147.00                  -40.90\n",
      "    3  10233   4A   5A C   N      22   24  -61.94  -68.20   11.98    0.77  -62.50  175.98   28.87\n",
      "    3          5A   5A N   CA     24   25  135.09  145.30                  -40.90\n",
      "    4  10234   5A   6A C   N      27   29  -80.54  -68.20   16.30    1.04  -62.50  164.14   27.82\n",
      "    4          6A   6A N   CA     29   30  155.96  145.30                  -40.90\n",
      "    5  10235   6A   7A C   N      32   34 -155.54 -134.00   25.12    0.59  -62.50 -175.63   35.18\n",
      "    5          7A   7A N   CA     34   35  159.93  147.00                  -40.90\n",
      "    6  10236   7A   8G C   N      37   39 -140.51 -167.20   42.32    1.06   82.20 -168.68   14.71\n",
      "    6          8G   8G N   CA     39   40  141.76  174.60                    8.50\n",
      "    7  10237   8G   9A C   N      41   43 -117.91 -134.00   17.60    0.40  -62.50 -172.38   33.70\n",
      "    7          9A   9A N   CA     43   44  139.85  147.00                  -40.90\n",
      "    8  10387 158N 159T C   N    1269 1271  -64.43  -63.20    9.02    1.24   55.90  150.58   18.34\n",
      "    8        159T 159T N   CA   1271 1272  -51.03  -42.10                   39.50\n",
      "    9  10388 159T 160T C   N    1276 1278  -84.17  -63.20   77.54    8.73  -63.20   77.54    8.73\n",
      "    9        160T 160T N   CA   1278 1279   32.55  -42.10                  -42.10\n",
      "   10  10587 358Y 359R C   N    2896 2898 -107.26 -125.20   19.43    0.96  -63.00  176.50   26.90\n",
      "   10        359R 359R N   CA   2898 2899  148.04  140.60                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    0    2   14   38  126  182  203  251  366  401  501\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "2y9q_MODLL.B99990001.pdb       101.35764\n",
      "\n",
      "Modelando proteína 2z7l\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    36893    34247\n",
      "313 (of 2915 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2739 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     313\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    34247    2164\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      447\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         296.3005\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     324       0      0   0.003   0.003      1.0062       1.000\n",
      " 2 Bond angle potential               :     441       0      0   1.929   1.929      31.306       1.000\n",
      " 3 Stereochemical cosine torsion poten:     201       0      8  50.046  50.046      73.253       1.000\n",
      " 4 Stereochemical improper torsion pot:     130       0      0   0.876   0.876      2.4564       1.000\n",
      " 5 Soft-sphere overlap restraints     :     447       0      0   0.003   0.003     0.38803       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     269       0      0   0.167   0.167      4.6841       1.000\n",
      "10 Distance restraints 2 (N-O)        :     273       0      0   0.268   0.268      7.8412       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      45       1      0  26.948  26.948      5.1146       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      28       0      0  72.613  72.613      6.4865       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      25       0      0  48.017  48.017      4.5442       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:      15       0      0  79.663  79.663      8.4737       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       3       0      0  90.074  90.074      2.3591       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     189       0      0   0.484   0.484     0.86245       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      47      30      9  35.546 134.115      123.03       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     174       0      0   1.337   1.337      24.495       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 2z7l_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    7710.2773\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 13                           : Mainchain Omega dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10620  33E  33E CA  C     228  234    0.43 -179.18  179.61   41.75 -179.18  179.61   41.75\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9  -73.99  -68.20    8.50    0.54  -62.50  167.97   28.10\n",
      "    1          2A   2A N   CA      9   10  151.53  145.30                  -40.90\n",
      "    2  10232   3A   4A C   N      17   19 -173.08 -134.00   62.91    2.16  -62.50  165.26   32.78\n",
      "    2          4A   4A N   CA     19   20 -163.70  147.00                  -40.90\n",
      "    3  10233   4A   5A C   N      22   24  -58.57  -68.20    9.79    0.77  -62.50  175.62   28.59\n",
      "    3          5A   5A N   CA     24   25  143.52  145.30                  -40.90\n",
      "    4  10234   5A   6A C   N      27   29  -64.03  -68.20   12.32    0.85  -62.50  174.61   28.54\n",
      "    4          6A   6A N   CA     29   30  133.71  145.30                  -40.90\n",
      "    5  10235   6A   7A C   N      32   34  -85.33  -68.20   23.59    1.49  -62.50  159.22   27.27\n",
      "    5          7A   7A N   CA     34   35  161.53  145.30                  -40.90\n",
      "    6  10237   8G   9A C   N      41   43  -71.52  -68.20   17.79    1.32  -62.50  156.59   26.11\n",
      "    6          9A   9A N   CA     43   44  162.78  145.30                  -40.90\n",
      "    7  10238   9A  10G C   N      46   48  -82.59  -80.20    9.91    0.28   82.20 -119.47    7.29\n",
      "    7         10G  10G N   CA     48   49 -176.28  174.10                    8.50\n",
      "    8  10239  10G  11P C   N      50   52  -70.22  -58.70   18.56    1.18  -64.50  163.24   12.49\n",
      "    8         11P  11P N   CA     52   53  -15.94  -30.50                  147.20\n",
      "    9  10240  11P  12E C   N      57   59  -60.62  -69.30    8.79    0.71  -63.60  175.84   23.28\n",
      "    9         12E  12E N   CA     59   60  143.89  142.50                  -40.30\n",
      "   10  10241  12E  13M C   N      66   68 -129.67 -125.60   26.77    1.19  -63.40  166.31   30.06\n",
      "   10         13M  13M N   CA     68   69  166.96  140.50                  -40.50\n",
      "   11  10243  14V  15R C   N      81   83 -114.31 -125.20   16.78    0.93  -63.00  173.31   26.96\n",
      "   11         15R  15R N   CA     83   84  153.36  140.60                  -41.10\n",
      "   12  10244  15R  16G C   N      92   94  -86.13  -80.20   29.43    0.88   82.20 -123.89   18.13\n",
      "   12         16G  16G N   CA     94   95 -157.07  174.10                    8.50\n",
      "   13  10245  16G  17Q C   N      96   98  -59.61  -73.00   15.10    0.93  -63.80  174.09   25.82\n",
      "   13         17Q  17Q N   CA     98   99  133.73  140.70                  -40.30\n",
      "   14  10259  30Y  31I C   N     213  215 -132.09  -63.40   68.75   12.18  -63.40   68.75   12.18\n",
      "   14         31I  31I N   CA    215  216  -40.62  -43.60                  -43.60\n",
      "   15  10262  33E  34G C   N     234  236 -143.10 -167.20   38.55    0.97   82.20 -168.59   14.67\n",
      "   15         34G  34G N   CA    236  237  144.50  174.60                    8.50\n",
      "   16  10263  34G  35A C   N     238  240  148.27 -134.00   92.34    2.22  -62.50 -167.09   38.94\n",
      "   16         35A  35A N   CA    240  241 -163.16  147.00                  -40.90\n",
      "   17  10264  35A  36Y C   N     243  245  -92.80  -98.40   49.23    5.23  -63.50  126.34   18.64\n",
      "   17         36Y  36Y N   CA    245  246   79.49  128.40                  -43.40\n",
      "   18  10265  36Y  37G C   N     255  257 -178.51 -167.20   18.77    0.74  -80.20   99.37    6.99\n",
      "   18         37G  37G N   CA    257  258  159.61  174.60                  174.10\n",
      "   19  10558 329F 330K C   N    2648 2650 -110.60 -118.00   28.04    1.24  -62.90  160.13   18.76\n",
      "   19        330K 330K N   CA   2650 2651  112.06  139.10                  -40.80\n",
      "   20  10559 330K 331F C   N    2657 2659 -136.06 -124.20   12.65    0.61  -63.20 -168.79   31.32\n",
      "   20        331F 331F N   CA   2659 2660  138.92  143.30                  -44.30\n",
      "   21  10560 331F 332D C   N    2668 2670 -105.97  -96.50   14.69    0.62  -63.30  170.85   18.91\n",
      "   21        332D 332D N   CA   2670 2671  125.44  114.20                  -40.00\n",
      "   22  10561 332D 333M C   N    2676 2678  -65.08  -73.00    9.38    0.75  -63.40  171.49   25.73\n",
      "   22        333M 333M N   CA   2678 2679  148.02  143.00                  -40.50\n",
      "   23  10562 333M 334E C   N    2684 2686 -142.75 -117.80   32.40    0.99  -63.60 -179.48   28.49\n",
      "   23        334E 334E N   CA   2686 2687  157.46  136.80                  -40.30\n",
      "   24  10563 334E 335L C   N    2693 2695  -90.89  -70.70   58.57    3.77  -63.50  125.26   19.06\n",
      "   24        335L 335L N   CA   2695 2696 -163.43  141.60                  -41.20\n",
      "   25  10564 335L 336D C   N    2701 2703  -78.07  -63.30   75.98    8.59  -63.30   75.98    8.59\n",
      "   25        336D 336D N   CA   2703 2704   34.54  -40.00                  -40.00\n",
      "   26  10566 337D 338L C   N    2717 2719  -60.98  -70.70   22.63    1.42   60.20  151.88    9.86\n",
      "   26        338L 338L N   CA   2719 2720  121.16  141.60                   29.60\n",
      "   27  10568 339P 340K C   N    2732 2734  -72.65  -62.90   97.20   12.12  -62.90   97.20   12.12\n",
      "   27        340K 340K N   CA   2734 2735   55.91  -40.80                  -40.80\n",
      "   28  10585 356P 357G C   N    2880 2882   88.70   78.70   19.57    0.30  -80.20  168.93   11.35\n",
      "   28        357G 357G N   CA   2882 2883  177.08 -166.10                  174.10\n",
      "   29  10586 357G 358Y C   N    2884 2886 -119.12 -124.30   10.93    0.80  -63.50 -179.64   31.43\n",
      "   29        358Y 358Y N   CA   2886 2887  145.03  135.40                  -43.40\n",
      "   30  10587 358Y 359R C   N    2896 2898  -78.16  -72.10   13.99    0.95  -63.00  165.09   23.34\n",
      "   30        359R 359R N   CA   2898 2899  154.51  141.90                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    1    4   11   38   84  128  210  237  279  380  426  495\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "2z7l_MODLL.B99990001.pdb       296.30054\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 3c9w\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    37421    34775\n",
      "236 (of 2915 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2849 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     236\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    34775    1575\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      651\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :        3139.6865\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     244       9     14   0.079   0.079      470.53       1.000\n",
      " 2 Bond angle potential               :     336      13     25   6.184   6.184      262.99       1.000\n",
      " 3 Stereochemical cosine torsion poten:     163       0     13  56.481  56.481      74.158       1.000\n",
      " 4 Stereochemical improper torsion pot:     110       3      7   5.489   5.489      60.943       1.000\n",
      " 5 Soft-sphere overlap restraints     :     651      17     29   0.090   0.090      627.32       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     208      25     35   2.005   2.005      685.44       1.000\n",
      "10 Distance restraints 2 (N-O)        :     182      20     30   2.081   2.081      584.16       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      33       2      7  14.660  14.660      83.637       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      21       0      3  90.610  90.610      24.073       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      16       0      1  88.413  88.413      10.297       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       7       0      0  75.781  75.781      3.4876       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       2       0      0  42.766  42.766      1.5056       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     116       0      0   0.659   0.659      1.4829       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      34      25     15  70.398 148.051      185.71       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     103       5      6   2.420   2.420      63.953       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 3c9w_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    9803.4043\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  1                           : Bond length potential                   \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   1528 187Y 188V C   N    1496 1498    1.48    1.35    0.13    4.66    1.35    0.13    4.66\n",
      "    2   1534 188V 188V C   CA   1503 1499    1.78    1.49    0.29    8.36    1.49    0.29    8.36\n",
      "    3   1535 188V 189A C   N    1503 1505    1.71    1.35    0.37   13.02    1.35    0.37   13.02\n",
      "    4   1537 189A 189A N   CA   1505 1506    1.92    1.43    0.49   16.20    1.43    0.49   16.20\n",
      "    5   1539 189A 189A C   CA   1508 1506    2.09    1.49    0.60   17.33    1.49    0.60   17.33\n",
      "    6   1540 189A 190T C   N    1508 1510    1.87    1.35    0.53   18.67    1.35    0.53   18.67\n",
      "    7   1544 190T 190T N   CA   1510 1511    1.92    1.43    0.49   16.20    1.43    0.49   16.20\n",
      "    8   1546 190T 190T C   CA   1515 1511    1.74    1.49    0.25    7.15    1.49    0.25    7.15\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   5056 188V 188V N   CA   1498 1499  137.22  107.00   30.22    8.69  107.00   30.22    8.69\n",
      "    2   5060 188V 188V CA  C    1499 1503  145.21  116.50   28.71    8.26  116.50   28.71    8.26\n",
      "    3   5061 188V 188V CG2 CB   1502 1500  143.74  114.00   29.74    6.98  114.00   29.74    6.98\n",
      "    4   5066 189A 189A N   CA   1505 1506  131.85  107.00   24.85    7.15  107.00   24.85    7.15\n",
      "    5   5068 189A 189A CA  C    1506 1508  138.76  116.50   22.26    6.40  116.50   22.26    6.40\n",
      "    6   5070 189A 190T C   N    1508 1510  142.60  120.00   22.60    5.13  120.00   22.60    5.13\n",
      "    7   5073 190T 190T N   CA   1510 1511  137.72  107.00   30.72    8.83  107.00   30.72    8.83\n",
      "    8   5077 190T 190T CA  C    1511 1515  132.55  116.50   16.05    4.62  116.50   16.05    4.62\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  9                           : Distance restraints 1 (CA-CA)           \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  12328  34G 190T CA  CA    237 1511   22.06   12.91    9.14   14.07   12.91    9.14   14.07\n",
      "    2  14405 113Y 190T CA  CA    883 1511   17.75   12.38    5.37    9.72   12.38    5.37    9.72\n",
      "    3  15694 149D 190T CA  CA   1194 1511   11.39    7.33    4.06    8.14    7.33    4.06    8.14\n",
      "    4  15732 150L 190T CA  CA   1202 1511   11.87    8.77    3.10    6.02    8.77    3.10    6.02\n",
      "    5  15772 151K 190T CA  CA   1210 1511   11.67    7.48    4.19    8.47    7.48    4.19    8.47\n",
      "    6  15807 152P 190T CA  CA   1219 1511   14.70   10.16    4.54    8.65   10.16    4.54    8.65\n",
      "    7  15843 153S 190T CA  CA   1226 1511   16.57    9.88    6.69   12.75    9.88    6.69   12.75\n",
      "    8  15870 154N 190T CA  CA   1232 1511   16.90   10.73    6.17   11.70   10.73    6.17   11.70\n",
      "    9  15893 155L 190T CA  CA   1240 1511   18.58   13.67    4.91    8.64   13.67    4.91    8.64\n",
      "   10  15980 167D 190T CA  CA   1331 1511   19.15   12.62    6.53    9.55   12.62    6.53    9.55\n",
      "   11  15987 168F 190T CA  CA   1339 1511   19.06   13.78    5.28    7.42   13.78    5.28    7.42\n",
      "   12  15992 169G 190T CA  CA   1350 1511   19.27   13.21    6.06    8.25   13.21    6.06    8.25\n",
      "   13  15998 170L 190T CA  CA   1354 1511   16.17   11.43    4.74    6.29   11.43    4.74    6.29\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 10                           : Distance restraints 2 (N-O)             \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  21451 149D 190T N   O    1193 1516   11.34    7.64    3.70    6.35    7.64    3.70    6.35\n",
      "    2  21482 150L 190T N   O    1201 1516   11.39    7.29    4.10    7.13    7.29    4.10    7.13\n",
      "    3  21509 151K 190T N   O    1209 1516   12.03    7.43    4.60    8.08    7.43    4.60    8.08\n",
      "    4  21539 152P 190T N   O    1218 1516   13.46    8.87    4.59    7.87    8.87    4.59    7.87\n",
      "    5  21569 153S 190T N   O    1225 1516   15.24   10.07    5.17    8.65   10.07    5.17    8.65\n",
      "    6  21589 154N 190T N   O    1231 1516   15.99   10.60    5.39    9.05   10.60    5.39    9.05\n",
      "    7  22071 190T 147H N   O    1510 1181   15.39   10.09    5.30    8.51   10.09    5.30    8.51\n",
      "    8  22072 190T 148R N   O    1510 1192   11.38    6.87    4.51    7.80    6.87    4.51    7.80\n",
      "    9  22073 190T 149D N   O    1510 1200   13.96    6.18    7.77   13.71    6.18    7.77   13.71\n",
      "   10  22074 190T 150L N   O    1510 1208   11.32    6.38    4.93    8.72    6.38    4.93    8.72\n",
      "   11  22075 190T 151K N   O    1510 1217   15.64    8.47    7.17   12.31    8.47    7.17   12.31\n",
      "   12  22076 190T 152P N   O    1510 1224   18.50   10.78    7.72   12.64   10.78    7.72   12.64\n",
      "   13  22077 190T 153S N   O    1510 1230   19.94   10.32    9.62   16.01   10.32    9.62   16.01\n",
      "   14  22078 190T 167D N   O    1510 1337   18.55   10.01    8.53   12.61   10.01    8.53   12.61\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 13                           : Mainchain Omega dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10774 187Y 187Y CA  C    1487 1496 -148.11 -180.00   31.89    6.38 -180.00   31.89    6.38\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9 -111.90 -134.00   22.17    0.67  -62.50 -179.31   32.23\n",
      "    1          2A   2A N   CA      9   10  145.29  147.00                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14  -69.55  -68.20    4.55    0.32  -62.50  169.61   28.15\n",
      "    2          3A   3A N   CA     14   15  149.64  145.30                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19 -118.59 -134.00   15.45    0.56  -62.50  179.94   32.48\n",
      "    3          4A   4A N   CA     19   20  148.13  147.00                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24 -109.21 -134.00   25.05    0.70  -62.50 -178.20   32.27\n",
      "    4          5A   5A N   CA     24   25  143.41  147.00                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29 -114.47 -134.00   19.63    0.58  -62.50 -178.37   32.53\n",
      "    5          6A   6A N   CA     29   30  145.06  147.00                  -40.90\n",
      "    6  10235   6A   7A C   N      32   34 -118.57 -134.00   15.54    0.59  -62.50  179.19   32.36\n",
      "    6          7A   7A N   CA     34   35  148.91  147.00                  -40.90\n",
      "    7  10237   8G   9A C   N      41   43   69.38  -68.20  145.42   13.70  -62.50 -177.14   28.39\n",
      "    7          9A   9A N   CA     43   44 -167.58  145.30                  -40.90\n",
      "    8  10404 175D 176P C   N    1395 1397  -75.40  -58.70   17.53    1.90  -64.50  172.73   13.47\n",
      "    8        176P 176P N   CA   1397 1398  -25.18  -30.50                  147.20\n",
      "    9  10405 176P 177D C   N    1402 1404   52.94   54.50    5.08    0.27  -63.30  144.44   24.51\n",
      "    9        177D 177D N   CA   1404 1405   45.73   40.90                  -40.00\n",
      "   10  10406 177D 178H C   N    1410 1412  -63.10  -67.60    4.90    0.42  -63.20  175.76   21.97\n",
      "   10        178H 178H N   CA   1412 1413  141.94  140.00                  -42.30\n",
      "   11  10407 178H 179D C   N    1420 1422 -165.37  -70.90  106.38    5.86  -63.30  158.13   25.85\n",
      "   11        179D 179D N   CA   1422 1423 -160.78  150.30                  -40.00\n",
      "   12  10408 179D 180H C   N    1428 1430 -118.15 -125.60   53.22    1.88  -63.20  139.67   14.98\n",
      "   12        180H 180H N   CA   1430 1431   86.10  138.80                  -42.30\n",
      "   13  10409 180H 181T C   N    1438 1440  136.37 -124.80  141.23    8.46  -63.20 -178.57   22.78\n",
      "   13        181T 181T N   CA   1440 1441   42.62  143.50                  -42.10\n",
      "   14  10410 181T 182G C   N    1445 1447 -131.52 -167.20   36.90    0.95   82.20 -131.53   17.17\n",
      "   14        182G 182G N   CA   1447 1448 -175.99  174.60                    8.50\n",
      "   15  10411 182G 183F C   N    1449 1451  -97.72 -124.20   36.36    1.99  -63.20  151.47   23.35\n",
      "   15        183F 183F N   CA   1451 1452  168.22  143.30                  -44.30\n",
      "   16  10412 183F 184L C   N    1460 1462 -139.54 -108.50   38.84    1.72  -63.50  179.82   29.67\n",
      "   16        184L 184L N   CA   1462 1463  155.84  132.50                  -41.20\n",
      "   17  10413 184L 185T C   N    1468 1470 -113.51  -63.20   84.59    8.97  -63.20   84.59    8.97\n",
      "   17        185T 185T N   CA   1470 1471   25.91  -42.10                  -42.10\n",
      "   18  10414 185T 186E C   N    1475 1477   78.35  -63.60  165.03   21.45  -63.60  165.03   21.45\n",
      "   18        186E 186E N   CA   1477 1478 -124.48  -40.30                  -40.30\n",
      "   19  10415 186E 187Y C   N    1484 1486  153.02 -124.30   98.99    5.15  -63.50 -168.53   36.23\n",
      "   19        187Y 187Y N   CA   1486 1487 -170.18  135.40                  -43.40\n",
      "   20  10417 188V 189A C   N    1503 1505  -92.73  -68.20   80.50    7.31  -62.50  113.62   17.40\n",
      "   20        189A 189A N   CA   1505 1506   68.62  145.30                  -40.90\n",
      "   21  10418 189A 190T C   N    1508 1510   75.99  -78.10  154.13    9.42  -78.10  154.13    9.42\n",
      "   21        190T 190T N   CA   1510 1511  153.48  149.80                  149.80\n",
      "   22  10419 190T 191R C   N    1515 1517 -102.93  -63.00   72.03   12.38  -63.00   72.03   12.38\n",
      "   22        191R 191R N   CA   1517 1518 -101.05  -41.10                  -41.10\n",
      "   23  10585 356P 357G C   N    2880 2882 -129.13 -167.20   41.38    0.61   82.20 -148.91   16.18\n",
      "   23        357G 357G N   CA   2882 2883  158.37  174.60                    8.50\n",
      "   24  10586 357G 358Y C   N    2884 2886 -120.44 -124.30    4.23    0.35  -63.50 -171.71   32.76\n",
      "   24        358Y 358Y N   CA   2886 2887  137.13  135.40                  -43.40\n",
      "   25  10587 358Y 359R C   N    2896 2898 -112.71 -125.20   13.34    0.65  -63.00 -179.41   27.83\n",
      "   25        359R 359R N   CA   2898 2899  145.28  140.60                  -41.10\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 26                           : Distance restraints 4 (SDCH-SDCH)       \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  33335 151K 190T CD  CB   1213 1512   11.86    4.55    7.32    4.63    4.55    7.32    4.63\n",
      "    2  33336 151K 190T CD  OG1  1213 1513   12.47    4.13    8.34    5.29    4.13    8.34    5.29\n",
      "    3  33344 151K 190T CE  OG1  1214 1513   13.45    4.50    8.95    5.66    4.50    8.95    5.66\n",
      "    4  33347 151K 190T NZ  CB   1215 1512   12.92    4.47    8.45    5.35    4.47    8.45    5.35\n",
      "    5  33348 151K 190T NZ  OG1  1215 1513   13.72    3.63   10.09    6.43    3.63   10.09    6.43\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:  1505 1573   2.050\n",
      "  serious non-bonded atom clash:  1506 1573   1.840\n",
      "  serious non-bonded atom clash:  1510 1599   2.265\n",
      "  serious non-bonded atom clash:  1510 1927   2.221\n",
      "  serious non-bonded atom clash:  1511 1564   2.018\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     3    0    2    0    5   16   40   86  128  170  217  278  368  460  485\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "3c9w_MODLL.B99990001.pdb      3139.68652\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 3erk\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    38758    36112\n",
      "88 (of 2915 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2694 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915      88\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    36112     524\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      108\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          89.8118\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      90       0      0   0.003   0.003     0.24237       1.000\n",
      " 2 Bond angle potential               :     123       0      0   1.663   1.663      6.5940       1.000\n",
      " 3 Stereochemical cosine torsion poten:      53       0      1  53.178  53.178      22.830       1.000\n",
      " 4 Stereochemical improper torsion pot:      36       0      0   0.958   0.958      1.0042       1.000\n",
      " 5 Soft-sphere overlap restraints     :     108       0      0   0.005   0.005     0.31237       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      71       0      0   0.209   0.209      2.6824       1.000\n",
      "10 Distance restraints 2 (N-O)        :      74       0      0   0.479   0.479      7.8217       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      14       0      0   2.832   2.832      1.3242       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       5       0      0 102.110 102.110      2.7293       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       4       0      0  83.030  83.030      1.9255       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       3       0      0  95.556  95.556      2.4952       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       1       0      0 124.236 124.236     0.81668       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      36       0      0   0.345   0.345     0.11102       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      14      11      2  46.703 147.409      38.923       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 3erk_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    2948.1338\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9 -129.97 -134.00    4.66    0.24  -62.50 -177.32   33.55\n",
      "    1          2A   2A N   CA      9   10  149.34  147.00                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14  -67.41  -68.20    2.08    0.14  -62.50  175.79   29.05\n",
      "    2          3A   3A N   CA     14   15  143.38  145.30                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19 -134.93 -134.00    6.95    0.37  -62.50 -179.61   33.44\n",
      "    3          4A   4A N   CA     19   20  153.89  147.00                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24  -72.73  -68.20    6.72    0.42  -62.50  169.15   28.23\n",
      "    4          5A   5A N   CA     24   25  150.27  145.30                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29  -87.38  -68.20   22.91    2.25  -62.50  175.44   27.63\n",
      "    5          6A   6A N   CA     29   30  132.76  145.30                  -40.90\n",
      "    6  10236   7A   8G C   N      37   39  -55.77  -62.40   13.90    2.43  -80.20  158.81    7.49\n",
      "    6          8G   8G N   CA     39   40  -28.99  -41.20                  174.10\n",
      "    7  10237   8G   9A C   N      41   43   74.36  -68.20  154.62   14.78  -62.50  178.06   28.12\n",
      "    7          9A   9A N   CA     43   44 -154.81  145.30                  -40.90\n",
      "    8  10238   9A  10G C   N      46   48  -57.84  -62.40   60.68    8.85  -62.40   60.68    8.85\n",
      "    8         10G  10G N   CA     48   49 -101.71  -41.20                  -41.20\n",
      "    9  10585 356P 357G C   N    2880 2882   57.53   78.70   23.98    0.83  -80.20  141.19   10.14\n",
      "    9        357G 357G N   CA   2882 2883 -154.85 -166.10                  174.10\n",
      "   10  10586 357G 358Y C   N    2884 2886 -127.32 -124.30    5.49    0.22  -63.50 -172.21   33.01\n",
      "   10        358Y 358Y N   CA   2886 2887  139.98  135.40                  -43.40\n",
      "   11  10587 358Y 359R C   N    2896 2898 -150.66 -125.20   34.18    1.02  -63.00  178.51   30.02\n",
      "   11        359R 359R N   CA   2898 2899  163.40  140.60                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:    64  124   2.272\n",
      "  serious non-bonded atom clash:  2669 2691   2.297\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    2    2    9   19   36   65  120  190  237  313  373  430  470\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "3erk_MODLL.B99990001.pdb        89.81176\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 3i5z\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    36667    34021\n",
      "303 (of 2915 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2732 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     303\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    34021    2233\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      512\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         178.7359\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     313       0      0   0.003   0.003     0.75633       1.000\n",
      " 2 Bond angle potential               :     429       0      0   1.493   1.493      17.803       1.000\n",
      " 3 Stereochemical cosine torsion poten:     201       0      6  48.637  48.637      72.135       1.000\n",
      " 4 Stereochemical improper torsion pot:     138       0      0   0.877   0.877      2.6361       1.000\n",
      " 5 Soft-sphere overlap restraints     :     512       0      0   0.002   0.002     0.19463       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     374       0      0   0.154   0.154      6.1600       1.000\n",
      "10 Distance restraints 2 (N-O)        :     334       0      0   0.360   0.360      18.782       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      43       0      0   2.643   2.643      3.5430       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      28       0      0  76.159  76.159      3.7237       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      21       0      0  73.889  73.889      10.766       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       9       0      0  70.727  70.727      4.2104       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       4       0      0  56.056  56.056      2.5566       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     142       0      0   0.395   0.395     0.48154       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      45      27      0  21.818 130.344      19.804       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     152       0      0   0.937   0.937      15.184       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 3i5z_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    7421.3076\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9  -63.31  -68.20    5.73    0.38  -62.50  176.78   29.01\n",
      "    1          2A   2A N   CA      9   10  142.32  145.30                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14  -75.52  -68.20    7.66    0.57  -62.50  172.04   28.85\n",
      "    2          3A   3A N   CA     14   15  147.55  145.30                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19  -76.43  -68.20   13.60    0.86  -62.50  163.57   27.51\n",
      "    3          4A   4A N   CA     19   20  156.13  145.30                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24  -62.03  -68.20    7.30    0.48  -62.50  177.72   29.10\n",
      "    4          5A   5A N   CA     24   25  141.38  145.30                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29  -71.91  -68.20    3.71    0.31  -62.50  174.02   28.99\n",
      "    5          6A   6A N   CA     29   30  145.34  145.30                  -40.90\n",
      "    6  10235   6A   7A C   N      32   34 -138.43 -134.00    6.28    0.19  -62.50 -175.95   34.23\n",
      "    6          7A   7A N   CA     34   35  151.45  147.00                  -40.90\n",
      "    7  10236   7A   8G C   N      37   39  -61.32  -62.40    1.68    0.23  -80.20  144.65    6.65\n",
      "    7          8G   8G N   CA     39   40  -42.49  -41.20                  174.10\n",
      "    8  10402 173V 174A C   N    1382 1384  170.43 -134.00   60.00    1.37  -62.50 -163.80   38.74\n",
      "    8        174A 174A N   CA   1384 1385  169.62  147.00                  -40.90\n",
      "    9  10403 174A 175D C   N    1387 1389 -124.10  -96.50   33.17    1.35  -63.30  148.79   15.85\n",
      "    9        175D 175D N   CA   1389 1390   95.81  114.20                  -40.00\n",
      "   10  10404 175D 176P C   N    1395 1397  -70.93  -58.70   15.08    1.24  -64.50  169.00   12.95\n",
      "   10        176P 176P N   CA   1397 1398  -21.68  -30.50                  147.20\n",
      "   11  10405 176P 177D C   N    1402 1404  -82.55  -70.90   27.36    0.92  -63.30  146.22   19.26\n",
      "   11        177D 177D N   CA   1404 1405  175.05  150.30                  -40.00\n",
      "   12  10406 177D 178H C   N    1410 1412  -63.41  -67.60    5.03    0.45  -63.20  174.92   21.88\n",
      "   12        178H 178H N   CA   1412 1413  142.78  140.00                  -42.30\n",
      "   13  10407 178H 179D C   N    1420 1422 -101.51  -96.50    5.93    0.25  -63.30  161.94   18.01\n",
      "   13        179D 179D N   CA   1422 1423  117.36  114.20                  -40.00\n",
      "   14  10408 179D 180H C   N    1428 1430  -75.89  -67.60   14.71    0.90  -63.20  166.04   21.56\n",
      "   14        180H 180H N   CA   1430 1431  152.15  140.00                  -42.30\n",
      "   15  10410 181T 182G C   N    1445 1447 -134.83 -167.20   45.88    1.03   82.20 -164.33   15.09\n",
      "   15        182G 182G N   CA   1447 1448  142.09  174.60                    8.50\n",
      "   16  10411 182G 183F C   N    1449 1451 -122.45 -124.20   26.68    1.30  -63.20  157.35   25.73\n",
      "   16        183F 183F N   CA   1451 1452  169.93  143.30                  -44.30\n",
      "   17  10413 184L 185T C   N    1468 1470  -80.45  -78.10   14.77    0.56  -63.20  154.49   20.96\n",
      "   17        185T 185T N   CA   1470 1471  164.38  149.80                  -42.10\n",
      "   18  10414 185T 186E C   N    1475 1477 -119.37 -117.80   22.21    1.09  -63.60  170.15   25.84\n",
      "   18        186E 186E N   CA   1477 1478  158.95  136.80                  -40.30\n",
      "   19  10415 186E 187Y C   N    1484 1486  -68.01  -98.40   30.41    1.19  -63.50  170.77   26.86\n",
      "   19        187Y 187Y N   CA   1486 1487  127.30  128.40                  -43.40\n",
      "   20  10429 200L 201N C   N    1608 1610   51.74   55.90   17.16    0.92  -63.20  150.56   24.57\n",
      "   20        201N 201N N   CA   1610 1611   56.15   39.50                  -41.10\n",
      "   21  10430 201N 202S C   N    1616 1618  -79.62  -72.40   25.04    1.25  -64.10  149.43   11.82\n",
      "   21        202S 202S N   CA   1618 1619  176.37  152.40                  -35.00\n",
      "   22  10431 202S 203K C   N    1622 1624  -77.67  -70.20   13.65    0.83  -62.90  168.03   22.66\n",
      "   22        203K 203K N   CA   1624 1625  151.82  140.40                  -40.80\n",
      "   23  10432 203K 204G C   N    1631 1633   85.91   78.70   23.84    0.41   82.20  162.72    8.09\n",
      "   23        204G 204G N   CA   1633 1634  171.18 -166.10                    8.50\n",
      "   24  10433 204G 205Y C   N    1635 1637 -123.60 -124.30    9.01    0.51  -63.50 -177.60   31.97\n",
      "   24        205Y 205Y N   CA   1637 1638  144.38  135.40                  -43.40\n",
      "   25  10585 356P 357G C   N    2880 2882  141.03 -167.20   51.85    0.92  -80.20  138.81    9.15\n",
      "   25        357G 357G N   CA   2882 2883  177.50  174.60                  174.10\n",
      "   26  10586 357G 358Y C   N    2884 2886 -129.82 -124.30   27.11    1.22  -63.50  168.28   29.99\n",
      "   26        358Y 358Y N   CA   2886 2887  161.94  135.40                  -43.40\n",
      "   27  10587 358Y 359R C   N    2896 2898  -64.74  -72.10    7.38    0.57  -63.00  176.52   23.96\n",
      "   27        359R 359R N   CA   2898 2899  142.39  141.90                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    1    6   23   54   94  141  191  286  334  409  490\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "3i5z_MODLL.B99990001.pdb       178.73592\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 3i60\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    37097    34451\n",
      "268 (of 2915 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2759 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:       91.2337\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     268\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    34451    2001\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      569\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         290.5158\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     277       0      0   0.007   0.007      3.5728       1.000\n",
      " 2 Bond angle potential               :     382       0      6   2.694   2.694      52.697       1.000\n",
      " 3 Stereochemical cosine torsion poten:     181       0      4  46.974  46.974      61.033       1.000\n",
      " 4 Stereochemical improper torsion pot:     129       0      0   1.619   1.619      7.5158       1.000\n",
      " 5 Soft-sphere overlap restraints     :     569       0      0   0.005   0.005      1.9537       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     330       0      0   0.280   0.280      13.762       1.000\n",
      "10 Distance restraints 2 (N-O)        :     292       0      0   0.474   0.474      25.097       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      39       0      4   7.321   7.321      24.650       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      24       0      0  85.712  85.712      11.946       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      17       0      0  80.790  80.790      8.5561       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      0 107.319 107.319      2.4984       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       2       0      0   6.884   6.884     0.83747       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     150       0      0   0.797   0.797      2.4531       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      41      22     10  36.046 127.493      60.009       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     132       0      0   1.604   1.604      13.934       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 3i60_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    6749.2734\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9  -69.73  -68.20    5.52    0.39  -62.50  168.66   28.00\n",
      "    1          2A   2A N   CA      9   10  150.60  145.30                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14  -69.29  -68.20    2.13    0.14  -62.50  172.10   28.54\n",
      "    2          3A   3A N   CA     14   15  147.13  145.30                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19  -73.79  -68.20   11.47    0.75  -62.50  164.18   27.47\n",
      "    3          4A   4A N   CA     19   20  155.31  145.30                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24  -68.48  -68.20    7.74    0.61  -62.50  166.17   27.53\n",
      "    4          5A   5A N   CA     24   25  153.04  145.30                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29  -62.86  -68.20    9.35    0.59  -62.50  178.53   29.24\n",
      "    5          6A   6A N   CA     29   30  137.63  145.30                  -40.90\n",
      "    6  10235   6A   7A C   N      32   34  -80.53  -68.20   19.05    1.20  -62.50  160.29   27.19\n",
      "    6          7A   7A N   CA     34   35  159.83  145.30                  -40.90\n",
      "    7  10236   7A   8G C   N      37   39   88.85   82.20    9.03    0.30  -80.20 -119.04    7.58\n",
      "    7          8G   8G N   CA     39   40    2.39    8.50                  174.10\n",
      "    8  10404 175D 176P C   N    1395 1397  -68.03  -58.70   36.88    2.38  -64.50  142.06   10.79\n",
      "    8        176P 176P N   CA   1397 1398    5.19  -30.50                  147.20\n",
      "    9  10406 177D 178H C   N    1410 1412 -126.35 -125.60   52.24    1.97  -63.20  141.54   21.80\n",
      "    9        178H 178H N   CA   1412 1413 -168.97  138.80                  -42.30\n",
      "   10  10407 178H 179D C   N    1420 1422  -55.34  -70.90   31.18    1.05  -63.30  163.47   20.67\n",
      "   10        179D 179D N   CA   1422 1423  123.27  150.30                  -40.00\n",
      "   11  10408 179D 180H C   N    1428 1430  172.16 -125.60   65.42    1.87  -63.20 -158.16   33.04\n",
      "   11        180H 180H N   CA   1430 1431  158.94  138.80                  -42.30\n",
      "   12  10409 180H 181T C   N    1438 1440  -98.29 -124.80   87.77    3.03  -63.20  107.80   11.87\n",
      "   12        181T 181T N   CA   1440 1441   59.83  143.50                  -42.10\n",
      "   13  10411 182G 183F C   N    1449 1451 -122.57 -124.20   41.71    2.01  -63.20  143.57   23.79\n",
      "   13        183F 183F N   CA   1451 1452 -175.02  143.30                  -44.30\n",
      "   14  10412 183F 184L C   N    1460 1462  -89.03  -70.70   33.31    2.06  -63.50  151.55   22.59\n",
      "   14        184L 184L N   CA   1462 1463  169.41  141.60                  -41.20\n",
      "   15  10413 184L 185T C   N    1468 1470  -62.65  -78.10   16.20    1.06  -63.20  163.23   20.90\n",
      "   15        185T 185T N   CA   1470 1471  154.67  149.80                  -42.10\n",
      "   16  10414 185T 186E C   N    1475 1477  -99.95 -117.80   42.21    2.31  -63.60  149.15   21.94\n",
      "   16        186E 186E N   CA   1477 1478  175.05  136.80                  -40.30\n",
      "   17  10415 186E 187Y C   N    1484 1486 -111.66 -124.30   12.90    1.04  -63.50 -175.04   31.80\n",
      "   17        187Y 187Y N   CA   1486 1487  138.02  135.40                  -43.40\n",
      "   18  10416 187Y 188V C   N    1496 1498 -159.03  -62.40   98.33   14.65 -125.40  170.84    9.21\n",
      "   18        188V 188V N   CA   1498 1499  -24.19  -42.40                  143.30\n",
      "   19  10429 200L 201N C   N    1608 1610  148.84 -119.90   91.50    3.63   55.90  139.58   16.21\n",
      "   19        201N 201N N   CA   1610 1611  143.64  137.00                   39.50\n",
      "   20  10431 202S 203K C   N    1622 1624  -53.41  -70.20   26.95    1.62  -62.90  160.40   21.36\n",
      "   20        203K 203K N   CA   1624 1625  119.32  140.40                  -40.80\n",
      "   21  10432 203K 204G C   N    1631 1633   95.34   78.70   34.20    0.51   82.20  156.07    8.23\n",
      "   21        204G 204G N   CA   1633 1634  164.02 -166.10                    8.50\n",
      "   22  10587 358Y 359R C   N    2896 2898 -113.11 -125.20   13.59    0.69  -63.00  179.26   27.68\n",
      "   22        359R 359R N   CA   2898 2899  146.79  140.60                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    2   10   20   64  100  176  191  290  331  423  492\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "3i60_MODLL.B99990001.pdb       290.51578\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 3o71\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    37612    34966\n",
      "229 (of 2915 total) atoms selected for optimization\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:       90.7647\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     229\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    34966    1814\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      408\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         203.2170\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     239       0      0   0.005   0.005      1.4241       1.000\n",
      " 2 Bond angle potential               :     329       0      2   2.166   2.166      29.155       1.000\n",
      " 3 Stereochemical cosine torsion poten:     159       0      2  46.090  46.090      45.617       1.000\n",
      " 4 Stereochemical improper torsion pot:     106       0      1   1.377   1.377      6.2282       1.000\n",
      " 5 Soft-sphere overlap restraints     :     408       0      0   0.003   0.003     0.46316       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     337       0      0   0.176   0.176      7.1433       1.000\n",
      "10 Distance restraints 2 (N-O)        :     305       1      1   0.432   0.432      24.189       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      34       0      1   4.322   4.322      6.9391       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      20       0      0  88.135  88.135      6.8065       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      17       0      0  73.627  73.627      5.5087       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       8       0      0  70.941  70.941      6.0676       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       2       0      0 124.984 124.984      2.1437       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     151       0      0   0.431   0.431     0.62872       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      36      15      9  29.661 109.086      58.087       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      71       0      0   0.837   0.837      2.8153       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 3o71_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    4987.3579\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 10                           : Distance restraints 2 (N-O)             \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  25075 334E 176P N   O    2686 1403   14.62   10.65    3.98    5.86   10.65    3.98    5.86\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9  -69.05  -68.20    2.56    0.18  -62.50  171.51   28.43\n",
      "    1          2A   2A N   CA      9   10  147.72  145.30                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14  -80.55  -68.20   19.42    1.22  -62.50  159.83   27.12\n",
      "    2          3A   3A N   CA     14   15  160.29  145.30                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19  -69.28  -68.20    5.57    0.41  -62.50  168.47   27.95\n",
      "    3          4A   4A N   CA     19   20  150.77  145.30                  -40.90\n",
      "    4  10234   5A   6A C   N      27   29  -79.42  -68.20   16.61    1.04  -62.50  162.43   27.48\n",
      "    4          6A   6A N   CA     29   30  157.55  145.30                  -40.90\n",
      "    5  10235   6A   7A C   N      32   34  -81.73  -68.20   23.52    1.49  -62.50  155.76   26.51\n",
      "    5          7A   7A N   CA     34   35  164.53  145.30                  -40.90\n",
      "    6  10236   7A   8G C   N      37   39 -128.64 -167.20   49.33    0.96   82.20 -158.61   15.57\n",
      "    6          8G   8G N   CA     39   40  143.82  174.60                    8.50\n",
      "    7  10237   8G   9A C   N      41   43 -104.36 -134.00   30.05    1.19  -62.50  172.31   30.45\n",
      "    7          9A   9A N   CA     43   44  151.95  147.00                  -40.90\n",
      "    8  10238   9A  10G C   N      46   48 -176.26 -167.20   27.44    1.13   82.20  173.10   12.75\n",
      "    8         10G  10G N   CA     48   49  148.69  174.60                    8.50\n",
      "    9  10406 177D 178H C   N    1410 1412 -118.93 -125.60   52.57    2.12  -63.20  138.47   20.95\n",
      "    9        178H 178H N   CA   1412 1413 -169.06  138.80                  -42.30\n",
      "   10  10407 178H 179D C   N    1420 1422  -55.20  -70.90   19.37    0.92  -63.30  179.13   22.61\n",
      "   10        179D 179D N   CA   1422 1423  138.95  150.30                  -40.00\n",
      "   11  10408 179D 180H C   N    1428 1430 -127.29 -125.60   31.70    1.17  -63.20  160.59   24.26\n",
      "   11        180H 180H N   CA   1430 1431  170.45  138.80                  -42.30\n",
      "   12  10409 180H 181T C   N    1438 1440  -77.77  -78.10   43.80    1.88  -63.20  125.15   17.02\n",
      "   12        181T 181T N   CA   1440 1441 -166.40  149.80                  -42.10\n",
      "   13  10430 201N 202S C   N    1616 1618 -115.14 -136.60   44.07    2.50  -64.10  144.62   14.08\n",
      "   13        202S 202S N   CA   1618 1619 -170.31  151.20                  -35.00\n",
      "   14  10586 357G 358Y C   N    2884 2886  -84.33  -98.40   14.07    0.54  -63.50  173.44   26.48\n",
      "   14        358Y 358Y N   CA   2886 2887  128.79  128.40                  -43.40\n",
      "   15  10587 358Y 359R C   N    2896 2898  -78.19  -72.10   20.75    1.46  -63.00  157.90   22.37\n",
      "   15        359R 359R N   CA   2898 2899  161.73  141.90                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    1    3   18   33   98  164  198  250  340  420  486\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2773 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "3o71_MODLL.B99990001.pdb       203.21703\n",
      "\n",
      "Modelando proteína 3qyw\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    37306    34660\n",
      "230 (of 2915 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2766 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     230\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    34660    1852\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      424\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         172.5613\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     236       0      0   0.005   0.005      1.4777       1.000\n",
      " 2 Bond angle potential               :     325       0      0   2.081   2.081      25.650       1.000\n",
      " 3 Stereochemical cosine torsion poten:     139       0      3  50.369  50.369      53.954       1.000\n",
      " 4 Stereochemical improper torsion pot:      92       0      0   1.483   1.483      4.4695       1.000\n",
      " 5 Soft-sphere overlap restraints     :     424       0      0   0.003   0.003     0.49959       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     298       0      0   0.135   0.135      4.9393       1.000\n",
      "10 Distance restraints 2 (N-O)        :     283       0      0   0.246   0.246      8.4056       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      34       0      0   2.965   2.965      1.1975       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      22       0      0  63.046  63.046      5.8194       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      18       0      0  86.387  86.387      10.060       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       9       0      0  95.584  95.584      6.9310       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       2       0      0 105.765 105.765      1.7238       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     235       0      0   0.504   0.504      1.4349       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      36      18      6  25.643 120.603      38.874       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     123       0      0   0.727   0.727      7.1250       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 3qyw_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    5279.7812\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9  -73.54  -68.20   32.71    2.45  -62.50  141.96   23.82\n",
      "    1          2A   2A N   CA      9   10  177.57  145.30                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14  -63.83  -68.20   12.79    1.17  -62.50  161.78   26.58\n",
      "    2          3A   3A N   CA     14   15  157.32  145.30                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19  -66.30  -68.20    3.76    0.24  -62.50  177.08   29.21\n",
      "    3          4A   4A N   CA     19   20  142.06  145.30                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24  -70.57  -68.20    3.08    0.30  -62.50  175.96   29.24\n",
      "    4          5A   5A N   CA     24   25  143.33  145.30                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29 -110.14 -134.00   23.96    0.71  -62.50 -179.31   32.14\n",
      "    5          6A   6A N   CA     29   30  144.81  147.00                  -40.90\n",
      "    6  10235   6A   7A C   N      32   34  -65.75  -68.20    8.20    0.58  -62.50  178.40   29.08\n",
      "    6          7A   7A N   CA     34   35  137.48  145.30                  -40.90\n",
      "    7  10236   7A   8G C   N      37   39   72.10   78.70   15.07    0.69   82.20  172.15    7.85\n",
      "    7          8G   8G N   CA     39   40 -179.65 -166.10                    8.50\n",
      "    8  10237   8G   9A C   N      41   43 -144.49 -134.00   15.39    0.91  -62.50 -165.26   29.23\n",
      "    8          9A   9A N   CA     43   44  135.74  147.00                  -40.90\n",
      "    9  10238   9A  10G C   N      46   48   60.02   78.70   24.20    0.64   82.20  160.75    8.90\n",
      "    9         10G  10G N   CA     48   49 -150.71 -166.10                    8.50\n",
      "   10  10389 160T 161C C   N    1283 1285  118.66   57.40   63.44    6.62 -117.90  173.25    7.65\n",
      "   10        161C 161C N   CA   1285 1286   19.53   36.00                  141.10\n",
      "   11  10560 331F 332D C   N    2668 2670 -124.16  -96.50   31.66    1.31  -63.30 -179.82   19.44\n",
      "   11        332D 332D N   CA   2670 2671  129.60  114.20                  -40.00\n",
      "   12  10561 332D 333M C   N    2676 2678  -57.65  -73.00   17.85    1.43  -63.40  167.50   24.57\n",
      "   12        333M 333M N   CA   2678 2679  152.10  143.00                  -40.50\n",
      "   13  10562 333M 334E C   N    2684 2686 -100.48 -117.80   40.66    1.63  -63.60  145.08   17.59\n",
      "   13        334E 334E N   CA   2686 2687  100.02  136.80                  -40.30\n",
      "   14  10565 336D 337D C   N    2709 2711  -64.05  -63.30    0.80    0.11   54.50  143.35   18.49\n",
      "   14        337D 337D N   CA   2711 2712  -39.71  -40.00                   40.90\n",
      "   15  10584 355Q 356P C   N    2873 2875  -71.92  -58.70   22.48    1.39  -64.50  159.69   12.31\n",
      "   15        356P 356P N   CA   2875 2876  -12.32  -30.50                  147.20\n",
      "   16  10585 356P 357G C   N    2880 2882  125.71   78.70   49.08    2.06   82.20  176.71   10.69\n",
      "   16        357G 357G N   CA   2882 2883  179.78 -166.10                    8.50\n",
      "   17  10586 357G 358Y C   N    2884 2886  -83.30  -98.40   18.54    1.22  -63.50  178.54   29.35\n",
      "   17        358Y 358Y N   CA   2886 2887  139.16  128.40                  -43.40\n",
      "   18  10587 358Y 359R C   N    2896 2898  -93.27  -72.10   35.06    2.29  -63.00  152.10   22.64\n",
      "   18        359R 359R N   CA   2898 2899  169.85  141.90                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    0    4   10   34  122  206  174  233  335  430  497\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "3qyw_MODLL.B99990001.pdb       172.56131\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 3qyz\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "    337     1  330   336      D     L   13.211\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    37554    34908\n",
      "238 (of 2915 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2828 atoms and 1 coordinate set(s) were parsed in 0.03s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     238\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    34908    1856\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      534\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :        1686.1497\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     244       7      8   0.061   0.061      261.14       1.000\n",
      " 2 Bond angle potential               :     335       8     10   4.585   4.585      127.73       1.000\n",
      " 3 Stereochemical cosine torsion poten:     142       0      8  55.463  55.463      64.065       1.000\n",
      " 4 Stereochemical improper torsion pot:      94       0      1   1.958   1.958      6.3315       1.000\n",
      " 5 Soft-sphere overlap restraints     :     534      13     17   0.079   0.079      388.87       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     317      19     25   1.410   1.410      457.74       1.000\n",
      "10 Distance restraints 2 (N-O)        :     293      10     18   1.385   1.385      249.37       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      35       0      1   4.760   4.760      7.7393       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      23       0      0  84.163  84.163      7.8443       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      19       0      0  44.679  44.679      5.3505       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:      10       0      0  65.340  65.340      6.6269       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       2       0      0  39.365  39.365      1.2911       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     186       0      0   0.645   0.645      1.9034       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      37      16      8  38.020 109.457      58.525       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     119       0      8   1.743   1.743      41.621       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 3qyz_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    6374.8584\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  1                           : Bond length potential                   \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   2783 337D 337D C   CA   2717 2712    1.91    1.49    0.42   12.23    1.49    0.42   12.23\n",
      "    2   2784 337D 338L C   N    2717 2719    1.66    1.35    0.32   11.25    1.35    0.32   11.25\n",
      "    3   2789 338L 338L N   CA   2719 2720    1.80    1.43    0.37   12.10    1.43    0.37   12.10\n",
      "    4   2791 338L 338L C   CA   2725 2720    1.93    1.49    0.44   12.83    1.49    0.44   12.83\n",
      "    5   2792 338L 339P C   N    2725 2727    1.75    1.30    0.45   13.32    1.30    0.45   13.32\n",
      "    6   2972 359R 359R NE  CD   2903 2902    1.75    1.49    0.26    7.67    1.49    0.26    7.67\n",
      "    7   2973 359R 359R CZ  NE   2904 2903    1.51    1.37    0.15    5.87    1.37    0.15    5.87\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6757 337D 337D CA  C    2712 2717  141.09  116.50   24.59    7.07  116.50   24.59    7.07\n",
      "    2   6762 337D 338L C   N    2717 2719  145.86  120.00   25.86    5.88  120.00   25.86    5.88\n",
      "    3   6765 338L 338L N   CA   2719 2720  138.26  107.00   31.26    8.99  107.00   31.26    8.99\n",
      "    4   6768 338L 338L CA  C    2720 2725  149.01  112.50   36.51    5.24  112.50   36.51    5.24\n",
      "    5   7018 359R 359R CD  NE   2902 2903  141.47  120.00   21.47    5.45  120.00   21.47    5.45\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  9                           : Distance restraints 1 (CA-CA)           \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  13301  74I 337D CA  CA    559 2712   13.75   11.07    2.68    5.64   11.07    2.68    5.64\n",
      "    2  15315 143A 337D CA  CA   1145 2712   18.28   11.90    6.39   11.86   11.90    6.39   11.86\n",
      "    3  15345 144N 337D CA  CA   1150 2712   16.69    9.97    6.72   12.90    9.97    6.72   12.90\n",
      "    4  15380 145V 337D CA  CA   1158 2712   14.79    9.11    5.67   10.97    9.11    5.67   10.97\n",
      "    5  15422 146L 337D CA  CA   1165 2712   15.24   10.76    4.48    8.23   10.76    4.48    8.23\n",
      "    6  15463 147H 337D CA  CA   1173 2712   15.90   12.81    3.09    5.30   12.81    3.09    5.30\n",
      "    7  15897 172R 337D CA  CA   1367 2712    9.94    6.30    3.64    6.68    6.30    3.64    6.68\n",
      "    8  15918 173V 337D CA  CA   1378 2712   11.33    5.19    6.14   11.64    5.19    6.14   11.64\n",
      "    9  15940 174A 337D CA  CA   1385 2712   14.31    8.37    5.94   10.97    8.37    5.94   10.97\n",
      "   10  15959 175D 337D CA  CA   1390 2712   14.57    9.14    5.43   10.15    9.14    5.43   10.15\n",
      "   11  15980 176P 337D CA  CA   1398 2712   18.11   12.94    5.18    8.63   12.94    5.18    8.63\n",
      "   12  15994 177D 337D CA  CA   1405 2712   18.74   13.97    4.77    7.06   13.97    4.77    7.06\n",
      "   13  16008 178H 337D CA  CA   1413 2712   16.30   12.80    3.50    5.78   12.80    3.50    5.78\n",
      "   14  16828 204G 337D CA  CA   1634 2712   16.34   13.85    2.49    4.85   13.85    2.49    4.85\n",
      "   15  16841 205Y 337D CA  CA   1638 2712   14.90   11.06    3.85    7.88   11.06    3.85    7.88\n",
      "   16  16857 206T 337D CA  CA   1650 2712   18.16   13.36    4.80    9.13   13.36    4.80    9.13\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 10                           : Distance restraints 2 (N-O)             \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  21966 172R 337D N   O    1366 2718   12.95    9.72    3.23    5.13    9.72    3.23    5.13\n",
      "    2  21988 173V 337D N   O    1377 2718   12.19    7.29    4.89    8.56    7.29    4.89    8.56\n",
      "    3  22012 174A 337D N   O    1384 2718   15.13    9.56    5.57    9.45    9.56    5.57    9.45\n",
      "    4  25190 337D 143A N   O    2711 1148   14.77    9.18    5.59   10.13    9.18    5.59   10.13\n",
      "    5  25191 337D 144N N   O    2711 1156   13.23    7.68    5.55   10.39    7.68    5.55   10.39\n",
      "    6  25194 337D 172R N   O    2711 1376   11.00    7.56    3.44    5.78    7.56    3.44    5.78\n",
      "    7  25195 337D 173V N   O    2711 1383   10.50    5.87    4.63    8.35    5.87    4.63    8.35\n",
      "    8  25196 337D 174A N   O    2711 1388   12.85    6.97    5.88   10.62    6.97    5.88   10.62\n",
      "    9  25197 337D 175D N   O    2711 1396   14.82   10.50    4.33    7.21   10.50    4.33    7.21\n",
      "   10  25198 337D 205Y N   O    2711 1648   15.03   10.76    4.27    7.50   10.76    4.27    7.50\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9  -78.93  -68.20   18.59    1.18  -62.50  159.46   26.97\n",
      "    1          2A   2A N   CA      9   10  160.49  145.30                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14  -62.51  -68.20    7.48    0.48  -62.50  178.65   29.28\n",
      "    2          3A   3A N   CA     14   15  140.45  145.30                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19  -71.23  -68.20   25.88    2.19  -62.50  160.73   25.93\n",
      "    3          4A   4A N   CA     19   20  119.60  145.30                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24  -75.27  -68.20    7.71    0.54  -62.50  171.21   28.70\n",
      "    4          5A   5A N   CA     24   25  148.37  145.30                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29 -133.35 -134.00    8.51    0.50  -62.50  178.30   33.02\n",
      "    5          6A   6A N   CA     29   30  155.49  147.00                  -40.90\n",
      "    6  10235   6A   7A C   N      32   34  -78.37  -68.20   26.05    1.76  -62.50  150.66   25.50\n",
      "    6          7A   7A N   CA     34   35  169.28  145.30                  -40.90\n",
      "    7  10236   7A   8G C   N      37   39 -110.70  -80.20   42.57    3.02   82.20 -144.62   16.75\n",
      "    7          8G   8G N   CA     39   40  144.39  174.10                    8.50\n",
      "    8  10237   8G   9A C   N      41   43 -168.51 -134.00   41.27    1.00  -62.50 -176.74   35.62\n",
      "    8          9A   9A N   CA     43   44  169.62  147.00                  -40.90\n",
      "    9  10238   9A  10G C   N      46   48   63.98   78.70   49.77    0.86   82.20  128.36    7.13\n",
      "    9         10G  10G N   CA     48   49 -118.56 -166.10                    8.50\n",
      "   10  10240  11P  12E C   N      57   59  -58.92  -69.30   13.83    1.27  -63.60  168.11   22.16\n",
      "   10         12E  12E N   CA     59   60  151.65  142.50                  -40.30\n",
      "   11  10561 332D 333M C   N    2676 2678  -85.92  -73.00   67.40    4.78  -63.40  119.49   16.22\n",
      "   11        333M 333M N   CA   2678 2679   76.85  143.00                  -40.50\n",
      "   12  10562 333M 334E C   N    2684 2686 -143.13 -117.80   60.40    2.44  -63.60  150.76   24.46\n",
      "   12        334E 334E N   CA   2686 2687 -168.37  136.80                  -40.30\n",
      "   13  10563 334E 335L C   N    2693 2695  -78.44 -108.50   71.47    3.48  -63.50  109.88   14.47\n",
      "   13        335L 335L N   CA   2695 2696   67.66  132.50                  -41.20\n",
      "   14  10564 335L 336D C   N    2701 2703 -101.30  -63.30   70.21   11.08  -63.30   70.21   11.08\n",
      "   14        336D 336D N   CA   2703 2704  -99.04  -40.00                  -40.00\n",
      "   15  10585 356P 357G C   N    2880 2882 -143.66 -167.20   46.86    1.37   82.20 -176.25   14.17\n",
      "   15        357G 357G N   CA   2882 2883  134.08  174.60                    8.50\n",
      "   16  10587 358Y 359R C   N    2896 2898  -69.42  -72.10   30.73    2.30  -63.00  152.52   20.18\n",
      "   16        359R 359R N   CA   2898 2899  111.28  141.90                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:    87  279   1.873\n",
      "  serious non-bonded atom clash:    88  203   2.268\n",
      "  serious non-bonded atom clash:    88  204   1.907\n",
      "  serious non-bonded atom clash:    88  205   2.279\n",
      "  serious non-bonded atom clash:    89  203   2.062\n",
      "  serious non-bonded atom clash:    89  204   1.405\n",
      "  serious non-bonded atom clash:    90  277   1.916\n",
      "  serious non-bonded atom clash:    90  278   1.828\n",
      "  serious non-bonded atom clash:    90  279   1.772\n",
      "  serious non-bonded atom clash:    91  203   1.978\n",
      "  serious non-bonded atom clash:    91  204   0.773\n",
      "  serious non-bonded atom clash:    91  205   2.098\n",
      "  serious non-bonded atom clash:    91  213   1.117\n",
      "  serious non-bonded atom clash:    91  214   2.247\n",
      "  serious non-bonded atom clash:    91  215   1.729\n",
      "  serious non-bonded atom clash:   139 2903   2.199\n",
      "  serious non-bonded atom clash:   167 2903   2.294\n",
      "  serious non-bonded atom clash:   380 2906   2.285\n",
      "  serious non-bonded atom clash:   699 2903   2.079\n",
      "  serious non-bonded atom clash:   699 2904   2.267\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:    13    1    6    4    3    9   22   53  164  188  206  272  371  440  470\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "3qyz_MODLL.B99990001.pdb      1686.14966\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 3r63\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    38201    35555\n",
      "145 (of 2915 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 5740 atoms and 1 coordinate set(s) were parsed in 0.04s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     145\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    35555    1171\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      228\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         224.4041\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     149       0      0   0.005   0.005     0.85642       1.000\n",
      " 2 Bond angle potential               :     203       1      1   2.330   2.330      18.977       1.000\n",
      " 3 Stereochemical cosine torsion poten:      75       0      6  57.704  57.704      39.888       1.000\n",
      " 4 Stereochemical improper torsion pot:      54       0      0   0.962   0.962      1.2453       1.000\n",
      " 5 Soft-sphere overlap restraints     :     228       0      0   0.003   0.003     0.27923       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     207       0      3   0.464   0.464      20.063       1.000\n",
      "10 Distance restraints 2 (N-O)        :     186       1      7   0.841   0.841      45.803       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      23       1      1  35.191  35.191      4.7060       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      12       0      0  64.186  64.186      2.9215       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      11       0      0  38.885  38.885      2.0594       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       7       0      0 111.141 111.141      4.3391       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       2       0      0 127.327 127.327     0.97162       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     117       0      0   0.586   0.586      1.1555       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      25      13      5  41.069 123.815      78.609       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     100       0      0   0.907   0.907      2.5301       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 3r63_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    3834.4219\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 13                           : Mainchain Omega dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10603  16G  16G CA  C      95   96  -11.19 -179.18  168.00   39.04 -179.18  168.00   39.04\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10233   4A   5A C   N      22   24 -173.20 -134.00   39.97    1.06  -62.50 -161.89   38.33\n",
      "    1          5A   5A N   CA     24   25  154.80  147.00                  -40.90\n",
      "    2  10234   5A   6A C   N      27   29 -118.09 -134.00   31.63    1.26  -62.50  169.91   25.76\n",
      "    2          6A   6A N   CA     29   30  119.66  147.00                  -40.90\n",
      "    3  10235   6A   7A C   N      32   34  -76.24  -68.20    8.72    0.62  -62.50  170.98   28.71\n",
      "    3          7A   7A N   CA     34   35  148.68  145.30                  -40.90\n",
      "    4  10236   7A   8G C   N      37   39  102.29   78.70   53.73    0.80   82.20  138.59    7.72\n",
      "    4          8G   8G N   CA     39   40  145.63 -166.10                    8.50\n",
      "    5  10237   8G   9A C   N      41   43  -75.19  -68.20    7.80    0.54  -62.50  170.81   28.63\n",
      "    5          9A   9A N   CA     43   44  148.76  145.30                  -40.90\n",
      "    6  10240  11P  12E C   N      57   59  -74.62  -69.30   20.11    1.71  -63.60  163.78   21.25\n",
      "    6         12E  12E N   CA     59   60  123.10  142.50                  -40.30\n",
      "    7  10241  12E  13M C   N      66   68 -112.37 -125.60   20.72    1.13  -63.40  170.24   29.33\n",
      "    7         13M  13M N   CA     68   69  156.45  140.50                  -40.50\n",
      "    8  10243  14V  15R C   N      81   83  -80.54  -72.10   14.74    0.97   57.30 -179.85   12.81\n",
      "    8         15R  15R N   CA     83   84  153.99  141.90                   38.00\n",
      "    9  10244  15R  16G C   N      92   94   58.00   78.70   22.08    1.23  -80.20  138.73    9.58\n",
      "    9         16G  16G N   CA     94   95 -173.78 -166.10                  174.10\n",
      "   10  10245  16G  17Q C   N      96   98  -94.98  -63.80   74.00   12.51  -63.80   74.00   12.51\n",
      "   10         17Q  17Q N   CA     98   99 -107.41  -40.30                  -40.30\n",
      "   11  10560 331F 332D C   N    2668 2670  -77.62  -70.90   45.83    1.94  -63.30  125.19   16.35\n",
      "   11        332D 332D N   CA   2670 2671 -164.37  150.30                  -40.00\n",
      "   12  10561 332D 333M C   N    2676 2678  -79.10  -73.00   72.80    5.02  -63.40  112.07   15.57\n",
      "   12        333M 333M N   CA   2678 2679   70.46  143.00                  -40.50\n",
      "   13  10562 333M 334E C   N    2684 2686 -152.90 -117.80   36.35    1.13  -63.60 -164.92   30.98\n",
      "   13        334E 334E N   CA   2686 2687  146.26  136.80                  -40.30\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    2    0    6   17   58  118  184  191  266  377  460  441\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "3r63_MODLL.B99990001.pdb       224.40405\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 3sa0\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    38755    36109\n",
      "111 (of 2915 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2725 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     111\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    36109    1240\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      322\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          78.9565\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     113       0      0   0.005   0.005     0.71935       1.000\n",
      " 2 Bond angle potential               :     157       0      0   1.805   1.805      9.5948       1.000\n",
      " 3 Stereochemical cosine torsion poten:      61       0      4  59.060  59.060      34.677       1.000\n",
      " 4 Stereochemical improper torsion pot:      43       0      0   1.013   1.013      1.0004       1.000\n",
      " 5 Soft-sphere overlap restraints     :     322       0      0   0.004   0.004     0.74211       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     284       0      0   0.145   0.145      4.5778       1.000\n",
      "10 Distance restraints 2 (N-O)        :     278       0      0   0.282   0.282      7.6665       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      18       0      0   4.646   4.646      3.7640       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      12       0      0  41.158  41.158     0.74390       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       8       0      0  74.281  74.281      3.5596       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       2       0      0  61.287  61.287      1.3320       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       1       0      0  12.253  12.253     0.31392       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     153       0      0   0.280   0.280     0.27173       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      20       3      2  20.457  86.975      6.6769       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      90       0      0   0.584   0.584      3.3163       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 3sa0_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    1674.6482\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9  -70.62  -68.20    3.85    0.24  -62.50  170.99   28.43\n",
      "    1          2A   2A N   CA      9   10  148.30  145.30                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14  -69.43  -68.20   12.37    1.04  -62.50  174.03   28.19\n",
      "    2          3A   3A N   CA     14   15  132.99  145.30                  -40.90\n",
      "    3  10389 160T 161C C   N    1283 1285   77.15   57.40   46.76    1.49 -117.90 -138.73    9.71\n",
      "    3        161C 161C N   CA   1285 1286   -6.38   36.00                  141.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:   200  285   2.249\n",
      "  serious non-bonded atom clash:   254  509   1.991\n",
      "  serious non-bonded atom clash:   650 1319   2.167\n",
      "  serious non-bonded atom clash:   650 1321   2.046\n",
      "  serious non-bonded atom clash:   651 1321   1.723\n",
      "  serious non-bonded atom clash:  2715 2718   2.262\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     3    1    2    2    8   18   16   59  150  196  207  269  388  435  472\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "3sa0_MODLL.B99990001.pdb        78.95652\n",
      "\n",
      "Modelando proteína 3tei\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    37385    34739\n",
      "240 (of 2915 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2733 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     240\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    34739    1855\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      447\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         195.2994\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     249       0      0   0.005   0.005      1.4564       1.000\n",
      " 2 Bond angle potential               :     342       0      1   1.980   1.980      25.569       1.000\n",
      " 3 Stereochemical cosine torsion poten:     165       0      3  45.460  45.460      54.682       1.000\n",
      " 4 Stereochemical improper torsion pot:     117       0      0   1.429   1.429      6.6295       1.000\n",
      " 5 Soft-sphere overlap restraints     :     447       0      0   0.002   0.002     0.26657       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     305       0      0   0.235   0.235      9.2744       1.000\n",
      "10 Distance restraints 2 (N-O)        :     262       0      0   0.424   0.424      16.683       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      35       2      2  41.179  41.179      12.973       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      21       0      0  73.591  73.591      4.8752       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      16       0      0  95.447  95.447      4.8292       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      0  90.704  90.704      3.4339       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       2       0      0 122.878 122.878     0.77066       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     216       0      0   0.898   0.898      4.2879       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      37      22      5  26.349 138.506      44.373       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      83       0      0   1.107   1.107      5.1957       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 3tei_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    6352.4790\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 13                           : Mainchain Omega dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10763 176P 176P CA  C    1398 1402   -0.40 -179.18  178.78   41.56 -179.18  178.78   41.56\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9  -75.02  -68.20    8.02    0.53  -62.50  170.04   28.50\n",
      "    1          2A   2A N   CA      9   10  149.53  145.30                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14  -74.77  -68.20   13.32    0.86  -62.50  162.68   27.28\n",
      "    2          3A   3A N   CA     14   15  156.88  145.30                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19  -75.81  -68.20   13.82    0.88  -62.50  162.81   27.35\n",
      "    3          4A   4A N   CA     19   20  156.84  145.30                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24  -80.74  -68.20   18.57    1.83  -62.50  173.46   27.59\n",
      "    4          5A   5A N   CA     24   25  131.60  145.30                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29  -71.62  -68.20    4.05    0.40  -62.50  176.20   29.33\n",
      "    5          6A   6A N   CA     29   30  143.14  145.30                  -40.90\n",
      "    6  10235   6A   7A C   N      32   34  -75.96  -68.20   12.19    0.77  -62.50  164.95   27.71\n",
      "    6          7A   7A N   CA     34   35  154.70  145.30                  -40.90\n",
      "    7  10236   7A   8G C   N      37   39 -105.88  -80.20   33.44    2.41   82.20 -135.63   17.42\n",
      "    7          8G   8G N   CA     39   40  152.67  174.10                    8.50\n",
      "    8  10237   8G   9A C   N      41   43 -134.66 -134.00    4.14    0.22  -62.50 -177.14   33.83\n",
      "    8          9A   9A N   CA     43   44  151.08  147.00                  -40.90\n",
      "    9  10407 178H 179D C   N    1420 1422  -95.97  -96.50    1.52    0.06  -63.30  156.23   17.55\n",
      "    9        179D 179D N   CA   1422 1423  112.78  114.20                  -40.00\n",
      "   10  10409 180H 181T C   N    1438 1440 -128.83 -124.80   28.15    1.09  -63.20  160.57   24.89\n",
      "   10        181T 181T N   CA   1440 1441  171.35  143.50                  -42.10\n",
      "   11  10410 181T 182G C   N    1445 1447  -91.38  -80.20   18.82    1.26   82.20 -130.29    7.81\n",
      "   11        182G 182G N   CA   1447 1448  158.95  174.10                    8.50\n",
      "   12  10411 182G 183F C   N    1449 1451 -141.63 -124.20   22.41    0.58  -63.20  176.68   29.58\n",
      "   12        183F 183F N   CA   1451 1452  157.38  143.30                  -44.30\n",
      "   13  10412 183F 184L C   N    1460 1462  -69.11  -70.70   19.88    1.45  -63.50  163.08   22.33\n",
      "   13        184L 184L N   CA   1462 1463  121.78  141.60                  -41.20\n",
      "   14  10413 184L 185T C   N    1468 1470  -98.37  -78.10   37.49    1.24  -63.20  141.02   20.43\n",
      "   14        185T 185T N   CA   1470 1471 -178.66  149.80                  -42.10\n",
      "   15  10414 185T 186E C   N    1475 1477  -82.14  -69.30   21.17    1.20   54.60  179.92   11.98\n",
      "   15        186E 186E N   CA   1477 1478  159.33  142.50                   42.40\n",
      "   16  10416 187Y 188V C   N    1496 1498   68.27   55.90   40.04    3.73 -125.40  178.84    8.92\n",
      "   16        188V 188V N   CA   1498 1499   77.59   39.50                  143.30\n",
      "   17  10429 200L 201N C   N    1608 1610  -88.82 -119.90   33.10    1.63  -63.20  172.40   23.00\n",
      "   17        201N 201N N   CA   1610 1611  148.41  137.00                  -41.10\n",
      "   18  10430 201N 202S C   N    1616 1618 -104.22 -136.60   65.48    2.49  -64.10  135.36    8.35\n",
      "   18        202S 202S N   CA   1618 1619   94.28  151.20                  -35.00\n",
      "   19  10431 202S 203K C   N    1622 1624 -140.72 -118.00   48.46    1.95  -62.90  157.82   25.14\n",
      "   19        203K 203K N   CA   1624 1625 -178.10  139.10                  -40.80\n",
      "   20  10432 203K 204G C   N    1631 1633  -86.65  -80.20   34.67    1.05   82.20 -127.15   17.94\n",
      "   20        204G 204G N   CA   1633 1634 -151.83  174.10                    8.50\n",
      "   21  10433 204G 205Y C   N    1635 1637 -120.63 -124.30   11.73    0.78   55.90 -153.55   15.91\n",
      "   21        205Y 205Y N   CA   1637 1638  146.54  135.40                   39.50\n",
      "   22  10587 358Y 359R C   N    2896 2898  -70.38  -72.10    1.73    0.13  -63.00  177.31   24.45\n",
      "   22        359R 359R N   CA   2898 2899  141.74  141.90                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:   412  779   2.221\n",
      "  serious non-bonded atom clash:   588 2640   2.290\n",
      "  serious non-bonded atom clash:   589 2640   1.648\n",
      "  serious non-bonded atom clash:   589 2648   2.197\n",
      "  serious non-bonded atom clash:   589 2650   2.117\n",
      "  serious non-bonded atom clash:   590 2638   1.764\n",
      "  serious non-bonded atom clash:  1601 1868   1.799\n",
      "  serious non-bonded atom clash:  2020 2022   1.935\n",
      "  serious non-bonded atom clash:  2707 2710   2.170\n",
      "  serious non-bonded atom clash:  2740 2749   2.280\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     4    3    3    5    9    9   21   50   91  158  207  290  354  470  441\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "3tei_MODLL.B99990001.pdb       195.29939\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 3w55\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    37326    34680\n",
      "234 (of 2915 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2817 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     234\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    34680    1527\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      355\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         152.7725\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     243       0      0   0.003   0.003     0.51199       1.000\n",
      " 2 Bond angle potential               :     336       0      0   1.671   1.671      17.572       1.000\n",
      " 3 Stereochemical cosine torsion poten:     166       0      3  48.098  48.098      56.745       1.000\n",
      " 4 Stereochemical improper torsion pot:     108       0      0   0.787   0.787      1.5587       1.000\n",
      " 5 Soft-sphere overlap restraints     :     355       0      0   0.004   0.004     0.52208       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     210       0      0   0.134   0.134      2.5961       1.000\n",
      "10 Distance restraints 2 (N-O)        :     194       0      0   0.407   0.407      10.759       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      34       0      0   2.854   2.854      2.6714       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      20       0      0  73.453  73.453      3.9384       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      16       0      0  73.118  73.118      4.0734       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       7       0      0  97.974  97.974      2.6653       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       1       0      0  82.814  82.814     0.76567       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     120       0      0   0.315   0.315     0.24916       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      35      24      5  31.908 145.932      46.824       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      37       0      0   0.597   0.597      1.3202       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 3w55_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    6097.5498\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9  -64.98  -68.20    8.35    0.56  -62.50  178.51   29.13\n",
      "    1          2A   2A N   CA      9   10  137.59  145.30                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14  -80.51  -68.20   12.33    1.06  -62.50  175.27   29.64\n",
      "    2          3A   3A N   CA     14   15  144.76  145.30                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19  -78.12  -68.20   12.32    0.80  -62.50  167.22   28.20\n",
      "    3          4A   4A N   CA     19   20  152.61  145.30                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24  -71.91  -68.20    4.86    0.31  -62.50  170.92   28.48\n",
      "    4          5A   5A N   CA     24   25  148.44  145.30                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29 -133.83 -134.00    6.62    0.38  -62.50 -179.79   33.35\n",
      "    5          6A   6A N   CA     29   30  153.62  147.00                  -40.90\n",
      "    6  10235   6A   7A C   N      32   34  -63.02  -68.20    7.55    0.48  -62.50  179.29   29.41\n",
      "    6          7A   7A N   CA     34   35  139.81  145.30                  -40.90\n",
      "    7  10236   7A   8G C   N      37   39   90.50   78.70   45.90    0.85   82.20  141.28    7.28\n",
      "    7          8G   8G N   CA     39   40  149.54 -166.10                    8.50\n",
      "    8  10237   8G   9A C   N      41   43  -73.22  -68.20   18.97    1.70  -62.50  168.25   27.07\n",
      "    8          9A   9A N   CA     43   44  127.01  145.30                  -40.90\n",
      "    9  10238   9A  10G C   N      46   48 -108.37  -80.20   34.71    1.38   82.20 -117.05    7.49\n",
      "    9         10G  10G N   CA     48   49 -165.62  174.10                    8.50\n",
      "   10  10403 174A 175D C   N    1387 1389  -61.97  -63.30    8.38    0.96   54.50  146.69   18.79\n",
      "   10        175D 175D N   CA   1389 1390  -48.28  -40.00                   40.90\n",
      "   11  10404 175D 176P C   N    1395 1397  -76.45  -58.70   19.52    1.92  -64.50  169.98   13.32\n",
      "   11        176P 176P N   CA   1397 1398  -22.36  -30.50                  147.20\n",
      "   12  10405 176P 177D C   N    1402 1404 -151.95  -96.50   81.28    3.40  -63.30  171.12   26.79\n",
      "   12        177D 177D N   CA   1404 1405  173.63  114.20                  -40.00\n",
      "   13  10406 177D 178H C   N    1410 1412  -64.28  -67.60    9.50    0.63  -63.20  173.40   21.61\n",
      "   13        178H 178H N   CA   1412 1413  131.10  140.00                  -42.30\n",
      "   14  10407 178H 179D C   N    1420 1422 -131.41  -96.50   46.65    1.95  -63.30 -172.33   27.57\n",
      "   14        179D 179D N   CA   1422 1423  145.13  114.20                  -40.00\n",
      "   15  10409 180H 181T C   N    1438 1440 -155.81 -124.80   58.95    1.83  -63.20  154.98   25.62\n",
      "   15        181T 181T N   CA   1440 1441 -166.37  143.50                  -42.10\n",
      "   16  10410 181T 182G C   N    1445 1447 -105.12  -80.20   39.72    2.70   82.20 -141.02   17.06\n",
      "   16        182G 182G N   CA   1447 1448  143.16  174.10                    8.50\n",
      "   17  10411 182G 183F C   N    1449 1451  -93.32  -71.40   22.25    1.39  -63.20  173.80   26.17\n",
      "   17        183F 183F N   CA   1451 1452  144.53  140.70                  -44.30\n",
      "   18  10412 183F 184L C   N    1460 1462 -107.10 -108.50   10.53    0.55  -63.50  168.99   21.39\n",
      "   18        184L 184L N   CA   1462 1463  122.07  132.50                  -41.20\n",
      "   19  10413 184L 185T C   N    1468 1470 -140.45 -124.80   19.34    0.69  -63.20 -179.59   28.17\n",
      "   19        185T 185T N   CA   1470 1471  154.87  143.50                  -42.10\n",
      "   20  10414 185T 186E C   N    1475 1477  -70.27  -69.30    8.84    0.71  -63.60  174.15   22.86\n",
      "   20        186E 186E N   CA   1477 1478  133.72  142.50                  -40.30\n",
      "   21  10415 186E 187Y C   N    1484 1486 -129.42 -124.30    9.68    0.38  -63.50 -174.88   32.68\n",
      "   21        187Y 187Y N   CA   1486 1487  143.61  135.40                  -43.40\n",
      "   22  10418 189A 190T C   N    1508 1510  -33.95  -63.20   63.72    6.77  -63.20   63.72    6.77\n",
      "   22        190T 190T N   CA   1510 1511  -98.72  -42.10                  -42.10\n",
      "   23  10586 357G 358Y C   N    2884 2886 -110.20  -98.40   13.84    0.84  -63.50  171.08   24.94\n",
      "   23        358Y 358Y N   CA   2886 2887  121.18  128.40                  -43.40\n",
      "   24  10587 358Y 359R C   N    2896 2898 -131.82 -125.20    8.42    0.45  -63.00 -170.56   21.92\n",
      "   24        359R 359R N   CA   2898 2899  135.40  140.60                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:    93   97   2.280\n",
      "  serious non-bonded atom clash:  1137 1149   2.260\n",
      "  serious non-bonded atom clash:  1360 1365   2.270\n",
      "  serious non-bonded atom clash:  1553 1712   2.220\n",
      "  serious non-bonded atom clash:  1732 1737   2.118\n",
      "  serious non-bonded atom clash:  2113 2135   2.299\n",
      "  serious non-bonded atom clash:  2742 2761   2.296\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    1    6   14   18   64   78  107  156  213  258  315  357  406  455\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "3w55_MODLL.B99990001.pdb       152.77249\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 3zu7\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    37917    35271\n",
      "136 (of 2915 total) atoms selected for optimization\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:      -93.0728\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     136\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    35271     842\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      495\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :        3599.0078\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     140      10     10   0.124   0.124      559.18       1.000\n",
      " 2 Bond angle potential               :     190      10     17   7.986   7.986      231.50       1.000\n",
      " 3 Stereochemical cosine torsion poten:      87       0      5  61.805  61.805      41.766       1.000\n",
      " 4 Stereochemical improper torsion pot:      56       3      3   8.232   8.232      75.256       1.000\n",
      " 5 Soft-sphere overlap restraints     :     495      59     81   0.207   0.207      2505.2       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      95       0      0   0.130   0.130      1.3657       1.000\n",
      "10 Distance restraints 2 (N-O)        :      95       0      0   0.380   0.380      6.7379       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      20       2      2  22.510  22.510      119.39       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      10       0      2  68.069  68.069      6.2604       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       9       0      0  79.068  79.068      3.1309       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       7       0      0  80.774  80.774      4.3502       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       1       0      0 110.906 110.906     0.73995       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      75       0      0   0.657   0.657     0.57737       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      20      13      7  54.015 144.150      37.588       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      37       0      0   1.130   1.130      5.9400       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 3zu7_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    6056.9570\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  1                           : Bond length potential                   \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   2954 357G 357G C   CA   2884 2883    1.70    1.49    0.21    6.12    1.49    0.21    6.12\n",
      "    2   2955 357G 358Y C   N    2884 2886    1.72    1.35    0.38   13.43    1.35    0.38   13.43\n",
      "    3   2956 358Y 358Y CB  CA   2888 2887    1.89    1.54    0.35    9.71    1.54    0.35    9.71\n",
      "    4   2957 358Y 358Y CG  CB   2889 2888    1.80    1.49    0.31    8.54    1.49    0.31    8.54\n",
      "    5   2965 358Y 358Y N   CA   2886 2887    2.02    1.43    0.59   19.52    1.43    0.59   19.52\n",
      "    6   2969 359R 359R CB  CA   2900 2899    2.16    1.54    0.62   17.01    1.54    0.62   17.01\n",
      "    7   2970 359R 359R CG  CB   2901 2900    2.12    1.53    0.59   16.31    1.53    0.59   16.31\n",
      "    8   2976 359R 359R N   CA   2898 2899    1.73    1.43    0.30    9.84    1.43    0.30    9.84\n",
      "    9   2978 359R 359R C   CA   2907 2899    2.14    1.49    0.65   18.75    1.49    0.65   18.75\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6993 358Y 358Y N   CA   2886 2887  138.79  113.50   25.29    6.80  113.50   25.29    6.80\n",
      "    2   6994 358Y 358Y N   CA   2886 2887  140.12  107.00   33.12    9.52  107.00   33.12    9.52\n",
      "    3   6995 358Y 358Y CA  CB   2887 2888  159.05  107.50   51.55   11.92  107.50   51.55   11.92\n",
      "    4   7009 358Y 359R C   N    2896 2898  154.42  120.00   34.42    7.82  120.00   34.42    7.82\n",
      "    5   7016 359R 359R CB  CG   2900 2901  139.58  113.60   25.98    6.38  113.60   25.98    6.38\n",
      "    6   7022 359R 359R C   CA   2907 2899  138.19  108.00   30.18    7.00  108.00   30.18    7.00\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 13                           : Mainchain Omega dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10944 357G 357G CA  C    2883 2884  -83.29 -180.00   96.71   19.33 -180.00   96.71   19.33\n",
      "    2  10945 358Y 358Y CA  C    2887 2896 -156.26 -180.00   23.74    4.75 -180.00   23.74    4.75\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9  -61.06  -68.20   14.47    0.94  -62.50  173.61   28.52\n",
      "    1          2A   2A N   CA      9   10  132.71  145.30                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14  -77.36  -68.20   11.05    0.73  -62.50  168.28   28.33\n",
      "    2          3A   3A N   CA     14   15  151.48  145.30                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19  -74.59  -68.20    6.78    0.49  -62.50  171.95   28.79\n",
      "    3          4A   4A N   CA     19   20  147.57  145.30                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24 -103.76 -134.00   30.82    1.25  -62.50  171.18   30.23\n",
      "    4          5A   5A N   CA     24   25  152.97  147.00                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29  -62.18  -68.20   10.84    0.69  -62.50  177.18   29.05\n",
      "    5          6A   6A N   CA     29   30  136.28  145.30                  -40.90\n",
      "    6  10235   6A   7A C   N      32   34 -152.29 -134.00   21.30    0.50  -62.50 -175.48   35.03\n",
      "    6          7A   7A N   CA     34   35  157.91  147.00                  -40.90\n",
      "    7  10236   7A   8G C   N      37   39  -62.84  -80.20   62.29    1.66   82.20  179.51    6.77\n",
      "    7          8G   8G N   CA     39   40  114.28  174.10                    8.50\n",
      "    8  10237   8G   9A C   N      41   43  164.43 -134.00   68.94    1.57  -62.50 -166.05   38.59\n",
      "    8          9A   9A N   CA     43   44  178.01  147.00                  -40.90\n",
      "    9  10238   9A  10G C   N      46   48 -102.60  -80.20   57.23    3.35   82.20 -151.56   16.29\n",
      "    9         10G  10G N   CA     48   49  121.43  174.10                    8.50\n",
      "   10  10583 354F 355Q C   N    2864 2866  127.30   55.10  158.74    7.46  -63.80  179.50   32.10\n",
      "   10        355Q 355Q N   CA   2866 2867 -101.07   40.30                  -40.30\n",
      "   11  10585 356P 357G C   N    2880 2882  157.83 -167.20   58.52    2.30   82.20  141.15   10.17\n",
      "   11        357G 357G N   CA   2882 2883  127.68  174.60                    8.50\n",
      "   12  10586 357G 358Y C   N    2884 2886  135.37 -124.30  107.08    8.82  -63.50 -145.63   30.09\n",
      "   12        358Y 358Y N   CA   2886 2887   97.99  135.40                  -43.40\n",
      "   13  10587 358Y 359R C   N    2896 2898  -95.00  -72.10   51.83    4.41  -63.00  140.21   17.02\n",
      "   13        359R 359R N   CA   2898 2899   95.41  141.90                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:   709 2883   2.159\n",
      "  serious non-bonded atom clash:   717 2883   2.247\n",
      "  serious non-bonded atom clash:   717 2884   2.067\n",
      "  serious non-bonded atom clash:   717 2886   1.564\n",
      "  serious non-bonded atom clash:   724 2885   2.300\n",
      "  serious non-bonded atom clash:   725 2886   2.063\n",
      "  serious non-bonded atom clash:   725 2887   1.630\n",
      "  serious non-bonded atom clash:   725 2888   2.248\n",
      "  serious non-bonded atom clash:   728 2899   2.022\n",
      "  serious non-bonded atom clash:   728 2907   2.242\n",
      "  serious non-bonded atom clash:   731 2898   1.716\n",
      "  serious non-bonded atom clash:   731 2899   1.984\n",
      "  serious non-bonded atom clash:   731 2900   2.049\n",
      "  serious non-bonded atom clash:   738 2900   2.110\n",
      "  serious non-bonded atom clash:   739 2900   1.872\n",
      "  serious non-bonded atom clash:   739 2901   1.958\n",
      "  serious non-bonded atom clash:   749 2899   1.557\n",
      "  serious non-bonded atom clash:   749 2900   2.081\n",
      "  serious non-bonded atom clash:   751 2896   2.274\n",
      "  serious non-bonded atom clash:   751 2898   2.171\n",
      "  serious non-bonded atom clash:   752 2898   2.151\n",
      "  serious non-bonded atom clash:   755 2889   2.180\n",
      "  serious non-bonded atom clash:   758 2900   1.843\n",
      "  serious non-bonded atom clash:   758 2901   2.249\n",
      "  serious non-bonded atom clash:  1426 1429   2.300\n",
      "  serious non-bonded atom clash:  1439 1446   2.300\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:    13    5    8   12   14   22   38   72  133  170  207  276  364  435  469\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "3zu7_MODLL.B99990001.pdb      3599.00781\n",
      "\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2924 atoms and 1 coordinate set(s) were parsed in 0.11s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 3zuv\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    38426    35780\n",
      "77 (of 2915 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2849 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915      77\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    35780     768\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      146\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         127.7683\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      80       0      0   0.014   0.014      4.6952       1.000\n",
      " 2 Bond angle potential               :     111       0      1   2.587   2.587      14.676       1.000\n",
      " 3 Stereochemical cosine torsion poten:      55       0      3  49.703  49.703      21.487       1.000\n",
      " 4 Stereochemical improper torsion pot:      39       0      0   1.394   1.394      1.6845       1.000\n",
      " 5 Soft-sphere overlap restraints     :     146       0      0   0.002   0.002     0.66844E-01   1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     147       0      4   0.585   0.585      26.224       1.000\n",
      "10 Distance restraints 2 (N-O)        :     145       0      1   0.576   0.576      20.134       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      11       1      1   6.547   6.547      3.2899       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       7       0      0  71.293  71.293      1.7506       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       5       0      0  48.981  48.981     0.85505       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       2       0      0 163.702 163.702      1.6606       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      75       0      0   0.543   0.543     0.46697       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      12       4      3  58.673 114.313      25.321       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      79       0      0   0.746   0.746      5.4564       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 3zuv_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    1931.2822\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9  -78.61  -68.20   20.49    1.32  -62.50  156.99   26.55\n",
      "    1          2A   2A N   CA      9   10  162.94  145.30                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14  -63.76  -68.20    6.97    0.68  -62.50  168.43   27.66\n",
      "    2          3A   3A N   CA     14   15  150.68  145.30                  -40.90\n",
      "    3  10414 185T 186E C   N    1475 1477  -99.87 -117.80   19.38    0.90  -63.60  179.26   25.95\n",
      "    3        186E 186E N   CA   1477 1478  144.15  136.80                  -40.30\n",
      "    4  10417 188V 189A C   N    1503 1505   68.35   55.40  115.83    8.05  -62.50 -148.78   31.99\n",
      "    4        189A 189A N   CA   1505 1506  153.30   38.20                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:   856 1275   2.054\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     1    0    0    1    3   11   22   84  121  178  226  241  371  428  433\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "3zuv_MODLL.B99990001.pdb       127.76831\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 4erk\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    38855    36209\n",
      "88 (of 2915 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2845 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915      88\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    36209     477\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      104\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          57.6049\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      90       0      0   0.002   0.002     0.10598       1.000\n",
      " 2 Bond angle potential               :     123       0      0   1.611   1.611      5.9786       1.000\n",
      " 3 Stereochemical cosine torsion poten:      53       0      2  52.698  52.698      21.260       1.000\n",
      " 4 Stereochemical improper torsion pot:      36       0      0   0.684   0.684     0.45396       1.000\n",
      " 5 Soft-sphere overlap restraints     :     104       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      54       0      0   0.340   0.340      3.4705       1.000\n",
      "10 Distance restraints 2 (N-O)        :      48       0      0   0.548   0.548      6.0963       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      14       0      0   2.178   2.178     0.78293       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       5       0      0  74.616  74.616     0.38426       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       4       0      0  10.449  10.449     0.35272       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       3       0      0 104.708 104.708      2.0086       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       1       0      0 119.499 119.499     0.78532       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      32       0      0   0.403   0.403     0.16757       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      14       7      1  34.847 128.057      15.758       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 4erk_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    2088.1423\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9  -76.49  -68.20    8.76    0.82  -62.50  177.18   29.74\n",
      "    1          2A   2A N   CA      9   10  142.48  145.30                  -40.90\n",
      "    2  10232   3A   4A C   N      17   19  -62.57  -68.20   21.76    1.56  -62.50  165.18   27.07\n",
      "    2          4A   4A N   CA     19   20  124.29  145.30                  -40.90\n",
      "    3  10234   5A   6A C   N      27   29 -162.18 -134.00   34.73    0.87  -62.50 -178.39   35.05\n",
      "    3          6A   6A N   CA     29   30  167.29  147.00                  -40.90\n",
      "    4  10235   6A   7A C   N      32   34 -113.95 -134.00   21.59    1.03  -62.50  171.97   30.92\n",
      "    4          7A   7A N   CA     34   35  155.01  147.00                  -40.90\n",
      "    5  10585 356P 357G C   N    2880 2882   76.43   78.70    2.27    0.11  -80.20  157.88   11.04\n",
      "    5        357G 357G N   CA   2882 2883 -166.10 -166.10                  174.10\n",
      "    6  10586 357G 358Y C   N    2884 2886 -123.62 -124.30    2.90    0.13  -63.50 -174.03   26.75\n",
      "    6        358Y 358Y N   CA   2886 2887  132.59  135.40                  -43.40\n",
      "    7  10587 358Y 359R C   N    2896 2898  -97.82  -72.10   27.60    2.30  -63.00  176.45   21.70\n",
      "    7        359R 359R N   CA   2898 2899  131.88  141.90                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:  2036 2082   2.296\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    1    0    9   20   41   81  115  194  273  309  335  441  490\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "4erk_MODLL.B99990001.pdb        57.60491\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 4fmq\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    38637    35991\n",
      "85 (of 2915 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2753 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915      85\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    35991     452\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       95\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          42.9882\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      86       0      0   0.003   0.003     0.15416       1.000\n",
      " 2 Bond angle potential               :     117       0      0   1.367   1.367      4.1268       1.000\n",
      " 3 Stereochemical cosine torsion poten:      47       0      1  49.278  49.278      19.960       1.000\n",
      " 4 Stereochemical improper torsion pot:      35       0      0   1.209   1.209      1.7530       1.000\n",
      " 5 Soft-sphere overlap restraints     :      95       0      0   0.001   0.001     0.66377E-02   1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      54       0      0   0.209   0.209      1.4213       1.000\n",
      "10 Distance restraints 2 (N-O)        :      48       0      0   0.422   0.422      2.7293       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      14       0      0   1.887   1.887     0.58813       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       4       0      0  79.943  79.943     0.27342       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       3       0      0   5.345   5.345     0.38236E-01   1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       2       0      0  73.012  73.012      1.2226       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       1       0      0 168.025 168.025     0.89240       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      19       0      0   0.247   0.247     0.25314E-01   1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      14       8      1  24.841 134.395      9.5443       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       8       0      0   0.840   0.840     0.25269       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 4fmq_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    2275.5791\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9  -74.44  -68.20   17.54    1.20  -62.50  157.86   26.47\n",
      "    1          2A   2A N   CA      9   10  161.69  145.30                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14  -64.41  -68.20   13.33    1.20  -62.50  161.03   26.48\n",
      "    2          3A   3A N   CA     14   15  158.08  145.30                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19 -127.40 -134.00    7.88    0.42  -62.50  179.90   32.96\n",
      "    3          4A   4A N   CA     19   20  151.32  147.00                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24 -135.51 -134.00    4.14    0.25  -62.50 -169.50   35.13\n",
      "    4          5A   5A N   CA     24   25  143.14  147.00                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29 -118.57 -134.00   15.54    0.59  -62.50  179.21   32.36\n",
      "    5          6A   6A N   CA     29   30  148.88  147.00                  -40.90\n",
      "    6  10235   6A   7A C   N      32   34  -68.80  -68.20    5.11    0.43  -62.50  178.98   29.64\n",
      "    6          7A   7A N   CA     34   35  140.23  145.30                  -40.90\n",
      "    7  10238   9A  10G C   N      46   48  -93.93 -167.20   74.94    1.13   78.70  176.15    9.55\n",
      "    7         10G  10G N   CA     48   49  158.87  174.60                 -166.10\n",
      "    8  10587 358Y 359R C   N    2896 2898 -133.97 -125.20    9.36    0.30  -63.00 -171.14   30.39\n",
      "    8        359R 359R N   CA   2898 2899  143.88  140.60                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:  1427 1628   2.055\n",
      "  serious non-bonded atom clash:  1427 1630   2.136\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     1    1    0    1    4    5   11   42   91  177  261  307  329  434  449\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "4fmq_MODLL.B99990001.pdb        42.98823\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 4fux\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    37389    34743\n",
      "213 (of 2915 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2654 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     213\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    34743    1744\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      346\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         126.9480\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     219       0      0   0.003   0.003     0.64553       1.000\n",
      " 2 Bond angle potential               :     301       0      0   1.683   1.683      15.827       1.000\n",
      " 3 Stereochemical cosine torsion poten:     129       0      7  54.767  54.767      58.260       1.000\n",
      " 4 Stereochemical improper torsion pot:      85       0      0   0.862   0.862      1.3472       1.000\n",
      " 5 Soft-sphere overlap restraints     :     346       0      0   0.003   0.003     0.38376       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     301       0      0   0.146   0.146      4.0698       1.000\n",
      "10 Distance restraints 2 (N-O)        :     262       0      0   0.245   0.245      5.4773       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      32       0      0   3.630   3.630      4.2644       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      20       0      0  72.025  72.025      3.8014       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      16       0      0  60.929  60.929      4.4376       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       8       0      0  90.313  90.313      4.0634       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       2       0      0  72.526  72.526     0.95508       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     208       0      0   0.350   0.350     0.61698       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      34      15      3  23.825 118.778      20.608       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     127       0      0   0.479   0.479      2.1906       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 4fux_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    4515.2256\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9  -70.47  -68.20    5.22    0.35  -62.50  169.29   28.14\n",
      "    1          2A   2A N   CA      9   10  150.00  145.30                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14  -67.41  -68.20    5.67    0.43  -62.50  179.49   29.66\n",
      "    2          3A   3A N   CA     14   15  139.68  145.30                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19 -162.18 -134.00   31.32    0.71  -62.50 -172.81   35.97\n",
      "    3          4A   4A N   CA     19   20  160.66  147.00                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24 -117.86 -134.00   18.26    0.42  -62.50 -172.29   28.61\n",
      "    4          5A   5A N   CA     24   25  138.47  147.00                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29  -69.37  -68.20    2.26    0.22  -62.50  175.87   29.16\n",
      "    5          6A   6A N   CA     29   30  143.37  145.30                  -40.90\n",
      "    6  10235   6A   7A C   N      32   34  177.73 -134.00   56.24    1.32  -62.50 -173.27   36.82\n",
      "    6          7A   7A N   CA     34   35  175.85  147.00                  -40.90\n",
      "    7  10236   7A   8G C   N      37   39   85.52   78.70    6.86    0.36   82.20  173.81    8.27\n",
      "    7          8G   8G N   CA     39   40 -165.28 -166.10                    8.50\n",
      "    8  10237   8G   9A C   N      41   43  -69.77  -68.20    3.78    0.25  -62.50  170.52   28.31\n",
      "    8          9A   9A N   CA     43   44  148.74  145.30                  -40.90\n",
      "    9  10238   9A  10G C   N      46   48  171.13 -167.20   22.02    0.52   82.20 -175.03   12.97\n",
      "    9         10G  10G N   CA     48   49  170.69  174.60                    8.50\n",
      "   10  10389 160T 161C C   N    1283 1285   73.43   57.40   21.74    1.33 -117.90 -153.12    8.91\n",
      "   10        161C 161C N   CA   1285 1286   21.32   36.00                  141.10\n",
      "   11  10561 332D 333M C   N    2676 2678 -125.84 -125.60   24.21    1.16  -63.40  168.77   21.33\n",
      "   11        333M 333M N   CA   2678 2679  116.29  140.50                  -40.50\n",
      "   12  10563 334E 335L C   N    2693 2695  -69.18  -70.70    7.30    0.61  -63.50  170.15   23.95\n",
      "   12        335L 335L N   CA   2695 2696  148.74  141.60                  -41.20\n",
      "   13  10585 356P 357G C   N    2880 2882  168.74 -167.20   29.55    1.02  -80.20  112.30    7.91\n",
      "   13        357G 357G N   CA   2882 2883  157.45  174.60                  174.10\n",
      "   14  10586 357G 358Y C   N    2884 2886  -92.81  -98.40    8.32    0.71  -63.50  168.22   25.25\n",
      "   14        358Y 358Y N   CA   2886 2887  122.24  128.40                  -43.40\n",
      "   15  10587 358Y 359R C   N    2896 2898 -105.31 -125.20   20.65    0.69  -63.00 -178.81   21.94\n",
      "   15        359R 359R N   CA   2898 2899  135.08  140.60                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:   249  402   2.205\n",
      "  serious non-bonded atom clash:   249  407   2.050\n",
      "  serious non-bonded atom clash:   249  409   1.547\n",
      "  serious non-bonded atom clash:   251  407   1.638\n",
      "  serious non-bonded atom clash:   251  409   0.582\n",
      "  serious non-bonded atom clash:   251  410   1.408\n",
      "  serious non-bonded atom clash:   251  413   2.003\n",
      "  serious non-bonded atom clash:   252  413   1.316\n",
      "  serious non-bonded atom clash:   252  414   1.334\n",
      "  serious non-bonded atom clash:   252  415   1.632\n",
      "  serious non-bonded atom clash:   252  416   1.973\n",
      "  serious non-bonded atom clash:   253  409   1.869\n",
      "  serious non-bonded atom clash:   253  410   1.286\n",
      "  serious non-bonded atom clash:   253  413   0.624\n",
      "  serious non-bonded atom clash:   253  414   1.649\n",
      "  serious non-bonded atom clash:   253  415   1.492\n",
      "  serious non-bonded atom clash:   254  410   1.384\n",
      "  serious non-bonded atom clash:   254  411   2.280\n",
      "  serious non-bonded atom clash:   254  413   1.135\n",
      "  serious non-bonded atom clash:   254  414   2.078\n",
      "  serious non-bonded atom clash:   254  415   1.750\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:    19    0    2    0    4    9   25   51   90  128  175  229  336  431  516\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "4fux_MODLL.B99990001.pdb       126.94798\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 4fuy\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    36007    33361\n",
      "388 (of 2915 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2737 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:      -91.0613\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     388\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    33361    3105\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      795\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         413.9116\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     400       0      1   0.009   0.009      9.9022       1.000\n",
      " 2 Bond angle potential               :     548       0      7   2.635   2.635      73.188       1.000\n",
      " 3 Stereochemical cosine torsion poten:     249       0      9  49.392  49.392      95.972       1.000\n",
      " 4 Stereochemical improper torsion pot:     162       0      0   1.604   1.604      8.5497       1.000\n",
      " 5 Soft-sphere overlap restraints     :     795       0      1   0.011   0.011      11.141       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     526       0      0   0.296   0.296      22.602       1.000\n",
      "10 Distance restraints 2 (N-O)        :     490       0      0   0.383   0.383      23.012       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      55       1      1  23.829  23.829      19.685       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      41       0      2  76.665  76.665      20.867       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      32       0      2  54.864  54.864      14.905       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:      15       0      0 111.133 111.133      10.020       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       5       0      0  74.192  74.192      3.0975       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     327       0      0   0.719   0.719      4.4568       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      59      27     10  40.155 119.167      80.104       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     196       0      0   1.260   1.260      16.409       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 4fuy_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    9039.3887\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9  -69.62  -68.20    6.13    0.44  -62.50  167.99   27.88\n",
      "    1          2A   2A N   CA      9   10  151.27  145.30                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14  -69.38  -68.20    7.31    0.55  -62.50  166.73   27.67\n",
      "    2          3A   3A N   CA     14   15  152.51  145.30                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19  -67.21  -68.20    3.23    0.29  -62.50  170.79   28.22\n",
      "    3          4A   4A N   CA     19   20  148.37  145.30                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24 -129.41 -134.00   10.94    0.68  -62.50  175.43   32.33\n",
      "    4          5A   5A N   CA     24   25  156.93  147.00                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29  -67.06  -68.20    7.08    0.61  -62.50  166.87   27.57\n",
      "    5          6A   6A N   CA     29   30  152.29  145.30                  -40.90\n",
      "    6  10235   6A   7A C   N      32   34  -64.75  -68.20    4.34    0.28  -62.50  176.44   29.03\n",
      "    6          7A   7A N   CA     34   35  142.67  145.30                  -40.90\n",
      "    7  10237   8G   9A C   N      41   43  -76.00  -68.20    8.67    0.60  -62.50  170.54   28.63\n",
      "    7          9A   9A N   CA     43   44  149.10  145.30                  -40.90\n",
      "    8  10238   9A  10G C   N      46   48  -77.72  -80.20   14.64    0.46   82.20 -139.94    7.11\n",
      "    8         10G  10G N   CA     48   49  159.67  174.10                    8.50\n",
      "    9  10240  11P  12E C   N      57   59 -108.58 -117.80    9.82    0.29  -63.60  179.44   21.77\n",
      "    9         12E  12E N   CA     59   60  133.41  136.80                  -40.30\n",
      "   10  10241  12E  13M C   N      66   68 -136.58 -125.60   18.31    0.60  -63.40  179.90   32.63\n",
      "   10         13M  13M N   CA     68   69  155.16  140.50                  -40.50\n",
      "   11  10285  56I  57S C   N     407  409 -119.35 -136.60   18.05    0.59  -64.10 -172.55   17.42\n",
      "   11         57S  57S N   CA    409  410  145.88  151.20                  -35.00\n",
      "   12  10286  57S  58P C   N     413  415  -48.98  -58.70   37.59    2.42  -64.50  146.81   11.79\n",
      "   12         58P  58P N   CA    415  416  -66.82  -30.50                  147.20\n",
      "   13  10287  58P  59F C   N     420  422   54.54  -71.40  125.96    8.44  -63.20 -147.19   24.03\n",
      "   13         59F  59F N   CA    422  423  138.43  140.70                  -44.30\n",
      "   14  10288  59F  60E C   N     431  433   32.48  -69.30  117.94   10.61  -63.60  151.86   17.73\n",
      "   14         60E  60E N   CA    433  434 -157.90  142.50                  -40.30\n",
      "   15  10289  60E  61H C   N     440  442 -116.53  -63.20   53.41    8.60  -63.20   53.41    8.60\n",
      "   15         61H  61H N   CA    442  443  -45.04  -42.30                  -42.30\n",
      "   16  10291  62Q  63T C   N     459  461  167.36 -124.80   69.99    3.23  -63.20 -156.39   34.00\n",
      "   16         63T  63T N   CA    461  462  160.72  143.50                  -42.10\n",
      "   17  10292  63T  64Y C   N     466  468 -140.52  -63.50   96.27   13.72   55.90  165.51   17.27\n",
      "   17         64Y  64Y N   CA    468  469   14.36  -43.40                   39.50\n",
      "   18  10390 161C 162D C   N    1289 1291   66.34   54.50   39.33    2.08  -63.30  136.71   23.14\n",
      "   18        162D 162D N   CA   1291 1292    3.39   40.90                  -40.00\n",
      "   19  10482 253N 254C C   N    2021 2023  -60.74  -63.00    2.63    0.47 -117.90 -171.96    8.11\n",
      "   19        254C 254C N   CA   2023 2024  -39.76  -41.10                  141.10\n",
      "   20  10560 331F 332D C   N    2668 2670  -62.95  -70.90   17.23    1.24  -63.30  154.42   19.04\n",
      "   20        332D 332D N   CA   2670 2671  165.58  150.30                  -40.00\n",
      "   21  10561 332D 333M C   N    2676 2678  -60.51  -73.00   14.35    1.15  -63.40  169.46   25.08\n",
      "   21        333M 333M N   CA   2678 2679  150.06  143.00                  -40.50\n",
      "   22  10562 333M 334E C   N    2684 2686  -63.74  -69.30    7.13    0.40  -63.60  178.33   23.77\n",
      "   22        334E 334E N   CA   2686 2687  138.03  142.50                  -40.30\n",
      "   23  10563 334E 335L C   N    2693 2695 -129.08 -108.50   54.18    2.67  -63.50  151.14   25.04\n",
      "   23        335L 335L N   CA   2695 2696 -177.38  132.50                  -41.20\n",
      "   24  10564 335L 336D C   N    2701 2703  -69.39  -70.90   22.84    1.20  -63.30  147.04   18.52\n",
      "   24        336D 336D N   CA   2703 2704  173.09  150.30                  -40.00\n",
      "   25  10565 336D 337D C   N    2709 2711  -58.49  -70.90   28.40    0.95  -63.30  164.81   20.64\n",
      "   25        337D 337D N   CA   2711 2712  124.75  150.30                  -40.00\n",
      "   26  10566 337D 338L C   N    2717 2719  -78.53  -70.70    9.68    0.66  -63.50  172.17   24.79\n",
      "   26        338L 338L N   CA   2719 2720  147.28  141.60                  -41.20\n",
      "   27  10587 358Y 359R C   N    2896 2898  -73.69  -72.10    4.64    0.32  -63.00  172.97   24.09\n",
      "   27        359R 359R N   CA   2898 2899  146.26  141.90                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:   104  279   1.551\n",
      "  serious non-bonded atom clash:   650 1319   2.208\n",
      "  serious non-bonded atom clash:   650 1321   2.204\n",
      "  serious non-bonded atom clash:   651 1321   1.832\n",
      "  serious non-bonded atom clash:  1417 1630   1.840\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     3    0    2    1    4    8   15   92   97  161  166  237  338  439  505\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "4fuy_MODLL.B99990001.pdb       413.91162\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 4fv0\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    36969    34323\n",
      "253 (of 2915 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2762 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     253\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    34323    2189\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      548\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         206.5630\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     259       0      0   0.005   0.005      1.8945       1.000\n",
      " 2 Bond angle potential               :     355       0      0   1.863   1.863      23.067       1.000\n",
      " 3 Stereochemical cosine torsion poten:     147       0      9  57.475  57.475      72.213       1.000\n",
      " 4 Stereochemical improper torsion pot:     101       0      0   1.190   1.190      3.3953       1.000\n",
      " 5 Soft-sphere overlap restraints     :     548       0      0   0.005   0.005      1.4847       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     411       0      0   0.209   0.209      12.740       1.000\n",
      "10 Distance restraints 2 (N-O)        :     376       0      0   0.440   0.440      17.555       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      38       1      0  27.825  27.825      9.1076       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      25       0      0  83.402  83.402      7.6394       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      20       0      0  52.740  52.740      8.7053       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       8       0      0  76.727  76.727      4.4299       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       3       0      0  61.527  61.527      1.7651       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     243       0      0   0.498   0.498      1.5525       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      41      14      9  30.494 111.401      31.477       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     162       0      0   1.011   1.011      9.5363       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 4fv0_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    5398.8867\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 13                           : Mainchain Omega dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10840 253N 253N CA  C    2016 2021   -9.71 -179.18  169.47   39.39 -179.18  169.47   39.39\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9  -75.12  -68.20    9.26    0.59  -62.50  168.12   28.19\n",
      "    1          2A   2A N   CA      9   10  151.45  145.30                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14  -62.05  -68.20   22.02    1.56  -62.50  165.05   27.07\n",
      "    2          3A   3A N   CA     14   15  124.16  145.30                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19  -83.27  -68.20   21.42    1.35  -62.50  159.94   27.28\n",
      "    3          4A   4A N   CA     19   20  160.52  145.30                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24  -63.78  -68.20   10.21    0.68  -62.50  177.00   28.94\n",
      "    4          5A   5A N   CA     24   25  136.10  145.30                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29  -75.95  -68.20    8.09    0.76  -62.50  176.63   29.63\n",
      "    5          6A   6A N   CA     29   30  142.98  145.30                  -40.90\n",
      "    6  10235   6A   7A C   N      32   34 -132.81 -134.00   11.59    0.63  -62.50 -170.13   28.63\n",
      "    6          7A   7A N   CA     34   35  135.47  147.00                  -40.90\n",
      "    7  10236   7A   8G C   N      37   39   72.48   78.70   14.38    0.65   82.20  172.71    7.89\n",
      "    7          8G   8G N   CA     39   40 -179.06 -166.10                    8.50\n",
      "    8  10237   8G   9A C   N      41   43  -72.88  -68.20    6.89    0.43  -62.50  169.06   28.23\n",
      "    8          9A   9A N   CA     43   44  150.36  145.30                  -40.90\n",
      "    9  10389 160T 161C C   N    1283 1285   73.91   57.40   41.62    1.32 -117.90 -139.03    9.66\n",
      "    9        161C 161C N   CA   1285 1286   -2.21   36.00                  141.10\n",
      "   10  10481 252L 253N C   N    2013 2015  -93.16 -119.90   32.05    1.69  -63.20  166.93   22.57\n",
      "   10        253N 253N N   CA   2015 2016  154.68  137.00                  -41.10\n",
      "   11  10561 332D 333M C   N    2676 2678 -105.80 -125.60   21.71    0.67  -63.40  177.23   23.51\n",
      "   11        333M 333M N   CA   2678 2679  131.58  140.50                  -40.50\n",
      "   12  10562 333M 334E C   N    2684 2686 -119.98 -117.80   69.38    3.52  -63.60  121.62   14.09\n",
      "   12        334E 334E N   CA   2686 2687   67.46  136.80                  -40.30\n",
      "   13  10563 334E 335L C   N    2693 2695 -164.34 -108.50   55.92    2.63  -63.50 -156.42   24.95\n",
      "   13        335L 335L N   CA   2695 2696  135.66  132.50                  -41.20\n",
      "   14  10587 358Y 359R C   N    2896 2898 -111.32 -125.20   14.44    0.68  -63.00 -179.10   27.78\n",
      "   14        359R 359R N   CA   2898 2899  144.57  140.60                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:   247  474   2.164\n",
      "  serious non-bonded atom clash:   248  472   2.189\n",
      "  serious non-bonded atom clash:   248  474   1.461\n",
      "  serious non-bonded atom clash:   249  256   2.275\n",
      "  serious non-bonded atom clash:   249  474   1.597\n",
      "  serious non-bonded atom clash:   249  476   1.751\n",
      "  serious non-bonded atom clash:   249  477   2.156\n",
      "  serious non-bonded atom clash:   250  472   2.225\n",
      "  serious non-bonded atom clash:   251  414   2.240\n",
      "  serious non-bonded atom clash:   254  414   2.038\n",
      "  serious non-bonded atom clash:   416  473   2.055\n",
      "  serious non-bonded atom clash:   418  473   1.383\n",
      "  serious non-bonded atom clash:   418  475   1.609\n",
      "  serious non-bonded atom clash:   650 1319   2.268\n",
      "  serious non-bonded atom clash:   650 1321   2.258\n",
      "  serious non-bonded atom clash:   651 1321   1.869\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     8    3    5    3    6   11   18   63  102  155  186  231  355  438  493\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "4fv0_MODLL.B99990001.pdb       206.56297\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 4fv1\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    37473    34827\n",
      "217 (of 2915 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2770 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     217\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    34827    1931\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      346\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         156.0365\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     224       0      0   0.004   0.004     0.95746       1.000\n",
      " 2 Bond angle potential               :     307       0      0   1.805   1.805      18.720       1.000\n",
      " 3 Stereochemical cosine torsion poten:     138       0      4  53.776  53.776      58.694       1.000\n",
      " 4 Stereochemical improper torsion pot:      91       0      0   0.991   0.991      2.1323       1.000\n",
      " 5 Soft-sphere overlap restraints     :     346       0      0   0.002   0.002     0.13202       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     326       0      0   0.135   0.135      4.4851       1.000\n",
      "10 Distance restraints 2 (N-O)        :     314       0      1   0.292   0.292      14.941       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      32       0      0   3.289   3.289      3.4073       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      20       0      0  64.665  64.665     0.50155       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      16       0      0  71.909  71.909      4.0494       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       9       0      0  80.899  80.899      5.7487       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       2       0      0  87.768  87.768      1.9326       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     226       0      0   0.489   0.489      1.0732       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      34      16      5  23.057 116.515      27.035       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     192       0      0   0.846   0.846      12.227       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 4fv1_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    4759.3843\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9  -72.92  -68.20   13.13    0.90  -62.50  161.88   27.05\n",
      "    1          2A   2A N   CA      9   10  157.55  145.30                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14 -112.77 -134.00   22.99    1.10  -62.50  170.84   30.67\n",
      "    2          3A   3A N   CA     14   15  155.82  147.00                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19  -71.76  -68.20    4.25    0.28  -62.50  171.73   28.61\n",
      "    3          4A   4A N   CA     19   20  147.62  145.30                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24  -76.42  -68.20   12.73    0.80  -62.50  164.67   27.69\n",
      "    4          5A   5A N   CA     24   25  155.02  145.30                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29  -69.98  -68.20   15.88    1.22  -62.50  158.20   26.30\n",
      "    5          6A   6A N   CA     29   30  161.08  145.30                  -40.90\n",
      "    6  10235   6A   7A C   N      32   34  -59.93  -68.20   11.36    1.12  -62.50  166.03   27.09\n",
      "    6          7A   7A N   CA     34   35  153.09  145.30                  -40.90\n",
      "    7  10236   7A   8G C   N      37   39  105.98   78.70   65.52    0.98   82.20  128.05    7.38\n",
      "    7          8G   8G N   CA     39   40  134.33 -166.10                    8.50\n",
      "    8  10237   8G   9A C   N      41   43 -145.83 -134.00   20.60    0.76  -62.50  176.19   33.33\n",
      "    8          9A   9A N   CA     43   44  163.87  147.00                  -40.90\n",
      "    9  10238   9A  10G C   N      46   48   82.60   78.70    8.58    0.13   82.20  177.76    8.65\n",
      "    9         10G  10G N   CA     48   49 -173.74 -166.10                    8.50\n",
      "   10  10389 160T 161C C   N    1283 1285   74.55   57.40   24.48    1.37 -117.90 -152.41    8.96\n",
      "   10        161C 161C N   CA   1285 1286   18.54   36.00                  141.10\n",
      "   11  10559 330K 331F C   N    2657 2659  -78.01  -71.40    6.86    0.41  -63.20  173.80   25.13\n",
      "   11        331F 331F N   CA   2659 2660  142.53  140.70                  -44.30\n",
      "   12  10561 332D 333M C   N    2676 2678 -145.33 -125.60   19.73    0.79  -63.40 -162.85   35.89\n",
      "   12        333M 333M N   CA   2678 2679  140.18  140.50                  -40.50\n",
      "   13  10562 333M 334E C   N    2684 2686  -59.07  -69.30   25.25    1.56   54.60  137.30   10.32\n",
      "   13        334E 334E N   CA   2686 2687  119.42  142.50                   42.40\n",
      "   14  10585 356P 357G C   N    2880 2882 -168.81 -167.20   46.99    1.86   82.20  161.46   12.27\n",
      "   14        357G 357G N   CA   2882 2883  127.63  174.60                    8.50\n",
      "   15  10586 357G 358Y C   N    2884 2886 -121.39 -124.30   13.14    0.58  -63.50  175.80   25.24\n",
      "   15        358Y 358Y N   CA   2886 2887  122.59  135.40                  -43.40\n",
      "   16  10587 358Y 359R C   N    2896 2898   51.44   57.30    7.64    0.54  -63.00  141.96   25.82\n",
      "   16        359R 359R N   CA   2898 2899   42.90   38.00                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:   250  256   2.294\n",
      "  serious non-bonded atom clash:   254  508   2.187\n",
      "  serious non-bonded atom clash:   254  509   1.102\n",
      "  serious non-bonded atom clash:  2257 2259   2.058\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     2    1    1    1    0    4   17   41   88  133  172  210  329  432  518\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "4fv1_MODLL.B99990001.pdb       156.03647\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 4fv2\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    37661    35015\n",
      "197 (of 2915 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2793 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     197\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    35015    1776\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      310\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         123.5324\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     203       0      0   0.004   0.004     0.80669       1.000\n",
      " 2 Bond angle potential               :     279       0      0   1.836   1.836      17.884       1.000\n",
      " 3 Stereochemical cosine torsion poten:     121       0      3  52.478  52.478      52.742       1.000\n",
      " 4 Stereochemical improper torsion pot:      79       0      0   1.171   1.171      2.0997       1.000\n",
      " 5 Soft-sphere overlap restraints     :     310       0      0   0.001   0.001     0.52432E-01   1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     319       0      0   0.114   0.114      3.1113       1.000\n",
      "10 Distance restraints 2 (N-O)        :     294       0      0   0.265   0.265      8.1307       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      30       0      0   3.599   3.599      3.9185       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      18       0      0  39.052  39.052     -1.1954       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      14       0      0  44.428  44.428      3.3341       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       8       0      0  70.811  70.811      4.3693       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       2       0      0 140.737 140.737      1.5733       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     217       0      0   0.559   0.559      1.8399       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      32      14      3  21.806 121.156      20.012       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     160       0      0   0.775   0.775      4.8547       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 4fv2_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    4212.6211\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9  -68.93  -68.20    3.91    0.29  -62.50  170.08   28.19\n",
      "    1          2A   2A N   CA      9   10  149.14  145.30                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14 -123.89 -134.00   17.56    1.07  -62.50  169.26   31.02\n",
      "    2          3A   3A N   CA     14   15  161.37  147.00                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19 -118.39 -134.00   19.67    1.10  -62.50  169.60   30.78\n",
      "    3          4A   4A N   CA     19   20  158.98  147.00                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24 -111.18 -134.00   23.41    0.98  -62.50  173.85   31.08\n",
      "    4          5A   5A N   CA     24   25  152.21  147.00                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29  -65.68  -68.20    5.95    0.56  -62.50  168.44   27.76\n",
      "    5          6A   6A N   CA     29   30  150.69  145.30                  -40.90\n",
      "    6  10235   6A   7A C   N      32   34  -70.56  -68.20   11.20    0.82  -62.50  163.05   27.12\n",
      "    6          7A   7A N   CA     34   35  156.25  145.30                  -40.90\n",
      "    7  10237   8G   9A C   N      41   43 -156.63 -134.00   27.31    0.67  -62.50 -177.09   34.99\n",
      "    7          9A   9A N   CA     43   44  162.28  147.00                  -40.90\n",
      "    8  10238   9A  10G C   N      46   48 -128.67 -167.20   45.43    0.77   82.20 -154.06   15.86\n",
      "    8         10G  10G N   CA     48   49  150.54  174.60                    8.50\n",
      "    9  10389 160T 161C C   N    1283 1285   73.49   57.40   19.66    1.45 -117.90 -155.11    8.80\n",
      "    9        161C 161C N   CA   1285 1286   24.70   36.00                  141.10\n",
      "   10  10561 332D 333M C   N    2676 2678 -134.70 -125.60   10.67    0.55  -63.40 -170.64   23.88\n",
      "   10        333M 333M N   CA   2678 2679  134.92  140.50                  -40.50\n",
      "   11  10562 333M 334E C   N    2684 2686  -68.19  -69.30   14.53    1.07   54.60  149.69   11.09\n",
      "   11        334E 334E N   CA   2686 2687  128.02  142.50                   42.40\n",
      "   12  10585 356P 357G C   N    2880 2882  -65.41  -62.40    3.47    0.53  -80.20  147.18    6.54\n",
      "   12        357G 357G N   CA   2882 2883  -39.46  -41.20                  174.10\n",
      "   13  10586 357G 358Y C   N    2884 2886  -87.57  -98.40   14.27    1.12  -63.50  164.29   24.87\n",
      "   13        358Y 358Y N   CA   2886 2887  119.12  128.40                  -43.40\n",
      "   14  10587 358Y 359R C   N    2896 2898 -109.93 -125.20   20.23    1.10  -63.00  171.58   26.42\n",
      "   14        359R 359R N   CA   2898 2899  153.86  140.60                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:    90  232   1.525\n",
      "  serious non-bonded atom clash:   250  414   1.779\n",
      "  serious non-bonded atom clash:   251  413   1.924\n",
      "  serious non-bonded atom clash:   251  415   1.618\n",
      "  serious non-bonded atom clash:   251  416   2.198\n",
      "  serious non-bonded atom clash:   251  417   2.243\n",
      "  serious non-bonded atom clash:   252  413   1.680\n",
      "  serious non-bonded atom clash:   252  414   1.132\n",
      "  serious non-bonded atom clash:   252  415   1.948\n",
      "  serious non-bonded atom clash:   252  416   1.814\n",
      "  serious non-bonded atom clash:   253  413   1.070\n",
      "  serious non-bonded atom clash:   253  414   1.721\n",
      "  serious non-bonded atom clash:   253  415   0.655\n",
      "  serious non-bonded atom clash:   253  416   1.431\n",
      "  serious non-bonded atom clash:   253  417   2.083\n",
      "  serious non-bonded atom clash:   254  413   1.658\n",
      "  serious non-bonded atom clash:   254  415   0.871\n",
      "  serious non-bonded atom clash:   254  416   1.990\n",
      "  serious non-bonded atom clash:   254  417   1.466\n",
      "  serious non-bonded atom clash:   431 2739   1.081\n",
      "  serious non-bonded atom clash:   431 2740   1.801\n",
      "  serious non-bonded atom clash:   432 2738   1.658\n",
      "  serious non-bonded atom clash:   432 2739   0.828\n",
      "  serious non-bonded atom clash:   432 2740   2.003\n",
      "  serious non-bonded atom clash:   433 2739   1.747\n",
      "  serious non-bonded atom clash:   433 2740   1.217\n",
      "  serious non-bonded atom clash:   434 2739   2.284\n",
      "  serious non-bonded atom clash:   434 2740   0.834\n",
      "  serious non-bonded atom clash:   440 2740   1.665\n",
      "  serious non-bonded atom clash:   490  529   1.875\n",
      "  serious non-bonded atom clash:   491  529   1.176\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:    28    1    2    4   10   11   23   52   88  153  166  225  344  450  498\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "4fv2_MODLL.B99990001.pdb       123.53244\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 4fv3\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    37852    35206\n",
      "144 (of 2915 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2616 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     144\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    35206    1179\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      258\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          81.0833\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     148       0      0   0.003   0.003     0.45363       1.000\n",
      " 2 Bond angle potential               :     205       0      0   1.857   1.857      13.109       1.000\n",
      " 3 Stereochemical cosine torsion poten:      90       0      2  52.740  52.740      39.146       1.000\n",
      " 4 Stereochemical improper torsion pot:      57       0      0   0.804   0.804     0.76321       1.000\n",
      " 5 Soft-sphere overlap restraints     :     258       0      0   0.002   0.002     0.14784       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     198       0      0   0.123   0.123      1.8213       1.000\n",
      "10 Distance restraints 2 (N-O)        :     194       0      0   0.267   0.267      3.1960       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      23       0      0   3.359   3.359      2.2285       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      12       0      0  76.359  76.359      1.2339       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       8       0      0  56.387  56.387      1.5657       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      0   7.687   7.687      1.5820       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       1       0      0 137.124 137.124     0.93745       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     138       0      0   0.328   0.328     0.31881       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      24      13      2  18.493 132.045      14.021       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      76       0      0   0.432   0.432     0.55883       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 4fv3_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    3476.1091\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9 -133.21 -134.00    7.58    0.45  -62.50  179.11   33.14\n",
      "    1          2A   2A N   CA      9   10  154.54  147.00                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14  -66.77  -68.20    2.67    0.17  -62.50  176.10   29.07\n",
      "    2          3A   3A N   CA     14   15  143.05  145.30                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19  -77.88  -68.20   14.37    1.41  -62.50  176.25   28.17\n",
      "    3          4A   4A N   CA     19   20  134.68  145.30                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24 -123.80 -134.00   11.18    0.25  -62.50 -172.98   33.92\n",
      "    4          5A   5A N   CA     24   25  142.42  147.00                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29 -114.59 -134.00   26.16    0.74  -62.50  178.14   27.16\n",
      "    5          6A   6A N   CA     29   30  129.45  147.00                  -40.90\n",
      "    6  10235   6A   7A C   N      32   34  -73.53  -68.20    7.79    0.49  -62.50  168.48   28.17\n",
      "    6          7A   7A N   CA     34   35  150.98  145.30                  -40.90\n",
      "    7  10236   7A   8G C   N      37   39  174.00 -167.20   20.27    0.30   82.20 -163.57   13.68\n",
      "    7          8G   8G N   CA     39   40 -177.83  174.60                    8.50\n",
      "    8  10237   8G   9A C   N      41   43  -75.12  -68.20   17.86    1.21  -62.50  157.84   26.50\n",
      "    8          9A   9A N   CA     43   44  161.77  145.30                  -40.90\n",
      "    9  10238   9A  10G C   N      46   48 -175.25 -167.20   16.83    0.68   82.20 -177.21   13.32\n",
      "    9         10G  10G N   CA     48   49  159.81  174.60                    8.50\n",
      "   10  10389 160T 161C C   N    1283 1285   71.51   57.40   17.68    1.25 -117.90 -153.84    8.84\n",
      "   10        161C 161C N   CA   1285 1286   25.34   36.00                  141.10\n",
      "   11  10585 356P 357G C   N    2880 2882  -58.05  -62.40    5.00    0.77  -80.20  143.95    6.79\n",
      "   11        357G 357G N   CA   2882 2883  -43.67  -41.20                  174.10\n",
      "   12  10586 357G 358Y C   N    2884 2886 -122.27 -124.30    7.78    0.33  -63.50 -178.90   26.04\n",
      "   12        358Y 358Y N   CA   2886 2887  127.89  135.40                  -43.40\n",
      "   13  10587 358Y 359R C   N    2896 2898  -88.75  -72.10   18.82    1.26  -63.00  170.20   24.76\n",
      "   13        359R 359R N   CA   2898 2899  150.66  141.90                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:   431 2739   1.153\n",
      "  serious non-bonded atom clash:   431 2740   1.701\n",
      "  serious non-bonded atom clash:   432 2738   1.900\n",
      "  serious non-bonded atom clash:   432 2739   0.977\n",
      "  serious non-bonded atom clash:   432 2740   1.911\n",
      "  serious non-bonded atom clash:   433 2739   1.827\n",
      "  serious non-bonded atom clash:   433 2740   1.172\n",
      "  serious non-bonded atom clash:   434 2740   0.908\n",
      "  serious non-bonded atom clash:   440 2740   1.736\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     9    0    0    4    2    4   18   54   61  100  178  223  331  426  539\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "4fv3_MODLL.B99990001.pdb        81.08333\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 4fv4\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    35851    33205\n",
      "366 (of 2915 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2664 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     366\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    33205    3078\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):     1072\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :        1252.1281\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     379       3      9   0.041   0.041      165.49       1.000\n",
      " 2 Bond angle potential               :     524       8     16   4.816   4.816      243.32       1.000\n",
      " 3 Stereochemical cosine torsion poten:     243       0     12  51.665  51.665      100.94       1.000\n",
      " 4 Stereochemical improper torsion pot:     169       0      2   1.804   1.804      12.731       1.000\n",
      " 5 Soft-sphere overlap restraints     :    1072      12     26   0.062   0.062      483.67       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     581       0      0   0.368   0.368      45.515       1.000\n",
      "10 Distance restraints 2 (N-O)        :     526       0      5   0.466   0.466      39.297       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      55       2      4   7.733   7.733      37.611       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      34       0      2  80.500  80.500      14.383       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      25       0      0  78.805  78.805      13.696       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       9       0      0  76.649  76.649      3.7743       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       2       0      0  97.563  97.563      1.1775       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     286       0      0   0.410   0.410      1.0258       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      59      29     20  41.077 129.331      78.778       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     186       0      0   0.813   0.813      10.718       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 4fv4_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    9876.7168\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  1                           : Bond length potential                   \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   1480 183F 183F CB  CA   1453 1452    1.79    1.54    0.25    6.79    1.54    0.25    6.79\n",
      "    2   1490 183F 183F C   CA   1460 1452    2.13    1.49    0.64   18.49    1.49    0.64   18.49\n",
      "    3   1496 184L 184L N   CA   1462 1463    1.63    1.43    0.20    6.66    1.43    0.20    6.66\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   4989 183F 183F N   CA   1451 1452  132.97  107.00   25.97    7.47  107.00   25.97    7.47\n",
      "    2   4990 183F 183F CA  CB   1452 1453  142.20  107.50   34.70    8.02  107.50   34.70    8.02\n",
      "    3   4992 183F 183F CA  C    1452 1460  140.26  116.50   23.77    6.83  116.50   23.77    6.83\n",
      "    4   5001 183F 183F C   CA   1460 1452  137.42  108.00   29.42    6.82  108.00   29.42    6.82\n",
      "    5   5005 184L 184L N   CA   1462 1463  126.30  107.00   19.30    5.55  107.00   19.30    5.55\n",
      "    6   5016 185T 185T N   CA   1470 1471  122.91  107.00   15.91    4.58  107.00   15.91    4.58\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 13                           : Mainchain Omega dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10770 183F 183F CA  C    1452 1460 -149.72 -180.00   30.28    6.05 -180.00   30.28    6.05\n",
      "    2  10771 184L 184L CA  C    1463 1468 -155.20 -180.00   24.80    4.96 -180.00   24.80    4.96\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9  -73.18  -68.20   16.18    1.13  -62.50  158.77   26.56\n",
      "    1          2A   2A N   CA      9   10  160.70  145.30                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14  -65.50  -68.20   12.04    1.06  -62.50  162.09   26.71\n",
      "    2          3A   3A N   CA     14   15  157.04  145.30                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19  -68.13  -68.20   12.65    1.02  -62.50  161.25   26.70\n",
      "    3          4A   4A N   CA     19   20  157.95  145.30                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24  -67.00  -68.20   13.77    1.15  -62.50  160.15   26.47\n",
      "    4          5A   5A N   CA     24   25  159.02  145.30                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29  -65.10  -68.20    3.78    0.37  -62.50  171.66   28.26\n",
      "    5          6A   6A N   CA     29   30  147.47  145.30                  -40.90\n",
      "    6  10236   7A   8G C   N      37   39  -92.05  -80.20   16.40    0.54   82.20 -111.66    7.70\n",
      "    6          8G   8G N   CA     39   40 -174.56  174.10                    8.50\n",
      "    7  10263  34G  35A C   N     238  240  -62.26  -68.20   26.67    1.94  -62.50  160.20   26.26\n",
      "    7         35A  35A N   CA    240  241  119.30  145.30                  -40.90\n",
      "    8  10264  35A  36Y C   N     243  245   -6.28  -98.40   92.39    3.72  -63.50  174.41   30.56\n",
      "    8         36Y  36Y N   CA    245  246  121.36  128.40                  -43.40\n",
      "    9  10265  36Y  37G C   N     255  257   58.47   78.70   21.74    0.85  -80.20  141.43   10.10\n",
      "    9         37G  37G N   CA    257  258 -158.14 -166.10                  174.10\n",
      "   10  10389 160T 161C C   N    1283 1285   71.48   57.40   17.74    1.24 -117.90 -153.74    8.84\n",
      "   10        161C 161C N   CA   1285 1286   25.20   36.00                  141.10\n",
      "   11  10405 176P 177D C   N    1402 1404 -145.88  -96.50   51.80    2.13  -63.30 -171.15   20.06\n",
      "   11        177D 177D N   CA   1404 1405  129.84  114.20                  -40.00\n",
      "   12  10406 177D 178H C   N    1410 1412 -158.23 -125.60   32.79    1.21  -63.20 -158.37   21.44\n",
      "   12        178H 178H N   CA   1412 1413  135.53  138.80                  -42.30\n",
      "   13  10407 178H 179D C   N    1420 1422 -172.05  -96.50   76.73    3.12  -63.30  177.90   19.28\n",
      "   13        179D 179D N   CA   1422 1423  100.79  114.20                  -40.00\n",
      "   14  10408 179D 180H C   N    1428 1430  173.35 -125.60   74.96    1.86  -63.20 -176.76   30.49\n",
      "   14        180H 180H N   CA   1430 1431 -177.71  138.80                  -42.30\n",
      "   15  10409 180H 181T C   N    1438 1440 -114.13 -124.80   53.29    2.60  -63.20  132.38   20.33\n",
      "   15        181T 181T N   CA   1440 1441 -164.29  143.50                  -42.10\n",
      "   16  10411 182G 183F C   N    1449 1451 -142.17 -124.20   27.89    0.79  -63.20  170.46   28.72\n",
      "   16        183F 183F N   CA   1451 1452  164.63  143.30                  -44.30\n",
      "   17  10412 183F 184L C   N    1460 1462  -90.90 -108.50   55.66    2.79  -63.50  123.96   15.86\n",
      "   17        184L 184L N   CA   1462 1463   79.69  132.50                  -41.20\n",
      "   18  10413 184L 185T C   N    1468 1470 -162.12 -124.80   41.37    2.49  -63.20 -165.26   20.58\n",
      "   18        185T 185T N   CA   1470 1471  125.64  143.50                  -42.10\n",
      "   19  10414 185T 186E C   N    1475 1477  104.63 -117.80  138.56    5.50  -63.60 -127.49   39.52\n",
      "   19        186E 186E N   CA   1477 1478  120.20  136.80                  -40.30\n",
      "   20  10415 186E 187Y C   N    1484 1486 -170.77 -124.30   62.74    2.88  -63.50  175.63   32.73\n",
      "   20        187Y 187Y N   CA   1486 1487  177.54  135.40                  -43.40\n",
      "   21  10417 188V 189A C   N    1503 1505  154.38 -134.00   76.19    1.77  -68.20  140.18   10.90\n",
      "   21        189A 189A N   CA   1505 1506  173.01  147.00                  145.30\n",
      "   22  10428 199M 200L C   N    1600 1602  -46.28  -70.70   29.69    2.99  -63.50  161.24   21.46\n",
      "   22        200L 200L N   CA   1602 1603  158.48  141.60                  -41.20\n",
      "   23  10430 201N 202S C   N    1616 1618  -59.52  -72.40   12.97    0.97  -64.10  171.18   12.14\n",
      "   23        202S 202S N   CA   1618 1619  153.88  152.40                  -35.00\n",
      "   24  10431 202S 203K C   N    1622 1624 -103.72 -118.00   15.42    0.69  -62.90  179.01   25.65\n",
      "   24        203K 203K N   CA   1624 1625  144.91  139.10                  -40.80\n",
      "   25  10432 203K 204G C   N    1631 1633   77.20   78.70   31.13    0.95   82.20  154.39    7.24\n",
      "   25        204G 204G N   CA   1633 1634  162.81 -166.10                    8.50\n",
      "   26  10433 204G 205Y C   N    1635 1637 -151.17 -124.30   86.41    3.60   55.90 -125.30   27.08\n",
      "   26        205Y 205Y N   CA   1637 1638 -142.47  135.40                   39.50\n",
      "   27  10585 356P 357G C   N    2880 2882   75.44   82.20   35.95    1.40  -80.20 -157.03    7.31\n",
      "   27        357G 357G N   CA   2882 2883   43.81    8.50                  174.10\n",
      "   28  10586 357G 358Y C   N    2884 2886 -110.36  -98.40   13.71    0.90  -63.50 -175.44   27.05\n",
      "   28        358Y 358Y N   CA   2886 2887  135.12  128.40                  -43.40\n",
      "   29  10587 358Y 359R C   N    2896 2898 -118.56 -125.20    8.83    0.26  -63.00 -175.55   21.76\n",
      "   29        359R 359R N   CA   2898 2899  134.78  140.60                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:  1452 2052   2.043\n",
      "  serious non-bonded atom clash:  1452 2077   2.219\n",
      "  serious non-bonded atom clash:  1460 2040   2.188\n",
      "  serious non-bonded atom clash:  1460 2052   2.031\n",
      "  serious non-bonded atom clash:  1461 2052   2.215\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     2    1    2    3    7   17   30   69   92  119  221  254  358  408  510\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "4fv4_MODLL.B99990001.pdb      1252.12805\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 4fv5\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    36341    33695\n",
      "331 (of 2915 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2667 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     331\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    33695    2713\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):     1000\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :        2408.5862\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     342      10     20   0.077   0.077      542.69       1.000\n",
      " 2 Bond angle potential               :     474       9     28   4.948   4.948      225.39       1.000\n",
      " 3 Stereochemical cosine torsion poten:     223       0     13  54.686  54.686      100.61       1.000\n",
      " 4 Stereochemical improper torsion pot:     152       3      4   4.432   4.432      60.071       1.000\n",
      " 5 Soft-sphere overlap restraints     :    1000      33     60   0.092   0.092      991.88       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     475       2     14   0.615   0.615      98.589       1.000\n",
      "10 Distance restraints 2 (N-O)        :     461       1      7   0.531   0.531      51.055       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      49       3      4  15.053  15.053      129.37       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      32       0      3  75.120  75.120      17.356       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      21       0      0  72.567  72.567      10.361       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       7       0      0  88.841  88.841      2.3549       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       3       0      0  92.632  92.632      2.1713       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     241       0      0   0.598   0.598      2.3481       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      52      29     17  47.233 128.676      133.83       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     181       0      5   1.508   1.508      40.505       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 4fv5_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:   10181.9609\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  1                           : Bond length potential                   \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   1515 186E 187Y C   N    1484 1486    1.52    1.35    0.18    6.33    1.35    0.18    6.33\n",
      "    2   1516 187Y 187Y CB  CA   1488 1487    2.20    1.54    0.66   18.08    1.54    0.66   18.08\n",
      "    3   1517 187Y 187Y CG  CB   1489 1488    1.77    1.49    0.28    7.86    1.49    0.28    7.86\n",
      "    4   1525 187Y 187Y N   CA   1486 1487    1.70    1.43    0.27    8.95    1.43    0.27    8.95\n",
      "    5   1527 187Y 187Y C   CA   1496 1487    2.32    1.49    0.83   24.07    1.49    0.83   24.07\n",
      "    6   1528 187Y 188V C   N    1496 1498    1.69    1.35    0.34   12.22    1.35    0.34   12.22\n",
      "    7   1532 188V 188V N   CA   1498 1499    1.84    1.43    0.41   13.42    1.43    0.41   13.42\n",
      "    8   1534 188V 188V C   CA   1503 1499    1.91    1.49    0.42   12.29    1.49    0.42   12.29\n",
      "    9   1535 188V 189A C   N    1503 1505    1.53    1.35    0.18    6.42    1.35    0.18    6.42\n",
      "   10   1537 189A 189A N   CA   1505 1506    1.60    1.43    0.17    5.73    1.43    0.17    5.73\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   5002 183F 184L C   N    1460 1462  143.69  120.00   23.69    5.38  120.00   23.69    5.38\n",
      "    2   5005 184L 184L N   CA   1462 1463  128.81  107.00   21.81    6.27  107.00   21.81    6.27\n",
      "    3   5013 184L 185T C   N    1468 1470  151.80  120.00   31.80    7.23  120.00   31.80    7.23\n",
      "    4   5016 185T 185T N   CA   1470 1471  128.57  107.00   21.57    6.20  107.00   21.57    6.20\n",
      "    5   5026 186E 186E N   CA   1477 1478  124.91  107.00   17.91    5.15  107.00   17.91    5.15\n",
      "    6   5029 186E 186E CA  C    1478 1484  134.39  116.50   17.89    5.14  116.50   17.89    5.14\n",
      "    7   5056 188V 188V N   CA   1498 1499  137.88  107.00   30.88    8.88  107.00   30.88    8.88\n",
      "    8   5063 188V 189A C   N    1503 1505  143.19  120.00   23.19    5.27  120.00   23.19    5.27\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  4                           : Stereochemical improper torsion potentia\n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   9620 187Y 187Y CB  CG   1488 1489 -164.83 -180.00   15.17    4.62 -180.00   15.17    4.62\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  9                           : Distance restraints 1 (CA-CA)           \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  12338  34G 189A CA  CA    237 1506   15.99   12.67    3.32    5.71   12.67    3.32    5.71\n",
      "    2  12365  35A 189A CA  CA    241 1506   14.88   11.97    2.91    5.23   11.97    2.91    5.23\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 13                           : Mainchain Omega dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10777 190T 190T CA  C    1511 1515 -143.20 -180.00   36.80    7.36 -180.00   36.80    7.36\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9  -63.40  -68.20    5.68    0.56  -62.50  170.76   28.03\n",
      "    1          2A   2A N   CA      9   10  148.34  145.30                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14  -70.81  -68.20    7.98    0.55  -62.50  166.47   27.70\n",
      "    2          3A   3A N   CA     14   15  152.84  145.30                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19  -68.48  -68.20    5.92    0.47  -62.50  167.99   27.83\n",
      "    3          4A   4A N   CA     19   20  151.21  145.30                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24  -67.77  -68.20    1.62    0.14  -62.50  172.32   28.50\n",
      "    4          5A   5A N   CA     24   25  146.86  145.30                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29 -134.60 -134.00   10.91    0.61  -62.50  176.60   32.80\n",
      "    5          6A   6A N   CA     29   30  157.89  147.00                  -40.90\n",
      "    6  10236   7A   8G C   N      37   39 -168.08 -167.20   23.41    0.93   82.20 -179.99   13.38\n",
      "    6          8G   8G N   CA     39   40  151.21  174.60                    8.50\n",
      "    7  10237   8G   9A C   N      41   43  -74.39  -68.20    6.84    0.48  -62.50  171.31   28.67\n",
      "    7          9A   9A N   CA     43   44  148.21  145.30                  -40.90\n",
      "    8  10389 160T 161C C   N    1283 1285  113.76   57.40   59.55    7.73 -117.90  154.42    6.61\n",
      "    8        161C 161C N   CA   1285 1286   55.22   36.00                  141.10\n",
      "    9  10403 174A 175D C   N    1387 1389 -144.61  -96.50   61.69    2.57  -63.30 -174.09   28.19\n",
      "    9        175D 175D N   CA   1389 1390  152.81  114.20                  -40.00\n",
      "   10  10405 176P 177D C   N    1402 1404 -126.40  -70.90   64.79    3.36  -63.30  150.18   22.61\n",
      "   10        177D 177D N   CA   1404 1405 -176.28  150.30                  -40.00\n",
      "   11  10406 177D 178H C   N    1410 1412 -145.30 -125.60   37.30    1.01  -63.20  168.57   26.38\n",
      "   11        178H 178H N   CA   1412 1413  170.48  138.80                  -42.30\n",
      "   12  10407 178H 179D C   N    1420 1422 -123.57  -96.50   27.09    1.10  -63.30  164.91   17.68\n",
      "   12        179D 179D N   CA   1422 1423  113.50  114.20                  -40.00\n",
      "   13  10408 179D 180H C   N    1428 1430 -138.90  -63.20   81.26   11.10  -63.20   81.26   11.10\n",
      "   13        180H 180H N   CA   1430 1431  -12.78  -42.30                  -42.30\n",
      "   14  10409 180H 181T C   N    1438 1440 -142.34 -124.80   97.11    4.68  -63.20  119.92   12.92\n",
      "   14        181T 181T N   CA   1440 1441   47.99  143.50                  -42.10\n",
      "   15  10410 181T 182G C   N    1445 1447  -94.29  -80.20   14.57    0.83   82.20 -115.43    7.83\n",
      "   15        182G 182G N   CA   1447 1448  177.81  174.10                    8.50\n",
      "   16  10411 182G 183F C   N    1449 1451  145.77 -124.20   92.13    3.03  -63.20 -145.12   38.40\n",
      "   16        183F 183F N   CA   1451 1452  162.85  143.30                  -44.30\n",
      "   17  10412 183F 184L C   N    1460 1462  -46.23  -70.70   50.43    3.13  -63.50  139.78   20.44\n",
      "   17        184L 184L N   CA   1462 1463   97.51  141.60                  -41.20\n",
      "   18  10413 184L 185T C   N    1468 1470  170.49 -124.80   76.11    2.84  -63.20 -175.60   31.15\n",
      "   18        185T 185T N   CA   1470 1471 -176.44  143.50                  -42.10\n",
      "   19  10414 185T 186E C   N    1475 1477  152.92 -117.80   94.45    4.25  -63.60 -155.09   24.38\n",
      "   19        186E 186E N   CA   1477 1478  105.99  136.80                  -40.30\n",
      "   20  10415 186E 187Y C   N    1484 1486  -98.13  -63.50   92.99   16.48  -63.50   92.99   16.48\n",
      "   20        187Y 187Y N   CA   1486 1487 -129.71  -43.40                  -43.40\n",
      "   21  10416 187Y 188V C   N    1496 1498  159.30  -62.40  138.32   21.85 -125.40 -168.20    7.30\n",
      "   21        188V 188V N   CA   1498 1499  -40.30  -42.40                  143.30\n",
      "   22  10417 188V 189A C   N    1503 1505  -61.16  -68.20  120.95   10.00 -134.00  139.57    8.56\n",
      "   22        189A 189A N   CA   1505 1506  -93.95  145.30                  147.00\n",
      "   23  10429 200L 201N C   N    1608 1610 -124.08 -119.90    7.24    0.39  -63.20 -177.36   20.14\n",
      "   23        201N 201N N   CA   1610 1611  131.09  137.00                  -41.10\n",
      "   24  10431 202S 203K C   N    1622 1624 -106.75 -118.00   18.07    0.67  -62.90  171.46   20.30\n",
      "   24        203K 203K N   CA   1624 1625  124.96  139.10                  -40.80\n",
      "   25  10432 203K 204G C   N    1631 1633 -155.31 -167.20   55.36    1.99   82.20  165.99   12.82\n",
      "   25        204G 204G N   CA   1633 1634  120.53  174.60                    8.50\n",
      "   26  10433 204G 205Y C   N    1635 1637 -120.98 -124.30   22.69    1.07  -63.50  166.59   23.83\n",
      "   26        205Y 205Y N   CA   1637 1638  112.95  135.40                  -43.40\n",
      "   27  10585 356P 357G C   N    2880 2882   95.43   78.70   35.37    0.53  -80.20  176.00   11.37\n",
      "   27        357G 357G N   CA   2882 2883  162.74 -166.10                  174.10\n",
      "   28  10586 357G 358Y C   N    2884 2886 -121.04 -124.30    3.90    0.32  -63.50 -171.94   32.75\n",
      "   28        358Y 358Y N   CA   2886 2887  137.56  135.40                  -43.40\n",
      "   29  10587 358Y 359R C   N    2896 2898 -112.83 -125.20   13.10    0.63  -63.00 -179.00   27.90\n",
      "   29        359R 359R N   CA   2898 2899  144.90  140.60                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:  1486 1576   2.268\n",
      "  serious non-bonded atom clash:  1487 1596   1.920\n",
      "  serious non-bonded atom clash:  1488 1596   2.041\n",
      "  serious non-bonded atom clash:  1496 1594   2.078\n",
      "  serious non-bonded atom clash:  1497 1569   1.856\n",
      "  serious non-bonded atom clash:  1497 1594   2.175\n",
      "  serious non-bonded atom clash:  1498 1597   1.968\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     5    1    1    5    4   22   38   69   88  108  183  253  328  497  499\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "4fv5_MODLL.B99990001.pdb      2408.58618\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 4fv6\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    36261    33615\n",
      "286 (of 2915 total) atoms selected for optimization\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:      -93.2710\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     286\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    33615    2016\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      854\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :        2751.8699\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     296      13     20   0.085   0.085      598.45       1.000\n",
      " 2 Bond angle potential               :     410      19     41   7.469   7.469      402.36       1.000\n",
      " 3 Stereochemical cosine torsion poten:     198       0     11  51.947  51.947      87.079       1.000\n",
      " 4 Stereochemical improper torsion pot:     135       3      9   5.202   5.202      72.134       1.000\n",
      " 5 Soft-sphere overlap restraints     :     854      41     60   0.099   0.099      992.53       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     341      12     24   0.967   0.967      204.48       1.000\n",
      "10 Distance restraints 2 (N-O)        :     316       5     16   0.925   0.925      117.51       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      42       3      3  11.126  11.126      59.829       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      26       0      3  81.486  81.486      19.431       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      18       0      2  52.063  52.063      14.022       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       6       0      0 104.906 104.906      3.5194       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       2       0      0  72.321  72.321      1.3210       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     125       0      0   0.569   0.569      1.1306       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      44      29     17  63.600 142.371      158.03       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      57       0      1   2.019   2.019      20.044       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 4fv6_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:   10698.6484\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  1                           : Bond length potential                   \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   1505 185T 185T C   CA   1475 1471    1.89    1.49    0.40   11.60    1.49    0.40   11.60\n",
      "    2   1506 185T 186E C   N    1475 1477    1.57    1.35    0.23    8.16    1.35    0.23    8.16\n",
      "    3   1512 186E 186E N   CA   1477 1478    1.62    1.43    0.19    6.09    1.43    0.19    6.09\n",
      "    4   1527 187Y 187Y C   CA   1496 1487    1.89    1.49    0.40   11.58    1.49    0.40   11.58\n",
      "    5   1528 187Y 188V C   N    1496 1498    1.75    1.35    0.40   14.22    1.35    0.40   14.22\n",
      "    6   1529 188V 188V CB  CA   1500 1499    2.17    1.50    0.67   18.52    1.50    0.67   18.52\n",
      "    7   1532 188V 188V N   CA   1498 1499    1.90    1.43    0.47   15.30    1.43    0.47   15.30\n",
      "    8   1534 188V 188V C   CA   1503 1499    2.07    1.49    0.58   16.82    1.49    0.58   16.82\n",
      "    9   1535 188V 189A C   N    1503 1505    1.79    1.35    0.45   15.94    1.35    0.45   15.94\n",
      "   10   1537 189A 189A N   CA   1505 1506    1.72    1.43    0.29    9.68    1.43    0.29    9.68\n",
      "   11   1539 189A 189A C   CA   1508 1506    1.70    1.49    0.21    6.05    1.49    0.21    6.05\n",
      "   12   1540 189A 190T C   N    1508 1510    1.49    1.35    0.14    4.96    1.35    0.14    4.96\n",
      "   13   1544 190T 190T N   CA   1510 1511    1.60    1.43    0.17    5.49    1.43    0.17    5.49\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   5005 184L 184L N   CA   1462 1463  126.82  107.00   19.82    5.70  107.00   19.82    5.70\n",
      "    2   5006 184L 184L CA  CB   1463 1464  133.04  113.50   19.54    4.80  113.50   19.54    4.80\n",
      "    3   5016 185T 185T N   CA   1470 1471  129.05  107.00   22.05    6.34  107.00   22.05    6.34\n",
      "    4   5023 185T 186E C   N    1475 1477  151.06  120.00   31.06    7.06  120.00   31.06    7.06\n",
      "    5   5025 186E 186E N   CA   1477 1478  132.27  113.50   18.77    5.05  113.50   18.77    5.05\n",
      "    6   5026 186E 186E N   CA   1477 1478  135.55  107.00   28.55    8.21  107.00   28.55    8.21\n",
      "    7   5030 186E 186E CB  CG   1479 1480  129.91  108.00   21.91    5.08  108.00   21.91    5.08\n",
      "    8   5038 187Y 187Y N   CA   1486 1487  125.59  107.00   18.59    5.35  107.00   18.59    5.35\n",
      "    9   5039 187Y 187Y CA  CB   1487 1488  127.39  107.50   19.89    4.60  107.50   19.89    4.60\n",
      "   10   5041 187Y 187Y CA  C    1487 1496  132.16  116.50   15.67    4.50  116.50   15.67    4.50\n",
      "   11   5052 187Y 187Y C   CA   1496 1487  142.67  108.00   34.67    8.03  108.00   34.67    8.03\n",
      "   12   5053 187Y 188V C   N    1496 1498  146.36  120.00   26.36    5.99  120.00   26.36    5.99\n",
      "   13   5057 188V 188V CA  CB   1499 1500  140.70  108.50   32.20    7.55  108.50   32.20    7.55\n",
      "   14   5058 188V 188V CA  CB   1499 1500  133.30  108.50   24.80    5.82  108.50   24.80    5.82\n",
      "   15   5063 188V 189A C   N    1503 1505  179.30  120.00   59.30   13.48  120.00   59.30   13.48\n",
      "   16   5070 189A 190T C   N    1508 1510  147.76  120.00   27.76    6.31  120.00   27.76    6.31\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  9                           : Distance restraints 1 (CA-CA)           \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  12329  34G 189A CA  CA    237 1506   16.23   12.18    4.05    7.09   12.18    4.05    7.09\n",
      "    2  12355  35A 189A CA  CA    241 1506   14.95   11.47    3.49    6.33   11.47    3.49    6.33\n",
      "    3  13099  67R 189A CA  CA    496 1506   15.16   12.74    2.42    4.89   12.74    2.42    4.89\n",
      "    4  15603 149D 189A CA  CA   1194 1506    8.27    5.20    3.07    6.08    5.20    3.07    6.08\n",
      "    5  15639 150L 189A CA  CA   1202 1506   11.09    8.49    2.60    4.93    8.49    2.60    4.93\n",
      "    6  15680 151K 189A CA  CA   1210 1506   12.31    9.42    2.89    5.56    9.42    2.89    5.56\n",
      "    7  15716 152P 189A CA  CA   1219 1506   16.00   12.93    3.07    5.37   12.93    3.07    5.37\n",
      "    8  15753 153S 189A CA  CA   1226 1506   16.19   12.43    3.76    6.78   12.43    3.76    6.78\n",
      "    9  15781 154N 189A CA  CA   1232 1506   15.44   11.60    3.84    6.98   11.60    3.84    6.98\n",
      "   10  15878 166C 189A CA  CA   1325 1506   17.63   13.84    3.78    5.25   13.84    3.78    5.25\n",
      "   11  15884 167D 189A CA  CA   1331 1506   15.23   11.29    3.95    5.58   11.29    3.95    5.58\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 10                           : Distance restraints 2 (N-O)             \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  21710 189A 147H N   O    1505 1181   11.50    8.43    3.06    5.13    8.43    3.06    5.13\n",
      "    2  21712 189A 149D N   O    1505 1200   11.47    8.07    3.40    5.81    8.07    3.40    5.81\n",
      "    3  21714 189A 167D N   O    1505 1337   14.92   10.44    4.48    6.09   10.44    4.48    6.09\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 13                           : Mainchain Omega dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10774 187Y 187Y CA  C    1487 1496 -146.72 -180.00   33.28    6.65 -180.00   33.28    6.65\n",
      "    2  10777 190T 190T CA  C    1511 1515 -132.39 -180.00   47.61    9.52 -180.00   47.61    9.52\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9 -111.92 -134.00   22.10    0.69  -62.50 -179.91   32.14\n",
      "    1          2A   2A N   CA      9   10  145.92  147.00                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14 -108.52 -134.00   25.68    0.73  -62.50 -178.71   32.14\n",
      "    2          3A   3A N   CA     14   15  143.75  147.00                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19 -116.47 -134.00   17.59    0.64  -62.50  178.93   32.20\n",
      "    3          4A   4A N   CA     19   20  148.51  147.00                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24 -106.44 -134.00   27.69    0.82  -62.50 -179.71   31.87\n",
      "    4          5A   5A N   CA     24   25  144.25  147.00                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29 -110.02 -134.00   24.23    0.68  -62.50 -178.10   32.33\n",
      "    5          6A   6A N   CA     29   30  143.52  147.00                  -40.90\n",
      "    6  10235   6A   7A C   N      32   34 -121.37 -134.00   13.68    0.66  -62.50  176.93   32.14\n",
      "    6          7A   7A N   CA     34   35  152.26  147.00                  -40.90\n",
      "    7  10236   7A   8G C   N      37   39 -107.03  -80.20   41.12    2.83   82.20 -142.66   16.93\n",
      "    7          8G   8G N   CA     39   40  142.94  174.10                    8.50\n",
      "    8  10237   8G   9A C   N      41   43 -147.95 -134.00   22.10    0.75  -62.50  176.96   33.57\n",
      "    8          9A   9A N   CA     43   44  164.14  147.00                  -40.90\n",
      "    9  10403 174A 175D C   N    1387 1389 -132.36  -96.50   49.72    2.08  -63.30 -175.24   27.27\n",
      "    9        175D 175D N   CA   1389 1390  148.64  114.20                  -40.00\n",
      "   10  10405 176P 177D C   N    1402 1404  -89.09  -70.90   18.23    1.42  -63.30  172.90   22.97\n",
      "   10        177D 177D N   CA   1404 1405  149.04  150.30                  -40.00\n",
      "   11  10406 177D 178H C   N    1410 1412  -61.07  -67.60    7.99    0.71  -63.20  173.10   21.51\n",
      "   11        178H 178H N   CA   1412 1413  144.61  140.00                  -42.30\n",
      "   12  10407 178H 179D C   N    1420 1422 -120.14  -96.50   24.22    0.99  -63.30  159.42   17.12\n",
      "   12        179D 179D N   CA   1422 1423  108.94  114.20                  -40.00\n",
      "   13  10408 179D 180H C   N    1428 1430 -147.51  -63.20   84.36   13.52  -63.20   84.36   13.52\n",
      "   13        180H 180H N   CA   1430 1431  -45.07  -42.30                  -42.30\n",
      "   14  10409 180H 181T C   N    1438 1440 -114.52 -124.80   73.26    3.45  -63.20  114.07   17.96\n",
      "   14        181T 181T N   CA   1440 1441 -143.97  143.50                  -42.10\n",
      "   15  10410 181T 182G C   N    1445 1447 -128.13  -62.40   86.92   16.25   82.20 -176.26    7.03\n",
      "   15        182G 182G N   CA   1447 1448  -98.08  -41.20                    8.50\n",
      "   16  10411 182G 183F C   N    1449 1451   89.94 -124.20  154.89    4.58  -63.20 -165.27   23.36\n",
      "   16        183F 183F N   CA   1451 1452 -164.59  143.30                  -44.30\n",
      "   17  10412 183F 184L C   N    1460 1462 -168.96 -108.50   98.89    5.61  -63.50  142.23   18.62\n",
      "   17        184L 184L N   CA   1462 1463   54.24  132.50                  -41.20\n",
      "   18  10413 184L 185T C   N    1468 1470 -140.55 -124.80   74.28    2.74  -63.20  127.86   21.18\n",
      "   18        185T 185T N   CA   1470 1471 -143.91  143.50                  -42.10\n",
      "   19  10414 185T 186E C   N    1475 1477  -82.32  -69.30   85.66    6.07  -63.60   94.41   13.64\n",
      "   19        186E 186E N   CA   1477 1478 -132.84  142.50                  -40.30\n",
      "   20  10415 186E 187Y C   N    1484 1486  113.93 -124.30  163.31    7.54  -63.50 -168.37   29.32\n",
      "   20        187Y 187Y N   CA   1486 1487 -115.77  135.40                  -43.40\n",
      "   21  10418 189A 190T C   N    1508 1510   70.38  -78.10  151.58    8.36  -78.10  151.58    8.36\n",
      "   21        190T 190T N   CA   1510 1511  119.30  149.80                  149.80\n",
      "   22  10429 200L 201N C   N    1608 1610 -115.75 -119.90   77.03    3.59  -63.20  117.41   17.63\n",
      "   22        201N 201N N   CA   1610 1611 -146.09  137.00                  -41.10\n",
      "   23  10430 201N 202S C   N    1616 1618  166.03 -136.60   63.43    3.21  -64.10 -154.60   15.26\n",
      "   23        202S 202S N   CA   1618 1619  124.14  151.20                  -35.00\n",
      "   24  10431 202S 203K C   N    1622 1624  174.14  -62.90  133.33   23.25  -62.90  133.33   23.25\n",
      "   24        203K 203K N   CA   1624 1625  -92.33  -40.80                  -40.80\n",
      "   25  10432 203K 204G C   N    1631 1633 -132.36 -167.20   36.10    0.53   82.20 -146.28   16.27\n",
      "   25        204G 204G N   CA   1633 1634  165.11  174.60                    8.50\n",
      "   26  10433 204G 205Y C   N    1635 1637 -117.79 -124.30   31.69    1.96   55.90 -144.88   15.19\n",
      "   26        205Y 205Y N   CA   1637 1638  166.42  135.40                   39.50\n",
      "   27  10585 356P 357G C   N    2880 2882   94.19   78.70   33.56    0.50  -80.20  174.68   11.33\n",
      "   27        357G 357G N   CA   2882 2883  164.13 -166.10                  174.10\n",
      "   28  10586 357G 358Y C   N    2884 2886 -120.81 -124.30    4.04    0.34  -63.50 -171.90   32.75\n",
      "   28        358Y 358Y N   CA   2886 2887  137.45  135.40                  -43.40\n",
      "   29  10587 358Y 359R C   N    2896 2898 -112.79 -125.20   13.13    0.63  -63.00 -178.99   27.89\n",
      "   29        359R 359R N   CA   2898 2899  144.88  140.60                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:  1496 1576   2.278\n",
      "  serious non-bonded atom clash:  1500 1599   2.278\n",
      "  serious non-bonded atom clash:  1503 1558   2.130\n",
      "  serious non-bonded atom clash:  1505 1558   1.966\n",
      "  serious non-bonded atom clash:  1505 1590   2.269\n",
      "  serious non-bonded atom clash:  1595 1868   1.823\n",
      "  serious non-bonded atom clash:  1596 1865   1.912\n",
      "  serious non-bonded atom clash:  1596 1867   2.008\n",
      "  serious non-bonded atom clash:  1596 1868   1.942\n",
      "  serious non-bonded atom clash:  1597 1865   1.991\n",
      "  serious non-bonded atom clash:  1600 1868   1.406\n",
      "  serious non-bonded atom clash:  1601 1868   1.537\n",
      "  serious non-bonded atom clash:  2674 2677   2.277\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     8    1    4   11   13    9   30   73   83  108  170  242  327  432  513\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "4fv6_MODLL.B99990001.pdb      2751.86987\n",
      "\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2770 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 4fv7\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    37591    34945\n",
      "244 (of 2915 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2682 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     244\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    34945    2173\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      476\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         156.4529\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     250       0      0   0.004   0.004      1.0641       1.000\n",
      " 2 Bond angle potential               :     344       0      0   1.696   1.696      18.473       1.000\n",
      " 3 Stereochemical cosine torsion poten:     144       0      5  51.137  51.137      60.218       1.000\n",
      " 4 Stereochemical improper torsion pot:      99       0      0   1.045   1.045      2.2681       1.000\n",
      " 5 Soft-sphere overlap restraints     :     476       0      0   0.003   0.003     0.35912       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     406       0      0   0.129   0.129      4.4988       1.000\n",
      "10 Distance restraints 2 (N-O)        :     383       0      0   0.349   0.349      12.643       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      37       0      0   3.901   3.901      4.7184       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      24       0      0  69.751  69.751      3.6689       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      19       0      0  59.660  59.660      9.9348       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       7       0      0  96.636  96.636      4.6054       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       2       0      0 144.547 144.547      1.5652       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     252       0      0   0.440   0.440      1.2939       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      40      14      4  28.245 109.027      29.559       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     166       0      0   0.397   0.397      1.5820       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 4fv7_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    4781.9814\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9  -76.20  -68.20    8.08    0.65  -62.50  173.21   29.08\n",
      "    1          2A   2A N   CA      9   10  146.44  145.30                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14  -75.41  -68.20   10.38    0.65  -62.50  166.83   27.99\n",
      "    2          3A   3A N   CA     14   15  152.77  145.30                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19  -66.74  -68.20    4.75    0.43  -62.50  169.34   27.96\n",
      "    3          4A   4A N   CA     19   20  149.82  145.30                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24 -120.11 -134.00   13.90    0.49  -62.50 -179.16   32.71\n",
      "    4          5A   5A N   CA     24   25  147.69  147.00                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29  -80.52  -68.20   27.02    1.78  -62.50  150.83   25.64\n",
      "    5          6A   6A N   CA     29   30  169.35  145.30                  -40.90\n",
      "    6  10235   6A   7A C   N      32   34  -60.09  -68.20    8.81    0.62  -62.50  177.27   28.93\n",
      "    6          7A   7A N   CA     34   35  141.85  145.30                  -40.90\n",
      "    7  10236   7A   8G C   N      37   39   85.55   78.70   31.83    0.64   82.20  154.35    7.66\n",
      "    7          8G   8G N   CA     39   40  162.81 -166.10                    8.50\n",
      "    8  10237   8G   9A C   N      41   43 -155.84 -134.00   24.15    0.55  -62.50 -173.21   35.59\n",
      "    8          9A   9A N   CA     43   44  157.31  147.00                  -40.90\n",
      "    9  10238   9A  10G C   N      46   48   68.06   78.70   12.32    0.41   82.20  168.98    8.91\n",
      "    9         10G  10G N   CA     48   49 -159.88 -166.10                    8.50\n",
      "   10  10389 160T 161C C   N    1283 1285   74.60   57.40   23.26    1.43 -117.90 -153.51    8.90\n",
      "   10        161C 161C N   CA   1285 1286   20.35   36.00                  141.10\n",
      "   11  10560 331F 332D C   N    2668 2670 -108.54  -96.50   17.07    0.71  -63.30  172.34   18.99\n",
      "   11        332D 332D N   CA   2670 2671  126.29  114.20                  -40.00\n",
      "   12  10563 334E 335L C   N    2693 2695 -105.97 -108.50    8.85    0.50  -63.50 -177.18   27.98\n",
      "   12        335L 335L N   CA   2695 2696  140.98  132.50                  -41.20\n",
      "   13  10564 335L 336D C   N    2701 2703  165.88  -63.30  132.68   21.94   54.50  151.75    9.69\n",
      "   13        336D 336D N   CA   2703 2704  -62.16  -40.00                   40.90\n",
      "   14  10587 358Y 359R C   N    2896 2898 -122.14 -125.20   10.55    0.56  -63.00  178.30   28.17\n",
      "   14        359R 359R N   CA   2898 2899  150.70  140.60                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    1    7   11   53  102  146  186  226  332  432  559\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "4fv7_MODLL.B99990001.pdb       156.45290\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 4fv8\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    36427    33781\n",
      "311 (of 2915 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2673 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:       92.4989\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     311\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    33781    2485\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      959\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :        1818.7019\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     322       9     23   0.047   0.047      208.10       1.000\n",
      " 2 Bond angle potential               :     447      24     33   6.232   6.232      346.26       1.000\n",
      " 3 Stereochemical cosine torsion poten:     212       0      8  51.615  51.615      93.415       1.000\n",
      " 4 Stereochemical improper torsion pot:     146       2      5   3.705   3.705      44.975       1.000\n",
      " 5 Soft-sphere overlap restraints     :     959      32     54   0.084   0.084      801.62       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     452       0      7   0.392   0.392      51.968       1.000\n",
      "10 Distance restraints 2 (N-O)        :     410       0      8   0.662   0.662      80.708       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      46       4      6  38.245  38.245      37.431       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      30       0      4  88.102  88.102      22.768       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      20       0      2  83.018  83.018      14.190       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       6       0      0  91.060  91.060      4.5734       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       2       0      0  12.742  12.742     0.99324       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     203       0      0   0.646   0.646      1.5642       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      49      27     16  50.981 137.577      88.957       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     140       0      2   1.194   1.194      21.183       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 4fv8_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:   10948.2725\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  1                           : Bond length potential                   \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   1491 183F 184L C   N    1460 1462    1.56    1.35    0.21    7.55    1.35    0.21    7.55\n",
      "    2   1492 184L 184L CB  CA   1464 1463    1.73    1.54    0.19    5.16    1.54    0.19    5.16\n",
      "    3   1496 184L 184L N   CA   1462 1463    1.72    1.43    0.29    9.61    1.43    0.29    9.61\n",
      "    4   1498 184L 184L C   CA   1468 1463    1.74    1.49    0.25    7.16    1.49    0.25    7.16\n",
      "    5   1499 184L 185T C   N    1468 1470    1.51    1.35    0.16    5.70    1.35    0.16    5.70\n",
      "    6   1503 185T 185T N   CA   1470 1471    1.63    1.43    0.20    6.61    1.43    0.20    6.61\n",
      "    7   1509 186E 186E CD  CG   1481 1480    1.76    1.52    0.24    6.30    1.52    0.24    6.30\n",
      "    8   1641 200L 200L N   CA   1602 1603    1.62    1.43    0.19    6.21    1.43    0.19    6.21\n",
      "    9   1643 200L 200L C   CA   1608 1603    1.87    1.49    0.38   11.05    1.49    0.38   11.05\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   4992 183F 183F CA  C    1452 1460  134.99  116.50   18.49    5.32  116.50   18.49    5.32\n",
      "    2   5002 183F 184L C   N    1460 1462  153.24  120.00   33.24    7.55  120.00   33.24    7.55\n",
      "    3   5005 184L 184L N   CA   1462 1463  128.53  107.00   21.53    6.19  107.00   21.53    6.19\n",
      "    4   5006 184L 184L CA  CB   1463 1464  133.62  113.50   20.12    4.94  113.50   20.12    4.94\n",
      "    5   5008 184L 184L CA  C    1463 1468  140.03  116.50   23.53    6.77  116.50   23.53    6.77\n",
      "    6   5013 184L 185T C   N    1468 1470  150.68  120.00   30.68    6.97  120.00   30.68    6.97\n",
      "    7   5015 185T 185T N   CA   1470 1471  133.75  113.50   20.25    5.45  113.50   20.25    5.45\n",
      "    8   5016 185T 185T N   CA   1470 1471  128.44  107.00   21.44    6.16  107.00   21.44    6.16\n",
      "    9   5030 186E 186E CB  CG   1479 1480  132.77  108.00   24.77    5.74  108.00   24.77    5.74\n",
      "   10   5031 186E 186E CG  CD   1480 1481  141.83  118.00   23.83    4.84  118.00   23.83    4.84\n",
      "   11   5038 187Y 187Y N   CA   1486 1487  123.94  107.00   16.94    4.87  107.00   16.94    4.87\n",
      "   12   5053 187Y 188V C   N    1496 1498  145.94  120.00   25.94    5.90  120.00   25.94    5.90\n",
      "   13   5056 188V 188V N   CA   1498 1499  127.07  107.00   20.07    5.77  107.00   20.07    5.77\n",
      "   14   5197 199M 199M CA  C    1595 1600  136.93  116.50   20.43    5.87  116.50   20.43    5.87\n",
      "   15   5201 199M 200L C   N    1600 1602  141.44  120.00   21.44    4.87  120.00   21.44    4.87\n",
      "   16   5204 200L 200L N   CA   1602 1603  132.25  107.00   25.25    7.26  107.00   25.25    7.26\n",
      "   17   5207 200L 200L CA  C    1603 1608  132.19  116.50   15.69    4.51  116.50   15.69    4.51\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  4                           : Stereochemical improper torsion potentia\n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   9671 199M 199M C   CA   1600 1595   18.59    0.00   18.59    6.54    0.00   18.59    6.54\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 13                           : Mainchain Omega dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10761 174A 174A CA  C    1385 1387   -0.35 -179.18  178.83   41.57 -179.18  178.83   41.57\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9  -72.89  -68.20   13.11    1.21  -62.50  174.27   28.07\n",
      "    1          2A   2A N   CA      9   10  133.07  145.30                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14 -129.85 -134.00    5.96    0.18  -62.50 -171.20   34.54\n",
      "    2          3A   3A N   CA     14   15  142.72  147.00                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19 -107.04 -134.00   30.10    0.69  -62.50 -179.91   27.69\n",
      "    3          4A   4A N   CA     19   20  133.60  147.00                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24 -121.00 -134.00   13.38    0.34  -62.50 -175.24   33.40\n",
      "    4          5A   5A N   CA     24   25  143.85  147.00                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29 -115.24 -134.00   23.64    0.61  -62.50 -178.66   27.65\n",
      "    5          6A   6A N   CA     29   30  132.60  147.00                  -40.90\n",
      "    6  10235   6A   7A C   N      32   34  -80.72  -68.20   12.88    0.98  -62.50  171.76   29.08\n",
      "    6          7A   7A N   CA     34   35  148.31  145.30                  -40.90\n",
      "    7  10403 174A 175D C   N    1387 1389 -148.64  -96.50   58.30    2.41  -63.30 -161.04   30.07\n",
      "    7        175D 175D N   CA   1389 1390  140.27  114.20                  -40.00\n",
      "    8  10405 176P 177D C   N    1402 1404 -138.69  -96.50   60.27    2.52  -63.30  179.37   27.01\n",
      "    8        177D 177D N   CA   1404 1405  157.25  114.20                  -40.00\n",
      "    9  10406 177D 178H C   N    1410 1412 -148.96 -125.60   28.19    1.25  -63.20 -173.77   19.81\n",
      "    9        178H 178H N   CA   1412 1413  123.01  138.80                  -42.30\n",
      "   10  10407 178H 179D C   N    1420 1422 -145.99  -96.50   66.85    2.79  -63.30 -179.12   27.65\n",
      "   10        179D 179D N   CA   1422 1423  159.14  114.20                  -40.00\n",
      "   11  10408 179D 180H C   N    1428 1430 -121.48 -125.60    7.25    0.33  -63.20 -177.50   26.63\n",
      "   11        180H 180H N   CA   1430 1431  144.76  138.80                  -42.30\n",
      "   12  10409 180H 181T C   N    1438 1440 -170.70 -124.80   70.03    2.20  -63.20  162.23   27.27\n",
      "   12        181T 181T N   CA   1440 1441 -163.60  143.50                  -42.10\n",
      "   13  10410 181T 182G C   N    1445 1447 -140.67 -167.20  102.09    3.58   82.20  152.84   11.87\n",
      "   13        182G 182G N   CA   1447 1448   76.02  174.60                    8.50\n",
      "   14  10411 182G 183F C   N    1449 1451 -157.71 -124.20   40.22    1.05  -63.20  177.43   30.53\n",
      "   14        183F 183F N   CA   1451 1452  165.54  143.30                  -44.30\n",
      "   15  10412 183F 184L C   N    1460 1462  -85.13 -108.50   80.19    4.05  -63.50   99.37   12.73\n",
      "   15        184L 184L N   CA   1462 1463   55.79  132.50                  -41.20\n",
      "   16  10413 184L 185T C   N    1468 1470 -135.08 -124.80   11.88    0.45  -63.20 -176.86   28.21\n",
      "   16        185T 185T N   CA   1470 1471  149.46  143.50                  -42.10\n",
      "   17  10414 185T 186E C   N    1475 1477 -176.52 -117.80   81.12    2.57  -63.60  169.89   28.55\n",
      "   17        186E 186E N   CA   1477 1478 -167.23  136.80                  -40.30\n",
      "   18  10415 186E 187Y C   N    1484 1486 -159.86 -124.30   46.20    2.20  -63.50  179.73   33.06\n",
      "   18        187Y 187Y N   CA   1486 1487  164.88  135.40                  -43.40\n",
      "   19  10417 188V 189A C   N    1503 1505  154.31 -134.00   80.67    1.84  -68.20  142.83   10.72\n",
      "   19        189A 189A N   CA   1505 1506 -176.01  147.00                  145.30\n",
      "   20  10429 200L 201N C   N    1608 1610  -96.39 -119.90   24.54    0.84  -63.20  174.26   20.13\n",
      "   20        201N 201N N   CA   1610 1611  129.97  137.00                  -41.10\n",
      "   21  10430 201N 202S C   N    1616 1618  -53.16  -72.40   19.37    1.44  -64.10  170.68   11.76\n",
      "   21        202S 202S N   CA   1618 1619  154.67  152.40                  -35.00\n",
      "   22  10431 202S 203K C   N    1622 1624   75.77  -70.20  155.02   12.41  -62.90 -172.23   23.20\n",
      "   22        203K 203K N   CA   1624 1625 -167.41  140.40                  -40.80\n",
      "   23  10432 203K 204G C   N    1631 1633  -79.08  -80.20   31.50    1.30   82.20 -130.75   17.56\n",
      "   23        204G 204G N   CA   1633 1634 -154.42  174.10                    8.50\n",
      "   24  10433 204G 205Y C   N    1635 1637 -134.39 -124.30   87.83    4.23   55.90 -114.89   14.31\n",
      "   24        205Y 205Y N   CA   1637 1638 -137.35  135.40                   39.50\n",
      "   25  10585 356P 357G C   N    2880 2882  -59.01  -62.40    3.84    0.59  -80.20  144.46    6.76\n",
      "   25        357G 357G N   CA   2882 2883  -43.00  -41.20                  174.10\n",
      "   26  10586 357G 358Y C   N    2884 2886  -64.09  -98.40   34.46    1.41  -63.50  168.60   26.72\n",
      "   26        358Y 358Y N   CA   2886 2887  125.20  128.40                  -43.40\n",
      "   27  10587 358Y 359R C   N    2896 2898 -101.28 -125.20   39.29    2.18  -63.00  152.03   23.18\n",
      "   27        359R 359R N   CA   2898 2899  171.77  140.60                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:  1462 2052   1.817\n",
      "  serious non-bonded atom clash:  1469 2072   2.194\n",
      "  serious non-bonded atom clash:  1470 1609   2.209\n",
      "  serious non-bonded atom clash:  1470 2045   2.149\n",
      "  serious non-bonded atom clash:  1481 1585   2.003\n",
      "  serious non-bonded atom clash:  1481 1602   2.210\n",
      "  serious non-bonded atom clash:  1483 1576   2.027\n",
      "  serious non-bonded atom clash:  1483 1584   2.190\n",
      "  serious non-bonded atom clash:  1483 1594   2.211\n",
      "  serious non-bonded atom clash:  1483 1602   2.208\n",
      "  serious non-bonded atom clash:  2690 2716   2.067\n",
      "  serious non-bonded atom clash:  2692 2716   1.877\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     5    3    4    5   13   20   35   85  107  146  177  258  341  452  530\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "4fv8_MODLL.B99990001.pdb      1818.70190\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 4fv9\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    36427    33781\n",
      "326 (of 2915 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2743 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     326\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    33781    2649\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):     1294\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :        1807.0321\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     337      10     12   0.043   0.043      147.79       1.000\n",
      " 2 Bond angle potential               :     467       9     40   5.435   5.435      245.48       1.000\n",
      " 3 Stereochemical cosine torsion poten:     220       0     14  51.832  51.832      90.789       1.000\n",
      " 4 Stereochemical improper torsion pot:     151       1      7   4.305   4.305      54.786       1.000\n",
      " 5 Soft-sphere overlap restraints     :    1294      46     79   0.079   0.079      942.43       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     475       0      2   0.393   0.393      48.407       1.000\n",
      "10 Distance restraints 2 (N-O)        :     430       0      3   0.491   0.491      41.386       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      48       1      6   8.421   8.421      38.706       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      32       0      3  66.025  66.025      21.737       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      21       0      5  95.039  95.039      27.674       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       7       0      1  92.181  92.181      5.8379       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       2       0      0 126.945 126.945     0.81696       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     228       0      0   0.533   0.533      1.4853       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      51      26     12  45.637 118.855      100.94       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     180       0      5   1.395   1.395      38.771       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 4fv9_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:   10053.9023\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  1                           : Bond length potential                   \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   1480 183F 183F CB  CA   1453 1452    1.77    1.54    0.24    6.49    1.54    0.24    6.49\n",
      "    2   1505 185T 185T C   CA   1475 1471    1.76    1.49    0.27    7.76    1.49    0.27    7.76\n",
      "    3   1528 187Y 188V C   N    1496 1498    1.47    1.35    0.13    4.60    1.35    0.13    4.60\n",
      "    4   1629 199M 199M CB  CA   1596 1595    1.72    1.54    0.18    4.97    1.54    0.18    4.97\n",
      "    5   1630 199M 199M CG  CB   1597 1596    1.72    1.53    0.19    5.33    1.53    0.19    5.33\n",
      "    6   1631 199M 199M SD  CG   1598 1597    2.11    1.82    0.30    7.65    1.82    0.30    7.65\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   4988 183F 183F N   CA   1451 1452  134.49  113.50   20.99    5.64  113.50   20.99    5.64\n",
      "    2   5003 184L 183F N   C    1462 1460  138.47  122.50   15.97    4.59  122.50   15.97    4.59\n",
      "    3   5022 185T 185T C   CA   1475 1471  135.85  108.00   27.85    6.46  108.00   27.85    6.46\n",
      "    4   5039 187Y 187Y CA  CB   1487 1488  135.75  107.50   28.25    6.53  107.50   28.25    6.53\n",
      "    5   5053 187Y 188V C   N    1496 1498  146.87  120.00   26.87    6.11  120.00   26.87    6.11\n",
      "    6   5195 199M 199M CA  CB   1595 1596  137.04  113.50   23.54    5.78  113.50   23.54    5.78\n",
      "    7   5199 199M 199M CG  SD   1597 1598  123.38   95.00   28.38    5.32   95.00   28.38    5.32\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9  -65.47  -68.20    3.65    0.23  -62.50  176.26   29.03\n",
      "    1          2A   2A N   CA      9   10  142.87  145.30                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14  -67.15  -68.20    1.10    0.10  -62.50  173.52   28.67\n",
      "    2          3A   3A N   CA     14   15  145.65  145.30                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19  -70.39  -68.20    2.32    0.17  -62.50  173.23   28.78\n",
      "    3          4A   4A N   CA     19   20  146.05  145.30                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24  -83.21  -68.20   21.26    1.34  -62.50  160.09   27.30\n",
      "    4          5A   5A N   CA     24   25  160.35  145.30                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29  -65.94  -68.20    4.06    0.26  -62.50  177.21   29.21\n",
      "    5          6A   6A N   CA     29   30  141.93  145.30                  -40.90\n",
      "    6  10235   6A   7A C   N      32   34  -78.18  -68.20   16.99    1.08  -62.50  160.81   27.15\n",
      "    6          7A   7A N   CA     34   35  159.06  145.30                  -40.90\n",
      "    7  10236   7A   8G C   N      37   39  164.40 -167.20   30.64    0.45  -80.20  116.02    7.34\n",
      "    7          8G   8G N   CA     39   40 -173.91  174.60                  174.10\n",
      "    8  10389 160T 161C C   N    1283 1285   77.32   57.40   26.48    1.68 -117.90 -154.64    8.88\n",
      "    8        161C 161C N   CA   1285 1286   18.55   36.00                  141.10\n",
      "    9  10403 174A 175D C   N    1387 1389 -120.72  -96.50   46.78    1.97  -63.30  175.45   25.37\n",
      "    9        175D 175D N   CA   1389 1390  154.21  114.20                  -40.00\n",
      "   10  10405 176P 177D C   N    1402 1404  -83.35  -70.90   25.23    0.85  -63.30  149.11   19.67\n",
      "   10        177D 177D N   CA   1404 1405  172.24  150.30                  -40.00\n",
      "   11  10406 177D 178H C   N    1410 1412 -147.31 -125.60   83.02    2.71  -63.20  129.73   21.43\n",
      "   11        178H 178H N   CA   1412 1413 -141.07  138.80                  -42.30\n",
      "   12  10407 178H 179D C   N    1420 1422  -74.14  -70.90   12.27    0.46  -63.30  158.24   20.20\n",
      "   12        179D 179D N   CA   1422 1423  162.13  150.30                  -40.00\n",
      "   13  10408 179D 180H C   N    1428 1430  -59.39  -67.60   12.41    0.74  -63.20  173.04   21.87\n",
      "   13        180H 180H N   CA   1430 1431  130.69  140.00                  -42.30\n",
      "   14  10409 180H 181T C   N    1438 1440 -110.61 -124.80   14.47    0.83  -63.20  177.99   25.98\n",
      "   14        181T 181T N   CA   1440 1441  146.34  143.50                  -42.10\n",
      "   15  10410 181T 182G C   N    1445 1447  121.06   78.70   97.19    1.44   82.20  105.35    6.93\n",
      "   15        182G 182G N   CA   1447 1448  106.42 -166.10                    8.50\n",
      "   16  10411 182G 183F C   N    1449 1451 -107.70 -124.20   42.14    1.51  -63.20  155.34   18.84\n",
      "   16        183F 183F N   CA   1451 1452  104.53  143.30                  -44.30\n",
      "   17  10412 183F 184L C   N    1460 1462   10.92  -70.70   87.29    6.77  -63.50  169.10   28.08\n",
      "   17        184L 184L N   CA   1462 1463  110.65  141.60                  -41.20\n",
      "   18  10413 184L 185T C   N    1468 1470 -134.17 -124.80   73.01    2.86  -63.20  124.26   20.38\n",
      "   18        185T 185T N   CA   1470 1471 -144.09  143.50                  -42.10\n",
      "   19  10414 185T 186E C   N    1475 1477 -176.65  -63.60  127.62   16.94  -63.60  127.62   16.94\n",
      "   19        186E 186E N   CA   1477 1478   18.92  -40.30                  -40.30\n",
      "   20  10415 186E 187Y C   N    1484 1486  -93.37  -98.40   59.15    6.25  -63.50  132.67   22.58\n",
      "   20        187Y 187Y N   CA   1486 1487 -172.67  128.40                  -43.40\n",
      "   21  10429 200L 201N C   N    1608 1610   61.98   55.90    6.35    0.77  -63.20  149.87   24.63\n",
      "   21        201N 201N N   CA   1610 1611   41.31   39.50                  -41.10\n",
      "   22  10430 201N 202S C   N    1616 1618  -84.05  -72.40   13.33    0.73  -64.10  167.32   13.41\n",
      "   22        202S 202S N   CA   1618 1619  158.87  152.40                  -35.00\n",
      "   23  10432 203K 204G C   N    1631 1633  -62.57  -80.20   59.52    1.55   82.20 -178.94    6.71\n",
      "   23        204G 204G N   CA   1633 1634  117.25  174.10                    8.50\n",
      "   24  10584 355Q 356P C   N    2873 2875  -65.16  -58.70   13.45    0.79  -64.50  165.90   12.41\n",
      "   24        356P 356P N   CA   2875 2876  -18.70  -30.50                  147.20\n",
      "   25  10586 357G 358Y C   N    2884 2886 -114.29 -124.30   10.54    0.67  -63.50 -177.32   26.59\n",
      "   25        358Y 358Y N   CA   2886 2887  132.07  135.40                  -43.40\n",
      "   26  10587 358Y 359R C   N    2896 2898  -72.15  -72.10    6.61    0.51  -63.00  170.64   23.67\n",
      "   26        359R 359R N   CA   2898 2899  148.51  141.90                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    9   10   33   49   84  117  173  207  220  376  497  535\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "4fv9_MODLL.B99990001.pdb      1807.03210\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 4g6n\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    37289    34643\n",
      "224 (of 2915 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2720 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:      -90.0082\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     224\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    34643    1795\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      417\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         138.6307\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     231       0      0   0.003   0.003     0.64207       1.000\n",
      " 2 Bond angle potential               :     317       0      0   1.671   1.671      16.559       1.000\n",
      " 3 Stereochemical cosine torsion poten:     142       0      5  51.030  51.030      53.850       1.000\n",
      " 4 Stereochemical improper torsion pot:      93       0      0   0.926   0.926      1.8281       1.000\n",
      " 5 Soft-sphere overlap restraints     :     417       0      0   0.002   0.002     0.12930       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     290       0      0   0.162   0.162      4.6626       1.000\n",
      "10 Distance restraints 2 (N-O)        :     273       0      0   0.275   0.275      7.6271       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      33       0      0   3.815   3.815      5.0013       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      21       0      0  80.804  80.804      4.5467       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      17       0      0  53.756  53.756      5.9294       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       8       0      0 101.095 101.095      5.2381       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       2       0      0  68.020  68.020      1.8567       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     194       0      0   0.484   0.484      1.2057       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      35      16      4  22.096 120.516      25.297       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     139       0      0   0.575   0.575      4.2574       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 4g6n_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    5034.6587\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9  -65.14  -68.20    8.78    0.81  -62.50  165.59   27.27\n",
      "    1          2A   2A N   CA      9   10  153.53  145.30                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14  -60.92  -68.20    7.46    0.68  -62.50  172.17   28.14\n",
      "    2          3A   3A N   CA     14   15  146.94  145.30                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19  -72.88  -68.20    7.13    0.45  -62.50  168.74   28.17\n",
      "    3          4A   4A N   CA     19   20  150.68  145.30                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24  -83.46  -68.20   15.67    1.44  -62.50  178.60   30.34\n",
      "    4          5A   5A N   CA     24   25  141.74  145.30                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29  -85.32  -68.20   18.69    1.32  -62.50  167.88   28.69\n",
      "    5          6A   6A N   CA     29   30  152.79  145.30                  -40.90\n",
      "    6  10235   6A   7A C   N      32   34  -75.48  -68.20   11.82    0.75  -62.50  165.00   27.70\n",
      "    6          7A   7A N   CA     34   35  154.62  145.30                  -40.90\n",
      "    7  10237   8G   9A C   N      41   43 -119.95 -134.00   17.95    1.01  -62.50  170.87   31.07\n",
      "    7          9A   9A N   CA     43   44  158.18  147.00                  -40.90\n",
      "    8  10238   9A  10G C   N      46   48   81.05   78.70   17.97    0.61   82.20  156.79    7.67\n",
      "    8         10G  10G N   CA     48   49 -148.28 -166.10                    8.50\n",
      "    9  10389 160T 161C C   N    1283 1285   73.82   57.40   22.91    1.34 -117.90 -152.68    8.94\n",
      "    9        161C 161C N   CA   1285 1286   20.01   36.00                  141.10\n",
      "   10  10558 329F 330K C   N    2648 2650 -138.52 -118.00   32.50    1.20  -62.90  172.37   26.91\n",
      "   10        330K 330K N   CA   2650 2651  164.30  139.10                  -40.80\n",
      "   11  10559 330K 331F C   N    2657 2659 -121.42 -124.20    9.87    0.52  -63.20  173.02   27.87\n",
      "   11        331F 331F N   CA   2659 2660  152.77  143.30                  -44.30\n",
      "   12  10560 331F 332D C   N    2668 2670  -80.03  -70.90   35.29    1.33  -63.30  136.64   17.92\n",
      "   12        332D 332D N   CA   2670 2671 -175.61  150.30                  -40.00\n",
      "   13  10561 332D 333M C   N    2676 2678  -71.71  -73.00   50.49    3.39  -63.40  133.29   19.27\n",
      "   13        333M 333M N   CA   2678 2679   92.52  143.00                  -40.50\n",
      "   14  10562 333M 334E C   N    2684 2686 -135.15 -117.80   21.77    1.12  -63.60  178.89   20.90\n",
      "   14        334E 334E N   CA   2686 2687  123.66  136.80                  -40.30\n",
      "   15  10563 334E 335L C   N    2693 2695 -117.39 -108.50   36.64    1.88  -63.50  160.09   25.56\n",
      "   15        335L 335L N   CA   2695 2696  168.05  132.50                  -41.20\n",
      "   16  10587 358Y 359R C   N    2896 2898  -63.87  -72.10   12.33    0.80  -63.00  173.82   23.42\n",
      "   16        359R 359R N   CA   2898 2899  132.72  141.90                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:   247  474   1.939\n",
      "  serious non-bonded atom clash:   247  476   1.753\n",
      "  serious non-bonded atom clash:   247  477   1.344\n",
      "  serious non-bonded atom clash:   248  476   1.691\n",
      "  serious non-bonded atom clash:   248  477   1.484\n",
      "  serious non-bonded atom clash:   249  407   2.134\n",
      "  serious non-bonded atom clash:   249  409   1.158\n",
      "  serious non-bonded atom clash:   249  410   2.199\n",
      "  serious non-bonded atom clash:   249  477   2.034\n",
      "  serious non-bonded atom clash:   250  413   2.228\n",
      "  serious non-bonded atom clash:   250  414   1.925\n",
      "  serious non-bonded atom clash:   250  475   2.160\n",
      "  serious non-bonded atom clash:   250  476   1.999\n",
      "  serious non-bonded atom clash:   251  407   2.093\n",
      "  serious non-bonded atom clash:   251  409   0.845\n",
      "  serious non-bonded atom clash:   251  410   0.798\n",
      "  serious non-bonded atom clash:   251  411   1.974\n",
      "  serious non-bonded atom clash:   251  413   1.666\n",
      "  serious non-bonded atom clash:   252  413   1.148\n",
      "  serious non-bonded atom clash:   252  414   0.690\n",
      "  serious non-bonded atom clash:   252  415   1.890\n",
      "  serious non-bonded atom clash:   252  416   2.278\n",
      "  serious non-bonded atom clash:   253  409   2.236\n",
      "  serious non-bonded atom clash:   253  410   1.184\n",
      "  serious non-bonded atom clash:   253  411   2.056\n",
      "  serious non-bonded atom clash:   253  413   0.490\n",
      "  serious non-bonded atom clash:   253  414   1.286\n",
      "  serious non-bonded atom clash:   253  415   1.809\n",
      "  serious non-bonded atom clash:   254  410   1.766\n",
      "  serious non-bonded atom clash:   254  411   2.059\n",
      "  serious non-bonded atom clash:   254  413   1.589\n",
      "  serious non-bonded atom clash:   254  414   2.069\n",
      "  serious non-bonded atom clash:   416  473   2.084\n",
      "  serious non-bonded atom clash:   416  475   2.285\n",
      "  serious non-bonded atom clash:   418  473   1.558\n",
      "  serious non-bonded atom clash:   418  475   1.726\n",
      "  serious non-bonded atom clash:   431 2739   1.464\n",
      "  serious non-bonded atom clash:   431 2740   1.877\n",
      "  serious non-bonded atom clash:   432 2738   2.106\n",
      "  serious non-bonded atom clash:   432 2739   0.818\n",
      "  serious non-bonded atom clash:   432 2740   1.800\n",
      "  serious non-bonded atom clash:   433 2739   2.138\n",
      "  serious non-bonded atom clash:   433 2740   1.580\n",
      "  serious non-bonded atom clash:   434 2740   1.138\n",
      "  serious non-bonded atom clash:   440 2740   1.892\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:    36    5    4    2    6   10   20   57   96  167  187  234  354  461  493\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "4g6n_MODLL.B99990001.pdb       138.63072\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 4g6o\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    37353    34707\n",
      "291 (of 2915 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2783 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:      -90.9932\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     291\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    34707    2746\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      562\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         256.4841\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     302       0      0   0.004   0.004      1.5420       1.000\n",
      " 2 Bond angle potential               :     417       0      0   1.962   1.962      28.756       1.000\n",
      " 3 Stereochemical cosine torsion poten:     188       0      6  49.764  49.764      69.625       1.000\n",
      " 4 Stereochemical improper torsion pot:     124       0      0   1.119   1.119      3.7600       1.000\n",
      " 5 Soft-sphere overlap restraints     :     562       0      0   0.005   0.005      1.5425       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     560       0      0   0.169   0.169      10.643       1.000\n",
      "10 Distance restraints 2 (N-O)        :     521       0      6   0.345   0.345      37.382       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      45       0      0   3.780   3.780      6.0227       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      27       0      0  59.475  59.475      1.2939       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      22       0      0  57.887  57.887      4.3552       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:      10       0      0  66.940  66.940      4.8972       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       2       0      0 141.152 141.152      1.7016       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     277       0      0   0.405   0.405      1.1145       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      49      17      8  31.455 112.966      79.170       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     202       0      0   0.515   0.515      4.6784       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 4g6o_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    5947.8481\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9  -71.30  -68.20    7.80    0.52  -62.50  166.88   27.79\n",
      "    1          2A   2A N   CA      9   10  152.46  145.30                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14  -77.99  -68.20   21.05    1.38  -62.50  155.94   26.34\n",
      "    2          3A   3A N   CA     14   15  163.93  145.30                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19  -60.99  -68.20    7.61    0.56  -62.50  176.23   28.81\n",
      "    3          4A   4A N   CA     19   20  142.88  145.30                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24 -127.27 -134.00   11.62    0.71  -62.50  175.05   32.15\n",
      "    4          5A   5A N   CA     24   25  156.48  147.00                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29  -92.86  -68.20   25.08    2.27  -62.50 -179.07   31.23\n",
      "    5          6A   6A N   CA     29   30  140.74  145.30                  -40.90\n",
      "    6  10235   6A   7A C   N      32   34 -101.13  -68.20   35.25    3.35  -62.50  177.86   27.52\n",
      "    6          7A   7A N   CA     34   35  132.71  145.30                  -40.90\n",
      "    7  10237   8G   9A C   N      41   43  -66.45  -68.20   17.77    1.49  -62.50  156.17   25.79\n",
      "    7          9A   9A N   CA     43   44  162.98  145.30                  -40.90\n",
      "    8  10259  30Y  31I C   N     213  215 -127.69  -63.40   65.82   12.15  -63.40   65.82   12.15\n",
      "    8         31I  31I N   CA    215  216  -57.70  -43.60                  -43.60\n",
      "    9  10262  33E  34G C   N     234  236 -131.44 -167.20   52.11    1.20   82.20 -165.44   15.07\n",
      "    9         34G  34G N   CA    236  237  136.70  174.60                    8.50\n",
      "   10  10263  34G  35A C   N     238  240   60.14  -68.20  134.88   12.65  -62.50 -179.61   27.68\n",
      "   10         35A  35A N   CA    240  241 -173.19  145.30                  -40.90\n",
      "   11  10389 160T 161C C   N    1283 1285   70.85   57.40   14.01    1.44 -117.90 -156.99    8.64\n",
      "   11        161C 161C N   CA   1285 1286   32.08   36.00                  141.10\n",
      "   12  10558 329F 330K C   N    2648 2650  -66.07  -70.20   14.44    1.14  -62.90  165.00   21.60\n",
      "   12        330K 330K N   CA   2650 2651  154.23  140.40                  -40.80\n",
      "   13  10559 330K 331F C   N    2657 2659  -63.78  -71.40   15.41    0.99  -63.20  171.61   23.79\n",
      "   13        331F 331F N   CA   2659 2660  127.31  140.70                  -44.30\n",
      "   14  10560 331F 332D C   N    2668 2670  -96.85  -70.90   37.48    1.48  -63.30  146.55   20.24\n",
      "   14        332D 332D N   CA   2670 2671  177.35  150.30                  -40.00\n",
      "   15  10561 332D 333M C   N    2676 2678  -84.07  -73.00   27.21    1.70  -63.40  153.05   24.49\n",
      "   15        333M 333M N   CA   2678 2679  167.85  143.00                  -40.50\n",
      "   16  10586 357G 358Y C   N    2884 2886 -116.24 -124.30   12.24    0.52  -63.50  177.61   25.72\n",
      "   16        358Y 358Y N   CA   2886 2887  126.19  135.40                  -43.40\n",
      "   17  10587 358Y 359R C   N    2896 2898  -74.39  -72.10    9.23    0.76  -63.00  174.42   22.82\n",
      "   17        359R 359R N   CA   2898 2899  132.95  141.90                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:   403  477   1.973\n",
      "  serious non-bonded atom clash:   405  477   2.212\n",
      "  serious non-bonded atom clash:   416  472   2.065\n",
      "  serious non-bonded atom clash:   418  472   1.507\n",
      "  serious non-bonded atom clash:   418  474   1.843\n",
      "  serious non-bonded atom clash:   423 2739   2.156\n",
      "  serious non-bonded atom clash:   431 2739   0.873\n",
      "  serious non-bonded atom clash:   431 2740   1.566\n",
      "  serious non-bonded atom clash:   432 2738   1.593\n",
      "  serious non-bonded atom clash:   432 2739   0.954\n",
      "  serious non-bonded atom clash:   432 2740   2.003\n",
      "  serious non-bonded atom clash:   433 2739   1.623\n",
      "  serious non-bonded atom clash:   433 2740   0.831\n",
      "  serious non-bonded atom clash:   434 2740   0.922\n",
      "  serious non-bonded atom clash:   440 2740   1.848\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:    13    1    1    4    2   10   22   60   89  141  186  247  365  441  512\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "4g6o_MODLL.B99990001.pdb       256.48413\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 4gsb\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    37874    35228\n",
      "228 (of 2915 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2769 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     228\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    35228    2327\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      496\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         151.9498\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     235       0      0   0.004   0.004      1.1446       1.000\n",
      " 2 Bond angle potential               :     323       0      1   1.825   1.825      21.585       1.000\n",
      " 3 Stereochemical cosine torsion poten:     137       0      1  47.331  47.331      50.622       1.000\n",
      " 4 Stereochemical improper torsion pot:      99       0      0   1.095   1.095      2.8445       1.000\n",
      " 5 Soft-sphere overlap restraints     :     496       0      0   0.002   0.002     0.23944       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     454       0      0   0.103   0.103      3.2269       1.000\n",
      "10 Distance restraints 2 (N-O)        :     467       0      0   0.237   0.237      7.0330       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      36       1      0  30.094  30.094      4.9329       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      19       0      0  77.572  77.572      6.5568       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      13       0      0  69.138  69.138      4.2559       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       8       0      0  80.519  80.519      4.4092       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       2       0      0 149.291 149.291      1.3750       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     290       0      0   0.591   0.591      1.7762       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      39      16      4  22.240 114.734      31.049       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     205       0      0   1.111   1.111      10.900       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 4gsb_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    5093.0967\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9  -64.18  -68.20   17.13    1.24  -62.50  169.55   27.70\n",
      "    1          2A   2A N   CA      9   10  128.65  145.30                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14  -86.86  -68.20   25.79    1.63  -62.50  157.89   27.14\n",
      "    2          3A   3A N   CA     14   15  163.10  145.30                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19 -114.66 -134.00   19.41    0.71  -62.50  178.25   31.99\n",
      "    3          4A   4A N   CA     19   20  148.66  147.00                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24 -175.24 -134.00   41.98    1.12  -62.50 -160.76   38.61\n",
      "    4          5A   5A N   CA     24   25  154.83  147.00                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29 -132.96 -134.00    2.80    0.17  -62.50 -176.43   33.86\n",
      "    5          6A   6A N   CA     29   30  149.60  147.00                  -40.90\n",
      "    6  10235   6A   7A C   N      32   34 -115.52 -134.00   18.79    0.50  -62.50 -176.63   32.87\n",
      "    6          7A   7A N   CA     34   35  143.57  147.00                  -40.90\n",
      "    7  10236   7A   8G C   N      37   39  100.16   78.70   36.08    0.63   82.20  157.42    8.54\n",
      "    7          8G   8G N   CA     39   40  164.90 -166.10                    8.50\n",
      "    8  10237   8G   9A C   N      41   43 -129.58 -134.00   14.50    0.89  -62.50  171.92   31.77\n",
      "    8          9A   9A N   CA     43   44  160.81  147.00                  -40.90\n",
      "    9  10238   9A  10G C   N      46   48 -132.46 -167.20   39.44    1.25   82.20 -132.33    6.53\n",
      "    9         10G  10G N   CA     48   49 -166.74  174.60                    8.50\n",
      "   10  10259  30Y  31I C   N     213  215 -105.56  -63.40   42.19    7.59  -63.40   42.19    7.59\n",
      "   10         31I  31I N   CA    215  216  -45.17  -43.60                  -43.60\n",
      "   11  10262  33E  34G C   N     234  236  -89.99  -80.20   26.76    0.65   82.20 -118.38   18.56\n",
      "   11         34G  34G N   CA    236  237 -161.00  174.10                    8.50\n",
      "   12  10264  35A  36Y C   N     243  245 -141.39 -124.30   21.83    1.06  -63.50 -175.17   33.16\n",
      "   12         36Y  36Y N   CA    245  246  148.98  135.40                  -43.40\n",
      "   13  10265  36Y  37G C   N     255  257   83.43   78.70    8.63    0.14  -80.20  164.11   11.29\n",
      "   13         37G  37G N   CA    257  258 -173.32 -166.10                  174.10\n",
      "   14  10293  64Y  65C C   N     478  480  -58.76  -63.00    4.53    0.65 -117.90 -174.13    8.03\n",
      "   14         65C  65C N   CA    480  481  -42.69  -41.10                  141.10\n",
      "   15  10294  65C  66Q C   N     484  486  -60.89  -63.80    9.86    1.32   55.10  146.83   19.19\n",
      "   15         66Q  66Q N   CA    486  487  -49.72  -40.30                   40.30\n",
      "   16  10587 358Y 359R C   N    2896 2898  -67.69  -72.10   14.58    1.21  -63.00  163.18   22.36\n",
      "   16        359R 359R N   CA   2898 2899  155.79  141.90                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    1   10   16   43  160  184  192  237  350  428  488\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "4gsb_MODLL.B99990001.pdb       151.94983\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 4gt3\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    38008    35362\n",
      "137 (of 2915 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2742 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     137\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    35362    1119\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      253\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          84.3767\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     140       0      0   0.004   0.004     0.49408       1.000\n",
      " 2 Bond angle potential               :     193       0      0   1.659   1.659      10.185       1.000\n",
      " 3 Stereochemical cosine torsion poten:      81       0      4  54.092  54.092      34.892       1.000\n",
      " 4 Stereochemical improper torsion pot:      55       0      0   1.033   1.033      1.5866       1.000\n",
      " 5 Soft-sphere overlap restraints     :     253       0      0   0.003   0.003     0.30757       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     188       0      0   0.154   0.154      3.0051       1.000\n",
      "10 Distance restraints 2 (N-O)        :     183       0      0   0.229   0.229      3.0776       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      22       0      0   2.960   2.960      1.4584       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      11       0      0  51.148  51.148    -0.31539       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       7       0      0  84.847  84.847      2.1179       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       4       0      0  85.872  85.872      2.0220       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       1       0      0   0.076   0.076     0.55498       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     127       0      0   0.440   0.440     0.46103       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      23      12      3  28.271 131.555      21.868       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      84       0      0   0.720   0.720      2.6615       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 4gt3_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    3092.1851\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9  -71.15  -68.20    9.62    0.67  -62.50  164.87   27.45\n",
      "    1          2A   2A N   CA      9   10  154.46  145.30                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14  -67.01  -68.20   19.14    1.59  -62.50  154.76   25.59\n",
      "    2          3A   3A N   CA     14   15  164.41  145.30                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19  -56.96  -68.20   14.27    0.92  -62.50  177.50   29.36\n",
      "    3          4A   4A N   CA     19   20  136.52  145.30                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24  -75.72  -68.20   18.22    1.22  -62.50  157.75   26.52\n",
      "    4          5A   5A N   CA     24   25  161.90  145.30                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29  -71.79  -68.20    6.53    0.42  -62.50  168.60   28.09\n",
      "    5          6A   6A N   CA     29   30  150.76  145.30                  -40.90\n",
      "    6  10235   6A   7A C   N      32   34  -77.28  -68.20   10.16    0.70  -62.50  169.87   28.59\n",
      "    6          7A   7A N   CA     34   35  149.88  145.30                  -40.90\n",
      "    7  10236   7A   8G C   N      37   39  -76.79  -80.20   19.58    0.61   82.20 -143.93    7.09\n",
      "    7          8G   8G N   CA     39   40  154.82  174.10                    8.50\n",
      "    8  10237   8G   9A C   N      41   43  -70.49  -68.20    3.85    0.24  -62.50  170.89   28.40\n",
      "    8          9A   9A N   CA     43   44  148.40  145.30                  -40.90\n",
      "    9  10238   9A  10G C   N      46   48 -167.60 -167.20    2.71    0.11   82.20 -162.90   14.35\n",
      "    9         10G  10G N   CA     48   49  171.92  174.60                    8.50\n",
      "   10  10389 160T 161C C   N    1283 1285   74.84   57.40   22.97    1.48 -117.90 -154.11    8.87\n",
      "   10        161C 161C N   CA   1285 1286   21.05   36.00                  141.10\n",
      "   11  10586 357G 358Y C   N    2884 2886  -68.89  -63.50    5.54    0.89 -124.30 -174.02    7.81\n",
      "   11        358Y 358Y N   CA   2886 2887  -42.13  -43.40                  135.40\n",
      "   12  10587 358Y 359R C   N    2896 2898   50.01   57.30   26.30    1.32  -63.00  153.83   27.69\n",
      "   12        359R 359R N   CA   2898 2899   63.27   38.00                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:    73   97   1.925\n",
      "  serious non-bonded atom clash:   590 2829   2.129\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     1    1    0    3    1   11   14   54  141  180  186  252  365  413  469\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "4gt3_MODLL.B99990001.pdb        84.37669\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 4gva\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    37657    35011\n",
      "152 (of 2915 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2741 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     152\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    35011    1203\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      279\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         148.8504\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     155       0      0   0.004   0.004     0.59695       1.000\n",
      " 2 Bond angle potential               :     213       0      0   1.592   1.592      10.548       1.000\n",
      " 3 Stereochemical cosine torsion poten:      89       0      4  55.575  55.575      42.372       1.000\n",
      " 4 Stereochemical improper torsion pot:      60       0      0   0.877   0.877      1.1254       1.000\n",
      " 5 Soft-sphere overlap restraints     :     279       0      0   0.002   0.002     0.79990E-01   1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     194       0      0   0.249   0.249      9.7321       1.000\n",
      "10 Distance restraints 2 (N-O)        :     192       0      2   0.466   0.466      21.067       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      24       0      0   3.540   3.540      2.8723       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      13       0      0  93.113  93.113      4.9904       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       8       0      0  93.269  93.269      4.6140       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      0 119.392 119.392      4.0784       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       1       0      0 122.449 122.449     0.80453       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     143       0      0   0.677   0.677      1.6934       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      25      13      4  37.791 129.330      37.771       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      81       0      0   1.021   1.021      6.5051       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 4gva_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    4080.3088\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9  -81.45  -68.20   25.06    1.61  -62.50  153.71   26.16\n",
      "    1          2A   2A N   CA      9   10  166.57  145.30                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14  -68.71  -68.20    4.36    0.33  -62.50  169.59   28.10\n",
      "    2          3A   3A N   CA     14   15  149.63  145.30                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19 -126.91 -134.00    8.23    0.19  -62.50 -172.31   34.20\n",
      "    3          4A   4A N   CA     19   20  142.81  147.00                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24  -71.72  -68.20    8.75    0.82  -62.50  178.43   28.80\n",
      "    4          5A   5A N   CA     24   25  137.29  145.30                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29  -84.81  -68.20   16.71    1.47  -62.50  176.99   30.15\n",
      "    5          6A   6A N   CA     29   30  143.52  145.30                  -40.90\n",
      "    6  10235   6A   7A C   N      32   34  -84.52  -68.20   20.24    1.31  -62.50  163.32   27.90\n",
      "    6          7A   7A N   CA     34   35  157.27  145.30                  -40.90\n",
      "    7  10236   7A   8G C   N      37   39  -79.43  -80.20    2.53    0.14   82.20 -126.86    7.14\n",
      "    7          8G   8G N   CA     39   40  176.51  174.10                    8.50\n",
      "    8  10237   8G   9A C   N      41   43  -85.50  -68.20   21.99    2.17  -62.50  174.15   27.50\n",
      "    8          9A   9A N   CA     43   44  131.73  145.30                  -40.90\n",
      "    9  10238   9A  10G C   N      46   48   80.25   78.70   12.71    0.30   82.20  172.79    8.29\n",
      "    9         10G  10G N   CA     48   49 -178.72 -166.10                    8.50\n",
      "   10  10240  11P  12E C   N      57   59 -131.36 -117.80   20.93    0.71  -63.60 -179.82   27.84\n",
      "   10         12E  12E N   CA     59   60  152.75  136.80                  -40.30\n",
      "   11  10241  12E  13M C   N      66   68  -90.66  -73.00   64.00    4.64  -63.40  125.00   16.74\n",
      "   11         13M  13M N   CA     68   69   81.49  143.00                  -40.50\n",
      "   12  10389 160T 161C C   N    1283 1285  159.54   57.40  108.25   10.67 -117.90  163.35    7.25\n",
      "   12        161C 161C N   CA   1285 1286    0.15   36.00                  141.10\n",
      "   13  10587 358Y 359R C   N    2896 2898  -81.15  -72.10    9.68    0.67  -63.00  174.53   24.82\n",
      "   13        359R 359R N   CA   2898 2899  145.32  141.90                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:    88  203   1.573\n",
      "  serious non-bonded atom clash:    88  204   1.936\n",
      "  serious non-bonded atom clash:    89  201   1.405\n",
      "  serious non-bonded atom clash:    89  202   1.905\n",
      "  serious non-bonded atom clash:    89  203   0.939\n",
      "  serious non-bonded atom clash:    89  204   1.386\n",
      "  serious non-bonded atom clash:    90  201   1.656\n",
      "  serious non-bonded atom clash:    90  202   1.333\n",
      "  serious non-bonded atom clash:    90  203   2.134\n",
      "  serious non-bonded atom clash:    91  201   1.553\n",
      "  serious non-bonded atom clash:    91  202   2.085\n",
      "  serious non-bonded atom clash:    91  203   0.917\n",
      "  serious non-bonded atom clash:    91  204   1.159\n",
      "  serious non-bonded atom clash:    91  213   1.618\n",
      "  serious non-bonded atom clash:    91  214   2.075\n",
      "  serious non-bonded atom clash:   434 2740   2.219\n",
      "  serious non-bonded atom clash:   440 2739   1.628\n",
      "  serious non-bonded atom clash:   440 2740   1.219\n",
      "  serious non-bonded atom clash:   441 2739   1.338\n",
      "  serious non-bonded atom clash:   441 2740   0.288\n",
      "  serious non-bonded atom clash:   442 2739   2.134\n",
      "  serious non-bonded atom clash:  2257 2259   1.954\n",
      "  serious non-bonded atom clash:  2658 2669   2.279\n",
      "  serious non-bonded atom clash:  2715 2718   2.106\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:    19    3    2    6    8   10   26   61  116  166  223  246  362  458  433\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "4gva_MODLL.B99990001.pdb       148.85036\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 4h3p\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    37732    35086\n",
      "191 (of 2915 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2815 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:      -90.3436\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     191\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    35086    1351\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      346\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         178.4137\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     198       0      0   0.004   0.004      1.0638       1.000\n",
      " 2 Bond angle potential               :     272       0      0   1.885   1.885      17.650       1.000\n",
      " 3 Stereochemical cosine torsion poten:     134       0      2  45.096  45.096      41.170       1.000\n",
      " 4 Stereochemical improper torsion pot:      93       0      1   1.378   1.378      4.7016       1.000\n",
      " 5 Soft-sphere overlap restraints     :     346       0      0   0.004   0.004     0.52895       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     175       0      0   0.193   0.193      5.1544       1.000\n",
      "10 Distance restraints 2 (N-O)        :     156       0      0   0.390   0.390      15.798       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      27       2      2  30.468  30.468      16.980       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      16       0      0  76.036  76.036      3.7082       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      13       0      1  71.963  71.963      4.2052       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      0  81.110  81.110      3.6671       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       1       0      0   7.039   7.039     0.62136       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     160       0      0   0.583   0.583      1.5501       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      28      16      6  41.141 138.653      52.425       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      73       0      0   1.079   1.079      9.1888       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 4h3p_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    4666.1729\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 13                           : Mainchain Omega dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10763 176P 176P CA  C    1398 1402  -24.02 -179.18  155.16   36.06 -179.18  155.16   36.06\n",
      "    2  10774 187Y 187Y CA  C    1487 1496 -153.31 -180.00   26.69    5.34 -180.00   26.69    5.34\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9 -125.60 -134.00    9.94    0.53  -62.50  178.32   32.60\n",
      "    1          2A   2A N   CA      9   10  152.32  147.00                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14  -65.03  -68.20    7.64    0.72  -62.50  166.87   27.47\n",
      "    2          3A   3A N   CA     14   15  152.25  145.30                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19  -68.87  -68.20    4.25    0.32  -62.50  169.72   28.13\n",
      "    3          4A   4A N   CA     19   20  149.50  145.30                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24  -73.46  -68.20   13.92    0.94  -62.50  161.29   26.98\n",
      "    4          5A   5A N   CA     24   25  158.18  145.30                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29  -67.16  -68.20    3.03    0.28  -62.50  171.02   28.26\n",
      "    5          6A   6A N   CA     29   30  148.15  145.30                  -40.90\n",
      "    6  10235   6A   7A C   N      32   34  -75.73  -68.20    8.33    0.58  -62.50  170.75   28.65\n",
      "    6          7A   7A N   CA     34   35  148.87  145.30                  -40.90\n",
      "    7  10236   7A   8G C   N      37   39  -91.04  -80.20   11.15    0.65   82.20 -118.53    7.68\n",
      "    7          8G   8G N   CA     39   40  176.71  174.10                    8.50\n",
      "    8  10237   8G   9A C   N      41   43  -81.70  -68.20   25.50    1.64  -62.50  153.38   26.12\n",
      "    8          9A   9A N   CA     43   44  166.93  145.30                  -40.90\n",
      "    9  10238   9A  10G C   N      46   48 -125.33 -167.20   43.51    0.64   82.20 -143.10   16.62\n",
      "    9         10G  10G N   CA     48   49  162.77  174.60                    8.50\n",
      "   10  10407 178H 179D C   N    1420 1422  -75.22  -70.90   21.20    0.84  -63.30  149.42   19.18\n",
      "   10        179D 179D N   CA   1422 1423  171.06  150.30                  -40.00\n",
      "   11  10408 179D 180H C   N    1428 1430  -84.97  -67.60   29.14    1.76  -63.20  155.83   20.89\n",
      "   11        180H 180H N   CA   1430 1431  163.40  140.00                  -42.30\n",
      "   12  10409 180H 181T C   N    1438 1440 -121.62 -124.80   25.33    1.18  -63.20  160.30   24.41\n",
      "   12        181T 181T N   CA   1440 1441  168.63  143.50                  -42.10\n",
      "   13  10410 181T 182G C   N    1445 1447  -76.85  -80.20    3.54    0.26   82.20 -129.56    7.03\n",
      "   13        182G 182G N   CA   1447 1448  175.26  174.10                    8.50\n",
      "   14  10411 182G 183F C   N    1449 1451 -105.07 -124.20   25.85    0.68  -63.20  175.29   21.71\n",
      "   14        183F 183F N   CA   1451 1452  125.92  143.30                  -44.30\n",
      "   15  10412 183F 184L C   N    1460 1462 -135.36 -108.50   60.52    2.93  -63.50  150.35   25.31\n",
      "   15        184L 184L N   CA   1462 1463 -173.27  132.50                  -41.20\n",
      "   16  10413 184L 185T C   N    1468 1470  -89.45  -78.10   75.77    3.62  -63.20  119.90   13.80\n",
      "   16        185T 185T N   CA   1470 1471   74.89  149.80                  -42.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:   586 2643   2.139\n",
      "  serious non-bonded atom clash:   588 2640   1.981\n",
      "  serious non-bonded atom clash:   589 2640   1.360\n",
      "  serious non-bonded atom clash:   589 2641   2.102\n",
      "  serious non-bonded atom clash:   589 2648   2.089\n",
      "  serious non-bonded atom clash:   589 2650   2.185\n",
      "  serious non-bonded atom clash:   590 2637   2.238\n",
      "  serious non-bonded atom clash:   590 2638   1.804\n",
      "  serious non-bonded atom clash:  1986 2083   2.155\n",
      "  serious non-bonded atom clash:  2017 2082   2.126\n",
      "  serious non-bonded atom clash:  2020 2022   2.017\n",
      "  serious non-bonded atom clash:  2306 2339   1.875\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     6    5    1    5    4   12   29   56  103  181  208  283  351  438  483\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "4h3p_MODLL.B99990001.pdb       178.41374\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 4h3q\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    38378    35732\n",
      "51 (of 2915 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2735 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915      51\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    35732     242\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):       48\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          26.4852\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :      51       0      0   0.003   0.003     0.14066       1.000\n",
      " 2 Bond angle potential               :      70       0      0   1.509   1.509      2.8268       1.000\n",
      " 3 Stereochemical cosine torsion poten:      20       0      0  56.856  56.856      13.403       1.000\n",
      " 4 Stereochemical improper torsion pot:      18       0      0   0.726   0.726     0.29545       1.000\n",
      " 5 Soft-sphere overlap restraints     :      48       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      25       0      0   0.136   0.136     0.19281       1.000\n",
      "10 Distance restraints 2 (N-O)        :      24       0      0   0.129   0.129     0.14271       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      10       0      0   1.941   1.941     0.44438       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       1       0      0  98.348  98.348     0.46433       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       1       0      0   9.942   9.942     0.24787       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       1       0      0  10.819  10.819     0.49703       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      11       0      0   0.109   0.109     0.24843E-02   1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      10       8      0  19.998 154.396      7.8273       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 4h3q_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    1855.7925\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9  -74.96  -68.20   14.11    0.92  -62.50  161.90   27.16\n",
      "    1          2A   2A N   CA      9   10  157.69  145.30                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14  -72.06  -68.20    3.86    0.32  -62.50  173.90   28.98\n",
      "    2          3A   3A N   CA     14   15  145.46  145.30                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19  -76.14  -68.20   30.18    2.15  -62.50  145.33   24.51\n",
      "    3          4A   4A N   CA     19   20  174.42  145.30                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24  -58.43  -68.20    9.97    0.91  -62.50  171.88   27.97\n",
      "    4          5A   5A N   CA     24   25  147.27  145.30                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29  -78.22  -68.20   19.85    1.90  -62.50  169.79   27.10\n",
      "    5          6A   6A N   CA     29   30  128.17  145.30                  -40.90\n",
      "    6  10235   6A   7A C   N      32   34 -154.02 -134.00   28.13    0.84  -62.50  177.73   34.00\n",
      "    6          7A   7A N   CA     34   35  166.75  147.00                  -40.90\n",
      "    7  10236   7A   8G C   N      37   39  -66.95  -80.20   24.39    0.59   82.20 -151.90    6.61\n",
      "    7          8G   8G N   CA     39   40  153.62  174.10                    8.50\n",
      "    8  10237   8G   9A C   N      41   43 -138.23 -134.00   25.89    1.36  -62.50  164.97   31.09\n",
      "    8          9A   9A N   CA     43   44  172.54  147.00                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:   589 2640   2.036\n",
      "  serious non-bonded atom clash:   590 2638   2.221\n",
      "  serious non-bonded atom clash:  1281 1296   2.194\n",
      "  serious non-bonded atom clash:  1407 1629   2.241\n",
      "  serious non-bonded atom clash:  1408 1629   1.808\n",
      "  serious non-bonded atom clash:  1408 1630   1.263\n",
      "  serious non-bonded atom clash:  1421 1633   2.272\n",
      "  serious non-bonded atom clash:  1615 2033   2.250\n",
      "  serious non-bonded atom clash:  1987 2083   2.292\n",
      "  serious non-bonded atom clash:  2199 2329   2.214\n",
      "  serious non-bonded atom clash:  2564 2578   2.264\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     3    1    7    6   10    7   18   54  117  181  230  293  384  407  458\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "4h3q_MODLL.B99990001.pdb        26.48522\n",
      "\n",
      "Modelando proteína 4i5h\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    36939    34293\n",
      "277 (of 2915 total) atoms selected for optimization\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:       91.2393\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:       93.3302\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     277\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    34293    1840\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      398\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         156.4222\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     287       0      0   0.002   0.002     0.36111       1.000\n",
      " 2 Bond angle potential               :     394       0      0   1.456   1.456      15.419       1.000\n",
      " 3 Stereochemical cosine torsion poten:     188       0      4  48.741  48.741      67.704       1.000\n",
      " 4 Stereochemical improper torsion pot:     125       0      0   0.791   0.791      1.8097       1.000\n",
      " 5 Soft-sphere overlap restraints     :     398       0      0   0.001   0.001     0.50304E-01   1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     211       0      0   0.298   0.298      7.6282       1.000\n",
      "10 Distance restraints 2 (N-O)        :     205       0      0   0.449   0.449      10.335       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      39       0      0   3.002   3.002      4.1439       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      26       0      0  77.585  77.585      2.9877       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      23       0      0  75.402  75.402      6.4791       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       8       0      0 110.744 110.744      5.3013       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       2       0      0  86.359  86.359      1.0606       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     181       0      0   0.683   0.683      1.4990       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      41      22      3  26.760 131.821      28.886       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     110       0      0   0.509   0.509      2.7557       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 4i5h_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    6405.6196\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9 -111.93 -134.00   22.38    0.88  -62.50  175.53   31.39\n",
      "    1          2A   2A N   CA      9   10  150.68  147.00                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14  -64.82  -68.20    3.52    0.26  -62.50  174.80   28.76\n",
      "    2          3A   3A N   CA     14   15  144.31  145.30                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19 -120.91 -134.00   13.08    0.44  -62.50 -178.32   32.89\n",
      "    3          4A   4A N   CA     19   20  147.07  147.00                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24  -74.52  -68.20   11.18    0.71  -62.50  165.01   27.65\n",
      "    4          5A   5A N   CA     24   25  154.53  145.30                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29 -127.13 -134.00   16.24    1.00  -62.50  170.14   31.34\n",
      "    5          6A   6A N   CA     29   30  161.71  147.00                  -40.90\n",
      "    6  10235   6A   7A C   N      32   34  -67.38  -68.20   10.17    0.85  -62.50  163.74   27.08\n",
      "    6          7A   7A N   CA     34   35  155.44  145.30                  -40.90\n",
      "    7  10236   7A   8G C   N      37   39  -72.54  -80.20    7.66    0.51   82.20 -133.33    6.84\n",
      "    7          8G   8G N   CA     39   40  174.14  174.10                    8.50\n",
      "    8  10237   8G   9A C   N      41   43  -58.56  -68.20    9.90    0.76  -62.50  176.12   28.67\n",
      "    8          9A   9A N   CA     43   44  143.03  145.30                  -40.90\n",
      "    9  10239  10G  11P C   N      50   52  -51.40  -58.70   32.56    2.16  -64.50  151.14   11.98\n",
      "    9         11P  11P N   CA     52   53  -62.23  -30.50                  147.20\n",
      "   10  10405 176P 177D C   N    1402 1404 -102.37  -96.50    5.89    0.24  -63.30  159.61   17.70\n",
      "   10        177D 177D N   CA   1404 1405  114.76  114.20                  -40.00\n",
      "   11  10407 178H 179D C   N    1420 1422  -65.61  -70.90    5.30    0.39  -63.30  170.01   21.13\n",
      "   11        179D 179D N   CA   1422 1423  150.00  150.30                  -40.00\n",
      "   12  10409 180H 181T C   N    1438 1440 -122.15 -124.80   21.83    1.02  -63.20  163.71   24.89\n",
      "   12        181T 181T N   CA   1440 1441  165.17  143.50                  -42.10\n",
      "   13  10410 181T 182G C   N    1445 1447  177.85 -167.20   15.08    0.34   82.20 -170.05   13.48\n",
      "   13        182G 182G N   CA   1447 1448  172.61  174.60                    8.50\n",
      "   14  10411 182G 183F C   N    1449 1451 -125.34 -124.20   26.06    1.20  -63.20  159.00   26.14\n",
      "   14        183F 183F N   CA   1451 1452  169.34  143.30                  -44.30\n",
      "   15  10412 183F 184L C   N    1460 1462 -123.77 -108.50   36.23    1.77   60.20 -137.70   26.32\n",
      "   15        184L 184L N   CA   1462 1463  165.35  132.50                   29.60\n",
      "   16  10559 330K 331F C   N    2657 2659 -142.87 -124.20   20.50    1.03  -63.20 -163.94   22.75\n",
      "   16        331F 331F N   CA   2659 2660  134.84  143.30                  -44.30\n",
      "   17  10561 332D 333M C   N    2676 2678 -142.63 -125.60   19.84    0.59  -63.40 -173.50   34.08\n",
      "   17        333M 333M N   CA   2678 2679  150.66  140.50                  -40.50\n",
      "   18  10562 333M 334E C   N    2684 2686  -91.64 -117.80   52.54    2.00  -63.60  134.49   16.55\n",
      "   18        334E 334E N   CA   2686 2687   91.23  136.80                  -40.30\n",
      "   19  10563 334E 335L C   N    2693 2695 -132.00 -108.50   33.76    1.90  -63.50  164.41   20.19\n",
      "   19        335L 335L N   CA   2695 2696  108.26  132.50                  -41.20\n",
      "   20  10564 335L 336D C   N    2701 2703 -129.65  -96.50   37.28    1.54  -63.30 -176.35   19.71\n",
      "   20        336D 336D N   CA   2703 2704  131.24  114.20                  -40.00\n",
      "   21  10565 336D 337D C   N    2709 2711 -122.12  -96.50   75.25    3.13   54.50  176.64   20.65\n",
      "   21        337D 337D N   CA   2711 2712   43.45  114.20                   40.90\n",
      "   22  10587 358Y 359R C   N    2896 2898 -111.52 -125.20   16.67    0.89  -63.00  175.62   27.08\n",
      "   22        359R 359R N   CA   2898 2899  150.12  140.60                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:   651  822   2.129\n",
      "  serious non-bonded atom clash:   652  822   1.663\n",
      "  serious non-bonded atom clash:   653  821   1.405\n",
      "  serious non-bonded atom clash:   653  822   0.934\n",
      "  serious non-bonded atom clash:   653  823   2.016\n",
      "  serious non-bonded atom clash:   655  822   1.782\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     5    1    0    2    3    9   12   58  118  176  206  264  359  406  457\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "4i5h_MODLL.B99990001.pdb       156.42221\n",
      "\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2836 atoms and 1 coordinate set(s) were parsed in 0.06s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 4iz5\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    38320    35674\n",
      "108 (of 2915 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2741 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     108\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    35674     651\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      101\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          56.9059\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     111       0      0   0.003   0.003     0.21389       1.000\n",
      " 2 Bond angle potential               :     152       0      0   1.800   1.800      9.1072       1.000\n",
      " 3 Stereochemical cosine torsion poten:      68       0      2  50.728  50.728      24.433       1.000\n",
      " 4 Stereochemical improper torsion pot:      42       0      0   0.793   0.793     0.57045       1.000\n",
      " 5 Soft-sphere overlap restraints     :     101       0      0   0.001   0.001     0.20485E-01   1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      77       0      0   0.128   0.128     0.89881       1.000\n",
      "10 Distance restraints 2 (N-O)        :      75       0      0   0.313   0.313      1.7988       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      17       0      0   2.033   2.033     0.82870       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       7       0      0  74.353  74.353     0.47898       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       6       0      0  60.582  60.582      1.9398       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      0  54.930  54.930      2.4873       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       1       0      0  98.900  98.900     0.72033       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      48       0      0   0.331   0.331     0.14554       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      17      12      0  19.578 142.447      10.873       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      25       0      0   0.877   0.877      2.3902       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 4iz5_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    2880.9341\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9  -66.12  -68.20    5.00    0.47  -62.50  169.29   27.92\n",
      "    1          2A   2A N   CA      9   10  149.85  145.30                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14  -68.73  -68.20    7.09    0.55  -62.50  166.85   27.65\n",
      "    2          3A   3A N   CA     14   15  152.37  145.30                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19  -69.76  -68.20   13.74    1.05  -62.50  160.31   26.63\n",
      "    3          4A   4A N   CA     19   20  158.95  145.30                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24  -68.58  -68.20   12.55    1.00  -62.50  161.37   26.75\n",
      "    4          5A   5A N   CA     24   25  157.85  145.30                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29  -64.65  -68.20    5.40    0.53  -62.50  169.74   27.92\n",
      "    5          6A   6A N   CA     29   30  149.37  145.30                  -40.90\n",
      "    6  10235   6A   7A C   N      32   34  -77.25  -68.20   18.45    1.20  -62.50  158.41   26.71\n",
      "    6          7A   7A N   CA     34   35  161.38  145.30                  -40.90\n",
      "    7  10236   7A   8G C   N      37   39 -114.51  -80.20   34.54    2.41   82.20 -130.32   17.63\n",
      "    7          8G   8G N   CA     39   40  170.02  174.10                    8.50\n",
      "    8  10237   8G   9A C   N      41   43  -73.46  -68.20   12.37    0.82  -62.50  162.97   27.26\n",
      "    8          9A   9A N   CA     43   44  156.50  145.30                  -40.90\n",
      "    9  10238   9A  10G C   N      46   48 -168.57 -167.20    6.42    0.26   82.20 -166.42   14.13\n",
      "    9         10G  10G N   CA     48   49  168.32  174.60                    8.50\n",
      "   10  10585 356P 357G C   N    2880 2882  163.95 -167.20   30.92    0.88  -80.20  116.33    8.04\n",
      "   10        357G 357G N   CA   2882 2883  163.46  174.60                  174.10\n",
      "   11  10586 357G 358Y C   N    2884 2886 -106.82  -98.40   42.61    4.42  -63.50  137.06   19.75\n",
      "   11        358Y 358Y N   CA   2886 2887   86.63  128.40                  -43.40\n",
      "   12  10587 358Y 359R C   N    2896 2898 -110.30 -125.20   16.90    0.51  -63.00 -179.95   21.55\n",
      "   12        359R 359R N   CA   2898 2899  132.63  140.60                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    1    4   22   43   84  131  195  251  338  451  467\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "4iz5_MODLL.B99990001.pdb        56.90592\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 4iz7\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    37546    34900\n",
      "206 (of 2915 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2823 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:      -90.3348\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     206\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    34900    1405\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      341\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         102.5159\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     214       0      0   0.003   0.003     0.53284       1.000\n",
      " 2 Bond angle potential               :     295       0      0   1.604   1.604      14.287       1.000\n",
      " 3 Stereochemical cosine torsion poten:     147       0      4  47.586  47.586      48.828       1.000\n",
      " 4 Stereochemical improper torsion pot:      98       0      0   0.793   0.793      1.4780       1.000\n",
      " 5 Soft-sphere overlap restraints     :     341       0      0   0.002   0.002     0.11654       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     173       0      0   0.121   0.121      2.0378       1.000\n",
      "10 Distance restraints 2 (N-O)        :     149       0      0   0.223   0.223      3.0742       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      29       0      0   2.670   2.670      1.8263       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      18       0      0  66.617  66.617      3.2085       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      15       0      0  66.966  66.966      4.3740       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       6       0      0  65.279  65.279      3.3431       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       1       0      0   4.490   4.490     0.56462       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     177       0      0   0.345   0.345     0.52968       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      30      17      2  23.991 134.194      16.272       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      53       0      0   0.567   0.567      2.0424       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 4iz7_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    4454.8340\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9 -120.92 -134.00   14.88    0.76  -62.50  175.03   31.80\n",
      "    1          2A   2A N   CA      9   10  154.11  147.00                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14 -102.84 -134.00   32.59    0.78  -62.50 -177.16   28.27\n",
      "    2          3A   3A N   CA     14   15  137.44  147.00                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19  -74.41  -68.20   13.12    0.86  -62.50  162.68   27.26\n",
      "    3          4A   4A N   CA     19   20  156.86  145.30                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24  -69.05  -68.20   10.22    0.79  -62.50  163.75   27.16\n",
      "    4          5A   5A N   CA     24   25  155.48  145.30                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29  -65.74  -68.20    4.19    0.41  -62.50  170.44   28.09\n",
      "    5          6A   6A N   CA     29   30  148.70  145.30                  -40.90\n",
      "    6  10235   6A   7A C   N      32   34  -69.94  -68.20    8.42    0.62  -62.50  165.73   27.53\n",
      "    6          7A   7A N   CA     34   35  153.54  145.30                  -40.90\n",
      "    7  10236   7A   8G C   N      37   39  -69.15  -80.20   27.56    0.65   82.20 -153.59    6.74\n",
      "    7          8G   8G N   CA     39   40  148.85  174.10                    8.50\n",
      "    8  10237   8G   9A C   N      41   43  -73.23  -68.20    5.08    0.45  -62.50  174.81   29.19\n",
      "    8          9A   9A N   CA     43   44  144.62  145.30                  -40.90\n",
      "    9  10405 176P 177D C   N    1402 1404  -96.90  -96.50   30.41    1.28  -63.30  128.27   14.14\n",
      "    9        177D 177D N   CA   1404 1405   83.79  114.20                  -40.00\n",
      "   10  10407 178H 179D C   N    1420 1422  -83.31  -70.90   22.85    0.78  -63.30  151.84   20.00\n",
      "   10        179D 179D N   CA   1422 1423  169.49  150.30                  -40.00\n",
      "   11  10408 179D 180H C   N    1428 1430 -110.75 -125.60   27.94    0.76  -63.20  164.46   18.12\n",
      "   11        180H 180H N   CA   1430 1431  115.13  138.80                  -42.30\n",
      "   12  10409 180H 181T C   N    1438 1440 -103.65  -78.10   36.30    1.33  -63.20  147.96   21.67\n",
      "   12        181T 181T N   CA   1440 1441  175.58  149.80                  -42.10\n",
      "   13  10410 181T 182G C   N    1445 1447  -70.14  -80.20   29.04    0.72   82.20 -154.21    6.80\n",
      "   13        182G 182G N   CA   1447 1448  146.86  174.10                    8.50\n",
      "   14  10411 182G 183F C   N    1449 1451  -84.86  -71.40   18.39    1.06  -63.20  163.91   24.22\n",
      "   14        183F 183F N   CA   1451 1452  153.23  140.70                  -44.30\n",
      "   15  10412 183F 184L C   N    1460 1462 -122.86 -108.50   19.66    0.88  -63.50 -177.22   29.05\n",
      "   15        184L 184L N   CA   1462 1463  145.92  132.50                  -41.20\n",
      "   16  10413 184L 185T C   N    1468 1470 -114.41 -124.80   21.19    0.66  -63.20  174.80   19.49\n",
      "   16        185T 185T N   CA   1470 1471  125.03  143.50                  -42.10\n",
      "   17  10414 185T 186E C   N    1475 1477 -112.98 -117.80    6.00    0.18  -63.60 -179.58   21.74\n",
      "   17        186E 186E N   CA   1477 1478  133.22  136.80                  -40.30\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:  1189 1371   2.277\n",
      "  serious non-bonded atom clash:  1642 1675   1.598\n",
      "  serious non-bonded atom clash:  1643 1674   1.924\n",
      "  serious non-bonded atom clash:  1643 1676   2.096\n",
      "  serious non-bonded atom clash:  1644 1673   1.672\n",
      "  serious non-bonded atom clash:  1644 1675   0.292\n",
      "  serious non-bonded atom clash:  1645 1673   1.365\n",
      "  serious non-bonded atom clash:  1645 1674   1.285\n",
      "  serious non-bonded atom clash:  1645 1675   1.527\n",
      "  serious non-bonded atom clash:  1645 1676   2.075\n",
      "  serious non-bonded atom clash:  1646 1671   2.289\n",
      "  serious non-bonded atom clash:  1646 1672   1.667\n",
      "  serious non-bonded atom clash:  1646 1673   1.108\n",
      "  serious non-bonded atom clash:  1646 1674   0.913\n",
      "  serious non-bonded atom clash:  1646 1677   2.265\n",
      "  serious non-bonded atom clash:  2715 2718   2.154\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:    12    1    3    4    2    9   14   43  134  182  190  243  374  436  453\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "4iz7_MODLL.B99990001.pdb       102.51588\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 4iza\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    37896    35250\n",
      "155 (of 2915 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 5628 atoms and 1 coordinate set(s) were parsed in 0.04s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:      -90.8424\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     155\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    35250    1235\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      253\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         160.1514\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     160       0      0   0.005   0.005      1.3177       1.000\n",
      " 2 Bond angle potential               :     220       0      0   1.896   1.896      14.378       1.000\n",
      " 3 Stereochemical cosine torsion poten:     104       0      4  49.920  49.920      37.177       1.000\n",
      " 4 Stereochemical improper torsion pot:      71       0      0   0.953   0.953      1.4327       1.000\n",
      " 5 Soft-sphere overlap restraints     :     253       0      0   0.002   0.002     0.93792E-01   1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     211       0      4   0.648   0.648      28.693       1.000\n",
      "10 Distance restraints 2 (N-O)        :     171       0      1   0.546   0.546      10.389       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      23       1      0  36.813  36.813      2.7074       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      12       0      0  65.987  65.987      1.4495       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       9       0      1  70.611  70.611      3.0919       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      0  92.319  92.319      3.0861       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       1       0      0 178.825 178.825     0.52131       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      90       0      0   0.419   0.419     0.33249       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      24      14      4  36.399 133.333      47.996       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     134       0      0   0.740   0.740      7.4858       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 4iza_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    3887.9097\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 13                           : Mainchain Omega dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10774 187Y 187Y CA  C    1487 1496   -3.27 -179.18  175.91   40.88 -179.18  175.91   40.88\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9 -176.59 -134.00   56.76    1.59  -62.50  176.43   34.83\n",
      "    1          2A   2A N   CA      9   10 -175.48  147.00                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14  -71.26  -68.20    9.26    0.64  -62.50  165.29   27.53\n",
      "    2          3A   3A N   CA     14   15  154.04  145.30                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19  -73.27  -68.20   13.16    0.89  -62.50  162.01   27.09\n",
      "    3          4A   4A N   CA     19   20  157.45  145.30                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24 -112.07 -134.00   23.10    0.55  -62.50 -173.92   33.12\n",
      "    4          5A   5A N   CA     24   25  139.75  147.00                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29  -77.38  -68.20   12.94    0.82  -62.50  165.35   27.85\n",
      "    5          6A   6A N   CA     29   30  154.43  145.30                  -40.90\n",
      "    6  10235   6A   7A C   N      32   34  -70.93  -68.20   13.84    1.02  -62.50  160.46   26.72\n",
      "    6          7A   7A N   CA     34   35  158.87  145.30                  -40.90\n",
      "    7  10237   8G   9A C   N      41   43  -66.96  -68.20   11.42    0.96  -62.50  162.51   26.85\n",
      "    7          9A   9A N   CA     43   44  156.65  145.30                  -40.90\n",
      "    8  10238   9A  10G C   N      46   48  149.66 -167.20   43.25    0.90   82.20  176.41   11.71\n",
      "    8         10G  10G N   CA     48   49  171.50  174.60                    8.50\n",
      "    9  10414 185T 186E C   N    1475 1477  -53.70  -69.30   19.81    1.12  -63.60  170.89   23.33\n",
      "    9        186E 186E N   CA   1477 1478  130.30  142.50                  -40.30\n",
      "   10  10415 186E 187Y C   N    1484 1486  -24.31  -98.40   75.22    3.03   55.90  129.70    6.86\n",
      "   10        187Y 187Y N   CA   1486 1487  141.42  128.40                   39.50\n",
      "   11  10416 187Y 188V C   N    1496 1498  163.11   55.90  123.62    9.75 -125.40 -179.85   10.21\n",
      "   11        188V 188V N   CA   1498 1499  -22.05   39.50                  143.30\n",
      "   12  10417 188V 189A C   N    1503 1505  -72.57  -62.50   49.73    7.72  -62.50   49.73    7.72\n",
      "   12        189A 189A N   CA   1505 1506    7.80  -40.90                  -40.90\n",
      "   13  10586 357G 358Y C   N    2884 2886   52.05   55.90    7.48    0.36 -124.30 -162.25   11.26\n",
      "   13        358Y 358Y N   CA   2886 2887   45.91   39.50                  135.40\n",
      "   14  10587 358Y 359R C   N    2896 2898  -70.66  -72.10    6.36    0.46  -63.00  176.97   23.40\n",
      "   14        359R 359R N   CA   2898 2899  135.71  141.90                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:   589 2640   2.059\n",
      "  serious non-bonded atom clash:   590 2638   2.270\n",
      "  serious non-bonded atom clash:   888  925   2.225\n",
      "  serious non-bonded atom clash:  2715 2718   2.129\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     1    1    2    2    5    8   12   44  118  193  189  225  371  421  492\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "4iza_MODLL.B99990001.pdb       160.15135\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 4n0s\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    38083    35437\n",
      "207 (of 2915 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2778 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     207\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    35437    2229\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      574\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         610.0853\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     213       2      3   0.024   0.024      26.485       1.000\n",
      " 2 Bond angle potential               :     295       2      3   3.131   3.131      46.641       1.000\n",
      " 3 Stereochemical cosine torsion poten:     132       0      7  51.734  51.734      56.825       1.000\n",
      " 4 Stereochemical improper torsion pot:      89       0      0   0.973   0.973      1.9377       1.000\n",
      " 5 Soft-sphere overlap restraints     :     574      13     16   0.077   0.077      404.11       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     473       0      0   0.179   0.179      9.8335       1.000\n",
      "10 Distance restraints 2 (N-O)        :     471       0      1   0.420   0.420      17.563       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      31       1      2  32.851  32.851      11.966       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      21       0      0  47.751  47.751      2.6974       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      14       0      1  50.809  50.809      7.1428       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       6       0      0  90.498  90.498      5.1229       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       3       0      0 110.100 110.100      1.8243       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     247       0      0   0.376   0.376     0.68705       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      34      12      4  25.062 104.801      10.827       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     200       0      0   0.588   0.588      6.4224       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 4n0s_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    4701.9595\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  1                           : Bond length potential                   \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1      3   1M   1M SD  CG      5    4    2.07    1.82    0.25    6.54    1.82    0.25    6.54\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   2991   1M   1M CB  CG      3    4  139.15  114.50   24.65    6.03  114.50   24.65    6.03\n",
      "    2   2992   1M   1M CG  SD      4    5  124.92   95.00   29.92    5.60   95.00   29.92    5.60\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9 -156.23 -134.00   22.28    0.68  -62.50 -165.34   36.91\n",
      "    1          2A   2A N   CA      9   10  148.50  147.00                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14 -140.66 -134.00    8.64    0.23  -62.50 -175.97   34.35\n",
      "    2          3A   3A N   CA     14   15  152.50  147.00                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19 -139.50 -134.00   18.93    0.92  -62.50  172.16   32.34\n",
      "    3          4A   4A N   CA     19   20  165.12  147.00                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24  -72.42  -68.20   18.71    1.36  -62.50  155.89   26.05\n",
      "    4          5A   5A N   CA     24   25  163.53  145.30                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29 -163.91 -134.00   89.30    4.19  -62.50  134.24   27.03\n",
      "    5          6A   6A N   CA     29   30 -128.86  147.00                  -40.90\n",
      "    6  10293  64Y  65C C   N     478  480  -62.13  -63.00    1.84    0.29 -117.90 -172.11    8.09\n",
      "    6         65C  65C N   CA    480  481  -39.48  -41.10                  141.10\n",
      "    7  10294  65C  66Q C   N     484  486  -63.50  -63.80    4.89    0.70   55.10  146.20   19.22\n",
      "    7         66Q  66Q N   CA    486  487  -45.18  -40.30                   40.30\n",
      "    8  10389 160T 161C C   N    1283 1285   75.07   57.40   37.48    1.26 -117.90 -143.23    9.46\n",
      "    8        161C 161C N   CA   1285 1286    2.94   36.00                  141.10\n",
      "    9  10481 252L 253N C   N    2013 2015 -103.96 -119.90   33.03    1.10  -63.20  154.63   17.41\n",
      "    9        253N 253N N   CA   2015 2016  108.06  137.00                  -41.10\n",
      "   10  10585 356P 357G C   N    2880 2882   86.14   78.70   22.01    0.36  -80.20  166.34   11.06\n",
      "   10        357G 357G N   CA   2882 2883  173.18 -166.10                  174.10\n",
      "   11  10586 357G 358Y C   N    2884 2886 -127.56 -124.30    6.91    0.27  -63.50 -173.55   32.80\n",
      "   11        358Y 358Y N   CA   2886 2887  141.50  135.40                  -43.40\n",
      "   12  10587 358Y 359R C   N    2896 2898 -102.21 -125.20   27.72    0.82  -63.00  170.78   20.72\n",
      "   12        359R 359R N   CA   2898 2899  125.12  140.60                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:     4  754   2.112\n",
      "  serious non-bonded atom clash:     5  754   2.184\n",
      "  serious non-bonded atom clash:     6  731   2.181\n",
      "  serious non-bonded atom clash:    88  411   1.395\n",
      "  serious non-bonded atom clash:    88  412   1.629\n",
      "  serious non-bonded atom clash:    89  409   2.071\n",
      "  serious non-bonded atom clash:    89  410   1.406\n",
      "  serious non-bonded atom clash:    89  411   0.730\n",
      "  serious non-bonded atom clash:    89  412   1.868\n",
      "  serious non-bonded atom clash:    89  413   2.171\n",
      "  serious non-bonded atom clash:    90  410   1.838\n",
      "  serious non-bonded atom clash:    90  411   1.961\n",
      "  serious non-bonded atom clash:    90  413   1.532\n",
      "  serious non-bonded atom clash:    90  414   1.233\n",
      "  serious non-bonded atom clash:    91  409   1.931\n",
      "  serious non-bonded atom clash:    91  410   0.875\n",
      "  serious non-bonded atom clash:    91  411   1.030\n",
      "  serious non-bonded atom clash:    91  412   1.672\n",
      "  serious non-bonded atom clash:    91  413   2.257\n",
      "  serious non-bonded atom clash:   650 1319   2.214\n",
      "  serious non-bonded atom clash:   650 1321   2.259\n",
      "  serious non-bonded atom clash:   651 1321   1.873\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:    15    4    3    4    3   14   32   52  125  192  225  247  379  429  508\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "4n0s_MODLL.B99990001.pdb       610.08533\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 4n4s\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    37053    34407\n",
      "181 (of 2915 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2812 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:      -90.7235\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     181\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    34407    1140\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      215\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          95.3366\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     188       0      0   0.003   0.003     0.51661       1.000\n",
      " 2 Bond angle potential               :     259       0      0   1.744   1.744      14.295       1.000\n",
      " 3 Stereochemical cosine torsion poten:     126       0      3  46.435  46.435      38.477       1.000\n",
      " 4 Stereochemical improper torsion pot:      83       0      0   0.795   0.795      1.1027       1.000\n",
      " 5 Soft-sphere overlap restraints     :     215       0      0   0.002   0.002     0.94194E-01   1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     134       0      0   0.168   0.168      2.3112       1.000\n",
      "10 Distance restraints 2 (N-O)        :     111       0      2   0.298   0.298      7.5791       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      27       0      0   2.797   2.797      1.6587       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      15       0      0  71.525  71.525      2.4249       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      12       0      0  59.087  59.087      2.4066       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      0  71.783  71.783      1.4178       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       1       0      0  94.520  94.520     0.72459       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     113       0      0   0.381   0.381     0.35599       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      28      15      1  25.453 129.166      20.024       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      38       0      0   0.848   0.848      1.9482       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 4n4s_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    4329.2778\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9  -65.66  -68.20    2.97    0.20  -62.50  175.37   28.90\n",
      "    1          2A   2A N   CA      9   10  143.76  145.30                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14  -65.67  -68.20    3.53    0.35  -62.50  171.36   28.24\n",
      "    2          3A   3A N   CA     14   15  147.77  145.30                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19  -73.48  -68.20   15.77    1.09  -62.50  159.32   26.66\n",
      "    3          4A   4A N   CA     19   20  160.16  145.30                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24  -63.12  -68.20    5.67    0.55  -62.50  171.30   28.10\n",
      "    4          5A   5A N   CA     24   25  147.80  145.30                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29  -67.26  -68.20    1.21    0.12  -62.50  173.11   28.60\n",
      "    5          6A   6A N   CA     29   30  146.06  145.30                  -40.90\n",
      "    6  10235   6A   7A C   N      32   34 -133.31 -134.00   12.59    0.73  -62.50  174.55   32.40\n",
      "    6          7A   7A N   CA     34   35  159.57  147.00                  -40.90\n",
      "    7  10236   7A   8G C   N      37   39 -138.01 -167.20   29.35    0.63   82.20 -140.60   16.47\n",
      "    7          8G   8G N   CA     39   40  177.60  174.60                    8.50\n",
      "    8  10237   8G   9A C   N      41   43  -63.24  -68.20    7.56    0.48  -62.50  179.51   29.45\n",
      "    8          9A   9A N   CA     43   44  139.60  145.30                  -40.90\n",
      "    9  10406 177D 178H C   N    1410 1412   55.16   56.30    3.12    0.22  -63.20  146.30   25.02\n",
      "    9        178H 178H N   CA   1412 1413   43.71   40.80                  -42.30\n",
      "   10  10407 178H 179D C   N    1420 1422  -77.67  -96.50   21.23    0.88  -63.30  145.11   17.09\n",
      "   10        179D 179D N   CA   1422 1423  104.40  114.20                  -40.00\n",
      "   11  10409 180H 181T C   N    1438 1440  -74.92  -78.10    4.76    0.30  -63.20  164.96   21.93\n",
      "   11        181T 181T N   CA   1440 1441  153.35  149.80                  -42.10\n",
      "   12  10410 181T 182G C   N    1445 1447 -123.20  -80.20   57.71    4.13   82.20 -159.86   15.56\n",
      "   12        182G 182G N   CA   1447 1448  135.61  174.10                    8.50\n",
      "   13  10411 182G 183F C   N    1449 1451 -147.19 -124.20   40.72    1.24  -63.20  162.22   27.79\n",
      "   13        183F 183F N   CA   1451 1452  176.92  143.30                  -44.30\n",
      "   14  10412 183F 184L C   N    1460 1462  -70.28  -70.70    0.50    0.05  -63.50  177.06   24.98\n",
      "   14        184L 184L N   CA   1462 1463  141.87  141.60                  -41.20\n",
      "   15  10587 358Y 359R C   N    2896 2898  -74.06  -72.10    7.14    0.51  -63.00  170.50   23.78\n",
      "   15        359R 359R N   CA   2898 2899  148.76  141.90                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:   253  500   2.168\n",
      "  serious non-bonded atom clash:   254  498   2.124\n",
      "  serious non-bonded atom clash:   254  499   1.632\n",
      "  serious non-bonded atom clash:   254  500   1.009\n",
      "  serious non-bonded atom clash:   254  501   2.099\n",
      "  serious non-bonded atom clash:   503 1334   2.220\n",
      "  serious non-bonded atom clash:   653  821   1.800\n",
      "  serious non-bonded atom clash:   653  822   1.223\n",
      "  serious non-bonded atom clash:   653  823   2.219\n",
      "  serious non-bonded atom clash:   655  821   1.910\n",
      "  serious non-bonded atom clash:   655  822   1.893\n",
      "  serious non-bonded atom clash:   655  823   1.504\n",
      "  serious non-bonded atom clash:  2715 2718   2.064\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     9    2    2    1    1    6   16   47   92  129  196  279  335  404  449\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "4n4s_MODLL.B99990001.pdb        95.33658\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 4nif\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    38195    35549\n",
      "109 (of 2915 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2676 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     109\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    35549     863\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      189\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          65.7582\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     113       0      0   0.004   0.004     0.44580       1.000\n",
      " 2 Bond angle potential               :     156       0      0   1.688   1.688      8.2296       1.000\n",
      " 3 Stereochemical cosine torsion poten:      69       0      1  50.225  50.225      26.501       1.000\n",
      " 4 Stereochemical improper torsion pot:      53       0      0   0.834   0.834      1.0616       1.000\n",
      " 5 Soft-sphere overlap restraints     :     189       0      0   0.000   0.000     0.25999E-02   1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     148       0      0   0.186   0.186      3.0667       1.000\n",
      "10 Distance restraints 2 (N-O)        :     116       0      0   0.381   0.381      6.3234       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      18       0      0   3.579   3.579      1.9384       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       7       0      0  55.174  55.174      1.5844       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       6       0      0  20.600  20.600      3.4850       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       1       0      0 144.928 144.928     0.51484       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      79       0      0   0.423   0.423     0.39974       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      19      11      3  22.488 128.731      11.110       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      78       0      0   0.361   0.361      1.0948       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 4nif_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    2895.8154\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9  -60.81  -68.20   10.51    1.04  -62.50  166.33   27.18\n",
      "    1          2A   2A N   CA      9   10  152.78  145.30                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14 -129.69 -134.00   10.38    0.64  -62.50  175.99   32.44\n",
      "    2          3A   3A N   CA     14   15  156.45  147.00                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19  -74.90  -68.20   29.93    2.18  -62.50  145.16   24.42\n",
      "    3          4A   4A N   CA     19   20  174.48  145.30                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24  -56.94  -68.20   15.44    0.98  -62.50  175.73   29.07\n",
      "    4          5A   5A N   CA     24   25  134.74  145.30                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29  -82.47  -68.20   15.12    1.10  -62.50  169.99   28.88\n",
      "    5          6A   6A N   CA     29   30  150.29  145.30                  -40.90\n",
      "    6  10235   6A   7A C   N      32   34  -71.99  -68.20   11.18    0.77  -62.50  163.56   27.28\n",
      "    6          7A   7A N   CA     34   35  155.82  145.30                  -40.90\n",
      "    7  10236   7A   8G C   N      37   39   79.99   78.70    1.39    0.05   82.20  175.14    8.61\n",
      "    7          8G   8G N   CA     39   40 -166.62 -166.10                    8.50\n",
      "    8  10237   8G   9A C   N      41   43  -71.34  -68.20    7.00    0.46  -62.50  167.78   27.94\n",
      "    8          9A   9A N   CA     43   44  151.55  145.30                  -40.90\n",
      "    9  10407 178H 179D C   N    1420 1422 -116.62  -96.50   56.00    2.33  -63.30  115.04   12.20\n",
      "    9        179D 179D N   CA   1422 1423   61.94  114.20                  -40.00\n",
      "   10  10408 179D 180H C   N    1428 1430 -153.65 -125.60   28.27    0.92  -63.20 -162.66   30.54\n",
      "   10        180H 180H N   CA   1430 1431  142.31  138.80                  -42.30\n",
      "   11  10409 180H 181T C   N    1438 1440 -120.56 -124.80   11.62    0.39  -63.20 -176.05   20.36\n",
      "   11        181T 181T N   CA   1440 1441  132.68  143.50                  -42.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:  1469 1474   2.263\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    1    0    6    4   21   51  107  181  214  253  384  432  470\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "4nif_MODLL.B99990001.pdb        65.75823\n",
      "\n",
      "Modelando proteína 4o6e\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    36277    33631\n",
      "362 (of 2915 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2794 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:       90.7144\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:      -93.5435\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:      -93.2682\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     362\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    33631    2723\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      725\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         310.0446\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     376       0      0   0.004   0.004      2.1604       1.000\n",
      " 2 Bond angle potential               :     516       0      3   2.310   2.310      49.035       1.000\n",
      " 3 Stereochemical cosine torsion poten:     250       0     10  51.033  51.033      97.696       1.000\n",
      " 4 Stereochemical improper torsion pot:     167       0      0   1.373   1.373      6.3437       1.000\n",
      " 5 Soft-sphere overlap restraints     :     725       0      0   0.003   0.003     0.85281       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     422       0      0   0.198   0.198      10.438       1.000\n",
      "10 Distance restraints 2 (N-O)        :     383       0      0   0.351   0.351      10.243       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      51       1      1   5.019   5.019      13.855       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      35       0      4  81.237  81.237      23.053       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      28       0      0  75.856  75.856      9.3273       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:      12       0      0  88.116  88.116      6.2471       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       3       0      0  87.472  87.472      2.0663       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     250       0      0   0.689   0.689      2.8468       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      54      30      9  34.365 130.738      64.338       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     176       0      0   1.498   1.498      11.543       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 4o6e_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    9220.3613\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 13                           : Mainchain Omega dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10770 183F 183F CA  C    1452 1460 -156.83 -180.00   23.17    4.63 -180.00   23.17    4.63\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9 -113.75 -134.00   20.31    0.62  -62.50 -178.94   32.39\n",
      "    1          2A   2A N   CA      9   10  145.45  147.00                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14 -112.83 -134.00   21.22    0.65  -62.50 -179.31   32.28\n",
      "    2          3A   3A N   CA     14   15  145.56  147.00                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19 -108.45 -134.00   26.41    0.66  -62.50 -175.38   32.68\n",
      "    3          4A   4A N   CA     19   20  140.30  147.00                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24 -130.95 -134.00    8.00    0.49  -62.50  178.36   32.90\n",
      "    4          5A   5A N   CA     24   25  154.40  147.00                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29  -66.99  -68.20    1.83    0.18  -62.50  172.49   28.49\n",
      "    5          6A   6A N   CA     29   30  146.67  145.30                  -40.90\n",
      "    6  10235   6A   7A C   N      32   34 -123.70 -134.00   10.38    0.39  -62.50 -178.54   33.01\n",
      "    6          7A   7A N   CA     34   35  148.27  147.00                  -40.90\n",
      "    7  10236   7A   8G C   N      37   39 -151.60 -167.20   28.28    0.78   82.20 -169.65   14.41\n",
      "    7          8G   8G N   CA     39   40  151.01  174.60                    8.50\n",
      "    8  10237   8G   9A C   N      41   43 -141.17 -134.00   12.27    0.44  -62.50 -179.78   33.75\n",
      "    8          9A   9A N   CA     43   44  156.96  147.00                  -40.90\n",
      "    9  10238   9A  10G C   N      46   48 -130.78 -167.20   48.78    1.02   82.20 -161.33   15.35\n",
      "    9         10G  10G N   CA     48   49  142.14  174.60                    8.50\n",
      "   10  10240  11P  12E C   N      57   59 -116.26 -117.80   10.58    0.56  -63.60 -179.70   27.02\n",
      "   10         12E  12E N   CA     59   60  147.26  136.80                  -40.30\n",
      "   11  10241  12E  13M C   N      66   68 -103.74 -125.60   23.25    0.74  -63.40  177.71   23.70\n",
      "   11         13M  13M N   CA     68   69  132.57  140.50                  -40.50\n",
      "   12  10404 175D 176P C   N    1395 1397  -64.90  -58.70   22.39    2.36  -64.50  160.79   11.98\n",
      "   12        176P 176P N   CA   1397 1398  -52.01  -30.50                  147.20\n",
      "   13  10405 176P 177D C   N    1402 1404   50.05   54.50   11.73    1.22  -63.30  133.24   22.72\n",
      "   13        177D 177D N   CA   1404 1405   30.04   40.90                  -40.00\n",
      "   14  10406 177D 178H C   N    1410 1412   53.39   56.30    4.67    0.68  -63.20  141.08   24.18\n",
      "   14        178H 178H N   CA   1412 1413   37.15   40.80                  -42.30\n",
      "   15  10407 178H 179D C   N    1420 1422 -109.93  -70.90   50.19    2.26  -63.30  145.81   21.01\n",
      "   15        179D 179D N   CA   1422 1423 -178.15  150.30                  -40.00\n",
      "   16  10408 179D 180H C   N    1428 1430 -118.13 -125.60   21.88    0.95  -63.20  167.59   24.55\n",
      "   16        180H 180H N   CA   1430 1431  159.37  138.80                  -42.30\n",
      "   17  10409 180H 181T C   N    1438 1440 -126.93 -124.80   48.42    2.01  -63.20  141.23   22.25\n",
      "   17        181T 181T N   CA   1440 1441 -168.13  143.50                  -42.10\n",
      "   18  10412 183F 184L C   N    1460 1462  -50.60  -70.70   21.71    1.66  -63.50  175.08   25.07\n",
      "   18        184L 184L N   CA   1462 1463  133.40  141.60                  -41.20\n",
      "   19  10413 184L 185T C   N    1468 1470  -83.36  -78.10   10.53    0.62  -63.20  178.37   24.21\n",
      "   19        185T 185T N   CA   1470 1471  140.68  149.80                  -42.10\n",
      "   20  10414 185T 186E C   N    1475 1477  -68.43  -69.30   50.74    3.84  -63.60  132.16   17.36\n",
      "   20        186E 186E N   CA   1477 1478   91.77  142.50                  -40.30\n",
      "   21  10415 186E 187Y C   N    1484 1486 -130.79 -124.30   15.55    1.10  -63.50  177.89   25.25\n",
      "   21        187Y 187Y N   CA   1486 1487  121.28  135.40                  -43.40\n",
      "   22  10430 201N 202S C   N    1616 1618 -157.12 -136.60   22.25    1.10  -64.10 -159.53   12.89\n",
      "   22        202S 202S N   CA   1618 1619  142.59  151.20                  -35.00\n",
      "   23  10431 202S 203K C   N    1622 1624  122.53 -118.00  146.17    4.91  -62.90 -160.83   34.79\n",
      "   23        203K 203K N   CA   1624 1625 -136.68  139.10                  -40.80\n",
      "   24  10432 203K 204G C   N    1631 1633  -95.17  -80.20   21.22    1.50   82.20 -127.34    8.01\n",
      "   24        204G 204G N   CA   1633 1634  159.07  174.10                    8.50\n",
      "   25  10559 330K 331F C   N    2657 2659 -125.60 -124.20   20.49    0.93  -63.20  164.27   26.89\n",
      "   25        331F 331F N   CA   2659 2660  163.75  143.30                  -44.30\n",
      "   26  10560 331F 332D C   N    2668 2670  -76.30  -70.90   67.13    3.04  -63.30  103.61   13.61\n",
      "   26        332D 332D N   CA   2670 2671 -142.79  150.30                  -40.00\n",
      "   27  10561 332D 333M C   N    2676 2678  -54.04  -73.00   21.09    1.69  -63.40  167.53   24.30\n",
      "   27        333M 333M N   CA   2678 2679  152.23  143.00                  -40.50\n",
      "   28  10562 333M 334E C   N    2684 2686  -56.41  -69.30   15.40    0.89   54.60  143.97    9.77\n",
      "   28        334E 334E N   CA   2686 2687  134.08  142.50                   42.40\n",
      "   29  10586 357G 358Y C   N    2884 2886 -119.43 -124.30    5.00    0.40  -63.50 -171.54   27.29\n",
      "   29        358Y 358Y N   CA   2886 2887  136.57  135.40                  -43.40\n",
      "   30  10587 358Y 359R C   N    2896 2898 -112.73 -125.20   13.33    0.65  -63.00 -179.41   27.83\n",
      "   30        359R 359R N   CA   2898 2899  145.30  140.60                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    0    4   10   58  114  137  203  213  319  427  484\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "4o6e_MODLL.B99990001.pdb       310.04459\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 4qp1\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    37685    35039\n",
      "202 (of 2915 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2736 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     202\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    35039    1762\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      445\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         190.6372\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     209       0      0   0.005   0.005      1.5339       1.000\n",
      " 2 Bond angle potential               :     286       0      1   2.250   2.250      27.246       1.000\n",
      " 3 Stereochemical cosine torsion poten:     125       0      9  54.137  54.137      54.683       1.000\n",
      " 4 Stereochemical improper torsion pot:      84       0      0   1.439   1.439      4.1414       1.000\n",
      " 5 Soft-sphere overlap restraints     :     445       0      0   0.003   0.003     0.56925       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     325       0      0   0.151   0.151      6.5678       1.000\n",
      "10 Distance restraints 2 (N-O)        :     298       0      0   0.313   0.313      16.478       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      30       0      0   4.176   4.176      6.1708       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      19       0      0  64.113  64.113      3.0276       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      18       0      0  64.396  64.396      8.4238       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       8       0      0  82.372  82.372      4.6520       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       1       0      0 118.159 118.159     0.94832       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     171       0      0   0.423   0.423     0.85138       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      32      15      6  35.067 126.138      37.773       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     156       0      0   1.043   1.043      17.571       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 4qp1_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    4898.6777\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9  -67.92  -68.20    1.47    0.13  -62.50  172.44   28.53\n",
      "    1          2A   2A N   CA      9   10  146.75  145.30                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14  -74.19  -68.20    8.10    0.51  -62.50  168.75   28.24\n",
      "    2          3A   3A N   CA     14   15  150.76  145.30                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19 -120.08 -134.00   16.92    0.42  -62.50 -172.66   28.49\n",
      "    3          4A   4A N   CA     19   20  137.38  147.00                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24  -69.55  -68.20    5.08    0.45  -62.50  178.84   29.66\n",
      "    4          5A   5A N   CA     24   25  140.40  145.30                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29 -160.37 -134.00   39.24    1.25  -62.50  173.33   33.58\n",
      "    5          6A   6A N   CA     29   30  176.05  147.00                  -40.90\n",
      "    6  10235   6A   7A C   N      32   34  -65.50  -68.20   11.95    1.06  -62.50  162.19   26.73\n",
      "    6          7A   7A N   CA     34   35  156.94  145.30                  -40.90\n",
      "    7  10236   7A   8G C   N      37   39 -132.01 -167.20   38.78    0.58   82.20 -150.97   15.99\n",
      "    7          8G   8G N   CA     39   40  158.30  174.60                    8.50\n",
      "    8  10237   8G   9A C   N      41   43 -162.79 -134.00   59.24    2.41  -62.50  156.65   30.88\n",
      "    8          9A   9A N   CA     43   44 -161.23  147.00                  -40.90\n",
      "    9  10240  11P  12E C   N      57   59 -157.41 -117.80   39.76    1.54  -63.60 -162.61   22.84\n",
      "    9         12E  12E N   CA     59   60  133.37  136.80                  -40.30\n",
      "   10  10354 125H 126I C   N     992  994  -58.55  -63.40    7.92    1.47 -120.60  178.75    8.01\n",
      "   10        126I 126I N   CA    994  995  -37.33  -43.60                  130.30\n",
      "   11  10558 329F 330K C   N    2648 2650  -94.17 -118.00   53.99    2.20  -62.90  135.13   16.11\n",
      "   11        330K 330K N   CA   2650 2651   90.66  139.10                  -40.80\n",
      "   12  10559 330K 331F C   N    2657 2659  -84.30  -71.40   38.48    3.37  -63.20  150.24   19.48\n",
      "   12        331F 331F N   CA   2659 2660  104.45  140.70                  -44.30\n",
      "   13  10560 331F 332D C   N    2668 2670  -71.58  -70.90   10.69    0.49  -63.30  159.25   20.17\n",
      "   13        332D 332D N   CA   2670 2671  160.96  150.30                  -40.00\n",
      "   14  10561 332D 333M C   N    2676 2678 -113.93 -125.60   47.87    2.02  -63.40  143.76   18.27\n",
      "   14        333M 333M N   CA   2678 2679   94.08  140.50                  -40.50\n",
      "   15  10563 334E 335L C   N    2693 2695  130.24 -108.50  121.39    5.85  -63.50 -123.56   30.39\n",
      "   15        335L 335L N   CA   2695 2696  126.91  132.50                  -41.20\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:  1509 1562   2.278\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    1    0    3   11   22   43   81  153  218  290  368  431  430\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "4qp1_MODLL.B99990001.pdb       190.63718\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 4qp2\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    37179    34533\n",
      "313 (of 2915 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2777 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     313\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    34533    3073\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      710\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         275.4376\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     324       0      0   0.004   0.004      1.3639       1.000\n",
      " 2 Bond angle potential               :     445       0      3   2.087   2.087      36.662       1.000\n",
      " 3 Stereochemical cosine torsion poten:     198       0      8  50.464  50.464      79.271       1.000\n",
      " 4 Stereochemical improper torsion pot:     143       0      0   1.424   1.424      6.1521       1.000\n",
      " 5 Soft-sphere overlap restraints     :     710       0      0   0.003   0.003     0.94103       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     568       0      0   0.162   0.162      7.9258       1.000\n",
      "10 Distance restraints 2 (N-O)        :     492       0      0   0.293   0.293      13.698       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      46       3      1  37.241  37.241      12.984       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      31       0      0  71.468  71.468      12.763       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      25       0      1  84.421  84.421      19.584       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       9       0      0 112.669 112.669      6.3381       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       2       0      0   4.592   4.592     0.50733       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     372       0      0   0.603   0.603      3.2484       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      50      23      8  30.503 116.007      43.350       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     368       0      0   0.976   0.976      30.650       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 4qp2_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    7626.4277\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 13                           : Mainchain Omega dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10765 178H 178H CA  C    1413 1420   -4.37 -179.18  174.81   40.63 -179.18  174.81   40.63\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9  -73.36  -68.20    8.27    0.52  -62.50  167.69   28.03\n",
      "    1          2A   2A N   CA      9   10  151.77  145.30                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14  -72.54  -68.20    6.78    0.43  -62.50  168.89   28.18\n",
      "    2          3A   3A N   CA     14   15  150.51  145.30                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19  -73.71  -68.20    6.76    0.44  -62.50  170.26   28.46\n",
      "    3          4A   4A N   CA     19   20  149.22  145.30                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24  -76.33  -68.20   19.18    1.28  -62.50  157.04   26.44\n",
      "    4          5A   5A N   CA     24   25  162.67  145.30                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29  -65.75  -68.20    4.99    0.48  -62.50  169.48   27.93\n",
      "    5          6A   6A N   CA     29   30  149.66  145.30                  -40.90\n",
      "    6  10235   6A   7A C   N      32   34  -73.00  -68.20   15.44    1.08  -62.50  159.47   26.66\n",
      "    6          7A   7A N   CA     34   35  159.97  145.30                  -40.90\n",
      "    7  10236   7A   8G C   N      37   39  102.49   78.70   68.12    1.09   82.20  123.24    6.98\n",
      "    7          8G   8G N   CA     39   40  130.07 -166.10                    8.50\n",
      "    8  10237   8G   9A C   N      41   43 -141.54 -134.00   16.66    0.70  -62.50  175.99   33.07\n",
      "    8          9A   9A N   CA     43   44  161.86  147.00                  -40.90\n",
      "    9  10238   9A  10G C   N      46   48 -132.07 -167.20   43.42    0.80   82.20 -157.52   15.58\n",
      "    9         10G  10G N   CA     48   49  149.08  174.60                    8.50\n",
      "   10  10354 125H 126I C   N     992  994  -58.73  -63.40    5.18    0.98 -120.60 -177.53    8.22\n",
      "   10        126I 126I N   CA    994  995  -41.36  -43.60                  130.30\n",
      "   11  10408 179D 180H C   N    1428 1430   55.02   56.30    9.30    0.73  -63.20  149.99   25.58\n",
      "   11        180H 180H N   CA   1430 1431   50.01   40.80                  -42.30\n",
      "   12  10409 180H 181T C   N    1438 1440 -103.24 -124.80   29.58    0.97  -63.20  170.13   19.44\n",
      "   12        181T 181T N   CA   1440 1441  123.25  143.50                  -42.10\n",
      "   13  10410 181T 182G C   N    1445 1447  166.55 -167.20   26.26    0.52   82.20 -174.49   12.83\n",
      "   13        182G 182G N   CA   1447 1448  173.73  174.60                    8.50\n",
      "   14  10412 183F 184L C   N    1460 1462  -59.47  -70.70   17.64    1.10  -63.50  169.25   23.73\n",
      "   14        184L 184L N   CA   1462 1463  128.00  141.60                  -41.20\n",
      "   15  10413 184L 185T C   N    1468 1470 -115.33 -124.80   12.48    0.42  -63.20 -175.02   20.72\n",
      "   15        185T 185T N   CA   1470 1471  135.38  143.50                  -42.10\n",
      "   16  10414 185T 186E C   N    1475 1477  -74.71  -69.30   19.41    1.29  -63.60  158.94   21.81\n",
      "   16        186E 186E N   CA   1477 1478  161.14  142.50                  -40.30\n",
      "   17  10429 200L 201N C   N    1608 1610 -150.97 -119.90   47.02    1.40  -63.20  170.88   26.24\n",
      "   17        201N 201N N   CA   1610 1611  172.29  137.00                  -41.10\n",
      "   18  10430 201N 202S C   N    1616 1618  -75.72  -72.40   66.78    4.04  -64.10  121.27    8.16\n",
      "   18        202S 202S N   CA   1618 1619   85.71  152.40                  -35.00\n",
      "   19  10431 202S 203K C   N    1622 1624  -77.70  -70.20   15.01    0.92  -62.90  166.46   22.45\n",
      "   19        203K 203K N   CA   1624 1625  153.40  140.40                  -40.80\n",
      "   20  10432 203K 204G C   N    1631 1633  -77.63  -80.20    3.25    0.12   82.20 -131.27    7.06\n",
      "   20        204G 204G N   CA   1633 1634  172.11  174.10                    8.50\n",
      "   21  10560 331F 332D C   N    2668 2670   64.26   54.50   26.56    1.33  -63.30  139.39   23.76\n",
      "   21        332D 332D N   CA   2670 2671   16.20   40.90                  -40.00\n",
      "   22  10561 332D 333M C   N    2676 2678 -103.57 -125.60   46.00    2.50  -63.40  144.31   24.75\n",
      "   22        333M 333M N   CA   2678 2679 -179.11  140.50                  -40.50\n",
      "   23  10562 333M 334E C   N    2684 2686  -74.00  -69.30   25.91    1.81  -63.60  152.07   20.85\n",
      "   23        334E 334E N   CA   2686 2687  167.98  142.50                  -40.30\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:   709 2813   2.224\n",
      "  serious non-bonded atom clash:  2019 2082   1.970\n",
      "  serious non-bonded atom clash:  2091 2094   2.046\n",
      "  serious non-bonded atom clash:  2091 2121   1.952\n",
      "  serious non-bonded atom clash:  2133 2138   2.137\n",
      "  serious non-bonded atom clash:  2740 2749   2.249\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     3    1    2    0    6    6   33   52  121  187  221  289  335  398  493\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "4qp2_MODLL.B99990001.pdb       275.43759\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 4qp3\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    37277    34631\n",
      "252 (of 2915 total) atoms selected for optimization\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:       90.0346\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     252\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    34631    2416\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      600\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         193.6507\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     261       0      0   0.004   0.004      1.3454       1.000\n",
      " 2 Bond angle potential               :     360       0      1   1.928   1.928      25.777       1.000\n",
      " 3 Stereochemical cosine torsion poten:     163       0      2  45.851  45.851      53.678       1.000\n",
      " 4 Stereochemical improper torsion pot:     118       0      1   1.513   1.513      6.3326       1.000\n",
      " 5 Soft-sphere overlap restraints     :     600       0      0   0.002   0.002     0.40222       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     449       0      0   0.285   0.285      8.5907       1.000\n",
      "10 Distance restraints 2 (N-O)        :     412       0      1   0.308   0.308      12.551       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      38       0      0   3.628   3.628      5.2011       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      24       0      0  83.581  83.581      6.9425       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      18       0      0  76.837  76.837      6.9670       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       6       0      0  84.641  84.641      4.3868       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       1       0      0 119.841 119.841     0.95999       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     239       0      0   0.667   0.667      2.0965       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      41      17      6  28.486 118.106      35.430       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     286       0      0   0.923   0.923      22.990       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 4qp3_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    5714.0811\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9  -76.15  -68.20   11.69    0.74  -62.50  165.79   27.86\n",
      "    1          2A   2A N   CA      9   10  153.88  145.30                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14  -67.28  -68.20   15.22    1.19  -62.50  171.08   27.81\n",
      "    2          3A   3A N   CA     14   15  130.11  145.30                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19  -90.49  -68.20   32.37    2.04  -62.50  152.91   26.52\n",
      "    3          4A   4A N   CA     19   20  168.78  145.30                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24  -69.17  -68.20    2.45    0.16  -62.50  171.68   28.47\n",
      "    4          5A   5A N   CA     24   25  147.55  145.30                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29  -72.62  -68.20    8.78    0.57  -62.50  166.52   27.80\n",
      "    5          6A   6A N   CA     29   30  152.89  145.30                  -40.90\n",
      "    6  10235   6A   7A C   N      32   34  -66.54  -68.20    1.66    0.14  -62.50  173.83   28.69\n",
      "    6          7A   7A N   CA     34   35  145.32  145.30                  -40.90\n",
      "    7  10236   7A   8G C   N      37   39  -72.77  -80.20   24.64    0.64   82.20 -149.74    6.91\n",
      "    7          8G   8G N   CA     39   40  150.61  174.10                    8.50\n",
      "    8  10237   8G   9A C   N      41   43  -68.09  -68.20    1.57    0.13  -62.50  172.32   28.52\n",
      "    8          9A   9A N   CA     43   44  146.87  145.30                  -40.90\n",
      "    9  10354 125H 126I C   N     992  994  -58.92  -63.40    7.96    1.46 -120.60  178.33    7.99\n",
      "    9        126I 126I N   CA    994  995  -37.02  -43.60                  130.30\n",
      "   10  10409 180H 181T C   N    1438 1440  -58.58  -78.10   30.92    1.06  -63.20  167.99   21.85\n",
      "   10        181T 181T N   CA   1440 1441  125.83  149.80                  -42.10\n",
      "   11  10412 183F 184L C   N    1460 1462 -113.50 -108.50   49.77    2.63  -63.50  145.63   23.32\n",
      "   11        184L 184L N   CA   1462 1463 -177.98  132.50                  -41.20\n",
      "   12  10413 184L 185T C   N    1468 1470 -117.91 -124.80   26.69    1.35  -63.20  158.37   23.93\n",
      "   12        185T 185T N   CA   1470 1471  169.28  143.50                  -42.10\n",
      "   13  10414 185T 186E C   N    1475 1477 -113.24 -117.80    8.94    0.49  -63.60 -177.89   27.09\n",
      "   13        186E 186E N   CA   1477 1478  144.49  136.80                  -40.30\n",
      "   14  10429 200L 201N C   N    1608 1610 -125.59 -119.90   25.86    1.04  -63.20  168.64   24.63\n",
      "   14        201N 201N N   CA   1610 1611  162.23  137.00                  -41.10\n",
      "   15  10430 201N 202S C   N    1616 1618  -98.59  -72.40   71.57    5.09  -64.10  125.63    7.77\n",
      "   15        202S 202S N   CA   1618 1619   85.80  152.40                  -35.00\n",
      "   16  10431 202S 203K C   N    1622 1624 -121.73 -118.00   57.08    2.72  -62.90  136.47   21.24\n",
      "   16        203K 203K N   CA   1624 1625 -163.94  139.10                  -40.80\n",
      "   17  10432 203K 204G C   N    1631 1633  -91.35  -80.20   25.50    0.59   82.20 -116.03   18.73\n",
      "   17        204G 204G N   CA   1633 1634 -162.96  174.10                    8.50\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:   710 2813   2.265\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    1    0    0   14   19   52   73  172  197  299  393  431  469\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "4qp3_MODLL.B99990001.pdb       193.65065\n",
      "\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2753 atoms and 1 coordinate set(s) were parsed in 0.03s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 4qp4\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    37395    34749\n",
      "216 (of 2915 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2777 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     216\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    34749    1424\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      336\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         169.3156\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     224       0      0   0.003   0.003     0.42956       1.000\n",
      " 2 Bond angle potential               :     309       0      0   1.809   1.809      18.175       1.000\n",
      " 3 Stereochemical cosine torsion poten:     149       0      3  50.223  50.223      53.339       1.000\n",
      " 4 Stereochemical improper torsion pot:      95       0      0   0.897   0.897      1.8066       1.000\n",
      " 5 Soft-sphere overlap restraints     :     336       0      0   0.001   0.001     0.44589E-01   1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     193       0      0   0.221   0.221      6.6612       1.000\n",
      "10 Distance restraints 2 (N-O)        :     179       0      1   0.623   0.623      21.656       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      32       1      1  31.097  31.097      6.5265       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      19       0      0  72.172  72.172      3.6843       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      15       0      0  86.088  86.088      6.6552       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       7       0      0  61.157  61.157      3.5423       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       1       0      0 172.561 172.561     0.52864       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     117       0      0   0.548   0.548     0.66679       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      33      19      5  33.078 132.102      37.755       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      51       0      0   1.173   1.173      7.8448       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 4qp4_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    5776.1953\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 13                           : Mainchain Omega dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10943 356P 356P CA  C    2876 2880   -4.74 -179.18  174.44   40.54 -179.18  174.44   40.54\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9  -66.79  -68.20    8.25    0.61  -62.50  178.13   28.98\n",
      "    1          2A   2A N   CA      9   10  137.18  145.30                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14  -82.09  -68.20   22.14    1.39  -62.50  157.78   26.86\n",
      "    2          3A   3A N   CA     14   15  162.54  145.30                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19 -116.80 -134.00   17.67    0.45  -62.50 -175.69   33.09\n",
      "    3          4A   4A N   CA     19   20  142.97  147.00                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24  -72.96  -68.20    7.46    0.47  -62.50  168.38   28.12\n",
      "    4          5A   5A N   CA     24   25  151.04  145.30                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29  -67.64  -68.20    0.56    0.05  -62.50  173.94   28.76\n",
      "    5          6A   6A N   CA     29   30  145.24  145.30                  -40.90\n",
      "    6  10235   6A   7A C   N      32   34  -78.85  -68.20   15.06    0.95  -62.50  163.98   27.70\n",
      "    6          7A   7A N   CA     34   35  155.94  145.30                  -40.90\n",
      "    7  10237   8G   9A C   N      41   43  -75.90  -68.20   16.77    1.10  -62.50  159.47   26.81\n",
      "    7          9A   9A N   CA     43   44  160.20  145.30                  -40.90\n",
      "    8  10238   9A  10G C   N      46   48  -79.86  -80.20   17.69    0.71   82.20 -120.22   18.23\n",
      "    8         10G  10G N   CA     48   49 -168.22  174.10                    8.50\n",
      "    9  10240  11P  12E C   N      57   59 -126.26 -117.80   17.10    0.65  -63.60  179.34   27.45\n",
      "    9         12E  12E N   CA     59   60  151.66  136.80                  -40.30\n",
      "   10  10241  12E  13M C   N      66   68  -77.54  -73.00   26.17    1.70  -63.40  151.39   23.72\n",
      "   10         13M  13M N   CA     68   69  168.77  143.00                  -40.50\n",
      "   11  10405 176P 177D C   N    1402 1404 -148.52  -96.50  102.74    4.25  -63.30  107.55   12.85\n",
      "   11        177D 177D N   CA   1404 1405   25.61  114.20                  -40.00\n",
      "   12  10406 177D 178H C   N    1410 1412 -142.17 -125.60   50.51    1.58  -63.20  153.12   24.22\n",
      "   12        178H 178H N   CA   1412 1413 -173.48  138.80                  -42.30\n",
      "   13  10408 179D 180H C   N    1428 1430  -67.08  -67.60    3.47    0.25  -63.20  178.92   22.13\n",
      "   13        180H 180H N   CA   1430 1431  136.57  140.00                  -42.30\n",
      "   14  10409 180H 181T C   N    1438 1440 -154.16 -124.80   51.16    1.58  -63.20  160.72   26.32\n",
      "   14        181T 181T N   CA   1440 1441 -174.61  143.50                  -42.10\n",
      "   15  10410 181T 182G C   N    1445 1447 -141.50 -167.20   57.10    1.75   82.20  178.40   13.85\n",
      "   15        182G 182G N   CA   1447 1448  123.60  174.60                    8.50\n",
      "   16  10411 182G 183F C   N    1449 1451 -132.55 -124.20   22.34    0.81  -63.20  166.78   27.66\n",
      "   16        183F 183F N   CA   1451 1452  164.02  143.30                  -44.30\n",
      "   17  10585 356P 357G C   N    2880 2882   87.16   78.70   49.45    1.07  -80.20  169.84   10.32\n",
      "   17        357G 357G N   CA   2882 2883  145.18 -166.10                  174.10\n",
      "   18  10586 357G 358Y C   N    2884 2886 -122.76 -124.30    5.35    0.23  -63.50 -176.49   26.39\n",
      "   18        358Y 358Y N   CA   2886 2887  130.28  135.40                  -43.40\n",
      "   19  10587 358Y 359R C   N    2896 2898  -79.26  -72.10    7.56    0.63  -63.00 -179.85   25.45\n",
      "   19        359R 359R N   CA   2898 2899  139.48  141.90                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    2    3    5   21   29   98  170  214  275  348  414  459\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "4qp4_MODLL.B99990001.pdb       169.31561\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 4qp6\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    37241    34595\n",
      "252 (of 2915 total) atoms selected for optimization\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     252\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    34595    2419\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      560\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         184.3659\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     261       0      0   0.004   0.004     0.99814       1.000\n",
      " 2 Bond angle potential               :     360       0      0   1.672   1.672      19.411       1.000\n",
      " 3 Stereochemical cosine torsion poten:     163       0      9  52.290  52.290      68.693       1.000\n",
      " 4 Stereochemical improper torsion pot:     118       0      0   1.113   1.113      3.5332       1.000\n",
      " 5 Soft-sphere overlap restraints     :     560       0      0   0.003   0.003     0.54695       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     455       0      0   0.134   0.134      4.8086       1.000\n",
      "10 Distance restraints 2 (N-O)        :     417       0      0   0.275   0.275      9.4513       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      38       2      1  39.769  39.769      4.8609       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      24       0      0  67.798  67.798      5.4750       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      18       0      0  62.170  62.170      6.4369       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       6       0      0  88.809  88.809      4.3339       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       1       0      0   2.096   2.096     0.23657       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     262       0      0   0.423   0.423      1.1587       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      41      21      6  30.621 122.666      45.929       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     255       0      0   0.548   0.548      8.4927       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 4qp6_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    5894.5400\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9  -59.75  -68.20    8.79    0.66  -62.50  176.23   28.75\n",
      "    1          2A   2A N   CA      9   10  142.90  145.30                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14  -72.45  -68.20    5.85    0.58  -62.50  178.10   29.68\n",
      "    2          3A   3A N   CA     14   15  141.29  145.30                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19 -170.52 -134.00   38.43    0.91  -62.50 -166.85   37.37\n",
      "    3          4A   4A N   CA     19   20  158.98  147.00                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24  -73.86  -68.20    5.66    0.48  -62.50  174.19   29.12\n",
      "    4          5A   5A N   CA     24   25  145.28  145.30                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29  -71.40  -68.20   12.40    1.11  -62.50  174.45   28.17\n",
      "    5          6A   6A N   CA     29   30  133.32  145.30                  -40.90\n",
      "    6  10235   6A   7A C   N      32   34  -82.87  -68.20   19.81    1.26  -62.50  161.78   27.56\n",
      "    6          7A   7A N   CA     34   35  158.61  145.30                  -40.90\n",
      "    7  10236   7A   8G C   N      37   39  -70.72  -80.20   30.84    0.79   82.20 -155.19    6.85\n",
      "    7          8G   8G N   CA     39   40  144.75  174.10                    8.50\n",
      "    8  10237   8G   9A C   N      41   43  -72.50  -68.20    5.01    0.34  -62.50  171.52   28.61\n",
      "    8          9A   9A N   CA     43   44  147.87  145.30                  -40.90\n",
      "    9  10238   9A  10G C   N      46   48   88.64   78.70   33.06    0.57   82.20  154.00    7.80\n",
      "    9         10G  10G N   CA     48   49  162.37 -166.10                    8.50\n",
      "   10  10240  11P  12E C   N      57   59 -140.73  -63.60   83.80   11.50  -63.60   83.80   11.50\n",
      "   10         12E  12E N   CA     59   60   -7.55  -40.30                  -40.30\n",
      "   11  10354 125H 126I C   N     992  994  -59.71  -63.40    5.38    1.01 -120.60 -179.44    8.14\n",
      "   11        126I 126I N   CA    994  995  -39.68  -43.60                  130.30\n",
      "   12  10408 179D 180H C   N    1428 1430   61.07   56.30    5.17    0.85  -63.20  150.61   25.81\n",
      "   12        180H 180H N   CA   1430 1431   42.80   40.80                  -42.30\n",
      "   13  10409 180H 181T C   N    1438 1440 -102.92  -78.10   28.64    1.28  -63.20  158.87   23.02\n",
      "   13        181T 181T N   CA   1440 1441  164.08  149.80                  -42.10\n",
      "   14  10410 181T 182G C   N    1445 1447   82.67   78.70   22.37    0.48   82.20  163.39    7.96\n",
      "   14        182G 182G N   CA   1447 1448  171.89 -166.10                    8.50\n",
      "   15  10411 182G 183F C   N    1449 1451  -93.58  -71.40   44.64    3.97  -63.20  149.38   18.80\n",
      "   15        183F 183F N   CA   1451 1452  101.96  140.70                  -44.30\n",
      "   16  10412 183F 184L C   N    1460 1462 -126.28 -108.50   25.24    1.14  -63.50  179.70   28.84\n",
      "   16        184L 184L N   CA   1462 1463  150.42  132.50                  -41.20\n",
      "   17  10413 184L 185T C   N    1468 1470  -97.21  -78.10   25.70    1.67  -63.20  178.00   20.75\n",
      "   17        185T 185T N   CA   1470 1471  132.62  149.80                  -42.10\n",
      "   18  10414 185T 186E C   N    1475 1477 -147.60 -117.80   40.52    1.27  -63.60  176.69   28.22\n",
      "   18        186E 186E N   CA   1477 1478  164.25  136.80                  -40.30\n",
      "   19  10430 201N 202S C   N    1616 1618  -91.26  -72.40   56.71    3.98  -64.10  136.65    8.66\n",
      "   19        202S 202S N   CA   1618 1619   98.92  152.40                  -35.00\n",
      "   20  10431 202S 203K C   N    1622 1624 -110.80 -118.00   41.96    2.12  -62.90  146.80   21.92\n",
      "   20        203K 203K N   CA   1624 1625 -179.56  139.10                  -40.80\n",
      "   21  10432 203K 204G C   N    1631 1633  -65.37  -80.20   16.07    0.82   82.20 -142.76    6.53\n",
      "   21        204G 204G N   CA   1633 1634  167.93  174.10                    8.50\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2753 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    1    4    5   21   48   62  164  201  279  381  448  457\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "4qp6_MODLL.B99990001.pdb       184.36586\n",
      "\n",
      "Modelando proteína 4qp7\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    37409    34763\n",
      "216 (of 2915 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2739 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     216\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    34763    1429\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      333\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         165.0084\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     224       0      0   0.003   0.003     0.46560       1.000\n",
      " 2 Bond angle potential               :     309       0      1   1.842   1.842      19.583       1.000\n",
      " 3 Stereochemical cosine torsion poten:     149       0      4  48.863  48.863      52.786       1.000\n",
      " 4 Stereochemical improper torsion pot:      95       0      0   0.867   0.867      1.7403       1.000\n",
      " 5 Soft-sphere overlap restraints     :     333       0      0   0.001   0.001     0.54150E-01   1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     192       0      0   0.261   0.261      9.4733       1.000\n",
      "10 Distance restraints 2 (N-O)        :     174       0      1   0.487   0.487      15.118       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      32       1      1  31.167  31.167      5.6770       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      19       0      0  82.120  82.120      5.0770       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      15       0      0  31.071  31.071      4.1427       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       7       0      0  68.118  68.118      2.9447       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       1       0      0  41.059  41.059      1.2138       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     120       0      0   0.491   0.491     0.59676       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      33      18      4  28.440 132.393      35.308       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      59       0      2   1.269   1.269      10.828       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 4qp7_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    5201.8926\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9  -71.61  -68.20    9.29    0.63  -62.50  165.41   27.56\n",
      "    1          2A   2A N   CA      9   10  153.94  145.30                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14 -120.89 -134.00   14.13    0.32  -62.50 -173.26   33.72\n",
      "    2          3A   3A N   CA     14   15  141.73  147.00                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19  -74.93  -68.20    6.95    0.53  -62.50  172.50   28.90\n",
      "    3          4A   4A N   CA     19   20  147.05  145.30                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24  -80.27  -68.20   20.16    1.27  -62.50  158.66   26.91\n",
      "    4          5A   5A N   CA     24   25  161.45  145.30                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29 -118.16 -134.00   16.08    0.64  -62.50  178.26   32.18\n",
      "    5          6A   6A N   CA     29   30  149.75  147.00                  -40.90\n",
      "    6  10235   6A   7A C   N      32   34  -69.64  -68.20    5.93    0.43  -62.50  168.20   27.92\n",
      "    6          7A   7A N   CA     34   35  151.06  145.30                  -40.90\n",
      "    7  10236   7A   8G C   N      37   39  -76.46  -80.20   15.28    0.43   82.20 -141.12    7.05\n",
      "    7          8G   8G N   CA     39   40  159.29  174.10                    8.50\n",
      "    8  10237   8G   9A C   N      41   43 -136.56 -134.00    7.74    0.36  -62.50 -179.32   33.58\n",
      "    8          9A   9A N   CA     43   44  154.30  147.00                  -40.90\n",
      "    9  10238   9A  10G C   N      46   48  -80.18  -80.20    7.53    0.30   82.20 -122.64    7.18\n",
      "    9         10G  10G N   CA     48   49 -178.37  174.10                    8.50\n",
      "   10  10240  11P  12E C   N      57   59 -155.59 -117.80   47.88    1.45  -63.60  178.96   28.92\n",
      "   10         12E  12E N   CA     59   60  166.20  136.80                  -40.30\n",
      "   11  10241  12E  13M C   N      66   68  -72.70  -73.00   29.54    2.00  -63.40  147.25   22.72\n",
      "   11         13M  13M N   CA     68   69  172.54  143.00                  -40.50\n",
      "   12  10406 177D 178H C   N    1410 1412 -135.35 -125.60    9.90    0.38  -63.20 -166.60   20.82\n",
      "   12        178H 178H N   CA   1412 1413  137.14  138.80                  -42.30\n",
      "   13  10407 178H 179D C   N    1420 1422 -119.23  -96.50   33.14    1.39  -63.30 -173.12   20.36\n",
      "   13        179D 179D N   CA   1422 1423  138.31  114.20                  -40.00\n",
      "   14  10409 180H 181T C   N    1438 1440  -89.34  -63.20   82.71   12.34  -63.20   82.71   12.34\n",
      "   14        181T 181T N   CA   1440 1441 -120.57  -42.10                  -42.10\n",
      "   15  10411 182G 183F C   N    1449 1451  -78.13  -71.40   38.60    3.27  -63.20  147.75   19.53\n",
      "   15        183F 183F N   CA   1451 1452  102.70  140.70                  -44.30\n",
      "   16  10585 356P 357G C   N    2880 2882   90.56   78.70   17.42    0.37  -80.20  170.91   11.60\n",
      "   16        357G 357G N   CA   2882 2883 -178.86 -166.10                  174.10\n",
      "   17  10586 357G 358Y C   N    2884 2886 -119.07 -124.30    7.41    0.60  -63.50 -175.49   32.09\n",
      "   17        358Y 358Y N   CA   2886 2887  140.66  135.40                  -43.40\n",
      "   18  10587 358Y 359R C   N    2896 2898  -62.14  -72.10   11.01    0.74  -63.00  178.31   24.15\n",
      "   18        359R 359R N   CA   2898 2899  137.21  141.90                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    1    4    6   18   37   85  162  227  281  332  453  456\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "4qp7_MODLL.B99990001.pdb       165.00841\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 4qp8\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    37237    34591\n",
      "237 (of 2915 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2737 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     237\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    34591    1546\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      599\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         761.5192\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     246       3     13   0.048   0.048      140.01       1.000\n",
      " 2 Bond angle potential               :     339       8     12   4.827   4.827      156.92       1.000\n",
      " 3 Stereochemical cosine torsion poten:     167       0      5  49.657  49.657      61.055       1.000\n",
      " 4 Stereochemical improper torsion pot:     108       2      2   3.635   3.635      25.638       1.000\n",
      " 5 Soft-sphere overlap restraints     :     599      10     18   0.059   0.059      244.53       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     196       0      4   0.382   0.382      22.117       1.000\n",
      "10 Distance restraints 2 (N-O)        :     181       0      1   0.479   0.479      17.406       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      34       0      1   5.775   5.775      13.000       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      21       0      1  81.814  81.814      10.261       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      17       0      0  79.287  79.287      6.3823       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       8       0      0  75.626  75.626      5.1202       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       1       0      0 176.524 176.524     0.60951       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     150       0      0   0.576   0.576      1.3946       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      35      23      3  41.432 143.342      50.534       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      43       0      0   1.314   1.314      6.5479       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 4qp8_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    7279.8193\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  1                           : Bond length potential                   \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   1469 181T 181T CB  CA   1442 1441    2.04    1.50    0.54   14.94    1.50    0.54   14.94\n",
      "    2   1474 181T 181T C   CA   1445 1441    1.74    1.49    0.25    7.17    1.49    0.25    7.17\n",
      "    3   1490 183F 183F C   CA   1460 1452    1.67    1.49    0.18    5.16    1.49    0.18    5.16\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   4975 181T 181T CA  CB   1441 1442  131.24  110.10   21.14    5.91  110.10   21.14    5.91\n",
      "    2   4976 181T 181T CA  CB   1441 1442  136.43  108.50   27.93    6.55  108.50   27.93    6.55\n",
      "    3   4980 181T 181T C   CA   1445 1441  134.19  108.00   26.19    6.07  108.00   26.19    6.07\n",
      "    4   4989 183F 183F N   CA   1451 1452  136.86  107.00   29.86    8.59  107.00   29.86    8.59\n",
      "    5   5002 183F 184L C   N    1460 1462  147.62  120.00   27.62    6.28  120.00   27.62    6.28\n",
      "    6   5005 184L 184L N   CA   1462 1463  126.02  107.00   19.02    5.47  107.00   19.02    5.47\n",
      "    7   5023 185T 186E C   N    1475 1477  140.49  120.00   20.49    4.66  120.00   20.49    4.66\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9  -75.30  -68.20   16.07    1.06  -62.50  159.90   26.85\n",
      "    1          2A   2A N   CA      9   10  159.72  145.30                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14 -106.70 -134.00   28.37    1.24  -62.50  170.21   30.23\n",
      "    2          3A   3A N   CA     14   15  154.73  147.00                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19  -85.78  -68.20   28.33    1.79  -62.50  153.36   26.34\n",
      "    3          4A   4A N   CA     19   20  167.52  145.30                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24  -78.69  -68.20   10.50    0.90  -62.50  175.00   29.50\n",
      "    4          5A   5A N   CA     24   25  144.85  145.30                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29 -130.17 -134.00    7.19    0.44  -62.50  179.28   33.00\n",
      "    5          6A   6A N   CA     29   30  153.08  147.00                  -40.90\n",
      "    6  10235   6A   7A C   N      32   34  -80.96  -68.20   26.97    1.76  -62.50  151.17   25.72\n",
      "    6          7A   7A N   CA     34   35  169.06  145.30                  -40.90\n",
      "    7  10236   7A   8G C   N      37   39  -76.78  -80.20    3.43    0.24   82.20 -130.24    7.03\n",
      "    7          8G   8G N   CA     39   40  174.38  174.10                    8.50\n",
      "    8  10237   8G   9A C   N      41   43  -70.67  -68.20    6.36    0.43  -62.50  168.14   27.96\n",
      "    8          9A   9A N   CA     43   44  151.16  145.30                  -40.90\n",
      "    9  10238   9A  10G C   N      46   48 -127.21 -167.20   40.97    1.01   82.20 -129.16   17.42\n",
      "    9         10G  10G N   CA     48   49 -176.54  174.60                    8.50\n",
      "   10  10239  10G  11P C   N      50   52  -65.81  -58.70   10.83    0.71  -64.50  169.52   12.72\n",
      "   10         11P  11P N   CA     52   53  -22.32  -30.50                  147.20\n",
      "   11  10403 174A 175D C   N    1387 1389 -117.50  -96.50   27.16    1.13  -63.30  179.79   19.58\n",
      "   11        175D 175D N   CA   1389 1390  131.42  114.20                  -40.00\n",
      "   12  10404 175D 176P C   N    1395 1397  -66.53  -58.70   19.31    1.14  -64.50  160.05   12.05\n",
      "   12        176P 176P N   CA   1397 1398  -12.84  -30.50                  147.20\n",
      "   13  10405 176P 177D C   N    1402 1404  -51.98  -70.90   44.00    1.48  -63.30  151.00   19.34\n",
      "   13        177D 177D N   CA   1404 1405  110.58  150.30                  -40.00\n",
      "   14  10406 177D 178H C   N    1410 1412 -115.90 -125.60   13.80    0.63  -63.20  177.11   25.59\n",
      "   14        178H 178H N   CA   1412 1413  148.61  138.80                  -42.30\n",
      "   15  10407 178H 179D C   N    1420 1422 -120.55  -96.50   24.35    1.00  -63.30  168.08   18.12\n",
      "   15        179D 179D N   CA   1422 1423  118.03  114.20                  -40.00\n",
      "   16  10409 180H 181T C   N    1438 1440  -81.34  -78.10    4.36    0.17  -63.20  166.18   22.52\n",
      "   16        181T 181T N   CA   1440 1441  152.72  149.80                  -42.10\n",
      "   17  10410 181T 182G C   N    1445 1447 -110.52  -80.20   34.17    2.52   82.20 -135.43   17.37\n",
      "   17        182G 182G N   CA   1447 1448  158.33  174.10                    8.50\n",
      "   18  10411 182G 183F C   N    1449 1451  133.25 -124.20  165.74    9.19  -63.20  173.33   24.18\n",
      "   18        183F 183F N   CA   1451 1452   13.09  143.30                  -44.30\n",
      "   19  10412 183F 184L C   N    1460 1462 -103.75  -63.50   94.87   15.66  -63.50   94.87   15.66\n",
      "   19        184L 184L N   CA   1462 1463 -127.11  -41.20                  -41.20\n",
      "   20  10413 184L 185T C   N    1468 1470 -141.13 -124.80   19.02    0.72  -63.20 -177.82   28.45\n",
      "   20        185T 185T N   CA   1470 1471  153.24  143.50                  -42.10\n",
      "   21  10414 185T 186E C   N    1475 1477  161.29 -117.80  100.64    3.01  -63.60 -177.25   31.14\n",
      "   21        186E 186E N   CA   1477 1478 -163.36  136.80                  -40.30\n",
      "   22  10586 357G 358Y C   N    2884 2886 -122.82 -124.30    1.73    0.14  -63.50 -170.77   27.28\n",
      "   22        358Y 358Y N   CA   2886 2887  136.29  135.40                  -43.40\n",
      "   23  10587 358Y 359R C   N    2896 2898 -119.36 -125.20   19.63    1.05  -63.00  169.22   26.75\n",
      "   23        359R 359R N   CA   2898 2899  159.34  140.60                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:  1441 2022   2.190\n",
      "  serious non-bonded atom clash:  1442 2032   2.273\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    1    1    3    7   15   30   66  104  162  208  284  390  427  495\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "4qp8_MODLL.B99990001.pdb       761.51923\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 4qp9\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    37263    34617\n",
      "269 (of 2915 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2695 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     269\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    34617    2028\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      398\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         197.5414\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     279       0      0   0.004   0.004      1.1377       1.000\n",
      " 2 Bond angle potential               :     383       0      0   1.769   1.769      22.123       1.000\n",
      " 3 Stereochemical cosine torsion poten:     180       0      6  50.512  50.512      68.301       1.000\n",
      " 4 Stereochemical improper torsion pot:     117       0      0   1.012   1.012      2.5849       1.000\n",
      " 5 Soft-sphere overlap restraints     :     398       0      0   0.002   0.002     0.24035       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     309       0      0   0.211   0.211      9.1971       1.000\n",
      "10 Distance restraints 2 (N-O)        :     261       0      0   0.449   0.449      20.953       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      39       1      0  27.455  27.455      7.5527       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      25       0      0  84.774  84.774      6.1168       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      21       0      0  63.860  63.860      9.3287       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:      10       0      0  85.889  85.889      5.7348       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       2       0      0 100.960 100.960      1.4510       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     218       0      0   0.704   0.704      2.6959       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      41      21      3  28.099 118.184      28.193       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     143       0      1   1.125   1.125      11.931       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 4qp9_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    6768.8516\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 13                           : Mainchain Omega dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10771 184L 184L CA  C    1463 1468   -9.44 -179.18  169.74   39.45 -179.18  169.74   39.45\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9  -69.26  -68.20    5.76    0.43  -62.50  168.27   27.91\n",
      "    1          2A   2A N   CA      9   10  150.97  145.30                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14  -88.50  -68.20   21.63    1.57  -62.50  168.36   28.94\n",
      "    2          3A   3A N   CA     14   15  152.76  145.30                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19 -114.65 -134.00   20.08    0.49  -62.50 -175.00   33.09\n",
      "    3          4A   4A N   CA     19   20  141.61  147.00                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24 -168.51 -134.00   47.42    1.38  -62.50  175.28   34.28\n",
      "    4          5A   5A N   CA     24   25  179.52  147.00                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29 -120.86 -134.00   13.37    0.36  -62.50 -175.92   33.28\n",
      "    5          6A   6A N   CA     29   30  144.52  147.00                  -40.90\n",
      "    6  10235   6A   7A C   N      32   34  -88.99  -68.20   31.70    1.99  -62.50  152.19   26.32\n",
      "    6          7A   7A N   CA     34   35  169.23  145.30                  -40.90\n",
      "    7  10236   7A   8G C   N      37   39  169.62 -167.20   27.44    0.91   82.20  174.84   12.38\n",
      "    7          8G   8G N   CA     39   40  159.92  174.60                    8.50\n",
      "    8  10237   8G   9A C   N      41   43 -113.81 -134.00   20.81    0.52  -62.50 -175.55   32.95\n",
      "    8          9A   9A N   CA     43   44  141.93  147.00                  -40.90\n",
      "    9  10238   9A  10G C   N      46   48  113.46   78.70   74.74    1.11   82.20  123.26    7.50\n",
      "    9         10G  10G N   CA     48   49  127.73 -166.10                    8.50\n",
      "   10  10240  11P  12E C   N      57   59  -96.54 -117.80   34.18    1.86  -63.60  159.56   23.13\n",
      "   10         12E  12E N   CA     59   60  163.57  136.80                  -40.30\n",
      "   11  10241  12E  13M C   N      66   68 -121.73 -125.60   25.08    1.27  -63.40  164.88   29.25\n",
      "   11         13M  13M N   CA     68   69  165.29  140.50                  -40.50\n",
      "   12  10405 176P 177D C   N    1402 1404 -144.74  -96.50   55.98    2.28  -63.30  149.85   15.98\n",
      "   12        177D 177D N   CA   1404 1405   85.79  114.20                  -40.00\n",
      "   13  10406 177D 178H C   N    1410 1412 -137.69 -125.60   32.08    1.40  -63.20  168.73   17.97\n",
      "   13        178H 178H N   CA   1412 1413  109.09  138.80                  -42.30\n",
      "   14  10407 178H 179D C   N    1420 1422  -92.09  -70.90   22.88    1.36  -63.30  163.65   22.03\n",
      "   14        179D 179D N   CA   1422 1423  158.91  150.30                  -40.00\n",
      "   15  10409 180H 181T C   N    1438 1440 -140.32 -124.80   45.04    1.51  -63.20  152.98   24.56\n",
      "   15        181T 181T N   CA   1440 1441 -174.22  143.50                  -42.10\n",
      "   16  10410 181T 182G C   N    1445 1447  -63.03  -62.40    2.66    0.37   82.20  152.68   11.57\n",
      "   16        182G 182G N   CA   1447 1448  -38.62  -41.20                    8.50\n",
      "   17  10411 182G 183F C   N    1449 1451  -74.75  -71.40   24.27    1.86  -63.20  151.40   21.80\n",
      "   17        183F 183F N   CA   1451 1452  164.74  140.70                  -44.30\n",
      "   18  10561 332D 333M C   N    2676 2678 -132.14 -125.60   34.37    1.50  -63.40  160.70   29.40\n",
      "   18        333M 333M N   CA   2678 2679  174.25  140.50                  -40.50\n",
      "   19  10585 356P 357G C   N    2880 2882  172.60 -167.20   20.20    0.39  -80.20  107.20    7.14\n",
      "   19        357G 357G N   CA   2882 2883  174.35  174.60                  174.10\n",
      "   20  10586 357G 358Y C   N    2884 2886  -93.24  -98.40    5.72    0.35  -63.50  171.92   25.81\n",
      "   20        358Y 358Y N   CA   2886 2887  125.93  128.40                  -43.40\n",
      "   21  10587 358Y 359R C   N    2896 2898  -69.36  -72.10    4.91    0.32  -63.00  179.04   23.77\n",
      "   21        359R 359R N   CA   2898 2899  137.83  141.90                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    2    2    5   16   56  108  172  184  278  355  399  478\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "4qp9_MODLL.B99990001.pdb       197.54141\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 4qpa\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    36399    33753\n",
      "324 (of 2915 total) atoms selected for optimization\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:      -90.6062\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     324\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    33753    2465\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      600\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         267.7403\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     334       0      0   0.005   0.005      2.0981       1.000\n",
      " 2 Bond angle potential               :     460       0      1   2.030   2.030      36.831       1.000\n",
      " 3 Stereochemical cosine torsion poten:     208       0      9  49.806  49.806      81.682       1.000\n",
      " 4 Stereochemical improper torsion pot:     138       0      0   1.163   1.163      4.0913       1.000\n",
      " 5 Soft-sphere overlap restraints     :     600       0      0   0.005   0.005      1.4641       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     419       0      1   0.306   0.306      17.725       1.000\n",
      "10 Distance restraints 2 (N-O)        :     365       0      0   0.315   0.315      14.954       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      48       0      1   4.635   4.635      10.844       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      33       0      0  76.272  76.272      5.4347       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      25       0      0  68.475  68.475      9.1494       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:      10       0      0 109.872 109.872      6.4780       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       2       0      0 147.751 147.751      1.2593       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     226       0      0   0.331   0.331     0.66800       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      51      27      9  36.974 129.556      69.468       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     146       0      0   0.606   0.606      5.5937       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 4qpa_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    7646.0732\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9  -67.87  -68.20    2.49    0.19  -62.50  176.35   29.17\n",
      "    1          2A   2A N   CA      9   10  142.83  145.30                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14  -73.44  -68.20   11.25    0.74  -62.50  164.21   27.46\n",
      "    2          3A   3A N   CA     14   15  155.26  145.30                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19  -65.81  -68.20    2.43    0.19  -62.50  174.26   28.72\n",
      "    3          4A   4A N   CA     19   20  144.87  145.30                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24 -103.79 -134.00   35.88    0.86  -62.50  173.53   26.73\n",
      "    4          5A   5A N   CA     24   25  127.65  147.00                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29  -77.78  -68.20   20.53    1.34  -62.50  156.39   26.41\n",
      "    5          6A   6A N   CA     29   30  163.46  145.30                  -40.90\n",
      "    6  10235   6A   7A C   N      32   34  -62.29  -68.20    6.46    0.62  -62.50  171.18   28.04\n",
      "    6          7A   7A N   CA     34   35  147.92  145.30                  -40.90\n",
      "    7  10237   8G   9A C   N      41   43  -68.64  -68.20   12.40    0.98  -62.50  161.52   26.78\n",
      "    7          9A   9A N   CA     43   44  157.70  145.30                  -40.90\n",
      "    8  10238   9A  10G C   N      46   48 -103.10  -80.20   38.06    2.55   82.20 -139.10   17.21\n",
      "    8         10G  10G N   CA     48   49  143.70  174.10                    8.50\n",
      "    9  10239  10G  11P C   N      50   52  -69.05  -58.70   18.81    1.13  -64.50  162.06   12.33\n",
      "    9         11P  11P N   CA     52   53  -14.79  -30.50                  147.20\n",
      "   10  10240  11P  12E C   N      57   59  -69.88  -69.30    2.95    0.20  -63.60  174.42   23.60\n",
      "   10         12E  12E N   CA     59   60  145.39  142.50                  -40.30\n",
      "   11  10241  12E  13M C   N      66   68 -128.52 -125.60   26.42    1.20  -63.40  166.04   29.94\n",
      "   11         13M  13M N   CA     68   69  166.76  140.50                  -40.50\n",
      "   12  10245  16G  17Q C   N      96   98 -105.32  -63.80   60.09   10.79  -63.80   60.09   10.79\n",
      "   12         17Q  17Q N   CA     98   99  -83.73  -40.30                  -40.30\n",
      "   13  10354 125H 126I C   N     992  994  -57.97  -63.40    7.64    1.43 -120.60  179.79    8.05\n",
      "   13        126I 126I N   CA    994  995  -38.22  -43.60                  130.30\n",
      "   14  10408 179D 180H C   N    1428 1430   48.75  -67.60  125.33   10.68  -63.20  172.41   18.89\n",
      "   14        180H 180H N   CA   1430 1431 -173.42  140.00                  -42.30\n",
      "   15  10409 180H 181T C   N    1438 1440 -149.25 -124.80   83.48    4.29  -63.20  136.36   14.57\n",
      "   15        181T 181T N   CA   1440 1441   63.68  143.50                  -42.10\n",
      "   16  10411 182G 183F C   N    1449 1451 -162.31 -124.20  101.91    3.71  -63.20  126.05   22.73\n",
      "   16        183F 183F N   CA   1451 1452 -122.19  143.30                  -44.30\n",
      "   17  10475 246S 247P C   N    1966 1968  -55.31  -64.50   23.16    1.36  -58.70  156.48   13.21\n",
      "   17        247P 247P N   CA   1968 1969  125.94  147.20                  -30.50\n",
      "   18  10476 247P 248S C   N    1973 1975 -142.74 -136.60   10.03    0.57  -64.10 -165.16   12.17\n",
      "   18        248S 248S N   CA   1975 1976  143.26  151.20                  -35.00\n",
      "   19  10477 248S 249Q C   N    1979 1981 -144.32 -121.10   23.70    0.80  -63.80 -167.14   32.57\n",
      "   19        249Q 249Q N   CA   1981 1982  144.46  139.70                  -40.30\n",
      "   20  10478 249Q 250E C   N    1988 1990  -86.47  -69.30   17.65    1.47  -63.60 -179.81   22.84\n",
      "   20        250E 250E N   CA   1990 1991  138.43  142.50                  -40.30\n",
      "   21  10479 250E 251D C   N    1997 1999   66.73   54.50   38.77    2.02  -63.30  137.30   23.25\n",
      "   21        251D 251D N   CA   1999 2000    4.10   40.90                  -40.00\n",
      "   22  10480 251D 252L C   N    2005 2007 -130.48 -108.50   35.37    1.63  -63.50  172.15   28.05\n",
      "   22        252L 252L N   CA   2007 2008  160.21  132.50                  -41.20\n",
      "   23  10481 252L 253N C   N    2013 2015 -113.27 -119.90   12.24    0.66  -63.20  178.76   25.20\n",
      "   23        253N 253N N   CA   2015 2016  147.30  137.00                  -41.10\n",
      "   24  10584 355Q 356P C   N    2873 2875  -70.34  -64.50   21.12    1.89  -58.70  157.83   12.12\n",
      "   24        356P 356P N   CA   2875 2876  126.90  147.20                  -30.50\n",
      "   25  10585 356P 357G C   N    2880 2882   95.98   78.70   44.72    0.68  -80.20  177.48   11.12\n",
      "   25        357G 357G N   CA   2882 2883  152.66 -166.10                  174.10\n",
      "   26  10586 357G 358Y C   N    2884 2886 -121.80 -124.30    2.79    0.16  -63.50 -173.11   26.96\n",
      "   26        358Y 358Y N   CA   2886 2887  134.16  135.40                  -43.40\n",
      "   27  10587 358Y 359R C   N    2896 2898 -107.79 -125.20   17.89    0.62  -63.00 -176.81   22.08\n",
      "   27        359R 359R N   CA   2898 2899  136.53  140.60                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:   129  133   2.230\n",
      "  serious non-bonded atom clash:   158  309   2.183\n",
      "  serious non-bonded atom clash:   412  779   2.256\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    1    2    2    6   20   19   49   94  152  237  277  384  413  462\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "4qpa_MODLL.B99990001.pdb       267.74030\n",
      "\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2715 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 4qta\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    37170    34524\n",
      "236 (of 2915 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2846 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     236\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    34524    1452\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      659\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :        4497.5562\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     244      18     20   0.129   0.129      1167.1       1.000\n",
      " 2 Bond angle potential               :     336      33     55  10.693  10.693      747.09       1.000\n",
      " 3 Stereochemical cosine torsion poten:     163       0     11  56.632  56.632      78.509       1.000\n",
      " 4 Stereochemical improper torsion pot:     109       6      9   6.531   6.531      90.182       1.000\n",
      " 5 Soft-sphere overlap restraints     :     659      31     58   0.120   0.120      1112.7       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     176      20     39   2.096   2.096      634.35       1.000\n",
      "10 Distance restraints 2 (N-O)        :     172      13     30   1.494   1.494      267.65       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      34       5      7  17.489  17.489      122.64       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      20       0      8  96.951  96.951      58.797       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      15       0      4  63.033  63.033      17.203       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       6       0      0  99.412  99.412      3.9421       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       2       0      0 125.733 125.733      1.5788       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      93       0      0   0.754   0.754      1.3330       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      35      21     13  63.816 130.603      154.14       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      47       3      5   3.448   3.448      40.378       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 4qta_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    9884.6328\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  1                           : Bond length potential                   \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   1507 186E 186E CB  CA   1479 1478    1.71    1.54    0.17    4.62    1.54    0.17    4.62\n",
      "    2   1514 186E 186E C   CA   1484 1478    1.72    1.49    0.23    6.66    1.49    0.23    6.66\n",
      "    3   1515 186E 187Y C   N    1484 1486    1.59    1.35    0.24    8.62    1.35    0.24    8.62\n",
      "    4   1516 187Y 187Y CB  CA   1488 1487    1.76    1.54    0.22    6.05    1.54    0.22    6.05\n",
      "    5   1517 187Y 187Y CG  CB   1489 1488    1.73    1.49    0.24    6.57    1.49    0.24    6.57\n",
      "    6   1525 187Y 187Y N   CA   1486 1487    1.68    1.43    0.25    8.28    1.43    0.25    8.28\n",
      "    7   1527 187Y 187Y C   CA   1496 1487    2.38    1.49    0.89   25.76    1.49    0.89   25.76\n",
      "    8   1528 187Y 188V C   N    1496 1498    1.80    1.35    0.46   16.25    1.35    0.46   16.25\n",
      "    9   1532 188V 188V N   CA   1498 1499    1.96    1.43    0.53   17.57    1.43    0.53   17.57\n",
      "   10   1534 188V 188V C   CA   1503 1499    2.18    1.49    0.69   20.07    1.49    0.69   20.07\n",
      "   11   1535 188V 189A C   N    1503 1505    1.78    1.35    0.43   15.34    1.35    0.43   15.34\n",
      "   12   1537 189A 189A N   CA   1505 1506    1.90    1.43    0.47   15.42    1.43    0.47   15.42\n",
      "   13   1539 189A 189A C   CA   1508 1506    2.28    1.49    0.79   23.03    1.49    0.79   23.03\n",
      "   14   1540 189A 190T C   N    1508 1510    1.85    1.35    0.50   17.86    1.35    0.50   17.86\n",
      "   15   1544 190T 190T N   CA   1510 1511    1.99    1.43    0.56   18.48    1.43    0.56   18.48\n",
      "   16   1546 190T 190T C   CA   1515 1511    2.00    1.49    0.51   14.97    1.49    0.51   14.97\n",
      "   17   1547 190T 191R C   N    1515 1517    1.51    1.35    0.17    5.97    1.35    0.17    5.97\n",
      "   18   1555 191R 191R N   CA   1517 1518    1.62    1.43    0.19    6.21    1.43    0.19    6.21\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   4974 181T 181T N   CA   1440 1441  123.41  107.00   16.41    4.72  107.00   16.41    4.72\n",
      "    2   4990 183F 183F CA  CB   1452 1453  131.01  107.50   23.52    5.44  107.50   23.52    5.44\n",
      "    3   5026 186E 186E N   CA   1477 1478  122.93  107.00   15.93    4.58  107.00   15.93    4.58\n",
      "    4   5027 186E 186E CA  CB   1478 1479  133.43  113.50   19.93    4.89  113.50   19.93    4.89\n",
      "    5   5029 186E 186E CA  C    1478 1484  147.16  116.50   30.66    8.81  116.50   30.66    8.81\n",
      "    6   5035 186E 187Y C   N    1484 1486  179.68  120.00   59.68   13.56  120.00   59.68   13.56\n",
      "    7   5038 187Y 187Y N   CA   1486 1487  157.11  107.00   50.11   14.41  107.00   50.11   14.41\n",
      "    8   5039 187Y 187Y CA  CB   1487 1488  145.19  107.50   37.69    8.71  107.50   37.69    8.71\n",
      "    9   5041 187Y 187Y CA  C    1487 1496  133.45  116.50   16.95    4.87  116.50   16.95    4.87\n",
      "   10   5042 187Y 187Y CB  CG   1488 1489  143.36  122.30   21.06    4.58  122.30   21.06    4.58\n",
      "   11   5044 187Y 187Y CG  CD1  1489 1490  148.74  120.00   28.74    5.84  120.00   28.74    5.84\n",
      "   12   5052 187Y 187Y C   CA   1496 1487  132.42  108.00   24.42    5.66  108.00   24.42    5.66\n",
      "   13   5053 187Y 188V C   N    1496 1498  157.25  120.00   37.25    8.47  120.00   37.25    8.47\n",
      "   14   5056 188V 188V N   CA   1498 1499  160.66  107.00   53.66   15.43  107.00   53.66   15.43\n",
      "   15   5060 188V 188V CA  C    1499 1503  139.75  116.50   23.25    6.68  116.50   23.25    6.68\n",
      "   16   5063 188V 189A C   N    1503 1505  154.93  120.00   34.93    7.94  120.00   34.93    7.94\n",
      "   17   5066 189A 189A N   CA   1505 1506  138.40  107.00   31.40    9.03  107.00   31.40    9.03\n",
      "   18   5068 189A 189A CA  C    1506 1508  139.97  116.50   23.47    6.75  116.50   23.47    6.75\n",
      "   19   5070 189A 190T C   N    1508 1510  149.06  120.00   29.06    6.60  120.00   29.06    6.60\n",
      "   20   5072 190T 190T N   CA   1510 1511  132.34  113.50   18.84    5.07  113.50   18.84    5.07\n",
      "   21   5073 190T 190T N   CA   1510 1511  141.88  107.00   34.88   10.03  107.00   34.88   10.03\n",
      "   22   5077 190T 190T CA  C    1511 1515  135.01  116.50   18.51    5.32  116.50   18.51    5.32\n",
      "   23   5083 191R 191R N   CA   1517 1518  123.73  107.00   16.73    4.81  107.00   16.73    4.81\n",
      "   24   5086 191R 191R CA  C    1518 1526  141.21  116.50   24.71    7.11  116.50   24.71    7.11\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  4                           : Stereochemical improper torsion potentia\n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   9636 191R 191R C   CA   1526 1518   13.67    0.00   13.67    4.81    0.00   13.67    4.81\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  9                           : Distance restraints 1 (CA-CA)           \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  12299  34G 190T CA  CA    237 1511   22.84   12.33   10.50   18.37   12.33   10.50   18.37\n",
      "    2  12321  35A 190T CA  CA    241 1511   22.80   12.52   10.28   16.66   12.52   10.28   16.66\n",
      "    3  14347 113Y 190T CA  CA    883 1511   17.99   12.24    5.75   10.88   12.24    5.75   10.88\n",
      "    4  15606 149D 190T CA  CA   1194 1511   11.94    9.11    2.83    5.46    9.11    2.83    5.46\n",
      "    5  15681 151K 190T CA  CA   1210 1511   12.02    8.74    3.28    6.43    8.74    3.28    6.43\n",
      "    6  15715 152P 190T CA  CA   1219 1511   14.85   10.81    4.04    7.53   10.81    4.04    7.53\n",
      "    7  15750 153S 190T CA  CA   1226 1511   17.09   10.54    6.55   12.40   10.54    6.55   12.40\n",
      "    8  15777 154N 190T CA  CA   1232 1511   17.29   11.92    5.37    9.81   11.92    5.37    9.81\n",
      "    9  15882 167D 190T CA  CA   1331 1511   19.69   13.98    5.70    7.77   13.98    5.70    7.77\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 10                           : Distance restraints 2 (N-O)             \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  21286 149D 190T N   O    1193 1516   11.77    8.20    3.57    6.06    8.20    3.57    6.06\n",
      "    2  21312 150L 190T N   O    1201 1516   11.01    8.05    2.95    5.05    8.05    2.95    5.05\n",
      "    3  21338 151K 190T N   O    1209 1516   10.93    8.16    2.77    4.79    8.16    2.77    4.79\n",
      "    4  21394 153S 190T N   O    1225 1516   13.89   10.01    3.88    6.56   10.01    3.88    6.56\n",
      "    5  21412 154N 190T N   O    1231 1516   15.01   10.57    4.45    7.44   10.57    4.45    7.44\n",
      "    6  21874 190T 149D N   O    1510 1200   14.36    8.77    5.59    9.36    8.77    5.59    9.36\n",
      "    7  21876 190T 151K N   O    1510 1217   16.16   10.55    5.61    9.17   10.55    5.61    9.17\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 13                           : Mainchain Omega dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10767 180H 180H CA  C    1431 1438 -154.06 -180.00   25.94    5.19 -180.00   25.94    5.19\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9 -110.18 -134.00   23.84    0.75  -62.50  179.59   31.96\n",
      "    1          2A   2A N   CA      9   10  145.96  147.00                  -40.90\n",
      "    2  10232   3A   4A C   N      17   19 -124.69 -134.00    9.59    0.24  -62.50 -174.85   33.66\n",
      "    2          4A   4A N   CA     19   20  144.71  147.00                  -40.90\n",
      "    3  10233   4A   5A C   N      22   24 -128.89 -134.00    5.26    0.22  -62.50 -176.70   33.59\n",
      "    3          5A   5A N   CA     24   25  148.25  147.00                  -40.90\n",
      "    4  10234   5A   6A C   N      27   29 -114.48 -134.00   21.38    0.49  -62.50 -173.44   28.51\n",
      "    4          6A   6A N   CA     29   30  138.27  147.00                  -40.90\n",
      "    5  10235   6A   7A C   N      32   34 -140.28 -134.00    9.13    0.28  -62.50 -177.15   34.13\n",
      "    5          7A   7A N   CA     34   35  153.62  147.00                  -40.90\n",
      "    6  10236   7A   8G C   N      37   39  179.42 -167.20   20.76    0.80   82.20  178.94   12.94\n",
      "    6          8G   8G N   CA     39   40  158.72  174.60                    8.50\n",
      "    7  10237   8G   9A C   N      41   43 -142.76 -134.00   12.96    0.41  -62.50 -178.71   34.01\n",
      "    7          9A   9A N   CA     43   44  156.55  147.00                  -40.90\n",
      "    8  10405 176P 177D C   N    1402 1404 -125.14  -63.30   90.03   14.89  -63.30   90.03   14.89\n",
      "    8        177D 177D N   CA   1404 1405 -105.43  -40.00                  -40.00\n",
      "    9  10406 177D 178H C   N    1410 1412  -78.40  -67.60   14.28    1.28  -63.20  173.63   20.80\n",
      "    9        178H 178H N   CA   1412 1413  130.66  140.00                  -42.30\n",
      "   10  10407 178H 179D C   N    1420 1422 -107.52  -96.50   32.21    1.36  -63.30 -178.98   25.18\n",
      "   10        179D 179D N   CA   1422 1423  144.46  114.20                  -40.00\n",
      "   11  10409 180H 181T C   N    1438 1440  -55.68  -78.10   43.82    2.61  -63.20  130.67   16.27\n",
      "   11        181T 181T N   CA   1440 1441 -172.55  149.80                  -42.10\n",
      "   12  10410 181T 182G C   N    1445 1447  116.19   78.70   58.96    1.12   82.20  143.97    8.64\n",
      "   12        182G 182G N   CA   1447 1448  148.40 -166.10                    8.50\n",
      "   13  10411 182G 183F C   N    1449 1451   92.11  -63.20  156.01   24.26  -63.20  156.01   24.26\n",
      "   13        183F 183F N   CA   1451 1452  -59.17  -44.30                  -44.30\n",
      "   14  10412 183F 184L C   N    1460 1462  176.88 -108.50   74.75    3.61  -63.50 -152.76   25.62\n",
      "   14        184L 184L N   CA   1462 1463  128.03  132.50                  -41.20\n",
      "   15  10413 184L 185T C   N    1468 1470 -123.54  -63.20   73.30   12.63  -63.20   73.30   12.63\n",
      "   15        185T 185T N   CA   1470 1471  -83.71  -42.10                  -42.10\n",
      "   16  10414 185T 186E C   N    1475 1477  176.05 -117.80   98.61    5.30  -63.60  159.04   19.38\n",
      "   16        186E 186E N   CA   1477 1478   63.67  136.80                  -40.30\n",
      "   17  10415 186E 187Y C   N    1484 1486   33.35  -98.40  140.59    7.71  -63.50  156.36   29.18\n",
      "   17        187Y 187Y N   CA   1486 1487   79.35  128.40                  -43.40\n",
      "   18  10417 188V 189A C   N    1503 1505 -100.09  -68.20   81.49    7.59  -62.50  117.39   17.82\n",
      "   18        189A 189A N   CA   1505 1506   70.31  145.30                  -40.90\n",
      "   19  10418 189A 190T C   N    1508 1510   82.73  -63.20  172.60   29.75  -78.10 -170.76    8.25\n",
      "   19        190T 190T N   CA   1510 1511   50.08  -42.10                  149.80\n",
      "   20  10419 190T 191R C   N    1515 1517  -32.98  -63.00   63.74    7.23  -63.00   63.74    7.23\n",
      "   20        191R 191R N   CA   1517 1518  -97.32  -41.10                  -41.10\n",
      "   21  10587 358Y 359R C   N    2896 2898 -110.09 -125.20   16.54    0.83  -63.00  177.93   27.29\n",
      "   21        359R 359R N   CA   2898 2899  147.32  140.60                  -41.10\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 26                           : Distance restraints 4 (SDCH-SDCH)       \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  33168 151K 190T CD  CG2  1213 1514   11.74    3.74    7.99    5.08    3.74    7.99    5.08\n",
      "    2  33176 151K 190T CE  CG2  1214 1514   13.09    4.62    8.47    5.34    4.62    8.47    5.34\n",
      "    3  33179 151K 190T NZ  CG2  1215 1514   13.48    4.16    9.32    5.90    4.16    9.32    5.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:  1485 2053   2.196\n",
      "  serious non-bonded atom clash:  1487 2033   2.185\n",
      "  serious non-bonded atom clash:  1496 2034   2.246\n",
      "  serious non-bonded atom clash:  1498 2034   2.102\n",
      "  serious non-bonded atom clash:  1503 1606   2.261\n",
      "  serious non-bonded atom clash:  1506 1573   1.867\n",
      "  serious non-bonded atom clash:  1511 1564   2.201\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     1    3    3    9   11   23   27   58  163  182  218  255  348  456  477\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "4qta_MODLL.B99990001.pdb      4497.55615\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 4qte\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    38601    35955\n",
      "101 (of 2915 total) atoms selected for optimization\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     101\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    35955     614\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      130\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          49.5416\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     103       0      0   0.002   0.002     0.14568       1.000\n",
      " 2 Bond angle potential               :     140       0      0   1.453   1.453      5.7617       1.000\n",
      " 3 Stereochemical cosine torsion poten:      59       0      1  49.581  49.581      20.968       1.000\n",
      " 4 Stereochemical improper torsion pot:      40       0      0   0.775   0.775     0.52530       1.000\n",
      " 5 Soft-sphere overlap restraints     :     130       0      0   0.001   0.001     0.58775E-02   1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      78       0      0   0.133   0.133     0.98119       1.000\n",
      "10 Distance restraints 2 (N-O)        :      78       0      0   0.431   0.431      4.8924       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      16       0      0   2.708   2.708      1.3836       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       6       0      0  70.848  70.848     0.43011       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       5       0      0   9.562   9.562     0.84161E-01   1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       4       0      0  72.511  72.511      1.5408       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       1       0      0  12.820  12.820     0.75451       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      34       0      0   0.310   0.310     0.66855E-01   1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      16      10      1  19.328 134.390      10.380       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      34       0      0   0.887   0.887      1.6212       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 4qte_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    2442.5825\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9  -68.27  -68.20    1.29    0.10  -62.50  172.61   28.57\n",
      "    1          2A   2A N   CA      9   10  146.59  145.30                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14  -73.97  -68.20    6.51    0.45  -62.50  171.18   28.63\n",
      "    2          3A   3A N   CA     14   15  148.31  145.30                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19  -70.34  -68.20    4.16    0.27  -62.50  170.41   28.32\n",
      "    3          4A   4A N   CA     19   20  148.87  145.30                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24  -76.43  -68.20   26.98    2.45  -62.50  161.11   25.75\n",
      "    4          5A   5A N   CA     24   25  119.60  145.30                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29  -86.82  -68.20   28.44    1.79  -62.50  154.23   26.53\n",
      "    5          6A   6A N   CA     29   30  166.80  145.30                  -40.90\n",
      "    6  10235   6A   7A C   N      32   34  -68.00  -68.20   13.67    1.11  -62.50  160.23   26.53\n",
      "    6          7A   7A N   CA     34   35  158.97  145.30                  -40.90\n",
      "    7  10236   7A   8G C   N      37   39   83.80   78.70   13.08    0.57   82.20  162.56    7.81\n",
      "    7          8G   8G N   CA     39   40 -154.05 -166.10                    8.50\n",
      "    8  10237   8G   9A C   N      41   43  -60.45  -68.20   12.26    0.77  -62.50  176.71   29.06\n",
      "    8          9A   9A N   CA     43   44  135.80  145.30                  -40.90\n",
      "    9  10238   9A  10G C   N      46   48 -155.19 -167.20   34.00    1.12   82.20 -178.16   13.82\n",
      "    9         10G  10G N   CA     48   49  142.79  174.60                    8.50\n",
      "   10  10587 358Y 359R C   N    2896 2898  -70.22  -72.10    2.54    0.16  -63.00  178.86   24.65\n",
      "   10        359R 359R N   CA   2898 2899  140.19  141.90                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    0    4   10   46  138  194  179  259  349  459  467\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "4qte_MODLL.B99990001.pdb        49.54163\n",
      "\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 5523 atoms and 1 coordinate set(s) were parsed in 0.07s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 4qyy\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    37429    34783\n",
      "230 (of 2915 total) atoms selected for optimization\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:      -91.0384\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:       91.0896\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     230\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    34783    1591\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      688\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :        1793.8978\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     237      10     10   0.054   0.054      203.54       1.000\n",
      " 2 Bond angle potential               :     320       9     17   6.331   6.331      232.82       1.000\n",
      " 3 Stereochemical cosine torsion poten:     145       0      9  53.922  53.922      63.499       1.000\n",
      " 4 Stereochemical improper torsion pot:      96       1      3   4.107   4.107      31.668       1.000\n",
      " 5 Soft-sphere overlap restraints     :     688      36     55   0.114   0.114      1046.8       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     194       0      0   0.178   0.178      3.5628       1.000\n",
      "10 Distance restraints 2 (N-O)        :     172       0      0   0.405   0.405      8.7124       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      32       3      5  12.150  12.150      54.354       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      20       0      0  62.523  62.523      3.3724       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      18       0      1  86.774  86.774      8.2048       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:      11       0      0  99.296  99.296      6.8699       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       3       0      0 119.676 119.676      2.2905       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     142       0      0   0.601   0.601      1.1749       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      33      21     11  56.521 141.560      118.94       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     168       0      0   0.598   0.598      8.1139       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 4qyy_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    7822.7310\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  1                           : Bond length potential                   \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1     46   9A   9A C   CA     46   44    1.69    1.49    0.20    5.88    1.49    0.20    5.88\n",
      "    2     47   9A  10G C   N      46   48    1.52    1.35    0.17    6.15    1.35    0.17    6.15\n",
      "    3     48  10G  10G N   CA     48   49    1.67    1.43    0.24    7.90    1.43    0.24    7.90\n",
      "    4     50  10G  10G C   CA     50   49    1.70    1.49    0.21    6.21    1.49    0.21    6.21\n",
      "    5     51  10G  11P C   N      50   52    1.56    1.30    0.26    7.72    1.30    0.26    7.72\n",
      "    6     54  11P  12E C   N      57   59    1.56    1.35    0.21    7.62    1.35    0.21    7.62\n",
      "    7     55  11P  11P N   CA     52   53    1.57    1.43    0.14    4.57    1.43    0.14    4.57\n",
      "    8     60  12E  12E CB  CA     61   60    1.75    1.54    0.22    5.92    1.54    0.22    5.92\n",
      "    9     65  12E  12E N   CA     59   60    1.88    1.43    0.45   14.77    1.43    0.45   14.77\n",
      "   10     67  12E  12E C   CA     66   60    1.84    1.49    0.35   10.23    1.49    0.35   10.23\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   3047   9A   9A C   CA     46   44  130.11  108.00   22.11    5.12  108.00   22.11    5.12\n",
      "    2   3048   9A  10G C   N      46   48  170.86  120.00   50.86   11.56  120.00   50.86   11.56\n",
      "    3   3050  10G  10G N   CA     48   49  143.81  107.00   36.81   10.59  107.00   36.81   10.59\n",
      "    4   3052  10G  10G CA  C      49   50  157.36  112.50   44.86    6.44  112.50   44.86    6.44\n",
      "    5   3060  11P  11P CA  C      53   57  147.30  116.50   30.80    8.86  116.50   30.80    8.86\n",
      "    6   3068  12E  12E N   CA     59   60  127.76  107.00   20.76    5.97  107.00   20.76    5.97\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 13                           : Mainchain Omega dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10598  11P  11P CA  C      53   57 -129.33 -180.00   50.67   10.13 -180.00   50.67   10.13\n",
      "    2  10599  12E  12E CA  C      60   66 -151.92 -180.00   28.08    5.61 -180.00   28.08    5.61\n",
      "    3  10603  16G  16G CA  C      95   96 -154.95 -180.00   25.05    5.01 -180.00   25.05    5.01\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9  -71.73  -68.20    4.91    0.48  -62.50  177.45   29.54\n",
      "    1          2A   2A N   CA      9   10  141.89  145.30                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14 -111.14 -134.00   22.87    0.79  -62.50  178.09   31.77\n",
      "    2          3A   3A N   CA     14   15  147.78  147.00                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19 -149.44 -134.00   32.80    1.36  -62.50  167.50   32.08\n",
      "    3          4A   4A N   CA     19   20  175.94  147.00                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24   44.99  -68.20  119.15    8.77  -62.50 -176.30   35.77\n",
      "    4          5A   5A N   CA     24   25  108.07  145.30                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29  161.01 -134.00   77.53    4.16  -62.50 -160.42   30.65\n",
      "    5          6A   6A N   CA     29   30  104.72  147.00                  -40.90\n",
      "    6  10235   6A   7A C   N      32   34 -153.85 -134.00   43.91    1.85  -62.50  161.30   31.26\n",
      "    6          7A   7A N   CA     34   35 -173.84  147.00                  -40.90\n",
      "    7  10237   8G   9A C   N      41   43 -159.63 -134.00   28.92    0.66  -62.50 -173.91   35.67\n",
      "    7          9A   9A N   CA     43   44  160.38  147.00                  -40.90\n",
      "    8  10239  10G  11P C   N      50   52    7.07  -64.50  100.54   11.00  -64.50  100.54   11.00\n",
      "    8         11P  11P N   CA     52   53 -142.20  147.20                  147.20\n",
      "    9  10240  11P  12E C   N      57   59   87.84  -63.60  167.27   28.69  -63.60  167.27   28.69\n",
      "    9         12E  12E N   CA     59   60   30.71  -40.30                  -40.30\n",
      "   10  10241  12E  13M C   N      66   68  -53.22  -73.00   20.67    1.64  -63.40  170.81   24.73\n",
      "   10         13M  13M N   CA     68   69  148.99  143.00                  -40.50\n",
      "   11  10244  15R  16G C   N      92   94  -64.61  -62.40   30.74    4.49   82.20  148.04   10.56\n",
      "   11         16G  16G N   CA     94   95  -10.54  -41.20                    8.50\n",
      "   12  10559 330K 331F C   N    2657 2659   95.77 -124.20  140.15    5.30  -63.20 -129.84   26.42\n",
      "   12        331F 331F N   CA   2659 2660  149.26  143.30                  -44.30\n",
      "   13  10560 331F 332D C   N    2668 2670  -90.18  -70.90   54.51    1.90  -63.30  121.73   16.75\n",
      "   13        332D 332D N   CA   2670 2671 -158.72  150.30                  -40.00\n",
      "   14  10561 332D 333M C   N    2676 2678  -59.97  -73.00   17.54    1.40  -63.40  164.80   24.34\n",
      "   14        333M 333M N   CA   2678 2679  154.74  143.00                  -40.50\n",
      "   15  10562 333M 334E C   N    2684 2686  -60.25  -69.30   15.90    0.91  -63.60  169.76   22.82\n",
      "   15        334E 334E N   CA   2686 2687  129.42  142.50                  -40.30\n",
      "   16  10563 334E 335L C   N    2693 2695  -68.32  -63.50   11.29    1.38   60.20  142.09   17.23\n",
      "   16        335L 335L N   CA   2695 2696  -31.00  -41.20                   29.60\n",
      "   17  10583 354F 355Q C   N    2864 2866  -80.03  -73.00    9.05    0.53  -63.80  174.06   26.45\n",
      "   17        355Q 355Q N   CA   2866 2867  146.40  140.70                  -40.30\n",
      "   18  10584 355Q 356P C   N    2873 2875  -53.03  -58.70    6.89    0.58  -64.50  178.77   13.95\n",
      "   18        356P 356P N   CA   2875 2876  -34.40  -30.50                  147.20\n",
      "   19  10585 356P 357G C   N    2880 2882 -169.30 -167.20    5.12    0.21   82.20 -165.50   14.15\n",
      "   19        357G 357G N   CA   2882 2883  169.92  174.60                    8.50\n",
      "   20  10586 357G 358Y C   N    2884 2886 -127.89 -124.30   18.13    0.82  -63.50  175.65   31.09\n",
      "   20        358Y 358Y N   CA   2886 2887  153.17  135.40                  -43.40\n",
      "   21  10587 358Y 359R C   N    2896 2898  -65.75  -72.10    6.37    0.50  -63.00  176.58   24.04\n",
      "   21        359R 359R N   CA   2898 2899  142.34  141.90                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:    48  355   2.029\n",
      "  serious non-bonded atom clash:    50  357   2.290\n",
      "  serious non-bonded atom clash:    52  188   2.272\n",
      "  serious non-bonded atom clash:    53  360   2.296\n",
      "  serious non-bonded atom clash:    57  287   2.232\n",
      "  serious non-bonded atom clash:    58  188   1.961\n",
      "  serious non-bonded atom clash:    58  287   1.901\n",
      "  serious non-bonded atom clash:    59  197   1.823\n",
      "  serious non-bonded atom clash:    59  287   1.945\n",
      "  serious non-bonded atom clash:    60  197   2.147\n",
      "  serious non-bonded atom clash:    60  287   2.289\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     5    1    5    6    8   18   33   62  146  186  209  286  369  426  475\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "4qyy_MODLL.B99990001.pdb      1793.89783\n",
      "\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2811 atoms and 1 coordinate set(s) were parsed in 0.03s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 4s2z\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    38256    35610\n",
      "134 (of 2915 total) atoms selected for optimization\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     134\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    35610     854\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      170\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         247.9648\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     138       0      0   0.007   0.007      1.8209       1.000\n",
      " 2 Bond angle potential               :     189       1      1   3.689   3.689      42.265       1.000\n",
      " 3 Stereochemical cosine torsion poten:      80       0      3  51.854  51.854      30.401       1.000\n",
      " 4 Stereochemical improper torsion pot:      52       0      0   1.569   1.569      3.9303       1.000\n",
      " 5 Soft-sphere overlap restraints     :     170       0      0   0.010   0.010      2.1564       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     128       1      9   0.672   0.672      46.506       1.000\n",
      "10 Distance restraints 2 (N-O)        :     118       1      3   1.107   1.107      40.447       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      21       1      1  30.278  30.278      13.859       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      10       0      0  79.754  79.754      2.1801       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       8       0      0  53.043  53.043      3.1632       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       7       0      0  85.081  85.081      5.4912       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       2       0      0 141.693 141.693      1.3896       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      55       0      0   0.768   0.768     0.95675       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      22      14      3  28.582 136.661      50.166       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      24       0      0   0.967   0.967      3.2320       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 4s2z_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    3813.0317\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 13                           : Mainchain Omega dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10603  16G  16G CA  C      95   96  -40.96 -179.18  138.22   32.12 -179.18  138.22   32.12\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10233   4A   5A C   N      22   24 -157.32 -134.00   27.18    0.64  -62.50 -175.62   35.27\n",
      "    1          5A   5A N   CA     24   25  160.97  147.00                  -40.90\n",
      "    2  10234   5A   6A C   N      27   29 -117.59 -134.00   16.55    0.63  -62.50  178.65   32.21\n",
      "    2          6A   6A N   CA     29   30  149.16  147.00                  -40.90\n",
      "    3  10235   6A   7A C   N      32   34 -139.35 -134.00   27.61    1.43  -62.50  164.12   31.01\n",
      "    3          7A   7A N   CA     34   35  174.09  147.00                  -40.90\n",
      "    4  10236   7A   8G C   N      37   39   96.35   78.70   17.72    0.93   82.20  173.57    7.71\n",
      "    4          8G   8G N   CA     39   40 -164.49 -166.10                    8.50\n",
      "    5  10237   8G   9A C   N      41   43  -64.73  -68.20   10.48    0.96  -62.50  163.93   26.97\n",
      "    5          9A   9A N   CA     43   44  155.19  145.30                  -40.90\n",
      "    6  10238   9A  10G C   N      46   48  100.97   78.70   34.13    0.67   82.20  160.63    8.73\n",
      "    6         10G  10G N   CA     48   49  168.03 -166.10                    8.50\n",
      "    7  10240  11P  12E C   N      57   59 -100.60 -117.80   24.40    0.78  -63.60  164.02   20.07\n",
      "    7         12E  12E N   CA     59   60  119.49  136.80                  -40.30\n",
      "    8  10241  12E  13M C   N      66   68  -82.73  -73.00   30.54    1.93  -63.40  148.81   23.75\n",
      "    8         13M  13M N   CA     68   69  171.95  143.00                  -40.50\n",
      "    9  10243  14V  15R C   N      81   83  -89.71  -72.10   22.54    1.47   57.30 -171.51   13.73\n",
      "    9         15R  15R N   CA     83   84  155.97  141.90                   38.00\n",
      "   10  10244  15R  16G C   N      92   94  125.92   78.70   47.45    2.49  -80.20  155.82    9.55\n",
      "   10         16G  16G N   CA     94   95 -161.40 -166.10                  174.10\n",
      "   11  10245  16G  17Q C   N      96   98  -88.85  -63.80   45.07    7.89  -63.80   45.07    7.89\n",
      "   11         17Q  17Q N   CA     98   99  -77.77  -40.30                  -40.30\n",
      "   12  10585 356P 357G C   N    2880 2882 -101.91  -80.20   29.38    2.10   82.20 -131.54   17.75\n",
      "   12        357G 357G N   CA   2882 2883  154.30  174.10                    8.50\n",
      "   13  10586 357G 358Y C   N    2884 2886 -105.95  -98.40    7.56    0.30  -63.50  177.18   26.08\n",
      "   13        358Y 358Y N   CA   2886 2887  128.62  128.40                  -43.40\n",
      "   14  10587 358Y 359R C   N    2896 2898  -81.93  -72.10   13.43    0.87  -63.00  168.91   24.11\n",
      "   14        359R 359R N   CA   2898 2899  151.05  141.90                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:     5  110   2.068\n",
      "  serious non-bonded atom clash:     6  110   1.486\n",
      "  serious non-bonded atom clash:     6  129   2.089\n",
      "  serious non-bonded atom clash:   458 2726   2.245\n",
      "  serious non-bonded atom clash:   503 1333   2.283\n",
      "  serious non-bonded atom clash:   503 1334   2.242\n",
      "  serious non-bonded atom clash:   503 1335   1.782\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2774 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  serious non-bonded atom clash:  2674 2685   2.230\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     4    0    4    0    4    5   17   44  127  168  218  245  344  458  483\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "4s2z_MODLL.B99990001.pdb       247.96483\n",
      "\n",
      "Modelando proteína 4s30\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    37972    35326\n",
      "223 (of 2915 total) atoms selected for optimization\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     223\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    35326    1900\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      386\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         281.8305\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     231       0      0   0.006   0.006      2.0849       1.000\n",
      " 2 Bond angle potential               :     318       0      2   2.544   2.544      39.598       1.000\n",
      " 3 Stereochemical cosine torsion poten:     134       0      3  50.320  50.320      53.368       1.000\n",
      " 4 Stereochemical improper torsion pot:      89       0      1   1.598   1.598      6.3477       1.000\n",
      " 5 Soft-sphere overlap restraints     :     386       0      0   0.003   0.003     0.52489       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     368       0      0   0.381   0.381      25.499       1.000\n",
      "10 Distance restraints 2 (N-O)        :     349       0      5   0.743   0.743      51.262       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      35       1      1  29.464  29.464      8.0465       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      19       0      0  71.882  71.882      4.5799       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      17       0      0  65.813  65.813      7.6847       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:      11       0      0 101.580 101.580      7.0906       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       2       0      0 143.326 143.326      1.4392       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     199       0      0   0.669   0.669      2.1526       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      38      17      8  35.166 123.854      61.643       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      90       0      2   1.645   1.645      10.509       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 4s30_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    5985.6719\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 13                           : Mainchain Omega dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10622  35A  35A CA  C     241  243   -6.57 -179.18  172.61   40.12 -179.18  172.61   40.12\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10233   4A   5A C   N      22   24 -109.04 -134.00   24.99    0.78  -62.50  179.52   31.88\n",
      "    1          5A   5A N   CA     24   25  145.72  147.00                  -40.90\n",
      "    2  10234   5A   6A C   N      27   29 -141.19 -134.00   24.45    1.18  -62.50  168.28   31.79\n",
      "    2          6A   6A N   CA     29   30  170.36  147.00                  -40.90\n",
      "    3  10235   6A   7A C   N      32   34 -120.72 -134.00   13.50    0.54  -62.50  179.39   32.51\n",
      "    3          7A   7A N   CA     34   35  149.42  147.00                  -40.90\n",
      "    4  10236   7A   8G C   N      37   39  125.39   78.70   47.68    2.14   82.20 -179.04   10.88\n",
      "    4          8G   8G N   CA     39   40 -175.76 -166.10                    8.50\n",
      "    5  10237   8G   9A C   N      41   43 -145.44 -134.00   12.75    0.29  -62.50 -174.00   34.92\n",
      "    5          9A   9A N   CA     43   44  152.62  147.00                  -40.90\n",
      "    6  10240  11P  12E C   N      57   59  -60.74  -69.30    9.78    0.88  -63.60  172.49   22.84\n",
      "    6         12E  12E N   CA     59   60  147.24  142.50                  -40.30\n",
      "    7  10241  12E  13M C   N      66   68 -113.67 -125.60   19.60    1.07  -63.40  171.00   29.54\n",
      "    7         13M  13M N   CA     68   69  156.05  140.50                  -40.50\n",
      "    8  10242  13M  14V C   N      74   76  -60.83  -62.40    2.65    0.43 -125.40 -172.12   10.54\n",
      "    8         14V  14V N   CA     76   77  -40.26  -42.40                  143.30\n",
      "    9  10243  14V  15R C   N      81   83  -67.70  -72.10    4.56    0.32  -63.00  178.26   24.40\n",
      "    9         15R  15R N   CA     83   84  140.70  141.90                  -41.10\n",
      "   10  10262  33E  34G C   N     234  236 -171.85   82.20  109.68    6.14  -62.40  111.51   18.94\n",
      "   10         34G  34G N   CA    236  237  -19.88    8.50                  -41.20\n",
      "   11  10264  35A  36Y C   N     243  245  -68.70  -98.40   30.02    1.19  -63.50  176.29   27.71\n",
      "   11         36Y  36Y N   CA    245  246  132.81  128.40                  -43.40\n",
      "   12  10563 334E 335L C   N    2693 2695 -149.43 -108.50   41.29    1.91  -63.50 -161.28   24.38\n",
      "   12        335L 335L N   CA   2695 2696  137.98  132.50                  -41.20\n",
      "   13  10565 336D 337D C   N    2709 2711 -114.08  -96.50   22.10    0.92  -63.30  175.12   19.13\n",
      "   13        337D 337D N   CA   2711 2712  127.59  114.20                  -40.00\n",
      "   14  10566 337D 338L C   N    2717 2719  -59.36  -63.50    9.76    1.61 -108.50  172.03    8.71\n",
      "   14        338L 338L N   CA   2719 2720  -32.36  -41.20                  132.50\n",
      "   15  10585 356P 357G C   N    2880 2882  -61.79  -62.40    1.95    0.27  -80.20  144.02    6.60\n",
      "   15        357G 357G N   CA   2882 2883  -43.06  -41.20                  174.10\n",
      "   16  10586 357G 358Y C   N    2884 2886  -69.69  -98.40   29.01    1.14  -63.50  176.08   27.62\n",
      "   16        358Y 358Y N   CA   2886 2887  132.57  128.40                  -43.40\n",
      "   17  10587 358Y 359R C   N    2896 2898 -136.29 -125.20   11.36    0.39  -63.00 -169.53   30.76\n",
      "   17        359R 359R N   CA   2898 2899  143.09  140.60                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:   667  689   1.822\n",
      "  serious non-bonded atom clash:   667  691   2.007\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     2    0    0    2    2   10   22   58  111  185  201  297  360  419  500\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "4s30_MODLL.B99990001.pdb       281.83051\n",
      "\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2820 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 4s31\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    38284    35638\n",
      "125 (of 2915 total) atoms selected for optimization\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     125\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    35638     767\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      171\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         117.8646\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     129       0      0   0.004   0.004     0.68627       1.000\n",
      " 2 Bond angle potential               :     177       0      0   1.867   1.867      12.364       1.000\n",
      " 3 Stereochemical cosine torsion poten:      76       0      1  48.773  48.773      27.117       1.000\n",
      " 4 Stereochemical improper torsion pot:      48       0      0   0.857   0.857     0.83218       1.000\n",
      " 5 Soft-sphere overlap restraints     :     171       0      0   0.003   0.003     0.15312       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     116       0      0   0.450   0.450      17.626       1.000\n",
      "10 Distance restraints 2 (N-O)        :     108       0      1   0.990   0.990      19.611       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      20       1      0  40.133  40.133      4.8175       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       9       0      0  81.461  81.461      1.4487       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       7       0      0  34.211  34.211     0.68849       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       6       0      0 108.117 108.117      4.0638       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       2       0      0  99.579  99.579      1.4418       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      44       0      0   0.383   0.383     0.15885       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      21      12      4  36.232 136.005      26.826       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       4       0      0   0.252   0.252     0.29709E-01   1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 4s31_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    3548.6125\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10233   4A   5A C   N      22   24 -142.01 -134.00   16.48    0.67  -62.50  176.61   33.20\n",
      "    1          5A   5A N   CA     24   25  161.40  147.00                  -40.90\n",
      "    2  10234   5A   6A C   N      27   29 -108.12 -134.00   25.88    0.87  -62.50  177.62   31.52\n",
      "    2          6A   6A N   CA     29   30  147.44  147.00                  -40.90\n",
      "    3  10235   6A   7A C   N      32   34  -70.18  -68.20    6.28    0.44  -62.50  168.02   27.92\n",
      "    3          7A   7A N   CA     34   35  151.26  145.30                  -40.90\n",
      "    4  10236   7A   8G C   N      37   39   82.12   78.70   18.19    0.38   82.20  167.53    8.13\n",
      "    4          8G   8G N   CA     39   40  176.03 -166.10                    8.50\n",
      "    5  10237   8G   9A C   N      41   43 -118.46 -134.00   16.47    0.76  -62.50  175.78   31.79\n",
      "    5          9A   9A N   CA     43   44  152.47  147.00                  -40.90\n",
      "    6  10240  11P  12E C   N      57   59  -66.76  -69.30    3.67    0.20  -63.60  179.88   24.16\n",
      "    6         12E  12E N   CA     59   60  139.85  142.50                  -40.30\n",
      "    7  10241  12E  13M C   N      66   68 -148.23 -125.60   88.95    3.73  -63.40  125.86   24.95\n",
      "    7         13M  13M N   CA     68   69 -133.47  140.50                  -40.50\n",
      "    8  10242  13M  14V C   N      74   76  -60.33  -62.40    7.37    1.06 -125.40 -169.90    7.56\n",
      "    8         14V  14V N   CA     76   77  -35.32  -42.40                  143.30\n",
      "    9  10243  14V  15R C   N      81   83  -59.94  -72.10   14.07    1.20   57.30  161.43   10.78\n",
      "    9         15R  15R N   CA     83   84  148.96  141.90                   38.00\n",
      "   10  10244  15R  16G C   N      92   94   77.40   78.70   59.09    1.74  -80.20  162.42    9.39\n",
      "   10         16G  16G N   CA     94   95  134.83 -166.10                  174.10\n",
      "   11  10586 357G 358Y C   N    2884 2886 -132.40 -124.30   32.05    1.39  -63.50  165.23   29.62\n",
      "   11        358Y 358Y N   CA   2886 2887  166.41  135.40                  -43.40\n",
      "   12  10587 358Y 359R C   N    2896 2898 -119.39 -125.20   14.62    0.80  -63.00  174.27   27.43\n",
      "   12        359R 359R N   CA   2898 2899  154.01  140.60                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:     5  110   1.146\n",
      "  serious non-bonded atom clash:     6  109   2.232\n",
      "  serious non-bonded atom clash:     6  110   1.091\n",
      "  serious non-bonded atom clash:   492 2702   2.297\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     2    0    2    0    1    6   14   33  127  184  192  263  397  430  474\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "4s31_MODLL.B99990001.pdb       117.86461\n",
      "\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2807 atoms and 1 coordinate set(s) were parsed in 0.03s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 4s32\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    38375    35729\n",
      "134 (of 2915 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2811 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     134\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    35729     872\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      229\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         138.0872\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     138       0      0   0.003   0.003     0.35809       1.000\n",
      " 2 Bond angle potential               :     189       0      0   1.838   1.838      12.022       1.000\n",
      " 3 Stereochemical cosine torsion poten:      80       0      4  54.634  54.634      36.620       1.000\n",
      " 4 Stereochemical improper torsion pot:      52       0      0   0.784   0.784      1.0923       1.000\n",
      " 5 Soft-sphere overlap restraints     :     229       0      0   0.002   0.002     0.14244       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     132       0      0   0.364   0.364      8.3375       1.000\n",
      "10 Distance restraints 2 (N-O)        :     124       0      1   0.925   0.925      17.749       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      21       1      1  20.060  20.060      22.282       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      10       0      0  84.454  84.454      3.0443       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       8       0      0  61.113  61.113      1.8938       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       7       0      0  78.188  78.188      3.2416       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       2       0      0   4.434   4.434      1.1572       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      67       0      0   0.524   0.524     0.52825       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      22      11      3  31.032 131.396      27.010       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      20       0      0   0.949   0.949      2.6084       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 4s32_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    3359.1960\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 13                           : Mainchain Omega dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10603  16G  16G CA  C      95   96  -87.74 -179.18   91.44   21.25 -179.18   91.44   21.25\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10233   4A   5A C   N      22   24  176.09 -134.00   55.74    1.27  -62.50 -169.12   37.60\n",
      "    1          5A   5A N   CA     24   25  171.81  147.00                  -40.90\n",
      "    2  10234   5A   6A C   N      27   29 -109.60 -134.00   28.50    0.67  -62.50  179.46   27.51\n",
      "    2          6A   6A N   CA     29   30  132.27  147.00                  -40.90\n",
      "    3  10235   6A   7A C   N      32   34  -75.97  -68.20    9.91    0.64  -62.50  168.19   28.24\n",
      "    3          7A   7A N   CA     34   35  151.45  145.30                  -40.90\n",
      "    4  10237   8G   9A C   N      41   43 -131.58 -134.00   12.93    0.78  -62.50  173.73   32.17\n",
      "    4          9A   9A N   CA     43   44  159.70  147.00                  -40.90\n",
      "    5  10238   9A  10G C   N      46   48  -94.16  -80.20   43.96    1.12   82.20 -126.71   18.09\n",
      "    5         10G  10G N   CA     48   49 -144.21  174.10                    8.50\n",
      "    6  10240  11P  12E C   N      57   59 -141.76 -117.80   44.79    1.66  -63.60  164.77   26.31\n",
      "    6         12E  12E N   CA     59   60  174.64  136.80                  -40.30\n",
      "    7  10243  14V  15R C   N      81   83 -166.20 -125.20   44.78    1.39   57.30 -177.84   21.69\n",
      "    7         15R  15R N   CA     83   84  158.62  140.60                   38.00\n",
      "    8  10244  15R  16G C   N      92   94   63.00   78.70   31.83    0.48  -80.20  150.88   11.03\n",
      "    8         16G  16G N   CA     94   95 -138.41 -166.10                  174.10\n",
      "    9  10585 356P 357G C   N    2880 2882 -172.90 -167.20    7.59    0.16   82.20 -159.30   14.37\n",
      "    9        357G 357G N   CA   2882 2883  179.61  174.60                    8.50\n",
      "   10  10586 357G 358Y C   N    2884 2886 -110.72  -98.40   14.75    0.93  -63.50  170.36   24.80\n",
      "   10        358Y 358Y N   CA   2886 2887  120.28  128.40                  -43.40\n",
      "   11  10587 358Y 359R C   N    2896 2898 -114.92 -125.20   10.30    0.40  -63.00 -173.75   28.75\n",
      "   11        359R 359R N   CA   2898 2899  140.03  140.60                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:  2715 2718   1.993\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     1    0    0    0    0    7    9   50  143  190  174  260  375  430  477\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "4s32_MODLL.B99990001.pdb       138.08720\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 4s33\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    38346    35700\n",
      "125 (of 2915 total) atoms selected for optimization\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     125\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    35700     773\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      177\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         140.2335\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     129       0      0   0.005   0.005      1.0364       1.000\n",
      " 2 Bond angle potential               :     177       1      1   2.613   2.613      21.421       1.000\n",
      " 3 Stereochemical cosine torsion poten:      76       0      2  53.208  53.208      31.292       1.000\n",
      " 4 Stereochemical improper torsion pot:      48       0      0   0.718   0.718     0.52247       1.000\n",
      " 5 Soft-sphere overlap restraints     :     177       0      0   0.002   0.002     0.12402       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     119       0      1   0.499   0.499      20.322       1.000\n",
      "10 Distance restraints 2 (N-O)        :     114       0      2   0.908   0.908      21.549       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      20       0      0   4.756   4.756      5.3346       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       9       0      0  64.828  64.828      2.3292       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       7       0      0  33.035  33.035     0.60033       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       6       0      0  76.574  76.574      3.4676       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       2       0      0 158.391 158.391      1.5032       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      41       0      0   0.320   0.320     0.95719E-01   1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      21      10      6  41.643 132.092      30.471       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :       4       0      0   0.589   0.589     0.16442       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 4s33_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    3298.2488\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10233   4A   5A C   N      22   24 -159.98 -134.00   40.73    1.37  -62.50  171.20   33.21\n",
      "    1          5A   5A N   CA     24   25  178.36  147.00                  -40.90\n",
      "    2  10234   5A   6A C   N      27   29 -102.06 -134.00   31.98    1.00  -62.50  178.16   31.28\n",
      "    2          6A   6A N   CA     29   30  145.39  147.00                  -40.90\n",
      "    3  10235   6A   7A C   N      32   34  -66.81  -68.20    9.42    0.81  -62.50  164.54   27.18\n",
      "    3          7A   7A N   CA     34   35  154.62  145.30                  -40.90\n",
      "    4  10237   8G   9A C   N      41   43 -113.46 -134.00   26.71    0.72  -62.50  178.26   27.21\n",
      "    4          9A   9A N   CA     43   44  129.92  147.00                  -40.90\n",
      "    5  10238   9A  10G C   N      46   48   67.13   78.70   14.04    0.42   82.20  167.34    8.88\n",
      "    5         10G  10G N   CA     48   49 -158.15 -166.10                    8.50\n",
      "    6  10240  11P  12E C   N      57   59 -158.46 -117.80   75.66    2.80  -63.60  152.25   25.35\n",
      "    6         12E  12E N   CA     59   60 -159.39  136.80                  -40.30\n",
      "    7  10241  12E  13M C   N      66   68  -74.70  -73.00   42.44    2.90  -63.40  141.55   20.28\n",
      "    7         13M  13M N   CA     68   69  100.60  143.00                  -40.50\n",
      "    8  10243  14V  15R C   N      81   83  -63.88  -63.00    4.90    0.61   57.30  142.14   17.54\n",
      "    8         15R  15R N   CA     83   84  -36.28  -41.10                   38.00\n",
      "    9  10586 357G 358Y C   N    2884 2886 -127.35 -124.30    3.54    0.19  -63.50 -169.56   33.43\n",
      "    9        358Y 358Y N   CA   2886 2887  137.19  135.40                  -43.40\n",
      "   10  10587 358Y 359R C   N    2896 2898 -120.93 -125.20    5.18    0.15  -63.00 -172.08   22.12\n",
      "   10        359R 359R N   CA   2898 2899  137.67  140.60                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:   503 1335   2.290\n",
      "  serious non-bonded atom clash:  2689 2707   2.096\n",
      "  serious non-bonded atom clash:  2692 2707   1.981\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     2    0    1    3    1    7   14   41  130  168  186  267  374  436  490\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "4s33_MODLL.B99990001.pdb       140.23349\n",
      "\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2782 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 4s34\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    37624    34978\n",
      "195 (of 2915 total) atoms selected for optimization\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:      -92.0623\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     195\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    34978    1526\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      310\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         233.9547\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     201       0      0   0.004   0.004     0.82477       1.000\n",
      " 2 Bond angle potential               :     274       0      0   2.066   2.066      22.206       1.000\n",
      " 3 Stereochemical cosine torsion poten:     115       0      8  55.231  55.231      54.793       1.000\n",
      " 4 Stereochemical improper torsion pot:      77       0      0   1.141   1.141      2.4563       1.000\n",
      " 5 Soft-sphere overlap restraints     :     310       0      0   0.002   0.002     0.13355       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     251       0      0   0.285   0.285      13.153       1.000\n",
      "10 Distance restraints 2 (N-O)        :     225       0      3   0.703   0.703      29.026       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      29       1      3  15.897  15.897      31.249       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      17       0      0  85.952  85.952      6.5746       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      15       0      0  60.492  60.492      5.4174       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:      10       0      0 113.204 113.204      7.3525       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       3       0      0  69.926  69.926      1.7635       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     166       0      0   0.573   0.573      1.2247       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      31      15      8  45.165 131.623      52.788       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     112       0      0   0.718   0.718      4.9927       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 4s34_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    4953.5312\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10233   4A   5A C   N      22   24 -140.20 -134.00   17.00    0.78  -62.50  174.52   32.76\n",
      "    1          5A   5A N   CA     24   25  162.83  147.00                  -40.90\n",
      "    2  10234   5A   6A C   N      27   29 -122.23 -134.00   11.80    0.36  -62.50 -177.05   33.17\n",
      "    2          6A   6A N   CA     29   30  146.17  147.00                  -40.90\n",
      "    3  10235   6A   7A C   N      32   34 -125.05 -134.00   10.22    0.24  -62.50 -172.25   34.11\n",
      "    3          7A   7A N   CA     34   35  142.08  147.00                  -40.90\n",
      "    4  10239  10G  11P C   N      50   52  -66.31  -58.70   11.16    0.76  -64.50  169.54   12.75\n",
      "    4         11P  11P N   CA     52   53  -22.34  -30.50                  147.20\n",
      "    5  10240  11P  12E C   N      57   59 -113.98 -117.80    9.52    0.52  -63.60 -178.68   27.03\n",
      "    5         12E  12E N   CA     59   60  145.52  136.80                  -40.30\n",
      "    6  10241  12E  13M C   N      66   68 -105.49 -125.60   23.19    1.18  -63.40  172.67   29.14\n",
      "    6         13M  13M N   CA     68   69  152.04  140.50                  -40.50\n",
      "    7  10243  14V  15R C   N      81   83  175.06 -125.20   76.62    2.27   57.30 -168.85   21.29\n",
      "    7         15R  15R N   CA     83   84 -171.43  140.60                   38.00\n",
      "    8  10244  15R  16G C   N      92   94   80.72   78.70  119.65    3.33  -80.20 -170.63    8.17\n",
      "    8         16G  16G N   CA     94   95   74.27 -166.10                  174.10\n",
      "    9  10560 331F 332D C   N    2668 2670  -71.62  -96.50   76.06    3.21  -63.30   82.74    9.74\n",
      "    9        332D 332D N   CA   2670 2671   42.33  114.20                  -40.00\n",
      "   10  10562 333M 334E C   N    2684 2686 -166.01 -117.80   63.35    3.31  -63.60  170.26   19.77\n",
      "   10        334E 334E N   CA   2686 2687   95.71  136.80                  -40.30\n",
      "   11  10563 334E 335L C   N    2693 2695 -128.93 -108.50   72.75    3.69   60.20 -117.04   27.73\n",
      "   11        335L 335L N   CA   2695 2696 -157.68  132.50                   29.60\n",
      "   12  10564 335L 336D C   N    2701 2703   68.37   54.50  118.25    7.62  -70.90 -167.31   15.67\n",
      "   12        336D 336D N   CA   2703 2704  -76.53   40.90                  150.30\n",
      "   13  10585 356P 357G C   N    2880 2882  -61.90  -62.40    4.27    0.61   82.20  153.87   11.77\n",
      "   13        357G 357G N   CA   2882 2883  -45.44  -41.20                    8.50\n",
      "   14  10586 357G 358Y C   N    2884 2886 -134.91 -124.30   10.82    0.87  -63.50 -169.44   27.07\n",
      "   14        358Y 358Y N   CA   2886 2887  133.28  135.40                  -43.40\n",
      "   15  10587 358Y 359R C   N    2896 2898  -90.29  -72.10   18.50    1.48  -63.00 -178.35   22.82\n",
      "   15        359R 359R N   CA   2898 2899  138.50  141.90                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:   667  689   1.985\n",
      "  serious non-bonded atom clash:   667  691   1.969\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     2    0    0    1    3    8   23   56  112  152  207  264  341  407  440\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "4s34_MODLL.B99990001.pdb       233.95473\n",
      "\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2817 atoms and 1 coordinate set(s) were parsed in 0.04s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 4xj0\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    37821    35175\n",
      "206 (of 2915 total) atoms selected for optimization\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     206\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    35175    2209\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      446\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         148.9870\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     212       0      0   0.004   0.004     0.91852       1.000\n",
      " 2 Bond angle potential               :     290       0      0   1.879   1.879      20.241       1.000\n",
      " 3 Stereochemical cosine torsion poten:     115       0      6  54.564  54.564      57.701       1.000\n",
      " 4 Stereochemical improper torsion pot:      83       0      0   1.162   1.162      2.4291       1.000\n",
      " 5 Soft-sphere overlap restraints     :     446       0      0   0.001   0.001     0.26605E-01   1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     441       0      0   0.108   0.108      3.9937       1.000\n",
      "10 Distance restraints 2 (N-O)        :     406       0      0   0.232   0.232      7.3493       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      32       0      0   4.768   4.768      7.1960       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      19       0      0  59.012  59.012      2.0228       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      18       0      0  74.024  74.024      6.3718       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       8       0      0  40.876  40.876      4.2322       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       2       0      0  20.058  20.058      1.0907       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     302       0      0   0.578   0.578      2.5116       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      35      14      2  24.067 112.890      15.475       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     246       0      0   0.898   0.898      17.427       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 4xj0_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    4734.6611\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9  -63.44  -68.20   10.67    0.70  -62.50  176.65   28.90\n",
      "    1          2A   2A N   CA      9   10  135.75  145.30                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14  -78.73  -68.20   16.84    1.06  -62.50  161.47   27.29\n",
      "    2          3A   3A N   CA     14   15  158.45  145.30                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19  -74.14  -68.20   19.41    1.76  -62.50  168.13   27.00\n",
      "    3          4A   4A N   CA     19   20  126.83  145.30                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24 -136.96 -134.00    7.39    0.46  -62.50 -166.25   35.74\n",
      "    4          5A   5A N   CA     24   25  140.23  147.00                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29 -139.56 -134.00    6.49    0.34  -62.50 -168.38   35.53\n",
      "    5          6A   6A N   CA     29   30  143.66  147.00                  -40.90\n",
      "    6  10235   6A   7A C   N      32   34 -127.94 -134.00   10.65    0.65  -62.50  175.96   32.34\n",
      "    6          7A   7A N   CA     34   35  155.76  147.00                  -40.90\n",
      "    7  10236   7A   8G C   N      37   39 -139.68 -167.20   35.78    0.71   82.20 -161.02   15.21\n",
      "    7          8G   8G N   CA     39   40  151.74  174.60                    8.50\n",
      "    8  10237   8G   9A C   N      41   43 -141.25 -134.00    9.92    0.29  -62.50 -176.86   34.23\n",
      "    8          9A   9A N   CA     43   44  153.76  147.00                  -40.90\n",
      "    9  10238   9A  10G C   N      46   48   91.55   78.70   46.16    0.82   82.20  141.37    7.33\n",
      "    9         10G  10G N   CA     48   49  149.56 -166.10                    8.50\n",
      "   10  10239  10G  11P C   N      50   52  -73.30  -58.70   22.71    1.48  -64.50  160.54   12.45\n",
      "   10         11P  11P N   CA     52   53  -13.10  -30.50                  147.20\n",
      "   11  10354 125H 126I C   N     992  994  -61.67  -63.40    2.21    0.42 -120.60 -177.69    8.29\n",
      "   11        126I 126I N   CA    994  995  -42.22  -43.60                  130.30\n",
      "   12  10487 258L 259K C   N    2059 2061  -74.25  -62.90   17.39    2.07   56.60  146.66   18.78\n",
      "   12        259K 259K N   CA   2061 2062  -27.63  -40.80                   38.60\n",
      "   13  10560 331F 332D C   N    2668 2670   68.05   54.50   19.83    1.17  -63.30  147.18   25.13\n",
      "   13        332D 332D N   CA   2670 2671   26.41   40.90                  -40.00\n",
      "   14  10561 332D 333M C   N    2676 2678 -171.75 -125.60   46.21    1.77  -63.40 -152.75   39.28\n",
      "   14        333M 333M N   CA   2678 2679  142.83  140.50                  -40.50\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    0    1    6   59  102  106  174  238  294  466  497\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "4xj0_MODLL.B99990001.pdb       148.98705\n",
      "\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2819 atoms and 1 coordinate set(s) were parsed in 0.03s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 4xne\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    37532    34886\n",
      "191 (of 2915 total) atoms selected for optimization\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:       90.5997\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     191\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    34886    1793\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      378\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         117.6236\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     196       0      0   0.004   0.004     0.94710       1.000\n",
      " 2 Bond angle potential               :     272       0      0   1.868   1.868      18.800       1.000\n",
      " 3 Stereochemical cosine torsion poten:     118       0      2  47.826  47.826      41.356       1.000\n",
      " 4 Stereochemical improper torsion pot:      77       0      0   1.106   1.106      2.0889       1.000\n",
      " 5 Soft-sphere overlap restraints     :     378       0      0   0.001   0.001     0.67833E-01   1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     322       0      0   0.093   0.093      1.9266       1.000\n",
      "10 Distance restraints 2 (N-O)        :     308       0      0   0.243   0.243      4.2164       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      30       1      0  33.045  33.045      4.8771       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      18       0      0  76.810  76.810      2.7230       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      13       0      0  67.920  67.920      4.1018       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       6       0      0 104.137 104.137      3.9902       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       1       0      0  64.721  64.721     0.89993       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     243       0      0   0.546   0.546      1.6610       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      32      12      2  20.207 119.874      23.167       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     157       0      0   0.690   0.690      6.8010       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 4xne_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    4303.8892\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9  -67.79  -68.20    3.29    0.28  -62.50  170.61   28.22\n",
      "    1          2A   2A N   CA      9   10  148.57  145.30                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14 -122.56 -134.00   14.14    0.78  -62.50  174.46   31.80\n",
      "    2          3A   3A N   CA     14   15  155.31  147.00                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19  -68.13  -68.20    1.78    0.14  -62.50  175.67   29.07\n",
      "    3          4A   4A N   CA     19   20  143.53  145.30                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24  -74.25  -68.20    6.07    0.50  -62.50  173.71   29.06\n",
      "    4          5A   5A N   CA     24   25  145.79  145.30                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29  -82.66  -68.20   25.39    1.61  -62.50  154.25   26.32\n",
      "    5          6A   6A N   CA     29   30  166.17  145.30                  -40.90\n",
      "    6  10235   6A   7A C   N      32   34  -72.45  -68.20   17.27    1.24  -62.50  157.37   26.29\n",
      "    6          7A   7A N   CA     34   35  162.04  145.30                  -40.90\n",
      "    7  10236   7A   8G C   N      37   39  -91.96  -80.20   22.50    0.54   82.20 -112.94   18.94\n",
      "    7          8G   8G N   CA     39   40 -166.72  174.10                    8.50\n",
      "    8  10237   8G   9A C   N      41   43  -78.55  -68.20   25.20    1.68  -62.50  151.68   25.68\n",
      "    8          9A   9A N   CA     43   44  168.28  145.30                  -40.90\n",
      "    9  10238   9A  10G C   N      46   48  -81.83  -80.20    5.29    0.14   82.20 -123.31    7.25\n",
      "    9         10G  10G N   CA     48   49  179.13  174.10                    8.50\n",
      "   10  10585 356P 357G C   N    2880 2882 -148.82 -167.20   22.53    0.41   82.20 -159.84   15.06\n",
      "   10        357G 357G N   CA   2882 2883  161.57  174.60                    8.50\n",
      "   11  10586 357G 358Y C   N    2884 2886  -77.43  -98.40   21.76    0.96  -63.50  178.18   27.57\n",
      "   11        358Y 358Y N   CA   2886 2887  134.23  128.40                  -43.40\n",
      "   12  10587 358Y 359R C   N    2896 2898 -115.43 -125.20   12.62    0.68  -63.00  178.20   27.70\n",
      "   12        359R 359R N   CA   2898 2899  148.59  140.60                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:   212  232   2.272\n",
      "  serious non-bonded atom clash:   492 2702   2.216\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    2    0    4    3   19   31  115  172  180  239  334  396  473\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "4xne_MODLL.B99990001.pdb       117.62361\n",
      "\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2647 atoms and 1 coordinate set(s) were parsed in 0.09s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 4xoy\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    35627    32981\n",
      "487 (of 2915 total) atoms selected for optimization\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     487\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    32981    4602\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):     1115\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         308.9439\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     508       0      0   0.004   0.004      1.8992       1.000\n",
      " 2 Bond angle potential               :     704       0      1   1.884   1.884      46.075       1.000\n",
      " 3 Stereochemical cosine torsion poten:     357       0     12  48.560  48.560      122.46       1.000\n",
      " 4 Stereochemical improper torsion pot:     232       0      0   1.045   1.045      5.7995       1.000\n",
      " 5 Soft-sphere overlap restraints     :    1115       0      0   0.002   0.002     0.76134       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     855       0      0   0.180   0.180      8.2406       1.000\n",
      "10 Distance restraints 2 (N-O)        :     783       0      1   0.399   0.399      30.025       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      68       0      0   3.766   3.766      8.4749       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      51       0      2  76.380  76.380      15.408       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      40       0      0  58.526  58.526      8.9973       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:      15       0      0 104.853 104.853      11.320       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       3       0      0 115.188 115.188      1.9994       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     468       0      0   0.455   0.455      2.4783       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      74      30      3  24.907 110.062      24.347       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     444       0      0   0.723   0.723      20.655       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 4xoy_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:   10062.3916\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9 -101.98 -134.00   32.10    1.15  -62.50  174.39   30.66\n",
      "    1          2A   2A N   CA      9   10  149.24  147.00                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14 -157.22 -134.00   42.28    1.60  -62.50  166.37   32.27\n",
      "    2          3A   3A N   CA     14   15 -177.68  147.00                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19  -65.18  -68.20   11.80    1.05  -62.50  162.42   26.75\n",
      "    3          4A   4A N   CA     19   20  156.71  145.30                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24  -63.20  -68.20    7.55    0.74  -62.50  168.15   27.59\n",
      "    4          5A   5A N   CA     24   25  150.96  145.30                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29  -70.46  -68.20    3.17    0.31  -62.50  176.20   29.27\n",
      "    5          6A   6A N   CA     29   30  143.08  145.30                  -40.90\n",
      "    6  10236   7A   8G C   N      37   39   78.10   78.70   10.24    0.27   82.20  164.43    8.19\n",
      "    6          8G   8G N   CA     39   40 -155.88 -166.10                    8.50\n",
      "    7  10237   8G   9A C   N      41   43  -74.54  -68.20   10.35    0.65  -62.50  166.05   27.82\n",
      "    7          9A   9A N   CA     43   44  153.49  145.30                  -40.90\n",
      "    8  10238   9A  10G C   N      46   48  150.57 -167.20   42.45    0.71   82.20 -176.33   12.11\n",
      "    8         10G  10G N   CA     48   49  178.98  174.60                    8.50\n",
      "    9  10285  56I  57S C   N     407  409 -112.87 -136.60   23.85    1.01  -64.10  178.19   16.28\n",
      "    9         57S  57S N   CA    409  410  153.61  151.20                  -35.00\n",
      "   10  10287  58P  59F C   N     420  422 -133.40 -124.20   11.12    0.29  -63.20 -179.62   29.63\n",
      "   10         59F  59F N   CA    422  423  149.54  143.30                  -44.30\n",
      "   11  10288  59F  60E C   N     431  433 -106.53 -117.80   20.33    1.11  -63.60  171.44   25.29\n",
      "   11         60E  60E N   CA    433  434  153.73  136.80                  -40.30\n",
      "   12  10289  60E  61H C   N     440  442   56.22   56.30   11.44    1.02  -63.20  139.27   23.92\n",
      "   12         61H  61H N   CA    442  443   29.36   40.80                  -42.30\n",
      "   13  10290  61H  62Q C   N     450  452 -139.75 -121.10   19.72    0.63  -63.80 -170.52   31.85\n",
      "   13         62Q  62Q N   CA    452  453  146.10  139.70                  -40.30\n",
      "   14  10355 126I 127C C   N    1000 1002  -61.12  -63.00    3.11    0.33 -117.90 -175.70    7.95\n",
      "   14        127C 127C N   CA   1002 1003  -43.56  -41.10                  141.10\n",
      "   15  10356 127C 128Y C   N    1006 1008  -65.44  -63.50    2.15    0.40   55.90  147.48   18.06\n",
      "   15        128Y 128Y N   CA   1008 1009  -44.33  -43.40                   39.50\n",
      "   16  10390 161C 162D C   N    1289 1291   54.79   54.50    2.06    0.13  -63.30  142.00   24.18\n",
      "   16        162D 162D N   CA   1291 1292   38.86   40.90                  -40.00\n",
      "   17  10414 185T 186E C   N    1475 1477 -135.68 -117.80   20.77    1.03  -63.60 -178.53   21.21\n",
      "   17        186E 186E N   CA   1477 1478  126.24  136.80                  -40.30\n",
      "   18  10415 186E 187Y C   N    1484 1486 -136.86 -124.30   19.95    0.83  -63.50 -178.79   32.38\n",
      "   18        187Y 187Y N   CA   1486 1487  150.90  135.40                  -43.40\n",
      "   19  10417 188V 189A C   N    1503 1505 -163.23 -134.00   32.24    0.73  -62.50 -172.21   36.13\n",
      "   19        189A 189A N   CA   1505 1506  160.61  147.00                  -40.90\n",
      "   20  10482 253N 254C C   N    2021 2023  -58.60  -63.00    4.85    0.87 -117.90 -170.65    8.18\n",
      "   20        254C 254C N   CA   2023 2024  -39.07  -41.10                  141.10\n",
      "   21  10558 329F 330K C   N    2648 2650  -87.07  -70.20   30.22    1.84  -62.90  155.62   21.61\n",
      "   21        330K 330K N   CA   2650 2651  165.47  140.40                  -40.80\n",
      "   22  10559 330K 331F C   N    2657 2659  -71.47  -71.40   11.20    0.90  -63.20  164.01   23.33\n",
      "   22        331F 331F N   CA   2659 2660  151.90  140.70                  -44.30\n",
      "   23  10560 331F 332D C   N    2668 2670  -83.58  -70.90   37.03    1.30  -63.30  136.42   18.12\n",
      "   23        332D 332D N   CA   2670 2671 -174.90  150.30                  -40.00\n",
      "   24  10561 332D 333M C   N    2676 2678  -73.10  -73.00    2.66    0.18  -63.40  174.11   26.76\n",
      "   24        333M 333M N   CA   2678 2679  145.66  143.00                  -40.50\n",
      "   25  10563 334E 335L C   N    2693 2695  -71.29  -70.70   39.43    3.01  -63.50  143.59   19.50\n",
      "   25        335L 335L N   CA   2695 2696  102.18  141.60                  -41.20\n",
      "   26  10565 336D 337D C   N    2709 2711  -68.78  -70.90    4.03    0.14   54.50  162.57   10.82\n",
      "   26        337D 337D N   CA   2711 2712  146.88  150.30                   40.90\n",
      "   27  10583 354F 355Q C   N    2864 2866  -75.74  -73.00    5.34    0.31  -63.80  174.83   26.34\n",
      "   27        355Q 355Q N   CA   2866 2867  145.28  140.70                  -40.30\n",
      "   28  10585 356P 357G C   N    2880 2882   55.61   78.70   30.70    0.77   82.20  156.65    8.92\n",
      "   28        357G 357G N   CA   2882 2883 -145.87 -166.10                    8.50\n",
      "   29  10586 357G 358Y C   N    2884 2886 -142.65 -124.30   46.90    1.88  -63.50  159.13   29.04\n",
      "   29        358Y 358Y N   CA   2886 2887  178.56  135.40                  -43.40\n",
      "   30  10587 358Y 359R C   N    2896 2898 -118.82 -125.20   10.76    0.60  -63.00  178.59   27.98\n",
      "   30        359R 359R N   CA   2898 2899  149.26  140.60                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:   651 1321   2.029\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2769 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     1    0    0    2    3    7   14   51  119  186  213  293  366  408  474\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "4xoy_MODLL.B99990001.pdb       308.94391\n",
      "\n",
      "Modelando proteína 4xoz\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    37140    34494\n",
      "221 (of 2915 total) atoms selected for optimization\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     221\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    34494    1860\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      401\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         156.9981\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     227       0      0   0.004   0.004     0.81067       1.000\n",
      " 2 Bond angle potential               :     313       0      0   1.800   1.800      18.768       1.000\n",
      " 3 Stereochemical cosine torsion poten:     135       0      5  50.513  50.513      55.934       1.000\n",
      " 4 Stereochemical improper torsion pot:      89       0      0   1.030   1.030      2.0397       1.000\n",
      " 5 Soft-sphere overlap restraints     :     401       0      0   0.002   0.002     0.14268       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     295       0      0   0.152   0.152      5.7342       1.000\n",
      "10 Distance restraints 2 (N-O)        :     278       0      0   0.270   0.270      9.0967       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      33       0      0   3.719   3.719      3.6177       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      21       0      0  67.212  67.212      1.5763       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      17       0      0  58.484  58.484      6.9806       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       8       0      0  91.580  91.580      4.7196       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       2       0      0  84.385  84.385      1.9899       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     255       0      0   0.637   0.637      2.6095       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      35      15      4  29.588 114.105      37.058       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     152       0      0   0.826   0.826      5.9204       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 4xoz_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    4622.9917\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9  -73.84  -68.20   19.65    1.39  -62.50  155.39   26.04\n",
      "    1          2A   2A N   CA      9   10  164.12  145.30                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14  -64.21  -68.20    8.18    0.78  -62.50  166.67   27.40\n",
      "    2          3A   3A N   CA     14   15  152.44  145.30                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19 -104.23 -134.00   30.11    0.84  -62.50 -178.56   31.93\n",
      "    3          4A   4A N   CA     19   20  142.52  147.00                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24  -77.72  -68.20   12.20    0.78  -62.50  166.87   28.12\n",
      "    4          5A   5A N   CA     24   25  152.93  145.30                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29  -78.16  -68.20   15.26    0.96  -62.50  162.99   27.51\n",
      "    5          6A   6A N   CA     29   30  156.87  145.30                  -40.90\n",
      "    6  10235   6A   7A C   N      32   34 -114.68 -134.00   20.21    0.90  -62.50  174.17   31.32\n",
      "    6          7A   7A N   CA     34   35  152.93  147.00                  -40.90\n",
      "    7  10236   7A   8G C   N      37   39  -71.20  -80.20   16.04    0.40   82.20 -143.82    6.79\n",
      "    7          8G   8G N   CA     39   40  160.82  174.10                    8.50\n",
      "    8  10237   8G   9A C   N      41   43 -113.14 -134.00   20.93    0.76  -62.50  177.77   31.82\n",
      "    8          9A   9A N   CA     43   44  148.70  147.00                  -40.90\n",
      "    9  10389 160T 161C C   N    1283 1285  157.26   57.40  105.69   10.45 -117.90  163.44    7.26\n",
      "    9        161C 161C N   CA   1285 1286    1.40   36.00                  141.10\n",
      "   10  10561 332D 333M C   N    2676 2678  -90.29  -73.00   22.71    1.81  -63.40  170.91   23.55\n",
      "   10        333M 333M N   CA   2678 2679  128.28  143.00                  -40.50\n",
      "   11  10563 334E 335L C   N    2693 2695  -95.21 -108.50   16.00    0.71  -63.50  167.82   21.69\n",
      "   11        335L 335L N   CA   2695 2696  123.60  132.50                  -41.20\n",
      "   12  10565 336D 337D C   N    2709 2711  -72.65  -96.50   27.97    1.16   54.50  140.04   12.43\n",
      "   12        337D 337D N   CA   2711 2712   99.57  114.20                   40.90\n",
      "   13  10585 356P 357G C   N    2880 2882 -142.54 -167.20   68.06    2.24   82.20  169.81   13.24\n",
      "   13        357G 357G N   CA   2882 2883  111.16  174.60                    8.50\n",
      "   14  10586 357G 358Y C   N    2884 2886 -120.46 -124.30   19.55    0.88  -63.50  169.49   24.30\n",
      "   14        358Y 358Y N   CA   2886 2887  116.23  135.40                  -43.40\n",
      "   15  10587 358Y 359R C   N    2896 2898  -98.61  -72.10   31.92    2.09  -63.00  163.14   24.50\n",
      "   15        359R 359R N   CA   2898 2899  159.69  141.90                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:   891  924   2.072\n",
      "  serious non-bonded atom clash:   891  925   1.993\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     2    0    0    1    0    4   16   47  107  157  205  237  346  413  438\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "4xoz_MODLL.B99990001.pdb       156.99809\n",
      "\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2738 atoms and 1 coordinate set(s) were parsed in 0.04s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 4xp0\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    36981    34335\n",
      "333 (of 2915 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2753 atoms and 1 coordinate set(s) were parsed in 0.03s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     333\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    34335    3156\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      859\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         516.9003\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     346       0      2   0.008   0.008      8.5950       1.000\n",
      " 2 Bond angle potential               :     478       1      9   3.013   3.013      79.608       1.000\n",
      " 3 Stereochemical cosine torsion poten:     231       0     11  49.243  49.243      86.425       1.000\n",
      " 4 Stereochemical improper torsion pot:     149       0      1   1.728   1.728      7.7752       1.000\n",
      " 5 Soft-sphere overlap restraints     :     859      10     13   0.041   0.041      170.30       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     586       0      0   0.205   0.205      13.472       1.000\n",
      "10 Distance restraints 2 (N-O)        :     541       0      0   0.425   0.425      38.930       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      48       1      1  26.263  26.263      12.997       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      34       0      0  72.946  72.946      13.512       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      28       0      1  59.080  59.080      11.971       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:      10       0      0  87.177  87.177      6.8422       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       3       0      0  71.463  71.463      2.4728       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     321       0      0   0.337   0.337     0.86427       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      52      23     10  31.557 119.174      53.364       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     329       0      0   0.600   0.600      9.7741       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 4xp0_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    7938.1875\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9  -73.66  -68.20   11.86    0.78  -62.50  163.65   27.38\n",
      "    1          2A   2A N   CA      9   10  155.83  145.30                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14  -67.34  -68.20    8.72    0.73  -62.50  165.19   27.31\n",
      "    2          3A   3A N   CA     14   15  153.98  145.30                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19  -69.20  -68.20    6.11    0.46  -62.50  167.91   27.85\n",
      "    3          4A   4A N   CA     19   20  151.33  145.30                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24  -74.44  -68.20   10.46    0.66  -62.50  165.84   27.78\n",
      "    4          5A   5A N   CA     24   25  153.69  145.30                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29  -74.85  -68.20   10.41    0.66  -62.50  166.25   27.87\n",
      "    5          6A   6A N   CA     29   30  153.31  145.30                  -40.90\n",
      "    6  10235   6A   7A C   N      32   34 -133.47 -134.00   10.89    0.63  -62.50  176.16   32.67\n",
      "    6          7A   7A N   CA     34   35  157.87  147.00                  -40.90\n",
      "    7  10236   7A   8G C   N      37   39 -110.57  -80.20   30.97    2.21   82.20 -128.89   17.80\n",
      "    7          8G   8G N   CA     39   40  168.03  174.10                    8.50\n",
      "    8  10237   8G   9A C   N      41   43 -112.60 -134.00   23.27    1.12  -62.50  170.50   30.61\n",
      "    8          9A   9A N   CA     43   44  156.13  147.00                  -40.90\n",
      "    9  10238   9A  10G C   N      46   48   92.64   78.70   14.66    0.60  -80.20  173.51   11.98\n",
      "    9         10G  10G N   CA     48   49 -170.66 -166.10                  174.10\n",
      "   10  10356 127C 128Y C   N    1006 1008  -71.46  -63.50   23.65    3.39   55.90  141.05   17.41\n",
      "   10        128Y 128Y N   CA   1008 1009  -21.13  -43.40                   39.50\n",
      "   11  10389 160T 161C C   N    1283 1285   74.81   57.40   41.33    1.32 -117.90 -140.19    9.61\n",
      "   11        161C 161C N   CA   1285 1286   -1.49   36.00                  141.10\n",
      "   12  10481 252L 253N C   N    2013 2015  -66.21 -119.90   58.59    2.95  -63.20  158.46   19.98\n",
      "   12        253N 253N N   CA   2015 2016  160.47  137.00                  -41.10\n",
      "   13  10482 253N 254C C   N    2021 2023   73.48   57.40   46.23    1.49 -117.90 -135.34    9.85\n",
      "   13        254C 254C N   CA   2023 2024   -7.35   36.00                  141.10\n",
      "   14  10558 329F 330K C   N    2648 2650 -125.76 -118.00   32.44    1.45  -62.90  161.35   24.71\n",
      "   14        330K 330K N   CA   2650 2651  170.60  139.10                  -40.80\n",
      "   15  10559 330K 331F C   N    2657 2659  -97.65  -71.40   26.73    1.97  -63.20 -176.78   23.23\n",
      "   15        331F 331F N   CA   2659 2660  135.65  140.70                  -44.30\n",
      "   16  10561 332D 333M C   N    2676 2678 -127.51 -125.60    7.02    0.37  -63.40 -174.34   23.64\n",
      "   16        333M 333M N   CA   2678 2679  133.74  140.50                  -40.50\n",
      "   17  10562 333M 334E C   N    2684 2686  -77.28  -69.30    8.06    0.65  -63.60  178.86   24.61\n",
      "   17        334E 334E N   CA   2686 2687  141.36  142.50                  -40.30\n",
      "   18  10563 334E 335L C   N    2693 2695 -108.01 -108.50   19.71    1.06  -63.50  160.30   20.18\n",
      "   18        335L 335L N   CA   2695 2696  112.80  132.50                  -41.20\n",
      "   19  10564 335L 336D C   N    2701 2703 -128.78  -63.30   87.24   14.66  -63.30   87.24   14.66\n",
      "   19        336D 336D N   CA   2703 2704  -97.65  -40.00                  -40.00\n",
      "   20  10565 336D 337D C   N    2709 2711 -109.40  -96.50   69.89    2.92   54.50  163.96   19.05\n",
      "   20        337D 337D N   CA   2711 2712   45.51  114.20                   40.90\n",
      "   21  10585 356P 357G C   N    2880 2882 -143.30 -167.20   45.35    1.29   82.20 -174.64   14.28\n",
      "   21        357G 357G N   CA   2882 2883  136.06  174.60                    8.50\n",
      "   22  10586 357G 358Y C   N    2884 2886 -123.82 -124.30   18.04    0.93  -63.50  171.71   24.52\n",
      "   22        358Y 358Y N   CA   2886 2887  117.37  135.40                  -43.40\n",
      "   23  10587 358Y 359R C   N    2896 2898 -165.17 -125.20   41.31    1.40  -63.00 -163.52   33.38\n",
      "   23        359R 359R N   CA   2898 2899  151.07  140.60                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:   132 2905   2.291\n",
      "  serious non-bonded atom clash:   651 1321   1.951\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     1    0    1    2    3   11   18   53  146  207  212  279  375  447  465\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "4xp0_MODLL.B99990001.pdb       516.90027\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 4xp2\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    37758    35112\n",
      "286 (of 2915 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2774 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     286\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    35112    2493\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      606\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         165.2900\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     295       0      0   0.004   0.004      1.1222       1.000\n",
      " 2 Bond angle potential               :     406       0      0   1.797   1.797      25.259       1.000\n",
      " 3 Stereochemical cosine torsion poten:     182       0      4  49.727  49.727      66.003       1.000\n",
      " 4 Stereochemical improper torsion pot:     117       0      0   1.326   1.326      4.7408       1.000\n",
      " 5 Soft-sphere overlap restraints     :     606       0      0   0.002   0.002     0.26377       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     415       0      0   0.112   0.112      3.5659       1.000\n",
      "10 Distance restraints 2 (N-O)        :     417       0      0   0.152   0.152      5.4524       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      42       0      0   3.935   3.935      5.7408       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      29       0      0  72.137  72.137      4.5966       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      24       0      0  74.572  74.572      9.2633       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:      10       0      0  73.161  73.161      4.3216       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       3       0      0 125.370 125.370      3.2884       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     309       0      0   0.574   0.574      2.0651       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      45      18      5  24.664 110.338      18.901       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     199       0      0   0.832   0.832      10.707       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 4xp2_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    5996.6460\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9  -62.71  -68.20    5.50    0.48  -62.50  173.46   28.44\n",
      "    1          2A   2A N   CA      9   10  145.65  145.30                  -40.90\n",
      "    2  10232   3A   4A C   N      17   19 -125.43 -134.00    9.10    0.42  -62.50 -179.62   32.93\n",
      "    2          4A   4A N   CA     19   20  150.06  147.00                  -40.90\n",
      "    3  10233   4A   5A C   N      22   24  -66.37  -68.20    5.36    0.49  -62.50  168.81   27.85\n",
      "    3          5A   5A N   CA     24   25  150.34  145.30                  -40.90\n",
      "    4  10234   5A   6A C   N      27   29  -67.84  -68.20    6.52    0.54  -62.50  167.38   27.69\n",
      "    4          6A   6A N   CA     29   30  151.81  145.30                  -40.90\n",
      "    5  10235   6A   7A C   N      32   34 -160.10 -134.00   40.63    1.36  -62.50  171.45   33.25\n",
      "    5          7A   7A N   CA     34   35  178.14  147.00                  -40.90\n",
      "    6  10236   7A   8G C   N      37   39  107.32   78.70   53.71    0.84   82.20  142.18    8.14\n",
      "    6          8G   8G N   CA     39   40  148.45 -166.10                    8.50\n",
      "    7  10237   8G   9A C   N      41   43  -72.08  -68.20    4.63    0.31  -62.50  171.54   28.59\n",
      "    7          9A   9A N   CA     43   44  147.83  145.30                  -40.90\n",
      "    8  10389 160T 161C C   N    1283 1285   62.90   57.40    8.05    0.96 -117.90 -155.16    8.59\n",
      "    8        161C 161C N   CA   1285 1286   41.88   36.00                  141.10\n",
      "    9  10482 253N 254C C   N    2021 2023  -60.95  -63.00    2.13    0.38 -117.90 -172.76    8.07\n",
      "    9        254C 254C N   CA   2023 2024  -40.53  -41.10                  141.10\n",
      "   10  10560 331F 332D C   N    2668 2670  -85.76  -70.90   42.02    1.47  -63.30  132.32   17.76\n",
      "   10        332D 332D N   CA   2670 2671 -170.40  150.30                  -40.00\n",
      "   11  10561 332D 333M C   N    2676 2678  -69.64  -73.00   10.15    0.75  -63.40  167.04   25.43\n",
      "   11        333M 333M N   CA   2678 2679  152.58  143.00                  -40.50\n",
      "   12  10563 334E 335L C   N    2693 2695  -72.41  -70.70   15.76    1.26  -63.50  167.37   22.74\n",
      "   12        335L 335L N   CA   2695 2696  125.93  141.60                  -41.20\n",
      "   13  10565 336D 337D C   N    2709 2711  -97.00  -70.90   35.64    1.49  -63.30  149.28   20.59\n",
      "   13        337D 337D N   CA   2711 2712  174.57  150.30                  -40.00\n",
      "   14  10566 337D 338L C   N    2717 2719 -148.10 -108.50   43.95    1.94  -63.50 -172.60   31.25\n",
      "   14        338L 338L N   CA   2719 2720  151.58  132.50                  -41.20\n",
      "   15  10583 354F 355Q C   N    2864 2866  -63.86  -73.00   16.96    0.97  -63.80  166.72   24.51\n",
      "   15        355Q 355Q N   CA   2866 2867  126.42  140.70                  -40.30\n",
      "   16  10585 356P 357G C   N    2880 2882 -145.97 -167.20   49.93    1.56   82.20  178.87   13.81\n",
      "   16        357G 357G N   CA   2882 2883  129.41  174.60                    8.50\n",
      "   17  10586 357G 358Y C   N    2884 2886 -135.61 -124.30   21.31    0.84  -63.50  178.36   31.87\n",
      "   17        358Y 358Y N   CA   2886 2887  153.46  135.40                  -43.40\n",
      "   18  10587 358Y 359R C   N    2896 2898  -79.05  -72.10    6.95    0.52  -63.00  177.40   25.06\n",
      "   18        359R 359R N   CA   2898 2899  142.23  141.90                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    0    5   22   45  131  187  216  264  373  467  465\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "4xp2_MODLL.B99990001.pdb       165.29004\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 4xp3\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    37761    35115\n",
      "266 (of 2915 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2819 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     266\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    35115    2355\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      491\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         186.2657\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     274       0      0   0.004   0.004      1.1799       1.000\n",
      " 2 Bond angle potential               :     379       0      0   1.835   1.835      24.116       1.000\n",
      " 3 Stereochemical cosine torsion poten:     167       0     10  53.967  53.967      70.831       1.000\n",
      " 4 Stereochemical improper torsion pot:     107       0      0   1.204   1.204      3.6365       1.000\n",
      " 5 Soft-sphere overlap restraints     :     491       0      0   0.002   0.002     0.13809       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     395       0      0   0.122   0.122      4.1977       1.000\n",
      "10 Distance restraints 2 (N-O)        :     397       0      0   0.270   0.270      7.8077       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      40       0      0   3.971   3.971      5.2656       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      27       0      0  51.056  51.056      3.0719       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      22       0      0  63.088  63.088      7.5629       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       9       0      0  97.271  97.271      5.0862       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       2       0      0  97.498  97.498      2.1558       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     315       0      0   0.386   0.386      1.2364       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      43      18      7  27.375 118.418      42.962       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     178       0      0   0.697   0.697      7.0174       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 4xp3_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    5541.0869\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9  -71.23  -68.20   12.75    0.92  -62.50  161.65   26.93\n",
      "    1          2A   2A N   CA      9   10  157.68  145.30                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14  -71.47  -68.20    3.49    0.33  -62.50  175.25   29.17\n",
      "    2          3A   3A N   CA     14   15  144.08  145.30                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19  -68.32  -68.20    0.24    0.02  -62.50  173.69   28.75\n",
      "    3          4A   4A N   CA     19   20  145.51  145.30                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24 -124.35 -134.00   12.34    0.33  -62.50 -169.86   34.46\n",
      "    4          5A   5A N   CA     24   25  139.30  147.00                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29 -105.04 -134.00   38.78    1.09  -62.50  167.59   25.73\n",
      "    5          6A   6A N   CA     29   30  121.21  147.00                  -40.90\n",
      "    6  10235   6A   7A C   N      32   34  -77.65  -68.20   12.28    0.78  -62.50  166.64   28.08\n",
      "    6          7A   7A N   CA     34   35  153.15  145.30                  -40.90\n",
      "    7  10236   7A   8G C   N      37   39  -87.95  -80.20   10.37    0.36   82.20 -117.71    7.52\n",
      "    7          8G   8G N   CA     39   40 -179.00  174.10                    8.50\n",
      "    8  10237   8G   9A C   N      41   43  -76.38  -68.20   14.36    0.91  -62.50  162.59   27.35\n",
      "    8          9A   9A N   CA     43   44  157.11  145.30                  -40.90\n",
      "    9  10389 160T 161C C   N    1283 1285   74.18   57.40   21.39    1.46 -117.90 -154.56    8.84\n",
      "    9        161C 161C N   CA   1285 1286   22.74   36.00                  141.10\n",
      "   10  10481 252L 253N C   N    2013 2015  -66.48 -119.90   58.56    2.96  -63.20  157.93   19.93\n",
      "   10        253N 253N N   CA   2015 2016  161.01  137.00                  -41.10\n",
      "   11  10482 253N 254C C   N    2021 2023   75.31   57.40   51.01    1.64 -117.90 -133.76    9.94\n",
      "   11        254C 254C N   CA   2023 2024  -11.76   36.00                  141.10\n",
      "   12  10560 331F 332D C   N    2668 2670 -125.39  -96.50   38.36    1.57  -63.30  143.13   15.20\n",
      "   12        332D 332D N   CA   2670 2671   88.96  114.20                  -40.00\n",
      "   13  10561 332D 333M C   N    2676 2678  -82.48  -73.00   20.88    1.58  -63.40  165.99   23.36\n",
      "   13        333M 333M N   CA   2678 2679  124.39  143.00                  -40.50\n",
      "   14  10562 333M 334E C   N    2684 2686 -163.53 -117.80   66.97    2.19  -63.60  167.14   27.65\n",
      "   14        334E 334E N   CA   2686 2687 -174.27  136.80                  -40.30\n",
      "   15  10563 334E 335L C   N    2693 2695 -114.04 -108.50   29.88    1.55  -63.50  164.87   26.01\n",
      "   15        335L 335L N   CA   2695 2696  161.86  132.50                  -41.20\n",
      "   16  10565 336D 337D C   N    2709 2711 -128.15  -96.50   31.65    1.29  -63.30  167.25   17.86\n",
      "   16        337D 337D N   CA   2711 2712  114.17  114.20                  -40.00\n",
      "   17  10586 357G 358Y C   N    2884 2886  -82.48  -98.40   16.66    0.76  -63.50  177.72   27.25\n",
      "   17        358Y 358Y N   CA   2886 2887  133.31  128.40                  -43.40\n",
      "   18  10587 358Y 359R C   N    2896 2898  -80.89  -72.10    8.85    0.65  -63.00  176.81   25.11\n",
      "   18        359R 359R N   CA   2898 2899  143.00  141.90                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    0    5   10   50  132  175  173  288  361  433  496\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "4xp3_MODLL.B99990001.pdb       186.26570\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 4xrj\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    37494    34848\n",
      "191 (of 2915 total) atoms selected for optimization\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:      -92.2108\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     191\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    34848    1789\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      371\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         125.2368\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     196       0      0   0.005   0.005      1.4999       1.000\n",
      " 2 Bond angle potential               :     272       0      0   2.084   2.084      22.973       1.000\n",
      " 3 Stereochemical cosine torsion poten:     118       0      2  50.322  50.322      46.499       1.000\n",
      " 4 Stereochemical improper torsion pot:      77       0      0   1.298   1.298      2.8009       1.000\n",
      " 5 Soft-sphere overlap restraints     :     371       0      0   0.003   0.003     0.27828       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     320       0      0   0.110   0.110      2.8240       1.000\n",
      "10 Distance restraints 2 (N-O)        :     310       0      0   0.276   0.276      6.0440       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      30       1      0  33.014  33.014      3.6706       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      18       0      0  82.668  82.668      2.8606       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      13       0      0  31.333  31.333      2.5232       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       6       0      0  87.977  87.977      3.2394       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       1       0      0 176.032 176.032     0.62251       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     237       0      0   0.541   0.541      1.4167       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      32      11      4  23.311 117.584      24.475       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     159       0      0   0.487   0.487      3.5097       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 4xrj_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    4222.5625\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 13                           : Mainchain Omega dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10747 160T 160T CA  C    1279 1283    0.74 -179.18  179.92   41.82 -179.18  179.92   41.82\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9 -123.76 -134.00   11.41    0.57  -62.50  177.95   32.44\n",
      "    1          2A   2A N   CA      9   10  152.03  147.00                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14  -68.83  -68.20    3.58    0.31  -62.50  177.44   29.39\n",
      "    2          3A   3A N   CA     14   15  141.78  145.30                  -40.90\n",
      "    3  10233   4A   5A C   N      22   24 -145.54 -134.00   12.76    0.29  -62.50 -173.78   34.96\n",
      "    3          5A   5A N   CA     24   25  152.43  147.00                  -40.90\n",
      "    4  10234   5A   6A C   N      27   29  -73.56  -68.20    6.52    0.43  -62.50  170.45   28.49\n",
      "    4          6A   6A N   CA     29   30  149.01  145.30                  -40.90\n",
      "    5  10235   6A   7A C   N      32   34  -94.42  -68.20   38.19    3.76  -62.50  161.61   25.12\n",
      "    5          7A   7A N   CA     34   35  117.53  145.30                  -40.90\n",
      "    6  10236   7A   8G C   N      37   39 -150.17 -167.20   18.87    0.58   82.20 -144.04   15.93\n",
      "    6          8G   8G N   CA     39   40 -177.29  174.60                    8.50\n",
      "    7  10237   8G   9A C   N      41   43 -156.45 -134.00   42.73    1.66  -62.50  165.09   32.02\n",
      "    7          9A   9A N   CA     43   44 -176.65  147.00                  -40.90\n",
      "    8  10238   9A  10G C   N      46   48  -88.46  -80.20   54.26    1.73   82.20 -138.77   17.19\n",
      "    8         10G  10G N   CA     48   49 -132.27  174.10                    8.50\n",
      "    9  10585 356P 357G C   N    2880 2882  -89.22  -80.20    9.02    0.61   82.20 -121.76    7.60\n",
      "    9        357G 357G N   CA   2882 2883  173.94  174.10                    8.50\n",
      "   10  10586 357G 358Y C   N    2884 2886  -89.82  -98.40   13.05    1.06  -63.50 -179.71   29.96\n",
      "   10        358Y 358Y N   CA   2886 2887  138.24  128.40                  -43.40\n",
      "   11  10587 358Y 359R C   N    2896 2898 -113.38 -125.20   16.29    0.89  -63.00  174.53   27.06\n",
      "   11        359R 359R N   CA   2898 2899  151.80  140.60                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:   256  477   2.107\n",
      "  serious non-bonded atom clash:  1476 1868   2.242\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    1    1    1    0    2   14   29  109  174  197  244  330  422  414\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "4xrj_MODLL.B99990001.pdb       125.23676\n",
      "\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2735 atoms and 1 coordinate set(s) were parsed in 0.04s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 4xrl\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    37806    35160\n",
      "259 (of 2915 total) atoms selected for optimization\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     259\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    35160    2165\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      464\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         182.1283\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     268       0      0   0.003   0.003     0.77359       1.000\n",
      " 2 Bond angle potential               :     369       0      0   1.855   1.855      23.433       1.000\n",
      " 3 Stereochemical cosine torsion poten:     173       0      6  51.116  51.116      70.260       1.000\n",
      " 4 Stereochemical improper torsion pot:     110       0      0   1.153   1.153      3.5652       1.000\n",
      " 5 Soft-sphere overlap restraints     :     464       0      0   0.002   0.002     0.30353       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     322       0      0   0.099   0.099      2.6536       1.000\n",
      "10 Distance restraints 2 (N-O)        :     315       0      0   0.281   0.281      12.950       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      37       0      0   2.766   2.766      1.8922       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      25       0      0  74.423  74.423      4.8976       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      21       0      0  76.001  76.001      7.9021       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:      10       0      0  82.364  82.364      4.7819       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       2       0      0  63.215  63.215     0.99042       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     257       0      0   0.552   0.552      1.4839       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      39      22      2  28.964 131.845      37.127       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     217       0      0   0.576   0.576      9.1149       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 4xrl_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    6397.4004\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9  -74.70  -68.20   16.58    1.12  -62.50  159.02   26.68\n",
      "    1          2A   2A N   CA      9   10  160.55  145.30                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14 -104.47 -134.00   29.88    0.83  -62.50 -178.40   31.97\n",
      "    2          3A   3A N   CA     14   15  142.42  147.00                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19  -71.52  -68.20   14.15    1.02  -62.50  160.30   26.72\n",
      "    3          4A   4A N   CA     19   20  159.05  145.30                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24  -69.46  -68.20    5.92    0.43  -62.50  168.16   27.90\n",
      "    4          5A   5A N   CA     24   25  151.09  145.30                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29 -130.98 -134.00   10.97    0.67  -62.50  175.47   32.42\n",
      "    5          6A   6A N   CA     29   30  157.54  147.00                  -40.90\n",
      "    6  10235   6A   7A C   N      32   34  -64.73  -68.20    3.50    0.31  -62.50  173.37   28.52\n",
      "    6          7A   7A N   CA     34   35  145.75  145.30                  -40.90\n",
      "    7  10236   7A   8G C   N      37   39  -73.22  -80.20   22.93    0.59   82.20 -148.29    6.92\n",
      "    7          8G   8G N   CA     39   40  152.25  174.10                    8.50\n",
      "    8  10237   8G   9A C   N      41   43 -163.35 -134.00   35.63    0.88  -62.50 -177.67   35.22\n",
      "    8          9A   9A N   CA     43   44  167.20  147.00                  -40.90\n",
      "    9  10238   9A  10G C   N      46   48   71.23   78.70   10.11    0.25   82.20  168.15    8.71\n",
      "    9         10G  10G N   CA     48   49 -159.29 -166.10                    8.50\n",
      "   10  10389 160T 161C C   N    1283 1285  156.91   57.40  105.43   10.40 -117.90  163.83    7.28\n",
      "   10        161C 161C N   CA   1285 1286    1.17   36.00                  141.10\n",
      "   11  10558 329F 330K C   N    2648 2650 -126.49 -118.00   34.43    1.76  -62.90  159.74   18.40\n",
      "   11        330K 330K N   CA   2650 2651  105.74  139.10                  -40.80\n",
      "   12  10559 330K 331F C   N    2657 2659 -146.28 -124.20   33.13    0.92  -63.20  169.46   28.79\n",
      "   12        331F 331F N   CA   2659 2660  168.00  143.30                  -44.30\n",
      "   13  10560 331F 332D C   N    2668 2670  -76.38  -70.90   14.31    0.98  -63.30  177.56   21.16\n",
      "   13        332D 332D N   CA   2670 2671  137.08  150.30                  -40.00\n",
      "   14  10561 332D 333M C   N    2676 2678  -85.51  -73.00   15.77    1.26  -63.40  175.29   24.53\n",
      "   14        333M 333M N   CA   2678 2679  133.39  143.00                  -40.50\n",
      "   15  10562 333M 334E C   N    2684 2686 -144.25 -117.80   35.85    1.12  -63.60  178.02   28.23\n",
      "   15        334E 334E N   CA   2686 2687  160.99  136.80                  -40.30\n",
      "   16  10563 334E 335L C   N    2693 2695  -71.19  -70.70    5.35    0.42  -63.50  177.64   24.24\n",
      "   16        335L 335L N   CA   2695 2696  136.27  141.60                  -41.20\n",
      "   17  10564 335L 336D C   N    2701 2703  -79.23  -70.90   16.36    1.21  -63.30  176.93   20.92\n",
      "   17        336D 336D N   CA   2703 2704  136.21  150.30                  -40.00\n",
      "   18  10565 336D 337D C   N    2709 2711 -144.29  -96.50   58.87    2.40   54.50  165.84   21.01\n",
      "   18        337D 337D N   CA   2711 2712   79.83  114.20                   40.90\n",
      "   19  10583 354F 355Q C   N    2864 2866  -79.71  -73.00   14.09    0.81  -63.80  167.37   25.46\n",
      "   19        355Q 355Q N   CA   2866 2867  153.09  140.70                  -40.30\n",
      "   20  10585 356P 357G C   N    2880 2882 -169.75 -167.20   22.92    0.92   82.20  179.48   13.30\n",
      "   20        357G 357G N   CA   2882 2883  151.82  174.60                    8.50\n",
      "   21  10586 357G 358Y C   N    2884 2886 -122.92 -124.30    5.79    0.25  -63.50 -176.92   26.32\n",
      "   21        358Y 358Y N   CA   2886 2887  129.77  135.40                  -43.40\n",
      "   22  10587 358Y 359R C   N    2896 2898  -76.75  -72.10   13.18    0.91  -63.00  165.25   23.26\n",
      "   22        359R 359R N   CA   2898 2899  154.23  141.90                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:   249  409   2.227\n",
      "  serious non-bonded atom clash:   250  414   2.197\n",
      "  serious non-bonded atom clash:   251  411   2.008\n",
      "  serious non-bonded atom clash:   252  414   1.762\n",
      "  serious non-bonded atom clash:   253  414   1.913\n",
      "  serious non-bonded atom clash:   709 2813   2.193\n",
      "  serious non-bonded atom clash:  1480 1488   1.920\n",
      "  serious non-bonded atom clash:  1490 1498   2.058\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     5    2    1    2    5   19   31   57  110  162  233  324  394  408  499\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "4xrl_MODLL.B99990001.pdb       182.12828\n",
      "\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 5634 atoms and 1 coordinate set(s) were parsed in 0.05s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 4zxt\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    38363    35717\n",
      "166 (of 2915 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2707 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     166\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    35717    1566\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      362\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         134.2420\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     170       0      0   0.006   0.006      1.6860       1.000\n",
      " 2 Bond angle potential               :     235       0      1   1.934   1.934      18.036       1.000\n",
      " 3 Stereochemical cosine torsion poten:     101       0      6  54.126  54.126      47.397       1.000\n",
      " 4 Stereochemical improper torsion pot:      67       0      0   1.146   1.146      1.9481       1.000\n",
      " 5 Soft-sphere overlap restraints     :     362       0      0   0.004   0.004     0.78272       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     325       0      0   0.193   0.193      7.6457       1.000\n",
      "10 Distance restraints 2 (N-O)        :     320       0      0   0.421   0.421      11.442       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      26       1      1   6.515   6.515      10.434       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      16       0      0  85.714  85.714      3.7075       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      11       0      0  69.298  69.298      3.9381       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       4       0      0  62.799  62.799      2.6125       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       2       0      0  35.583  35.583      1.3470       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     177       0      0   0.444   0.444     0.80794       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      28       9      2  36.769 110.782      18.816       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      84       0      0   0.718   0.718      3.6414       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 4zxt_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    3691.2085\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 13                           : Mainchain Omega dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10747 160T 160T CA  C    1279 1283 -154.46 -179.18   24.72    5.74 -179.18   24.72    5.74\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9  -73.24  -68.20    6.03    0.40  -62.50  170.83   28.54\n",
      "    1          2A   2A N   CA      9   10  148.61  145.30                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14 -130.21 -134.00    9.33    0.41  -62.50 -168.27   28.97\n",
      "    2          3A   3A N   CA     14   15  138.48  147.00                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19  -72.68  -68.20    4.96    0.48  -62.50  176.21   29.39\n",
      "    3          4A   4A N   CA     19   20  143.18  145.30                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24  -69.70  -68.20    3.47    0.23  -62.50  170.82   28.35\n",
      "    4          5A   5A N   CA     24   25  148.44  145.30                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29  -60.90  -68.20   11.64    0.73  -62.50  177.15   29.11\n",
      "    5          6A   6A N   CA     29   30  136.24  145.30                  -40.90\n",
      "    6  10235   6A   7A C   N      32   34  -64.95  -68.20   14.45    1.05  -62.50  172.14   28.09\n",
      "    6          7A   7A N   CA     34   35  131.22  145.30                  -40.90\n",
      "    7  10390 161C 162D C   N    1289 1291   65.88   54.50   42.58    2.32  -63.30  135.19   22.83\n",
      "    7        162D 162D N   CA   1291 1292   -0.13   40.90                  -40.00\n",
      "    8  10586 357G 358Y C   N    2884 2886 -114.92  -98.40   32.45    2.96  -63.50  152.79   21.90\n",
      "    8        358Y 358Y N   CA   2886 2887  100.47  128.40                  -43.40\n",
      "    9  10587 358Y 359R C   N    2896 2898  -83.87  -72.10   12.11    0.98  -63.00 -178.91   25.89\n",
      "    9        359R 359R N   CA   2898 2899  139.02  141.90                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:    79  232   2.268\n",
      "  serious non-bonded atom clash:    87  411   2.168\n",
      "  serious non-bonded atom clash:    87  412   2.204\n",
      "  serious non-bonded atom clash:    88  411   1.215\n",
      "  serious non-bonded atom clash:    88  412   0.718\n",
      "  serious non-bonded atom clash:    89  409   1.925\n",
      "  serious non-bonded atom clash:    89  410   1.142\n",
      "  serious non-bonded atom clash:    89  411   0.648\n",
      "  serious non-bonded atom clash:    89  412   1.313\n",
      "  serious non-bonded atom clash:    90  409   1.805\n",
      "  serious non-bonded atom clash:    90  410   0.953\n",
      "  serious non-bonded atom clash:    90  411   1.360\n",
      "  serious non-bonded atom clash:    90  413   1.251\n",
      "  serious non-bonded atom clash:    90  414   1.658\n",
      "  serious non-bonded atom clash:    91  409   2.286\n",
      "  serious non-bonded atom clash:    91  410   1.535\n",
      "  serious non-bonded atom clash:    91  411   1.807\n",
      "  serious non-bonded atom clash:    91  412   1.815\n",
      "  serious non-bonded atom clash:   650 1319   2.250\n",
      "  serious non-bonded atom clash:   651 1321   1.880\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:    15    1    4    4    7    7   23   67  156  188  203  274  354  446  483\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "4zxt_MODLL.B99990001.pdb       134.24199\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 4zzm\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    37308    34662\n",
      "164 (of 2915 total) atoms selected for optimization\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     164\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    34662    1390\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      335\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         120.0533\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     169       0      0   0.005   0.005      1.3761       1.000\n",
      " 2 Bond angle potential               :     233       0      0   2.148   2.148      20.609       1.000\n",
      " 3 Stereochemical cosine torsion poten:     106       0      2  49.454  49.454      40.275       1.000\n",
      " 4 Stereochemical improper torsion pot:      69       0      0   1.176   1.176      2.6952       1.000\n",
      " 5 Soft-sphere overlap restraints     :     335       0      0   0.004   0.004     0.54408       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     224       0      0   0.102   0.102      1.9125       1.000\n",
      "10 Distance restraints 2 (N-O)        :     215       0      0   0.366   0.366      11.435       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      25       1      0  36.135  36.135      6.2477       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      14       0      0  55.921  55.921     0.98418       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      10       0      0  76.929  76.929      4.0266       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       6       0      0  59.014  59.014      4.1233       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       1       0      0  98.078  98.078     0.72082       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     156       0      0   0.553   0.553     0.83044       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      26      12      2  25.724 122.181      19.004       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     136       0      0   0.572   0.572      5.2703       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 4zzm_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    4263.1279\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 13                           : Mainchain Omega dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10747 160T 160T CA  C    1279 1283    0.04 -179.18  179.23   41.66 -179.18  179.23   41.66\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9 -168.23 -134.00   36.77    0.84  -62.50 -169.32   36.85\n",
      "    1          2A   2A N   CA      9   10  160.42  147.00                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14  -96.92  -68.20   28.86    2.54  -62.50  179.93   31.29\n",
      "    2          3A   3A N   CA     14   15  142.50  145.30                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19  -55.11  -68.20   19.12    1.20  -62.50  172.42   28.62\n",
      "    3          4A   4A N   CA     19   20  131.36  145.30                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24  -64.79  -68.20   16.61    1.46  -62.50  157.56   25.93\n",
      "    4          5A   5A N   CA     24   25  161.56  145.30                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29  -69.25  -68.20    4.78    0.42  -62.50  178.59   29.60\n",
      "    5          6A   6A N   CA     29   30  140.64  145.30                  -40.90\n",
      "    6  10235   6A   7A C   N      32   34  -88.20  -68.20   25.83    1.65  -62.50  159.54   27.48\n",
      "    6          7A   7A N   CA     34   35  161.65  145.30                  -40.90\n",
      "    7  10237   8G   9A C   N      41   43 -168.41 -134.00   44.63    1.20  -62.50  178.51   34.82\n",
      "    7          9A   9A N   CA     43   44  175.41  147.00                  -40.90\n",
      "    8  10238   9A  10G C   N      46   48 -153.28 -167.20   48.10    1.66   82.20  172.97   13.31\n",
      "    8         10G  10G N   CA     48   49  128.56  174.60                    8.50\n",
      "    9  10583 354F 355Q C   N    2864 2866  -65.40  -73.00   12.86    0.74  -63.80  170.64   25.01\n",
      "    9        355Q 355Q N   CA   2866 2867  130.33  140.70                  -40.30\n",
      "   10  10585 356P 357G C   N    2880 2882 -149.20 -167.20   20.44    0.32   82.20 -157.52   15.19\n",
      "   10        357G 357G N   CA   2882 2883  164.90  174.60                    8.50\n",
      "   11  10586 357G 358Y C   N    2884 2886 -101.13  -98.40    7.48    0.76  -63.50 -177.32   27.15\n",
      "   11        358Y 358Y N   CA   2886 2887  135.36  128.40                  -43.40\n",
      "   12  10587 358Y 359R C   N    2896 2898 -122.89 -125.20    5.79    0.32  -63.00 -176.93   28.86\n",
      "   12        359R 359R N   CA   2898 2899  145.91  140.60                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:   431 2739   1.113\n",
      "  serious non-bonded atom clash:   431 2740   1.566\n",
      "  serious non-bonded atom clash:   432 2738   1.704\n",
      "  serious non-bonded atom clash:   432 2739   0.623\n",
      "  serious non-bonded atom clash:   432 2740   1.746\n",
      "  serious non-bonded atom clash:   433 2739   1.938\n",
      "  serious non-bonded atom clash:   433 2740   1.173\n",
      "  serious non-bonded atom clash:   434 2740   1.030\n",
      "  serious non-bonded atom clash:   440 2740   2.041\n",
      "  serious non-bonded atom clash:   547 2829   2.289\n",
      "  serious non-bonded atom clash:   549 2829   2.148\n",
      "  serious non-bonded atom clash:  1482 1622   2.189\n",
      "  serious non-bonded atom clash:  1482 1623   1.292\n",
      "  serious non-bonded atom clash:  1483 1502   1.934\n",
      "  serious non-bonded atom clash:  1483 1632   2.282\n",
      "  serious non-bonded atom clash:  2664 2677   2.008\n",
      "  serious non-bonded atom clash:  2666 2677   2.218\n",
      "  serious non-bonded atom clash:  2715 2718   2.137\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:    12    3    3    3    5    7    8   57   95  157  176  220  346  479  503\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "4zzm_MODLL.B99990001.pdb       120.05327\n",
      "\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2738 atoms and 1 coordinate set(s) were parsed in 0.03s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 4zzn\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    37601    34955\n",
      "251 (of 2915 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2705 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     251\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    34955    2093\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      457\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         164.2628\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     260       0      0   0.004   0.004      1.0923       1.000\n",
      " 2 Bond angle potential               :     357       0      0   1.689   1.689      19.069       1.000\n",
      " 3 Stereochemical cosine torsion poten:     167       0      5  51.412  51.412      61.241       1.000\n",
      " 4 Stereochemical improper torsion pot:     107       0      0   1.030   1.030      3.2473       1.000\n",
      " 5 Soft-sphere overlap restraints     :     457       0      0   0.002   0.002     0.12801       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     321       0      0   0.117   0.117      3.4335       1.000\n",
      "10 Distance restraints 2 (N-O)        :     310       0      0   0.237   0.237      6.9472       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      36       0      1   3.709   3.709      4.4267       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      24       0      0  67.098  67.098      5.5251       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      20       0      0  72.217  72.217      8.4564       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:      10       0      0 110.746 110.746      6.6373       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       2       0      0 118.367 118.367      1.9094       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     230       0      0   0.461   0.461     0.82864       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      38      21      4  29.975 127.737      37.337       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     211       0      0   0.450   0.450      3.9845       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 4zzn_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    5960.0889\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9  -67.81  -68.20    4.99    0.42  -62.50  168.90   27.94\n",
      "    1          2A   2A N   CA      9   10  150.28  145.30                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14  -69.85  -68.20    1.69    0.13  -62.50  173.56   28.81\n",
      "    2          3A   3A N   CA     14   15  145.70  145.30                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19  -67.31  -68.20    1.93    0.18  -62.50  172.16   28.45\n",
      "    3          4A   4A N   CA     19   20  147.01  145.30                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24 -120.55 -134.00   13.45    0.45  -62.50 -178.47   32.85\n",
      "    4          5A   5A N   CA     24   25  147.10  147.00                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29 -158.14 -134.00   27.22    0.62  -62.50 -173.99   35.58\n",
      "    5          6A   6A N   CA     29   30  159.56  147.00                  -40.90\n",
      "    6  10235   6A   7A C   N      32   34 -101.63 -134.00   32.38    1.11  -62.50  175.64   30.84\n",
      "    6          7A   7A N   CA     34   35  147.88  147.00                  -40.90\n",
      "    7  10236   7A   8G C   N      37   39 -162.86 -167.20   10.16    0.32   82.20 -165.50   14.35\n",
      "    7          8G   8G N   CA     39   40  165.41  174.60                    8.50\n",
      "    8  10237   8G   9A C   N      41   43  -71.47  -68.20    3.75    0.25  -62.50  172.21   28.67\n",
      "    8          9A   9A N   CA     43   44  147.13  145.30                  -40.90\n",
      "    9  10238   9A  10G C   N      46   48  158.86 -167.20   34.53    0.82   82.20  177.17   12.11\n",
      "    9         10G  10G N   CA     48   49  168.23  174.60                    8.50\n",
      "   10  10389 160T 161C C   N    1283 1285  160.54   57.40  111.85   10.48 -117.90  169.30    7.50\n",
      "   10        161C 161C N   CA   1285 1286   -7.25   36.00                  141.10\n",
      "   11  10558 329F 330K C   N    2648 2650  -99.98 -118.00   34.02    1.76  -62.90  155.73   22.41\n",
      "   11        330K 330K N   CA   2650 2651  167.95  139.10                  -40.80\n",
      "   12  10559 330K 331F C   N    2657 2659  -83.53  -71.40   42.59    3.70  -63.20  145.60   18.89\n",
      "   12        331F 331F N   CA   2659 2660   99.88  140.70                  -44.30\n",
      "   13  10560 331F 332D C   N    2668 2670  -89.56  -70.90   29.02    1.08  -63.30  149.81   20.16\n",
      "   13        332D 332D N   CA   2670 2671  172.51  150.30                  -40.00\n",
      "   14  10561 332D 333M C   N    2676 2678  -60.62  -73.00   12.55    0.98  -63.40  174.45   25.83\n",
      "   14        333M 333M N   CA   2678 2679  145.07  143.00                  -40.50\n",
      "   15  10562 333M 334E C   N    2684 2686  -72.41  -69.30    6.89    0.42  -63.60  171.28   23.32\n",
      "   15        334E 334E N   CA   2686 2687  148.65  142.50                  -40.30\n",
      "   16  10563 334E 335L C   N    2693 2695 -106.31  -63.50   77.45    9.54  -63.50   77.45    9.54\n",
      "   16        335L 335L N   CA   2695 2696   23.34  -41.20                  -41.20\n",
      "   17  10564 335L 336D C   N    2701 2703  -67.77  -63.30    4.47    0.72   54.50  146.71   18.95\n",
      "   17        336D 336D N   CA   2703 2704  -40.19  -40.00                   40.90\n",
      "   18  10583 354F 355Q C   N    2864 2866  -74.76  -73.00   11.47    0.72  -63.80  168.02   25.29\n",
      "   18        355Q 355Q N   CA   2866 2867  152.03  140.70                  -40.30\n",
      "   19  10585 356P 357G C   N    2880 2882 -123.01  -80.20   47.02    2.33   82.20 -126.40    6.87\n",
      "   19        357G 357G N   CA   2882 2883 -166.46  174.10                    8.50\n",
      "   20  10586 357G 358Y C   N    2884 2886 -107.86  -98.40   12.13    0.85  -63.50  170.10   24.88\n",
      "   20        358Y 358Y N   CA   2886 2887  120.81  128.40                  -43.40\n",
      "   21  10587 358Y 359R C   N    2896 2898 -125.46 -125.20   19.52    0.94  -63.00  170.63   27.35\n",
      "   21        359R 359R N   CA   2898 2899  160.11  140.60                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    0    0   10   42  138  166  171  228  365  429  499\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "4zzn_MODLL.B99990001.pdb       164.26282\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 4zzo\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    37447    34801\n",
      "244 (of 2915 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2705 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:      -91.3213\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     244\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    34801    2003\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      449\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         161.3030\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     252       0      0   0.004   0.004      1.0359       1.000\n",
      " 2 Bond angle potential               :     345       0      0   1.764   1.764      20.265       1.000\n",
      " 3 Stereochemical cosine torsion poten:     158       0      7  50.280  50.280      60.733       1.000\n",
      " 4 Stereochemical improper torsion pot:     105       0      0   1.022   1.022      2.7697       1.000\n",
      " 5 Soft-sphere overlap restraints     :     449       0      0   0.002   0.002     0.21972       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     316       0      0   0.167   0.167      5.9718       1.000\n",
      "10 Distance restraints 2 (N-O)        :     294       0      0   0.316   0.316      12.262       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      35       0      1   4.090   4.090      5.2372       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      23       0      0  58.733  58.733      3.6037       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      19       0      0  60.246  60.246      8.5677       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       9       0      0  75.851  75.851      3.6251       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       2       0      0  10.602  10.602      1.0585       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     225       0      0   0.500   0.500     0.99550       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      37      18      4  30.321 122.021      33.075       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     183       0      0   0.396   0.396      1.8819       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 4zzo_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    5245.8145\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9  -76.06  -68.20   13.06    1.27  -62.50  176.29   28.26\n",
      "    1          2A   2A N   CA      9   10  134.87  145.30                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14  -68.92  -68.20    8.35    0.70  -62.50  178.00   28.86\n",
      "    2          3A   3A N   CA     14   15  136.98  145.30                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19  -83.23  -68.20   23.06    1.45  -62.50  157.68   26.91\n",
      "    3          4A   4A N   CA     19   20  162.79  145.30                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24  -67.02  -68.20    5.39    0.47  -62.50  168.60   27.85\n",
      "    4          5A   5A N   CA     24   25  150.56  145.30                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29 -115.55 -134.00   18.62    0.53  -62.50 -177.46   32.74\n",
      "    5          6A   6A N   CA     29   30  144.45  147.00                  -40.90\n",
      "    6  10235   6A   7A C   N      32   34  -66.49  -68.20    6.50    0.58  -62.50  167.58   27.66\n",
      "    6          7A   7A N   CA     34   35  151.57  145.30                  -40.90\n",
      "    7  10236   7A   8G C   N      37   39  104.04   78.70   51.38    0.77   82.20  142.39    7.99\n",
      "    7          8G   8G N   CA     39   40  149.20 -166.10                    8.50\n",
      "    8  10237   8G   9A C   N      41   43 -122.54 -134.00   13.68    0.33  -62.50 -170.64   34.23\n",
      "    8          9A   9A N   CA     43   44  139.52  147.00                  -40.90\n",
      "    9  10238   9A  10G C   N      46   48  -77.36  -80.20   25.86    0.88   82.20 -147.79    7.16\n",
      "    9         10G  10G N   CA     48   49  148.40  174.10                    8.50\n",
      "   10  10389 160T 161C C   N    1283 1285  155.40   57.40  104.04   10.22 -117.90  164.72    7.32\n",
      "   10        161C 161C N   CA   1285 1286    1.05   36.00                  141.10\n",
      "   11  10559 330K 331F C   N    2657 2659 -166.16 -124.20   60.52    1.64  -63.20  164.89   29.03\n",
      "   11        331F 331F N   CA   2659 2660 -173.09  143.30                  -44.30\n",
      "   12  10560 331F 332D C   N    2668 2670 -159.61  -96.50   80.70    3.36  -63.30 -177.09   28.72\n",
      "   12        332D 332D N   CA   2670 2671  164.50  114.20                  -40.00\n",
      "   13  10561 332D 333M C   N    2676 2678  -88.59  -73.00   25.40    1.98  -63.40  165.38   22.84\n",
      "   13        333M 333M N   CA   2678 2679  122.95  143.00                  -40.50\n",
      "   14  10563 334E 335L C   N    2693 2695 -114.07 -108.50    7.04    0.39  -63.50  176.78   22.21\n",
      "   14        335L 335L N   CA   2695 2696  128.20  132.50                  -41.20\n",
      "   15  10564 335L 336D C   N    2701 2703  -93.22  -96.50    3.41    0.14   54.50  165.31   14.23\n",
      "   15        336D 336D N   CA   2703 2704  115.12  114.20                   40.90\n",
      "   16  10583 354F 355Q C   N    2864 2866  -80.38  -73.00   10.47    0.81  -63.80  174.36   24.79\n",
      "   16        355Q 355Q N   CA   2866 2867  133.27  140.70                  -40.30\n",
      "   17  10586 357G 358Y C   N    2884 2886 -111.65  -98.40   26.55    2.44  -63.50  156.39   22.58\n",
      "   17        358Y 358Y N   CA   2886 2887  105.39  128.40                  -43.40\n",
      "   18  10587 358Y 359R C   N    2896 2898  -83.46  -72.10   11.56    0.93  -63.00 -179.71   25.76\n",
      "   18        359R 359R N   CA   2898 2899  139.78  141.90                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:   891  925   2.294\n",
      "  serious non-bonded atom clash:  1482 1622   2.093\n",
      "  serious non-bonded atom clash:  1482 1623   1.162\n",
      "  serious non-bonded atom clash:  1483 1502   2.194\n",
      "  serious non-bonded atom clash:  2724 2758   2.300\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     2    1    2    2    7    4   17   58  124  153  173  233  393  418  491\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "4zzo_MODLL.B99990001.pdb       161.30301\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 5ax3\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "    174     1  165   174      A     D   14.382\n",
      "    203     1  191   194      K     G    9.058\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    35758    33112\n",
      "350 (of 2915 total) atoms selected for optimization\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:       90.1761\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     350\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    33112    2626\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):     1534\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :        9607.9854\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     362      37     65   0.139   0.139      2068.0       1.000\n",
      " 2 Bond angle potential               :     498      73    117  13.887  13.887      1742.5       1.000\n",
      " 3 Stereochemical cosine torsion poten:     239       0     22  49.962  49.962      141.09       1.000\n",
      " 4 Stereochemical improper torsion pot:     162      13     27   8.808   8.808      297.45       1.000\n",
      " 5 Soft-sphere overlap restraints     :    1534     114    178   0.139   0.139      3495.0       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     431      20     59   1.430   1.430      571.17       1.000\n",
      "10 Distance restraints 2 (N-O)        :     400      30     67   1.745   1.745      609.81       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      50      11     16  31.592  31.592      206.74       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      30       0      8  94.601  94.601      47.150       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      24       0      3  96.462  96.462      23.350       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       9       0      0  80.680  80.680      7.3159       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       4       0      1  66.079  66.079      7.3909       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     229       0      0   0.920   0.920      4.8032       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      53      30     26  66.946 132.843      348.69       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     135       0      1   2.769   2.769      37.530       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 5ax3_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:   18776.6484\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  1                           : Bond length potential                   \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   1457 179D 180H C   N    1428 1430    1.53    1.35    0.18    6.50    1.35    0.18    6.50\n",
      "    2   1464 180H 180H N   CA   1430 1431    1.65    1.43    0.22    7.40    1.43    0.22    7.40\n",
      "    3   1466 180H 180H C   CA   1438 1431    1.69    1.49    0.20    5.83    1.49    0.20    5.83\n",
      "    4   1490 183F 183F C   CA   1460 1452    1.76    1.49    0.27    7.82    1.49    0.27    7.82\n",
      "    5   1491 183F 184L C   N    1460 1462    1.76    1.35    0.42   14.89    1.35    0.42   14.89\n",
      "    6   1496 184L 184L N   CA   1462 1463    2.14    1.43    0.71   23.25    1.43    0.71   23.25\n",
      "    7   1498 184L 184L C   CA   1468 1463    1.70    1.49    0.21    6.24    1.49    0.21    6.24\n",
      "    8   1499 184L 185T C   N    1468 1470    1.75    1.35    0.40   14.35    1.35    0.40   14.35\n",
      "    9   1503 185T 185T N   CA   1470 1471    1.96    1.43    0.53   17.28    1.43    0.53   17.28\n",
      "   10   1505 185T 185T C   CA   1475 1471    2.12    1.49    0.63   18.22    1.49    0.63   18.22\n",
      "   11   1506 185T 186E C   N    1475 1477    1.55    1.35    0.21    7.28    1.35    0.21    7.28\n",
      "   12   1512 186E 186E N   CA   1477 1478    1.65    1.43    0.22    7.28    1.43    0.22    7.28\n",
      "   13   1560 192W 192W CG  CB   1531 1530    1.85    1.51    0.34    9.58    1.51    0.34    9.58\n",
      "   14   1570 192W 192W C   CA   1540 1529    1.90    1.49    0.41   12.06    1.49    0.41   12.06\n",
      "   15   1571 192W 193Y C   N    1540 1542    1.76    1.35    0.42   14.87    1.35    0.42   14.87\n",
      "   16   1575 193Y 193Y CB  CA   1544 1543    1.73    1.54    0.19    5.31    1.54    0.19    5.31\n",
      "   17   1576 193Y 193Y CG  CB   1545 1544    1.83    1.49    0.34    9.45    1.49    0.34    9.45\n",
      "   18   1584 193Y 193Y N   CA   1542 1543    1.94    1.43    0.51   16.66    1.43    0.51   16.66\n",
      "   19   1586 193Y 193Y C   CA   1552 1543    2.50    1.49    1.01   29.31    1.49    1.01   29.31\n",
      "   20   1587 193Y 194R C   N    1552 1554    1.85    1.35    0.51   18.05    1.35    0.51   18.05\n",
      "   21   1588 194R 194R CB  CA   1556 1555    1.80    1.54    0.26    7.17    1.54    0.26    7.17\n",
      "   22   1589 194R 194R CG  CB   1557 1556    1.80    1.53    0.27    7.49    1.53    0.27    7.49\n",
      "   23   1595 194R 194R N   CA   1554 1555    2.07    1.43    0.64   21.07    1.43    0.64   21.07\n",
      "   24   1597 194R 194R C   CA   1563 1555    1.84    1.49    0.35   10.14    1.49    0.35   10.14\n",
      "   25   1600 195A 195A N   CA   1565 1566    1.57    1.43    0.14    4.68    1.43    0.14    4.68\n",
      "   26   2066 253N 253N C   CA   2021 2016    1.75    1.49    0.26    7.63    1.49    0.26    7.63\n",
      "   27   2067 253N 254C C   N    2021 2023    1.92    1.35    0.57   20.32    1.35    0.57   20.32\n",
      "   28   2068 254C 254C CB  CA   2025 2024    1.88    1.54    0.34    9.46    1.54    0.34    9.46\n",
      "   29   2069 254C 254C SG  CB   2026 2025    2.13    1.82    0.31    8.10    1.82    0.31    8.10\n",
      "   30   2070 254C 254C N   CA   2023 2024    2.36    1.43    0.93   30.46    1.43    0.93   30.46\n",
      "   31   2072 254C 254C C   CA   2027 2024    1.73    1.49    0.24    7.00    1.49    0.24    7.00\n",
      "   32   2073 254C 255I C   N    2027 2029    1.55    1.35    0.21    7.35    1.35    0.21    7.35\n",
      "   33   2078 255I 255I N   CA   2029 2030    1.57    1.43    0.14    4.70    1.43    0.14    4.70\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   4959 180H 180H N   CA   1430 1431  123.15  107.00   16.15    4.64  107.00   16.15    4.64\n",
      "    2   4964 180H 180H CG  CB   1433 1434  133.03  113.00   20.03    4.92  113.00   20.03    4.92\n",
      "    3   4986 182G 183F C   N    1449 1451  142.74  120.00   22.74    5.17  120.00   22.74    5.17\n",
      "    4   4990 183F 183F CA  CB   1452 1453  128.95  107.50   21.46    4.96  107.50   21.46    4.96\n",
      "    5   4992 183F 183F CA  C    1452 1460  162.70  116.50   46.20   13.28  116.50   46.20   13.28\n",
      "    6   5002 183F 184L C   N    1460 1462  152.39  120.00   32.39    7.36  120.00   32.39    7.36\n",
      "    7   5004 184L 184L N   CA   1462 1463  144.13  113.50   30.63    8.24  113.50   30.63    8.24\n",
      "    8   5005 184L 184L N   CA   1462 1463  152.05  107.00   45.05   12.95  107.00   45.05   12.95\n",
      "    9   5006 184L 184L CA  CB   1463 1464  150.51  113.50   37.01    9.09  113.50   37.01    9.09\n",
      "   10   5008 184L 184L CA  C    1463 1468  150.58  116.50   34.08    9.80  116.50   34.08    9.80\n",
      "   11   5013 184L 185T C   N    1468 1470  153.51  120.00   33.51    7.62  120.00   33.51    7.62\n",
      "   12   5016 185T 185T N   CA   1470 1471  138.06  107.00   31.06    8.93  107.00   31.06    8.93\n",
      "   13   5017 185T 185T CA  CB   1471 1472  139.10  110.10   29.00    8.11  110.10   29.00    8.11\n",
      "   14   5023 185T 186E C   N    1475 1477  153.14  120.00   33.14    7.53  120.00   33.14    7.53\n",
      "   15   5025 186E 186E N   CA   1477 1478  133.34  113.50   19.85    5.34  113.50   19.85    5.34\n",
      "   16   5026 186E 186E N   CA   1477 1478  125.48  107.00   18.48    5.31  107.00   18.48    5.31\n",
      "   17   5027 186E 186E CA  CB   1478 1479  154.85  113.50   41.35   10.15  113.50   41.35   10.15\n",
      "   18   5030 186E 186E CB  CG   1479 1480  130.91  108.00   22.91    5.31  108.00   22.91    5.31\n",
      "   19   5070 189A 190T C   N    1508 1510  141.37  120.00   21.37    4.86  120.00   21.37    4.86\n",
      "   20   5097 192W 192W N   CA   1528 1529  127.63  107.00   20.63    5.93  107.00   20.63    5.93\n",
      "   21   5098 192W 192W CA  CB   1529 1530  148.79  114.00   34.79    8.54  114.00   34.79    8.54\n",
      "   22   5100 192W 192W CA  C    1529 1540  138.53  116.50   22.03    6.33  116.50   22.03    6.33\n",
      "   23   5116 192W 192W C   CA   1540 1529  136.05  108.00   28.05    6.50  108.00   28.05    6.50\n",
      "   24   5117 192W 193Y C   N    1540 1542  166.91  120.00   46.91   10.66  120.00   46.91   10.66\n",
      "   25   5120 193Y 193Y N   CA   1542 1543  144.26  107.00   37.26   10.71  107.00   37.26   10.71\n",
      "   26   5121 193Y 193Y CA  CB   1543 1544  134.46  107.50   26.96    6.23  107.50   26.96    6.23\n",
      "   27   5123 193Y 193Y CA  C    1543 1552  148.88  116.50   32.38    9.31  116.50   32.38    9.31\n",
      "   28   5124 193Y 193Y CB  CG   1544 1545  145.86  122.30   23.56    5.12  122.30   23.56    5.12\n",
      "   29   5126 193Y 193Y CG  CD1  1545 1546  179.84  120.00   59.84   12.16  120.00   59.84   12.16\n",
      "   30   5130 193Y 193Y CD2 CE2  1547 1549  159.88  120.00   39.88    8.10  120.00   39.88    8.10\n",
      "   31   5131 193Y 193Y CE1 CZ   1548 1550  151.01  120.00   31.01    6.70  120.00   31.01    6.70\n",
      "   32   5135 193Y 194R C   N    1552 1554  153.06  120.00   33.06    7.51  120.00   33.06    7.51\n",
      "   33   5138 194R 194R N   CA   1554 1555  140.21  107.00   33.21    9.55  107.00   33.21    9.55\n",
      "   34   5139 194R 194R CA  CB   1555 1556  141.82  113.50   28.33    6.95  113.50   28.33    6.95\n",
      "   35   5141 194R 194R CA  C    1555 1563  139.12  116.50   22.62    6.51  116.50   22.62    6.51\n",
      "   36   5142 194R 194R CB  CG   1556 1557  133.46  113.60   19.86    4.88  113.60   19.86    4.88\n",
      "   37   5149 194R 195A C   N    1563 1565  143.67  120.00   23.67    5.38  120.00   23.67    5.38\n",
      "   38   5151 195A 195A N   CA   1565 1566  136.14  113.50   22.64    6.09  113.50   22.64    6.09\n",
      "   39   5156 195A 196P C   N    1568 1570  139.58  117.00   22.59    5.62  117.00   22.59    5.62\n",
      "   40   5768 252L 252L CA  CB   2008 2009  132.13  113.50   18.63    4.57  113.50   18.63    4.57\n",
      "   41   5778 253N 253N N   CA   2015 2016  132.36  107.00   25.36    7.29  107.00   25.36    7.29\n",
      "   42   5779 253N 253N CA  CB   2016 2017  138.52  108.00   30.52    7.07  108.00   30.52    7.07\n",
      "   43   5781 253N 253N CA  C    2016 2021  142.65  116.50   26.15    7.52  116.50   26.15    7.52\n",
      "   44   5786 253N 254C C   N    2021 2023  143.83  120.00   23.83    5.41  120.00   23.83    5.41\n",
      "   45   5788 254C 254C N   CA   2023 2024  149.59  113.50   36.09    9.71  113.50   36.09    9.71\n",
      "   46   5789 254C 254C N   CA   2023 2024  125.23  107.00   18.23    5.24  107.00   18.23    5.24\n",
      "   47   5790 254C 254C CA  CB   2024 2025  142.00  112.50   29.50    7.22  112.50   29.50    7.22\n",
      "   48   5794 254C 255I C   N    2027 2029  154.59  120.00   34.59    7.86  120.00   34.59    7.86\n",
      "   49   5808 256I 256I N   CA   2037 2038  127.38  107.00   20.38    5.86  107.00   20.38    5.86\n",
      "   50   5819 257N 257N N   CA   2045 2046  126.88  107.00   19.88    5.72  107.00   19.88    5.72\n",
      "   51   5827 257N 258L C   N    2051 2053  143.33  120.00   23.33    5.30  120.00   23.33    5.30\n",
      "   52   5830 258L 258L N   CA   2053 2054  139.67  107.00   32.67    9.39  107.00   32.67    9.39\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  4                           : Stereochemical improper torsion potentia\n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   9636 191R 191R C   CA   1526 1518   13.67    0.00   13.67    4.81    0.00   13.67    4.81\n",
      "    2   9651 193Y 193Y CG  CD1  1545 1546   16.97    0.00   16.97    5.17    0.00   16.97    5.17\n",
      "    3   9653 193Y 193Y CE1 CZ   1548 1550   30.87    0.00   30.87    9.40    0.00   30.87    9.40\n",
      "    4   9654 193Y 193Y CZ  CE2  1550 1549   45.65    0.00   45.65   13.91    0.00   45.65   13.91\n",
      "    5   9657 193Y 193Y CD2 CE2  1547 1549 -164.45 -180.00   15.55    6.11 -180.00   15.55    6.11\n",
      "    6   9658 194R 194R C   CA   1563 1555   12.80    0.00   12.80    4.50    0.00   12.80    4.50\n",
      "    7   9661 195A 195A C   CA   1568 1566   22.44    0.00   22.44    7.90    0.00   22.44    7.90\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  9                           : Distance restraints 1 (CA-CA)           \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  15544 149D 194R CA  CA   1194 1555   11.83    9.20    2.63    5.13    9.20    2.63    5.13\n",
      "    2  15699 153S 194R CA  CA   1226 1555   15.02   12.33    2.69    4.83   12.33    2.69    4.83\n",
      "    3  15868 170L 194R CA  CA   1354 1555   17.96   13.85    4.11    6.79   13.85    4.11    6.79\n",
      "    4  16072 194R 230G CA  CA   1555 1837   10.12    7.52    2.60    5.49    7.52    2.60    5.49\n",
      "    5  16073 194R 231K CA  CA   1555 1841    9.97    4.64    5.34   12.48    4.64    5.34   12.48\n",
      "    6  16074 194R 232H CA  CA   1555 1850   11.43    7.16    4.27    9.16    7.16    4.27    9.16\n",
      "    7  17393 258L 266S CA  CA   2054 2123   16.13   12.72    3.41    4.55   12.72    3.41    4.55\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 10                           : Distance restraints 2 (N-O)             \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  21582 194R 149D N   O    1554 1200   13.91   10.23    3.69    6.16   10.23    3.69    6.16\n",
      "    2  21584 194R 176P N   O    1554 1403   16.69   10.81    5.88    5.44   10.81    5.88    5.44\n",
      "    3  21585 194R 177D N   O    1554 1411   18.68   10.71    7.96    5.41   10.71    7.96    5.41\n",
      "    4  21586 194R 178H N   O    1554 1421   18.43   10.07    8.36    4.68   10.07    8.36    4.68\n",
      "    5  21597 194R 230G N   O    1554 1839    9.94    5.81    4.13    8.11    5.81    4.13    8.11\n",
      "    6  21598 194R 231K N   O    1554 1848   13.07    5.28    7.79   15.35    5.28    7.79   15.35\n",
      "    7  21599 194R 232H N   O    1554 1858   12.08    8.85    3.23    5.94    8.85    3.23    5.94\n",
      "    8  21620 195A 231K N   O    1565 1848    9.13    6.43    2.70    5.16    6.43    2.70    5.16\n",
      "    9  21799 203K 194R N   O    1624 1564   13.81   10.10    3.72    5.62   10.10    3.72    5.62\n",
      "   10  22214 219A 195A N   O    1748 1569   12.54    9.91    2.63    4.57    9.91    2.63    4.57\n",
      "   11  22244 220E 195A N   O    1753 1569   11.99    9.18    2.81    4.90    9.18    2.81    4.90\n",
      "   12  22389 226P 195A N   O    1803 1569   11.14    8.51    2.64    4.67    8.51    2.64    4.67\n",
      "   13  22405 227I 195A N   O    1810 1569    9.24    6.37    2.87    5.32    6.37    2.87    5.32\n",
      "   14  22431 228F 195A N   O    1818 1569    7.56    4.99    2.57    4.89    4.99    2.57    4.89\n",
      "   15  22493 231K 194R N   O    1840 1564    9.67    5.11    4.56    9.02    5.11    4.56    9.02\n",
      "   16  22508 232H 194R N   O    1849 1564   10.74    5.43    5.31   10.62    5.43    5.31   10.62\n",
      "   17  22521 233Y 194R N   O    1859 1564   12.81    7.88    4.93    9.46    7.88    4.93    9.46\n",
      "   18  22536 234L 194R N   O    1871 1564   13.97    9.72    4.25    7.68    9.72    4.25    7.68\n",
      "   19  22550 235D 194R N   O    1879 1564   12.36    8.86    3.50    6.51    8.86    3.50    6.51\n",
      "   20  22565 236Q 194R N   O    1887 1564    9.34    6.32    3.02    5.99    6.32    3.02    5.99\n",
      "   21  22583 237L 194R N   O    1896 1564    9.32    6.83    2.49    4.84    6.83    2.49    4.84\n",
      "   22  22942 266S 258L N   O    2122 2060   12.76    9.39    3.36    4.63    9.39    3.36    4.63\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 13                           : Mainchain Omega dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10770 183F 183F CA  C    1452 1460 -153.98 -180.00   26.02    5.20 -180.00   26.02    5.20\n",
      "    2  10772 185T 185T CA  C    1471 1475 -143.66 -180.00   36.34    7.27 -180.00   36.34    7.27\n",
      "    3  10774 187Y 187Y CA  C    1487 1496 -154.58 -180.00   25.42    5.08 -180.00   25.42    5.08\n",
      "    4  10780 193Y 193Y CA  C    1543 1552 -130.01 -180.00   49.99    9.99 -180.00   49.99    9.99\n",
      "    5  10782 195A 195A CA  C    1566 1568 -130.04 -180.00   49.96    9.99 -180.00   49.96    9.99\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9 -111.90 -134.00   22.11    0.70  -62.50  179.79   32.09\n",
      "    1          2A   2A N   CA      9   10  146.23  147.00                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14 -109.37 -134.00   24.74    0.73  -62.50 -179.37   32.08\n",
      "    2          3A   3A N   CA     14   15  144.66  147.00                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19 -114.16 -134.00   19.84    0.67  -62.50  179.39   32.15\n",
      "    3          4A   4A N   CA     19   20  147.31  147.00                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24 -109.25 -134.00   24.84    0.74  -62.50 -179.58   32.04\n",
      "    4          5A   5A N   CA     24   25  144.85  147.00                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29 -111.82 -134.00   22.23    0.68  -62.50 -179.45   32.21\n",
      "    5          6A   6A N   CA     29   30  145.42  147.00                  -40.90\n",
      "    6  10235   6A   7A C   N      32   34 -114.27 -134.00   19.74    0.68  -62.50  179.09   32.10\n",
      "    6          7A   7A N   CA     34   35  147.66  147.00                  -40.90\n",
      "    7  10236   7A   8G C   N      37   39 -157.71 -167.20    9.57    0.16   82.20 -156.03   15.04\n",
      "    7          8G   8G N   CA     39   40  173.38  174.60                    8.50\n",
      "    8  10237   8G   9A C   N      41   43 -111.52 -134.00   22.52    0.80  -62.50  177.67   31.72\n",
      "    8          9A   9A N   CA     43   44  148.32  147.00                  -40.90\n",
      "    9  10263  34G  35A C   N     238  240   88.89  -62.50  165.11   28.35  -62.50  165.11   28.35\n",
      "    9         35A  35A N   CA    240  241 -106.81  -40.90                  -40.90\n",
      "   10  10410 181T 182G C   N    1445 1447 -152.00 -167.20   28.49    0.80   82.20 -170.29   14.37\n",
      "   10        182G 182G N   CA   1447 1448  150.51  174.60                    8.50\n",
      "   11  10411 182G 183F C   N    1449 1451 -160.91 -124.20   57.64    1.64  -63.20  161.00   28.24\n",
      "   11        183F 183F N   CA   1451 1452 -172.26  143.30                  -44.30\n",
      "   12  10412 183F 184L C   N    1460 1462  158.37 -108.50  100.09    5.24  -63.50 -165.43   25.08\n",
      "   12        184L 184L N   CA   1462 1463   95.84  132.50                  -41.20\n",
      "   13  10413 184L 185T C   N    1468 1470   -5.09  -78.10   74.64    4.82  -63.20  163.28   17.80\n",
      "   13        185T 185T N   CA   1470 1471  165.31  149.80                  -42.10\n",
      "   14  10414 185T 186E C   N    1475 1477 -144.20 -117.80   46.09    1.66  -63.60  166.00   26.60\n",
      "   14        186E 186E N   CA   1477 1478  174.58  136.80                  -40.30\n",
      "   15  10415 186E 187Y C   N    1484 1486 -144.04  -63.50   83.31   13.29  -63.50   83.31   13.29\n",
      "   15        187Y 187Y N   CA   1486 1487  -22.11  -43.40                  -43.40\n",
      "   16  10416 187Y 188V C   N    1496 1498  -54.85  -62.40   12.70    2.06 -125.40 -170.87    7.33\n",
      "   16        188V 188V N   CA   1498 1499  -32.18  -42.40                  143.30\n",
      "   17  10417 188V 189A C   N    1503 1505 -136.99  -62.50   81.16   16.47  -62.50   81.16   16.47\n",
      "   17        189A 189A N   CA   1505 1506  -73.11  -40.90                  -40.90\n",
      "   18  10419 190T 191R C   N    1515 1517   34.03  -72.10  106.13    8.10  -63.00 -158.21   22.86\n",
      "   18        191R 191R N   CA   1517 1518  141.98  141.90                  -41.10\n",
      "   19  10420 191R 192W C   N    1526 1528  130.56 -124.90  107.78    5.38  -63.00 -128.19   27.81\n",
      "   19        192W 192W N   CA   1528 1529  117.16  143.40                  -44.20\n",
      "   20  10421 192W 193Y C   N    1540 1542 -178.84  -63.50  117.34   21.16  -63.50  117.34   21.16\n",
      "   20        193Y 193Y N   CA   1542 1543  -64.96  -43.40                  -43.40\n",
      "   21  10422 193Y 194R C   N    1552 1554  -50.76 -125.20   76.18    3.47  -63.00  162.57   21.16\n",
      "   21        194R 194R N   CA   1554 1555  156.79  140.60                  -41.10\n",
      "   22  10423 194R 195A C   N    1563 1565 -107.42  -62.50  144.70   21.99  -62.50  144.70   21.99\n",
      "   22        195A 195A N   CA   1565 1566   96.65  -40.90                  -40.90\n",
      "   23  10424 195A 196P C   N    1568 1570  -26.46  -58.70   58.70    7.39  -58.70   58.70    7.39\n",
      "   23        196P 196P N   CA   1570 1571   18.56  -30.50                  -30.50\n",
      "   24  10480 251D 252L C   N    2005 2007 -155.69 -108.50   54.22    2.39  -63.50 -175.68   31.26\n",
      "   24        252L 252L N   CA   2007 2008  159.20  132.50                  -41.20\n",
      "   25  10481 252L 253N C   N    2013 2015  115.15 -119.90  130.15    4.49  -63.20 -129.83   26.59\n",
      "   25        253N 253N N   CA   2015 2016  173.39  137.00                  -41.10\n",
      "   26  10484 255I 256I C   N    2035 2037  -79.50  -63.40   21.61    4.06 -120.60  176.55   11.65\n",
      "   26        256I 256I N   CA   2037 2038  -58.00  -43.60                  130.30\n",
      "   27  10485 256I 257N C   N    2043 2045 -156.40 -119.90   66.82    2.12  -63.20  156.68   24.65\n",
      "   27        257N 257N N   CA   2045 2046 -167.04  137.00                  -41.10\n",
      "   28  10487 258L 259K C   N    2059 2061  -64.64  -62.90   83.53   10.94  -62.90   83.53   10.94\n",
      "   28        259K 259K N   CA   2061 2062 -124.32  -40.80                  -40.80\n",
      "   29  10488 259K 260A C   N    2068 2070   72.95  -62.50  136.06   26.65  -62.50  136.06   26.65\n",
      "   29        260A 260A N   CA   2070 2071  -28.01  -40.90                  -40.90\n",
      "   30  10587 358Y 359R C   N    2896 2898  -75.76  -72.10    9.70    0.67  -63.00  168.51   23.63\n",
      "   30        359R 359R N   CA   2898 2899  150.88  141.90                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:   188  360   2.218\n",
      "  serious non-bonded atom clash:  1107 1141   2.243\n",
      "  serious non-bonded atom clash:  1389 1399   2.199\n",
      "  serious non-bonded atom clash:  1408 1411   2.232\n",
      "  serious non-bonded atom clash:  1452 1828   2.172\n",
      "  serious non-bonded atom clash:  1460 1565   2.186\n",
      "  serious non-bonded atom clash:  1461 1890   2.226\n",
      "  serious non-bonded atom clash:  1461 1891   2.295\n",
      "  serious non-bonded atom clash:  1462 1820   2.053\n",
      "  serious non-bonded atom clash:  1462 1893   2.123\n",
      "  serious non-bonded atom clash:  1463 1893   2.154\n",
      "  serious non-bonded atom clash:  1469 1924   2.110\n",
      "  serious non-bonded atom clash:  1470 1706   1.933\n",
      "  serious non-bonded atom clash:  1470 1925   2.272\n",
      "  serious non-bonded atom clash:  1471 1927   2.287\n",
      "  serious non-bonded atom clash:  1473 1705   2.191\n",
      "  serious non-bonded atom clash:  1477 1954   2.249\n",
      "  serious non-bonded atom clash:  1530 2332   2.101\n",
      "  serious non-bonded atom clash:  1540 2357   2.213\n",
      "  serious non-bonded atom clash:  1542 1955   2.252\n",
      "  serious non-bonded atom clash:  1542 2357   2.103\n",
      "  serious non-bonded atom clash:  1545 1955   2.256\n",
      "  serious non-bonded atom clash:  1547 1955   2.229\n",
      "  serious non-bonded atom clash:  1549 1949   2.177\n",
      "  serious non-bonded atom clash:  1551 1949   2.102\n",
      "  serious non-bonded atom clash:  1551 2185   1.917\n",
      "  serious non-bonded atom clash:  1552 1825   2.035\n",
      "  serious non-bonded atom clash:  1554 1704   2.173\n",
      "  serious non-bonded atom clash:  1554 1825   2.240\n",
      "  serious non-bonded atom clash:  1557 1585   2.296\n",
      "  serious non-bonded atom clash:  1569 1577   2.227\n",
      "  serious non-bonded atom clash:  1589 2418   2.211\n",
      "  serious non-bonded atom clash:  1590 2023   2.098\n",
      "  serious non-bonded atom clash:  1590 2024   1.854\n",
      "  serious non-bonded atom clash:  1591 2417   2.258\n",
      "  serious non-bonded atom clash:  1591 2418   2.015\n",
      "  serious non-bonded atom clash:  1599 1705   2.137\n",
      "  serious non-bonded atom clash:  1605 2021   2.235\n",
      "  serious non-bonded atom clash:  1605 2023   1.776\n",
      "  serious non-bonded atom clash:  1607 2023   2.054\n",
      "  serious non-bonded atom clash:  1607 2024   1.698\n",
      "  serious non-bonded atom clash:  1615 1617   2.175\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:    10   14   18   21   34   53   78   93  132  209  269  338  416  416  521\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "5ax3_MODLL.B99990001.pdb      9607.98535\n",
      "\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2694 atoms and 1 coordinate set(s) were parsed in 0.03s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 5bue\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "    189     1  172   188      A     T   22.140\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    36513    33867\n",
      "313 (of 2915 total) atoms selected for optimization\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     313\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    33867    2340\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      669\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :        8606.9219\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     323       6      7   0.117   0.117      1356.5       1.000\n",
      " 2 Bond angle potential               :     443       7     15   6.006   6.006      332.23       1.000\n",
      " 3 Stereochemical cosine torsion poten:     206       0     11  53.535  53.535      92.349       1.000\n",
      " 4 Stereochemical improper torsion pot:     142       2      3   3.795   3.795      37.476       1.000\n",
      " 5 Soft-sphere overlap restraints     :     669      11     24   0.070   0.070      385.31       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     386      42     51   2.849   2.849      2923.1       1.000\n",
      "10 Distance restraints 2 (N-O)        :     367      41     47   3.320   3.320      2987.9       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      45       2      3  10.460  10.460      55.303       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      28       0      2  65.962  65.962      15.259       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      21       0      0  81.285  81.285      11.892       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       9       0      0  87.414  87.414      6.2970       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       4       0      0 113.126 113.126      2.5887       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     143       0      0   0.809   0.809      1.7024       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      47      30      9  42.222 143.006      103.70       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     176      16     18   4.112   4.112      295.36       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 5bue_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:   12028.7773\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  1                           : Bond length potential                   \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   1534 188V 188V C   CA   1503 1499    1.86    1.49    0.37   10.72    1.49    0.37   10.72\n",
      "    2   1535 188V 189A C   N    1503 1505    1.77    1.35    0.43   15.19    1.35    0.43   15.19\n",
      "    3   1537 189A 189A N   CA   1505 1506    2.19    1.43    0.76   24.93    1.43    0.76   24.93\n",
      "    4   1538 189A 189A O   C    1509 1508    1.34    1.23    0.11    5.07    1.23    0.11    5.07\n",
      "    5   1539 189A 189A C   CA   1508 1506    2.88    1.49    1.39   40.33    1.49    1.39   40.33\n",
      "    6   1540 189A 190T C   N    1508 1510    2.59    1.35    1.25   44.20    1.35    1.25   44.20\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   5053 187Y 188V C   N    1496 1498  175.36  120.00   55.36   12.58  120.00   55.36   12.58\n",
      "    2   5060 188V 188V CA  C    1499 1503  138.30  116.50   21.80    6.27  116.50   21.80    6.27\n",
      "    3   5063 188V 189A C   N    1503 1505  149.11  120.00   29.11    6.61  120.00   29.11    6.61\n",
      "    4   5066 189A 189A N   CA   1505 1506  143.87  107.00   36.87   10.60  107.00   36.87   10.60\n",
      "    5   5068 189A 189A CA  C    1506 1508  166.61  116.50   50.11   14.41  116.50   50.11   14.41\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  4                           : Stereochemical improper torsion potentia\n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   9629 188V 188V CA  N    1499 1498   54.56   35.27   19.30    4.60   35.27   19.30    4.60\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  9                           : Distance restraints 1 (CA-CA)           \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  13333  73K 188V CA  CA    550 1499   15.98   12.06    3.91    8.10   12.06    3.91    8.10\n",
      "    2  13386  74I 188V CA  CA    559 1499   13.96    9.94    4.03    9.01    9.94    4.03    9.01\n",
      "    3  13387  74I 189A CA  CA    559 1506   17.11   13.38    3.73    7.53   13.38    3.73    7.53\n",
      "    4  13497  77R 188V CA  CA    583 1499   19.17   13.55    5.62   10.26   13.55    5.62   10.26\n",
      "    5  13539  78F 188V CA  CA    594 1499   18.07   13.42    4.65    9.34   13.42    4.65    9.34\n",
      "    6  15173 137L 189A CA  CA   1092 1506   16.55   13.28    3.26    6.16   13.28    3.26    6.16\n",
      "    7  15214 138K 189A CA  CA   1100 1506   20.16   13.77    6.39   11.58   13.77    6.39   11.58\n",
      "    8  15257 139Y 188V CA  CA   1109 1499   17.42   13.18    4.24    7.55   13.18    4.24    7.55\n",
      "    9  15258 139Y 189A CA  CA   1109 1506   20.44   13.60    6.84   12.21   13.60    6.84   12.21\n",
      "   10  15298 140I 188V CA  CA   1121 1499   13.71    9.83    3.87    7.39    9.83    3.87    7.39\n",
      "   11  15299 140I 189A CA  CA   1121 1506   17.08   10.57    6.51   12.37   10.57    6.51   12.37\n",
      "   12  15331 141H 188V CA  CA   1129 1499   14.02    9.09    4.93    9.45    9.09    4.93    9.45\n",
      "   13  15332 141H 189A CA  CA   1129 1506   17.96    8.43    9.54   18.56    8.43    9.54   18.56\n",
      "   14  15362 142S 188V CA  CA   1139 1499   17.53   10.70    6.84   12.67   10.70    6.84   12.67\n",
      "   15  15363 142S 189A CA  CA   1139 1506   21.59   10.32   11.27   21.18   10.32   11.27   21.18\n",
      "   16  15389 143A 188V CA  CA   1145 1499   16.27    8.19    8.08   15.49    8.19    8.08   15.49\n",
      "   17  15390 143A 189A CA  CA   1145 1506   20.60    9.12   11.48   21.68    9.12   11.48   21.68\n",
      "   18  15411 144N 188V CA  CA   1150 1499   14.38    5.61    8.77   18.04    5.61    8.77   18.04\n",
      "   19  15412 144N 189A CA  CA   1150 1506   19.20    5.42   13.78   28.18    5.42   13.78   28.18\n",
      "   20  15441 145V 188V CA  CA   1158 1499   10.91    4.35    6.56   12.72    4.35    6.56   12.72\n",
      "   21  15442 145V 189A CA  CA   1158 1506   15.57    5.52   10.05   19.25    5.52   10.05   19.25\n",
      "   22  15472 146L 189A CA  CA   1165 1506   12.38    6.37    6.00   11.62    6.37    6.00   11.62\n",
      "   23  15916 188V 325A CA  CA   1499 2614   20.57   13.28    7.29   12.95   13.28    7.29   12.95\n",
      "   24  15917 188V 327A CA  CA   1499 2628   22.04   12.47    9.57   16.62   12.47    9.57   16.62\n",
      "   25  15918 188V 328P CA  CA   1499 2633   18.66    8.68    9.99   19.34    8.68    9.99   19.34\n",
      "   26  15919 188V 329F CA  CA   1499 2640   16.55    6.96    9.59   18.78    6.96    9.59   18.78\n",
      "   27  15920 188V 330K CA  CA   1499 2651   17.19    7.12   10.07   20.16    7.12   10.07   20.16\n",
      "   28  15921 188V 331F CA  CA   1499 2660   16.67    8.31    8.36   16.08    8.31    8.36   16.08\n",
      "   29  15922 188V 332D CA  CA   1499 2671   20.16   11.89    8.27   14.06   11.89    8.27   14.06\n",
      "   30  15923 188V 333M CA  CA   1499 2679   19.19   12.43    6.76   12.00   12.43    6.76   12.00\n",
      "   31  15924 188V 334E CA  CA   1499 2687   17.29   11.76    5.52    9.82   11.76    5.52    9.82\n",
      "   32  15925 188V 335L CA  CA   1499 2696   15.25   11.94    3.31    5.96   11.94    3.31    5.96\n",
      "   33  15933 189A 327A CA  CA   1506 2628   26.49   13.66   12.83   21.78   13.66   12.83   21.78\n",
      "   34  15934 189A 328P CA  CA   1506 2633   23.39    9.87   13.52   25.60    9.87   13.52   25.60\n",
      "   35  15935 189A 329F CA  CA   1506 2640   21.16    9.58   11.58   21.86    9.58   11.58   21.86\n",
      "   36  15936 189A 330K CA  CA   1506 2651   22.05   10.01   12.03   22.17   10.01   12.03   22.17\n",
      "   37  15937 189A 331F CA  CA   1506 2660   21.24   11.75    9.49   17.03   11.75    9.49   17.03\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 10                           : Distance restraints 2 (N-O)             \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  20866 141H 189A N   O    1128 1509   18.53   10.89    7.65   12.05   10.89    7.65   12.05\n",
      "    2  20890 142S 189A N   O    1138 1509   21.30   10.85   10.45   16.32   10.85   10.45   16.32\n",
      "    3  20915 143A 188V N   O    1144 1504   16.13   10.67    5.46    8.59   10.67    5.46    8.59\n",
      "    4  20916 143A 189A N   O    1144 1509   21.92   10.02   11.90   19.06   10.02   11.90   19.06\n",
      "    5  20935 144N 188V N   O    1149 1504   15.15    8.19    6.96   11.66    8.19    6.96   11.66\n",
      "    6  20936 144N 189A N   O    1149 1509   21.18    7.40   13.78   23.38    7.40   13.78   23.38\n",
      "    7  20961 145V 188V N   O    1157 1504   12.38    6.47    5.92   10.26    6.47    5.92   10.26\n",
      "    8  20962 145V 189A N   O    1157 1509   18.51    6.71   11.80   20.32    6.71   11.80   20.32\n",
      "    9  20988 146L 188V N   O    1164 1504    8.88    6.22    2.67    4.67    6.22    2.67    4.67\n",
      "   10  20989 146L 189A N   O    1164 1509   14.96    7.67    7.29   12.37    7.67    7.29   12.37\n",
      "   11  21589 188V  70R N   O    1498  531   12.30    9.17    3.13    5.96    9.17    3.13    5.96\n",
      "   12  21590 188V 140I N   O    1498 1127   14.59    8.48    6.11   10.25    8.48    6.11   10.25\n",
      "   13  21591 188V 141H N   O    1498 1137   16.29    9.51    6.78   11.00    9.51    6.78   11.00\n",
      "   14  21592 188V 143A N   O    1498 1148   15.92    7.15    8.77   14.94    7.15    8.77   14.94\n",
      "   15  21593 188V 144N N   O    1498 1156   13.12    4.36    8.76   15.72    4.36    8.76   15.72\n",
      "   16  21594 188V 145V N   O    1498 1163   11.65    6.63    5.02    8.68    6.63    5.02    8.68\n",
      "   17  21603 188V 328P N   O    1498 2638   19.41    9.85    9.56   15.28    9.85    9.56   15.28\n",
      "   18  21604 188V 329F N   O    1498 2649   16.22    6.47    9.75   16.80    6.47    9.75   16.80\n",
      "   19  21605 188V 330K N   O    1498 2658   17.88    9.15    8.73   14.14    9.15    8.73   14.14\n",
      "   20  21606 188V 331F N   O    1498 2669   17.91    9.57    8.34   13.58    9.57    8.34   13.58\n",
      "   21  21607 188V 333M N   O    1498 2685   16.25   10.08    6.17   10.12   10.08    6.17   10.12\n",
      "   22  21608 189A 140I N   O    1505 1127   15.74    7.80    7.94   13.76    7.80    7.94   13.76\n",
      "   23  21609 189A 141H N   O    1505 1137   17.66    7.49   10.17   17.47    7.49   10.17   17.47\n",
      "   24  21610 189A 142S N   O    1505 1143   20.72    9.64   11.07   17.87    9.64   11.07   17.87\n",
      "   25  21611 189A 143A N   O    1505 1148   18.00    6.50   11.50   19.89    6.50   11.50   19.89\n",
      "   26  21612 189A 144N N   O    1505 1156   15.53    2.78   12.74   23.06    2.78   12.74   23.06\n",
      "   27  21613 189A 145V N   O    1505 1163   12.96    5.86    7.10   12.39    5.86    7.10   12.39\n",
      "   28  21621 189A 328P N   O    1505 2638   21.89    9.68   12.20   19.62    9.68   12.20   19.62\n",
      "   29  21622 189A 329F N   O    1505 2649   18.97    6.74   12.23   20.99    6.74   12.23   20.99\n",
      "   30  21623 189A 330K N   O    1505 2658   20.80   10.75   10.05   15.60   10.75   10.05   15.60\n",
      "   31  24206 328P 189A N   O    2632 1509   26.45   10.58   15.87   24.91   10.58   15.87   24.91\n",
      "   32  24224 329F 188V N   O    2639 1504   17.10    9.25    7.86   12.80    9.25    7.86   12.80\n",
      "   33  24225 329F 189A N   O    2639 1509   23.50    8.99   14.51   23.80    8.99   14.51   23.80\n",
      "   34  24243 330K 188V N   O    2650 1504   17.34    8.89    8.45   13.78    8.89    8.45   13.78\n",
      "   35  24244 330K 189A N   O    2650 1509   23.87    9.24   14.63   23.62    9.24   14.63   23.62\n",
      "   36  24258 331F 188V N   O    2659 1504   17.04    8.68    8.37   13.91    8.68    8.37   13.91\n",
      "   37  24259 331F 189A N   O    2659 1509   23.51    9.77   13.74   22.17    9.77   13.74   22.17\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 13                           : Mainchain Omega dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10776 189A 189A CA  C    1506 1508 -153.49 -179.18   25.69    5.97 -179.18   25.69    5.97\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9 -121.88 -134.00   12.11    0.40  -62.50 -177.98   33.00\n",
      "    1          2A   2A N   CA      9   10  147.04  147.00                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14 -111.88 -134.00   22.15    0.69  -62.50 -179.80   32.15\n",
      "    2          3A   3A N   CA     14   15  145.80  147.00                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19 -110.98 -134.00   23.06    0.71  -62.50 -179.86   32.09\n",
      "    3          4A   4A N   CA     19   20  145.61  147.00                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24  -69.38  -68.20    5.03    0.36  -62.50  169.05   28.05\n",
      "    4          5A   5A N   CA     24   25  150.19  145.30                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29  -70.41  -68.20    6.61    0.46  -62.50  167.76   27.89\n",
      "    5          6A   6A N   CA     29   30  151.53  145.30                  -40.90\n",
      "    6  10235   6A   7A C   N      32   34  -69.79  -68.20    6.63    0.48  -62.50  167.52   27.82\n",
      "    6          7A   7A N   CA     34   35  151.74  145.30                  -40.90\n",
      "    7  10236   7A   8G C   N      37   39  -76.42  -80.20    8.81    0.21   82.20 -136.37    7.02\n",
      "    7          8G   8G N   CA     39   40  166.15  174.10                    8.50\n",
      "    8  10237   8G   9A C   N      41   43 -118.18 -134.00   15.94    0.60  -62.50  179.04   32.31\n",
      "    8          9A   9A N   CA     43   44  148.94  147.00                  -40.90\n",
      "    9  10238   9A  10G C   N      46   48 -145.58 -167.20   24.12    0.37   82.20 -155.95   15.38\n",
      "    9         10G  10G N   CA     48   49  163.92  174.60                    8.50\n",
      "   10  10402 173V 174A C   N    1382 1384 -169.29 -134.00   44.81    1.17  -62.50  179.68   35.06\n",
      "   10        174A 174A N   CA   1384 1385  174.60  147.00                  -40.90\n",
      "   11  10403 174A 175D C   N    1387 1389  -92.51  -96.50    8.22    0.35  -63.30  149.88   16.93\n",
      "   11        175D 175D N   CA   1389 1390  107.01  114.20                  -40.00\n",
      "   12  10406 177D 178H C   N    1410 1412 -158.77 -125.60   58.48    1.55  -63.20  161.94   26.28\n",
      "   12        178H 178H N   CA   1412 1413 -173.04  138.80                  -42.30\n",
      "   13  10407 178H 179D C   N    1420 1422 -121.71  -96.50   25.55    1.05  -63.30  168.73   18.17\n",
      "   13        179D 179D N   CA   1422 1423  118.30  114.20                  -40.00\n",
      "   14  10408 179D 180H C   N    1428 1430 -123.77 -125.60   12.09    0.49  -63.20  177.60   26.17\n",
      "   14        180H 180H N   CA   1430 1431  150.75  138.80                  -42.30\n",
      "   15  10409 180H 181T C   N    1438 1440  -79.64  -78.10   22.06    0.99  -63.20  170.69   20.84\n",
      "   15        181T 181T N   CA   1440 1441  127.80  149.80                  -42.10\n",
      "   16  10411 182G 183F C   N    1449 1451 -158.44 -124.20   58.60    1.75  -63.20  157.03   27.54\n",
      "   16        183F 183F N   CA   1451 1452 -169.15  143.30                  -44.30\n",
      "   17  10412 183F 184L C   N    1460 1462 -102.05 -108.50    8.65    0.39  -63.50  172.32   22.03\n",
      "   17        184L 184L N   CA   1462 1463  126.75  132.50                  -41.20\n",
      "   18  10413 184L 185T C   N    1468 1470 -116.76 -124.80   19.05    0.61  -63.20  176.65   19.62\n",
      "   18        185T 185T N   CA   1470 1471  126.23  143.50                  -42.10\n",
      "   19  10415 186E 187Y C   N    1484 1486  162.27 -124.30   74.17    5.23  -63.50 -142.80   40.53\n",
      "   19        187Y 187Y N   CA   1486 1487  145.84  135.40                  -43.40\n",
      "   20  10416 187Y 188V C   N    1496 1498 -116.33  -62.40   56.63    9.57  -73.50  166.70   13.54\n",
      "   20        188V 188V N   CA   1498 1499  -59.70  -42.40                  139.20\n",
      "   21  10417 188V 189A C   N    1503 1505  167.17 -134.00   59.92    2.43  -68.20  125.01   10.91\n",
      "   21        189A 189A N   CA   1505 1506  135.64  147.00                  145.30\n",
      "   22  10418 189A 190T C   N    1508 1510   41.61   55.90   97.62    5.82 -124.80  166.58    8.63\n",
      "   22        190T 190T N   CA   1510 1511  136.07   39.50                  143.50\n",
      "   23  10429 200L 201N C   N    1608 1610  119.18   55.90  103.94   11.75   55.90  103.94   11.75\n",
      "   23        201N 201N N   CA   1610 1611  121.95   39.50                   39.50\n",
      "   24  10430 201N 202S C   N    1616 1618  -69.63  -72.40   19.31    1.04  -64.10  168.38   11.88\n",
      "   24        202S 202S N   CA   1618 1619  133.29  152.40                  -35.00\n",
      "   25  10431 202S 203K C   N    1622 1624 -127.02 -118.00   32.52    1.67  -62.90  161.90   18.66\n",
      "   25        203K 203K N   CA   1624 1625  107.86  139.10                  -40.80\n",
      "   26  10432 203K 204G C   N    1631 1633   83.64   78.70   15.38    0.26   82.20  170.84    8.37\n",
      "   26        204G 204G N   CA   1633 1634  179.34 -166.10                    8.50\n",
      "   27  10433 204G 205Y C   N    1635 1637 -109.73  -98.40   11.41    0.48  -63.50  179.29   26.25\n",
      "   27        205Y 205Y N   CA   1637 1638  129.83  128.40                  -43.40\n",
      "   28  10434 205Y 206T C   N    1647 1649 -132.33 -124.80   13.25    0.77   55.90 -164.62   24.11\n",
      "   28        206T 206T N   CA   1649 1650  132.59  143.50                   39.50\n",
      "   29  10586 357G 358Y C   N    2884 2886  -63.16  -63.50    1.77    0.26 -124.30 -170.42   12.69\n",
      "   29        358Y 358Y N   CA   2886 2887  -45.14  -43.40                  135.40\n",
      "   30  10587 358Y 359R C   N    2896 2898   91.75   57.30  130.37   11.20  -63.00 -140.85   25.90\n",
      "   30        359R 359R N   CA   2898 2899  163.73   38.00                  -41.10\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 26                           : Distance restraints 4 (SDCH-SDCH)       \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  32274 144N 188V CB  CG1  1151 1501   15.19    4.65   10.54    6.51    4.65   10.54    6.51\n",
      "    2  32275 144N 189A CB  CB   1151 1507   20.55    4.79   15.76    9.68    4.79   15.76    9.68\n",
      "    3  32277 144N 188V CG  CG1  1152 1501   14.91    4.65   10.26    6.34    4.65   10.26    6.34\n",
      "    4  32278 144N 189A CG  CB   1152 1507   20.70    3.86   16.84   10.42    3.86   16.84   10.42\n",
      "    5  32282 144N 189A OD1 CB   1153 1507   20.64    3.38   17.25   10.71    3.38   17.25   10.71\n",
      "    6  32285 144N 188V ND2 CG1  1154 1501   15.20    3.99   11.21    6.96    3.99   11.21    6.96\n",
      "    7  32286 144N 189A ND2 CB   1154 1507   21.06    4.17   16.89   10.43    4.17   16.89   10.43\n",
      "    8  32288 145V 188V CG2 CB   1161 1500   12.95    4.92    8.03    4.92    4.92    8.03    4.92\n",
      "    9  32289 145V 188V CG2 CG2  1161 1502   14.17    4.34    9.83    6.05    4.34    9.83    6.05\n",
      "   10  32301 146L 189A CG  CB   1167 1507   13.41    4.61    8.79    5.40    4.61    8.79    5.40\n",
      "   11  32310 146L 189A CD2 CB   1169 1507   14.76    3.65   11.11    6.86    3.65   11.11    6.86\n",
      "   12  32577 188V 329F CG2 CB   1502 2641   17.52    4.78   12.74    7.79    4.78   12.74    7.79\n",
      "   13  32578 188V 329F CG2 CG   1502 2642   17.42    4.95   12.47    7.62    4.95   12.47    7.62\n",
      "   14  32579 188V 329F CG2 CD1  1502 2643   16.84    4.17   12.68    7.79    4.17   12.68    7.79\n",
      "   15  32580 188V 331F CG2 CD2  1502 2664   15.09    3.84   11.25    6.95    3.84   11.25    6.95\n",
      "   16  32581 188V 331F CG2 CE2  1502 2666   14.13    3.91   10.23    6.32    3.91   10.23    6.32\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:  1357 1503   2.094\n",
      "  serious non-bonded atom clash:  1506 1562   2.001\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     2    0    0    3    5    7   16   79   87  118  182  217  317  472  485\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "5bue_MODLL.B99990001.pdb      8606.92188\n",
      "\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2743 atoms and 1 coordinate set(s) were parsed in 0.03s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 5bui\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    37253    34607\n",
      "217 (of 2915 total) atoms selected for optimization\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:      -91.9616\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     217\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    34607    1482\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      352\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         130.8572\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     224       0      0   0.003   0.003     0.78272       1.000\n",
      " 2 Bond angle potential               :     307       0      0   1.699   1.699      16.228       1.000\n",
      " 3 Stereochemical cosine torsion poten:     147       0      4  47.513  47.513      52.646       1.000\n",
      " 4 Stereochemical improper torsion pot:     102       0      0   0.949   0.949      1.9479       1.000\n",
      " 5 Soft-sphere overlap restraints     :     352       0      0   0.001   0.001     0.81491E-01   1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     193       0      0   0.136   0.136      2.7189       1.000\n",
      "10 Distance restraints 2 (N-O)        :     192       0      0   0.285   0.285      6.7389       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      31       0      0   3.829   3.829      5.3608       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      18       0      0  62.565  62.565      3.6450       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      13       0      0  59.574  59.574      4.7086       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       6       0      0 108.032 108.032      4.4515       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       2       0      0 124.786 124.786      1.2635       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     127       0      0   0.356   0.356     0.39250       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      32      20      2  24.007 137.874      28.515       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      88       0      0   0.561   0.561      1.3754       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 5bui_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    5382.8311\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9 -132.07 -134.00    2.36    0.06  -62.50 -173.10   34.35\n",
      "    1          2A   2A N   CA      9   10  145.63  147.00                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14 -116.10 -134.00   21.55    0.53  -62.50 -176.12   28.04\n",
      "    2          3A   3A N   CA     14   15  135.00  147.00                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19  -74.08  -68.20    8.57    0.84  -62.50 -179.66   29.01\n",
      "    3          4A   4A N   CA     19   20  139.07  145.30                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24  -84.17  -68.20   17.39    1.23  -62.50  168.33   28.70\n",
      "    4          5A   5A N   CA     24   25  152.18  145.30                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29  -72.40  -68.20   11.59    0.79  -62.50  163.30   27.26\n",
      "    5          6A   6A N   CA     29   30  156.10  145.30                  -40.90\n",
      "    6  10235   6A   7A C   N      32   34  -60.90  -68.20   17.55    1.17  -62.50  170.25   27.98\n",
      "    6          7A   7A N   CA     34   35  129.34  145.30                  -40.90\n",
      "    7  10236   7A   8G C   N      37   39 -168.71 -167.20    1.60    0.04   82.20 -161.74   14.38\n",
      "    7          8G   8G N   CA     39   40  174.05  174.60                    8.50\n",
      "    8  10237   8G   9A C   N      41   43  -74.25  -68.20   13.11    0.86  -62.50  162.59   27.24\n",
      "    8          9A   9A N   CA     43   44  156.93  145.30                  -40.90\n",
      "    9  10238   9A  10G C   N      46   48  143.07 -167.20   50.24    0.81   82.20 -176.39   11.79\n",
      "    9         10G  10G N   CA     48   49 -178.27  174.60                    8.50\n",
      "   10  10407 178H 179D C   N    1420 1422 -136.04  -96.50   44.32    1.80  -70.90   85.97    7.07\n",
      "   10        179D 179D N   CA   1422 1423   94.20  114.20                  150.30\n",
      "   11  10408 179D 180H C   N    1428 1430 -103.96 -125.60   38.50    1.02  -63.20  154.72   17.20\n",
      "   11        180H 180H N   CA   1430 1431  106.95  138.80                  -42.30\n",
      "   12  10409 180H 181T C   N    1438 1440 -126.91 -124.80   12.25    0.46  -63.20  174.39   26.57\n",
      "   12        181T 181T N   CA   1440 1441  155.56  143.50                  -42.10\n",
      "   13  10410 181T 182G C   N    1445 1447 -124.31 -167.20   44.32    0.65   82.20 -141.94   16.71\n",
      "   13        182G 182G N   CA   1447 1448  163.40  174.60                    8.50\n",
      "   14  10411 182G 183F C   N    1449 1451  -72.43  -71.40   17.57    1.44  -63.20  167.72   22.67\n",
      "   14        183F 183F N   CA   1451 1452  123.16  140.70                  -44.30\n",
      "   15  10412 183F 184L C   N    1460 1462 -110.95 -108.50   34.55    1.84  -63.50  159.07   25.02\n",
      "   15        184L 184L N   CA   1462 1463  166.97  132.50                  -41.20\n",
      "   16  10413 184L 185T C   N    1468 1470  -60.56  -78.10   17.58    1.09  -63.20  167.02   21.25\n",
      "   16        185T 185T N   CA   1470 1471  150.90  149.80                  -42.10\n",
      "   17  10414 185T 186E C   N    1475 1477  -98.29 -117.80   23.93    1.22  -63.60  172.57   24.97\n",
      "   17        186E 186E N   CA   1477 1478  150.65  136.80                  -40.30\n",
      "   18  10415 186E 187Y C   N    1484 1486 -113.33 -124.30   27.20    1.91  -63.50  164.06   28.56\n",
      "   18        187Y 187Y N   CA   1486 1487  160.29  135.40                  -43.40\n",
      "   19  10417 188V 189A C   N    1503 1505 -162.62 -134.00   33.44    0.79  -62.50 -175.62   35.53\n",
      "   19        189A 189A N   CA   1505 1506  164.28  147.00                  -40.90\n",
      "   20  10587 358Y 359R C   N    2896 2898  -87.99  -72.10   15.90    1.21  -63.00  178.43   25.82\n",
      "   20        359R 359R N   CA   2898 2899  142.23  141.90                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:  2674 2677   2.187\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    1    0    0    0    7   16   39   91  132  182  189  344  433  521\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "5bui_MODLL.B99990001.pdb       130.85724\n",
      "\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2679 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 5buj\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "    189     1  172   188      A     T   22.211\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    36573    33927\n",
      "297 (of 2915 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2698 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     297\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    33927    2185\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      680\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :        9114.4609\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     306       9      9   0.124   0.124      1456.0       1.000\n",
      " 2 Bond angle potential               :     420      10     17   6.547   6.547      395.43       1.000\n",
      " 3 Stereochemical cosine torsion poten:     194       0      6  50.622  50.622      81.315       1.000\n",
      " 4 Stereochemical improper torsion pot:     137       1      1   4.237   4.237      43.081       1.000\n",
      " 5 Soft-sphere overlap restraints     :     680      18     27   0.078   0.078      488.31       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     369      43     56   2.985   2.985      3006.8       1.000\n",
      "10 Distance restraints 2 (N-O)        :     351      42     49   3.510   3.510      3196.9       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      43       4      3  28.567  28.567      36.239       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      26       0      4  86.758  86.758      20.837       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      19       0      0  91.065  91.065      10.832       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       7       0      0  91.214  91.214      4.2630       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       4       0      0  99.222  99.222      3.6253       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     123       0      0   0.874   0.874      1.9598       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      45      28     12  43.162 140.985      89.685       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     141      14     18   4.527   4.527      279.17       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 5buj_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:   12304.8594\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  1                           : Bond length potential                   \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   1528 187Y 188V C   N    1496 1498    1.54    1.35    0.19    6.87    1.35    0.19    6.87\n",
      "    2   1529 188V 188V CB  CA   1500 1499    1.77    1.50    0.27    7.52    1.50    0.27    7.52\n",
      "    3   1532 188V 188V N   CA   1498 1499    1.73    1.43    0.30    9.71    1.43    0.30    9.71\n",
      "    4   1534 188V 188V C   CA   1503 1499    1.85    1.49    0.36   10.38    1.49    0.36   10.38\n",
      "    5   1535 188V 189A C   N    1503 1505    1.74    1.35    0.40   14.05    1.35    0.40   14.05\n",
      "    6   1537 189A 189A N   CA   1505 1506    2.20    1.43    0.77   25.48    1.43    0.77   25.48\n",
      "    7   1538 189A 189A O   C    1509 1508    1.35    1.23    0.12    5.68    1.23    0.12    5.68\n",
      "    8   1539 189A 189A C   CA   1508 1506    2.90    1.49    1.41   40.92    1.49    1.41   40.92\n",
      "    9   1540 189A 190T C   N    1508 1510    2.63    1.35    1.28   45.39    1.35    1.28   45.39\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   5041 187Y 187Y CA  C    1487 1496  137.66  116.50   21.16    6.08  116.50   21.16    6.08\n",
      "    2   5055 188V 188V N   CA   1498 1499  152.68  113.50   39.18   10.54  113.50   39.18   10.54\n",
      "    3   5060 188V 188V CA  C    1499 1503  139.44  116.50   22.94    6.60  116.50   22.94    6.60\n",
      "    4   5063 188V 189A C   N    1503 1505  162.43  120.00   42.43    9.64  120.00   42.43    9.64\n",
      "    5   5066 189A 189A N   CA   1505 1506  158.82  107.00   51.82   14.90  107.00   51.82   14.90\n",
      "    6   5068 189A 189A CA  C    1506 1508  166.76  116.50   50.26   14.45  116.50   50.26   14.45\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  9                           : Distance restraints 1 (CA-CA)           \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  13397  73K 188V CA  CA    550 1499   15.04   12.08    2.96    6.10   12.08    2.96    6.10\n",
      "    2  13449  74I 188V CA  CA    559 1499   12.88   10.02    2.86    6.37   10.02    2.86    6.37\n",
      "    3  13450  74I 189A CA  CA    559 1506   17.52   13.36    4.16    8.42   13.36    4.16    8.42\n",
      "    4  13560  77R 188V CA  CA    583 1499   17.81   13.48    4.33    8.19   13.48    4.33    8.19\n",
      "    5  13602  78F 188V CA  CA    594 1499   16.58   13.43    3.15    6.30   13.43    3.15    6.30\n",
      "    6  15241 137L 189A CA  CA   1092 1506   16.89   13.16    3.72    7.06   13.16    3.72    7.06\n",
      "    7  15280 138K 189A CA  CA   1100 1506   20.42   13.48    6.93   12.67   13.48    6.93   12.67\n",
      "    8  15321 139Y 188V CA  CA   1109 1499   16.30   13.19    3.10    5.52   13.19    3.10    5.52\n",
      "    9  15322 139Y 189A CA  CA   1109 1506   20.76   13.39    7.37   13.24   13.39    7.37   13.24\n",
      "   10  15362 140I 188V CA  CA   1121 1499   12.65    9.84    2.80    5.35    9.84    2.80    5.35\n",
      "   11  15363 140I 189A CA  CA   1121 1506   17.41   10.36    7.05   13.45   10.36    7.05   13.45\n",
      "   12  15395 141H 188V CA  CA   1129 1499   13.44    9.18    4.26    8.18    9.18    4.26    8.18\n",
      "   13  15396 141H 189A CA  CA   1129 1506   18.32    8.30   10.02   19.49    8.30   10.02   19.49\n",
      "   14  15426 142S 188V CA  CA   1139 1499   16.91   10.81    6.10   11.27   10.81    6.10   11.27\n",
      "   15  15427 142S 189A CA  CA   1139 1506   21.93   10.28   11.65   21.92   10.28   11.65   21.92\n",
      "   16  15454 143A 188V CA  CA   1145 1499   15.58    8.21    7.38   14.18    8.21    7.38   14.18\n",
      "   17  15455 143A 189A CA  CA   1145 1506   20.93    8.97   11.96   22.59    8.97   11.96   22.59\n",
      "   18  15477 144N 188V CA  CA   1150 1499   14.06    5.64    8.43   17.39    5.64    8.43   17.39\n",
      "   19  15478 144N 189A CA  CA   1150 1506   19.49    5.28   14.21   29.01    5.28   14.21   29.01\n",
      "   20  15507 145V 188V CA  CA   1158 1499   10.45    4.38    6.08   11.88    4.38    6.08   11.88\n",
      "   21  15508 145V 189A CA  CA   1158 1506   15.92    5.34   10.58   20.17    5.34   10.58   20.17\n",
      "   22  15538 146L 189A CA  CA   1165 1506   12.70    5.94    6.76   13.07    5.94    6.76   13.07\n",
      "   23  15987 188V 325A CA  CA   1499 2614   19.52   13.24    6.28   11.16   13.24    6.28   11.16\n",
      "   24  15988 188V 327A CA  CA   1499 2628   21.20   12.34    8.86   15.46   12.34    8.86   15.46\n",
      "   25  15989 188V 328P CA  CA   1499 2633   18.18    8.62    9.56   18.30    8.62    9.56   18.30\n",
      "   26  15990 188V 329F CA  CA   1499 2640   15.99    6.92    9.07   17.73    6.92    9.07   17.73\n",
      "   27  15991 188V 330K CA  CA   1499 2651   16.93    7.16    9.78   19.47    7.16    9.78   19.47\n",
      "   28  15992 188V 331F CA  CA   1499 2660   16.48    8.34    8.13   15.67    8.34    8.13   15.67\n",
      "   29  15993 188V 332D CA  CA   1499 2671   20.07   11.92    8.15   13.69   11.92    8.15   13.69\n",
      "   30  15994 188V 333M CA  CA   1499 2679   19.49   12.56    6.93   12.26   12.56    6.93   12.26\n",
      "   31  15995 188V 334E CA  CA   1499 2687   17.88   11.72    6.16   10.87   11.72    6.16   10.87\n",
      "   32  15996 188V 335L CA  CA   1499 2696   15.55   11.71    3.84    6.97   11.71    3.84    6.97\n",
      "   33  16004 189A 303E CA  CA   1506 2430   21.26   13.88    7.39   13.24   13.88    7.39   13.24\n",
      "   34  16005 189A 327A CA  CA   1506 2628   26.63   13.49   13.14   22.59   13.49   13.14   22.59\n",
      "   35  16006 189A 328P CA  CA   1506 2633   23.71    9.75   13.96   26.25    9.75   13.96   26.25\n",
      "   36  16007 189A 329F CA  CA   1506 2640   21.48    9.48   12.00   22.54    9.48   12.00   22.54\n",
      "   37  16008 189A 330K CA  CA   1506 2651   22.34   10.13   12.21   22.87   10.13   12.21   22.87\n",
      "   38  16009 189A 331F CA  CA   1506 2660   21.53   11.86    9.66   17.25   11.86    9.66   17.25\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 10                           : Distance restraints 2 (N-O)             \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  20976 141H 188V N   O    1128 1504   14.46   10.75    3.70    5.81   10.75    3.70    5.81\n",
      "    2  20977 141H 189A N   O    1128 1509   18.81   10.68    8.12   12.90   10.68    8.12   12.90\n",
      "    3  21001 142S 189A N   O    1138 1509   21.59   10.74   10.84   17.00   10.74   10.84   17.00\n",
      "    4  21026 143A 188V N   O    1144 1504   17.26   10.64    6.62   10.41   10.64    6.62   10.41\n",
      "    5  21027 143A 189A N   O    1144 1509   22.17    9.84   12.34   19.88    9.84   12.34   19.88\n",
      "    6  21045 144N 188V N   O    1149 1504   16.04    8.16    7.88   13.21    8.16    7.88   13.21\n",
      "    7  21046 144N 189A N   O    1149 1509   21.44    7.16   14.28   24.34    7.16   14.28   24.34\n",
      "    8  21070 145V 188V N   O    1157 1504   13.31    6.36    6.96   12.08    6.36    6.96   12.08\n",
      "    9  21071 145V 189A N   O    1157 1509   18.80    6.49   12.31   21.28    6.49   12.31   21.28\n",
      "   10  21096 146L 188V N   O    1164 1504    9.74    5.70    4.03    7.12    5.70    4.03    7.12\n",
      "   11  21097 146L 189A N   O    1164 1509   15.30    7.29    8.01   13.69    7.29    8.01   13.69\n",
      "   12  21709 188V 140I N   O    1498 1127   13.23    8.58    4.65    7.77    8.58    4.65    7.77\n",
      "   13  21710 188V 141H N   O    1498 1137   15.01    9.63    5.37    8.69    9.63    5.37    8.69\n",
      "   14  21711 188V 143A N   O    1498 1148   14.73    7.18    7.56   12.86    7.18    7.56   12.86\n",
      "   15  21712 188V 144N N   O    1498 1156   11.94    4.44    7.50   13.45    4.44    7.50   13.45\n",
      "   16  21713 188V 145V N   O    1498 1163   10.25    6.65    3.59    6.20    6.65    3.59    6.20\n",
      "   17  21722 188V 328P N   O    1498 2638   18.38    9.90    8.48   13.40    9.90    8.48   13.40\n",
      "   18  21723 188V 329F N   O    1498 2649   15.22    6.38    8.85   15.26    6.38    8.85   15.26\n",
      "   19  21724 188V 330K N   O    1498 2658   17.12    9.10    8.02   13.16    9.10    8.02   13.16\n",
      "   20  21725 188V 331F N   O    1498 2669   17.43    9.50    7.93   12.95    9.50    7.93   12.95\n",
      "   21  21726 188V 333M N   O    1498 2685   16.27   10.09    6.18   10.13   10.09    6.18   10.13\n",
      "   22  21727 189A 139Y N   O    1505 1119   18.27   10.81    7.46   12.03   10.81    7.46   12.03\n",
      "   23  21728 189A 140I N   O    1505 1127   15.93    7.73    8.20   14.26    7.73    8.20   14.26\n",
      "   24  21729 189A 141H N   O    1505 1137   17.93    7.49   10.44   17.97    7.49   10.44   17.97\n",
      "   25  21730 189A 142S N   O    1505 1143   20.83    9.54   11.29   18.27    9.54   11.29   18.27\n",
      "   26  21731 189A 143A N   O    1505 1148   18.15    6.41   11.74   20.34    6.41   11.74   20.34\n",
      "   27  21732 189A 144N N   O    1505 1156   15.63    2.61   13.02   23.56    2.61   13.02   23.56\n",
      "   28  21733 189A 145V N   O    1505 1163   13.16    5.66    7.49   13.11    5.66    7.49   13.11\n",
      "   29  21741 189A 327A N   O    1505 2631   22.15   10.89   11.27   17.58   10.89   11.27   17.58\n",
      "   30  21742 189A 328P N   O    1505 2638   21.95    9.76   12.19   19.47    9.76   12.19   19.47\n",
      "   31  21743 189A 329F N   O    1505 2649   19.01    6.75   12.27   21.11    6.75   12.27   21.11\n",
      "   32  21744 189A 330K N   O    1505 2658   20.75   10.82    9.93   15.54   10.82    9.93   15.54\n",
      "   33  24330 328P 189A N   O    2632 1509   26.67   10.41   16.26   25.52   10.41   16.26   25.52\n",
      "   34  24348 329F 188V N   O    2639 1504   18.20    9.27    8.92   14.53    9.27    8.92   14.53\n",
      "   35  24349 329F 189A N   O    2639 1509   23.74    8.70   15.04   24.90    8.70   15.04   24.90\n",
      "   36  24367 330K 188V N   O    2650 1504   18.46    9.23    9.23   15.10    9.23    9.23   15.10\n",
      "   37  24368 330K 189A N   O    2650 1509   24.16    9.28   14.88   24.21    9.28   14.88   24.21\n",
      "   38  24382 331F 188V N   O    2659 1504   17.95    9.01    8.93   14.75    9.01    8.93   14.75\n",
      "   39  24383 331F 189A N   O    2659 1509   23.73    9.83   13.90   22.41    9.83   13.90   22.41\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 13                           : Mainchain Omega dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10776 189A 189A CA  C    1506 1508 -149.25 -179.18   29.93    6.96 -179.18   29.93    6.96\n",
      "    2  10788 201N 201N CA  C    1611 1616   -1.13 -179.18  178.06   41.38 -179.18  178.06   41.38\n",
      "    3  10943 356P 356P CA  C    2876 2880 -158.76 -179.18   20.43    4.75 -179.18   20.43    4.75\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9 -122.10 -134.00   11.92    0.42  -62.50 -178.44   32.94\n",
      "    1          2A   2A N   CA      9   10  147.60  147.00                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14  -69.55  -68.20    5.23    0.37  -62.50  168.90   28.03\n",
      "    2          3A   3A N   CA     14   15  150.35  145.30                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19 -116.27 -134.00   17.76    0.63  -62.50  179.18   32.23\n",
      "    3          4A   4A N   CA     19   20  148.19  147.00                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24 -105.80 -134.00   28.57    0.78  -62.50 -178.07   32.10\n",
      "    4          5A   5A N   CA     24   25  142.40  147.00                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29 -117.82 -134.00   16.21    0.58  -62.50  179.80   32.41\n",
      "    5          6A   6A N   CA     29   30  148.03  147.00                  -40.90\n",
      "    6  10235   6A   7A C   N      32   34 -111.28 -134.00   22.72    0.76  -62.50  178.57   31.85\n",
      "    6          7A   7A N   CA     34   35  147.33  147.00                  -40.90\n",
      "    7  10236   7A   8G C   N      37   39  -67.89  -80.20   30.65    0.72   82.20 -156.43    6.69\n",
      "    7          8G   8G N   CA     39   40  146.03  174.10                    8.50\n",
      "    8  10237   8G   9A C   N      41   43 -150.09 -134.00   23.68    0.75  -62.50  177.81   33.82\n",
      "    8          9A   9A N   CA     43   44  164.37  147.00                  -40.90\n",
      "    9  10398 169G 170L C   N    1351 1353 -100.08  -63.50   55.75    7.04  -63.50   55.75    7.04\n",
      "    9        170L 170L N   CA   1353 1354    0.87  -41.20                  -41.20\n",
      "   10  10402 173V 174A C   N    1382 1384  -65.97  -68.20    8.25    0.59  -62.50  178.29   29.05\n",
      "   10        174A 174A N   CA   1384 1385  137.36  145.30                  -40.90\n",
      "   11  10403 174A 175D C   N    1387 1389 -159.80  -96.50   70.69    2.93  -63.30 -160.74   30.79\n",
      "   11        175D 175D N   CA   1389 1390  145.66  114.20                  -40.00\n",
      "   12  10405 176P 177D C   N    1402 1404  -93.49  -96.50   19.67    0.82  -63.30  176.25   20.11\n",
      "   12        177D 177D N   CA   1404 1405  133.64  114.20                  -40.00\n",
      "   13  10406 177D 178H C   N    1410 1412  -79.94  -67.60   37.33    3.19  -63.20  148.02   17.52\n",
      "   13        178H 178H N   CA   1412 1413  104.77  140.00                  -42.30\n",
      "   14  10407 178H 179D C   N    1420 1422  -65.40  -70.90    5.78    0.48  -63.30  167.93   20.86\n",
      "   14        179D 179D N   CA   1422 1423  152.09  150.30                  -40.00\n",
      "   15  10408 179D 180H C   N    1428 1430  156.48 -125.60   78.88    2.50  -63.20 -142.12   35.95\n",
      "   15        180H 180H N   CA   1430 1431  151.02  138.80                  -42.30\n",
      "   16  10409 180H 181T C   N    1438 1440  -88.97  -78.10   16.94    0.59  -63.20  157.24   21.88\n",
      "   16        181T 181T N   CA   1440 1441  162.79  149.80                  -42.10\n",
      "   17  10411 182G 183F C   N    1449 1451 -102.74 -124.20   30.75    0.83  -63.20  170.24   21.15\n",
      "   17        183F 183F N   CA   1451 1452  121.28  143.30                  -44.30\n",
      "   18  10412 183F 184L C   N    1460 1462 -145.32 -108.50   37.05    1.72  -63.50 -164.18   24.05\n",
      "   18        184L 184L N   CA   1462 1463  136.70  132.50                  -41.20\n",
      "   19  10413 184L 185T C   N    1468 1470 -124.44  -63.20   94.77   10.17  -63.20   94.77   10.17\n",
      "   19        185T 185T N   CA   1470 1471   30.23  -42.10                  -42.10\n",
      "   20  10415 186E 187Y C   N    1484 1486  -11.15  -98.40   87.26    3.36  -63.50 -179.43   31.31\n",
      "   20        187Y 187Y N   CA   1486 1487  129.41  128.40                  -43.40\n",
      "   21  10418 189A 190T C   N    1508 1510   40.64   55.90   89.10    5.17 -124.80  166.23    8.38\n",
      "   21        190T 190T N   CA   1510 1511  127.29   39.50                  143.50\n",
      "   22  10429 200L 201N C   N    1608 1610  -99.72 -119.90   43.81    2.34  -63.20  147.60   20.53\n",
      "   22        201N 201N N   CA   1610 1611  175.89  137.00                  -41.10\n",
      "   23  10430 201N 202S C   N    1616 1618   88.84 -136.60  137.26    4.78  -64.10 -148.06   17.47\n",
      "   23        202S 202S N   CA   1618 1619  178.29  151.20                  -35.00\n",
      "   24  10431 202S 203K C   N    1622 1624  -75.24  -70.20    5.58    0.35  -62.90  176.85   23.66\n",
      "   24        203K 203K N   CA   1624 1625  142.78  140.40                  -40.80\n",
      "   25  10432 203K 204G C   N    1631 1633 -178.06 -167.20   57.07    2.33   82.20  148.53   11.28\n",
      "   25        204G 204G N   CA   1633 1634  118.57  174.60                    8.50\n",
      "   26  10433 204G 205Y C   N    1635 1637 -125.69 -124.30   30.42    1.67  -63.50  160.91   22.80\n",
      "   26        205Y 205Y N   CA   1637 1638  105.01  135.40                  -43.40\n",
      "   27  10585 356P 357G C   N    2880 2882   98.29   82.20   17.98    1.40 -167.20 -175.83    7.43\n",
      "   27        357G 357G N   CA   2882 2883   16.53    8.50                  174.60\n",
      "   28  10587 358Y 359R C   N    2896 2898  -73.43  -72.10    1.37    0.10  -63.00  176.98   24.62\n",
      "   28        359R 359R N   CA   2898 2899  142.23  141.90                  -41.10\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 26                           : Distance restraints 4 (SDCH-SDCH)       \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  32421 144N 188V CB  CG1  1151 1501   16.66    4.75   11.90    7.35    4.75   11.90    7.35\n",
      "    2  32422 144N 189A CB  CB   1151 1507   20.95    4.80   16.16    9.92    4.80   16.16    9.92\n",
      "    3  32424 144N 188V CG  CG1  1152 1501   17.01    4.61   12.41    7.67    4.61   12.41    7.67\n",
      "    4  32425 144N 189A CG  CB   1152 1507   21.10    3.96   17.14   10.59    3.96   17.14   10.59\n",
      "    5  32427 144N 189A OD1 CB   1153 1507   20.97    3.41   17.56   10.89    3.41   17.56   10.89\n",
      "    6  32430 144N 188V ND2 CG1  1154 1501   17.47    3.89   13.58    8.45    3.89   13.58    8.45\n",
      "    7  32431 144N 189A ND2 CB   1154 1507   21.48    4.39   17.09   10.53    4.39   17.09   10.53\n",
      "    8  32443 146L 189A CG  CB   1167 1507   14.09    3.99   10.10    6.23    3.99   10.10    6.23\n",
      "    9  32447 146L 189A CD1 CB   1168 1507   13.76    4.52    9.24    5.68    4.52    9.24    5.68\n",
      "   10  32449 146L 189A CD2 CB   1169 1507   15.14    3.45   11.69    7.22    3.45   11.69    7.22\n",
      "   11  32715 188V 329F CG2 CB   1502 2641   13.15    4.73    8.42    5.15    4.73    8.42    5.15\n",
      "   12  32716 188V 329F CG2 CG   1502 2642   12.79    4.95    7.84    4.79    4.95    7.84    4.79\n",
      "   13  32717 188V 329F CG2 CD2  1502 2644   13.11    4.17    8.94    5.49    4.17    8.94    5.49\n",
      "   14  32718 188V 331F CG2 CD2  1502 2664   12.04    4.07    7.98    4.93    4.07    7.98    4.93\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:  1192 1504   2.175\n",
      "  serious non-bonded atom clash:  1196 1505   2.274\n",
      "  serious non-bonded atom clash:  1355 1499   2.153\n",
      "  serious non-bonded atom clash:  1357 1505   2.100\n",
      "  serious non-bonded atom clash:  1505 1562   2.016\n",
      "  serious non-bonded atom clash:  1524 1860   2.062\n",
      "  serious non-bonded atom clash:  1524 1861   1.501\n",
      "  serious non-bonded atom clash:  1525 1597   2.212\n",
      "  serious non-bonded atom clash:  1525 1598   2.175\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     3    4    2    2    8   15   22   65  106  178  181  210  337  476  471\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "5buj_MODLL.B99990001.pdb      9114.46094\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 5bvd\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    36662    34016\n",
      "310 (of 2915 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2698 atoms and 1 coordinate set(s) were parsed in 0.03s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     310\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    34016    2347\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      814\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :        3487.5496\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     319      14     15   0.094   0.094      827.58       1.000\n",
      " 2 Bond angle potential               :     437      17     30   6.865   6.865      424.26       1.000\n",
      " 3 Stereochemical cosine torsion poten:     202       0     11  52.308  52.308      89.680       1.000\n",
      " 4 Stereochemical improper torsion pot:     141       3      4   4.336   4.336      52.715       1.000\n",
      " 5 Soft-sphere overlap restraints     :     814      14     28   0.071   0.071      485.56       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     363      24     30   1.665   1.665      666.61       1.000\n",
      "10 Distance restraints 2 (N-O)        :     374      18     35   1.443   1.443      488.30       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      44       2      7  13.431  13.431      92.599       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      28       0      4  80.209  80.209      19.809       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      20       0      2  54.480  54.480      19.214       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       8       0      0  84.274  84.274      3.6689       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       5       0      0  93.536  93.536      3.9595       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     160       0      0   0.520   0.520      1.1234       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      46      26     11  47.104 128.718      145.60       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     200      16     19   2.678   2.678      166.88       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 5bvd_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:   10480.6250\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  1                           : Bond length potential                   \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   1516 187Y 187Y CB  CA   1488 1487    1.82    1.54    0.29    7.88    1.54    0.29    7.88\n",
      "    2   1527 187Y 187Y C   CA   1496 1487    2.10    1.49    0.61   17.59    1.49    0.61   17.59\n",
      "    3   1528 187Y 188V C   N    1496 1498    1.74    1.35    0.40   14.13    1.35    0.40   14.13\n",
      "    4   1532 188V 188V N   CA   1498 1499    1.92    1.43    0.49   16.20    1.43    0.49   16.20\n",
      "    5   1534 188V 188V C   CA   1503 1499    2.06    1.49    0.57   16.62    1.49    0.57   16.62\n",
      "    6   1535 188V 189A C   N    1503 1505    1.75    1.35    0.41   14.37    1.35    0.41   14.37\n",
      "    7   1537 189A 189A N   CA   1505 1506    1.90    1.43    0.47   15.30    1.43    0.47   15.30\n",
      "    8   1539 189A 189A C   CA   1508 1506    2.20    1.49    0.71   20.57    1.49    0.71   20.57\n",
      "    9   1540 189A 190T C   N    1508 1510    1.84    1.35    0.49   17.38    1.35    0.49   17.38\n",
      "   10   1541 190T 190T CB  CA   1512 1511    1.68    1.50    0.18    5.03    1.50    0.18    5.03\n",
      "   11   1544 190T 190T N   CA   1510 1511    1.96    1.43    0.53   17.42    1.43    0.53   17.42\n",
      "   12   1546 190T 190T C   CA   1515 1511    1.85    1.49    0.36   10.37    1.49    0.36   10.37\n",
      "   13   1547 190T 191R C   N    1515 1517    1.48    1.35    0.13    4.64    1.35    0.13    4.64\n",
      "   14   1555 191R 191R N   CA   1517 1518    1.59    1.43    0.16    5.14    1.43    0.16    5.14\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   5038 187Y 187Y N   CA   1486 1487  131.03  107.00   24.03    6.91  107.00   24.03    6.91\n",
      "    2   5039 187Y 187Y CA  CB   1487 1488  136.66  107.50   29.16    6.74  107.50   29.16    6.74\n",
      "    3   5041 187Y 187Y CA  C    1487 1496  142.74  116.50   26.24    7.54  116.50   26.24    7.54\n",
      "    4   5052 187Y 187Y C   CA   1496 1487  147.22  108.00   39.22    9.09  108.00   39.22    9.09\n",
      "    5   5053 187Y 188V C   N    1496 1498  158.78  120.00   38.78    8.81  120.00   38.78    8.81\n",
      "    6   5056 188V 188V N   CA   1498 1499  153.25  107.00   46.25   13.30  107.00   46.25   13.30\n",
      "    7   5060 188V 188V CA  C    1499 1503  142.02  116.50   25.52    7.34  116.50   25.52    7.34\n",
      "    8   5063 188V 189A C   N    1503 1505  145.96  120.00   25.96    5.90  120.00   25.96    5.90\n",
      "    9   5066 189A 189A N   CA   1505 1506  149.12  107.00   42.12   12.11  107.00   42.12   12.11\n",
      "   10   5068 189A 189A CA  C    1506 1508  146.23  116.50   29.73    8.55  116.50   29.73    8.55\n",
      "   11   5070 189A 190T C   N    1508 1510  145.72  120.00   25.72    5.84  120.00   25.72    5.84\n",
      "   12   5073 190T 190T N   CA   1510 1511  132.94  107.00   25.94    7.46  107.00   25.94    7.46\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  4                           : Stereochemical improper torsion potentia\n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   9633 190T 190T C   CA   1515 1511   14.21    0.00   14.21    5.00    0.00   14.21    5.00\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  9                           : Distance restraints 1 (CA-CA)           \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  12346  34G 190T CA  CA    237 1511   21.25   12.13    9.12   15.50   12.13    9.12   15.50\n",
      "    2  12371  35A 190T CA  CA    241 1511   21.16   12.26    8.90   15.51   12.26    8.90   15.51\n",
      "    3  14406 113Y 190T CA  CA    883 1511   18.39   13.56    4.84    8.73   13.56    4.84    8.73\n",
      "    4  15554 147H 190T CA  CA   1173 1511   15.87   12.03    3.84    6.89   12.03    3.84    6.89\n",
      "    5  15585 148R 190T CA  CA   1183 1511   12.47    8.92    3.55    6.92    8.92    3.55    6.92\n",
      "    6  15623 149D 190T CA  CA   1194 1511   12.10    6.41    5.69   11.56    6.41    5.69   11.56\n",
      "    7  15658 150L 190T CA  CA   1202 1511   12.58    8.39    4.19    8.18    8.39    4.19    8.18\n",
      "    8  15697 151K 190T CA  CA   1210 1511   12.40    7.82    4.57    9.25    7.82    4.57    9.25\n",
      "    9  15731 152P 190T CA  CA   1219 1511   15.32   10.86    4.46    8.34   10.86    4.46    8.34\n",
      "   10  15767 153S 190T CA  CA   1226 1511   17.36   10.90    6.46   12.12   10.90    6.46   12.12\n",
      "   11  15794 154N 190T CA  CA   1232 1511   17.54   11.07    6.47   12.08   11.07    6.47   12.08\n",
      "   12  15815 155L 190T CA  CA   1240 1511   19.01   13.96    5.06    8.67   13.96    5.06    8.67\n",
      "   13  15898 167D 190T CA  CA   1331 1511   19.96   12.60    7.35    9.57   12.60    7.35    9.57\n",
      "   14  15904 168F 190T CA  CA   1339 1511   20.07   13.55    6.53    5.82   13.55    6.53    5.82\n",
      "   15  15910 169G 190T CA  CA   1350 1511   20.04   12.73    7.31    5.05   12.73    7.31    5.05\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 10                           : Distance restraints 2 (N-O)             \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  21125 148R 190T N   O    1182 1516   13.09   10.06    3.03    4.96   10.06    3.03    4.96\n",
      "    2  21148 149D 190T N   O    1193 1516   11.76    7.88    3.88    6.75    7.88    3.88    6.75\n",
      "    3  21175 150L 190T N   O    1201 1516   11.79    7.60    4.19    7.36    7.60    4.19    7.36\n",
      "    4  21201 151K 190T N   O    1209 1516   12.54    8.00    4.54    8.06    8.00    4.54    8.06\n",
      "    5  21229 152P 190T N   O    1218 1516   13.92    9.44    4.47    7.69    9.44    4.47    7.69\n",
      "    6  21259 153S 190T N   O    1225 1516   15.79   10.86    4.93    8.26   10.86    4.93    8.26\n",
      "    7  21694 190T 147H N   O    1510 1181   16.10    9.19    6.91   11.44    9.19    6.91   11.44\n",
      "    8  21695 190T 148R N   O    1510 1192   11.95    5.30    6.64   12.07    5.30    6.64   12.07\n",
      "    9  21696 190T 149D N   O    1510 1200   14.69    6.57    8.12   14.47    6.57    8.12   14.47\n",
      "   10  21697 190T 150L N   O    1510 1208   12.14    6.75    5.39    9.62    6.75    5.39    9.62\n",
      "   11  21698 190T 151K N   O    1510 1217   16.47    9.98    6.49   10.97    9.98    6.49   10.97\n",
      "   12  21699 190T 167D N   O    1510 1337   19.45   10.56    8.89   11.41   10.56    8.89   11.41\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 13                           : Mainchain Omega dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10774 187Y 187Y CA  C    1487 1496 -154.53 -180.00   25.47    5.09 -180.00   25.47    5.09\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9 -125.45 -134.00    8.99    0.41  -62.50 -179.36   32.97\n",
      "    1          2A   2A N   CA      9   10  149.79  147.00                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14 -109.42 -134.00   24.71    0.73  -62.50 -179.18   32.12\n",
      "    2          3A   3A N   CA     14   15  144.47  147.00                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19 -115.25 -134.00   18.75    0.63  -62.50  179.78   32.27\n",
      "    3          4A   4A N   CA     19   20  147.23  147.00                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24 -108.03 -134.00   26.05    0.78  -62.50 -179.91   31.92\n",
      "    4          5A   5A N   CA     24   25  144.86  147.00                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29 -108.55 -134.00   25.74    0.72  -62.50 -178.12   32.24\n",
      "    5          6A   6A N   CA     29   30  143.15  147.00                  -40.90\n",
      "    6  10236   7A   8G C   N      37   39   95.55   78.70   31.96    0.50   82.20  158.80    8.38\n",
      "    6          8G   8G N   CA     39   40  166.74 -166.10                    8.50\n",
      "    7  10237   8G   9A C   N      41   43 -124.60 -134.00   10.74    0.55  -62.50  178.09   32.51\n",
      "    7          9A   9A N   CA     43   44  152.19  147.00                  -40.90\n",
      "    8  10402 173V 174A C   N    1382 1384   60.97  -68.20  129.78   11.41  -62.50 -156.93   30.70\n",
      "    8        174A 174A N   CA   1384 1385  157.88  145.30                  -40.90\n",
      "    9  10403 174A 175D C   N    1387 1389 -124.97  -96.50   29.15    1.19  -63.30  160.30   17.13\n",
      "    9        175D 175D N   CA   1389 1390  107.97  114.20                  -40.00\n",
      "   10  10405 176P 177D C   N    1402 1404 -120.92  -96.50   27.62    1.14  -63.30  176.75   19.12\n",
      "   10        177D 177D N   CA   1404 1405  127.10  114.20                  -40.00\n",
      "   11  10406 177D 178H C   N    1410 1412 -107.50 -125.60   37.63    1.06  -63.20  154.59   17.05\n",
      "   11        178H 178H N   CA   1412 1413  105.80  138.80                  -42.30\n",
      "   12  10407 178H 179D C   N    1420 1422 -104.82  -96.50   24.37    1.01  -63.30  137.70   14.99\n",
      "   12        179D 179D N   CA   1422 1423   91.29  114.20                  -40.00\n",
      "   13  10409 180H 181T C   N    1438 1440  -44.06  -78.10   54.30    1.86  -63.20  150.81   20.61\n",
      "   13        181T 181T N   CA   1440 1441  107.49  149.80                  -42.10\n",
      "   14  10410 181T 182G C   N    1445 1447 -114.94  -80.20   45.57    3.28   82.20 -147.76   16.49\n",
      "   14        182G 182G N   CA   1447 1448  144.61  174.10                    8.50\n",
      "   15  10411 182G 183F C   N    1449 1451 -119.22 -124.20    8.73    0.26  -63.20 -171.89   29.85\n",
      "   15        183F 183F N   CA   1451 1452  136.13  143.30                  -44.30\n",
      "   16  10412 183F 184L C   N    1460 1462   11.61  -70.70   83.41    7.11  -63.50 -174.84   30.36\n",
      "   16        184L 184L N   CA   1462 1463  128.04  141.60                  -41.20\n",
      "   17  10413 184L 185T C   N    1468 1470   12.36  -78.10   94.78    4.93  -63.20 -179.77   28.05\n",
      "   17        185T 185T N   CA   1470 1471  121.52  149.80                  -42.10\n",
      "   18  10414 185T 186E C   N    1475 1477 -148.64 -117.80   35.80    1.77  -63.60 -179.74   20.86\n",
      "   18        186E 186E N   CA   1477 1478  118.63  136.80                  -40.30\n",
      "   19  10415 186E 187Y C   N    1484 1486 -147.92 -124.30   30.30    2.49  -63.50 -179.25   25.22\n",
      "   19        187Y 187Y N   CA   1486 1487  116.42  135.40                  -43.40\n",
      "   20  10417 188V 189A C   N    1503 1505 -178.57 -134.00  107.00    6.60  -62.50  147.26   23.45\n",
      "   20        189A 189A N   CA   1505 1506   49.73  147.00                  -40.90\n",
      "   21  10418 189A 190T C   N    1508 1510   71.92   55.90  124.23    9.90 -124.80  164.40    8.20\n",
      "   21        190T 190T N   CA   1510 1511  162.70   39.50                  143.50\n",
      "   22  10419 190T 191R C   N    1515 1517 -116.09  -63.00   80.70   14.28  -63.00   80.70   14.28\n",
      "   22        191R 191R N   CA   1517 1518 -101.87  -41.10                  -41.10\n",
      "   23  10429 200L 201N C   N    1608 1610 -109.00 -119.90   63.96    3.14  -63.20  127.40   18.53\n",
      "   23        201N 201N N   CA   1610 1611 -159.98  137.00                  -41.10\n",
      "   24  10430 201N 202S C   N    1616 1618 -123.01 -136.60   27.46    1.04  -64.10  172.69   10.65\n",
      "   24        202S 202S N   CA   1618 1619  127.34  151.20                  -35.00\n",
      "   25  10431 202S 203K C   N    1622 1624 -147.83 -118.00   92.53    4.00  -62.90  125.58   21.20\n",
      "   25        203K 203K N   CA   1624 1625 -133.31  139.10                  -40.80\n",
      "   26  10587 358Y 359R C   N    2896 2898 -108.47 -125.20   17.49    0.83  -63.00  179.08   27.34\n",
      "   26        359R 359R N   CA   2898 2899  145.69  140.60                  -41.10\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 26                           : Distance restraints 4 (SDCH-SDCH)       \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  32607 149D 190T CG  CB   1196 1512   12.31    4.35    7.96    5.02    4.35    7.96    5.02\n",
      "    2  32608 149D 190T CG  OG1  1196 1513   11.02    3.40    7.61    4.83    3.40    7.61    4.83\n",
      "    3  32614 149D 190T OD1 CB   1197 1512   11.09    3.34    7.75    4.92    3.34    7.75    4.92\n",
      "    4  32615 149D 190T OD1 OG1  1197 1513    9.79    2.71    7.08    4.51    2.71    7.08    4.51\n",
      "    5  32616 149D 190T OD1 CG2  1197 1514   11.98    4.71    7.27    4.57    4.71    7.27    4.57\n",
      "    6  32623 149D 190T OD2 CB   1198 1512   13.21    4.55    8.65    5.45    4.55    8.65    5.45\n",
      "    7  32624 149D 190T OD2 OG1  1198 1513   11.91    3.32    8.59    5.46    3.32    8.59    5.46\n",
      "    8  32662 151K 190T CD  CB   1213 1512   12.19    3.93    8.25    5.33    3.93    8.25    5.33\n",
      "    9  32663 151K 190T CD  OG1  1213 1513   10.76    3.39    7.37    4.79    3.39    7.37    4.79\n",
      "   10  32664 151K 190T CD  CG2  1213 1514   12.64    4.02    8.62    5.57    4.02    8.62    5.57\n",
      "   11  32673 151K 190T CE  CB   1214 1512   13.49    4.60    8.89    5.71    4.60    8.89    5.71\n",
      "   12  32674 151K 190T CE  OG1  1214 1513   12.08    3.61    8.47    5.49    3.61    8.47    5.49\n",
      "   13  32675 151K 190T CE  CG2  1214 1514   14.00    4.78    9.22    5.91    4.78    9.22    5.91\n",
      "   14  32678 151K 190T NZ  CB   1215 1512   13.48    4.03    9.44    6.10    4.03    9.44    6.10\n",
      "   15  32679 151K 190T NZ  OG1  1215 1513   12.10    2.86    9.24    6.02    2.86    9.24    6.02\n",
      "   16  32680 151K 190T NZ  CG2  1215 1514   14.00    4.19    9.82    6.33    4.19    9.82    6.33\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:  1486 2041   2.274\n",
      "  serious non-bonded atom clash:  1487 2014   2.005\n",
      "  serious non-bonded atom clash:  1488 2014   2.093\n",
      "  serious non-bonded atom clash:  1510 1599   2.224\n",
      "  serious non-bonded atom clash:  1511 1564   1.963\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     3    0    2    1    7   13   35   59  121  198  202  249  371  438  502\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "5bvd_MODLL.B99990001.pdb      3487.54956\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 5bve\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    36684    34038\n",
      "310 (of 2915 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2675 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:      -90.0239\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     310\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    34038    2315\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      783\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :        3487.3174\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     319      12     16   0.092   0.092      785.57       1.000\n",
      " 2 Bond angle potential               :     437      20     32   6.971   6.971      434.27       1.000\n",
      " 3 Stereochemical cosine torsion poten:     202       0     10  52.188  52.188      84.315       1.000\n",
      " 4 Stereochemical improper torsion pot:     141       4      6   4.669   4.669      62.429       1.000\n",
      " 5 Soft-sphere overlap restraints     :     783      15     32   0.074   0.074      500.76       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     363      24     32   1.684   1.684      681.42       1.000\n",
      "10 Distance restraints 2 (N-O)        :     360      18     32   1.485   1.485      490.26       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      44       1      4   8.539   8.539      37.019       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      28       0      6  71.120  71.120      28.429       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      20       0      2  73.064  73.064      16.267       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       8       0      0  84.713  84.713      5.6351       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       5       0      0  97.491  97.491      4.6401       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     159       0      0   0.680   0.680      1.7772       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      46      27     15  52.258 129.437      166.32       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     183      15     20   2.953   2.953      188.21       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 5bve_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:   10716.9268\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  1                           : Bond length potential                   \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   1516 187Y 187Y CB  CA   1488 1487    1.73    1.54    0.19    5.35    1.54    0.19    5.35\n",
      "    2   1525 187Y 187Y N   CA   1486 1487    1.58    1.43    0.15    4.80    1.43    0.15    4.80\n",
      "    3   1527 187Y 187Y C   CA   1496 1487    2.14    1.49    0.65   18.98    1.49    0.65   18.98\n",
      "    4   1528 187Y 188V C   N    1496 1498    1.71    1.35    0.36   12.87    1.35    0.36   12.87\n",
      "    5   1532 188V 188V N   CA   1498 1499    1.88    1.43    0.45   14.84    1.43    0.45   14.84\n",
      "    6   1534 188V 188V C   CA   1503 1499    2.02    1.49    0.53   15.45    1.49    0.53   15.45\n",
      "    7   1535 188V 189A C   N    1503 1505    1.75    1.35    0.41   14.39    1.35    0.41   14.39\n",
      "    8   1537 189A 189A N   CA   1505 1506    1.89    1.43    0.46   15.24    1.43    0.46   15.24\n",
      "    9   1539 189A 189A C   CA   1508 1506    2.20    1.49    0.71   20.64    1.49    0.71   20.64\n",
      "   10   1540 189A 190T C   N    1508 1510    1.83    1.35    0.48   17.13    1.35    0.48   17.13\n",
      "   11   1544 190T 190T N   CA   1510 1511    1.96    1.43    0.53   17.36    1.43    0.53   17.36\n",
      "   12   1546 190T 190T C   CA   1515 1511    1.82    1.49    0.33    9.49    1.49    0.33    9.49\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   5035 186E 187Y C   N    1484 1486  143.55  120.00   23.55    5.35  120.00   23.55    5.35\n",
      "    2   5038 187Y 187Y N   CA   1486 1487  136.36  107.00   29.36    8.44  107.00   29.36    8.44\n",
      "    3   5039 187Y 187Y CA  CB   1487 1488  136.01  107.50   28.51    6.59  107.50   28.51    6.59\n",
      "    4   5041 187Y 187Y CA  C    1487 1496  145.22  116.50   28.72    8.26  116.50   28.72    8.26\n",
      "    5   5052 187Y 187Y C   CA   1496 1487  141.96  108.00   33.96    7.87  108.00   33.96    7.87\n",
      "    6   5053 187Y 188V C   N    1496 1498  157.80  120.00   37.80    8.59  120.00   37.80    8.59\n",
      "    7   5056 188V 188V N   CA   1498 1499  148.67  107.00   41.67   11.98  107.00   41.67   11.98\n",
      "    8   5060 188V 188V CA  C    1499 1503  141.44  116.50   24.94    7.17  116.50   24.94    7.17\n",
      "    9   5063 188V 189A C   N    1503 1505  147.08  120.00   27.08    6.15  120.00   27.08    6.15\n",
      "   10   5066 189A 189A N   CA   1505 1506  148.21  107.00   41.21   11.85  107.00   41.21   11.85\n",
      "   11   5068 189A 189A CA  C    1506 1508  146.94  116.50   30.44    8.75  116.50   30.44    8.75\n",
      "   12   5070 189A 190T C   N    1508 1510  144.26  120.00   24.26    5.51  120.00   24.26    5.51\n",
      "   13   5073 190T 190T N   CA   1510 1511  134.59  107.00   27.59    7.93  107.00   27.59    7.93\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  4                           : Stereochemical improper torsion potentia\n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   9633 190T 190T C   CA   1515 1511   16.74    0.00   16.74    5.89    0.00   16.74    5.89\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  9                           : Distance restraints 1 (CA-CA)           \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  12346  34G 190T CA  CA    237 1511   21.59   12.47    9.12   15.35   12.47    9.12   15.35\n",
      "    2  12369  35A 190T CA  CA    241 1511   21.32   12.42    8.90   15.32   12.42    8.90   15.32\n",
      "    3  14414 113Y 190T CA  CA    883 1511   18.41   13.49    4.92    8.99   13.49    4.92    8.99\n",
      "    4  15560 147H 190T CA  CA   1173 1511   15.92   12.00    3.91    7.03   12.00    3.91    7.03\n",
      "    5  15591 148R 190T CA  CA   1183 1511   12.48    8.92    3.56    6.95    8.92    3.56    6.95\n",
      "    6  15629 149D 190T CA  CA   1194 1511   12.16    6.42    5.73   11.64    6.42    5.73   11.64\n",
      "    7  15663 150L 190T CA  CA   1202 1511   12.62    8.34    4.27    8.35    8.34    4.27    8.35\n",
      "    8  15702 151K 190T CA  CA   1210 1511   12.48    7.83    4.65    9.40    7.83    4.65    9.40\n",
      "    9  15736 152P 190T CA  CA   1219 1511   15.43   10.86    4.56    8.54   10.86    4.56    8.54\n",
      "   10  15772 153S 190T CA  CA   1226 1511   17.43   10.88    6.55   12.28   10.88    6.55   12.28\n",
      "   11  15799 154N 190T CA  CA   1232 1511   17.63   11.07    6.55   12.24   11.07    6.55   12.24\n",
      "   12  15820 155L 190T CA  CA   1240 1511   19.12   13.95    5.18    8.89   13.95    5.18    8.89\n",
      "   13  15902 167D 190T CA  CA   1331 1511   20.07   12.63    7.44    9.66   12.63    7.44    9.66\n",
      "   14  15908 168F 190T CA  CA   1339 1511   20.18   13.62    6.56    5.85   13.62    6.56    5.85\n",
      "   15  15914 169G 190T CA  CA   1350 1511   20.23   12.91    7.33    5.05   12.91    7.33    5.05\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 10                           : Distance restraints 2 (N-O)             \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  21121 148R 190T N   O    1182 1516   13.02   10.10    2.93    4.80   10.10    2.93    4.80\n",
      "    2  21144 149D 190T N   O    1193 1516   11.70    7.88    3.82    6.64    7.88    3.82    6.64\n",
      "    3  21171 150L 190T N   O    1201 1516   11.81    7.60    4.21    7.40    7.60    4.21    7.40\n",
      "    4  21197 151K 190T N   O    1209 1516   12.57    7.94    4.63    8.23    7.94    4.63    8.23\n",
      "    5  21224 152P 190T N   O    1218 1516   14.02    9.40    4.62    7.95    9.40    4.62    7.95\n",
      "    6  21254 153S 190T N   O    1225 1516   15.87   10.79    5.08    8.54   10.79    5.08    8.54\n",
      "    7  21690 190T 147H N   O    1510 1181   16.04    9.13    6.91   11.45    9.13    6.91   11.45\n",
      "    8  21691 190T 148R N   O    1510 1192   11.96    5.28    6.68   12.15    5.28    6.68   12.15\n",
      "    9  21692 190T 149D N   O    1510 1200   14.61    6.45    8.16   14.58    6.45    8.16   14.58\n",
      "   10  21693 190T 150L N   O    1510 1208   12.11    6.72    5.39    9.62    6.72    5.39    9.62\n",
      "   11  21694 190T 151K N   O    1510 1217   16.50    9.97    6.53   11.05    9.97    6.53   11.05\n",
      "   12  21695 190T 167D N   O    1510 1337   19.50   10.56    8.94   11.46   10.56    8.94   11.46\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9 -110.83 -134.00   23.30    0.68  -62.50 -178.90   32.24\n",
      "    1          2A   2A N   CA      9   10  144.57  147.00                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14 -115.79 -134.00   18.24    0.65  -62.50  179.11   32.19\n",
      "    2          3A   3A N   CA     14   15  148.11  147.00                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19  -68.59  -68.20    4.94    0.38  -62.50  168.99   28.00\n",
      "    3          4A   4A N   CA     19   20  150.22  145.30                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24  -69.31  -68.20    3.46    0.24  -62.50  170.66   28.31\n",
      "    4          5A   5A N   CA     24   25  148.58  145.30                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29  -72.99  -68.20   10.43    0.68  -62.50  164.88   27.55\n",
      "    5          6A   6A N   CA     29   30  154.56  145.30                  -40.90\n",
      "    6  10235   6A   7A C   N      32   34  -67.68  -68.20    5.55    0.47  -62.50  168.36   27.85\n",
      "    6          7A   7A N   CA     34   35  150.83  145.30                  -40.90\n",
      "    7  10236   7A   8G C   N      37   39  -69.67  -80.20   26.23    0.62   82.20 -152.37    6.76\n",
      "    7          8G   8G N   CA     39   40  150.08  174.10                    8.50\n",
      "    8  10237   8G   9A C   N      41   43 -142.70 -134.00   16.29    0.63  -62.50  177.49   33.38\n",
      "    8          9A   9A N   CA     43   44  160.77  147.00                  -40.90\n",
      "    9  10402 173V 174A C   N    1382 1384   61.48  -68.20  129.68   10.96  -62.50 -145.71   32.27\n",
      "    9        174A 174A N   CA   1384 1385  144.31  145.30                  -40.90\n",
      "   10  10403 174A 175D C   N    1387 1389  -88.82  -70.90   18.09    1.44  -63.30  174.03   23.09\n",
      "   10        175D 175D N   CA   1389 1390  147.85  150.30                  -40.00\n",
      "   11  10405 176P 177D C   N    1402 1404 -107.39  -96.50   32.22    1.34  -63.30  131.49   14.19\n",
      "   11        177D 177D N   CA   1404 1405   83.88  114.20                  -40.00\n",
      "   12  10406 177D 178H C   N    1410 1412  -59.98  -67.60    9.68    0.58  -63.20  176.35   22.25\n",
      "   12        178H 178H N   CA   1412 1413  134.02  140.00                  -42.30\n",
      "   13  10407 178H 179D C   N    1420 1422  -80.10  -96.50   44.05    1.86  -63.30  114.55   13.21\n",
      "   13        179D 179D N   CA   1422 1423   73.31  114.20                  -40.00\n",
      "   14  10409 180H 181T C   N    1438 1440 -134.24 -124.80   72.89    2.85  -63.20  124.41   20.40\n",
      "   14        181T 181T N   CA   1440 1441 -144.23  143.50                  -42.10\n",
      "   15  10410 181T 182G C   N    1445 1447  113.15   78.70   73.48    1.09   82.20  124.40    7.54\n",
      "   15        182G 182G N   CA   1447 1448  128.99 -166.10                    8.50\n",
      "   16  10411 182G 183F C   N    1449 1451 -105.35  -63.20   89.10   15.06  -63.20   89.10   15.06\n",
      "   16        183F 183F N   CA   1451 1452 -122.80  -44.30                  -44.30\n",
      "   17  10412 183F 184L C   N    1460 1462  -56.50  -70.70   60.91    4.11  -63.50  123.77   17.60\n",
      "   17        184L 184L N   CA   1462 1463   82.37  141.60                  -41.20\n",
      "   18  10413 184L 185T C   N    1468 1470 -116.42 -124.80   19.81    1.08  -63.20  165.25   24.72\n",
      "   18        185T 185T N   CA   1470 1471  161.45  143.50                  -42.10\n",
      "   19  10414 185T 186E C   N    1475 1477 -178.61 -117.80   60.82    2.25  -63.60 -149.54   24.32\n",
      "   19        186E 186E N   CA   1477 1478  135.95  136.80                  -40.30\n",
      "   20  10415 186E 187Y C   N    1484 1486  171.49 -124.30  131.01    5.15  -63.50  141.83   26.72\n",
      "   20        187Y 187Y N   CA   1486 1487 -110.40  135.40                  -43.40\n",
      "   21  10417 188V 189A C   N    1503 1505 -176.81 -134.00  105.92    6.54  -62.50  146.12   23.21\n",
      "   21        189A 189A N   CA   1505 1506   50.12  147.00                  -40.90\n",
      "   22  10418 189A 190T C   N    1508 1510   73.69   55.90  122.90    9.94 -124.80  162.47    8.14\n",
      "   22        190T 190T N   CA   1510 1511  161.10   39.50                  143.50\n",
      "   23  10419 190T 191R C   N    1515 1517 -127.81  -63.00  105.38   18.44  -63.00  105.38   18.44\n",
      "   23        191R 191R N   CA   1517 1518 -124.20  -41.10                  -41.10\n",
      "   24  10429 200L 201N C   N    1608 1610 -117.53 -119.90   28.29    1.34  -63.20  163.03   23.47\n",
      "   24        201N 201N N   CA   1610 1611  165.19  137.00                  -41.10\n",
      "   25  10431 202S 203K C   N    1622 1624 -162.08 -118.00   50.83    1.69  -62.90 -176.15   29.76\n",
      "   25        203K 203K N   CA   1624 1625  164.40  139.10                  -40.80\n",
      "   26  10433 204G 205Y C   N    1635 1637 -124.04 -124.30    1.72    0.08  -63.50 -172.83   26.92\n",
      "   26        205Y 205Y N   CA   1637 1638  133.70  135.40                  -43.40\n",
      "   27  10587 358Y 359R C   N    2896 2898 -108.23 -125.20   18.23    0.90  -63.00  177.52   27.11\n",
      "   27        359R 359R N   CA   2898 2899  147.24  140.60                  -41.10\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 26                           : Distance restraints 4 (SDCH-SDCH)       \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  32646 149D 190T CG  CB   1196 1512   12.68    4.54    8.14    5.13    4.54    8.14    5.13\n",
      "    2  32647 149D 190T CG  OG1  1196 1513   13.17    3.55    9.62    6.10    3.55    9.62    6.10\n",
      "    3  32654 149D 190T OD1 CB   1197 1512   11.48    3.46    8.02    5.09    3.46    8.02    5.09\n",
      "    4  32655 149D 190T OD1 OG1  1197 1513   11.99    2.73    9.26    5.90    2.73    9.26    5.90\n",
      "    5  32664 149D 190T OD2 CB   1198 1512   13.60    4.84    8.76    5.50    4.84    8.76    5.50\n",
      "    6  32665 149D 190T OD2 OG1  1198 1513   14.16    3.59   10.57    6.70    3.59   10.57    6.70\n",
      "    7  32701 151K 190T CG  OG1  1212 1513   12.83    4.69    8.14    5.22    4.69    8.14    5.22\n",
      "    8  32707 151K 190T CD  CB   1213 1512   12.24    3.89    8.35    5.39    3.89    8.35    5.39\n",
      "    9  32708 151K 190T CD  OG1  1213 1513   12.70    3.34    9.37    6.08    3.34    9.37    6.08\n",
      "   10  32717 151K 190T CE  CB   1214 1512   13.53    4.54    8.99    5.77    4.54    8.99    5.77\n",
      "   11  32718 151K 190T CE  OG1  1214 1513   14.05    3.55   10.50    6.80    3.55   10.50    6.80\n",
      "   12  32719 151K 190T CE  CG2  1214 1514   12.13    4.77    7.36    4.71    4.77    7.36    4.71\n",
      "   13  32722 151K 190T NZ  CB   1215 1512   13.40    3.91    9.50    6.13    3.91    9.50    6.13\n",
      "   14  32723 151K 190T NZ  OG1  1215 1513   14.06    2.74   11.32    7.37    2.74   11.32    7.37\n",
      "   15  32724 151K 190T NZ  CG2  1215 1514   12.07    4.10    7.97    5.14    4.10    7.97    5.14\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:   458 2726   2.286\n",
      "  serious non-bonded atom clash:  1486 2014   2.078\n",
      "  serious non-bonded atom clash:  1487 2014   2.167\n",
      "  serious non-bonded atom clash:  1488 2041   2.288\n",
      "  serious non-bonded atom clash:  1510 1599   2.257\n",
      "  serious non-bonded atom clash:  1511 1564   1.889\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     2    1    3    3    5   14   32   65  133  200  195  249  357  448  470\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "5bve_MODLL.B99990001.pdb      3487.31738\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 5bvf\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "    205     1  197   204      Y     T   13.419\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    36282    33636\n",
      "313 (of 2915 total) atoms selected for optimization\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     313\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    33636    1892\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      797\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :        2270.0437\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     324      13     17   0.069   0.069      445.39       1.000\n",
      " 2 Bond angle potential               :     446      19     29   6.978   6.978      408.97       1.000\n",
      " 3 Stereochemical cosine torsion poten:     230       0     10  50.761  50.761      91.312       1.000\n",
      " 4 Stereochemical improper torsion pot:     145       1      4   3.096   3.096      27.954       1.000\n",
      " 5 Soft-sphere overlap restraints     :     797      25     48   0.088   0.088      729.74       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     224       8     23   0.980   0.980      189.40       1.000\n",
      "10 Distance restraints 2 (N-O)        :     233       7     16   1.017   1.017      165.63       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      42       2      3   7.769   7.769      29.895       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      27       0      1  90.164  90.164      10.364       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      21       0      1  69.631  69.631      11.827       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       9       0      0  87.854  87.854      4.4140       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       4       0      0 112.469 112.469      2.9171       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      73       0      0   0.822   0.822      1.7059       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      43      31     11  46.586 142.112      144.22       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      71       0      0   0.839   0.839      6.3139       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 5bvf_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:   10619.7969\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  1                           : Bond length potential                   \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   1586 193Y 193Y C   CA   1552 1543    1.67    1.49    0.18    5.29    1.49    0.18    5.29\n",
      "    2   1587 193Y 194R C   N    1552 1554    1.54    1.35    0.20    6.98    1.35    0.20    6.98\n",
      "    3   1589 194R 194R CG  CB   1557 1556    1.80    1.53    0.27    7.50    1.53    0.27    7.50\n",
      "    4   1595 194R 194R N   CA   1554 1555    1.72    1.43    0.29    9.42    1.43    0.29    9.42\n",
      "    5   1597 194R 194R C   CA   1563 1555    1.98    1.49    0.49   14.29    1.49    0.49   14.29\n",
      "    6   1598 194R 195A C   N    1563 1565    1.66    1.35    0.31   11.00    1.35    0.31   11.00\n",
      "    7   1600 195A 195A N   CA   1565 1566    1.73    1.43    0.30   10.00    1.43    0.30   10.00\n",
      "    8   1602 195A 195A C   CA   1568 1566    1.84    1.49    0.35   10.13    1.49    0.35   10.13\n",
      "    9   1603 195A 196P C   N    1568 1570    1.65    1.30    0.35   10.25    1.30    0.35   10.25\n",
      "   10   1605 196P 196P C   CA   1575 1571    1.93    1.49    0.44   12.66    1.49    0.44   12.66\n",
      "   11   1606 196P 197E C   N    1575 1577    1.63    1.35    0.28   10.07    1.35    0.28   10.07\n",
      "   12   1607 196P 196P N   CA   1570 1571    1.89    1.43    0.46   15.05    1.43    0.46   15.05\n",
      "   13   1617 197E 197E N   CA   1577 1578    1.73    1.43    0.30    9.93    1.43    0.30    9.93\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   5120 193Y 193Y N   CA   1542 1543  127.35  107.00   20.35    5.85  107.00   20.35    5.85\n",
      "    2   5123 193Y 193Y CA  C    1543 1552  132.44  116.50   15.94    4.58  116.50   15.94    4.58\n",
      "    3   5135 193Y 194R C   N    1552 1554  170.96  120.00   50.96   11.58  120.00   50.96   11.58\n",
      "    4   5138 194R 194R N   CA   1554 1555  147.44  107.00   40.44   11.63  107.00   40.44   11.63\n",
      "    5   5142 194R 194R CB  CG   1556 1557  140.55  113.60   26.95    6.62  113.60   26.95    6.62\n",
      "    6   5148 194R 194R C   CA   1563 1555  129.12  108.00   21.12    4.89  108.00   21.12    4.89\n",
      "    7   5149 194R 195A C   N    1563 1565  164.42  120.00   44.42   10.10  120.00   44.42   10.10\n",
      "    8   5152 195A 195A N   CA   1565 1566  144.26  107.00   37.26   10.71  107.00   37.26   10.71\n",
      "    9   5156 195A 196P C   N    1568 1570  142.31  117.00   25.31    6.30  117.00   25.31    6.30\n",
      "   10   5159 196P 196P N   CA   1570 1571  148.58  108.20   40.38    9.18  108.20   40.38    9.18\n",
      "   11   5163 196P 196P CA  C    1571 1575  142.58  116.50   26.08    7.50  116.50   26.08    7.50\n",
      "   12   5168 196P 197E C   N    1575 1577  140.08  120.00   20.08    4.56  120.00   20.08    4.56\n",
      "   13   5170 197E 197E N   CA   1577 1578  143.67  113.50   30.17    8.11  113.50   30.17    8.11\n",
      "   14   5171 197E 197E N   CA   1577 1578  136.28  107.00   29.28    8.42  107.00   29.28    8.42\n",
      "   15   5172 197E 197E CA  CB   1578 1579  136.07  113.50   22.57    5.54  113.50   22.57    5.54\n",
      "   16   5175 197E 197E CB  CG   1579 1580  128.05  108.00   20.05    4.65  108.00   20.05    4.65\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  9                           : Distance restraints 1 (CA-CA)           \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  15778 153S 197E CA  CA   1226 1578   16.78   13.52    3.26    5.83   13.52    3.26    5.83\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 10                           : Distance restraints 2 (N-O)             \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  21679 197E 148R N   O    1577 1192   11.25    8.44    2.81    4.93    8.44    2.81    4.93\n",
      "    2  21680 197E 149D N   O    1577 1200   13.89    9.27    4.62    8.06    9.27    4.62    8.06\n",
      "    3  21681 197E 150L N   O    1577 1208   11.51    8.28    3.24    5.82    8.28    3.24    5.82\n",
      "    4  21689 197E 231K N   O    1577 1848   11.21    8.24    2.97    4.94    8.24    2.97    4.94\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9 -111.45 -134.00   22.60    0.69  -62.50 -179.62   32.16\n",
      "    1          2A   2A N   CA      9   10  145.49  147.00                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14 -110.94 -134.00   23.13    0.70  -62.50 -179.43   32.16\n",
      "    2          3A   3A N   CA     14   15  145.15  147.00                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19 -112.98 -134.00   21.04    0.66  -62.50 -179.71   32.23\n",
      "    3          4A   4A N   CA     19   20  146.03  147.00                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24 -111.09 -134.00   22.96    0.70  -62.50 -179.70   32.12\n",
      "    4          5A   5A N   CA     24   25  145.48  147.00                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29 -109.83 -134.00   24.42    0.68  -62.50 -178.16   32.31\n",
      "    5          6A   6A N   CA     29   30  143.53  147.00                  -40.90\n",
      "    6  10235   6A   7A C   N      32   34 -118.74 -134.00   15.52    0.62  -62.50  178.37   32.23\n",
      "    6          7A   7A N   CA     34   35  149.83  147.00                  -40.90\n",
      "    7  10236   7A   8G C   N      37   39 -131.32 -167.20   39.84    0.61   82.20 -151.20   15.99\n",
      "    7          8G   8G N   CA     39   40  157.29  174.60                    8.50\n",
      "    8  10237   8G   9A C   N      41   43 -121.38 -134.00   12.65    0.45  -62.50 -178.89   32.82\n",
      "    8          9A   9A N   CA     43   44  147.83  147.00                  -40.90\n",
      "    9  10402 173V 174A C   N    1382 1384   65.84  -68.20  134.25   11.63  -62.50 -149.85   31.79\n",
      "    9        174A 174A N   CA   1384 1385  152.70  145.30                  -40.90\n",
      "   10  10403 174A 175D C   N    1387 1389 -116.95  -96.50   24.83    1.03  -63.30  176.63   19.22\n",
      "   10        175D 175D N   CA   1389 1390  128.29  114.20                  -40.00\n",
      "   11  10405 176P 177D C   N    1402 1404 -122.93  -96.50   26.55    1.08  -63.30  163.03   17.48\n",
      "   11        177D 177D N   CA   1404 1405  111.74  114.20                  -40.00\n",
      "   12  10406 177D 178H C   N    1410 1412 -113.80 -125.60   37.33    1.18  -63.20  154.23   16.79\n",
      "   12        178H 178H N   CA   1412 1413  103.39  138.80                  -42.30\n",
      "   13  10407 178H 179D C   N    1420 1422 -113.65  -96.50   24.84    1.04  -63.30  179.37   19.65\n",
      "   13        179D 179D N   CA   1422 1423  132.16  114.20                  -40.00\n",
      "   14  10408 179D 180H C   N    1428 1430 -116.75 -125.60   17.06    0.47  -63.20  174.91   19.17\n",
      "   14        180H 180H N   CA   1430 1431  124.21  138.80                  -42.30\n",
      "   15  10409 180H 181T C   N    1438 1440 -141.13 -124.80   22.64    0.74  -63.20  176.82   27.75\n",
      "   15        181T 181T N   CA   1440 1441  159.18  143.50                  -42.10\n",
      "   16  10410 181T 182G C   N    1445 1447   88.47   78.70   26.15    0.41   82.20  161.26    8.14\n",
      "   16        182G 182G N   CA   1447 1448  169.64 -166.10                    8.50\n",
      "   17  10411 182G 183F C   N    1449 1451 -101.69 -124.20   22.66    0.96  -63.20  174.07   26.76\n",
      "   17        183F 183F N   CA   1451 1452  145.94  143.30                  -44.30\n",
      "   18  10412 183F 184L C   N    1460 1462  -95.43 -108.50   15.28    0.67  -63.50  168.83   21.82\n",
      "   18        184L 184L N   CA   1462 1463  124.59  132.50                  -41.20\n",
      "   19  10413 184L 185T C   N    1468 1470 -151.63 -124.80   54.16    1.70  -63.20  155.05   25.42\n",
      "   19        185T 185T N   CA   1470 1471 -169.45  143.50                  -42.10\n",
      "   20  10414 185T 186E C   N    1475 1477  -77.78  -69.30    9.85    0.58  -63.60  172.78   23.82\n",
      "   20        186E 186E N   CA   1477 1478  147.51  142.50                  -40.30\n",
      "   21  10415 186E 187Y C   N    1484 1486 -130.42 -124.30    9.76    0.40  -63.50 -173.95   32.87\n",
      "   21        187Y 187Y N   CA   1486 1487  143.00  135.40                  -43.40\n",
      "   22  10416 187Y 188V C   N    1496 1498  -59.18  -62.40   22.77    2.71 -125.40  165.58    9.39\n",
      "   22        188V 188V N   CA   1498 1499  -64.94  -42.40                  143.30\n",
      "   23  10417 188V 189A C   N    1503 1505  173.33 -134.00  119.36    7.36  -62.50  148.14   24.17\n",
      "   23        189A 189A N   CA   1505 1506   39.89  147.00                  -40.90\n",
      "   24  10418 189A 190T C   N    1508 1510  -98.50  -63.20   85.43   13.27  -63.20   85.43   13.27\n",
      "   24        190T 190T N   CA   1510 1511 -119.90  -42.10                  -42.10\n",
      "   25  10419 190T 191R C   N    1515 1517   59.27   57.30   18.77    1.38  -63.00  156.55   28.39\n",
      "   25        191R 191R N   CA   1517 1518   56.66   38.00                  -41.10\n",
      "   26  10420 191R 192W C   N    1526 1528  -95.94  -71.30   65.96    5.53  -63.00  126.38   14.23\n",
      "   26        192W 192W N   CA   1528 1529   77.82  139.00                  -44.20\n",
      "   27  10422 193Y 194R C   N    1552 1554   -7.68   57.30   79.79    6.16  -63.00  137.07   22.30\n",
      "   27        194R 194R N   CA   1554 1555   84.31   38.00                  -41.10\n",
      "   28  10423 194R 195A C   N    1563 1565 -135.03 -134.00   39.10    2.24  -62.50  165.54   24.83\n",
      "   28        195A 195A N   CA   1565 1566  107.91  147.00                  -40.90\n",
      "   29  10425 196P 197E C   N    1575 1577  -95.69  -63.60  139.87   20.46  -63.60  139.87   20.46\n",
      "   29        197E 197E N   CA   1577 1578 -176.44  -40.30                  -40.30\n",
      "   30  10427 198I 199M C   N    1592 1594  -94.85  -63.40   57.08    7.06  -63.40   57.08    7.06\n",
      "   30        199M 199M N   CA   1594 1595    7.13  -40.50                  -40.50\n",
      "   31  10587 358Y 359R C   N    2896 2898 -108.36 -125.20   17.73    0.85  -63.00  178.61   27.27\n",
      "   31        359R 359R N   CA   2898 2899  146.15  140.60                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:  1211 1598   2.209\n",
      "  serious non-bonded atom clash:  1554 2036   1.897\n",
      "  serious non-bonded atom clash:  1554 2077   2.221\n",
      "  serious non-bonded atom clash:  1557 2014   2.295\n",
      "  serious non-bonded atom clash:  1557 2036   2.083\n",
      "  serious non-bonded atom clash:  1565 2033   2.153\n",
      "  serious non-bonded atom clash:  1570 1901   2.038\n",
      "  serious non-bonded atom clash:  1571 1901   2.297\n",
      "  serious non-bonded atom clash:  1604 1646   2.199\n",
      "  serious non-bonded atom clash:  1605 1644   1.968\n",
      "  serious non-bonded atom clash:  1606 1644   2.096\n",
      "  serious non-bonded atom clash:  1606 1645   1.567\n",
      "  serious non-bonded atom clash:  1606 1646   1.693\n",
      "  serious non-bonded atom clash:  1614 1628   1.816\n",
      "  serious non-bonded atom clash:  1614 1629   2.175\n",
      "  serious non-bonded atom clash:  1614 1630   1.903\n",
      "  serious non-bonded atom clash:  1614 2417   2.215\n",
      "  serious non-bonded atom clash:  1615 1618   2.218\n",
      "  serious non-bonded atom clash:  1630 2416   1.381\n",
      "  serious non-bonded atom clash:  1630 2417   1.443\n",
      "  serious non-bonded atom clash:  1630 2418   1.319\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:    12    3    6    8    7   16   39   85  126  208  196  296  365  434  484\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "5bvf_MODLL.B99990001.pdb      2270.04370\n",
      "\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 5673 atoms and 1 coordinate set(s) were parsed in 0.05s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 5hd4\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    38556    35910\n",
      "108 (of 2915 total) atoms selected for optimization\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:      -90.4912\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     108\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    35910     683\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      136\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          65.8292\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     111       0      0   0.003   0.003     0.38249       1.000\n",
      " 2 Bond angle potential               :     152       0      0   1.962   1.962      11.481       1.000\n",
      " 3 Stereochemical cosine torsion poten:      68       0      0  47.562  47.562      22.536       1.000\n",
      " 4 Stereochemical improper torsion pot:      42       0      0   0.914   0.914     0.84561       1.000\n",
      " 5 Soft-sphere overlap restraints     :     136       0      0   0.001   0.001     0.10746E-01   1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      81       0      0   0.227   0.227      3.2526       1.000\n",
      "10 Distance restraints 2 (N-O)        :      81       0      2   0.452   0.452      11.528       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      17       0      0   2.625   2.625      1.3811       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       7       0      0  73.153  73.153     0.58327       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       6       0      0  51.642  51.642     0.16305       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      0  82.118  82.118      2.4026       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       1       0      0   1.848   1.848     0.55532       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      60       0      0   0.416   0.416     0.24383       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      17      12      0  20.794 148.388      9.8676       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      35       0      0   0.392   0.392     0.59585       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 5hd4_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    3048.1465\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9  -61.95  -68.20    6.99    0.48  -62.50  176.95   28.97\n",
      "    1          2A   2A N   CA      9   10  142.15  145.30                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14  -75.78  -68.20    7.59    0.64  -62.50  174.09   29.20\n",
      "    2          3A   3A N   CA     14   15  145.52  145.30                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19  -75.43  -68.20   23.70    1.66  -62.50  151.78   25.53\n",
      "    3          4A   4A N   CA     19   20  167.88  145.30                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24  -76.92  -68.20    8.95    0.69  -62.50  172.37   28.98\n",
      "    4          5A   5A N   CA     24   25  147.33  145.30                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29  -68.71  -68.20   16.56    1.35  -62.50  169.77   27.52\n",
      "    5          6A   6A N   CA     29   30  128.75  145.30                  -40.90\n",
      "    6  10235   6A   7A C   N      32   34  -77.52  -68.20    9.43    0.75  -62.50  173.03   29.12\n",
      "    6          7A   7A N   CA     34   35  146.73  145.30                  -40.90\n",
      "    7  10236   7A   8G C   N      37   39  106.89   78.70   58.73    0.88   82.20  136.13    7.82\n",
      "    7          8G   8G N   CA     39   40  142.38 -166.10                    8.50\n",
      "    8  10237   8G   9A C   N      41   43  -81.28  -68.20   18.77    1.18  -62.50  161.44   27.42\n",
      "    8          9A   9A N   CA     43   44  158.76  145.30                  -40.90\n",
      "    9  10238   9A  10G C   N      46   48  -91.68  -80.20   35.72    0.90   82.20 -123.31   18.27\n",
      "    9         10G  10G N   CA     48   49 -152.08  174.10                    8.50\n",
      "   10  10584 355Q 356P C   N    2873 2875  -55.99  -58.70   13.83    0.94  -64.50  168.96   13.06\n",
      "   10        356P 356P N   CA   2875 2876  -44.06  -30.50                  147.20\n",
      "   11  10586 357G 358Y C   N    2884 2886 -129.45 -124.30    5.21    0.41  -63.50 -170.17   27.14\n",
      "   11        358Y 358Y N   CA   2886 2887  134.60  135.40                  -43.40\n",
      "   12  10587 358Y 359R C   N    2896 2898 -127.99 -125.20    3.86    0.12  -63.00 -172.73   29.77\n",
      "   12        359R 359R N   CA   2898 2899  143.27  140.60                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    0    3    7   39  131  160  176  246  368  422  524\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "5hd4_MODLL.B99990001.pdb        65.82924\n",
      "\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 5674 atoms and 1 coordinate set(s) were parsed in 0.04s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 5hd7\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    38289    35643\n",
      "108 (of 2915 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2742 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     108\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    35643     648\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      124\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          69.5634\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     111       0      0   0.002   0.002     0.17702       1.000\n",
      " 2 Bond angle potential               :     152       0      0   2.002   2.002      11.818       1.000\n",
      " 3 Stereochemical cosine torsion poten:      68       0      0  46.661  46.661      22.757       1.000\n",
      " 4 Stereochemical improper torsion pot:      42       0      0   0.756   0.756     0.62821       1.000\n",
      " 5 Soft-sphere overlap restraints     :     124       0      0   0.001   0.001     0.15303E-01   1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      78       0      0   0.175   0.175      1.3796       1.000\n",
      "10 Distance restraints 2 (N-O)        :      74       0      2   0.523   0.523      14.139       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      17       0      0   2.455   2.455      1.2085       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       7       0      0  72.935  72.935      1.9832       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       6       0      1  61.050  61.050      2.9776       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      0  73.326  73.326      1.3079       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       1       0      0  14.141  14.141     0.75669       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      47       0      0   0.565   0.565     0.39143       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      17      11      1  17.188 139.491      9.1595       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      23       0      0   0.541   0.541     0.86433       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 5hd7_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    2811.0605\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10231   2A   3A C   N      12   14 -145.87 -134.00   22.65    0.88  -62.50  174.09   32.99\n",
      "    1          3A   3A N   CA     14   15  166.28  147.00                  -40.90\n",
      "    2  10232   3A   4A C   N      17   19  -69.47  -68.20    9.92    0.76  -62.50  164.10   27.24\n",
      "    2          4A   4A N   CA     19   20  155.15  145.30                  -40.90\n",
      "    3  10233   4A   5A C   N      22   24  -68.80  -68.20    7.31    0.57  -62.50  166.64   27.62\n",
      "    3          5A   5A N   CA     24   25  152.58  145.30                  -40.90\n",
      "    4  10234   5A   6A C   N      27   29  -77.92  -68.20   23.85    1.60  -62.50  152.80   25.83\n",
      "    4          6A   6A N   CA     29   30  167.08  145.30                  -40.90\n",
      "    5  10235   6A   7A C   N      32   34  -60.93  -68.20    8.23    0.80  -62.50  169.95   27.77\n",
      "    5          7A   7A N   CA     34   35  149.16  145.30                  -40.90\n",
      "    6  10236   7A   8G C   N      37   39  -75.85  -80.20   10.05    0.23   82.20 -137.55    7.00\n",
      "    6          8G   8G N   CA     39   40  165.04  174.10                    8.50\n",
      "    7  10237   8G   9A C   N      41   43 -131.69 -134.00    8.92    0.54  -62.50  177.53   32.80\n",
      "    7          9A   9A N   CA     43   44  155.61  147.00                  -40.90\n",
      "    8  10238   9A  10G C   N      46   48   69.54   78.70   30.00    0.51   82.20  146.59    7.75\n",
      "    8         10G  10G N   CA     48   49 -137.54 -166.10                    8.50\n",
      "    9  10584 355Q 356P C   N    2873 2875  -47.35  -58.70   33.92    2.06  -64.50  151.31   12.21\n",
      "    9        356P 356P N   CA   2875 2876  -62.47  -30.50                  147.20\n",
      "   10  10586 357G 358Y C   N    2884 2886 -125.11 -124.30    5.97    0.28  -63.50 -174.21   32.58\n",
      "   10        358Y 358Y N   CA   2886 2887  141.32  135.40                  -43.40\n",
      "   11  10587 358Y 359R C   N    2896 2898  -89.41  -72.10   17.92    1.46  -63.00 -179.69   22.69\n",
      "   11        359R 359R N   CA   2898 2899  137.26  141.90                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    0    2    7   45  109  158  190  247  328  427  495\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "5hd7_MODLL.B99990001.pdb        69.56343\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 5k4i\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    37249    34603\n",
      "229 (of 2915 total) atoms selected for optimization\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     229\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    34603    1428\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      369\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         156.5744\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     236       0      0   0.003   0.003     0.61543       1.000\n",
      " 2 Bond angle potential               :     325       0      1   1.811   1.811      19.535       1.000\n",
      " 3 Stereochemical cosine torsion poten:     155       0      7  51.406  51.406      60.266       1.000\n",
      " 4 Stereochemical improper torsion pot:     107       0      0   0.809   0.809      1.6450       1.000\n",
      " 5 Soft-sphere overlap restraints     :     369       0      0   0.004   0.004     0.68333       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     182       0      0   0.163   0.163      3.4995       1.000\n",
      "10 Distance restraints 2 (N-O)        :     172       0      0   0.369   0.369      6.7848       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      33       1      0  31.371  31.371      4.1272       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      19       0      0  76.337  76.337      5.6375       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      14       0      0  29.585  29.585      4.7273       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      0  95.941  95.941      2.7057       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       2       0      0 101.631 101.631      1.6127       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      92       0      0   0.327   0.327     0.24651       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      34      19      5  32.942 124.340      41.688       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      52       0      0   0.834   0.834      2.8011       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 5k4i_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    5306.8428\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9  -74.52  -68.20   10.10    0.64  -62.50  166.36   27.87\n",
      "    1          2A   2A N   CA      9   10  153.18  145.30                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14  -68.93  -68.20    3.83    0.28  -62.50  170.16   28.21\n",
      "    2          3A   3A N   CA     14   15  149.06  145.30                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19  -62.19  -68.20   18.91    1.32  -62.50  168.27   27.59\n",
      "    3          4A   4A N   CA     19   20  127.37  145.30                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24  -82.16  -68.20   18.63    1.18  -62.50  162.67   27.67\n",
      "    4          5A   5A N   CA     24   25  157.63  145.30                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29  -74.01  -68.20    5.87    0.47  -62.50  173.30   28.98\n",
      "    5          6A   6A N   CA     29   30  146.19  145.30                  -40.90\n",
      "    6  10235   6A   7A C   N      32   34  -81.30  -68.20   13.41    1.03  -62.50  171.96   29.14\n",
      "    6          7A   7A N   CA     34   35  148.17  145.30                  -40.90\n",
      "    7  10236   7A   8G C   N      37   39  164.56 -167.20   32.17    1.02   82.20  171.72   12.04\n",
      "    7          8G   8G N   CA     39   40  159.19  174.60                    8.50\n",
      "    8  10403 174A 175D C   N    1387 1389  106.06   54.50   87.41   10.32   54.50   87.41   10.32\n",
      "    8        175D 175D N   CA   1389 1390  111.48   40.90                   40.90\n",
      "    9  10405 176P 177D C   N    1402 1404  -91.75  -70.90   26.96    1.21  -63.30  155.25   20.97\n",
      "    9        177D 177D N   CA   1404 1405  167.38  150.30                  -40.00\n",
      "   10  10406 177D 178H C   N    1410 1412  -96.79 -125.60   63.71    1.82  -63.20  128.74   14.32\n",
      "   10        178H 178H N   CA   1412 1413   81.98  138.80                  -42.30\n",
      "   11  10407 178H 179D C   N    1420 1422 -119.17  -96.50   23.29    0.95  -63.30  159.02   17.10\n",
      "   11        179D 179D N   CA   1422 1423  108.89  114.20                  -40.00\n",
      "   12  10408 179D 180H C   N    1428 1430  -69.51  -67.60    1.92    0.15  -63.20  177.95   22.64\n",
      "   12        180H 180H N   CA   1430 1431  139.86  140.00                  -42.30\n",
      "   13  10409 180H 181T C   N    1438 1440 -130.12 -124.80   55.48    2.22  -63.20  136.69   21.83\n",
      "   13        181T 181T N   CA   1440 1441 -161.28  143.50                  -42.10\n",
      "   14  10412 183F 184L C   N    1460 1462  -67.72  -70.70    7.31    0.67  -63.50  170.57   23.93\n",
      "   14        184L 184L N   CA   1462 1463  148.28  141.60                  -41.20\n",
      "   15  10413 184L 185T C   N    1468 1470  -73.47  -78.10    6.86    0.44  -63.20  163.36   21.63\n",
      "   15        185T 185T N   CA   1470 1471  154.86  149.80                  -42.10\n",
      "   16  10414 185T 186E C   N    1475 1477 -131.47 -117.80   15.28    0.45  -63.60 -171.30   28.99\n",
      "   16        186E 186E N   CA   1477 1478  143.63  136.80                  -40.30\n",
      "   17  10415 186E 187Y C   N    1484 1486 -121.23 -124.30    3.17    0.26  -63.50 -171.34   27.26\n",
      "   17        187Y 187Y N   CA   1486 1487  136.21  135.40                  -43.40\n",
      "   18  10417 188V 189A C   N    1503 1505  -93.80  -68.20   37.55    3.69  -62.50  161.80   25.17\n",
      "   18        189A 189A N   CA   1505 1506  117.84  145.30                  -40.90\n",
      "   19  10587 358Y 359R C   N    2896 2898 -102.67 -125.20   29.03    0.86  -63.00  168.14   20.34\n",
      "   19        359R 359R N   CA   2898 2899  122.30  140.60                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:   491  528   2.251\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    1    0    3    8   11   41  108  184  190  266  358  449  484\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "5k4i_MODLL.B99990001.pdb       156.57437\n",
      "\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 5562 atoms and 1 coordinate set(s) were parsed in 0.04s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 5ke0\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    37876    35230\n",
      "165 (of 2915 total) atoms selected for optimization\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     165\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    35230     937\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      189\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          85.1507\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     169       0      0   0.003   0.003     0.30916       1.000\n",
      " 2 Bond angle potential               :     229       0      0   1.635   1.635      11.518       1.000\n",
      " 3 Stereochemical cosine torsion poten:     102       0      1  46.663  46.663      32.476       1.000\n",
      " 4 Stereochemical improper torsion pot:      66       0      0   0.580   0.580     0.54541       1.000\n",
      " 5 Soft-sphere overlap restraints     :     189       0      0   0.001   0.001     0.23531E-01   1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      92       0      0   0.084   0.084     0.51075       1.000\n",
      "10 Distance restraints 2 (N-O)        :      80       0      0   0.149   0.149     0.90599       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      24       0      0   2.650   2.650      1.9880       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      13       0      0  86.861  86.861      5.7423       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      10       0      0  67.368  67.368      2.0543       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       8       0      0  92.666  92.666      4.9980       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       2       0      0 139.245 139.245      1.3008       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      58       0      0   0.370   0.370     0.16854       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      24      17      3  25.569 147.478      21.577       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      60       0      0   0.459   0.459      1.0334       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 5ke0_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    4367.2627\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9 -119.27 -134.00   14.76    0.53  -62.50 -179.74   32.57\n",
      "    1          2A   2A N   CA      9   10  148.02  147.00                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14 -127.85 -134.00    6.89    0.16  -62.50 -173.00   34.14\n",
      "    2          3A   3A N   CA     14   15  143.89  147.00                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19  -83.82  -68.20   20.90    1.33  -62.50  161.33   27.54\n",
      "    3          4A   4A N   CA     19   20  159.19  145.30                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24  -79.27  -68.20   12.04    0.85  -62.50  169.90   28.70\n",
      "    4          5A   5A N   CA     24   25  150.04  145.30                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29  -87.97  -68.20   28.39    1.79  -62.50  155.52   26.81\n",
      "    5          6A   6A N   CA     29   30  165.68  145.30                  -40.90\n",
      "    6  10235   6A   7A C   N      32   34  -76.50  -68.20   22.20    1.51  -62.50  153.84   25.92\n",
      "    6          7A   7A N   CA     34   35  165.90  145.30                  -40.90\n",
      "    7  10236   7A   8G C   N      37   39 -134.53 -167.20   36.19    0.55   82.20 -152.18   15.86\n",
      "    7          8G   8G N   CA     39   40  159.05  174.60                    8.50\n",
      "    8  10237   8G   9A C   N      41   43  -78.96  -68.20   10.87    0.87  -62.50  172.99   29.19\n",
      "    8          9A   9A N   CA     43   44  146.90  145.30                  -40.90\n",
      "    9  10238   9A  10G C   N      46   48   90.16   78.70   44.38    0.82   82.20  142.74    7.33\n",
      "    9         10G  10G N   CA     48   49  151.02 -166.10                    8.50\n",
      "   10  10239  10G  11P C   N      50   52  -69.06  -58.70   14.27    1.03  -64.50  167.95   12.77\n",
      "   10         11P  11P N   CA     52   53  -20.69  -30.50                  147.20\n",
      "   11  10240  11P  12E C   N      57   59 -104.31 -117.80   15.35    0.45  -63.60  174.59   21.31\n",
      "   11         12E  12E N   CA     59   60  129.47  136.80                  -40.30\n",
      "   12  10243  14V  15R C   N      81   83  -65.23  -72.10   27.95    2.29  -63.00  149.92   20.40\n",
      "   12         15R  15R N   CA     83   84  169.00  141.90                  -41.10\n",
      "   13  10244  15R  16G C   N      92   94  -76.72  -80.20    6.18    0.41   82.20 -126.76    7.03\n",
      "   13         16G  16G N   CA     94   95  179.21  174.10                    8.50\n",
      "   14  10245  16G  17Q C   N      96   98  -78.03  -73.00   12.37    0.72  -63.80  168.29   25.50\n",
      "   14         17Q  17Q N   CA     98   99  152.01  140.70                  -40.30\n",
      "   15  10585 356P 357G C   N    2880 2882   90.80   78.70   38.83    0.66  -80.20  171.86   10.90\n",
      "   15        357G 357G N   CA   2882 2883  157.00 -166.10                  174.10\n",
      "   16  10586 357G 358Y C   N    2884 2886 -135.26 -124.30   27.76    1.11  -63.50  171.43   30.74\n",
      "   16        358Y 358Y N   CA   2886 2887  160.91  135.40                  -43.40\n",
      "   17  10587 358Y 359R C   N    2896 2898  -76.18  -72.10   12.60    0.88  -63.00  165.61   23.27\n",
      "   17        359R 359R N   CA   2898 2899  153.82  141.90                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    0    0    8   45  115  180  162  242  349  435  479\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "5ke0_MODLL.B99990001.pdb        85.15075\n",
      "\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2817 atoms and 1 coordinate set(s) were parsed in 0.03s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 5lcj\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    38082    35436\n",
      "188 (of 2915 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2822 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     188\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    35436    1758\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      319\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         170.4357\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     194       0      0   0.004   0.004      1.0607       1.000\n",
      " 2 Bond angle potential               :     268       0      1   2.319   2.319      28.026       1.000\n",
      " 3 Stereochemical cosine torsion poten:     118       0      6  55.165  55.165      55.298       1.000\n",
      " 4 Stereochemical improper torsion pot:      77       0      0   1.323   1.323      3.4996       1.000\n",
      " 5 Soft-sphere overlap restraints     :     319       0      0   0.001   0.001     0.76396E-01   1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     327       0      0   0.182   0.182      7.7804       1.000\n",
      "10 Distance restraints 2 (N-O)        :     299       0      1   0.290   0.290      12.109       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      29       0      1   5.713   5.713      9.7164       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      17       0      0  74.147  74.147      3.5865       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      13       0      0  49.010  49.010      4.7520       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       7       0      0  82.825  82.825      4.3710       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       1       0      0  35.365  35.365      1.2156       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     235       0      0   0.674   0.674      1.5910       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      31      13      6  39.869 117.678      33.323       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     142       0      0   0.624   0.624      4.0312       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 5lcj_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    4261.7803\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9 -160.81 -134.00   29.08    0.66  -62.50 -171.50   36.12\n",
      "    1          2A   2A N   CA      9   10  158.27  147.00                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14 -166.69 -134.00   52.66    1.81  -62.50  167.24   32.85\n",
      "    2          3A   3A N   CA     14   15 -171.72  147.00                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19 -131.96 -134.00   10.72    0.64  -62.50  175.88   32.54\n",
      "    3          4A   4A N   CA     19   20  157.52  147.00                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24  -70.93  -68.20    7.54    0.70  -62.50  179.37   28.99\n",
      "    4          5A   5A N   CA     24   25  138.28  145.30                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29 -135.54 -134.00   10.19    0.61  -62.50 -167.76   28.96\n",
      "    5          6A   6A N   CA     29   30  136.92  147.00                  -40.90\n",
      "    6  10237   8G   9A C   N      41   43  -79.75  -68.20   17.56    1.10  -62.50  161.49   27.34\n",
      "    6          9A   9A N   CA     43   44  158.54  145.30                  -40.90\n",
      "    7  10238   9A  10G C   N      46   48  158.38 -167.20   35.68    0.52   82.20 -168.71   12.82\n",
      "    7         10G  10G N   CA     48   49 -176.02  174.60                    8.50\n",
      "    8  10389 160T 161C C   N    1283 1285   76.86   57.40   24.90    1.69 -117.90 -155.40    8.83\n",
      "    8        161C 161C N   CA   1285 1286   20.46   36.00                  141.10\n",
      "    9  10560 331F 332D C   N    2668 2670   96.76  -70.90  175.20   14.47  -96.50 -171.96    7.79\n",
      "    9        332D 332D N   CA   2670 2671 -158.87  150.30                  114.20\n",
      "   10  10561 332D 333M C   N    2676 2678 -131.13 -125.60   57.75    2.87  -63.40  140.86   17.42\n",
      "   10        333M 333M N   CA   2678 2679   83.01  140.50                  -40.50\n",
      "   11  10584 355Q 356P C   N    2873 2875  -48.23  -58.70   19.10    1.14  -64.50  167.13   13.35\n",
      "   11        356P 356P N   CA   2875 2876  -46.47  -30.50                  147.20\n",
      "   12  10586 357G 358Y C   N    2884 2886 -126.13 -124.30   17.42    0.85  -63.50  175.43   30.97\n",
      "   12        358Y 358Y N   CA   2886 2887  152.73  135.40                  -43.40\n",
      "   13  10587 358Y 359R C   N    2896 2898  -57.02  -72.10   20.59    1.79  -63.00  163.09   21.64\n",
      "   13        359R 359R N   CA   2898 2899  155.92  141.90                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    1    3   11   55  117  141  166  245  339  447  483\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "5lcj_MODLL.B99990001.pdb       170.43575\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 5lck\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    38054    35408\n",
      "191 (of 2915 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2692 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     191\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    35408    1942\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      423\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         104.4000\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     199       0      0   0.004   0.004     0.79102       1.000\n",
      " 2 Bond angle potential               :     276       0      0   2.106   2.106      23.229       1.000\n",
      " 3 Stereochemical cosine torsion poten:     137       0      0  43.071  43.071      38.969       1.000\n",
      " 4 Stereochemical improper torsion pot:      89       0      0   0.934   0.934      1.7178       1.000\n",
      " 5 Soft-sphere overlap restraints     :     423       0      0   0.002   0.002     0.12425       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     374       0      0   0.127   0.127      3.0953       1.000\n",
      "10 Distance restraints 2 (N-O)        :     324       0      0   0.289   0.289      3.9993       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      29       1      0  33.453  33.453      5.2505       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      17       0      0  71.621  71.621      2.5852       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      12       0      0  36.141  36.141      3.8753       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      0  24.515  24.515      2.3946       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       1       0      0 176.315 176.315     0.50457       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     222       0      0   0.645   0.645      2.3229       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      31      12      2  24.351 110.135      8.7383       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     226       0      0   0.650   0.650      6.8020       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 5lck_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    4070.1938\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9  -76.69  -68.20    9.89    0.66  -62.50  169.32   28.47\n",
      "    1          2A   2A N   CA      9   10  150.37  145.30                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14  -79.39  -68.20   12.70    0.87  -62.50  168.63   28.50\n",
      "    2          3A   3A N   CA     14   15  151.32  145.30                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19 -137.13 -134.00   25.77    1.39  -62.50  164.44   30.94\n",
      "    3          4A   4A N   CA     19   20  172.57  147.00                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24  -57.78  -68.20   10.66    0.82  -62.50  176.12   28.63\n",
      "    4          5A   5A N   CA     24   25  143.05  145.30                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29 -114.63 -134.00   19.38    0.66  -62.50  179.38   32.17\n",
      "    5          6A   6A N   CA     29   30  147.47  147.00                  -40.90\n",
      "    6  10235   6A   7A C   N      32   34  -66.55  -68.20    5.47    0.50  -62.50  168.64   27.84\n",
      "    6          7A   7A N   CA     34   35  150.51  145.30                  -40.90\n",
      "    7  10237   8G   9A C   N      41   43  -53.57  -68.20   14.74    1.31  -62.50  172.21   27.80\n",
      "    7          9A   9A N   CA     43   44  147.12  145.30                  -40.90\n",
      "    8  10355 126I 127C C   N    1000 1002  -61.61  -63.00    4.10    0.43 -117.90 -177.18    7.89\n",
      "    8        127C 127C N   CA   1002 1003  -44.96  -41.10                  141.10\n",
      "    9  10356 127C 128Y C   N    1006 1008  -64.00  -63.50    0.70    0.13   55.90  146.05   17.87\n",
      "    9        128Y 128Y N   CA   1008 1009  -43.89  -43.40                   39.50\n",
      "   10  10585 356P 357G C   N    2880 2882  109.38   78.70   31.21    1.42  -80.20  171.00   10.95\n",
      "   10        357G 357G N   CA   2882 2883 -171.82 -166.10                  174.10\n",
      "   11  10586 357G 358Y C   N    2884 2886 -124.12 -124.30   22.66    1.21  -63.50  169.73   29.97\n",
      "   11        358Y 358Y N   CA   2886 2887  158.06  135.40                  -43.40\n",
      "   12  10587 358Y 359R C   N    2896 2898  -59.84  -72.10   12.38    0.91  -63.00  178.69   23.93\n",
      "   12        359R 359R N   CA   2898 2899  140.24  141.90                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    0    2   12   56  105  133  176  235  335  437  484\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "5lck_MODLL.B99990001.pdb       104.40004\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 5ngu\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "    337     1  332   338      D     L   13.444\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    37245    34599\n",
      "241 (of 2915 total) atoms selected for optimization\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     241\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    34599    1920\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      444\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :        1243.5934\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     248       5      5   0.058   0.058      231.60       1.000\n",
      " 2 Bond angle potential               :     341       8      8   4.381   4.381      117.09       1.000\n",
      " 3 Stereochemical cosine torsion poten:     151       0      5  47.298  47.298      55.502       1.000\n",
      " 4 Stereochemical improper torsion pot:     100       0      0   1.359   1.359      3.8305       1.000\n",
      " 5 Soft-sphere overlap restraints     :     444       0      0   0.005   0.005      1.0645       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     317      21     26   1.445   1.445      489.03       1.000\n",
      "10 Distance restraints 2 (N-O)        :     299      11     17   1.443   1.443      253.37       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      35       0      0   2.975   2.975      2.2703       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      23       0      0  66.942  66.942      2.3073       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      19       0      0  72.237  72.237      7.5862       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       9       0      0  96.454  96.454      5.8182       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       2       0      0   5.983   5.983     0.86377       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     183       0      0   0.554   0.554      1.1378       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      37      16      6  29.599 117.426      31.203       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     156       0      7   1.552   1.552      40.913       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 5ngu_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    5931.4570\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  1                           : Bond length potential                   \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   2783 337D 337D C   CA   2717 2712    1.89    1.49    0.40   11.76    1.49    0.40   11.76\n",
      "    2   2784 337D 338L C   N    2717 2719    1.67    1.35    0.32   11.39    1.35    0.32   11.39\n",
      "    3   2789 338L 338L N   CA   2719 2720    1.81    1.43    0.38   12.48    1.43    0.38   12.48\n",
      "    4   2791 338L 338L C   CA   2725 2720    1.94    1.49    0.45   13.16    1.49    0.45   13.16\n",
      "    5   2792 338L 339P C   N    2725 2727    1.75    1.30    0.45   13.43    1.30    0.45   13.43\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6754 337D 337D N   CA   2711 2712  123.79  107.00   16.79    4.83  107.00   16.79    4.83\n",
      "    2   6757 337D 337D CA  C    2712 2717  140.01  116.50   23.51    6.76  116.50   23.51    6.76\n",
      "    3   6762 337D 338L C   N    2717 2719  147.49  120.00   27.49    6.25  120.00   27.49    6.25\n",
      "    4   6765 338L 338L N   CA   2719 2720  139.54  107.00   32.54    9.36  107.00   32.54    9.36\n",
      "    5   6768 338L 338L CA  C    2720 2725  150.94  112.50   38.44    5.52  112.50   38.44    5.52\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  9                           : Distance restraints 1 (CA-CA)           \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  13352  74I 337D CA  CA    559 2712   13.91   11.12    2.78    5.88   11.12    2.78    5.88\n",
      "    2  15278 141H 337D CA  CA   1129 2712   20.02   13.99    6.03   10.45   13.99    6.03   10.45\n",
      "    3  15333 143A 337D CA  CA   1145 2712   18.36   11.94    6.42   11.66   11.94    6.42   11.66\n",
      "    4  15363 144N 337D CA  CA   1150 2712   16.70    9.93    6.78   12.82    9.93    6.78   12.82\n",
      "    5  15397 145V 337D CA  CA   1158 2712   14.91    9.21    5.70   10.80    9.21    5.70   10.80\n",
      "    6  15440 146L 337D CA  CA   1165 2712   15.65   11.10    4.55    8.13   11.10    4.55    8.13\n",
      "    7  15480 147H 337D CA  CA   1173 2712   16.31   13.15    3.16    5.26   13.15    3.16    5.26\n",
      "    8  15903 172R 337D CA  CA   1367 2712   10.34    6.60    3.74    6.71    6.60    3.74    6.71\n",
      "    9  15923 173V 337D CA  CA   1378 2712   11.44    5.26    6.18   11.58    5.26    6.18   11.58\n",
      "   10  15943 174A 337D CA  CA   1385 2712   14.32    8.29    6.04   10.97    8.29    6.04   10.97\n",
      "   11  15962 175D 337D CA  CA   1390 2712   14.42    9.02    5.40    9.97    9.02    5.40    9.97\n",
      "   12  15982 176P 337D CA  CA   1398 2712   18.01   12.87    5.14    8.48   12.87    5.14    8.48\n",
      "   13  15996 177D 337D CA  CA   1405 2712   18.26   13.68    4.58    6.82   13.68    4.58    6.82\n",
      "   14  16011 178H 337D CA  CA   1413 2712   15.60   12.30    3.29    5.52   12.30    3.29    5.52\n",
      "   15  16040 179D 337D CA  CA   1423 2712   17.05   13.90    3.15    5.41   13.90    3.15    5.41\n",
      "   16  16803 204G 337D CA  CA   1634 2712   16.56   13.93    2.63    5.08   13.93    2.63    5.08\n",
      "   17  16815 205Y 337D CA  CA   1638 2712   15.02   11.06    3.96    8.15   11.06    3.96    8.15\n",
      "   18  16830 206T 337D CA  CA   1650 2712   18.38   13.53    4.86    9.14   13.53    4.86    9.14\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 10                           : Distance restraints 2 (N-O)             \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  21902 172R 337D N   O    1366 2718   13.27   10.06    3.20    4.96   10.06    3.20    4.96\n",
      "    2  21925 173V 337D N   O    1377 2718   12.50    7.56    4.94    8.41    7.56    4.94    8.41\n",
      "    3  21948 174A 337D N   O    1384 2718   15.13    9.46    5.66    9.43    9.46    5.66    9.43\n",
      "    4  21969 175D 337D N   O    1389 2718   15.45   10.50    4.95    8.07   10.50    4.95    8.07\n",
      "    5  25112 337D 143A N   O    2711 1148   15.26    9.15    6.11   10.82    9.15    6.11   10.82\n",
      "    6  25113 337D 144N N   O    2711 1156   13.42    7.62    5.80   10.70    7.62    5.80   10.70\n",
      "    7  25115 337D 172R N   O    2711 1376   11.43    7.80    3.63    5.97    7.80    3.63    5.97\n",
      "    8  25116 337D 173V N   O    2711 1383   10.36    6.04    4.32    7.63    6.04    4.32    7.63\n",
      "    9  25117 337D 174A N   O    2711 1388   12.94    7.25    5.68    9.99    7.25    5.68    9.99\n",
      "   10  25118 337D 175D N   O    2711 1396   14.04   10.54    3.50    5.71   10.54    3.50    5.71\n",
      "   11  25119 337D 205Y N   O    2711 1648   14.66   10.81    3.85    6.68   10.81    3.85    6.68\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9  -63.18  -68.20    5.04    0.44  -62.50  173.37   28.45\n",
      "    1          2A   2A N   CA      9   10  145.73  145.30                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14 -129.01 -134.00   11.44    0.71  -62.50  174.94   32.23\n",
      "    2          3A   3A N   CA     14   15  157.30  147.00                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19  -70.21  -68.20   10.67    0.79  -62.50  163.50   27.18\n",
      "    3          4A   4A N   CA     19   20  155.78  145.30                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24 -101.03  -68.20   36.51    3.52  -62.50  174.54   26.98\n",
      "    4          5A   5A N   CA     24   25  129.34  145.30                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29  -78.23  -68.20   16.10    1.01  -62.50  161.98   27.35\n",
      "    5          6A   6A N   CA     29   30  157.89  145.30                  -40.90\n",
      "    6  10236   7A   8G C   N      37   39  -72.04  -80.20   18.96    0.44   82.20 -145.90    6.84\n",
      "    6          8G   8G N   CA     39   40  156.99  174.10                    8.50\n",
      "    7  10237   8G   9A C   N      41   43  -93.75  -68.20   25.57    2.13  -62.50  175.62   30.41\n",
      "    7          9A   9A N   CA     43   44  146.29  145.30                  -40.90\n",
      "    8  10238   9A  10G C   N      46   48   63.85   78.70   23.37    0.44   82.20  157.63    8.57\n",
      "    8         10G  10G N   CA     48   49 -148.05 -166.10                    8.50\n",
      "    9  10389 160T 161C C   N    1283 1285   74.16   57.40   22.48    1.40 -117.90 -153.54    8.89\n",
      "    9        161C 161C N   CA   1285 1286   21.02   36.00                  141.10\n",
      "   10  10561 332D 333M C   N    2676 2678 -172.93 -125.60   78.34    2.57  -63.40  159.96   31.80\n",
      "   10        333M 333M N   CA   2678 2679 -157.08  140.50                  -40.50\n",
      "   11  10563 334E 335L C   N    2693 2695 -118.15 -108.50   42.02    2.16  -63.50  155.34   24.95\n",
      "   11        335L 335L N   CA   2695 2696  173.39  132.50                  -41.20\n",
      "   12  10564 335L 336D C   N    2701 2703  -78.75  -70.90   43.23    1.76  -63.30  128.12   16.78\n",
      "   12        336D 336D N   CA   2703 2704 -167.19  150.30                  -40.00\n",
      "   13  10583 354F 355Q C   N    2864 2866  -89.93  -73.00   16.97    1.23  -63.80 -178.20   25.43\n",
      "   13        355Q 355Q N   CA   2866 2867  139.61  140.70                  -40.30\n",
      "   14  10585 356P 357G C   N    2880 2882  -65.07  -80.20   22.16    0.67   82.20 -150.21    6.51\n",
      "   14        357G 357G N   CA   2882 2883  157.91  174.10                    8.50\n",
      "   15  10586 357G 358Y C   N    2884 2886  -71.51  -98.40   26.92    1.06  -63.50  170.86   26.70\n",
      "   15        358Y 358Y N   CA   2886 2887  127.28  128.40                  -43.40\n",
      "   16  10587 358Y 359R C   N    2896 2898  -98.56  -72.10   29.36    1.98  -63.00  168.10   25.17\n",
      "   16        359R 359R N   CA   2898 2899  154.61  141.90                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:   104  266   1.432\n",
      "  serious non-bonded atom clash:   431 2739   1.781\n",
      "  serious non-bonded atom clash:   431 2740   2.122\n",
      "  serious non-bonded atom clash:   432 2738   2.216\n",
      "  serious non-bonded atom clash:   432 2739   0.997\n",
      "  serious non-bonded atom clash:   432 2740   1.932\n",
      "  serious non-bonded atom clash:   433 2740   1.854\n",
      "  serious non-bonded atom clash:   434 2740   1.223\n",
      "  serious non-bonded atom clash:   440 2740   1.193\n",
      "  serious non-bonded atom clash:   441 2740   1.853\n",
      "  serious non-bonded atom clash:   442 2740   2.145\n",
      "  serious non-bonded atom clash:   707 2835   2.237\n",
      "  serious non-bonded atom clash:   728  754   2.006\n",
      "  serious non-bonded atom clash:   740  745   1.905\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2649 atoms and 1 coordinate set(s) were parsed in 0.06s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:    10    2    2    4    2   14   20   56   92  134  201  231  385  417  468\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "5ngu_MODLL.B99990001.pdb      1243.59338\n",
      "\n",
      "Modelando proteína 5nhf\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "    336     1  332   337      D     D   13.003\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    37053    34407\n",
      "241 (of 2915 total) atoms selected for optimization\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     241\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    34407    1923\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      453\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         732.4971\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     248       5      5   0.037   0.037      103.83       1.000\n",
      " 2 Bond angle potential               :     339       5      6   3.219   3.219      74.380       1.000\n",
      " 3 Stereochemical cosine torsion poten:     148       0      4  51.989  51.989      60.504       1.000\n",
      " 4 Stereochemical improper torsion pot:      99       0      0   1.260   1.260      4.2242       1.000\n",
      " 5 Soft-sphere overlap restraints     :     453       0      1   0.007   0.007      2.8518       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     318      13     23   1.121   1.121      251.73       1.000\n",
      "10 Distance restraints 2 (N-O)        :     298       6     15   1.168   1.168      127.48       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      35       0      1   5.287   5.287      10.441       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      23       0      0  87.102  87.102      7.9743       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      19       0      0  71.976  71.976      8.3808       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:      10       0      0  97.079  97.079      8.6927       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       2       0      0  91.535  91.535      1.1104       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     198       0      0   0.642   0.642      2.4483       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      37      20      9  38.828 129.074      47.422       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     149       0      2   1.112   1.112      21.030       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 5nhf_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    6705.0674\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  1                           : Bond length potential                   \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   2775 336D 336D C   CA   2709 2704    1.77    1.49    0.28    8.07    1.49    0.28    8.07\n",
      "    2   2776 336D 337D C   N    2709 2711    1.56    1.35    0.22    7.63    1.35    0.22    7.63\n",
      "    3   2781 337D 337D N   CA   2711 2712    1.68    1.43    0.25    8.36    1.43    0.25    8.36\n",
      "    4   2783 337D 337D C   CA   2717 2712    1.77    1.49    0.28    8.23    1.49    0.28    8.23\n",
      "    5   2784 337D 338L C   N    2717 2719    1.61    1.35    0.26    9.31    1.35    0.26    9.31\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6746 336D 336D CA  C    2704 2709  134.79  116.50   18.29    5.26  116.50   18.29    5.26\n",
      "    2   6751 336D 337D C   N    2709 2711  141.33  120.00   21.33    4.85  120.00   21.33    4.85\n",
      "    3   6754 337D 337D N   CA   2711 2712  126.23  107.00   19.23    5.53  107.00   19.23    5.53\n",
      "    4   6757 337D 337D CA  C    2712 2717  137.60  116.50   21.10    6.07  116.50   21.10    6.07\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  9                           : Distance restraints 1 (CA-CA)           \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  13330  74I 336D CA  CA    559 2704   13.42   11.23    2.19    4.62   11.23    2.19    4.62\n",
      "    2  15240 141H 336D CA  CA   1129 2704   18.16   13.70    4.46    7.79   13.70    4.46    7.79\n",
      "    3  15295 143A 336D CA  CA   1145 2704   16.50   11.54    4.95    9.08   11.54    4.95    9.08\n",
      "    4  15324 144N 336D CA  CA   1150 2704   14.61    9.55    5.07    9.74    9.55    5.07    9.74\n",
      "    5  15358 145V 336D CA  CA   1158 2704   13.07    8.92    4.15    7.89    8.92    4.15    7.89\n",
      "    6  15402 146L 336D CA  CA   1165 2704   13.70   10.65    3.04    5.48   10.65    3.04    5.48\n",
      "    7  15883 173V 336D CA  CA   1378 2704    9.21    4.81    4.40    8.22    4.81    4.40    8.22\n",
      "    8  15903 174A 336D CA  CA   1385 2704   11.94    7.77    4.17    7.61    7.77    4.17    7.61\n",
      "    9  15922 175D 336D CA  CA   1390 2704   11.87    8.45    3.42    6.44    8.45    3.42    6.44\n",
      "   10  15943 176P 336D CA  CA   1398 2704   15.47   12.27    3.20    5.39   12.27    3.20    5.39\n",
      "   11  16794 206T 336D CA  CA   1650 2704   16.19   13.14    3.05    5.80   13.14    3.05    5.80\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 10                           : Distance restraints 2 (N-O)             \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  21866 173V 336D N   O    1377 2710   10.13    7.09    3.04    5.23    7.09    3.04    5.23\n",
      "    2  21889 174A 336D N   O    1384 2710   12.72    9.08    3.63    6.10    9.08    3.63    6.10\n",
      "    3  25016 336D 143A N   O    2703 1148   13.71    8.89    4.82    8.58    8.89    4.82    8.58\n",
      "    4  25017 336D 144N N   O    2703 1156   11.68    7.56    4.12    7.63    7.56    4.12    7.63\n",
      "    5  25022 336D 174A N   O    2703 1388   10.43    6.72    3.71    6.59    6.72    3.71    6.59\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9  -72.79  -68.20    5.07    0.35  -62.50  171.96   28.70\n",
      "    1          2A   2A N   CA      9   10  147.45  145.30                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14  -71.04  -68.20    2.91    0.22  -62.50  173.38   28.84\n",
      "    2          3A   3A N   CA     14   15  145.93  145.30                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19  -76.52  -68.20   10.25    0.67  -62.50  168.41   28.31\n",
      "    3          4A   4A N   CA     19   20  151.28  145.30                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24  -82.79  -68.20   19.53    1.93  -62.50  174.41   27.66\n",
      "    4          5A   5A N   CA     24   25  132.33  145.30                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29  -72.39  -68.20    7.81    0.50  -62.50  167.50   27.95\n",
      "    5          6A   6A N   CA     29   30  151.89  145.30                  -40.90\n",
      "    6  10235   6A   7A C   N      32   34  -69.46  -68.20    7.90    0.59  -62.50  166.15   27.57\n",
      "    6          7A   7A N   CA     34   35  153.10  145.30                  -40.90\n",
      "    7  10236   7A   8G C   N      37   39   83.42   78.70   19.91    0.38   82.20  166.06    8.12\n",
      "    7          8G   8G N   CA     39   40  174.56 -166.10                    8.50\n",
      "    8  10237   8G   9A C   N      41   43  -73.54  -68.20    7.53    0.47  -62.50  168.86   28.23\n",
      "    8          9A   9A N   CA     43   44  150.60  145.30                  -40.90\n",
      "    9  10238   9A  10G C   N      46   48  -66.04  -80.20   29.85    0.70   82.20 -156.56    6.59\n",
      "    9         10G  10G N   CA     48   49  147.82  174.10                    8.50\n",
      "   10  10240  11P  12E C   N      57   59  -77.03  -69.30    7.76    0.56  -63.60  177.03   24.35\n",
      "   10         12E  12E N   CA     59   60  143.17  142.50                  -40.30\n",
      "   11  10389 160T 161C C   N    1283 1285   70.68   57.40   16.71    1.17 -117.90 -153.45    8.84\n",
      "   11        161C 161C N   CA   1285 1286   25.87   36.00                  141.10\n",
      "   12  10560 331F 332D C   N    2668 2670 -135.86  -63.30   76.69   12.99  -63.30   76.69   12.99\n",
      "   12        332D 332D N   CA   2670 2671  -64.82  -40.00                  -40.00\n",
      "   13  10561 332D 333M C   N    2676 2678 -136.59 -125.60   61.14    2.68  -63.40  139.58   26.47\n",
      "   13        333M 333M N   CA   2678 2679 -159.35  140.50                  -40.50\n",
      "   14  10563 334E 335L C   N    2693 2695 -144.76 -108.50   46.11    2.05  -63.50  177.51   29.67\n",
      "   14        335L 335L N   CA   2695 2696  160.98  132.50                  -41.20\n",
      "   15  10565 336D 337D C   N    2709 2711 -151.42  -70.90   90.93    4.98   54.50 -143.83   26.98\n",
      "   15        337D 337D N   CA   2711 2712 -167.47  150.30                   40.90\n",
      "   16  10566 337D 338L C   N    2717 2719  112.20 -108.50  139.32    6.66  -70.70  177.46   16.48\n",
      "   16        338L 338L N   CA   2719 2720  130.32  132.50                  141.60\n",
      "   17  10583 354F 355Q C   N    2864 2866  -73.23  -73.00    0.36    0.03  -63.80  179.52   26.89\n",
      "   17        355Q 355Q N   CA   2866 2867  140.42  140.70                  -40.30\n",
      "   18  10585 356P 357G C   N    2880 2882  144.06 -167.20   50.03    1.25   82.20  166.73   10.99\n",
      "   18        357G 357G N   CA   2882 2883  163.33  174.60                    8.50\n",
      "   19  10586 357G 358Y C   N    2884 2886 -105.96  -98.40    8.56    0.49  -63.50  173.07   25.43\n",
      "   19        358Y 358Y N   CA   2886 2887  124.38  128.40                  -43.40\n",
      "   20  10587 358Y 359R C   N    2896 2898  -67.88  -72.10    8.62    0.58  -63.00  175.54   23.39\n",
      "   20        359R 359R N   CA   2898 2899  134.38  141.90                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:   113  398   2.064\n",
      "  serious non-bonded atom clash:   395  780   1.829\n",
      "  serious non-bonded atom clash:   407  779   2.214\n",
      "  serious non-bonded atom clash:   408  779   1.552\n",
      "  serious non-bonded atom clash:   431 2739   1.160\n",
      "  serious non-bonded atom clash:   431 2740   1.424\n",
      "  serious non-bonded atom clash:   432 2738   1.812\n",
      "  serious non-bonded atom clash:   432 2739   0.536\n",
      "  serious non-bonded atom clash:   432 2740   1.432\n",
      "  serious non-bonded atom clash:   433 2739   2.121\n",
      "  serious non-bonded atom clash:   433 2740   1.340\n",
      "  serious non-bonded atom clash:   434 2740   1.341\n",
      "  serious non-bonded atom clash:   765 2775   2.176\n",
      "  serious non-bonded atom clash:   779  783   2.108\n",
      "  serious non-bonded atom clash:  1426 1637   2.183\n",
      "  serious non-bonded atom clash:  1938 2159   1.671\n",
      "  serious non-bonded atom clash:  1939 2158   1.554\n",
      "  serious non-bonded atom clash:  1939 2159   0.515\n",
      "  serious non-bonded atom clash:  1940 2158   2.014\n",
      "  serious non-bonded atom clash:  1940 2159   1.937\n",
      "  serious non-bonded atom clash:  1941 2158   2.045\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2796 atoms and 1 coordinate set(s) were parsed in 0.05s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:    16    4    1    5    7   13   23   66   96  178  219  296  420  446  467\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "5nhf_MODLL.B99990001.pdb       732.49713\n",
      "\n",
      "Modelando proteína 5nhh\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    38285    35639\n",
      "101 (of 2915 total) atoms selected for optimization\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     101\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    35639     618\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      124\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          48.0620\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     103       0      0   0.002   0.002     0.18007       1.000\n",
      " 2 Bond angle potential               :     140       0      0   1.603   1.603      7.0036       1.000\n",
      " 3 Stereochemical cosine torsion poten:      59       0      0  50.895  50.895      21.493       1.000\n",
      " 4 Stereochemical improper torsion pot:      40       0      0   0.874   0.874     0.78913       1.000\n",
      " 5 Soft-sphere overlap restraints     :     124       0      0   0.002   0.002     0.44148E-01   1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      79       0      0   0.086   0.086     0.39767       1.000\n",
      "10 Distance restraints 2 (N-O)        :      80       0      0   0.286   0.286      2.5766       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      16       0      0   2.831   2.831      1.5125       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       6       0      0  71.282  71.282      2.3435       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       5       0      0  49.206  49.206      1.1464       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       4       0      0  61.788  61.788      1.5218       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       1       0      0  87.198  87.198     0.74399       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      39       0      0   0.254   0.254     0.59605E-01   1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      16       9      0  16.086 129.602      5.9340       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      30       0      0   0.899   0.899      2.3159       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 5nhh_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    2344.9033\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9  -72.87  -68.20    5.62    0.37  -62.50  170.98   28.54\n",
      "    1          2A   2A N   CA      9   10  148.44  145.30                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14  -85.10  -68.20   22.83    1.45  -62.50  160.06   27.40\n",
      "    2          3A   3A N   CA     14   15  160.65  145.30                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19 -152.89 -134.00   29.80    1.01  -62.50  174.32   33.38\n",
      "    3          4A   4A N   CA     19   20  170.04  147.00                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24  -66.88  -68.20   10.80    0.92  -62.50  163.14   26.95\n",
      "    4          5A   5A N   CA     24   25  156.03  145.30                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29 -113.05 -134.00   21.67    0.93  -62.50  174.06   31.21\n",
      "    5          6A   6A N   CA     29   30  152.55  147.00                  -40.90\n",
      "    6  10236   7A   8G C   N      37   39   83.05   78.70    4.61    0.26   82.20  173.08    8.36\n",
      "    6          8G   8G N   CA     39   40 -164.58 -166.10                    8.50\n",
      "    7  10237   8G   9A C   N      41   43  -66.89  -68.20    5.22    0.47  -62.50  168.80   27.88\n",
      "    7          9A   9A N   CA     43   44  150.36  145.30                  -40.90\n",
      "    8  10238   9A  10G C   N      46   48  160.12 -167.20   34.21    0.50   82.20 -167.34   12.95\n",
      "    8         10G  10G N   CA     48   49 -175.30  174.60                    8.50\n",
      "    9  10587 358Y 359R C   N    2896 2898  -83.21  -72.10   12.34    0.83  -63.00  172.82   24.73\n",
      "    9        359R 359R N   CA   2898 2899  147.27  141.90                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:   855 1275   2.268\n",
      "  serious non-bonded atom clash:  2663 2677   1.973\n",
      "  serious non-bonded atom clash:  2667 2692   2.204\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     1    0    2    4    3    4    9   48  102  129  189  236  337  440  491\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "5nhh_MODLL.B99990001.pdb        48.06200\n",
      "\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2737 atoms and 1 coordinate set(s) were parsed in 0.03s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 5nhj\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    37889    35243\n",
      "128 (of 2915 total) atoms selected for optimization\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     128\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    35243     833\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      166\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          65.5589\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     132       0      0   0.003   0.003     0.33525       1.000\n",
      " 2 Bond angle potential               :     180       0      0   1.785   1.785      10.367       1.000\n",
      " 3 Stereochemical cosine torsion poten:      84       0      1  47.611  47.611      26.472       1.000\n",
      " 4 Stereochemical improper torsion pot:      54       0      0   0.913   0.913      1.1782       1.000\n",
      " 5 Soft-sphere overlap restraints     :     166       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     103       0      0   0.102   0.102     0.99777       1.000\n",
      "10 Distance restraints 2 (N-O)        :     103       0      0   0.221   0.221      3.5270       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      19       0      0   1.313   1.313    -0.29223       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       9       0      0  68.867  68.867     0.78622       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       8       0      0  61.057  61.057      1.6562       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       6       0      1  93.823  93.823      5.4736       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       1       0      0   0.511   0.511     0.55610       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      52       0      0   0.276   0.276     0.73946E-01   1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      19      13      0  21.931 146.904      12.413       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      63       0      0   0.473   0.473      2.0150       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 5nhj_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    3480.8186\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9  -78.17  -68.20   11.49    0.77  -62.50  168.82   28.46\n",
      "    1          2A   2A N   CA      9   10  151.01  145.30                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14  -68.68  -68.20   14.94    1.22  -62.50  171.38   27.79\n",
      "    2          3A   3A N   CA     14   15  130.37  145.30                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19 -113.93 -134.00   20.17    0.59  -62.50 -178.34   32.50\n",
      "    3          4A   4A N   CA     19   20  144.88  147.00                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24 -111.68 -134.00   22.55    0.87  -62.50  175.84   31.43\n",
      "    4          5A   5A N   CA     24   25  150.28  147.00                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29 -107.62 -134.00   27.03    0.70  -62.50 -176.36   32.48\n",
      "    5          6A   6A N   CA     29   30  141.09  147.00                  -40.90\n",
      "    6  10235   6A   7A C   N      32   34  -67.10  -68.20    2.43    0.16  -62.50  176.02   29.07\n",
      "    6          7A   7A N   CA     34   35  143.14  145.30                  -40.90\n",
      "    7  10236   7A   8G C   N      37   39  142.29 -167.20   50.71    0.87   82.20 -179.08   11.63\n",
      "    7          8G   8G N   CA     39   40  179.15  174.60                    8.50\n",
      "    8  10237   8G   9A C   N      41   43  -73.65  -68.20    7.86    0.49  -62.50  168.50   28.18\n",
      "    8          9A   9A N   CA     43   44  150.97  145.30                  -40.90\n",
      "    9  10238   9A  10G C   N      46   48  160.04 -167.20   33.22    0.78   82.20  178.41   12.22\n",
      "    9         10G  10G N   CA     48   49  169.04  174.60                    8.50\n",
      "   10  10583 354F 355Q C   N    2864 2866  -62.39  -73.00   10.87    0.73  -63.80  178.64   26.34\n",
      "   10        355Q 355Q N   CA   2866 2867  138.33  140.70                  -40.30\n",
      "   11  10585 356P 357G C   N    2880 2882 -138.85 -167.20   32.99    0.55   82.20 -156.11   15.53\n",
      "   11        357G 357G N   CA   2882 2883  157.72  174.60                    8.50\n",
      "   12  10586 357G 358Y C   N    2884 2886  -85.93  -98.40   18.04    1.52  -63.50  160.33   24.32\n",
      "   12        358Y 358Y N   CA   2886 2887  115.36  128.40                  -43.40\n",
      "   13  10587 358Y 359R C   N    2896 2898  -80.05  -72.10    9.80    0.64  -63.00  172.11   24.42\n",
      "   13        359R 359R N   CA   2898 2899  147.63  141.90                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:   398  799   2.219\n",
      "  serious non-bonded atom clash:   431 2739   1.170\n",
      "  serious non-bonded atom clash:   431 2740   1.816\n",
      "  serious non-bonded atom clash:   432 2738   1.420\n",
      "  serious non-bonded atom clash:   432 2739   0.257\n",
      "  serious non-bonded atom clash:   432 2740   1.711\n",
      "  serious non-bonded atom clash:   433 2739   2.086\n",
      "  serious non-bonded atom clash:   433 2740   1.716\n",
      "  serious non-bonded atom clash:   434 2740   1.519\n",
      "  serious non-bonded atom clash:  1509 1562   2.152\n",
      "  serious non-bonded atom clash:  1939 2159   1.684\n",
      "  serious non-bonded atom clash:  1940 2159   2.115\n",
      "  serious non-bonded atom clash:  1941 2158   2.163\n",
      "  serious non-bonded atom clash:  1941 2159   2.021\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:    10    3    1    3    6   12   18   43   83  139  193  278  386  434  483\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2720 atoms and 1 coordinate set(s) were parsed in 0.03s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "5nhj_MODLL.B99990001.pdb        65.55887\n",
      "\n",
      "Modelando proteína 5nhl\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    38003    35357\n",
      "128 (of 2915 total) atoms selected for optimization\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:       90.4065\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     128\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    35357     832\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      180\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          71.2022\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     132       0      0   0.003   0.003     0.29061       1.000\n",
      " 2 Bond angle potential               :     180       0      0   1.983   1.983      13.566       1.000\n",
      " 3 Stereochemical cosine torsion poten:      84       0      0  48.344  48.344      26.435       1.000\n",
      " 4 Stereochemical improper torsion pot:      54       0      0   1.007   1.007      1.4744       1.000\n",
      " 5 Soft-sphere overlap restraints     :     180       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      99       0      0   0.115   0.115      1.1184       1.000\n",
      "10 Distance restraints 2 (N-O)        :     103       0      0   0.273   0.273      4.8464       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      19       0      0   2.704   2.704     0.91895       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       9       0      0  81.103  81.103      1.2279       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       8       0      0  63.147  63.147     0.99202       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       6       0      0  72.607  72.607      4.0850       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       1       0      0  56.658  56.658      1.0009       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      58       0      0   0.534   0.534     0.55394       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      19      11      3  21.812 136.298      8.6292       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      60       0      0   0.834   0.834      6.0640       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 5nhl_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    3197.9976\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9  -80.19  -68.20   15.34    0.98  -62.50  165.19   27.97\n",
      "    1          2A   2A N   CA      9   10  154.87  145.30                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14  -74.11  -68.20   13.20    1.25  -62.50  174.78   28.10\n",
      "    2          3A   3A N   CA     14   15  133.50  145.30                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19  -75.96  -68.20    7.95    0.73  -62.50  176.03   29.53\n",
      "    3          4A   4A N   CA     19   20  143.59  145.30                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24  -67.08  -68.20    4.89    0.35  -62.50  178.62   29.50\n",
      "    4          5A   5A N   CA     24   25  140.54  145.30                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29  -61.63  -68.20    7.34    0.51  -62.50  177.07   28.98\n",
      "    5          6A   6A N   CA     29   30  142.03  145.30                  -40.90\n",
      "    6  10236   7A   8G C   N      37   39  128.14   78.70   52.02    2.14   82.20  175.34   10.73\n",
      "    6          8G   8G N   CA     39   40  177.72 -166.10                    8.50\n",
      "    7  10237   8G   9A C   N      41   43  -89.86  -68.20   34.55    2.18  -62.50  149.41   25.91\n",
      "    7          9A   9A N   CA     43   44  172.22  145.30                  -40.90\n",
      "    8  10583 354F 355Q C   N    2864 2866  -62.37  -73.00   10.84    0.73  -63.80  178.91   26.38\n",
      "    8        355Q 355Q N   CA   2866 2867  138.60  140.70                  -40.30\n",
      "    9  10585 356P 357G C   N    2880 2882  -64.10  -62.40    3.05    0.42   82.20  153.72   11.64\n",
      "    9        357G 357G N   CA   2882 2883  -38.66  -41.20                    8.50\n",
      "   10  10586 357G 358Y C   N    2884 2886 -117.87 -124.30    6.47    0.51  -63.50 -172.41   27.22\n",
      "   10        358Y 358Y N   CA   2886 2887  136.14  135.40                  -43.40\n",
      "   11  10587 358Y 359R C   N    2896 2898  -67.41  -72.10    7.60    0.66  -63.00  171.08   23.41\n",
      "   11        359R 359R N   CA   2898 2899  147.88  141.90                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:   254  499   1.807\n",
      "  serious non-bonded atom clash:   431 2739   1.899\n",
      "  serious non-bonded atom clash:   432 2738   1.877\n",
      "  serious non-bonded atom clash:   432 2739   0.777\n",
      "  serious non-bonded atom clash:   432 2740   1.721\n",
      "  serious non-bonded atom clash:   589 2631   2.127\n",
      "  serious non-bonded atom clash:   728  754   2.078\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2686 atoms and 1 coordinate set(s) were parsed in 0.03s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  serious non-bonded atom clash:  2700 2766   1.620\n",
      "  serious non-bonded atom clash:  2715 2718   2.189\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     7    2    0    3    9   11   34   63  108  157  214  280  388  465  467\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "5nhl_MODLL.B99990001.pdb        71.20222\n",
      "\n",
      "Modelando proteína 5nho\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    37467    34821\n",
      "181 (of 2915 total) atoms selected for optimization\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:      -96.2918\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     181\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    34821    1292\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      244\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         124.8777\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     187       0      0   0.005   0.005      1.3960       1.000\n",
      " 2 Bond angle potential               :     254       0      0   2.183   2.183      20.430       1.000\n",
      " 3 Stereochemical cosine torsion poten:     115       0      2  48.089  48.089      39.101       1.000\n",
      " 4 Stereochemical improper torsion pot:      76       0      0   1.341   1.341      3.6623       1.000\n",
      " 5 Soft-sphere overlap restraints     :     244       0      0   0.003   0.003     0.30454       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     201       0      0   0.153   0.153      3.1544       1.000\n",
      "10 Distance restraints 2 (N-O)        :     191       0      0   0.278   0.278      8.4057       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      26       0      0   2.950   2.950      1.3471       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      15       0      0  67.576  67.576      3.6872       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      14       0      1  61.632  61.632      4.7513       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       9       0      1  81.279  81.279      6.4459       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       2       0      0 129.486 129.486      2.2792       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      90       0      0   0.350   0.350     0.31659       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      27      14      3  24.936 128.040      24.065       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      85       0      0   0.679   0.679      5.5317       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 5nho_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    4342.1543\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9 -170.18 -134.00   46.47    1.24  -62.50  178.97   34.98\n",
      "    1          2A   2A N   CA      9   10  176.15  147.00                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14  -62.35  -68.20    6.34    0.61  -62.50  171.34   28.07\n",
      "    2          3A   3A N   CA     14   15  147.76  145.30                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19  -66.08  -68.20    2.69    0.17  -62.50  175.49   28.94\n",
      "    3          4A   4A N   CA     19   20  143.65  145.30                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24  -62.32  -68.20    6.14    0.57  -62.50  172.04   28.19\n",
      "    4          5A   5A N   CA     24   25  147.06  145.30                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29  -61.37  -68.20   12.86    1.24  -62.50  162.91   26.64\n",
      "    5          6A   6A N   CA     29   30  156.20  145.30                  -40.90\n",
      "    6  10235   6A   7A C   N      32   34 -110.39 -134.00   23.76    0.89  -62.50  176.08   31.40\n",
      "    6          7A   7A N   CA     34   35  149.66  147.00                  -40.90\n",
      "    7  10236   7A   8G C   N      37   39  102.29   78.70   27.82    0.88   82.20  171.83    9.34\n",
      "    7          8G   8G N   CA     39   40  179.15 -166.10                    8.50\n",
      "    8  10237   8G   9A C   N      41   43  -78.87  -68.20   15.93    1.00  -62.50  162.79   27.51\n",
      "    8          9A   9A N   CA     43   44  157.14  145.30                  -40.90\n",
      "    9  10239  10G  11P C   N      50   52  -75.79  -58.70   42.78    2.52  -64.50  138.94   10.97\n",
      "    9         11P  11P N   CA     52   53    8.72  -30.50                  147.20\n",
      "   10  10560 331F 332D C   N    2668 2670  -83.85  -70.90   27.00    0.90  -63.30  147.46   19.50\n",
      "   10        332D 332D N   CA   2670 2671  173.99  150.30                  -40.00\n",
      "   11  10561 332D 333M C   N    2676 2678 -107.41 -125.60   29.31    1.60  -63.40  162.10   27.72\n",
      "   11        333M 333M N   CA   2678 2679  163.49  140.50                  -40.50\n",
      "   12  10585 356P 357G C   N    2880 2882  -95.94  -80.20   20.91    0.73   82.20 -107.20    7.88\n",
      "   12        357G 357G N   CA   2882 2883 -172.14  174.10                    8.50\n",
      "   13  10586 357G 358Y C   N    2884 2886 -110.25 -124.30   20.01    1.61  -63.50  173.36   29.89\n",
      "   13        358Y 358Y N   CA   2886 2887  149.66  135.40                  -43.40\n",
      "   14  10587 358Y 359R C   N    2896 2898 -117.32 -125.20    7.88    0.32  -63.00 -173.67   28.93\n",
      "   14        359R 359R N   CA   2898 2899  140.67  140.60                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:    89  232   2.254\n",
      "  serious non-bonded atom clash:    90  212   1.689\n",
      "  serious non-bonded atom clash:   113  398   2.298\n",
      "  serious non-bonded atom clash:   397  780   1.765\n",
      "  serious non-bonded atom clash:   431 2739   1.277\n",
      "  serious non-bonded atom clash:   431 2740   1.969\n",
      "  serious non-bonded atom clash:   432 2737   2.097\n",
      "  serious non-bonded atom clash:   432 2738   1.249\n",
      "  serious non-bonded atom clash:   432 2739   0.816\n",
      "  serious non-bonded atom clash:   432 2740   2.006\n",
      "  serious non-bonded atom clash:   433 2739   2.108\n",
      "  serious non-bonded atom clash:   433 2740   1.746\n",
      "  serious non-bonded atom clash:   434 2740   1.582\n",
      "  serious non-bonded atom clash:   549 2829   2.167\n",
      "  serious non-bonded atom clash:   682  686   2.243\n",
      "  serious non-bonded atom clash:  1939 2159   1.813\n",
      "  serious non-bonded atom clash:  1941 2158   2.190\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:    11    3    3    4    4   12   26   58   91  129  189  246  361  467  476\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "5nho_MODLL.B99990001.pdb       124.87766\n",
      "\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2692 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 5nhp\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    37502    34856\n",
      "223 (of 2915 total) atoms selected for optimization\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     223\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    34856    1558\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      375\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         142.3461\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     231       0      0   0.004   0.004      1.0674       1.000\n",
      " 2 Bond angle potential               :     315       0      0   1.986   1.986      22.655       1.000\n",
      " 3 Stereochemical cosine torsion poten:     150       0      5  49.841  49.841      49.581       1.000\n",
      " 4 Stereochemical improper torsion pot:      95       0      0   1.187   1.187      3.0242       1.000\n",
      " 5 Soft-sphere overlap restraints     :     375       0      0   0.003   0.003     0.39188       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     192       0      0   0.171   0.171      3.9396       1.000\n",
      "10 Distance restraints 2 (N-O)        :     210       0      0   0.212   0.212      5.9888       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      31       0      0   3.485   3.485      3.1022       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      20       0      0  81.316  81.316      7.0894       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      19       0      1  64.396  64.396      9.3424       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:      10       0      0  87.162  87.162      3.7430       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       2       0      0 110.359 110.359      1.8478       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     114       0      0   0.342   0.342     0.42303       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      32      17      3  28.927 126.078      23.342       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     137       0      0   0.612   0.612      6.8088       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 5nhp_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    5275.7041\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9  -75.28  -68.20   23.48    1.65  -62.50  151.95   25.55\n",
      "    1          2A   2A N   CA      9   10  167.69  145.30                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14  -54.55  -68.20   14.42    1.06  -62.50  178.61   28.89\n",
      "    2          3A   3A N   CA     14   15  140.67  145.30                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19  -74.94  -68.20    7.53    0.73  -62.50  177.60   29.73\n",
      "    3          4A   4A N   CA     19   20  141.94  145.30                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24  -85.00  -68.20   18.63    1.29  -62.50  167.26   28.57\n",
      "    4          5A   5A N   CA     24   25  153.36  145.30                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29  -63.30  -68.20    5.16    0.38  -62.50  175.42   28.79\n",
      "    5          6A   6A N   CA     29   30  143.68  145.30                  -40.90\n",
      "    6  10235   6A   7A C   N      32   34  -53.01  -68.20   15.21    1.32  -62.50  173.22   27.94\n",
      "    6          7A   7A N   CA     34   35  146.15  145.30                  -40.90\n",
      "    7  10236   7A   8G C   N      37   39  -74.35  -80.20   16.03    0.39   82.20 -142.72    6.94\n",
      "    7          8G   8G N   CA     39   40  159.17  174.10                    8.50\n",
      "    8  10237   8G   9A C   N      41   43 -118.99 -134.00   17.53    0.92  -62.50  172.57   31.30\n",
      "    8          9A   9A N   CA     43   44  156.04  147.00                  -40.90\n",
      "    9  10559 330K 331F C   N    2657 2659 -171.25 -124.20   58.02    1.50  -63.20  175.62   30.86\n",
      "    9        331F 331F N   CA   2659 2660  177.25  143.30                  -44.30\n",
      "   10  10560 331F 332D C   N    2668 2670 -131.69  -96.50   68.23    2.82  -63.30  117.66   12.65\n",
      "   10        332D 332D N   CA   2670 2671   55.75  114.20                  -40.00\n",
      "   11  10563 334E 335L C   N    2693 2695  -74.48  -70.70   24.84    2.03  -63.50  158.63   21.42\n",
      "   11        335L 335L N   CA   2695 2696  117.05  141.60                  -41.20\n",
      "   12  10564 335L 336D C   N    2701 2703 -129.92  -70.90   71.56    3.50  -63.30  145.40   22.24\n",
      "   12        336D 336D N   CA   2703 2704 -169.23  150.30                  -40.00\n",
      "   13  10565 336D 337D C   N    2709 2711  -68.81  -96.50   49.61    2.08  -63.30  113.18   13.64\n",
      "   13        337D 337D N   CA   2711 2712   73.04  114.20                  -40.00\n",
      "   14  10583 354F 355Q C   N    2864 2866  -63.11  -73.00   10.90    0.68  -63.80  176.43   25.98\n",
      "   14        355Q 355Q N   CA   2866 2867  136.13  140.70                  -40.30\n",
      "   15  10585 356P 357G C   N    2880 2882  -93.83  -80.20   46.54    1.22   82.20 -128.79   17.94\n",
      "   15        357G 357G N   CA   2882 2883 -141.40  174.10                    8.50\n",
      "   16  10586 357G 358Y C   N    2884 2886 -103.09  -98.40    8.13    0.71  -63.50  169.84   25.04\n",
      "   16        358Y 358Y N   CA   2886 2887  121.76  128.40                  -43.40\n",
      "   17  10587 358Y 359R C   N    2896 2898 -143.98 -125.20   29.57    0.94  -63.00  175.29   29.17\n",
      "   17        359R 359R N   CA   2898 2899  163.44  140.60                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  serious non-bonded atom clash:    73   95   2.041\n",
      "  serious non-bonded atom clash:   431 2739   1.241\n",
      "  serious non-bonded atom clash:   431 2740   1.753\n",
      "  serious non-bonded atom clash:   432 2738   1.771\n",
      "  serious non-bonded atom clash:   432 2739   0.584\n",
      "  serious non-bonded atom clash:   432 2740   1.706\n",
      "  serious non-bonded atom clash:   433 2739   2.036\n",
      "  serious non-bonded atom clash:   433 2740   1.482\n",
      "  serious non-bonded atom clash:   434 2740   1.102\n",
      "  serious non-bonded atom clash:   440 2740   1.898\n",
      "  serious non-bonded atom clash:   590 2631   2.236\n",
      "  serious non-bonded atom clash:   855 1275   2.040\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:    11    0    1    0    1   10   15   69  101  170  219  275  379  436  494\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "5nhp_MODLL.B99990001.pdb       142.34605\n",
      "\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2771 atoms and 1 coordinate set(s) were parsed in 0.18s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 5nhv\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    37949    35303\n",
      "128 (of 2915 total) atoms selected for optimization\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:       91.3612\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     128\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    35303     868\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      169\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          79.1304\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     132       0      0   0.004   0.004     0.56550       1.000\n",
      " 2 Bond angle potential               :     180       0      0   1.939   1.939      12.712       1.000\n",
      " 3 Stereochemical cosine torsion poten:      84       0      1  48.492  48.492      28.577       1.000\n",
      " 4 Stereochemical improper torsion pot:      54       0      0   0.990   0.990      1.4746       1.000\n",
      " 5 Soft-sphere overlap restraints     :     169       0      0   0.000   0.000      0.0000       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     102       0      0   0.114   0.114      1.1426       1.000\n",
      "10 Distance restraints 2 (N-O)        :      98       0      0   0.353   0.353      7.4558       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      19       0      0   2.434   2.434     0.65728       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       9       0      0  54.971  54.971      1.2586       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       8       0      1  80.324  80.324      2.8789       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       6       0      0  81.121  81.121      3.4912       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       1       0      0 172.715 172.715     0.52637       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      75       0      0   0.383   0.383     0.20828       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      19      11      3  26.455 136.124      14.138       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      81       0      0   0.616   0.616      4.0441       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 5nhv_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    3190.0913\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9 -163.77 -134.00   36.04    0.89  -62.50 -177.52   35.27\n",
      "    1          2A   2A N   CA      9   10  167.30  147.00                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14 -100.42  -68.20   32.93    3.00  -62.50 -176.63   28.44\n",
      "    2          3A   3A N   CA     14   15  138.51  145.30                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19  -64.57  -68.20    7.54    0.72  -62.50  167.21   27.50\n",
      "    3          4A   4A N   CA     19   20  151.91  145.30                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24  -66.65  -68.20   28.24    2.33  -62.50  145.66   24.08\n",
      "    4          5A   5A N   CA     24   25  173.50  145.30                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29  -86.39  -68.20   21.67    2.12  -62.50  176.05   27.77\n",
      "    5          6A   6A N   CA     29   30  133.52  145.30                  -40.90\n",
      "    6  10237   8G   9A C   N      41   43  -67.56  -68.20    7.28    0.61  -62.50  166.63   27.56\n",
      "    6          9A   9A N   CA     43   44  152.55  145.30                  -40.90\n",
      "    7  10238   9A  10G C   N      46   48 -177.38 -167.20   19.18    0.77   82.20 -179.62   13.12\n",
      "    7         10G  10G N   CA     48   49  158.35  174.60                    8.50\n",
      "    8  10583 354F 355Q C   N    2864 2866  -80.44  -73.00   13.91    1.05  -63.80  170.07   24.16\n",
      "    8        355Q 355Q N   CA   2866 2867  128.95  140.70                  -40.30\n",
      "    9  10585 356P 357G C   N    2880 2882 -179.09 -167.20   40.97    1.68   82.20  160.76   11.97\n",
      "    9        357G 357G N   CA   2882 2883  135.38  174.60                    8.50\n",
      "   10  10586 357G 358Y C   N    2884 2886 -129.94 -124.30    6.18    0.51  -63.50 -171.62   26.90\n",
      "   10        358Y 358Y N   CA   2886 2887  132.88  135.40                  -43.40\n",
      "   11  10587 358Y 359R C   N    2896 2898 -122.68 -125.20   11.94    0.62  -63.00  177.00   28.03\n",
      "   11        359R 359R N   CA   2898 2899  152.26  140.60                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:   423 2739   1.996\n",
      "  serious non-bonded atom clash:   431 2738   2.160\n",
      "  serious non-bonded atom clash:   431 2739   0.837\n",
      "  serious non-bonded atom clash:   431 2740   1.450\n",
      "  serious non-bonded atom clash:   432 2738   1.157\n",
      "  serious non-bonded atom clash:   432 2739   0.442\n",
      "  serious non-bonded atom clash:   432 2740   1.564\n",
      "  serious non-bonded atom clash:   433 2739   2.004\n",
      "  serious non-bonded atom clash:   433 2740   1.434\n",
      "  serious non-bonded atom clash:   434 2740   1.672\n",
      "  serious non-bonded atom clash:   492 2702   2.163\n",
      "  serious non-bonded atom clash:   855 1275   1.900\n",
      "  serious non-bonded atom clash:  2664 2677   2.138\n",
      "  serious non-bonded atom clash:  2666 2677   2.101\n",
      "  serious non-bonded atom clash:  2666 2692   2.040\n",
      "  serious non-bonded atom clash:  2667 2692   2.214\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:    11    4    1    2    1    3   10   53   76  116  171  234  346  423  488\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 5541 atoms and 1 coordinate set(s) were parsed in 0.06s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "5nhv_MODLL.B99990001.pdb        79.13036\n",
      "\n",
      "Modelando proteína 5u6i\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    37831    35185\n",
      "198 (of 2915 total) atoms selected for optimization\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     198\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    35185    1423\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      284\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         106.4995\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     204       0      0   0.003   0.003     0.63490       1.000\n",
      " 2 Bond angle potential               :     277       0      0   1.780   1.780      15.925       1.000\n",
      " 3 Stereochemical cosine torsion poten:     122       0      2  48.730  48.730      43.541       1.000\n",
      " 4 Stereochemical improper torsion pot:      83       0      0   0.854   0.854      1.4639       1.000\n",
      " 5 Soft-sphere overlap restraints     :     284       0      0   0.001   0.001     0.27119E-01   1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     251       0      0   0.102   0.102      2.0836       1.000\n",
      "10 Distance restraints 2 (N-O)        :     208       0      0   0.163   0.163      2.3102       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      30       0      0   2.935   2.935      2.4563       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      15       0      1  81.816  81.816      3.9995       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      12       0      0  76.328  76.328      2.1608       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       9       0      0  76.138  76.138      5.4176       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       2       0      0 123.486 123.486      1.8013       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      91       0      0   0.228   0.228     0.15878       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      31      17      2  25.114 139.058      21.832       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      88       0      0   0.677   0.677      2.6873       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 5u6i_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    4927.0811\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9 -125.95 -134.00   12.16    0.39  -62.50 -170.29   28.73\n",
      "    1          2A   2A N   CA      9   10  137.89  147.00                  -40.90\n",
      "    2  10232   3A   4A C   N      17   19  -72.18  -68.20   20.54    1.51  -62.50  153.95   25.72\n",
      "    2          4A   4A N   CA     19   20  165.45  145.30                  -40.90\n",
      "    3  10233   4A   5A C   N      22   24  -56.47  -68.20   19.08    1.86  -62.50  158.86   25.75\n",
      "    3          5A   5A N   CA     24   25  160.36  145.30                  -40.90\n",
      "    4  10234   5A   6A C   N      27   29  -53.65  -68.20   15.60    1.12  -62.50  179.65   29.02\n",
      "    4          6A   6A N   CA     29   30  139.67  145.30                  -40.90\n",
      "    5  10235   6A   7A C   N      32   34  -77.15  -68.20   28.56    1.99  -62.50  147.41   24.90\n",
      "    5          7A   7A N   CA     34   35  172.42  145.30                  -40.90\n",
      "    6  10236   7A   8G C   N      37   39  -90.92  -80.20   56.53    1.71   82.20 -138.04   17.27\n",
      "    6          8G   8G N   CA     39   40 -130.40  174.10                    8.50\n",
      "    7  10237   8G   9A C   N      41   43  -69.51  -68.20    6.67    0.49  -62.50  167.41   27.78\n",
      "    7          9A   9A N   CA     43   44  151.84  145.30                  -40.90\n",
      "    8  10239  10G  11P C   N      50   52  -48.37  -58.70   15.28    1.03  -64.50  171.79   13.69\n",
      "    8         11P  11P N   CA     52   53  -41.77  -30.50                  147.20\n",
      "    9  10240  11P  12E C   N      57   59  -84.90  -69.30   15.83    1.29  -63.60 -178.89   25.34\n",
      "    9         12E  12E N   CA     59   60  139.84  142.50                  -40.30\n",
      "   10  10241  12E  13M C   N      66   68 -158.98 -125.60   33.46    1.27  -63.40 -159.07   37.44\n",
      "   10         13M  13M N   CA     68   69  142.76  140.50                  -40.50\n",
      "   11  10242  13M  14V C   N      74   76  -65.96  -62.40    6.63    1.05 -125.40  178.86   10.01\n",
      "   11         14V  14V N   CA     76   77  -48.00  -42.40                  143.30\n",
      "   12  10243  14V  15R C   N      81   83  -89.08  -72.10   23.70    1.54  -63.00  162.57   23.76\n",
      "   12         15R  15R N   CA     83   84  158.44  141.90                  -41.10\n",
      "   13  10244  15R  16G C   N      92   94 -142.21 -167.20   34.48    0.75   82.20 -163.42   15.00\n",
      "   13         16G  16G N   CA     94   95  150.84  174.60                    8.50\n",
      "   14  10245  16G  17Q C   N      96   98 -123.71 -121.10    8.28    0.34  -63.80 -177.73   29.98\n",
      "   14         17Q  17Q N   CA     98   99  147.56  139.70                  -40.30\n",
      "   15  10264  35A  36Y C   N     243  245 -144.70 -124.30   20.88    1.41  -63.50 -165.47   34.87\n",
      "   15         36Y  36Y N   CA    245  246  139.83  135.40                  -43.40\n",
      "   16  10586 357G 358Y C   N    2884 2886 -101.28  -98.40    2.89    0.12  -63.50  176.09   26.11\n",
      "   16        358Y 358Y N   CA   2886 2887  128.58  128.40                  -43.40\n",
      "   17  10587 358Y 359R C   N    2896 2898  -78.73  -72.10    6.63    0.50  -63.00  177.61   25.07\n",
      "   17        359R 359R N   CA   2898 2899  141.98  141.90                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    0    1    9   42  118  170  171  257  348  438  472\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "5u6i_MODLL.B99990001.pdb       106.49953\n",
      "\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2825 atoms and 1 coordinate set(s) were parsed in 0.03s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 5umo\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    37165    34519\n",
      "130 (of 2915 total) atoms selected for optimization\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     130\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    34519     851\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      242\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         285.6080\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     134       0      0   0.013   0.013      6.4208       1.000\n",
      " 2 Bond angle potential               :     184       1      4   3.418   3.418      41.503       1.000\n",
      " 3 Stereochemical cosine torsion poten:      79       0      6  58.680  58.680      40.822       1.000\n",
      " 4 Stereochemical improper torsion pot:      51       0      0   1.410   1.410      2.4379       1.000\n",
      " 5 Soft-sphere overlap restraints     :     242       0      1   0.016   0.016      6.9092       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     132       0      7   0.698   0.698      41.657       1.000\n",
      "10 Distance restraints 2 (N-O)        :     128       1      7   1.071   1.071      46.307       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      20       0      1   6.258   6.258      9.2359       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      10       0      0  92.077  92.077      4.0514       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       8       0      0  35.781  35.781      1.3506       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       7       0      0  85.531  85.531      4.8364       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       2       0      0  52.473  52.473      2.1345       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      49       0      0   0.809   0.809      1.1999       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      21      12      8  63.741 152.102      72.335       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      26       0      0   1.079   1.079      4.4078       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 5umo_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    4155.8789\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   3087  13M  14V C   N      74   76  141.80  120.00   21.80    4.95  120.00   21.80    4.95\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 10                           : Distance restraints 2 (N-O)             \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  18535  14V  22G N   O      76  143   13.94   10.70    3.25    4.64   10.70    3.25    4.64\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10233   4A   5A C   N      22   24 -168.55 -134.00   36.55    0.86  -62.50 -167.87   37.11\n",
      "    1          5A   5A N   CA     24   25  158.90  147.00                  -40.90\n",
      "    2  10234   5A   6A C   N      27   29 -139.87 -134.00   31.60    1.64  -62.50  160.87   30.50\n",
      "    2          6A   6A N   CA     29   30  178.05  147.00                  -40.90\n",
      "    3  10235   6A   7A C   N      32   34  -67.18  -68.20   20.62    1.70  -62.50  153.28   25.35\n",
      "    3          7A   7A N   CA     34   35  165.90  145.30                  -40.90\n",
      "    4  10237   8G   9A C   N      41   43 -173.13 -134.00   58.64    1.88  -62.50  169.52   33.51\n",
      "    4          9A   9A N   CA     43   44 -169.33  147.00                  -40.90\n",
      "    5  10238   9A  10G C   N      46   48  -80.38  -80.20   23.25    0.91   82.20 -123.93   18.01\n",
      "    5         10G  10G N   CA     48   49 -162.65  174.10                    8.50\n",
      "    6  10240  11P  12E C   N      57   59  143.96 -117.80   99.56    3.28  -63.60 -134.11   38.05\n",
      "    6         12E  12E N   CA     59   60  152.99  136.80                  -40.30\n",
      "    7  10241  12E  13M C   N      66   68 -156.75 -125.60   31.16    1.23  -63.40 -158.12   37.43\n",
      "    7         13M  13M N   CA     68   69  140.50  140.50                  -40.50\n",
      "    8  10242  13M  14V C   N      74   76  126.09 -125.40 -151.58    7.02 -125.40 -151.58    7.02\n",
      "    8         14V  14V N   CA     76   77  -38.75  143.30                  143.30\n",
      "    9  10243  14V  15R C   N      81   83 -112.13 -125.20   65.86    2.87  -72.10   77.06    6.62\n",
      "    9         15R  15R N   CA     83   84   76.05  140.60                  141.90\n",
      "   10  10585 356P 357G C   N    2880 2882 -164.35 -167.20    8.13    0.33   82.20 -152.52   15.02\n",
      "   10        357G 357G N   CA   2882 2883 -177.78  174.60                    8.50\n",
      "   11  10586 357G 358Y C   N    2884 2886 -124.10 -124.30    2.47    0.14  -63.50 -171.27   33.01\n",
      "   11        358Y 358Y N   CA   2886 2887  137.86  135.40                  -43.40\n",
      "   12  10587 358Y 359R C   N    2896 2898 -148.61 -125.20   30.07    0.89  -63.00 -179.04   30.24\n",
      "   12        359R 359R N   CA   2898 2899  159.47  140.60                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    0    8   14   32   72  130  204  211  330  428  431\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "5umo_MODLL.B99990001.pdb       285.60797\n",
      "\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2858 atoms and 1 coordinate set(s) were parsed in 0.04s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 5v60\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    37944    35298\n",
      "146 (of 2915 total) atoms selected for optimization\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     146\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    35298    1180\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      234\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         134.1731\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     151       0      0   0.007   0.007      2.2073       1.000\n",
      " 2 Bond angle potential               :     208       0      0   1.818   1.818      13.373       1.000\n",
      " 3 Stereochemical cosine torsion poten:     100       0      3  46.869  46.869      32.346       1.000\n",
      " 4 Stereochemical improper torsion pot:      68       0      0   1.080   1.080      2.2687       1.000\n",
      " 5 Soft-sphere overlap restraints     :     234       0      0   0.001   0.001     0.57208E-01   1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     203       0      3   0.463   0.463      23.733       1.000\n",
      "10 Distance restraints 2 (N-O)        :     191       0      1   0.524   0.524      15.610       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      22       0      0   3.054   3.054      2.0645       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      11       0      0  76.473  76.473      2.7805       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       8       0      0  60.986  60.986      1.2230       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       4       0      0  99.901  99.901      2.6963       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       1       0      0   0.103   0.103     0.55464       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     102       0      0   0.496   0.496     0.70395       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      23      11      5  39.819 121.719      24.669       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      88       0      0   0.980   0.980      9.8856       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 5v60_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    3700.9114\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9  -74.87  -68.20    9.00    0.57  -62.50  168.22   28.19\n",
      "    1          2A   2A N   CA      9   10  151.34  145.30                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14 -117.85 -134.00   16.45    0.44  -62.50 -176.24   33.06\n",
      "    2          3A   3A N   CA     14   15  143.88  147.00                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19  -71.68  -68.20    5.16    0.32  -62.50  170.25   28.36\n",
      "    3          4A   4A N   CA     19   20  149.10  145.30                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24 -147.73 -134.00   24.52    0.92  -62.50  174.08   33.08\n",
      "    4          5A   5A N   CA     24   25  167.31  147.00                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29  -66.79  -68.20    3.79    0.35  -62.50  170.33   28.13\n",
      "    5          6A   6A N   CA     29   30  148.82  145.30                  -40.90\n",
      "    6  10235   6A   7A C   N      32   34  -69.78  -68.20    5.85    0.42  -62.50  168.33   27.95\n",
      "    6          7A   7A N   CA     34   35  150.93  145.30                  -40.90\n",
      "    7  10237   8G   9A C   N      41   43 -104.51 -134.00   31.29    0.73  -62.50 -177.68   28.13\n",
      "    7          9A   9A N   CA     43   44  136.52  147.00                  -40.90\n",
      "    8  10238   9A  10G C   N      46   48   91.67   78.70   61.85    1.25  -80.20  176.62   10.30\n",
      "    8         10G  10G N   CA     48   49  133.42 -166.10                  174.10\n",
      "    9  10414 185T 186E C   N    1475 1477 -103.35 -117.80   58.56    3.15  -63.60  132.26   19.88\n",
      "    9        186E 186E N   CA   1477 1478 -166.45  136.80                  -40.30\n",
      "   10  10417 188V 189A C   N    1503 1505   70.70   55.40  131.12    9.17  -62.50 -158.89   30.72\n",
      "   10        189A 189A N   CA   1505 1506  168.43   38.20                  -40.90\n",
      "   11  10587 358Y 359R C   N    2896 2898  -78.35  -72.10   13.25    0.89  -63.00  166.03   23.48\n",
      "   11        359R 359R N   CA   2898 2899  153.58  141.90                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    1    1    2    5   37   94  155  190  242  346  424  465\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "5v60_MODLL.B99990001.pdb       134.17313\n",
      "\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2869 atoms and 1 coordinate set(s) were parsed in 0.03s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 5v61\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    38270    35624\n",
      "142 (of 2915 total) atoms selected for optimization\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     142\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    35624    1226\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      213\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         146.4836\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     147       0      0   0.007   0.007      2.3354       1.000\n",
      " 2 Bond angle potential               :     203       0      0   2.113   2.113      17.776       1.000\n",
      " 3 Stereochemical cosine torsion poten:      99       0      6  52.342  52.342      39.830       1.000\n",
      " 4 Stereochemical improper torsion pot:      67       0      0   1.185   1.185      2.7777       1.000\n",
      " 5 Soft-sphere overlap restraints     :     213       0      0   0.002   0.002     0.13965       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     216       0      3   0.436   0.436      22.878       1.000\n",
      "10 Distance restraints 2 (N-O)        :     216       0      1   0.415   0.415      14.878       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      21       0      0   3.463   3.463      1.9510       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      11       0      0  80.827  80.827      3.5077       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       8       0      0  55.620  55.620      2.1237       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       4       0      0  61.996  61.996      2.5711       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       1       0      0   7.467   7.467     0.62919       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     105       0      0   0.526   0.526     0.78950       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      22      10      3  37.887 129.746      24.819       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     106       0      0   0.860   0.860      9.4781       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 5v61_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    3655.8340\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9  -73.16  -68.20   10.44    0.68  -62.50  164.96   27.57\n",
      "    1          2A   2A N   CA      9   10  154.49  145.30                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14 -120.67 -134.00   14.60    0.33  -62.50 -172.66   33.80\n",
      "    2          3A   3A N   CA     14   15  141.02  147.00                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19  -77.86  -68.20   12.67    0.81  -62.50  166.31   28.03\n",
      "    3          4A   4A N   CA     19   20  153.50  145.30                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24 -101.39  -68.20   37.02    3.58  -62.50  174.21   26.92\n",
      "    4          5A   5A N   CA     24   25  128.91  145.30                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29 -159.80 -134.00   33.13    0.88  -62.50  179.91   34.65\n",
      "    5          6A   6A N   CA     29   30  167.78  147.00                  -40.90\n",
      "    6  10235   6A   7A C   N      32   34 -110.25 -134.00   23.84    0.87  -62.50  176.69   31.49\n",
      "    6          7A   7A N   CA     34   35  148.99  147.00                  -40.90\n",
      "    7  10236   7A   8G C   N      37   39 -158.65 -167.20   32.12    1.12   82.20 -179.84   13.63\n",
      "    7          8G   8G N   CA     39   40  143.64  174.60                    8.50\n",
      "    8  10237   8G   9A C   N      41   43 -106.28 -134.00   30.35    0.69  -62.50 -179.10   27.84\n",
      "    8          9A   9A N   CA     43   44  134.62  147.00                  -40.90\n",
      "    9  10414 185T 186E C   N    1475 1477  -76.87  -69.30   17.19    1.05  -63.60  162.31   22.38\n",
      "    9        186E 186E N   CA   1477 1478  157.93  142.50                  -40.30\n",
      "   10  10417 188V 189A C   N    1503 1505   71.74   55.40  128.04    9.07  -62.50 -155.77   31.17\n",
      "   10        189A 189A N   CA   1505 1506  165.19   38.20                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    1   11   12   42  100  172  191  263  330  432  474\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2876 atoms and 1 coordinate set(s) were parsed in 0.03s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "5v61_MODLL.B99990001.pdb       146.48361\n",
      "\n",
      "Modelando proteína 5v62\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    38371    35725\n",
      "112 (of 2915 total) atoms selected for optimization\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     112\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    35725     968\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      174\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         129.4772\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     116       0      0   0.008   0.008      2.1879       1.000\n",
      " 2 Bond angle potential               :     161       0      0   2.304   2.304      16.718       1.000\n",
      " 3 Stereochemical cosine torsion poten:      73       0      3  47.054  47.054      24.946       1.000\n",
      " 4 Stereochemical improper torsion pot:      51       0      0   1.423   1.423      3.0195       1.000\n",
      " 5 Soft-sphere overlap restraints     :     174       0      0   0.002   0.002     0.69369E-01   1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     174       0      3   0.503   0.503      23.208       1.000\n",
      "10 Distance restraints 2 (N-O)        :     166       0      1   0.447   0.447      11.582       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      18       0      1   4.350   4.350      3.3816       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       8       0      0  64.434  64.434      1.7234       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       6       0      0  53.598  53.598     0.99085       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       3       0      0 108.306 108.306      2.8015       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      87       0      0   0.527   0.527     0.67830       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      19      11      3  43.232 129.556      27.235       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      86       0      0   1.037   1.037      10.936       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 5v62_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    3247.0591\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9  -74.90  -68.20    9.18    0.58  -62.50  167.99   28.15\n",
      "    1          2A   2A N   CA      9   10  151.57  145.30                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14  -70.21  -68.20    5.30    0.49  -62.50  178.87   29.70\n",
      "    2          3A   3A N   CA     14   15  140.40  145.30                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19  -88.70  -68.20   25.12    1.64  -62.50  161.43   27.81\n",
      "    3          4A   4A N   CA     19   20  159.81  145.30                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24  -80.34  -68.20   18.53    1.17  -62.50  160.80   27.26\n",
      "    4          5A   5A N   CA     24   25  159.30  145.30                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29  -75.00  -68.20   17.31    1.17  -62.50  158.37   26.59\n",
      "    5          6A   6A N   CA     29   30  161.23  145.30                  -40.90\n",
      "    6  10235   6A   7A C   N      32   34  -69.30  -68.20   11.23    0.87  -62.50  162.76   27.01\n",
      "    6          7A   7A N   CA     34   35  156.48  145.30                  -40.90\n",
      "    7  10236   7A   8G C   N      37   39 -101.76  -80.20   42.66    1.01   82.20 -123.73    7.88\n",
      "    7          8G   8G N   CA     39   40 -149.09  174.10                    8.50\n",
      "    8  10237   8G   9A C   N      41   43  -71.57  -68.20    3.90    0.26  -62.50  172.08   28.65\n",
      "    8          9A   9A N   CA     43   44  147.26  145.30                  -40.90\n",
      "    9  10238   9A  10G C   N      46   48 -173.36 -167.20   23.29    0.96  -80.20   95.71    6.89\n",
      "    9         10G  10G N   CA     48   49  152.14  174.60                  174.10\n",
      "   10  10414 185T 186E C   N    1475 1477 -114.63 -117.80   83.92    4.27  -63.60  111.41   17.70\n",
      "   10        186E 186E N   CA   1477 1478 -139.34  136.80                  -40.30\n",
      "   11  10417 188V 189A C   N    1503 1505   68.03   55.40  129.06    8.82  -62.50 -159.30   30.58\n",
      "   11        189A 189A N   CA   1505 1506  166.64   38.20                  -40.90\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:  2105 2125   2.253\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    1    0    3    4    8   41  125  175  208  232  376  421  468\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "5v62_MODLL.B99990001.pdb       129.47723\n",
      "\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 5483 atoms and 1 coordinate set(s) were parsed in 0.05s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 5wp1\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    37526    34880\n",
      "230 (of 2915 total) atoms selected for optimization\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     230\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    34880    1977\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      528\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         231.8936\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     238       0      0   0.006   0.006      2.7747       1.000\n",
      " 2 Bond angle potential               :     326       0      0   1.914   1.914      23.294       1.000\n",
      " 3 Stereochemical cosine torsion poten:     153       0      8  50.756  50.756      60.828       1.000\n",
      " 4 Stereochemical improper torsion pot:     100       0      0   1.319   1.319      4.4274       1.000\n",
      " 5 Soft-sphere overlap restraints     :     528       0      0   0.004   0.004     0.98192       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     379       0      0   0.323   0.323      23.013       1.000\n",
      "10 Distance restraints 2 (N-O)        :     350       0      3   0.480   0.480      30.078       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      34       0      0   3.906   3.906      5.0409       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      21       0      0  97.379  97.379      10.883       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      15       0      1  59.996  59.996      8.4723       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       8       0      0 105.147 105.147      5.5385       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       3       0      0  18.570  18.570      2.1635       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     160       0      0   0.341   0.341     0.37795       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      36      14      7  37.516 113.300      46.643       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     154       0      0   0.707   0.707      7.3764       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 5wp1_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    5273.9878\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9 -106.94 -134.00   36.85    1.06  -62.50  168.83   25.88\n",
      "    1          2A   2A N   CA      9   10  121.97  147.00                  -40.90\n",
      "    2  10232   3A   4A C   N      17   19 -131.49 -134.00   14.79    0.89  -62.50  171.98   31.88\n",
      "    2          4A   4A N   CA     19   20  161.57  147.00                  -40.90\n",
      "    3  10233   4A   5A C   N      22   24  -63.31  -68.20   12.51    0.84  -62.50  174.69   28.59\n",
      "    3          5A   5A N   CA     24   25  133.79  145.30                  -40.90\n",
      "    4  10234   5A   6A C   N      27   29  -80.97  -68.20   20.93    1.32  -62.50  158.30   26.89\n",
      "    4          6A   6A N   CA     29   30  161.89  145.30                  -40.90\n",
      "    5  10235   6A   7A C   N      32   34  -70.34  -68.20    2.32    0.22  -62.50  174.88   29.05\n",
      "    5          7A   7A N   CA     34   35  144.40  145.30                  -40.90\n",
      "    6  10236   7A   8G C   N      37   39   73.12   78.70    5.71    0.25   82.20  173.62    8.89\n",
      "    6          8G   8G N   CA     39   40 -164.88 -166.10                    8.50\n",
      "    7  10237   8G   9A C   N      41   43 -156.97 -134.00   29.98    0.82  -62.50  179.69   34.47\n",
      "    7          9A   9A N   CA     43   44  166.25  147.00                  -40.90\n",
      "    8  10238   9A  10G C   N      46   48 -128.80 -167.20   49.77    0.99   82.20 -159.31   15.52\n",
      "    8         10G  10G N   CA     48   49  142.95  174.60                    8.50\n",
      "    9  10240  11P  12E C   N      57   59  -89.02  -69.30   21.52    1.33  -63.60  170.49   24.16\n",
      "    9         12E  12E N   CA     59   60  151.11  142.50                  -40.30\n",
      "   10  10241  12E  13M C   N      66   68 -119.50 -125.60   45.72    2.05  -63.40  146.83   18.49\n",
      "   10         13M  13M N   CA     68   69   95.19  140.50                  -40.50\n",
      "   11  10390 161C 162D C   N    1289 1291   66.61   54.50   32.57    1.63  -63.30  139.44   23.71\n",
      "   11        162D 162D N   CA   1291 1292   10.66   40.90                  -40.00\n",
      "   12  10585 356P 357G C   N    2880 2882   84.71   78.70    6.02    0.30  -80.20  166.07   11.59\n",
      "   12        357G 357G N   CA   2882 2883 -166.34 -166.10                  174.10\n",
      "   13  10586 357G 358Y C   N    2884 2886  -61.14  -98.40   37.53    1.57  -63.50  167.28   26.66\n",
      "   13        358Y 358Y N   CA   2886 2887  123.86  128.40                  -43.40\n",
      "   14  10587 358Y 359R C   N    2896 2898  -78.19  -72.10   25.82    1.85  -63.00  152.67   21.66\n",
      "   14        359R 359R N   CA   2898 2899  166.99  141.90                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:   249  405   2.234\n",
      "  serious non-bonded atom clash:   249  406   2.005\n",
      "  serious non-bonded atom clash:   251  402   1.595\n",
      "  serious non-bonded atom clash:   251  403   1.399\n",
      "  serious non-bonded atom clash:   251  405   1.244\n",
      "  serious non-bonded atom clash:   252  405   1.674\n",
      "  serious non-bonded atom clash:   253  403   1.994\n",
      "  serious non-bonded atom clash:   253  405   0.680\n",
      "  serious non-bonded atom clash:   254  405   1.719\n",
      "  serious non-bonded atom clash:   254  417   2.130\n",
      "  serious non-bonded atom clash:   650 1319   2.231\n",
      "  serious non-bonded atom clash:   650 1321   2.232\n",
      "  serious non-bonded atom clash:   651 1321   1.854\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 5367 atoms and 1 coordinate set(s) were parsed in 0.07s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     9    1    3    2    7    7   23   65  165  175  188  260  361  457  478\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "5wp1_MODLL.B99990001.pdb       231.89360\n",
      "\n",
      "Modelando proteína 6cpw\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    36713    34067\n",
      "336 (of 2915 total) atoms selected for optimization\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:      -91.3035\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:      -93.5564\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     336\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    34067    2276\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      518\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         222.2490\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     346       0      0   0.003   0.003      1.1355       1.000\n",
      " 2 Bond angle potential               :     472       0      0   1.720   1.720      25.918       1.000\n",
      " 3 Stereochemical cosine torsion poten:     217       0     10  50.453  50.453      84.539       1.000\n",
      " 4 Stereochemical improper torsion pot:     143       0      0   0.847   0.847      2.6238       1.000\n",
      " 5 Soft-sphere overlap restraints     :     518       0      0   0.002   0.002     0.29263       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     304       0      0   0.124   0.124      2.6789       1.000\n",
      "10 Distance restraints 2 (N-O)        :     282       0      0   0.336   0.336      8.4100       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      46       0      0   2.945   2.945      4.7058       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      33       0      0  79.003  79.003      9.5242       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      26       0      0  73.106  73.106      9.5219       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:      14       0      0 102.042 102.042      9.0724       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       5       0      0  73.753  73.753      2.7287       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     143       0      0   0.337   0.337     0.33875       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      48      28      7  30.439 134.259      53.185       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     197       0      0   0.877   0.877      7.5742       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 6cpw_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    8026.3481\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9  -69.56  -68.20    5.10    0.46  -62.50  178.85   29.66\n",
      "    1          2A   2A N   CA      9   10  140.39  145.30                  -40.90\n",
      "    2  10232   3A   4A C   N      17   19  -75.42  -68.20    7.23    0.62  -62.50  174.47   29.24\n",
      "    2          4A   4A N   CA     19   20  145.11  145.30                  -40.90\n",
      "    3  10233   4A   5A C   N      22   24  -75.04  -68.20    8.30    0.54  -62.50  169.57   28.42\n",
      "    3          5A   5A N   CA     24   25  150.00  145.30                  -40.90\n",
      "    4  10234   5A   6A C   N      27   29  -67.53  -68.20    3.33    0.29  -62.50  170.61   28.21\n",
      "    4          6A   6A N   CA     29   30  148.56  145.30                  -40.90\n",
      "    5  10235   6A   7A C   N      32   34 -116.98 -134.00   17.04    0.60  -62.50  179.67   32.35\n",
      "    5          7A   7A N   CA     34   35  147.90  147.00                  -40.90\n",
      "    6  10236   7A   8G C   N      37   39 -113.67  -80.20   36.46    2.69   82.20 -136.89   17.23\n",
      "    6          8G   8G N   CA     39   40  159.63  174.10                    8.50\n",
      "    7  10237   8G   9A C   N      41   43  -76.31  -68.20   16.23    1.05  -62.50  160.34   26.98\n",
      "    7          9A   9A N   CA     43   44  159.36  145.30                  -40.90\n",
      "    8  10238   9A  10G C   N      46   48  -90.51  -80.20   26.03    0.62   82.20 -117.31   18.63\n",
      "    8         10G  10G N   CA     48   49 -161.99  174.10                    8.50\n",
      "    9  10240  11P  12E C   N      57   59 -155.62 -117.80   58.96    2.00  -63.60  165.59   27.09\n",
      "    9         12E  12E N   CA     59   60 -177.97  136.80                  -40.30\n",
      "   10  10241  12E  13M C   N      66   68  -66.03  -73.00   16.64    1.04  -63.40  168.41   24.94\n",
      "   10         13M  13M N   CA     68   69  127.88  143.00                  -40.50\n",
      "   11  10242  13M  14V C   N      74   76  -63.97  -62.40    9.65    1.32 -125.40  175.85    9.88\n",
      "   11         14V  14V N   CA     76   77  -51.93  -42.40                  143.30\n",
      "   12  10243  14V  15R C   N      81   83 -171.26 -125.20   46.11    1.91  -63.00 -150.32   23.72\n",
      "   12         15R  15R N   CA     83   84  138.47  140.60                  -41.10\n",
      "   13  10244  15R  16G C   N      92   94   80.28   78.70   27.26    0.71   82.20  158.19    7.58\n",
      "   13         16G  16G N   CA     94   95  166.68 -166.10                    8.50\n",
      "   14  10245  16G  17Q C   N      96   98 -137.82 -121.10   23.23    0.75  -63.80  179.81   30.32\n",
      "   14         17Q  17Q N   CA     98   99  155.83  139.70                  -40.30\n",
      "   15  10247  18V  19F C   N     112  114  -96.96  -71.40   27.61    2.20  -63.20  177.78   22.52\n",
      "   15         19F  19F N   CA    114  115  130.24  140.70                  -44.30\n",
      "   16  10248  19F  20D C   N     123  125   63.73   54.50   25.10    1.26  -63.30  139.46   23.78\n",
      "   16         20D  20D N   CA    125  126   17.56   40.90                  -40.00\n",
      "   17  10273  44D  45N C   N     311  313   66.39   55.90   31.27    1.56  -63.20  139.31   22.73\n",
      "   17         45N  45N N   CA    313  314   10.04   39.50                  -41.10\n",
      "   18  10274  45N  46V C   N     319  321  -63.70  -62.40    2.81    0.31 -125.40 -172.75   10.48\n",
      "   18         46V  46V N   CA    321  322  -39.91  -42.40                  143.30\n",
      "   19  10275  46V  47N C   N     326  328 -135.39 -119.90   51.57    1.96  -63.20  151.07   22.93\n",
      "   19         47N  47N N   CA    328  329 -173.81  137.00                  -41.10\n",
      "   20  10558 329F 330K C   N    2648 2650 -135.75 -118.00   23.13    0.79  -62.90 -179.39   27.82\n",
      "   20        330K 330K N   CA   2650 2651  153.93  139.10                  -40.80\n",
      "   21  10559 330K 331F C   N    2657 2659 -128.70 -124.20    8.26    0.26  -63.20  177.96   29.01\n",
      "   21        331F 331F N   CA   2659 2660  150.23  143.30                  -44.30\n",
      "   22  10561 332D 333M C   N    2676 2678 -166.43 -125.60   59.14    1.83  -63.40  170.80   33.22\n",
      "   22        333M 333M N   CA   2678 2679 -176.73  140.50                  -40.50\n",
      "   23  10562 333M 334E C   N    2684 2686  -58.75  -69.30   29.55    1.88  -63.60  155.28   20.97\n",
      "   23        334E 334E N   CA   2686 2687  114.90  142.50                  -40.30\n",
      "   24  10563 334E 335L C   N    2693 2695  -69.87  -63.50   12.20    1.50   60.20  143.41   17.39\n",
      "   24        335L 335L N   CA   2695 2696  -30.79  -41.20                   29.60\n",
      "   25  10565 336D 337D C   N    2709 2711  -99.52  -63.30   74.61    7.92  -63.30   74.61    7.92\n",
      "   25        337D 337D N   CA   2711 2712   25.23  -40.00                  -40.00\n",
      "   26  10585 356P 357G C   N    2880 2882  105.03   78.70   44.87    0.77  -80.20  175.55   12.15\n",
      "   26        357G 357G N   CA   2882 2883  157.56 -166.10                  174.10\n",
      "   27  10586 357G 358Y C   N    2884 2886 -108.17  -98.40   10.05    0.48  -63.50  179.79   26.39\n",
      "   27        358Y 358Y N   CA   2886 2887  130.76  128.40                  -43.40\n",
      "   28  10587 358Y 359R C   N    2896 2898  -68.04  -72.10    4.49    0.38  -63.00  175.16   24.00\n",
      "   28        359R 359R N   CA   2898 2899  143.81  141.90                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    0    3    9   39  119  179  191  235  356  403  468\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "6cpw_MODLL.B99990001.pdb       222.24896\n",
      "\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2757 atoms and 1 coordinate set(s) were parsed in 0.03s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 6d5y\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    37162    34516\n",
      "216 (of 2915 total) atoms selected for optimization\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     216\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    34516    1421\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      301\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         221.8780\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     224       0      0   0.005   0.005      1.4807       1.000\n",
      " 2 Bond angle potential               :     309       0      1   2.294   2.294      30.092       1.000\n",
      " 3 Stereochemical cosine torsion poten:     149       0      6  51.087  51.087      56.525       1.000\n",
      " 4 Stereochemical improper torsion pot:      95       0      0   0.995   0.995      2.0154       1.000\n",
      " 5 Soft-sphere overlap restraints     :     301       0      0   0.001   0.001     0.77105E-01   1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     190       0      0   0.218   0.218      6.7364       1.000\n",
      "10 Distance restraints 2 (N-O)        :     169       0      0   0.471   0.471      12.709       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      32       2      1  44.906  44.906      8.3982       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      19       0      0  55.974  55.974      6.9567       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      15       0      0  74.067  74.067      7.7188       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       7       0      0  86.238  86.238      3.1913       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       1       0      0   1.980   1.980     0.55566       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     132       0      0   0.574   0.574     0.87934       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      33      20      5  33.907 132.777      77.719       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      46       0      1   1.374   1.374      6.8223       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 6d5y_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    5961.4512\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 13                           : Mainchain Omega dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10943 356P 356P CA  C    2876 2880   -0.33 -179.18  178.85   41.57 -179.18  178.85   41.57\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9  -70.90  -68.20    6.88    0.64  -62.50 -179.93   29.11\n",
      "    1          2A   2A N   CA      9   10  138.97  145.30                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14  -69.27  -68.20    1.37    0.09  -62.50  173.08   28.70\n",
      "    2          3A   3A N   CA     14   15  146.15  145.30                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19  -75.09  -68.20   11.92    0.76  -62.50  164.55   27.60\n",
      "    3          4A   4A N   CA     19   20  155.03  145.30                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24  -67.43  -68.20    6.25    0.53  -62.50  167.67   27.72\n",
      "    4          5A   5A N   CA     24   25  151.51  145.30                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29  -62.78  -68.20   15.02    1.39  -62.50  159.80   26.20\n",
      "    5          6A   6A N   CA     29   30  159.30  145.30                  -40.90\n",
      "    6  10235   6A   7A C   N      32   34 -127.81 -134.00   13.28    0.82  -62.50  173.14   31.87\n",
      "    6          7A   7A N   CA     34   35  158.76  147.00                  -40.90\n",
      "    7  10237   8G   9A C   N      41   43  -89.68  -68.20   29.34    1.86  -62.50  156.19   27.01\n",
      "    7          9A   9A N   CA     43   44  165.29  145.30                  -40.90\n",
      "    8  10238   9A  10G C   N      46   48  -99.23  -80.20   30.07    2.05   82.20 -131.66   17.77\n",
      "    8         10G  10G N   CA     48   49  150.81  174.10                    8.50\n",
      "    9  10240  11P  12E C   N      57   59  -66.79  -69.30    8.43    0.72  -63.60  169.18   22.73\n",
      "    9         12E  12E N   CA     59   60  150.54  142.50                  -40.30\n",
      "   10  10241  12E  13M C   N      66   68  -68.86  -73.00   42.31    2.93  -63.40  134.50   20.51\n",
      "   10         13M  13M N   CA     68   69 -174.89  143.00                  -40.50\n",
      "   11  10403 174A 175D C   N    1387 1389  -80.37  -63.30  133.22   15.48  -63.30  133.22   15.48\n",
      "   11        175D 175D N   CA   1389 1390   92.13  -40.00                  -40.00\n",
      "   12  10404 175D 176P C   N    1395 1397  -48.63  -58.70   22.81    2.70  -64.50  158.04   10.98\n",
      "   12        176P 176P N   CA   1397 1398  -10.04  -30.50                  147.20\n",
      "   13  10405 176P 177D C   N    1402 1404   79.00   54.50   30.63    2.20  -63.30  155.42   26.49\n",
      "   13        177D 177D N   CA   1404 1405   22.51   40.90                  -40.00\n",
      "   14  10407 178H 179D C   N    1420 1422 -128.43  -96.50   42.87    1.79  -63.30 -171.22   27.51\n",
      "   14        179D 179D N   CA   1422 1423  142.81  114.20                  -40.00\n",
      "   15  10408 179D 180H C   N    1428 1430 -132.28 -125.60    7.37    0.31  -63.20 -169.08   20.61\n",
      "   15        180H 180H N   CA   1430 1431  135.68  138.80                  -42.30\n",
      "   16  10409 180H 181T C   N    1438 1440 -154.89 -124.80   31.88    1.39  -63.20 -172.25   29.97\n",
      "   16        181T 181T N   CA   1440 1441  154.05  143.50                  -42.10\n",
      "   17  10410 181T 182G C   N    1445 1447  116.57   78.70   63.64    1.11   82.20  138.59    8.39\n",
      "   17        182G 182G N   CA   1447 1448  142.76 -166.10                    8.50\n",
      "   18  10411 182G 183F C   N    1449 1451 -143.22 -124.20   41.07    1.38  -63.20  157.79   26.95\n",
      "   18        183F 183F N   CA   1451 1452  179.71  143.30                  -44.30\n",
      "   19  10586 357G 358Y C   N    2884 2886 -126.41 -124.30   10.47    0.65  -63.50  179.91   25.70\n",
      "   19        358Y 358Y N   CA   2886 2887  125.15  135.40                  -43.40\n",
      "   20  10587 358Y 359R C   N    2896 2898  -76.96  -72.10   21.78    1.57  -63.00  156.39   22.08\n",
      "   20        359R 359R N   CA   2898 2899  163.13  141.90                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    3    4   11   20   41   55  139  206  293  355  431  447\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "6d5y_MODLL.B99990001.pdb       221.87798\n",
      "\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 5338 atoms and 1 coordinate set(s) were parsed in 0.05s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 6dcg\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    36742    34096\n",
      "353 (of 2915 total) atoms selected for optimization\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:      -92.8680\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     353\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    34096    2474\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      625\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         229.3336\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     364       0      0   0.003   0.003     0.64086       1.000\n",
      " 2 Bond angle potential               :     497       0      0   1.595   1.595      23.120       1.000\n",
      " 3 Stereochemical cosine torsion poten:     231       0      8  49.272  49.272      82.294       1.000\n",
      " 4 Stereochemical improper torsion pot:     149       0      0   0.864   0.864      2.7919       1.000\n",
      " 5 Soft-sphere overlap restraints     :     625       0      0   0.001   0.001     0.87903E-01   1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     318       0      0   0.138   0.138      2.7293       1.000\n",
      "10 Distance restraints 2 (N-O)        :     299       0      0   0.215   0.215      5.8188       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      48       0      0   2.929   2.929      4.8554       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      35       0      1  83.334  83.334      15.643       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      29       0      0  76.693  76.693      9.5756       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:      16       0      0 104.590 104.590      11.628       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       5       0      0 109.182 109.182      3.4672       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     180       0      0   0.540   0.540      1.2829       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      50      25      5  29.464 122.772      46.304       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     253       0      2   0.945   0.945      19.095       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 6dcg_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    7697.9888\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10232   3A   4A C   N      17   19  -67.18  -68.20    3.29    0.30  -62.50  170.73   28.21\n",
      "    1          4A   4A N   CA     19   20  148.43  145.30                  -40.90\n",
      "    2  10233   4A   5A C   N      22   24  -68.21  -68.20    4.91    0.40  -62.50  168.99   27.98\n",
      "    2          5A   5A N   CA     24   25  150.21  145.30                  -40.90\n",
      "    3  10234   5A   6A C   N      27   29  -67.92  -68.20    1.57    0.12  -62.50  175.43   29.02\n",
      "    3          6A   6A N   CA     29   30  143.76  145.30                  -40.90\n",
      "    4  10235   6A   7A C   N      32   34 -114.16 -134.00   22.12    0.50  -62.50 -174.55   28.34\n",
      "    4          7A   7A N   CA     34   35  137.22  147.00                  -40.90\n",
      "    5  10236   7A   8G C   N      37   39 -105.10  -80.20   28.12    2.08   82.20 -129.59   17.84\n",
      "    5          8G   8G N   CA     39   40  161.03  174.10                    8.50\n",
      "    6  10237   8G   9A C   N      41   43 -103.94 -134.00   30.30    0.86  -62.50 -179.22   31.81\n",
      "    6          9A   9A N   CA     43   44  143.13  147.00                  -40.90\n",
      "    7  10238   9A  10G C   N      46   48  -81.24  -80.20    5.26    0.16   82.20 -123.64    7.22\n",
      "    7         10G  10G N   CA     48   49  179.26  174.10                    8.50\n",
      "    8  10239  10G  11P C   N      50   52  -65.92  -58.70   11.22    0.73  -64.50  169.12   12.69\n",
      "    8         11P  11P N   CA     52   53  -21.91  -30.50                  147.20\n",
      "    9  10240  11P  12E C   N      57   59 -125.06 -117.80   24.92    1.09  -63.60  170.52   26.20\n",
      "    9         12E  12E N   CA     59   60  160.64  136.80                  -40.30\n",
      "   10  10241  12E  13M C   N      66   68 -122.16 -125.60   33.39    1.66  -63.40  157.18   28.12\n",
      "   10         13M  13M N   CA     68   69  173.71  140.50                  -40.50\n",
      "   11  10243  14V  15R C   N      81   83  173.62 -125.20   73.10    2.16  -63.00 -174.66   32.87\n",
      "   11         15R  15R N   CA     83   84 -179.40  140.60                  -41.10\n",
      "   12  10244  15R  16G C   N      92   94  151.57 -167.20   42.56    1.09   82.20  170.31   11.47\n",
      "   12         16G  16G N   CA     94   95  164.04  174.60                    8.50\n",
      "   13  10245  16G  17Q C   N      96   98  -96.89  -73.00   24.80    1.64  -63.80  175.52   27.58\n",
      "   13         17Q  17Q N   CA     98   99  147.33  140.70                  -40.30\n",
      "   14  10273  44D  45N C   N     311  313 -153.61 -119.90   89.26    3.20  -63.20  134.26   21.54\n",
      "   14         45N  45N N   CA    313  314 -140.35  137.00                  -41.10\n",
      "   15  10278  49V  50R C   N     350  352 -163.94 -125.20   40.58    1.34  -72.10   92.47    6.82\n",
      "   15         50R  50R N   CA    352  353  152.70  140.60                  141.90\n",
      "   16  10558 329F 330K C   N    2648 2650 -105.10 -118.00   38.57    1.66  -62.90  149.63   17.59\n",
      "   16        330K 330K N   CA   2650 2651  102.76  139.10                  -40.80\n",
      "   17  10559 330K 331F C   N    2657 2659 -131.28 -124.20   16.88    0.92  -63.20 -174.75   21.77\n",
      "   17        331F 331F N   CA   2659 2660  127.98  143.30                  -44.30\n",
      "   18  10560 331F 332D C   N    2668 2670 -133.97  -63.30   77.78   13.27  -63.30   77.78   13.27\n",
      "   18        332D 332D N   CA   2670 2671  -72.49  -40.00                  -40.00\n",
      "   19  10561 332D 333M C   N    2676 2678  -61.01  -73.00   25.60    1.60  -63.40  160.90   24.21\n",
      "   19        333M 333M N   CA   2678 2679  120.38  143.00                  -40.50\n",
      "   20  10562 333M 334E C   N    2684 2686 -107.05 -117.80   10.76    0.38  -63.60 -178.18   22.15\n",
      "   20        334E 334E N   CA   2686 2687  136.24  136.80                  -40.30\n",
      "   21  10563 334E 335L C   N    2693 2695  -66.17  -70.70   10.28    0.64  -63.50  173.59   23.95\n",
      "   21        335L 335L N   CA   2695 2696  132.37  141.60                  -41.20\n",
      "   22  10565 336D 337D C   N    2709 2711 -128.14  -96.50   32.46    1.32  -63.30  160.64   17.12\n",
      "   22        337D 337D N   CA   2711 2712  106.97  114.20                  -40.00\n",
      "   23  10585 356P 357G C   N    2880 2882  -65.66  -62.40    3.29    0.57   82.20  155.84   11.83\n",
      "   23        357G 357G N   CA   2882 2883  -40.73  -41.20                    8.50\n",
      "   24  10586 357G 358Y C   N    2884 2886 -105.51  -98.40    7.31    0.34  -63.50  178.49   26.30\n",
      "   24        358Y 358Y N   CA   2886 2887  130.08  128.40                  -43.40\n",
      "   25  10587 358Y 359R C   N    2896 2898  -68.17  -72.10    6.03    0.52  -63.00  172.50   23.65\n",
      "   25        359R 359R N   CA   2898 2899  146.48  141.90                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    0    2   12   43  133  166  198  240  354  411  474\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "6dcg_MODLL.B99990001.pdb       229.33359\n",
      "\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 5566 atoms and 1 coordinate set(s) were parsed in 0.06s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 6dmg\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    38097    35451\n",
      "144 (of 2915 total) atoms selected for optimization\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     144\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    35451    1190\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      268\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          77.7111\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     148       0      0   0.003   0.003     0.32082       1.000\n",
      " 2 Bond angle potential               :     205       0      0   1.935   1.935      14.070       1.000\n",
      " 3 Stereochemical cosine torsion poten:      90       0      2  52.364  52.364      38.208       1.000\n",
      " 4 Stereochemical improper torsion pot:      57       0      0   0.847   0.847     0.86664       1.000\n",
      " 5 Soft-sphere overlap restraints     :     268       0      0   0.002   0.002     0.10531       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     198       0      0   0.130   0.130      2.1055       1.000\n",
      "10 Distance restraints 2 (N-O)        :     197       0      0   0.175   0.175      1.6903       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      23       0      0   3.059   3.059      2.3049       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      12       0      0  74.665  74.665     0.61856       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       8       0      0  63.783  63.783      3.2742       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      0  74.270  74.270      1.5423       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       1       0      0  14.119  14.119     0.78817       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     145       0      0   0.320   0.320     0.30398       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      24      12      2  17.496 128.317      10.977       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      77       0      0   0.403   0.403     0.53586       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 6dmg_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    3355.2939\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9  -71.78  -68.20    3.59    0.31  -62.50  174.28   29.03\n",
      "    1          2A   2A N   CA      9   10  145.07  145.30                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14  -82.03  -68.20   13.91    1.12  -62.50  173.34   29.41\n",
      "    2          3A   3A N   CA     14   15  146.86  145.30                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19 -144.79 -134.00   12.19    0.28  -62.50 -174.34   34.83\n",
      "    3          4A   4A N   CA     19   20  152.68  147.00                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24  -71.63  -68.20    3.83    0.26  -62.50  172.34   28.70\n",
      "    4          5A   5A N   CA     24   25  147.00  145.30                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29  -69.05  -68.20    0.99    0.10  -62.50  174.44   28.91\n",
      "    5          6A   6A N   CA     29   30  144.79  145.30                  -40.90\n",
      "    6  10236   7A   8G C   N      37   39 -169.12 -167.20    2.84    0.11   82.20 -163.26   14.28\n",
      "    6          8G   8G N   CA     39   40  172.51  174.60                    8.50\n",
      "    7  10237   8G   9A C   N      41   43 -154.02 -134.00   25.51    0.67  -62.50 -178.88   34.56\n",
      "    7          9A   9A N   CA     43   44  162.81  147.00                  -40.90\n",
      "    8  10238   9A  10G C   N      46   48 -148.27 -167.20   24.43    0.48   82.20 -161.32   14.99\n",
      "    8         10G  10G N   CA     48   49  159.15  174.60                    8.50\n",
      "    9  10389 160T 161C C   N    1283 1285   71.09   57.40   19.12    1.11 -117.90 -151.97    8.93\n",
      "    9        161C 161C N   CA   1285 1286   22.65   36.00                  141.10\n",
      "   10  10585 356P 357G C   N    2880 2882  -64.49  -62.40    2.27    0.36  -80.20  146.43    6.56\n",
      "   10        357G 357G N   CA   2882 2883  -40.31  -41.20                  174.10\n",
      "   11  10586 357G 358Y C   N    2884 2886 -101.22  -98.40    5.45    0.50  -63.50  171.34   25.36\n",
      "   11        358Y 358Y N   CA   2886 2887  123.73  128.40                  -43.40\n",
      "   12  10587 358Y 359R C   N    2896 2898 -114.01 -125.20   22.70    1.25  -63.00  166.56   26.02\n",
      "   12        359R 359R N   CA   2898 2899  160.35  140.60                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:   431 2739   1.088\n",
      "  serious non-bonded atom clash:   431 2740   1.619\n",
      "  serious non-bonded atom clash:   432 2738   1.927\n",
      "  serious non-bonded atom clash:   432 2739   1.013\n",
      "  serious non-bonded atom clash:   432 2740   1.905\n",
      "  serious non-bonded atom clash:   433 2739   1.786\n",
      "  serious non-bonded atom clash:   433 2740   1.055\n",
      "  serious non-bonded atom clash:   434 2740   0.911\n",
      "  serious non-bonded atom clash:   440 2740   1.827\n",
      "  serious non-bonded atom clash:   491  529   2.253\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     9    0    1    6    1    7   22   49  123  193  194  269  341  478  419\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "6dmg_MODLL.B99990001.pdb        77.71114\n",
      "\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2740 atoms and 1 coordinate set(s) were parsed in 0.04s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 6fi3\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    37758    35112\n",
      "259 (of 2915 total) atoms selected for optimization\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     259\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    35112    2179\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      463\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         154.3079\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     268       0      0   0.003   0.003     0.91483       1.000\n",
      " 2 Bond angle potential               :     368       0      0   1.889   1.889      24.933       1.000\n",
      " 3 Stereochemical cosine torsion poten:     172       0      1  46.794  46.794      54.922       1.000\n",
      " 4 Stereochemical improper torsion pot:     110       0      0   1.151   1.151      3.3505       1.000\n",
      " 5 Soft-sphere overlap restraints     :     463       0      0   0.003   0.003     0.42904       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     320       0      0   0.139   0.139      4.4490       1.000\n",
      "10 Distance restraints 2 (N-O)        :     323       0      0   0.271   0.271      7.9206       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      37       1      0  29.623  29.623      4.1722       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      25       0      0  67.733  67.733      1.6865       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      21       0      0  78.240  78.240      9.0478       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:      10       0      0  98.362  98.362      6.4440       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       2       0      0  11.324  11.324      1.0124       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     257       0      0   0.570   0.570      1.9586       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      39      19      3  21.172 127.275      22.766       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     227       0      0   0.600   0.600      10.301       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 6fi3_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    6271.9922\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 13                           : Mainchain Omega dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10747 160T 160T CA  C    1279 1283   -0.07 -179.18  179.11   41.63 -179.18  179.11   41.63\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9 -115.86 -134.00   22.88    1.27  -62.50  166.92   30.20\n",
      "    1          2A   2A N   CA      9   10  160.94  147.00                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14  -61.17  -68.20    7.03    0.59  -62.50  174.13   28.47\n",
      "    2          3A   3A N   CA     14   15  144.98  145.30                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19  -67.74  -68.20    1.55    0.11  -62.50  175.36   29.00\n",
      "    3          4A   4A N   CA     19   20  143.82  145.30                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24 -152.82 -134.00   22.01    0.52  -62.50 -175.67   35.03\n",
      "    4          5A   5A N   CA     24   25  158.42  147.00                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29  -73.07  -68.20   10.04    0.65  -62.50  165.37   27.63\n",
      "    5          6A   6A N   CA     29   30  154.07  145.30                  -40.90\n",
      "    6  10235   6A   7A C   N      32   34  -70.59  -68.20   13.06    0.97  -62.50  161.17   26.82\n",
      "    6          7A   7A N   CA     34   35  158.14  145.30                  -40.90\n",
      "    7  10236   7A   8G C   N      37   39  -93.56  -80.20   33.73    0.80   82.20 -120.00   18.52\n",
      "    7          8G   8G N   CA     39   40 -154.92  174.10                    8.50\n",
      "    8  10237   8G   9A C   N      41   43 -108.80 -134.00   25.19    0.83  -62.50  178.30   31.67\n",
      "    8          9A   9A N   CA     43   44  146.92  147.00                  -40.90\n",
      "    9  10238   9A  10G C   N      46   48 -163.96 -167.20   10.10    0.42   82.20 -150.68   15.14\n",
      "    9         10G  10G N   CA     48   49 -175.84  174.60                    8.50\n",
      "   10  10559 330K 331F C   N    2657 2659  -66.61  -71.40    5.12    0.40  -63.20  173.23   24.28\n",
      "   10        331F 331F N   CA   2659 2660  142.50  140.70                  -44.30\n",
      "   11  10560 331F 332D C   N    2668 2670  -87.41  -70.90   53.60    1.93  -63.30  121.14   16.49\n",
      "   11        332D 332D N   CA   2670 2671 -158.71  150.30                  -40.00\n",
      "   12  10561 332D 333M C   N    2676 2678  -83.81  -73.00   24.47    1.53  -63.40  155.89   24.90\n",
      "   12        333M 333M N   CA   2678 2679  164.95  143.00                  -40.50\n",
      "   13  10562 333M 334E C   N    2684 2686  -76.37  -69.30    9.72    0.54  -63.60  171.00   23.51\n",
      "   13        334E 334E N   CA   2686 2687  149.18  142.50                  -40.30\n",
      "   14  10564 335L 336D C   N    2701 2703  -81.94  -70.90   12.06    0.70  -63.30  165.92   21.65\n",
      "   14        336D 336D N   CA   2703 2704  155.14  150.30                  -40.00\n",
      "   15  10565 336D 337D C   N    2709 2711  -81.62  -70.90   19.19    0.66  -63.30  154.87   20.27\n",
      "   15        337D 337D N   CA   2711 2712  166.21  150.30                  -40.00\n",
      "   16  10583 354F 355Q C   N    2864 2866  -73.68  -73.00    0.85    0.05  -63.80  178.76   26.81\n",
      "   16        355Q 355Q N   CA   2866 2867  141.21  140.70                  -40.30\n",
      "   17  10585 356P 357G C   N    2880 2882  -86.53  -80.20   19.68    0.50   82.20 -116.35   18.61\n",
      "   17        357G 357G N   CA   2882 2883 -167.27  174.10                    8.50\n",
      "   18  10586 357G 358Y C   N    2884 2886 -122.05 -124.30    2.47    0.15  -63.50 -172.83   26.99\n",
      "   18        358Y 358Y N   CA   2886 2887  134.37  135.40                  -43.40\n",
      "   19  10587 358Y 359R C   N    2896 2898 -113.26 -125.20   12.37    0.42  -63.00 -174.59   22.12\n",
      "   19        359R 359R N   CA   2898 2899  137.37  140.60                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:   457  492   2.138\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    1    0    0    3    5   12   50  146  177  180  275  351  423  481\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "6fi3_MODLL.B99990001.pdb       154.30786\n",
      "\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2737 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 6fi6\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    37314    34668\n",
      "264 (of 2915 total) atoms selected for optimization\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     264\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    34668    1993\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      594\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         674.6744\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     273       2      3   0.019   0.019      30.706       1.000\n",
      " 2 Bond angle potential               :     375       1      4   2.553   2.553      45.424       1.000\n",
      " 3 Stereochemical cosine torsion poten:     172       0      3  49.820  49.820      62.563       1.000\n",
      " 4 Stereochemical improper torsion pot:     106       0      0   1.734   1.734      6.0364       1.000\n",
      " 5 Soft-sphere overlap restraints     :     594      14     18   0.078   0.078      426.64       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     312       0      2   0.324   0.324      19.375       1.000\n",
      "10 Distance restraints 2 (N-O)        :     303       0      0   0.326   0.326      8.9682       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      38       0      1   3.771   3.771      4.8006       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      26       0      0  58.077  58.077      3.2871       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      22       0      1  60.758  60.758      8.8588       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:      12       0      0  78.596  78.596      7.3666       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       3       0      0 115.391 115.391      2.3525       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     181       0      0   0.440   0.440     0.87205       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      40      19      4  24.582 121.135      37.358       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     130       0      0   0.906   0.906      10.062       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 6fi6_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    6259.9482\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  1                           : Bond length potential                   \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   2972 359R 359R NE  CD   2903 2902    1.74    1.49    0.25    7.49    1.49    0.25    7.49\n",
      "    2   2973 359R 359R CZ  NE   2904 2903    1.51    1.37    0.14    5.63    1.37    0.14    5.63\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   7018 359R 359R CD  NE   2902 2903  141.65  120.00   21.65    5.49  120.00   21.65    5.49\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9  -68.42  -68.20    4.41    0.35  -62.50  169.50   28.07\n",
      "    1          2A   2A N   CA      9   10  149.71  145.30                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14  -78.38  -68.20   17.53    1.11  -62.50  160.32   27.08\n",
      "    2          3A   3A N   CA     14   15  159.57  145.30                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19  -70.76  -68.20    4.41    0.28  -62.50  170.41   28.34\n",
      "    3          4A   4A N   CA     19   20  148.90  145.30                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24  -64.80  -68.20   13.34    0.96  -62.50  173.32   28.29\n",
      "    4          5A   5A N   CA     24   25  132.41  145.30                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29  -77.16  -68.20    9.02    0.80  -62.50  175.42   29.49\n",
      "    5          6A   6A N   CA     29   30  144.29  145.30                  -40.90\n",
      "    6  10235   6A   7A C   N      32   34 -156.50 -134.00   26.72    0.64  -62.50 -176.41   35.10\n",
      "    6          7A   7A N   CA     34   35  161.40  147.00                  -40.90\n",
      "    7  10236   7A   8G C   N      37   39  -71.93  -80.20   16.31    0.39   82.20 -143.85    6.82\n",
      "    7          8G   8G N   CA     39   40  160.04  174.10                    8.50\n",
      "    8  10237   8G   9A C   N      41   43  -66.92  -68.20    6.06    0.53  -62.50  167.93   27.74\n",
      "    8          9A   9A N   CA     43   44  151.23  145.30                  -40.90\n",
      "    9  10238   9A  10G C   N      46   48   67.58   78.70   42.20    0.77   82.20  134.69    7.27\n",
      "    9         10G  10G N   CA     48   49 -125.39 -166.10                    8.50\n",
      "   10  10390 161C 162D C   N    1289 1291  -87.01  -96.50   75.30    3.17  -63.30   82.96    9.08\n",
      "   10        162D 162D N   CA   1291 1292   39.50  114.20                  -40.00\n",
      "   11  10558 329F 330K C   N    2648 2650 -138.80 -118.00   20.96    0.75  -62.90 -166.91   29.62\n",
      "   11        330K 330K N   CA   2650 2651  141.65  139.10                  -40.80\n",
      "   12  10559 330K 331F C   N    2657 2659 -117.83 -124.20   18.28    0.98  -63.20  164.59   26.47\n",
      "   12        331F 331F N   CA   2659 2660  160.44  143.30                  -44.30\n",
      "   13  10560 331F 332D C   N    2668 2670  -86.02  -70.90   19.32    0.88  -63.30  159.31   21.10\n",
      "   13        332D 332D N   CA   2670 2671  162.32  150.30                  -40.00\n",
      "   14  10561 332D 333M C   N    2676 2678 -142.44 -125.60   16.87    0.64  -63.40 -165.24   35.31\n",
      "   14        333M 333M N   CA   2678 2679  141.50  140.50                  -40.50\n",
      "   15  10562 333M 334E C   N    2684 2686 -132.09 -117.80   16.10    0.47  -63.60 -171.63   28.98\n",
      "   15        334E 334E N   CA   2686 2687  144.22  136.80                  -40.30\n",
      "   16  10564 335L 336D C   N    2701 2703  -77.64  -70.90   10.53    0.39  -63.30  162.25   20.92\n",
      "   16        336D 336D N   CA   2703 2704  158.38  150.30                  -40.00\n",
      "   17  10565 336D 337D C   N    2709 2711  -73.27  -70.90    7.14    0.25  -63.30  163.28   20.77\n",
      "   17        337D 337D N   CA   2711 2712  157.03  150.30                  -40.00\n",
      "   18  10585 356P 357G C   N    2880 2882 -125.70 -167.20   43.76    0.63   82.20 -144.83   16.50\n",
      "   18        357G 357G N   CA   2882 2883  160.71  174.60                    8.50\n",
      "   19  10587 358Y 359R C   N    2896 2898 -100.17 -125.20   27.82    0.85  -63.00  173.58   21.19\n",
      "   19        359R 359R N   CA   2898 2899  128.45  140.60                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:   139 2903   2.140\n",
      "  serious non-bonded atom clash:   167 2903   2.265\n",
      "  serious non-bonded atom clash:   380 2906   2.257\n",
      "  serious non-bonded atom clash:   650 1319   2.235\n",
      "  serious non-bonded atom clash:   650 1321   2.268\n",
      "  serious non-bonded atom clash:   651 1321   1.789\n",
      "  serious non-bonded atom clash:   652 1321   2.242\n",
      "  serious non-bonded atom clash:   699 2903   2.101\n",
      "  serious non-bonded atom clash:   699 2904   2.246\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     1    2    6    2    2   10   23   58  137  197  183  277  366  465  488\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "6fi6_MODLL.B99990001.pdb       674.67444\n",
      "\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2756 atoms and 1 coordinate set(s) were parsed in 0.03s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 6fj0\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    37733    35087\n",
      "241 (of 2915 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2744 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     241\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    35087    1943\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      470\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         166.0370\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     248       0      0   0.005   0.005      1.8873       1.000\n",
      " 2 Bond angle potential               :     341       0      1   2.005   2.005      26.302       1.000\n",
      " 3 Stereochemical cosine torsion poten:     151       0      5  50.461  50.461      59.011       1.000\n",
      " 4 Stereochemical improper torsion pot:     100       0      0   0.991   0.991      2.2966       1.000\n",
      " 5 Soft-sphere overlap restraints     :     470       0      0   0.004   0.004      1.0172       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     302       0      0   0.119   0.119      3.7803       1.000\n",
      "10 Distance restraints 2 (N-O)        :     299       0      0   0.370   0.370      16.285       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      35       1      0  30.601  30.601      4.8905       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      23       0      0  60.631  60.631      2.2495       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      19       0      0  69.459  69.459      7.0793       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       9       0      0  73.183  73.183      7.3439       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       2       0      0  57.036  57.036      1.0156       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     210       0      0   0.555   0.555      1.1572       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      37      16      5  26.321 115.812      26.653       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     167       0      0   0.562   0.562      5.0692       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 6fj0_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    5431.6338\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9 -115.84 -134.00   21.65    0.52  -62.50 -175.99   28.07\n",
      "    1          2A   2A N   CA      9   10  135.21  147.00                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14  -71.18  -68.20    7.88    0.53  -62.50  166.73   27.76\n",
      "    2          3A   3A N   CA     14   15  152.60  145.30                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19 -117.31 -134.00   17.13    0.72  -62.50  176.97   31.92\n",
      "    3          4A   4A N   CA     19   20  150.83  147.00                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24  -61.92  -68.20   11.90    0.76  -62.50  176.09   28.89\n",
      "    4          5A   5A N   CA     24   25  135.19  145.30                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29  -76.77  -68.20   17.78    1.16  -62.50  158.87   26.76\n",
      "    5          6A   6A N   CA     29   30  160.88  145.30                  -40.90\n",
      "    6  10235   6A   7A C   N      32   34 -160.62 -134.00   46.65    1.72  -62.50  165.92   32.35\n",
      "    6          7A   7A N   CA     34   35 -174.70  147.00                  -40.90\n",
      "    7  10237   8G   9A C   N      41   43  -57.49  -68.20   11.47    0.83  -62.50  177.98   28.93\n",
      "    7          9A   9A N   CA     43   44  141.20  145.30                  -40.90\n",
      "    8  10238   9A  10G C   N      46   48  -91.06  -80.20   47.04    1.34   82.20 -131.72   17.71\n",
      "    8         10G  10G N   CA     48   49 -140.13  174.10                    8.50\n",
      "    9  10560 331F 332D C   N    2668 2670  -92.95  -70.90   33.08    1.26  -63.30  148.04   20.17\n",
      "    9        332D 332D N   CA   2670 2671  174.96  150.30                  -40.00\n",
      "   10  10561 332D 333M C   N    2676 2678 -119.40 -125.60   30.65    1.58  -63.40  159.16   28.21\n",
      "   10        333M 333M N   CA   2678 2679  170.51  140.50                  -40.50\n",
      "   11  10562 333M 334E C   N    2684 2686  -84.62  -69.30   65.16    5.47  -63.60  121.30   15.11\n",
      "   11        334E 334E N   CA   2686 2687   79.16  142.50                  -40.30\n",
      "   12  10565 336D 337D C   N    2709 2711 -109.48  -96.50   45.89    1.91   54.50  166.58   17.89\n",
      "   12        337D 337D N   CA   2711 2712   70.18  114.20                   40.90\n",
      "   13  10583 354F 355Q C   N    2864 2866  -69.70  -73.00    4.82    0.37  -63.80  175.59   26.13\n",
      "   13        355Q 355Q N   CA   2866 2867  144.21  140.70                  -40.30\n",
      "   14  10585 356P 357G C   N    2880 2882 -159.01 -167.20    8.25    0.18   82.20 -154.95   15.06\n",
      "   14        357G 357G N   CA   2882 2883  175.64  174.60                    8.50\n",
      "   15  10586 357G 358Y C   N    2884 2886  -77.22  -98.40   23.37    1.41  -63.50  162.50   25.09\n",
      "   15        358Y 358Y N   CA   2886 2887  118.52  128.40                  -43.40\n",
      "   16  10587 358Y 359R C   N    2896 2898 -117.12 -125.20    8.80    0.44  -63.00 -177.00   28.46\n",
      "   16        359R 359R N   CA   2898 2899  144.08  140.60                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    1    6   19   44  145  177  196  237  373  455  469\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "6fj0_MODLL.B99990001.pdb       166.03697\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 6fjb\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    37151    34505\n",
      "263 (of 2915 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2812 atoms and 1 coordinate set(s) were parsed in 0.03s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     263\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    34505    2038\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      602\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         662.1087\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     272       3      4   0.026   0.026      48.798       1.000\n",
      " 2 Bond angle potential               :     374       1      4   2.729   2.729      53.024       1.000\n",
      " 3 Stereochemical cosine torsion poten:     172       0      6  52.863  52.863      67.391       1.000\n",
      " 4 Stereochemical improper torsion pot:     106       0      0   1.515   1.515      4.1756       1.000\n",
      " 5 Soft-sphere overlap restraints     :     602      13     16   0.074   0.074      394.14       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     310       0      0   0.151   0.151      5.6156       1.000\n",
      "10 Distance restraints 2 (N-O)        :     314       0      0   0.215   0.215      6.7197       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      38       0      1   4.043   4.043      5.8530       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      26       0      1  68.496  68.496      7.9651       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      22       0      0  58.087  58.087      10.475       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:      12       0      1  66.722  66.722      7.9446       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       2       0      0 116.381 116.381      1.9443       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     203       0      0   0.472   0.472      1.0777       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      40      22      5  26.539 133.057      39.946       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     147       0      0   0.655   0.655      7.0385       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 6fjb_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    6556.7729\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  1                           : Bond length potential                   \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   2971 359R 359R CD  CG   2902 2901    1.85    1.53    0.32    8.90    1.53    0.32    8.90\n",
      "    2   2972 359R 359R NE  CD   2903 2902    1.73    1.49    0.24    7.11    1.49    0.24    7.11\n",
      "    3   2973 359R 359R CZ  NE   2904 2903    1.49    1.37    0.13    5.05    1.37    0.13    5.05\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   7017 359R 359R CG  CD   2901 2902  131.25  107.50   23.75    6.28  107.50   23.75    6.28\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9  -79.86  -68.20   11.93    1.09  -62.50  177.16   29.92\n",
      "    1          2A   2A N   CA      9   10  142.79  145.30                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14  -81.56  -68.20   20.18    1.27  -62.50  159.81   27.17\n",
      "    2          3A   3A N   CA     14   15  160.43  145.30                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19  -75.14  -68.20   11.87    0.75  -62.50  164.66   27.62\n",
      "    3          4A   4A N   CA     19   20  154.93  145.30                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24  -70.97  -68.20   10.27    0.73  -62.50  164.13   27.32\n",
      "    4          5A   5A N   CA     24   25  155.19  145.30                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29  -64.85  -68.20    6.04    0.58  -62.50  168.79   27.78\n",
      "    5          6A   6A N   CA     29   30  150.33  145.30                  -40.90\n",
      "    6  10235   6A   7A C   N      32   34 -122.50 -134.00   21.61    1.32  -62.50  165.10   30.27\n",
      "    6          7A   7A N   CA     34   35  165.29  147.00                  -40.90\n",
      "    7  10236   7A   8G C   N      37   39  -99.38  -80.20   53.69    1.31   82.20 -130.56    8.14\n",
      "    7          8G   8G N   CA     39   40 -135.75  174.10                    8.50\n",
      "    8  10237   8G   9A C   N      41   43  -69.64  -68.20    7.93    0.59  -62.50  166.15   27.58\n",
      "    8          9A   9A N   CA     43   44  153.10  145.30                  -40.90\n",
      "    9  10238   9A  10G C   N      46   48 -115.03  -80.20   35.40    2.14   82.20 -123.25   18.06\n",
      "    9         10G  10G N   CA     48   49 -179.58  174.10                    8.50\n",
      "   10  10240  11P  12E C   N      57   59  -50.68  -69.30   18.62    1.40  -63.60  177.96   23.04\n",
      "   10         12E  12E N   CA     59   60  142.21  142.50                  -40.30\n",
      "   11  10389 160T 161C C   N    1283 1285   73.72   57.40   21.38    1.39 -117.90 -153.86    8.87\n",
      "   11        161C 161C N   CA   1285 1286   22.19   36.00                  141.10\n",
      "   12  10558 329F 330K C   N    2648 2650 -109.37 -118.00   16.79    0.87  -62.90  172.08   25.10\n",
      "   12        330K 330K N   CA   2650 2651  153.51  139.10                  -40.80\n",
      "   13  10559 330K 331F C   N    2657 2659 -133.41 -124.20   10.81    0.28  -63.20 -179.09   29.71\n",
      "   13        331F 331F N   CA   2659 2660  148.97  143.30                  -44.30\n",
      "   14  10560 331F 332D C   N    2668 2670  -91.26  -96.50   21.83    0.91  -63.30  177.61   20.38\n",
      "   14        332D 332D N   CA   2670 2671  135.39  114.20                  -40.00\n",
      "   15  10561 332D 333M C   N    2676 2678  -88.75  -73.00   27.09    2.10  -63.40  163.43   22.54\n",
      "   15        333M 333M N   CA   2678 2679  120.95  143.00                  -40.50\n",
      "   16  10562 333M 334E C   N    2684 2686 -163.20 -117.80   54.97    1.63  -63.60 -178.36   29.65\n",
      "   16        334E 334E N   CA   2686 2687  167.80  136.80                  -40.30\n",
      "   17  10563 334E 335L C   N    2693 2695  -66.94  -70.70   14.01    0.93  -63.50  169.34   23.32\n",
      "   17        335L 335L N   CA   2695 2696  128.10  141.60                  -41.20\n",
      "   18  10564 335L 336D C   N    2701 2703  -90.50  -96.50   32.49    1.37  -63.30  125.26   14.03\n",
      "   18        336D 336D N   CA   2703 2704   82.27  114.20                  -40.00\n",
      "   19  10565 336D 337D C   N    2709 2711 -129.06  -96.50   36.19    1.47  -63.30  153.24   16.29\n",
      "   19        337D 337D N   CA   2711 2712   98.42  114.20                  -40.00\n",
      "   20  10566 337D 338L C   N    2717 2719 -116.62 -108.50   13.02    0.74   60.20 -160.35   15.75\n",
      "   20        338L 338L N   CA   2719 2720  122.32  132.50                   29.60\n",
      "   21  10585 356P 357G C   N    2880 2882  -69.66  -80.20   33.48    0.85   82.20 -157.59    6.81\n",
      "   21        357G 357G N   CA   2882 2883  142.33  174.10                    8.50\n",
      "   22  10587 358Y 359R C   N    2896 2898  -79.55  -72.10   62.92    5.00  -63.00  121.66   15.38\n",
      "   22        359R 359R N   CA   2898 2899   79.43  141.90                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:    88  203   2.093\n",
      "  serious non-bonded atom clash:    88  204   1.947\n",
      "  serious non-bonded atom clash:    88  205   2.090\n",
      "  serious non-bonded atom clash:    89  201   2.211\n",
      "  serious non-bonded atom clash:    89  203   1.544\n",
      "  serious non-bonded atom clash:    89  204   1.089\n",
      "  serious non-bonded atom clash:    89  205   2.187\n",
      "  serious non-bonded atom clash:    90  201   2.289\n",
      "  serious non-bonded atom clash:    90  202   1.964\n",
      "  serious non-bonded atom clash:    90  204   2.209\n",
      "  serious non-bonded atom clash:    90  279   2.091\n",
      "  serious non-bonded atom clash:    91  203   1.499\n",
      "  serious non-bonded atom clash:    91  204   0.359\n",
      "  serious non-bonded atom clash:    91  205   1.749\n",
      "  serious non-bonded atom clash:    91  213   1.198\n",
      "  serious non-bonded atom clash:    91  214   2.073\n",
      "  serious non-bonded atom clash:    91  215   2.191\n",
      "  serious non-bonded atom clash:   139 2901   2.001\n",
      "  serious non-bonded atom clash:   139 2902   2.286\n",
      "  serious non-bonded atom clash:   162 2902   2.122\n",
      "  serious non-bonded atom clash:   173 2903   1.884\n",
      "  serious non-bonded atom clash:   173 2904   2.298\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:    14    3    5    5    7   12   14   48  144  191  204  282  379  430  458\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "6fjb_MODLL.B99990001.pdb       662.10870\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 6fjz\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    38462    35816\n",
      "153 (of 2915 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2757 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     153\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    35816    1317\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      516\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :        1299.1804\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     157       7      7   0.054   0.054      136.26       1.000\n",
      " 2 Bond angle potential               :     217       7     12   6.803   6.803      172.99       1.000\n",
      " 3 Stereochemical cosine torsion poten:      94       0      3  53.109  53.109      43.742       1.000\n",
      " 4 Stereochemical improper torsion pot:      61       0      0   1.607   1.607      2.7348       1.000\n",
      " 5 Soft-sphere overlap restraints     :     516      30     38   0.121   0.121      893.80       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     222       0      0   0.131   0.131      2.7695       1.000\n",
      "10 Distance restraints 2 (N-O)        :     205       0      0   0.139   0.139      2.1507       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      24       0      0   3.183   3.183      1.3399       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      13       0      0  66.857  66.857      1.4925       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       9       0      0  81.324  81.324      2.3503       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       6       0      1  49.952  49.952      5.7273       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       1       0      0  50.645  50.645      1.0957       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     165       0      0   0.387   0.387     0.55905       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      25      14      4  25.742 132.615      28.330       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     118       0      0   0.571   0.571      3.8432       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 6fjz_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    4866.7891\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  1                           : Bond length potential                   \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   2970 359R 359R CG  CB   2901 2900    1.75    1.53    0.22    6.03    1.53    0.22    6.03\n",
      "    2   2971 359R 359R CD  CG   2902 2901    1.76    1.53    0.23    6.41    1.53    0.23    6.41\n",
      "    3   2972 359R 359R NE  CD   2903 2902    1.68    1.49    0.19    5.60    1.49    0.19    5.60\n",
      "    4   2973 359R 359R CZ  NE   2904 2903    1.49    1.37    0.12    4.91    1.37    0.12    4.91\n",
      "    5   2979 359R 360S C   N    2907 2909    1.68    1.35    0.33   11.74    1.35    0.33   11.74\n",
      "    6   2980 360S 360S CB  CA   2911 2910    1.77    1.54    0.23    6.38    1.54    0.23    6.38\n",
      "    7   2982 360S 360S N   CA   2909 2910    1.79    1.43    0.36   11.72    1.43    0.36   11.72\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   7016 359R 359R CB  CG   2900 2901  148.41  113.60   34.81    8.54  113.60   34.81    8.54\n",
      "    2   7018 359R 359R CD  NE   2902 2903  152.42  120.00   32.42    8.22  120.00   32.42    8.22\n",
      "    3   7023 359R 360S C   N    2907 2909  158.05  120.00   38.05    8.65  120.00   38.05    8.65\n",
      "    4   7025 360S 360S N   CA   2909 2910  132.60  113.50   19.11    5.14  113.50   19.11    5.14\n",
      "    5   7026 360S 360S N   CA   2909 2910  147.97  107.00   40.98    9.31  107.00   40.98    9.31\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9 -142.12 -134.00   14.72    0.56  -62.50  178.56   33.53\n",
      "    1          2A   2A N   CA      9   10  159.28  147.00                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14  -72.46  -68.20    6.39    0.40  -62.50  169.33   28.25\n",
      "    2          3A   3A N   CA     14   15  150.06  145.30                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19  -70.68  -68.20    2.48    0.21  -62.50  173.91   28.91\n",
      "    3          4A   4A N   CA     19   20  145.39  145.30                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24  -71.52  -68.20    3.42    0.31  -62.50  174.82   29.10\n",
      "    4          5A   5A N   CA     24   25  144.52  145.30                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29  -80.53  -68.20   21.53    1.37  -62.50  157.19   26.68\n",
      "    5          6A   6A N   CA     29   30  162.95  145.30                  -40.90\n",
      "    6  10235   6A   7A C   N      32   34  -68.60  -68.20    1.26    0.09  -62.50  172.71   28.61\n",
      "    6          7A   7A N   CA     34   35  146.50  145.30                  -40.90\n",
      "    7  10236   7A   8G C   N      37   39   84.57   78.70   18.43    0.31   82.20  167.94    8.27\n",
      "    7          8G   8G N   CA     39   40  176.43 -166.10                    8.50\n",
      "    8  10237   8G   9A C   N      41   43 -133.10 -134.00    7.13    0.42  -62.50  179.50   33.20\n",
      "    8          9A   9A N   CA     43   44  154.07  147.00                  -40.90\n",
      "    9  10238   9A  10G C   N      46   48  -70.75  -80.20   28.62    0.72   82.20 -153.60    6.83\n",
      "    9         10G  10G N   CA     48   49  147.09  174.10                    8.50\n",
      "   10  10389 160T 161C C   N    1283 1285   75.77   57.40   26.05    1.48 -117.90 -152.78    8.96\n",
      "   10        161C 161C N   CA   1285 1286   17.52   36.00                  141.10\n",
      "   11  10584 355Q 356P C   N    2873 2875  -58.21  -58.70    7.02    0.54  -64.50  175.42   13.42\n",
      "   11        356P 356P N   CA   2875 2876  -37.50  -30.50                  147.20\n",
      "   12  10585 356P 357G C   N    2880 2882  104.18   78.70   73.45    1.18   82.20  118.57    6.84\n",
      "   12        357G 357G N   CA   2882 2883  125.02 -166.10                    8.50\n",
      "   13  10586 357G 358Y C   N    2884 2886 -116.03 -124.30   18.48    0.73  -63.50  170.56   24.63\n",
      "   13        358Y 358Y N   CA   2886 2887  118.87  135.40                  -43.40\n",
      "   14  10587 358Y 359R C   N    2896 2898 -153.39 -125.20   35.94    1.06  -63.00 -179.70   30.43\n",
      "   14        359R 359R N   CA   2898 2899  162.90  140.60                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:   139 2901   2.193\n",
      "  serious non-bonded atom clash:   139 2907   2.093\n",
      "  serious non-bonded atom clash:   139 2909   2.040\n",
      "  serious non-bonded atom clash:   141 2909   2.272\n",
      "  serious non-bonded atom clash:   141 2910   2.220\n",
      "  serious non-bonded atom clash:   151 2907   2.238\n",
      "  serious non-bonded atom clash:   151 2909   2.291\n",
      "  serious non-bonded atom clash:   162 2909   2.201\n",
      "  serious non-bonded atom clash:   164 2903   2.171\n",
      "  serious non-bonded atom clash:   173 2910   2.083\n",
      "  serious non-bonded atom clash:   380 2903   2.248\n",
      "  serious non-bonded atom clash:   699 2901   2.159\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     3    3    6    2    9   14   35   52  162  178  226  285  363  472  481\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "6fjz_MODLL.B99990001.pdb      1299.18042\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 6fle\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    37667    35021\n",
      "243 (of 2915 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2788 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     243\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    35021    2089\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      476\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         207.7645\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     252       0      0   0.004   0.004      1.0230       1.000\n",
      " 2 Bond angle potential               :     346       0      0   1.781   1.781      20.024       1.000\n",
      " 3 Stereochemical cosine torsion poten:     163       0      4  50.768  50.768      56.110       1.000\n",
      " 4 Stereochemical improper torsion pot:     104       0      0   0.928   0.928      2.2652       1.000\n",
      " 5 Soft-sphere overlap restraints     :     476       0      0   0.002   0.002     0.17924       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     332       0      0   0.201   0.201      9.1876       1.000\n",
      "10 Distance restraints 2 (N-O)        :     316       0      0   0.322   0.322      16.741       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      35       0      1   4.112   4.112      5.4127       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      23       0      0  73.173  73.173      5.5790       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      19       0      0  50.658  50.658      4.0844       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:      10       0      0  68.331  68.331      7.3453       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       2       0      0 125.923 125.923     0.93354       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     219       0      0   0.558   0.558      1.2773       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      37      18      6  35.131 118.521      72.522       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     231       0      0   0.466   0.466      5.0810       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 6fle_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    5290.3828\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9  -69.04  -68.20    7.72    0.59  -62.50  166.26   27.57\n",
      "    1          2A   2A N   CA      9   10  152.97  145.30                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14  -68.66  -68.20    5.89    0.46  -62.50  168.05   27.85\n",
      "    2          3A   3A N   CA     14   15  151.17  145.30                  -40.90\n",
      "    3  10234   5A   6A C   N      27   29  -77.22  -68.20   23.35    1.58  -62.50  152.98   25.82\n",
      "    3          6A   6A N   CA     29   30  166.84  145.30                  -40.90\n",
      "    4  10235   6A   7A C   N      32   34 -116.91 -134.00   30.01    1.83  -62.50  157.15   28.66\n",
      "    4          7A   7A N   CA     34   35  171.67  147.00                  -40.90\n",
      "    5  10236   7A   8G C   N      37   39 -174.29 -167.20   10.37    0.24   82.20 -157.83   14.40\n",
      "    5          8G   8G N   CA     39   40 -177.83  174.60                    8.50\n",
      "    6  10237   8G   9A C   N      41   43  -64.15  -68.20    4.68    0.31  -62.50  176.15   28.95\n",
      "    6          9A   9A N   CA     43   44  142.96  145.30                  -40.90\n",
      "    7  10238   9A  10G C   N      46   48 -134.13 -167.20   35.54    0.52   82.20 -150.06   16.00\n",
      "    7         10G  10G N   CA     48   49  161.59  174.60                    8.50\n",
      "    8  10558 329F 330K C   N    2648 2650 -116.36 -118.00   29.97    1.48  -62.90  159.40   23.89\n",
      "    8        330K 330K N   CA   2650 2651  169.03  139.10                  -40.80\n",
      "    9  10559 330K 331F C   N    2657 2659  -72.99  -71.40   31.65    2.59  -63.20  153.70   20.69\n",
      "    9        331F 331F N   CA   2659 2660  109.09  140.70                  -44.30\n",
      "   10  10560 331F 332D C   N    2668 2670 -110.47  -70.90   72.63    2.48  -63.30  118.58   17.69\n",
      "   10        332D 332D N   CA   2670 2671 -148.80  150.30                  -40.00\n",
      "   11  10561 332D 333M C   N    2676 2678  -86.29  -73.00   85.73    6.03  -63.40  101.43   13.53\n",
      "   11        333M 333M N   CA   2678 2679   58.31  143.00                  -40.50\n",
      "   12  10562 333M 334E C   N    2684 2686 -120.22 -117.80    9.80    0.53  -63.60  176.91   21.04\n",
      "   12        334E 334E N   CA   2686 2687  127.30  136.80                  -40.30\n",
      "   13  10563 334E 335L C   N    2693 2695 -151.91 -108.50   49.79    2.20  -70.70   82.64    6.97\n",
      "   13        335L 335L N   CA   2695 2696  156.90  132.50                  141.60\n",
      "   14  10565 336D 337D C   N    2709 2711  -97.15  -63.30  101.43   10.95  -63.30  101.43   10.95\n",
      "   14        337D 337D N   CA   2711 2712   55.62  -40.00                  -40.00\n",
      "   15  10583 354F 355Q C   N    2864 2866  -75.90  -73.00   13.09    0.80  -63.80  166.67   25.15\n",
      "   15        355Q 355Q N   CA   2866 2867  153.47  140.70                  -40.30\n",
      "   16  10585 356P 357G C   N    2880 2882  -94.81  -80.20   22.22    0.64   82.20 -109.21   19.23\n",
      "   16        357G 357G N   CA   2882 2883 -169.16  174.10                    8.50\n",
      "   17  10586 357G 358Y C   N    2884 2886  -64.77  -98.40   33.77    1.38  -63.50  168.73   26.71\n",
      "   17        358Y 358Y N   CA   2886 2887  125.32  128.40                  -43.40\n",
      "   18  10587 358Y 359R C   N    2896 2898  -73.08  -72.10    8.58    0.68  -63.00  174.76   22.95\n",
      "   18        359R 359R N   CA   2898 2899  133.37  141.90                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:    73   96   2.054\n",
      "  serious non-bonded atom clash:    73   97   1.338\n",
      "  serious non-bonded atom clash:   113  398   2.258\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     2    0    1    2    1    6   15   44  158  168  199  267  356  444  478\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "6fle_MODLL.B99990001.pdb       207.76448\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 6flv\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    37896    35250\n",
      "194 (of 2915 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2767 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:       90.4684\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     194\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    35250    1818\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      403\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         185.8168\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     199       0      0   0.005   0.005      1.3373       1.000\n",
      " 2 Bond angle potential               :     275       0      0   2.013   2.013      21.142       1.000\n",
      " 3 Stereochemical cosine torsion poten:     114       0      4  53.253  53.253      48.981       1.000\n",
      " 4 Stereochemical improper torsion pot:      76       0      0   1.340   1.340      3.4371       1.000\n",
      " 5 Soft-sphere overlap restraints     :     403       0      0   0.003   0.003     0.47545       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     345       0      0   0.199   0.199      7.6078       1.000\n",
      "10 Distance restraints 2 (N-O)        :     323       0      0   0.366   0.366      17.576       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      30       0      0   4.353   4.353      5.2232       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      18       0      0  49.835  49.835      3.4938       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      14       0      0  82.877  82.877      8.4691       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       8       0      0  83.634  83.634      3.7373       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       1       0      0   2.706   2.706     0.55825       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     240       0      0   0.567   0.567      1.4680       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      32      16      7  26.445 121.425      56.836       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     143       0      0   0.640   0.640      5.4740       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 6flv_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    4635.2759\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9  -69.94  -68.20    9.18    0.68  -62.50  164.96   27.40\n",
      "    1          2A   2A N   CA      9   10  154.31  145.30                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14  -65.55  -68.20   14.85    1.29  -62.50  159.21   26.24\n",
      "    2          3A   3A N   CA     14   15  159.92  145.30                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19  -64.67  -68.20   18.13    1.58  -62.50  156.03   25.68\n",
      "    3          4A   4A N   CA     19   20  163.09  145.30                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24  -60.19  -68.20    8.19    0.75  -62.50  172.10   28.09\n",
      "    4          5A   5A N   CA     24   25  147.02  145.30                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29 -147.28 -134.00   18.58    0.55  -62.50 -179.72   34.08\n",
      "    5          6A   6A N   CA     29   30  160.00  147.00                  -40.90\n",
      "    6  10235   6A   7A C   N      32   34  -66.55  -68.20    3.61    0.24  -62.50  177.06   29.22\n",
      "    6          7A   7A N   CA     34   35  142.09  145.30                  -40.90\n",
      "    7  10236   7A   8G C   N      37   39 -169.93 -167.20   25.96    1.04   82.20  176.96   13.15\n",
      "    7          8G   8G N   CA     39   40  148.78  174.60                    8.50\n",
      "    8  10237   8G   9A C   N      41   43  -76.98  -68.20    9.00    0.69  -62.50  172.44   28.99\n",
      "    8          9A   9A N   CA     43   44  147.28  145.30                  -40.90\n",
      "    9  10238   9A  10G C   N      46   48  170.94 -167.20   21.99    0.48   82.20 -173.77   13.03\n",
      "    9         10G  10G N   CA     48   49  172.23  174.60                    8.50\n",
      "   10  10389 160T 161C C   N    1283 1285   71.87   57.40   16.14    1.43 -117.90 -156.10    8.71\n",
      "   10        161C 161C N   CA   1285 1286   28.87   36.00                  141.10\n",
      "   11  10561 332D 333M C   N    2676 2678  -67.36  -63.40    3.96    0.73  -73.00  176.65   12.03\n",
      "   11        333M 333M N   CA   2678 2679  -40.45  -40.50                  143.00\n",
      "   12  10562 333M 334E C   N    2684 2686  -22.58  -69.30   50.14    3.15  -63.60  169.62   24.94\n",
      "   12        334E 334E N   CA   2686 2687  124.28  142.50                  -40.30\n",
      "   13  10565 336D 337D C   N    2709 2711  -88.29  -63.30   61.09    6.50  -63.30   61.09    6.50\n",
      "   13        337D 337D N   CA   2711 2712   15.74  -40.00                  -40.00\n",
      "   14  10584 355Q 356P C   N    2873 2875  -72.14  -58.70   30.02    1.75  -64.50  151.05   11.68\n",
      "   14        356P 356P N   CA   2875 2876   -3.66  -30.50                  147.20\n",
      "   15  10586 357G 358Y C   N    2884 2886  -77.24  -98.40   21.28    0.88  -63.50  170.16   26.30\n",
      "   15        358Y 358Y N   CA   2886 2887  126.20  128.40                  -43.40\n",
      "   16  10587 358Y 359R C   N    2896 2898 -123.99 -125.20    5.99    0.31  -63.00 -177.09   28.91\n",
      "   16        359R 359R N   CA   2898 2899  146.46  140.60                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:    90  212   2.057\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     1    0    0    0    2    8   16   44  125  180  187  247  349  471  468\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "6flv_MODLL.B99990001.pdb       185.81676\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 6fma\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    37925    35279\n",
      "230 (of 2915 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2742 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:      -90.7473\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     230\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    35279    1842\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      648\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :        1280.5914\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     236       6      8   0.045   0.045      124.93       1.000\n",
      " 2 Bond angle potential               :     325       6     10   4.548   4.548      127.01       1.000\n",
      " 3 Stereochemical cosine torsion poten:     139       0      5  52.080  52.080      57.852       1.000\n",
      " 4 Stereochemical improper torsion pot:      92       1      1   2.767   2.767      12.505       1.000\n",
      " 5 Soft-sphere overlap restraints     :     648      29     38   0.106   0.106      861.89       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     303       0      0   0.114   0.114      3.6210       1.000\n",
      "10 Distance restraints 2 (N-O)        :     286       0      0   0.185   0.185      3.2723       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      34       1      1   7.607   7.607      20.928       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      22       0      1  62.061  62.061      6.2121       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      18       0      0  70.895  70.895      6.9077       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       9       0      0  74.833  74.833      5.7682       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       2       0      0  35.427  35.427      1.4038       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     207       0      0   0.453   0.453     0.75806       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      36      16      5  31.465 118.619      43.039       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     133       0      0   0.588   0.588      4.4992       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 6fma_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    5856.7754\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  1                           : Bond length potential                   \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   2969 359R 359R CB  CA   2900 2899    1.79    1.54    0.26    7.05    1.54    0.26    7.05\n",
      "    2   2970 359R 359R CG  CB   2901 2900    1.78    1.53    0.25    6.75    1.53    0.25    6.75\n",
      "    3   2971 359R 359R CD  CG   2902 2901    1.73    1.53    0.20    5.54    1.53    0.20    5.54\n",
      "    4   2978 359R 359R C   CA   2907 2899    1.95    1.49    0.46   13.49    1.49    0.46   13.49\n",
      "    5   2979 359R 360S C   N    2907 2909    1.54    1.35    0.20    6.97    1.35    0.20    6.97\n",
      "    6   2982 360S 360S N   CA   2909 2910    1.60    1.43    0.17    5.55    1.43    0.17    5.55\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   7015 359R 359R CA  C    2899 2907  139.92  116.50   23.42    6.73  116.50   23.42    6.73\n",
      "    2   7016 359R 359R CB  CG   2900 2901  144.76  113.60   31.16    7.65  113.60   31.16    7.65\n",
      "    3   7018 359R 359R CD  NE   2902 2903  152.60  120.00   32.60    8.27  120.00   32.60    8.27\n",
      "    4   7023 359R 360S C   N    2907 2909  140.31  120.00   20.31    4.62  120.00   20.31    4.62\n",
      "    5   7026 360S 360S N   CA   2909 2910  134.22  107.00   27.22    6.19  107.00   27.22    6.19\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 13                           : Mainchain Omega dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10946 359R 359R CA  C    2899 2907 -143.78 -180.00   36.22    7.24 -180.00   36.22    7.24\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9  -66.02  -68.20    9.02    0.80  -62.50  165.09   27.23\n",
      "    1          2A   2A N   CA      9   10  154.05  145.30                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14  -67.75  -68.20    8.65    0.71  -62.50  165.24   27.34\n",
      "    2          3A   3A N   CA     14   15  153.94  145.30                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19  -77.87  -68.20   14.26    0.90  -62.50  164.04   27.66\n",
      "    3          4A   4A N   CA     19   20  155.78  145.30                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24  -84.03  -68.20   15.90    1.30  -62.50  173.68   29.57\n",
      "    4          5A   5A N   CA     24   25  146.76  145.30                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29 -144.86 -134.00   16.89    0.56  -62.50  179.22   33.78\n",
      "    5          6A   6A N   CA     29   30  159.93  147.00                  -40.90\n",
      "    6  10235   6A   7A C   N      32   34  -77.62  -68.20   16.01    1.01  -62.50  161.57   27.25\n",
      "    6          7A   7A N   CA     34   35  158.24  145.30                  -40.90\n",
      "    7  10237   8G   9A C   N      41   43  -64.11  -68.20    5.80    0.57  -62.50  169.69   27.89\n",
      "    7          9A   9A N   CA     43   44  149.42  145.30                  -40.90\n",
      "    8  10238   9A  10G C   N      46   48 -158.41 -167.20    8.80    0.18   82.20 -155.13   15.07\n",
      "    8         10G  10G N   CA     48   49  174.99  174.60                    8.50\n",
      "    9  10389 160T 161C C   N    1283 1285  160.14   57.40  110.51   10.54 -117.90  167.26    7.41\n",
      "    9        161C 161C N   CA   1285 1286   -4.70   36.00                  141.10\n",
      "   10  10560 331F 332D C   N    2668 2670  -96.34  -96.50   19.53    0.82  -63.30  138.67   15.41\n",
      "   10        332D 332D N   CA   2670 2671   94.68  114.20                  -40.00\n",
      "   11  10561 332D 333M C   N    2676 2678  -65.69  -73.00    8.19    0.66  -63.40  172.83   25.98\n",
      "   11        333M 333M N   CA   2678 2679  146.68  143.00                  -40.50\n",
      "   12  10562 333M 334E C   N    2684 2686  -83.73  -69.30   30.22    2.71  -63.60  157.54   19.97\n",
      "   12        334E 334E N   CA   2686 2687  115.95  142.50                  -40.30\n",
      "   13  10565 336D 337D C   N    2709 2711  -60.17  -63.30    6.51    0.69   54.50  143.70   18.42\n",
      "   13        337D 337D N   CA   2711 2712  -45.71  -40.00                   40.90\n",
      "   14  10585 356P 357G C   N    2880 2882 -141.60 -167.20   52.38    1.55   82.20 -178.22   14.07\n",
      "   14        357G 357G N   CA   2882 2883  128.90  174.60                    8.50\n",
      "   15  10586 357G 358Y C   N    2884 2886 -130.12 -124.30   25.67    1.13  -63.50  169.81   30.25\n",
      "   15        358Y 358Y N   CA   2886 2887  160.40  135.40                  -43.40\n",
      "   16  10587 358Y 359R C   N    2896 2898 -170.44 -125.20   92.16    5.09  -63.00  147.74   17.65\n",
      "   16        359R 359R N   CA   2898 2899   60.31  140.60                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:   139 2899   2.042\n",
      "  serious non-bonded atom clash:   139 2900   2.074\n",
      "  serious non-bonded atom clash:   139 2901   2.181\n",
      "  serious non-bonded atom clash:   139 2907   2.225\n",
      "  serious non-bonded atom clash:   139 2908   2.174\n",
      "  serious non-bonded atom clash:   162 2907   2.065\n",
      "  serious non-bonded atom clash:   164 2903   2.206\n",
      "  serious non-bonded atom clash:   173 2908   2.104\n",
      "  serious non-bonded atom clash:   173 2909   2.293\n",
      "  serious non-bonded atom clash:   699 2901   2.270\n",
      "  serious non-bonded atom clash:  1473 1477   2.298\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     3    3    5    6    5   20   28   58  154  195  195  258  404  431  501\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "6fma_MODLL.B99990001.pdb      1280.59143\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 6fn5\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    37218    34572\n",
      "293 (of 2915 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2783 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     293\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    34572    2595\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      755\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :        1396.7993\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     304       5      5   0.049   0.049      182.56       1.000\n",
      " 2 Bond angle potential               :     419       7     14   4.916   4.916      182.00       1.000\n",
      " 3 Stereochemical cosine torsion poten:     190       0     13  56.316  56.316      89.569       1.000\n",
      " 4 Stereochemical improper torsion pot:     123       0      0   1.513   1.513      7.6898       1.000\n",
      " 5 Soft-sphere overlap restraints     :     755      22     32   0.091   0.091      744.92       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     436       0      0   0.228   0.228      17.013       1.000\n",
      "10 Distance restraints 2 (N-O)        :     410       0      0   0.200   0.200      6.2392       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      44       0      2   4.039   4.039      6.0007       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      27       0      3  82.460  82.460      17.669       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      23       0      1  75.068  75.068      13.659       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:      12       0      0  84.657  84.657      8.2530       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       2       0      0 126.493 126.493      1.8908       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     288       0      0   0.585   0.585      1.8395       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      47      23      9  33.702 124.135      101.93       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     270       0      0   0.768   0.768      15.573       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 6fn5_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    8118.1240\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  1                           : Bond length potential                   \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   2968 358Y 359R C   N    2896 2898    1.56    1.35    0.22    7.77    1.35    0.22    7.77\n",
      "    2   2969 359R 359R CB  CA   2900 2899    1.94    1.54    0.40   11.12    1.54    0.40   11.12\n",
      "    3   2970 359R 359R CG  CB   2901 2900    2.05    1.53    0.52   14.18    1.53    0.52   14.18\n",
      "    4   2976 359R 359R N   CA   2898 2899    1.77    1.43    0.34   11.16    1.43    0.34   11.16\n",
      "    5   2978 359R 359R C   CA   2907 2899    1.80    1.49    0.31    8.93    1.49    0.31    8.93\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   6977 356P 356P N   CA   2875 2876  130.50  108.20   22.30    5.07  108.20   22.30    5.07\n",
      "    2   6986 356P 357G C   N    2880 2882  149.15  120.00   29.15    6.62  120.00   29.15    6.62\n",
      "    3   6988 357G 357G N   CA   2882 2883  126.08  107.00   19.08    5.49  107.00   19.08    5.49\n",
      "    4   7009 358Y 359R C   N    2896 2898  173.44  120.00   53.44   12.14  120.00   53.44   12.14\n",
      "    5   7012 359R 359R N   CA   2898 2899  135.06  107.00   28.06    8.07  107.00   28.06    8.07\n",
      "    6   7013 359R 359R CA  CB   2899 2900  141.87  113.50   28.38    6.97  113.50   28.38    6.97\n",
      "    7   7015 359R 359R CA  C    2899 2907  133.70  116.50   17.20    4.95  116.50   17.20    4.95\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9 -129.04 -134.00    5.03    0.20  -62.50 -176.28   33.67\n",
      "    1          2A   2A N   CA      9   10  147.85  147.00                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14 -128.62 -134.00    7.36    0.43  -62.50  179.69   32.99\n",
      "    2          3A   3A N   CA     14   15  152.01  147.00                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19  -68.61  -68.20    9.18    0.73  -62.50  164.75   27.30\n",
      "    3          4A   4A N   CA     19   20  154.47  145.30                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24  -69.31  -68.20   11.22    0.87  -62.50  162.78   27.02\n",
      "    4          5A   5A N   CA     24   25  156.46  145.30                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29  -65.48  -68.20    4.44    0.28  -62.50  177.33   29.21\n",
      "    5          6A   6A N   CA     29   30  141.79  145.30                  -40.90\n",
      "    6  10235   6A   7A C   N      32   34  -75.55  -68.20    8.10    0.57  -62.50  170.90   28.67\n",
      "    6          7A   7A N   CA     34   35  148.70  145.30                  -40.90\n",
      "    7  10236   7A   8G C   N      37   39  -70.26  -80.20   39.13    1.08   82.20 -161.09    6.90\n",
      "    7          8G   8G N   CA     39   40  136.25  174.10                    8.50\n",
      "    8  10237   8G   9A C   N      41   43 -110.30 -134.00   23.79    0.87  -62.50  176.61   31.48\n",
      "    8          9A   9A N   CA     43   44  149.08  147.00                  -40.90\n",
      "    9  10238   9A  10G C   N      46   48  -94.17  -80.20   19.79    0.63   82.20 -108.27    7.80\n",
      "    9         10G  10G N   CA     48   49 -171.87  174.10                    8.50\n",
      "   10  10259  30Y  31I C   N     213  215 -124.99  -63.40   61.92   11.26  -63.40   61.92   11.26\n",
      "   10         31I  31I N   CA    215  216  -50.00  -43.60                  -43.60\n",
      "   11  10262  33E  34G C   N     234  236 -151.49 -167.20   17.72    0.28   82.20 -157.81   15.11\n",
      "   11         34G  34G N   CA    236  237  166.39  174.60                    8.50\n",
      "   12  10263  34G  35A C   N     238  240  -64.90  -68.20   12.58    0.90  -62.50  174.08   28.41\n",
      "   12         35A  35A N   CA    240  241  133.17  145.30                  -40.90\n",
      "   13  10389 160T 161C C   N    1283 1285  120.41   57.40   63.72    7.15 -117.90  167.16    7.36\n",
      "   13        161C 161C N   CA   1285 1286   26.51   36.00                  141.10\n",
      "   14  10558 329F 330K C   N    2648 2650 -106.51 -118.00   12.60    0.42  -62.90 -179.92   21.40\n",
      "   14        330K 330K N   CA   2650 2651  133.92  139.10                  -40.80\n",
      "   15  10559 330K 331F C   N    2657 2659 -107.19 -124.20   32.10    1.01  -63.20  166.30   20.36\n",
      "   15        331F 331F N   CA   2659 2660  116.08  143.30                  -44.30\n",
      "   16  10560 331F 332D C   N    2668 2670  -83.41  -70.90   17.36    0.71  -63.30  158.95   20.88\n",
      "   16        332D 332D N   CA   2670 2671  162.33  150.30                  -40.00\n",
      "   17  10561 332D 333M C   N    2676 2678  -98.50 -125.60   44.61    1.46  -63.40  149.74   19.90\n",
      "   17        333M 333M N   CA   2678 2679  105.06  140.50                  -40.50\n",
      "   18  10562 333M 334E C   N    2684 2686 -157.07 -117.80   42.11    1.26  -63.60 -168.01   30.78\n",
      "   18        334E 334E N   CA   2686 2687  152.00  136.80                  -40.30\n",
      "   19  10563 334E 335L C   N    2693 2695  -76.13  -70.70    5.69    0.56  -63.50  179.32   25.64\n",
      "   19        335L 335L N   CA   2695 2696  139.93  141.60                  -41.20\n",
      "   20  10564 335L 336D C   N    2701 2703 -107.87  -96.50   27.38    1.14  -63.30  136.76   14.79\n",
      "   20        336D 336D N   CA   2703 2704   89.29  114.20                  -40.00\n",
      "   21  10565 336D 337D C   N    2709 2711 -115.47  -96.50   34.06    1.41  -63.30  136.30   14.57\n",
      "   21        337D 337D N   CA   2711 2712   85.92  114.20                  -40.00\n",
      "   22  10586 357G 358Y C   N    2884 2886 -163.47 -124.30   49.66    2.42  -63.50 -179.17   33.37\n",
      "   22        358Y 358Y N   CA   2886 2887  165.92  135.40                  -43.40\n",
      "   23  10587 358Y 359R C   N    2896 2898  -50.11  -72.10   79.27    5.61  -63.00  107.61   15.42\n",
      "   23        359R 359R N   CA   2898 2899   65.74  141.90                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:    73   97   1.839\n",
      "  serious non-bonded atom clash:   137 2900   2.231\n",
      "  serious non-bonded atom clash:   137 2901   2.177\n",
      "  serious non-bonded atom clash:   139 2898   1.772\n",
      "  serious non-bonded atom clash:   139 2900   2.284\n",
      "  serious non-bonded atom clash:   151 2898   2.206\n",
      "  serious non-bonded atom clash:   162 2899   2.133\n",
      "  serious non-bonded atom clash:   173 2900   1.907\n",
      "  serious non-bonded atom clash:   173 2901   2.244\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     3    2    4    5    9   12   19   60  135  185  203  278  361  439  497\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "6fn5_MODLL.B99990001.pdb      1396.79932\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 6fq7\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    37589    34943\n",
      "225 (of 2915 total) atoms selected for optimization\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     225\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    34943    1911\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      366\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         157.2031\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     232       0      0   0.003   0.003     0.76842       1.000\n",
      " 2 Bond angle potential               :     318       0      0   1.619   1.619      15.855       1.000\n",
      " 3 Stereochemical cosine torsion poten:     142       0      4  49.670  49.670      52.223       1.000\n",
      " 4 Stereochemical improper torsion pot:      94       0      0   1.094   1.094      2.6353       1.000\n",
      " 5 Soft-sphere overlap restraints     :     366       0      0   0.001   0.001     0.69260E-01   1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     312       0      0   0.149   0.149      4.8549       1.000\n",
      "10 Distance restraints 2 (N-O)        :     301       0      0   0.177   0.177      4.5695       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      33       0      0   3.692   3.692      4.1077       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      21       0      0  74.591  74.591      3.2242       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      17       0      0  49.192  49.192      4.6897       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       9       0      0 104.173 104.173      8.0187       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       2       0      0  52.037  52.037      1.1132       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     229       0      0   0.487   0.487      1.4353       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      35      18      7  26.788 127.488      47.478       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     166       0      0   0.571   0.571      6.1604       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 6fq7_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    5203.2852\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9  -64.10  -68.20    8.05    0.52  -62.50  179.28   29.30\n",
      "    1          2A   2A N   CA      9   10  138.38  145.30                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14  -72.57  -68.20    4.98    0.48  -62.50  176.46   29.42\n",
      "    2          3A   3A N   CA     14   15  142.93  145.30                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19  -82.59  -68.20   22.39    1.41  -62.50  157.92   26.91\n",
      "    3          4A   4A N   CA     19   20  162.46  145.30                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24  -76.95  -68.20   38.01    2.75  -62.50  137.58   23.28\n",
      "    4          5A   5A N   CA     24   25 -177.71  145.30                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29  -54.88  -68.20   13.44    1.20  -62.50  172.17   27.85\n",
      "    5          6A   6A N   CA     29   30  147.11  145.30                  -40.90\n",
      "    6  10235   6A   7A C   N      32   34 -111.10 -134.00   22.93    0.81  -62.50  177.66   31.69\n",
      "    6          7A   7A N   CA     34   35  148.22  147.00                  -40.90\n",
      "    7  10236   7A   8G C   N      37   39  137.14 -167.20   57.45    1.47   82.20  161.49   10.43\n",
      "    7          8G   8G N   CA     39   40  160.36  174.60                    8.50\n",
      "    8  10237   8G   9A C   N      41   43 -111.48 -134.00   30.06    0.84  -62.50  174.98   26.74\n",
      "    8          9A   9A N   CA     43   44  127.09  147.00                  -40.90\n",
      "    9  10238   9A  10G C   N      46   48 -156.41 -167.20   37.23    1.28   82.20  178.20   13.57\n",
      "    9         10G  10G N   CA     48   49  138.97  174.60                    8.50\n",
      "   10  10389 160T 161C C   N    1283 1285   74.43   57.40   24.77    1.35 -117.90 -152.00    8.98\n",
      "   10        161C 161C N   CA   1285 1286   18.02   36.00                  141.10\n",
      "   11  10559 330K 331F C   N    2657 2659 -148.79 -124.20   26.52    1.31  -63.20 -162.78   22.71\n",
      "   11        331F 331F N   CA   2659 2660  133.38  143.30                  -44.30\n",
      "   12  10560 331F 332D C   N    2668 2670 -110.79  -70.90   40.16    2.88  -63.30  171.79   24.27\n",
      "   12        332D 332D N   CA   2670 2671  154.90  150.30                  -40.00\n",
      "   13  10561 332D 333M C   N    2676 2678  -74.51  -73.00   12.08    0.79  -63.40  164.88   25.49\n",
      "   13        333M 333M N   CA   2678 2679  154.99  143.00                  -40.50\n",
      "   14  10562 333M 334E C   N    2684 2686  -77.55  -69.30   27.11    2.33  -63.60  157.59   20.28\n",
      "   14        334E 334E N   CA   2686 2687  116.67  142.50                  -40.30\n",
      "   15  10563 334E 335L C   N    2693 2695  -73.57  -63.50   17.48    2.16   60.20  145.21   17.60\n",
      "   15        335L 335L N   CA   2695 2696  -26.91  -41.20                   29.60\n",
      "   16  10585 356P 357G C   N    2880 2882  -80.04  -80.20   20.86    0.83   82.20 -122.43   18.10\n",
      "   16        357G 357G N   CA   2882 2883 -165.05  174.10                    8.50\n",
      "   17  10586 357G 358Y C   N    2884 2886  -90.14  -98.40    9.46    0.55  -63.50  178.41   26.99\n",
      "   17        358Y 358Y N   CA   2886 2887  133.01  128.40                  -43.40\n",
      "   18  10587 358Y 359R C   N    2896 2898  -67.10  -72.10   22.30    1.82  -63.00  155.33   21.26\n",
      "   18        359R 359R N   CA   2898 2899  163.63  141.90                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    3    2   11   33  132  190  183  245  344  414  475\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "6fq7_MODLL.B99990001.pdb       157.20309\n",
      "\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2776 atoms and 1 coordinate set(s) were parsed in 0.03s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 6fr1\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    37878    35232\n",
      "225 (of 2915 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2748 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     225\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    35232    1954\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      629\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         898.7486\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     232       3      5   0.040   0.040      111.45       1.000\n",
      " 2 Bond angle potential               :     318       5     12   5.078   5.078      149.44       1.000\n",
      " 3 Stereochemical cosine torsion poten:     142       0      4  50.735  50.735      59.070       1.000\n",
      " 4 Stereochemical improper torsion pot:      94       0      0   1.446   1.446      4.6981       1.000\n",
      " 5 Soft-sphere overlap restraints     :     629      11     20   0.078   0.078      446.99       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     329       0      0   0.239   0.239      15.722       1.000\n",
      "10 Distance restraints 2 (N-O)        :     313       0      0   0.336   0.336      13.257       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      33       1      0  31.588  31.588      8.6011       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      21       0      1  84.515  84.515      8.2678       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      17       0      0  75.632  75.632      8.5246       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       9       0      0  83.829  83.829      5.7264       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       2       0      0 112.565 112.565      1.6843       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     216       0      0   0.593   0.593      1.4373       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      35      16      8  40.867 124.692      55.316       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     193       0      0   0.636   0.636      8.5660       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 6fr1_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    6318.1475\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  1                           : Bond length potential                   \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   2968 358Y 359R C   N    2896 2898    1.65    1.35    0.31   10.92    1.35    0.31   10.92\n",
      "    2   2969 359R 359R CB  CA   2900 2899    1.87    1.54    0.33    9.07    1.54    0.33    9.07\n",
      "    3   2976 359R 359R N   CA   2898 2899    1.78    1.43    0.35   11.54    1.43    0.35   11.54\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature  2                           : Bond angle potential                    \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1   7009 358Y 359R C   N    2896 2898  171.55  120.00   51.55   11.72  120.00   51.55   11.72\n",
      "    2   7011 359R 359R N   CA   2898 2899  131.25  113.50   17.76    4.77  113.50   17.76    4.77\n",
      "    3   7012 359R 359R N   CA   2898 2899  134.10  107.00   27.10    7.79  107.00   27.10    7.79\n",
      "    4   7013 359R 359R CA  CB   2899 2900  140.49  113.50   26.99    6.63  113.50   26.99    6.63\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9  -70.36  -68.20    2.21    0.17  -62.50  173.50   28.83\n",
      "    1          2A   2A N   CA      9   10  145.78  145.30                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14  -70.11  -68.20    3.49    0.22  -62.50  171.04   28.41\n",
      "    2          3A   3A N   CA     14   15  148.23  145.30                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19  -67.35  -68.20    4.66    0.41  -62.50  169.28   27.98\n",
      "    3          4A   4A N   CA     19   20  149.89  145.30                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24  -65.22  -68.20    3.46    0.34  -62.50  172.06   28.33\n",
      "    4          5A   5A N   CA     24   25  147.07  145.30                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29  -74.26  -68.20   11.15    0.71  -62.50  164.86   27.61\n",
      "    5          6A   6A N   CA     29   30  154.66  145.30                  -40.90\n",
      "    6  10235   6A   7A C   N      32   34 -149.40 -134.00   31.46    1.28  -62.50  168.76   32.29\n",
      "    6          7A   7A N   CA     34   35  174.43  147.00                  -40.90\n",
      "    7  10237   8G   9A C   N      41   43 -172.98 -134.00   51.29    1.41  -62.50  177.37   34.84\n",
      "    7          9A   9A N   CA     43   44 -179.66  147.00                  -40.90\n",
      "    8  10238   9A  10G C   N      46   48  -78.00  -80.20    4.57    0.29   82.20 -126.71    7.08\n",
      "    8         10G  10G N   CA     48   49  178.10  174.10                    8.50\n",
      "    9  10559 330K 331F C   N    2657 2659   89.32 -124.20  146.52    5.62  -63.20 -132.41   25.98\n",
      "    9        331F 331F N   CA   2659 2660  146.77  143.30                  -44.30\n",
      "   10  10560 331F 332D C   N    2668 2670 -155.70  -96.50   64.38    2.66  -63.30 -158.11   21.42\n",
      "   10        332D 332D N   CA   2670 2671  139.51  114.20                  -40.00\n",
      "   11  10561 332D 333M C   N    2676 2678  -99.28 -125.60   38.52    1.20  -63.40  157.03   20.93\n",
      "   11        333M 333M N   CA   2678 2679  112.37  140.50                  -40.50\n",
      "   12  10562 333M 334E C   N    2684 2686 -148.74  -63.60   87.48   12.84  -63.60   87.48   12.84\n",
      "   12        334E 334E N   CA   2686 2687  -20.20  -40.30                  -40.30\n",
      "   13  10563 334E 335L C   N    2693 2695  -59.51  -63.50    4.03    0.67   60.20  139.33   16.81\n",
      "   13        335L 335L N   CA   2695 2696  -41.70  -41.20                   29.60\n",
      "   14  10585 356P 357G C   N    2880 2882 -143.86 -167.20   31.19    0.65   82.20 -162.31   15.04\n",
      "   14        357G 357G N   CA   2882 2883  153.91  174.60                    8.50\n",
      "   15  10586 357G 358Y C   N    2884 2886 -146.74 -124.30   36.80    1.50  -63.50  173.33   31.52\n",
      "   15        358Y 358Y N   CA   2886 2887  164.57  135.40                  -43.40\n",
      "   16  10587 358Y 359R C   N    2896 2898 -162.02 -125.20   77.33    2.79  -63.00  148.23   26.30\n",
      "   16        359R 359R N   CA   2898 2899 -151.40  140.60                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:   139 2896   2.125\n",
      "  serious non-bonded atom clash:   139 2898   1.951\n",
      "  serious non-bonded atom clash:   141 2898   2.139\n",
      "  serious non-bonded atom clash:   141 2899   2.221\n",
      "  serious non-bonded atom clash:   151 2896   2.265\n",
      "  serious non-bonded atom clash:   151 2898   2.080\n",
      "  serious non-bonded atom clash:   173 2899   2.197\n",
      "  serious non-bonded atom clash:   173 2900   2.231\n",
      "  serious non-bonded atom clash:  2715 2718   2.138\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     2    4    3    3    2    9   23   53  168  200  194  280  367  456  480\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "6fr1_MODLL.B99990001.pdb       898.74860\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 6frp\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    37595    34949\n",
      "240 (of 2915 total) atoms selected for optimization\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     240\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    34949    1926\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      461\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         158.2746\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     248       0      0   0.004   0.004     0.93927       1.000\n",
      " 2 Bond angle potential               :     341       0      0   1.723   1.723      19.288       1.000\n",
      " 3 Stereochemical cosine torsion poten:     155       0      6  50.747  50.747      58.046       1.000\n",
      " 4 Stereochemical improper torsion pot:      99       0      0   1.045   1.045      2.5642       1.000\n",
      " 5 Soft-sphere overlap restraints     :     461       0      0   0.001   0.001     0.88636E-01   1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     311       0      0   0.135   0.135      4.2985       1.000\n",
      "10 Distance restraints 2 (N-O)        :     289       0      0   0.205   0.205      5.5566       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      35       0      0   2.912   2.912      2.8042       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      23       0      0  74.642  74.642      5.2689       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      19       0      0  57.586  57.586      6.9710       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:      10       0      0 100.430 100.430      8.4721       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       2       0      0 112.701 112.701      1.4391       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     199       0      0   0.260   0.260     0.30455       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      37      17      6  24.540 117.144      38.250       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     158       0      0   0.481   0.481      3.9831       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 6frp_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    5211.4668\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9  -71.13  -68.20   10.59    0.75  -62.50  163.85   27.28\n",
      "    1          2A   2A N   CA      9   10  155.48  145.30                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14  -75.29  -68.20   16.95    1.13  -62.50  158.92   26.69\n",
      "    2          3A   3A N   CA     14   15  160.70  145.30                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19 -138.43 -134.00   18.46    0.93  -62.50  171.87   32.23\n",
      "    3          4A   4A N   CA     19   20  164.92  147.00                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24  -70.50  -68.20    4.22    0.41  -62.50  177.52   29.49\n",
      "    4          5A   5A N   CA     24   25  141.76  145.30                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29  176.00 -134.00   55.44    1.26  -62.50 -168.39   37.72\n",
      "    5          6A   6A N   CA     29   30  170.94  147.00                  -40.90\n",
      "    6  10237   8G   9A C   N      41   43  -71.75  -68.20    6.29    0.40  -62.50  168.86   28.14\n",
      "    6          9A   9A N   CA     43   44  150.50  145.30                  -40.90\n",
      "    7  10238   9A  10G C   N      46   48  169.03 -167.20   26.78    0.84   82.20  176.57   12.45\n",
      "    7         10G  10G N   CA     48   49  162.25  174.60                    8.50\n",
      "    8  10389 160T 161C C   N    1283 1285   72.97   57.40   18.08    1.47 -117.90 -155.87    8.75\n",
      "    8        161C 161C N   CA   1285 1286   26.81   36.00                  141.10\n",
      "    9  10558 329F 330K C   N    2648 2650 -101.92 -118.00   29.44    1.52  -62.90  160.27   23.11\n",
      "    9        330K 330K N   CA   2650 2651  163.76  139.10                  -40.80\n",
      "   10  10559 330K 331F C   N    2657 2659  -72.87  -71.40   33.38    2.73  -63.20  151.96   20.46\n",
      "   10        331F 331F N   CA   2659 2660  107.35  140.70                  -44.30\n",
      "   11  10560 331F 332D C   N    2668 2670  -86.17  -70.90   43.75    1.53  -63.30  130.72   17.59\n",
      "   11        332D 332D N   CA   2670 2671 -168.70  150.30                  -40.00\n",
      "   12  10561 332D 333M C   N    2676 2678  -64.33  -73.00    9.45    0.75  -63.40  172.74   25.86\n",
      "   12        333M 333M N   CA   2678 2679  146.76  143.00                  -40.50\n",
      "   13  10562 333M 334E C   N    2684 2686  -65.96  -69.30   10.82    0.71  -63.60  172.52   22.88\n",
      "   13        334E 334E N   CA   2686 2687  132.21  142.50                  -40.30\n",
      "   14  10564 335L 336D C   N    2701 2703  -68.85  -70.90   10.65    0.63   54.50  171.99   10.68\n",
      "   14        336D 336D N   CA   2703 2704  160.75  150.30                   40.90\n",
      "   15  10585 356P 357G C   N    2880 2882  -62.85  -62.40    1.42    0.20   82.20  152.90   11.60\n",
      "   15        357G 357G N   CA   2882 2883  -39.85  -41.20                    8.50\n",
      "   16  10586 357G 358Y C   N    2884 2886  -78.09  -98.40   20.42    0.79  -63.50  174.54   26.96\n",
      "   16        358Y 358Y N   CA   2886 2887  130.53  128.40                  -43.40\n",
      "   17  10587 358Y 359R C   N    2896 2898  -69.22  -72.10    5.36    0.46  -63.00  172.60   23.74\n",
      "   17        359R 359R N   CA   2898 2899  146.41  141.90                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    2    0    7   21   28  151  187  205  251  367  436  476\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "6frp_MODLL.B99990001.pdb       158.27461\n",
      "\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2722 atoms and 1 coordinate set(s) were parsed in 0.11s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 6fxv\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "     17     1   13    16      Q     V    8.019\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    37171    34525\n",
      "327 (of 2915 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2834 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     327\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    34525    2886\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      683\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         256.2875\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     338       0      0   0.005   0.005      2.1839       1.000\n",
      " 2 Bond angle potential               :     466       0      0   1.928   1.928      32.372       1.000\n",
      " 3 Stereochemical cosine torsion poten:     208       0      4  48.092  48.092      71.824       1.000\n",
      " 4 Stereochemical improper torsion pot:     138       0      0   1.175   1.175      4.4421       1.000\n",
      " 5 Soft-sphere overlap restraints     :     683       0      0   0.005   0.005      1.9248       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     548       0      0   0.176   0.176      11.772       1.000\n",
      "10 Distance restraints 2 (N-O)        :     501       0      3   0.382   0.382      34.732       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      49       1      2  26.158  26.158      14.353       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      32       0      1  65.112  65.112      5.6578       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      26       0      0  59.952  59.952      7.3704       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:      12       0      0  94.091  94.091      7.7666       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       2       0      0  57.510  57.510      1.0607       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     284       0      0   0.515   0.515      1.4065       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      53      19      5  27.334 112.955      46.895       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     229       0      0   0.755   0.755      12.527       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 6fxv_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    7060.3091\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9  -61.73  -68.20   14.34    0.94  -62.50  173.41   28.46\n",
      "    1          2A   2A N   CA      9   10  132.51  145.30                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14  -69.12  -68.20    6.59    0.56  -62.50  179.80   29.15\n",
      "    2          3A   3A N   CA     14   15  138.78  145.30                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19  -69.41  -68.20   17.29    1.44  -62.50  169.10   27.38\n",
      "    3          4A   4A N   CA     19   20  128.06  145.30                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24  -95.92  -68.20   27.72    2.34  -62.50  176.61   30.69\n",
      "    4          5A   5A N   CA     24   25  145.68  145.30                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29  -69.35  -68.20    4.77    0.34  -62.50  169.31   28.09\n",
      "    5          6A   6A N   CA     29   30  149.93  145.30                  -40.90\n",
      "    6  10235   6A   7A C   N      32   34  -74.82  -68.20   13.96    0.91  -62.50  161.98   27.17\n",
      "    6          7A   7A N   CA     34   35  157.59  145.30                  -40.90\n",
      "    7  10236   7A   8G C   N      37   39  -71.57  -80.20   11.99    0.39   82.20 -140.04    6.80\n",
      "    7          8G   8G N   CA     39   40  165.78  174.10                    8.50\n",
      "    8  10237   8G   9A C   N      41   43  -70.19  -68.20    8.92    0.65  -62.50  165.29   27.47\n",
      "    8          9A   9A N   CA     43   44  153.99  145.30                  -40.90\n",
      "    9  10238   9A  10G C   N      46   48 -140.72 -167.20   26.82    0.43   82.20 -147.89   15.97\n",
      "    9         10G  10G N   CA     48   49  170.37  174.60                    8.50\n",
      "   10  10241  12E  13M C   N      66   68   58.62   56.30   89.31    5.99  -63.40 -155.12   39.76\n",
      "   10         13M  13M N   CA     68   69  124.08   34.80                  -40.50\n",
      "   11  10263  34G  35A C   N     238  240  -66.84  -68.20   18.98    1.58  -62.50  154.93   25.60\n",
      "   11         35A  35A N   CA    240  241  164.23  145.30                  -40.90\n",
      "   12  10559 330K 331F C   N    2657 2659  -56.53  -71.40   15.93    1.26  -63.20  169.41   23.08\n",
      "   12        331F 331F N   CA   2659 2660  146.42  140.70                  -44.30\n",
      "   13  10560 331F 332D C   N    2668 2670 -104.81  -96.50   24.50    1.03  -63.30 -177.96   20.31\n",
      "   13        332D 332D N   CA   2670 2671  137.25  114.20                  -40.00\n",
      "   14  10561 332D 333M C   N    2676 2678  -69.13  -73.00    4.77    0.31  -63.40  179.37   27.23\n",
      "   14        333M 333M N   CA   2678 2679  140.22  143.00                  -40.50\n",
      "   15  10562 333M 334E C   N    2684 2686 -128.57 -117.80   28.01    1.16  -63.60  169.95   26.32\n",
      "   15        334E 334E N   CA   2686 2687  162.66  136.80                  -40.30\n",
      "   16  10563 334E 335L C   N    2693 2695  -63.31  -70.70   10.33    0.66  -63.50  175.58   24.39\n",
      "   16        335L 335L N   CA   2695 2696  134.38  141.60                  -41.20\n",
      "   17  10585 356P 357G C   N    2880 2882  131.39   78.70   55.76    2.26   82.20  174.22   10.82\n",
      "   17        357G 357G N   CA   2882 2883  175.63 -166.10                    8.50\n",
      "   18  10586 357G 358Y C   N    2884 2886 -137.41 -124.30   23.82    0.95  -63.50  177.43   31.80\n",
      "   18        358Y 358Y N   CA   2886 2887  155.29  135.40                  -43.40\n",
      "   19  10587 358Y 359R C   N    2896 2898  -78.63  -72.10    8.10    0.53  -63.00  172.91   24.43\n",
      "   19        359R 359R N   CA   2898 2899  146.70  141.90                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:   113  398   2.254\n",
      "  serious non-bonded atom clash:   457 2726   1.854\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     1    0    1    0    2    7   14   58  156  195  179  279  350  454  485\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "6fxv_MODLL.B99990001.pdb       256.28754\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 6g54\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    38751    36105\n",
      "108 (of 2915 total) atoms selected for optimization\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     108\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    36105     678\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      135\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          63.6489\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     111       0      0   0.003   0.003     0.21011       1.000\n",
      " 2 Bond angle potential               :     152       0      0   1.802   1.802      9.7050       1.000\n",
      " 3 Stereochemical cosine torsion poten:      68       0      0  48.762  48.762      21.047       1.000\n",
      " 4 Stereochemical improper torsion pot:      43       0      0   0.988   0.988      1.1689       1.000\n",
      " 5 Soft-sphere overlap restraints     :     135       0      0   0.001   0.001     0.17453E-01   1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :      82       0      0   0.145   0.145      1.3009       1.000\n",
      "10 Distance restraints 2 (N-O)        :      75       0      0   0.197   0.197      1.1849       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      17       0      0   1.909   1.909     0.73027       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:       7       0      0  96.551  96.551      2.6122       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       6       0      0  46.619  46.619    -0.39522E-01   1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      0  60.919  60.919      2.4449       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       1       0      0  76.329  76.329     0.80300       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      57       0      0   0.480   0.480     0.39703       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      17      11      1  19.138 137.297      16.103       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      37       0      0   1.059   1.059      5.9640       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 6g54_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    2798.5312\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9  -72.51  -68.20   10.40    0.69  -62.50  164.64   27.48\n",
      "    1          2A   2A N   CA      9   10  154.77  145.30                  -40.90\n",
      "    2  10232   3A   4A C   N      17   19  -74.89  -68.20    8.81    0.87  -62.50  179.95   30.11\n",
      "    2          4A   4A N   CA     19   20  139.57  145.30                  -40.90\n",
      "    3  10233   4A   5A C   N      22   24  -66.58  -68.20   13.79    1.05  -62.50  172.55   28.08\n",
      "    3          5A   5A N   CA     24   25  131.61  145.30                  -40.90\n",
      "    4  10234   5A   6A C   N      27   29  -82.26  -68.20   25.62    1.64  -62.50  153.66   26.20\n",
      "    4          6A   6A N   CA     29   30  166.72  145.30                  -40.90\n",
      "    5  10235   6A   7A C   N      32   34  -63.81  -68.20   10.86    1.01  -62.50  163.87   26.92\n",
      "    5          7A   7A N   CA     34   35  155.23  145.30                  -40.90\n",
      "    6  10236   7A   8G C   N      37   39   89.48   78.70   29.10    0.45   82.20  158.54    8.06\n",
      "    6          8G   8G N   CA     39   40  166.87 -166.10                    8.50\n",
      "    7  10237   8G   9A C   N      41   43 -117.68 -134.00   22.08    0.63  -62.50 -178.39   27.63\n",
      "    7          9A   9A N   CA     43   44  132.13  147.00                  -40.90\n",
      "    8  10584 355Q 356P C   N    2873 2875  -67.69  -58.70   16.97    1.01  -64.50  163.33   12.36\n",
      "    8        356P 356P N   CA   2875 2876  -16.10  -30.50                  147.20\n",
      "    9  10585 356P 357G C   N    2880 2882   87.50   78.70    8.80    0.44   82.20  174.72    8.21\n",
      "    9        357G 357G N   CA   2882 2883 -166.14 -166.10                    8.50\n",
      "   10  10586 357G 358Y C   N    2884 2886 -101.59  -98.40    3.21    0.14  -63.50  176.38   26.14\n",
      "   10        358Y 358Y N   CA   2886 2887  128.81  128.40                  -43.40\n",
      "   11  10587 358Y 359R C   N    2896 2898 -108.11 -125.20   18.59    0.92  -63.00  176.85   27.01\n",
      "   11        359R 359R N   CA   2898 2899  147.90  140.60                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:   458  491   2.228\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    1    0    2    8   10   58  118  174  210  255  374  486  466\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "6g54_MODLL.B99990001.pdb        63.64891\n",
      "\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2759 atoms and 1 coordinate set(s) were parsed in 0.03s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 6g8x\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    37653    35007\n",
      "246 (of 2915 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2784 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:      -93.4813\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     246\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    35007    1989\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      423\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         167.2547\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     255       0      0   0.005   0.005      1.6638       1.000\n",
      " 2 Bond angle potential               :     352       0      1   2.139   2.139      30.781       1.000\n",
      " 3 Stereochemical cosine torsion poten:     165       0      1  49.003  49.003      61.661       1.000\n",
      " 4 Stereochemical improper torsion pot:     100       0      0   1.034   1.034      2.3526       1.000\n",
      " 5 Soft-sphere overlap restraints     :     423       0      0   0.004   0.004     0.72482       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     300       0      0   0.164   0.164      5.4617       1.000\n",
      "10 Distance restraints 2 (N-O)        :     300       0      0   0.316   0.316      8.4241       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      36       1      1  29.989  29.989      7.1252       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      24       0      0  64.818  64.818      4.1542       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      20       0      0  61.518  61.518      7.6400       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:      10       0      1 104.464 104.464      7.3001       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       2       0      0 116.498 116.498      1.6365       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     224       0      0   0.525   0.525      1.1417       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      38      18      3  25.861 121.742      23.595       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     163       0      0   0.629   0.629      3.5925       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 6g8x_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    6056.0723\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9 -129.26 -134.00   12.88    0.59  -62.50 -171.84   28.42\n",
      "    1          2A   2A N   CA      9   10  135.02  147.00                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14 -102.85 -134.00   36.11    0.84  -62.50  174.37   26.90\n",
      "    2          3A   3A N   CA     14   15  128.74  147.00                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19 -169.09 -134.00   47.15    1.33  -62.50  176.44   34.51\n",
      "    3          4A   4A N   CA     19   20  178.49  147.00                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24  -73.41  -68.20    5.36    0.49  -62.50  175.40   29.29\n",
      "    4          5A   5A N   CA     24   25  144.04  145.30                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29  -70.47  -68.20    4.47    0.43  -62.50  177.83   29.54\n",
      "    5          6A   6A N   CA     29   30  141.45  145.30                  -40.90\n",
      "    6  10235   6A   7A C   N      32   34  -69.83  -68.20    2.27    0.22  -62.50  175.53   29.13\n",
      "    6          7A   7A N   CA     34   35  143.73  145.30                  -40.90\n",
      "    7  10237   8G   9A C   N      41   43 -140.37 -134.00   12.10    0.47  -62.50  179.57   33.60\n",
      "    7          9A   9A N   CA     43   44  157.29  147.00                  -40.90\n",
      "    8  10238   9A  10G C   N      46   48   77.67   78.70    7.06    0.24   82.20  178.47    8.44\n",
      "    8         10G  10G N   CA     48   49 -173.08 -166.10                    8.50\n",
      "    9  10558 329F 330K C   N    2648 2650 -152.49 -118.00   34.57    1.35  -62.90 -161.15   22.84\n",
      "    9        330K 330K N   CA   2650 2651  136.72  139.10                  -40.80\n",
      "   10  10559 330K 331F C   N    2657 2659 -111.75 -124.20   14.43    0.75  -63.20  172.08   27.14\n",
      "   10        331F 331F N   CA   2659 2660  150.61  143.30                  -44.30\n",
      "   11  10561 332D 333M C   N    2676 2678 -143.43 -125.60   35.93    1.27  -63.40  168.08   31.32\n",
      "   11        333M 333M N   CA   2678 2679  171.69  140.50                  -40.50\n",
      "   12  10562 333M 334E C   N    2684 2686  -76.53  -69.30   10.92    0.61  -63.60  169.51   23.32\n",
      "   12        334E 334E N   CA   2686 2687  150.68  142.50                  -40.30\n",
      "   13  10563 334E 335L C   N    2693 2695 -120.96 -108.50   21.73    1.01  -63.50  178.03   28.27\n",
      "   13        335L 335L N   CA   2695 2696  150.30  132.50                  -41.20\n",
      "   14  10564 335L 336D C   N    2701 2703  -83.15  -70.90   34.78    1.21  -63.30  138.58   18.36\n",
      "   14        336D 336D N   CA   2703 2704 -177.15  150.30                  -40.00\n",
      "   15  10565 336D 337D C   N    2709 2711  -72.82  -70.90   15.78    0.68  -63.30  154.34   19.64\n",
      "   15        337D 337D N   CA   2711 2712  165.96  150.30                  -40.00\n",
      "   16  10585 356P 357G C   N    2880 2882  115.30   78.70   75.40    1.13  -80.20  170.84   12.40\n",
      "   16        357G 357G N   CA   2882 2883  127.98 -166.10                  174.10\n",
      "   17  10586 357G 358Y C   N    2884 2886 -121.15 -124.30    3.66    0.20  -63.50 -173.91   26.86\n",
      "   17        358Y 358Y N   CA   2886 2887  133.53  135.40                  -43.40\n",
      "   18  10587 358Y 359R C   N    2896 2898  -86.33  -72.10   17.80    1.54  -63.00  173.88   22.01\n",
      "   18        359R 359R N   CA   2898 2899  131.21  141.90                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    0    0   10   44  120  153  178  262  341  435  475\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "6g8x_MODLL.B99990001.pdb       167.25465\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 6g91\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    37899    35253\n",
      "203 (of 2915 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2788 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:      -91.4712\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     203\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    35253    1684\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      365\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         137.2298\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     210       0      0   0.005   0.005      1.1089       1.000\n",
      " 2 Bond angle potential               :     289       0      1   2.010   2.010      23.026       1.000\n",
      " 3 Stereochemical cosine torsion poten:     132       0      1  48.797  48.797      47.825       1.000\n",
      " 4 Stereochemical improper torsion pot:      86       0      0   1.046   1.046      2.3509       1.000\n",
      " 5 Soft-sphere overlap restraints     :     365       0      0   0.004   0.004     0.63175       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     281       0      0   0.169   0.169      5.5512       1.000\n",
      "10 Distance restraints 2 (N-O)        :     268       0      0   0.340   0.340      8.6280       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      30       1      1  32.997  32.997      7.6654       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      19       0      0  80.269  80.269      5.3341       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      15       0      0  58.346  58.346      3.4915       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       7       0      0  95.961  95.961      4.0405       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       2       0      0 120.750 120.750      2.2261       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     178       0      0   0.555   0.555     0.92248       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      32      13      2  22.779 112.624      17.093       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     135       0      0   0.698   0.698      7.3351       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 6g91_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    4695.1309\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9  -61.31  -68.20   15.58    1.03  -62.50  172.23   28.28\n",
      "    1          2A   2A N   CA      9   10  131.33  145.30                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14  -77.40  -68.20   12.93    0.82  -62.50  165.39   27.86\n",
      "    2          3A   3A N   CA     14   15  154.38  145.30                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19  -78.99  -68.20   10.80    0.94  -62.50  175.16   29.54\n",
      "    3          4A   4A N   CA     19   20  144.72  145.30                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24 -131.50 -134.00    5.92    0.37  -62.50 -179.55   33.27\n",
      "    4          5A   5A N   CA     24   25  152.36  147.00                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29 -144.58 -134.00   16.56    0.56  -62.50  179.26   33.77\n",
      "    5          6A   6A N   CA     29   30  159.74  147.00                  -40.90\n",
      "    6  10558 329F 330K C   N    2648 2650 -149.97 -118.00   32.85    1.13  -62.90 -166.74   30.31\n",
      "    6        330K 330K N   CA   2650 2651  146.66  139.10                  -40.80\n",
      "    7  10559 330K 331F C   N    2657 2659 -113.17 -124.20   21.17    0.67  -63.20  176.75   21.47\n",
      "    7        331F 331F N   CA   2659 2660  125.24  143.30                  -44.30\n",
      "    8  10561 332D 333M C   N    2676 2678 -124.72 -125.60   26.12    1.27  -63.40  164.73   29.45\n",
      "    8        333M 333M N   CA   2678 2679  166.60  140.50                  -40.50\n",
      "    9  10562 333M 334E C   N    2684 2686  -85.56  -69.30   31.69    2.86  -63.60  157.15   19.83\n",
      "    9        334E 334E N   CA   2686 2687  115.31  142.50                  -40.30\n",
      "   10  10563 334E 335L C   N    2693 2695  -69.44  -63.50   11.16    1.37   60.20  143.42   17.39\n",
      "   10        335L 335L N   CA   2695 2696  -31.75  -41.20                   29.60\n",
      "   11  10585 356P 357G C   N    2880 2882  102.97   78.70   40.39    0.71  -80.20  177.27   12.16\n",
      "   11        357G 357G N   CA   2882 2883  161.61 -166.10                  174.10\n",
      "   12  10586 357G 358Y C   N    2884 2886  -76.70  -98.40   21.86    0.85  -63.50  174.96   27.09\n",
      "   12        358Y 358Y N   CA   2886 2887  131.06  128.40                  -43.40\n",
      "   13  10587 358Y 359R C   N    2896 2898  -73.73  -72.10    5.03    0.35  -63.00  172.58   24.04\n",
      "   13        359R 359R N   CA   2898 2899  146.66  141.90                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:  2715 2718   2.184\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    1    0    0    0    2    8   58  115  152  173  245  326  444  500\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "6g91_MODLL.B99990001.pdb       137.22981\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 6g92\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    37646    35000\n",
      "211 (of 2915 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2790 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:       90.9093\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     211\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    35000    1685\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      378\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         157.9589\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     218       0      0   0.006   0.006      1.7170       1.000\n",
      " 2 Bond angle potential               :     300       0      1   1.965   1.965      21.473       1.000\n",
      " 3 Stereochemical cosine torsion poten:     136       0      2  48.801  48.801      50.178       1.000\n",
      " 4 Stereochemical improper torsion pot:      89       0      0   1.056   1.056      2.3475       1.000\n",
      " 5 Soft-sphere overlap restraints     :     378       0      0   0.005   0.005     0.99010       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     263       0      0   0.213   0.213      7.3947       1.000\n",
      "10 Distance restraints 2 (N-O)        :     253       0      0   0.373   0.373      11.689       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      31       1      2  32.426  32.426      10.289       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      20       0      0  58.812  58.812      2.6242       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      16       0      0  75.017  75.017      6.7461       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       7       0      0 104.372 104.372      5.4684       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       2       0      0 140.578 140.578      1.7424       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     190       0      0   0.584   0.584      1.0930       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      33      13      2  24.446 110.022      21.793       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     127       0      0   0.886   0.886      12.412       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 6g92_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    4911.1475\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9  -70.96  -68.20    3.28    0.22  -62.50  172.24   28.65\n",
      "    1          2A   2A N   CA      9   10  147.07  145.30                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14  -70.82  -68.20    3.45    0.22  -62.50  171.75   28.56\n",
      "    2          3A   3A N   CA     14   15  147.55  145.30                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19 -149.08 -134.00   17.52    0.41  -62.50 -175.27   34.90\n",
      "    3          4A   4A N   CA     19   20  155.92  147.00                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24  -80.25  -68.20   19.16    1.21  -62.50  159.90   27.11\n",
      "    4          5A   5A N   CA     24   25  160.19  145.30                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29  -69.36  -68.20   25.59    2.02  -62.50  148.40   24.66\n",
      "    5          6A   6A N   CA     29   30  170.86  145.30                  -40.90\n",
      "    6  10235   6A   7A C   N      32   34  -56.66  -68.20   21.17    1.35  -62.50  168.56   27.91\n",
      "    6          7A   7A N   CA     34   35  127.56  145.30                  -40.90\n",
      "    7  10558 329F 330K C   N    2648 2650 -117.98 -118.00    3.54    0.17  -62.90 -175.24   21.64\n",
      "    7        330K 330K N   CA   2650 2651  135.56  139.10                  -40.80\n",
      "    8  10559 330K 331F C   N    2657 2659 -126.48 -124.20   10.40    0.43  -63.20  174.15   28.34\n",
      "    8        331F 331F N   CA   2659 2660  153.45  143.30                  -44.30\n",
      "    9  10561 332D 333M C   N    2676 2678 -159.37 -125.60   33.77    1.33  -63.40 -157.11   37.77\n",
      "    9        333M 333M N   CA   2678 2679  140.75  140.50                  -40.50\n",
      "   10  10563 334E 335L C   N    2693 2695 -108.36 -108.50   22.34    1.21  -63.50  157.88   19.84\n",
      "   10        335L 335L N   CA   2695 2696  110.17  132.50                  -41.20\n",
      "   11  10585 356P 357G C   N    2880 2882  129.35   78.70   59.93    1.89  -80.20  150.95   10.40\n",
      "   11        357G 357G N   CA   2882 2883  161.87 -166.10                  174.10\n",
      "   12  10586 357G 358Y C   N    2884 2886 -131.68 -124.30   24.15    1.01  -63.50  172.26   30.72\n",
      "   12        358Y 358Y N   CA   2886 2887  158.40  135.40                  -43.40\n",
      "   13  10587 358Y 359R C   N    2896 2898  -84.23  -72.10   18.05    1.17  -63.00  165.00   23.75\n",
      "   13        359R 359R N   CA   2898 2899  155.27  141.90                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    0    2   11   56   96  148  197  226  323  430  487\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "6g92_MODLL.B99990001.pdb       157.95888\n",
      "\n",
      "Modelando proteína 6g93\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    37932    35286\n",
      "205 (of 2915 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2790 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     205\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    35286    1718\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      356\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         138.0650\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     211       0      0   0.003   0.003     0.50448       1.000\n",
      " 2 Bond angle potential               :     290       0      0   1.839   1.839      19.250       1.000\n",
      " 3 Stereochemical cosine torsion poten:     125       0      1  49.027  49.027      45.894       1.000\n",
      " 4 Stereochemical improper torsion pot:      82       0      0   1.008   1.008      1.9115       1.000\n",
      " 5 Soft-sphere overlap restraints     :     356       0      0   0.002   0.002     0.19157       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     302       0      0   0.154   0.154      3.7490       1.000\n",
      "10 Distance restraints 2 (N-O)        :     283       0      0   0.494   0.494      14.979       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      31       1      0  32.569  32.569      7.3875       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      19       0      0  75.558  75.558      2.2306       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      15       0      0  44.283  44.283      4.9555       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       8       0      0  88.615  88.615      5.3933       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       2       0      0  67.496  67.496      1.7101       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     195       0      0   0.554   0.554      1.1953       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      33      15      4  25.909 129.360      25.387       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     122       0      0   0.591   0.591      3.3260       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 6g93_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    4865.5898\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9 -125.13 -134.00    8.87    0.29  -62.50 -176.85   33.36\n",
      "    1          2A   2A N   CA      9   10  147.00  147.00                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14 -120.54 -134.00   16.33    0.40  -62.50 -172.16   28.56\n",
      "    2          3A   3A N   CA     14   15  137.74  147.00                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19  -74.40  -68.20    6.90    0.48  -62.50  171.18   28.65\n",
      "    3          4A   4A N   CA     19   20  148.34  145.30                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24 -120.11 -134.00   13.90    0.44  -62.50 -178.03   32.89\n",
      "    4          5A   5A N   CA     24   25  146.50  147.00                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29  -69.36  -68.20    3.92    0.28  -62.50  170.20   28.23\n",
      "    5          6A   6A N   CA     29   30  149.05  145.30                  -40.90\n",
      "    6  10235   6A   7A C   N      32   34  -65.16  -68.20    4.53    0.29  -62.50  177.19   29.17\n",
      "    6          7A   7A N   CA     34   35  141.94  145.30                  -40.90\n",
      "    7  10236   7A   8G C   N      37   39  -82.81  -80.20    2.60    0.17   82.20 -126.22    7.30\n",
      "    7          8G   8G N   CA     39   40  174.12  174.10                    8.50\n",
      "    8  10237   8G   9A C   N      41   43  -68.62  -68.20    9.55    0.76  -62.50  164.37   27.24\n",
      "    8          9A   9A N   CA     43   44  154.85  145.30                  -40.90\n",
      "    9  10238   9A  10G C   N      46   48 -103.10  -80.20   57.45    3.38   82.20 -151.99   16.25\n",
      "    9         10G  10G N   CA     48   49  121.41  174.10                    8.50\n",
      "   10  10560 331F 332D C   N    2668 2670  -84.92  -70.90   29.86    1.00  -63.30  144.97   19.26\n",
      "   10        332D 332D N   CA   2670 2671  176.65  150.30                  -40.00\n",
      "   11  10562 333M 334E C   N    2684 2686  -66.69  -69.30    8.60    0.74  -63.60  169.03   22.71\n",
      "   11        334E 334E N   CA   2686 2687  150.70  142.50                  -40.30\n",
      "   12  10563 334E 335L C   N    2693 2695 -147.03 -108.50   70.22    3.30   60.20 -137.61   25.18\n",
      "   12        335L 335L N   CA   2695 2696 -168.79  132.50                   29.60\n",
      "   13  10585 356P 357G C   N    2880 2882   86.28   82.20    8.71    0.61  -80.20 -130.55    7.58\n",
      "   13        357G 357G N   CA   2882 2883   16.20    8.50                  174.10\n",
      "   14  10586 357G 358Y C   N    2884 2886  -61.12  -98.40   37.32    1.47  -63.50  170.23   27.13\n",
      "   14        358Y 358Y N   CA   2886 2887  126.81  128.40                  -43.40\n",
      "   15  10587 358Y 359R C   N    2896 2898 -125.85 -125.20    0.78    0.04  -63.00 -170.54   29.93\n",
      "   15        359R 359R N   CA   2898 2899  140.16  140.60                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:  2715 2718   2.203\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    1    0    0    3   10   43  128  153  168  259  349  447  468\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "6g93_MODLL.B99990001.pdb       138.06499\n",
      "\n",
      "Modelando proteína 6g97\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    38150    35504\n",
      "197 (of 2915 total) atoms selected for optimization\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     197\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    35504    1705\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      318\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         157.2369\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     203       0      0   0.005   0.005      1.1909       1.000\n",
      " 2 Bond angle potential               :     279       0      0   2.276   2.276      28.237       1.000\n",
      " 3 Stereochemical cosine torsion poten:     121       0      4  50.241  50.241      47.082       1.000\n",
      " 4 Stereochemical improper torsion pot:      79       0      0   1.177   1.177      2.3754       1.000\n",
      " 5 Soft-sphere overlap restraints     :     318       0      0   0.002   0.002     0.17259       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     301       0      0   0.156   0.156      6.4939       1.000\n",
      "10 Distance restraints 2 (N-O)        :     292       0      2   0.360   0.360      17.981       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      30       1      0  33.162  33.162      9.6947       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      18       0      0  82.017  82.017      2.1905       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      14       0      0  51.335  51.335      3.1443       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       8       0      0  84.163  84.163      4.9839       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       2       0      0 126.274 126.274     0.97365       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     204       0      0   0.554   0.554      1.3536       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      32      12      3  29.849 102.514      22.734       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     122       0      0   0.849   0.849      8.6293       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 6g97_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    4357.2661\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9  -68.78  -68.20    3.06    0.27  -62.50  176.92   29.31\n",
      "    1          2A   2A N   CA      9   10  142.30  145.30                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14  -78.58  -68.20   10.38    0.88  -62.50  174.61   29.43\n",
      "    2          3A   3A N   CA     14   15  145.23  145.30                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19  -82.44  -68.20   18.59    1.19  -62.50  163.07   27.75\n",
      "    3          4A   4A N   CA     19   20  157.26  145.30                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24  -65.80  -68.20   15.76    1.35  -62.50  158.26   26.10\n",
      "    4          5A   5A N   CA     24   25  160.88  145.30                  -40.90\n",
      "    5  10237   8G   9A C   N      41   43  -76.65  -68.20   15.23    0.97  -62.50  161.75   27.22\n",
      "    5          9A   9A N   CA     43   44  157.98  145.30                  -40.90\n",
      "    6  10238   9A  10G C   N      46   48 -175.32 -167.20    9.29    0.30   82.20 -168.66   13.78\n",
      "    6         10G  10G N   CA     48   49  170.08  174.60                    8.50\n",
      "    7  10560 331F 332D C   N    2668 2670 -172.46  -96.50   79.52    3.24  -63.30  170.26   18.65\n",
      "    7        332D 332D N   CA   2670 2671   90.66  114.20                  -40.00\n",
      "    8  10561 332D 333M C   N    2676 2678 -158.61  -63.40  107.69   16.02  -63.40  107.69   16.02\n",
      "    8        333M 333M N   CA   2678 2679    9.81  -40.50                  -40.50\n",
      "    9  10562 333M 334E C   N    2684 2686  -66.80  -63.60    4.46    0.53   54.60  145.16   18.97\n",
      "    9        334E 334E N   CA   2686 2687  -37.20  -40.30                   42.40\n",
      "   10  10584 355Q 356P C   N    2873 2875  -64.76  -58.70   24.65    1.60  -64.50  153.80   11.49\n",
      "   10        356P 356P N   CA   2875 2876   -6.61  -30.50                  147.20\n",
      "   11  10586 357G 358Y C   N    2884 2886 -118.89 -124.30    5.82    0.48  -63.50 -172.58   32.55\n",
      "   11        358Y 358Y N   CA   2886 2887  137.55  135.40                  -43.40\n",
      "   12  10587 358Y 359R C   N    2896 2898  -69.70  -72.10    8.80    0.73  -63.00  168.67   23.24\n",
      "   12        359R 359R N   CA   2898 2899  150.37  141.90                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:  2715 2718   2.171\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    1    0    1    0    1    9   48  120  151  180  238  370  449  512\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "6g97_MODLL.B99990001.pdb       157.23694\n",
      "\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2772 atoms and 1 coordinate set(s) were parsed in 0.04s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 6g9a\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    37776    35130\n",
      "226 (of 2915 total) atoms selected for optimization\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:       90.9335\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     226\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    35130    1801\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      347\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         148.6165\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     234       0      0   0.005   0.005      1.7635       1.000\n",
      " 2 Bond angle potential               :     323       0      0   2.168   2.168      27.650       1.000\n",
      " 3 Stereochemical cosine torsion poten:     150       0      2  48.862  48.862      53.047       1.000\n",
      " 4 Stereochemical improper torsion pot:      94       0      0   1.327   1.327      3.7286       1.000\n",
      " 5 Soft-sphere overlap restraints     :     347       0      0   0.005   0.005     0.85047       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     266       0      0   0.179   0.179      4.0743       1.000\n",
      "10 Distance restraints 2 (N-O)        :     262       0      0   0.228   0.228      4.6348       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      33       0      0   3.742   3.742      4.7902       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      22       0      0  49.148  49.148     0.68452       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      18       0      0  59.686  59.686      8.2762       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       8       0      0  84.032  84.032      3.4883       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       2       0      0 122.886 122.886      1.3217       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     196       0      0   0.331   0.331     0.47963       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      35      18      5  24.962 123.845      30.837       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     158       0      0   0.539   0.539      2.9905       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 6g9a_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    5121.1631\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9  -67.98  -68.20   12.92    1.03  -62.50  173.36   28.15\n",
      "    1          2A   2A N   CA      9   10  132.38  145.30                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14  -63.97  -68.20   14.45    1.02  -62.50  172.39   28.18\n",
      "    2          3A   3A N   CA     14   15  131.48  145.30                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19 -154.79 -134.00   25.51    0.64  -62.50 -177.60   34.81\n",
      "    3          4A   4A N   CA     19   20  161.77  147.00                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24  -76.37  -68.20   24.57    1.70  -62.50  151.27   25.49\n",
      "    4          5A   5A N   CA     24   25  168.47  145.30                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29  -71.95  -68.20    5.31    0.52  -62.50  177.82   29.61\n",
      "    5          6A   6A N   CA     29   30  141.54  145.30                  -40.90\n",
      "    6  10236   7A   8G C   N      37   39  158.82 -167.20   44.29    0.89  -80.20  124.38    7.24\n",
      "    6          8G   8G N   CA     39   40 -157.00  174.60                  174.10\n",
      "    7  10389 160T 161C C   N    1283 1285   74.05   57.40   24.60    1.30 -117.90 -151.62    9.00\n",
      "    7        161C 161C N   CA   1285 1286   17.89   36.00                  141.10\n",
      "    8  10558 329F 330K C   N    2648 2650 -100.37 -118.00   29.15    1.50  -62.90  161.30   23.15\n",
      "    8        330K 330K N   CA   2650 2651  162.31  139.10                  -40.80\n",
      "    9  10559 330K 331F C   N    2657 2659  -69.08  -71.40    2.42    0.14  -63.20  175.79   24.80\n",
      "    9        331F 331F N   CA   2659 2660  140.01  140.70                  -44.30\n",
      "   10  10560 331F 332D C   N    2668 2670  -93.51  -70.90   28.54    1.32  -63.30  155.26   21.09\n",
      "   10        332D 332D N   CA   2670 2671  167.71  150.30                  -40.00\n",
      "   11  10561 332D 333M C   N    2676 2678  -72.52  -73.00    5.65    0.39  -63.40  171.12   26.26\n",
      "   11        333M 333M N   CA   2678 2679  148.63  143.00                  -40.50\n",
      "   12  10562 333M 334E C   N    2684 2686  -72.92  -69.30    6.21    0.57  -63.60  177.99   23.23\n",
      "   12        334E 334E N   CA   2686 2687  137.45  142.50                  -40.30\n",
      "   13  10563 334E 335L C   N    2693 2695 -141.78 -108.50   47.30    2.13  -63.50  171.58   28.66\n",
      "   13        335L 335L N   CA   2695 2696  166.12  132.50                  -41.20\n",
      "   14  10564 335L 336D C   N    2701 2703  -70.64  -70.90   16.93    0.85  -63.30  152.95   19.33\n",
      "   14        336D 336D N   CA   2703 2704  167.23  150.30                  -40.00\n",
      "   15  10565 336D 337D C   N    2709 2711  -66.93  -70.90    4.35    0.36  -63.30  167.98   20.95\n",
      "   15        337D 337D N   CA   2711 2712  152.06  150.30                  -40.00\n",
      "   16  10585 356P 357G C   N    2880 2882   82.34   78.70   39.63    0.98  -80.20  163.72   10.26\n",
      "   16        357G 357G N   CA   2882 2883  154.44 -166.10                  174.10\n",
      "   17  10586 357G 358Y C   N    2884 2886 -120.45 -124.30    7.56    0.56  -63.50 -176.26   32.04\n",
      "   17        358Y 358Y N   CA   2886 2887  141.90  135.40                  -43.40\n",
      "   18  10587 358Y 359R C   N    2896 2898 -100.45 -125.20   25.65    0.86  -63.00  178.93   21.89\n",
      "   18        359R 359R N   CA   2898 2899  133.87  140.60                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    0    3   13   46  104  142  197  225  358  418  498\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "6g9a_MODLL.B99990001.pdb       148.61653\n",
      "\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2798 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 6g9d\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    37941    35295\n",
      "195 (of 2915 total) atoms selected for optimization\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     195\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    35295    1705\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      340\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         158.0699\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     202       0      0   0.005   0.005      1.1226       1.000\n",
      " 2 Bond angle potential               :     278       0      0   1.956   1.956      19.713       1.000\n",
      " 3 Stereochemical cosine torsion poten:     128       0      7  53.084  53.084      52.782       1.000\n",
      " 4 Stereochemical improper torsion pot:      83       0      0   1.257   1.257      2.6994       1.000\n",
      " 5 Soft-sphere overlap restraints     :     340       0      0   0.003   0.003     0.25735       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     303       0      0   0.124   0.124      2.7875       1.000\n",
      "10 Distance restraints 2 (N-O)        :     277       0      0   0.277   0.277      10.396       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      29       0      0   3.720   3.720      4.0763       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      18       0      0  72.560  72.560      5.9604       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      14       0      0  75.376  75.376      2.8717       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       7       0      0  96.400  96.400      4.3664       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       2       0      0  85.031  85.031      1.2142       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     199       0      0   0.464   0.464     0.98436       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      31      14      6  27.996 112.862      40.720       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     134       0      0   0.797   0.797      8.1179       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 6g9d_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    4308.4458\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9  -79.78  -68.20   21.14    1.35  -62.50  157.06   26.62\n",
      "    1          2A   2A N   CA      9   10  162.99  145.30                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14  -61.82  -68.20   14.61    0.97  -62.50  173.06   28.39\n",
      "    2          3A   3A N   CA     14   15  132.16  145.30                  -40.90\n",
      "    3  10233   4A   5A C   N      22   24  -74.36  -68.20   11.77    0.76  -62.50  164.20   27.51\n",
      "    3          5A   5A N   CA     24   25  155.33  145.30                  -40.90\n",
      "    4  10234   5A   6A C   N      27   29 -127.88 -134.00   12.94    0.80  -62.50  173.49   31.93\n",
      "    4          6A   6A N   CA     29   30  158.40  147.00                  -40.90\n",
      "    5  10235   6A   7A C   N      32   34  -64.30  -68.20    3.92    0.34  -62.50  173.45   28.51\n",
      "    5          7A   7A N   CA     34   35  145.67  145.30                  -40.90\n",
      "    6  10236   7A   8G C   N      37   39   83.98   78.70    5.89    0.33  -80.20  165.71   11.63\n",
      "    6          8G   8G N   CA     39   40 -163.50 -166.10                  174.10\n",
      "    7  10389 160T 161C C   N    1283 1285   65.09   57.40    7.78    0.99 -117.90 -154.76    8.66\n",
      "    7        161C 161C N   CA   1285 1286   37.23   36.00                  141.10\n",
      "    8  10558 329F 330K C   N    2648 2650 -124.14 -118.00   40.33    2.03  -62.90  152.85   17.61\n",
      "    8        330K 330K N   CA   2650 2651   99.25  139.10                  -40.80\n",
      "    9  10559 330K 331F C   N    2657 2659 -136.35 -124.20   20.50    0.61  -63.20  172.19   28.64\n",
      "    9        331F 331F N   CA   2659 2660  159.82  143.30                  -44.30\n",
      "   10  10561 332D 333M C   N    2676 2678 -168.11 -125.60   71.77    2.37  -63.40  160.15   31.59\n",
      "   10        333M 333M N   CA   2678 2679 -161.67  140.50                  -40.50\n",
      "   11  10564 335L 336D C   N    2701 2703  -87.51  -63.30   71.26    7.68  -63.30   71.26    7.68\n",
      "   11        336D 336D N   CA   2703 2704   27.03  -40.00                  -40.00\n",
      "   12  10585 356P 357G C   N    2880 2882  123.40   78.70   48.85    1.83  -80.20  156.39   10.42\n",
      "   12        357G 357G N   CA   2882 2883  174.21 -166.10                  174.10\n",
      "   13  10586 357G 358Y C   N    2884 2886  -90.04  -98.40   12.07    0.94  -63.50 -178.56   30.16\n",
      "   13        358Y 358Y N   CA   2886 2887  137.11  128.40                  -43.40\n",
      "   14  10587 358Y 359R C   N    2896 2898  -65.77  -72.10    6.46    0.52  -63.00  175.75   23.93\n",
      "   14        359R 359R N   CA   2898 2899  143.18  141.90                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    0    0    8   45  124  137  179  249  359  405  501\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "6g9d_MODLL.B99990001.pdb       158.06995\n",
      "\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2793 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 6g9h\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    37896    35250\n",
      "205 (of 2915 total) atoms selected for optimization\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:      -95.0233\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     205\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    35250    1710\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      388\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         142.8165\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     211       0      0   0.004   0.004     0.84413       1.000\n",
      " 2 Bond angle potential               :     290       0      1   1.974   1.974      20.998       1.000\n",
      " 3 Stereochemical cosine torsion poten:     125       0      5  52.082  52.082      49.642       1.000\n",
      " 4 Stereochemical improper torsion pot:      82       0      0   1.070   1.070      2.2210       1.000\n",
      " 5 Soft-sphere overlap restraints     :     388       0      0   0.002   0.002     0.14471       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     301       0      0   0.155   0.155      3.8863       1.000\n",
      "10 Distance restraints 2 (N-O)        :     290       0      0   0.316   0.316      14.540       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      31       0      0   4.728   4.728      7.4916       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      19       0      0  73.585  73.585      2.8085       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      15       0      0  62.662  62.662      2.9593       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       8       0      0 101.040 101.040      5.0883       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       2       0      0  89.362  89.362      1.0606       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     191       0      0   0.344   0.344     0.45167       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      33      12      8  25.865 117.582      29.094       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     112       0      0   0.464   0.464      1.5859       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 6g9h_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    4304.3623\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9  -82.23  -68.20   16.39    1.10  -62.50  166.51   28.30\n",
      "    1          2A   2A N   CA      9   10  153.77  145.30                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14  -79.42  -68.20   19.91    1.27  -62.50  158.27   26.80\n",
      "    2          3A   3A N   CA     14   15  161.74  145.30                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19  -73.40  -68.20    7.61    0.48  -62.50  168.59   28.18\n",
      "    3          4A   4A N   CA     19   20  150.86  145.30                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24  -98.21  -68.20   39.85    2.53  -62.50  151.84   26.77\n",
      "    4          5A   5A N   CA     24   25  171.52  145.30                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29 -114.66 -134.00   20.87    0.48  -62.50 -172.64   33.47\n",
      "    5          6A   6A N   CA     29   30  139.15  147.00                  -40.90\n",
      "    6  10237   8G   9A C   N      41   43 -147.67 -134.00   17.84    0.49  -62.50 -178.17   34.35\n",
      "    6          9A   9A N   CA     43   44  158.46  147.00                  -40.90\n",
      "    7  10389 160T 161C C   N    1283 1285   75.24   57.40   23.04    1.54 -117.90 -154.65    8.85\n",
      "    7        161C 161C N   CA   1285 1286   21.42   36.00                  141.10\n",
      "    8  10560 331F 332D C   N    2668 2670  -59.35  -70.90   16.00    1.30  -63.30  158.68   19.35\n",
      "    8        332D 332D N   CA   2670 2671  161.37  150.30                  -40.00\n",
      "    9  10562 333M 334E C   N    2684 2686 -163.80 -117.80   48.63    1.47  -63.60 -165.13   31.50\n",
      "    9        334E 334E N   CA   2686 2687  152.57  136.80                  -40.30\n",
      "   10  10585 356P 357G C   N    2880 2882   91.99   82.20   10.06    0.58  -80.20 -119.48    7.79\n",
      "   10        357G 357G N   CA   2882 2883    6.18    8.50                  174.10\n",
      "   11  10586 357G 358Y C   N    2884 2886 -106.81  -98.40   11.41    0.85  -63.50  169.70   24.86\n",
      "   11        358Y 358Y N   CA   2886 2887  120.68  128.40                  -43.40\n",
      "   12  10587 358Y 359R C   N    2896 2898  -75.51  -72.10    8.25    0.70  -63.00  175.93   22.95\n",
      "   12        359R 359R N   CA   2898 2899  134.39  141.90                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    0    2   16   37  122  157  198  238  375  454  472\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "6g9h_MODLL.B99990001.pdb       142.81651\n",
      "\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2815 atoms and 1 coordinate set(s) were parsed in 0.03s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 6g9j\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    37986    35340\n",
      "180 (of 2915 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2774 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:      -91.8430\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     180\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    35340    1717\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      429\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         118.8455\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     187       0      0   0.005   0.005      1.2471       1.000\n",
      " 2 Bond angle potential               :     258       0      0   1.751   1.751      14.777       1.000\n",
      " 3 Stereochemical cosine torsion poten:     125       0      3  46.458  46.458      44.001       1.000\n",
      " 4 Stereochemical improper torsion pot:      85       0      0   0.954   0.954      2.0480       1.000\n",
      " 5 Soft-sphere overlap restraints     :     429       0      0   0.003   0.003     0.45547       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     348       0      0   0.249   0.249      9.7700       1.000\n",
      "10 Distance restraints 2 (N-O)        :     305       0      0   0.384   0.384      10.986       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      27       0      0   4.626   4.626      5.7760       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      16       0      0  44.258  44.258      2.6194       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      11       0      1  52.618  52.618      5.5848       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       4       0      0  85.571  85.571      1.6194       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       2       0      0  59.762  59.762      1.8402       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     160       0      0   0.277   0.277     0.26508       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      29      10      3  27.005 107.059      15.860       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     160       0      0   0.487   0.487      1.9964       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 6g9j_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    3484.8032\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9  -74.86  -68.20    8.82    0.56  -62.50  168.47   28.23\n",
      "    1          2A   2A N   CA      9   10  151.09  145.30                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14  -71.35  -68.20    6.89    0.45  -62.50  167.90   27.96\n",
      "    2          3A   3A N   CA     14   15  151.43  145.30                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19  -72.67  -68.20   14.08    0.98  -62.50  160.78   26.86\n",
      "    3          4A   4A N   CA     19   20  158.65  145.30                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24  -61.19  -68.20    7.11    0.56  -62.50  175.01   28.62\n",
      "    4          5A   5A N   CA     24   25  144.10  145.30                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29 -131.40 -134.00   12.47    0.75  -62.50  174.12   32.23\n",
      "    5          6A   6A N   CA     29   30  159.20  147.00                  -40.90\n",
      "    6  10235   6A   7A C   N      32   34 -131.35 -134.00    4.58    0.28  -62.50 -178.10   33.50\n",
      "    6          7A   7A N   CA     34   35  150.74  147.00                  -40.90\n",
      "    7  10389 160T 161C C   N    1283 1285   75.11   57.40   43.16    1.37 -117.90 -139.19    9.67\n",
      "    7        161C 161C N   CA   1285 1286   -3.36   36.00                  141.10\n",
      "    8  10585 356P 357G C   N    2880 2882  108.03   78.70   74.74    1.14  -80.20  178.60   12.97\n",
      "    8        357G 357G N   CA   2882 2883  125.16 -166.10                  174.10\n",
      "    9  10586 357G 358Y C   N    2884 2886 -114.46 -124.30   13.55    1.10  -63.50  179.28   31.04\n",
      "    9        358Y 358Y N   CA   2886 2887  144.71  135.40                  -43.40\n",
      "   10  10587 358Y 359R C   N    2896 2898  -57.38  -72.10   17.51    1.50  -63.00  167.61   22.27\n",
      "   10        359R 359R N   CA   2898 2899  151.38  141.90                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:   650 1321   2.264\n",
      "  serious non-bonded atom clash:   651 1321   1.938\n",
      "  serious non-bonded atom clash:  2683 2723   1.720\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     2    0    1    1    0    7   14   59  113  143  193  250  357  443  472\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "6g9j_MODLL.B99990001.pdb       118.84552\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 6g9m\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    37809    35163\n",
      "224 (of 2915 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2802 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     224\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    35163    1947\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      394\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         134.3591\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     232       0      0   0.003   0.003     0.44241       1.000\n",
      " 2 Bond angle potential               :     319       0      0   1.653   1.653      16.128       1.000\n",
      " 3 Stereochemical cosine torsion poten:     147       0      4  49.948  49.948      50.380       1.000\n",
      " 4 Stereochemical improper torsion pot:      93       0      0   1.049   1.049      2.3832       1.000\n",
      " 5 Soft-sphere overlap restraints     :     394       0      0   0.001   0.001     0.71331E-01   1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     327       0      0   0.172   0.172      5.4492       1.000\n",
      "10 Distance restraints 2 (N-O)        :     311       0      0   0.191   0.191      4.2627       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      33       0      0   3.834   3.834      4.9761       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      21       0      0  64.576  64.576      4.3617       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      17       0      0  62.011  62.011      4.4104       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:      10       0      0  85.616  85.616      6.3504       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       2       0      0 123.520 123.520     0.78074       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     203       0      0   0.403   0.403     0.63956       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      35      16      3  22.917 121.422      27.131       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     197       0      0   0.558   0.558      6.5917       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 6g9m_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    4913.8247\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9  -61.65  -68.20    7.05    0.50  -62.50  176.42   28.87\n",
      "    1          2A   2A N   CA      9   10  142.69  145.30                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14  -63.95  -68.20    4.79    0.46  -62.50  171.59   28.19\n",
      "    2          3A   3A N   CA     14   15  147.52  145.30                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19  -72.76  -68.20   10.68    0.71  -62.50  164.46   27.47\n",
      "    3          4A   4A N   CA     19   20  154.96  145.30                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24 -123.18 -134.00   25.01    1.54  -62.50  161.40   29.70\n",
      "    4          5A   5A N   CA     24   25  169.54  147.00                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29  -60.58  -68.20    9.45    0.61  -62.50  179.40   29.31\n",
      "    5          6A   6A N   CA     29   30  139.72  145.30                  -40.90\n",
      "    6  10235   6A   7A C   N      32   34  -59.86  -68.20    8.50    0.66  -62.50  175.48   28.63\n",
      "    6          7A   7A N   CA     34   35  143.64  145.30                  -40.90\n",
      "    7  10236   7A   8G C   N      37   39   90.33   78.70   21.34    0.34   82.20  167.70    8.55\n",
      "    7          8G   8G N   CA     39   40  176.01 -166.10                    8.50\n",
      "    8  10237   8G   9A C   N      41   43  -70.03  -68.20    6.99    0.50  -62.50  167.22   27.78\n",
      "    8          9A   9A N   CA     43   44  152.05  145.30                  -40.90\n",
      "    9  10238   9A  10G C   N      46   48 -132.90 -167.20   45.64    0.94   82.20 -161.29   15.32\n",
      "    9         10G  10G N   CA     48   49  144.48  174.60                    8.50\n",
      "   10  10389 160T 161C C   N    1283 1285   69.79   57.40   14.83    1.14 -117.90 -153.79    8.81\n",
      "   10        161C 161C N   CA   1285 1286   27.84   36.00                  141.10\n",
      "   11  10558 329F 330K C   N    2648 2650  -89.99  -70.20   27.97    1.67  -62.90  161.34   22.52\n",
      "   11        330K 330K N   CA   2650 2651  160.15  140.40                  -40.80\n",
      "   12  10559 330K 331F C   N    2657 2659  -73.33  -71.40   32.09    2.64  -63.20  153.31   20.61\n",
      "   12        331F 331F N   CA   2659 2660  108.67  140.70                  -44.30\n",
      "   13  10560 331F 332D C   N    2668 2670  -82.52  -70.90   36.26    1.29  -63.30  136.71   18.09\n",
      "   13        332D 332D N   CA   2670 2671 -175.35  150.30                  -40.00\n",
      "   14  10561 332D 333M C   N    2676 2678  -62.31  -73.00   15.61    1.23  -63.40  165.13   24.57\n",
      "   14        333M 333M N   CA   2678 2679  154.37  143.00                  -40.50\n",
      "   15  10586 357G 358Y C   N    2884 2886 -123.81 -124.30   21.90    1.18  -63.50  170.33   30.05\n",
      "   15        358Y 358Y N   CA   2886 2887  157.30  135.40                  -43.40\n",
      "   16  10587 358Y 359R C   N    2896 2898  -73.62  -72.10    2.11    0.14  -63.00  175.87   24.48\n",
      "   16        359R 359R N   CA   2898 2899  143.35  141.90                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:   431 2739   1.500\n",
      "  serious non-bonded atom clash:   431 2740   2.012\n",
      "  serious non-bonded atom clash:   432 2738   2.010\n",
      "  serious non-bonded atom clash:   432 2739   0.659\n",
      "  serious non-bonded atom clash:   432 2740   1.816\n",
      "  serious non-bonded atom clash:   433 2739   2.195\n",
      "  serious non-bonded atom clash:   433 2740   1.765\n",
      "  serious non-bonded atom clash:   434 2740   1.180\n",
      "  serious non-bonded atom clash:   440 2740   2.045\n",
      "  serious non-bonded atom clash:  2715 2718   2.160\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     8    2    0    0    2    1   14   52  106  155  191  230  359  444  480\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "6g9m_MODLL.B99990001.pdb       134.35909\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 6g9n\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    38193    35547\n",
      "200 (of 2915 total) atoms selected for optimization\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:      -92.2727\n",
      "iupac_m_484W> Dihedral still outside +-90:      -94.4236\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     200\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    35547    1841\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      337\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         167.2720\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     207       0      0   0.005   0.005      1.4282       1.000\n",
      " 2 Bond angle potential               :     284       0      0   2.033   2.033      21.738       1.000\n",
      " 3 Stereochemical cosine torsion poten:     130       0      7  53.865  53.865      54.048       1.000\n",
      " 4 Stereochemical improper torsion pot:      84       0      0   1.085   1.085      2.5000       1.000\n",
      " 5 Soft-sphere overlap restraints     :     337       0      0   0.001   0.001     0.85430E-01   1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     323       0      0   0.160   0.160      4.5300       1.000\n",
      "10 Distance restraints 2 (N-O)        :     302       0      2   0.481   0.481      16.616       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      30       0      0   4.090   4.090      5.0976       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      18       0      0  75.559  75.559      2.8383       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      14       0      0  73.574  73.574      4.6083       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       8       0      0  75.242  75.242      5.9578       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       2       0      0  80.056  80.056      2.0201       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     234       0      0   0.542   0.542      1.3383       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      32      14      4  28.648 118.053      36.971       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     173       0      0   0.662   0.662      7.4935       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 6g9n_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    4736.9771\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9  -72.49  -68.20    4.51    0.42  -62.50  175.47   29.26\n",
      "    1          2A   2A N   CA      9   10  143.92  145.30                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14 -106.85 -134.00   27.73    0.72  -62.50 -176.77   32.37\n",
      "    2          3A   3A N   CA     14   15  141.33  147.00                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19  -85.34  -68.20   21.98    1.41  -62.50  161.67   27.67\n",
      "    3          4A   4A N   CA     19   20  159.05  145.30                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24  -64.15  -68.20    5.00    0.32  -62.50  176.74   29.04\n",
      "    4          5A   5A N   CA     24   25  142.37  145.30                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29 -125.03 -134.00   12.16    0.70  -62.50  175.42   32.09\n",
      "    5          6A   6A N   CA     29   30  155.21  147.00                  -40.90\n",
      "    6  10235   6A   7A C   N      32   34  -61.73  -68.20   16.82    1.56  -62.50  158.28   25.90\n",
      "    6          7A   7A N   CA     34   35  160.82  145.30                  -40.90\n",
      "    7  10236   7A   8G C   N      37   39 -134.02 -167.20   37.62    0.59   82.20 -153.41   15.80\n",
      "    7          8G   8G N   CA     39   40  156.86  174.60                    8.50\n",
      "    8  10237   8G   9A C   N      41   43  -65.50  -68.20    3.34    0.22  -62.50  175.78   28.96\n",
      "    8          9A   9A N   CA     43   44  143.34  145.30                  -40.90\n",
      "    9  10238   9A  10G C   N      46   48 -163.19 -167.20   12.46    0.42   82.20 -167.79   14.21\n",
      "    9         10G  10G N   CA     48   49  162.80  174.60                    8.50\n",
      "   10  10389 160T 161C C   N    1283 1285  162.37   57.40  111.62   10.92 -117.90  163.78    7.26\n",
      "   10        161C 161C N   CA   1285 1286   -1.96   36.00                  141.10\n",
      "   11  10559 330K 331F C   N    2657 2659  -63.58  -71.40   18.97    1.68  -63.20  157.71   21.92\n",
      "   11        331F 331F N   CA   2659 2660  157.99  140.70                  -44.30\n",
      "   12  10560 331F 332D C   N    2668 2670   49.19   54.50   44.77    2.88  -63.30  168.43   27.71\n",
      "   12        332D 332D N   CA   2670 2671   85.35   40.90                  -40.00\n",
      "   13  10586 357G 358Y C   N    2884 2886 -128.51 -124.30   16.86    0.73  -63.50  177.23   31.37\n",
      "   13        358Y 358Y N   CA   2886 2887  151.72  135.40                  -43.40\n",
      "   14  10587 358Y 359R C   N    2896 2898  -91.53  -72.10   21.30    1.79  -63.00  176.59   22.07\n",
      "   14        359R 359R N   CA   2898 2899  133.17  141.90                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:  2707 2710   2.290\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    1    0    0    1    8   51  136  140  189  231  368  462  489\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2786 atoms and 1 coordinate set(s) were parsed in 0.03s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "6g9n_MODLL.B99990001.pdb       167.27202\n",
      "\n",
      "Modelando proteína 6gdm\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    37383    34737\n",
      "294 (of 2915 total) atoms selected for optimization\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     294\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    34737    2683\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      640\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         243.3118\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     302       0      0   0.005   0.005      1.8577       1.000\n",
      " 2 Bond angle potential               :     415       0      1   1.901   1.901      28.484       1.000\n",
      " 3 Stereochemical cosine torsion poten:     177       0      8  52.516  52.516      77.241       1.000\n",
      " 4 Stereochemical improper torsion pot:     124       0      1   1.380   1.380      7.2826       1.000\n",
      " 5 Soft-sphere overlap restraints     :     640       0      0   0.005   0.005      2.2224       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     531       0      0   0.273   0.273      17.819       1.000\n",
      "10 Distance restraints 2 (N-O)        :     495       0      1   0.489   0.489      25.541       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      43       1      1  27.728  27.728      13.049       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      30       0      0  63.713  63.713      4.1951       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      23       0      0  64.886  64.886      9.2014       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:      10       0      0  99.126  99.126      4.8515       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       4       0      0  54.343  54.343      2.1283       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     309       0      0   0.417   0.417      1.2856       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      47      17      8  28.790 106.021      41.474       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     173       0      0   0.982   0.982      6.6793       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 6gdm_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    6204.8906\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 13                           : Mainchain Omega dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10840 253N 253N CA  C    2016 2021   -0.68 -179.18  178.51   41.49 -179.18  178.51   41.49\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9  -62.99  -68.20    9.05    0.57  -62.50  178.80   29.28\n",
      "    1          2A   2A N   CA      9   10  137.90  145.30                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14  -79.09  -68.20   15.97    1.00  -62.50  162.97   27.55\n",
      "    2          3A   3A N   CA     14   15  156.98  145.30                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19 -150.17 -134.00   27.30    0.98  -62.50  173.84   33.17\n",
      "    3          4A   4A N   CA     19   20  168.99  147.00                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24 -136.42 -134.00    6.80    0.31  -62.50 -178.51   33.70\n",
      "    4          5A   5A N   CA     24   25  153.35  147.00                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29 -115.25 -134.00   30.30    1.05  -62.50  172.36   26.22\n",
      "    5          6A   6A N   CA     29   30  123.19  147.00                  -40.90\n",
      "    6  10235   6A   7A C   N      32   34  176.44 -134.00   52.20    1.23  -62.50 -162.76   38.66\n",
      "    6          7A   7A N   CA     34   35  163.39  147.00                  -40.90\n",
      "    7  10236   7A   8G C   N      37   39  169.66 -167.20   25.11    0.74  -80.20  110.53    7.62\n",
      "    7          8G   8G N   CA     39   40  164.84  174.60                  174.10\n",
      "    8  10293  64Y  65C C   N     478  480  -62.26  -63.00    1.64    0.17 -117.90 -175.09    7.96\n",
      "    8         65C  65C N   CA    480  481  -42.56  -41.10                  141.10\n",
      "    9  10294  65C  66Q C   N     484  486  -61.67  -63.80    3.97    0.51   55.10  143.82   18.92\n",
      "    9         66Q  66Q N   CA    486  487  -43.66  -40.30                   40.30\n",
      "   10  10389 160T 161C C   N    1283 1285   77.51   57.40   44.79    1.46 -117.90 -140.56    9.62\n",
      "   10        161C 161C N   CA   1285 1286   -4.02   36.00                  141.10\n",
      "   11  10481 252L 253N C   N    2013 2015 -107.20 -119.90   32.89    1.17  -63.20  154.17   17.23\n",
      "   11        253N 253N N   CA   2015 2016  106.66  137.00                  -41.10\n",
      "   12  10560 331F 332D C   N    2668 2670  -71.90  -70.90   18.57    0.97  -63.30  171.97   20.72\n",
      "   12        332D 332D N   CA   2670 2671  131.75  150.30                  -40.00\n",
      "   13  10561 332D 333M C   N    2676 2678 -127.77 -125.60   12.58    0.55  -63.40  178.60   31.78\n",
      "   13        333M 333M N   CA   2678 2679  152.89  140.50                  -40.50\n",
      "   14  10562 333M 334E C   N    2684 2686 -160.90 -117.80  104.18    4.22  -63.60  131.22   22.37\n",
      "   14        334E 334E N   CA   2686 2687 -128.35  136.80                  -40.30\n",
      "   15  10563 334E 335L C   N    2693 2695  -59.21  -70.70   11.96    0.97  -63.50  179.54   25.18\n",
      "   15        335L 335L N   CA   2695 2696  138.28  141.60                  -41.20\n",
      "   16  10586 357G 358Y C   N    2884 2886  -71.08  -98.40   27.36    1.05  -63.50  173.37   27.12\n",
      "   16        358Y 358Y N   CA   2886 2887  129.80  128.40                  -43.40\n",
      "   17  10587 358Y 359R C   N    2896 2898  -74.17  -72.10    2.07    0.16  -63.00  177.45   24.73\n",
      "   17        359R 359R N   CA   2898 2899  141.81  141.90                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:   650 1319   2.235\n",
      "  serious non-bonded atom clash:   651 1321   1.971\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2764 atoms and 1 coordinate set(s) were parsed in 0.05s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     1    0    1    1    3    8   21   56  130  160  202  284  365  446  477\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "6gdm_MODLL.B99990001.pdb       243.31181\n",
      "\n",
      "Modelando proteína 6gdq\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    37671    35025\n",
      "239 (of 2915 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2805 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     239\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    35025    1923\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      427\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         171.0348\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     247       0      0   0.004   0.004      1.0804       1.000\n",
      " 2 Bond angle potential               :     339       0      0   1.900   1.900      23.206       1.000\n",
      " 3 Stereochemical cosine torsion poten:     155       0      6  51.815  51.815      58.600       1.000\n",
      " 4 Stereochemical improper torsion pot:      97       0      0   1.158   1.158      2.8047       1.000\n",
      " 5 Soft-sphere overlap restraints     :     427       0      0   0.002   0.002     0.21919       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     282       0      0   0.200   0.200      6.6799       1.000\n",
      "10 Distance restraints 2 (N-O)        :     295       0      0   0.321   0.321      12.057       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      35       0      1   4.417   4.417      7.2478       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      23       0      0  67.708  67.708      3.1477       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      19       0      0  56.433  56.433      6.9835       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       9       0      0  92.953  92.953      5.1717       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       3       0      0 130.141 130.141      2.7318       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     228       0      0   0.541   0.541      1.4708       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      37      19      4  23.652 130.883      28.172       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     154       0      0   0.822   0.822      11.461       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 6gdq_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    5900.4961\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9  -72.72  -68.20    6.82    0.43  -62.50  169.01   28.21\n",
      "    1          2A   2A N   CA      9   10  150.40  145.30                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14 -124.10 -134.00   10.20    0.26  -62.50 -174.90   33.62\n",
      "    2          3A   3A N   CA     14   15  144.56  147.00                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19  -70.88  -68.20   14.79    1.10  -62.50  159.47   26.55\n",
      "    3          4A   4A N   CA     19   20  159.85  145.30                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24 -123.98 -134.00   23.86    1.47  -62.50  162.53   29.93\n",
      "    4          5A   5A N   CA     24   25  168.65  147.00                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29  -61.48  -68.20   14.52    1.38  -62.50  160.93   26.32\n",
      "    5          6A   6A N   CA     29   30  158.18  145.30                  -40.90\n",
      "    6  10235   6A   7A C   N      32   34  -57.65  -68.20   13.26    1.31  -62.50  165.84   26.94\n",
      "    6          7A   7A N   CA     34   35  153.34  145.30                  -40.90\n",
      "    7  10237   8G   9A C   N      41   43  -69.38  -68.20   17.86    1.40  -62.50  156.14   25.93\n",
      "    7          9A   9A N   CA     43   44  163.12  145.30                  -40.90\n",
      "    8  10389 160T 161C C   N    1283 1285   65.28   57.40    8.39    0.82 -117.90 -152.82    8.78\n",
      "    8        161C 161C N   CA   1285 1286   33.12   36.00                  141.10\n",
      "    9  10559 330K 331F C   N    2657 2659 -174.26 -124.20   51.61    1.65  -63.20 -165.36   33.83\n",
      "    9        331F 331F N   CA   2659 2660  155.86  143.30                  -44.30\n",
      "   10  10560 331F 332D C   N    2668 2670  -89.06  -96.50   24.07    1.01  -63.30  133.81   15.13\n",
      "   10        332D 332D N   CA   2670 2671   91.31  114.20                  -40.00\n",
      "   11  10561 332D 333M C   N    2676 2678 -139.48 -125.60   18.48    0.55  -63.40 -176.68   33.36\n",
      "   11        333M 333M N   CA   2678 2679  152.71  140.50                  -40.50\n",
      "   12  10562 333M 334E C   N    2684 2686 -156.72 -117.80   81.72    3.17  -63.60  144.92   24.24\n",
      "   12        334E 334E N   CA   2686 2687 -151.34  136.80                  -40.30\n",
      "   13  10563 334E 335L C   N    2693 2695  -73.31  -70.70   18.15    1.28  -63.50  159.54   22.73\n",
      "   13        335L 335L N   CA   2695 2696  159.56  141.60                  -41.20\n",
      "   14  10565 336D 337D C   N    2709 2711 -100.46  -96.50   20.73    0.87  -63.30  178.46   20.06\n",
      "   14        337D 337D N   CA   2711 2712  134.55  114.20                  -40.00\n",
      "   15  10566 337D 338L C   N    2717 2719 -106.59 -108.50   18.05    0.95  -63.50  161.60   20.41\n",
      "   15        338L 338L N   CA   2719 2720  114.55  132.50                  -41.20\n",
      "   16  10567 338L 339P C   N    2725 2727  -82.05  -64.50   20.73    1.54  -58.70  172.86   16.21\n",
      "   16        339P 339P N   CA   2727 2728  158.22  147.20                  -30.50\n",
      "   17  10585 356P 357G C   N    2880 2882   77.65   82.20   17.49    0.62  -80.20 -143.13    7.20\n",
      "   17        357G 357G N   CA   2882 2883   25.39    8.50                  174.10\n",
      "   18  10586 357G 358Y C   N    2884 2886  -86.75  -98.40   11.95    0.50  -63.50  176.04   26.77\n",
      "   18        358Y 358Y N   CA   2886 2887  131.10  128.40                  -43.40\n",
      "   19  10587 358Y 359R C   N    2896 2898 -132.28 -125.20   13.43    0.47  -63.00 -179.30   29.17\n",
      "   19        359R 359R N   CA   2898 2899  152.01  140.60                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    0    2   13   41  118  155  199  246  326  443  511\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "6gdq_MODLL.B99990001.pdb       171.03476\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 6ge0\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    38039    35393\n",
      "243 (of 2915 total) atoms selected for optimization\n",
      "conmin__610W> Precision problem; starting from scratch.\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2825 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     243\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    35393    2502\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      533\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         160.0134\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     253       0      0   0.004   0.004      1.0639       1.000\n",
      " 2 Bond angle potential               :     350       0      0   2.008   2.008      26.450       1.000\n",
      " 3 Stereochemical cosine torsion poten:     169       0      3  46.489  46.489      53.154       1.000\n",
      " 4 Stereochemical improper torsion pot:     112       0      0   1.013   1.013      2.6674       1.000\n",
      " 5 Soft-sphere overlap restraints     :     533       0      0   0.002   0.002     0.24046       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     490       0      0   0.160   0.160      4.6879       1.000\n",
      "10 Distance restraints 2 (N-O)        :     419       0      0   0.370   0.370      12.690       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      36       1      0  30.090  30.090      5.4563       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      23       0      0  69.766  69.766      4.8528       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      18       0      0  32.437  32.437      3.4880       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       7       0      0 104.146 104.146      5.0934       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       1       0      0  76.785  76.785     0.80008       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     281       0      0   0.817   0.817      3.8998       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      39      16      4  25.623 111.549      18.235       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     304       0      0   0.959   0.959      17.234       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 6ge0_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    5107.5713\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9  -63.30  -68.20   10.80    1.02  -62.50  164.18   26.95\n",
      "    1          2A   2A N   CA      9   10  154.92  145.30                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14  -63.40  -68.20    7.00    0.69  -62.50  168.71   27.69\n",
      "    2          3A   3A N   CA     14   15  150.40  145.30                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19  -65.55  -68.20    6.86    0.64  -62.50  167.50   27.60\n",
      "    3          4A   4A N   CA     19   20  151.63  145.30                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24  -63.88  -68.20    7.00    0.44  -62.50  179.31   29.45\n",
      "    4          5A   5A N   CA     24   25  139.80  145.30                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29  -76.62  -68.20    8.46    0.74  -62.50  175.17   29.42\n",
      "    5          6A   6A N   CA     29   30  144.51  145.30                  -40.90\n",
      "    6  10235   6A   7A C   N      32   34  -88.29  -68.20   25.68    1.65  -62.50  159.90   27.54\n",
      "    6          7A   7A N   CA     34   35  161.30  145.30                  -40.90\n",
      "    7  10236   7A   8G C   N      37   39  -89.74  -80.20   17.29    0.43   82.20 -111.09   19.00\n",
      "    7          8G   8G N   CA     39   40 -171.47  174.10                    8.50\n",
      "    8  10237   8G   9A C   N      41   43  -96.08  -68.20   38.67    3.81  -62.50  162.91   25.27\n",
      "    8          9A   9A N   CA     43   44  118.51  145.30                  -40.90\n",
      "    9  10238   9A  10G C   N      46   48  -66.89  -62.40    4.61    0.86   82.20  157.49   11.97\n",
      "    9         10G  10G N   CA     48   49  -42.26  -41.20                    8.50\n",
      "   10  10355 126I 127C C   N    1000 1002  -61.57  -63.00    4.93    0.52 -117.90 -177.98    7.85\n",
      "   10        127C 127C N   CA   1002 1003  -45.82  -41.10                  141.10\n",
      "   11  10356 127C 128Y C   N    1006 1008  -64.19  -63.50    0.76    0.14   55.90  146.11   17.89\n",
      "   11        128Y 128Y N   CA   1008 1009  -43.73  -43.40                   39.50\n",
      "   12  10561 332D 333M C   N    2676 2678 -149.74 -125.60   91.14    4.77  -63.40  126.98   16.21\n",
      "   12        333M 333M N   CA   2678 2679   52.61  140.50                  -40.50\n",
      "   13  10562 333M 334E C   N    2684 2686  -99.57 -117.80   49.33    2.06  -63.60  136.10   16.45\n",
      "   13        334E 334E N   CA   2686 2687   90.96  136.80                  -40.30\n",
      "   14  10585 356P 357G C   N    2880 2882   98.74   78.70   31.52    0.60  -80.20  179.00   11.79\n",
      "   14        357G 357G N   CA   2882 2883  169.57 -166.10                  174.10\n",
      "   15  10586 357G 358Y C   N    2884 2886 -119.25 -124.30    7.97    0.33  -63.50 -178.59   26.20\n",
      "   15        358Y 358Y N   CA   2886 2887  129.23  135.40                  -43.40\n",
      "   16  10587 358Y 359R C   N    2896 2898 -101.81 -125.20   23.80    0.84  -63.00 -178.49   22.16\n",
      "   16        359R 359R N   CA   2898 2899  136.22  140.60                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    0    2   13   52  104  181  203  234  350  437  478\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "6ge0_MODLL.B99990001.pdb       160.01338\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 6gjb\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    38136    35490\n",
      "177 (of 2915 total) atoms selected for optimization\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     177\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    35490    1606\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      298\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         159.1999\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     182       0      0   0.005   0.005      1.0962       1.000\n",
      " 2 Bond angle potential               :     250       0      1   2.131   2.131      21.955       1.000\n",
      " 3 Stereochemical cosine torsion poten:     106       0      6  54.860  54.860      48.820       1.000\n",
      " 4 Stereochemical improper torsion pot:      73       0      0   1.157   1.157      1.9500       1.000\n",
      " 5 Soft-sphere overlap restraints     :     298       0      0   0.004   0.004     0.52292       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     295       0      0   0.177   0.177      7.8234       1.000\n",
      "10 Distance restraints 2 (N-O)        :     267       0      0   0.481   0.481      15.063       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      27       1      0  34.994  34.994      8.7736       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      16       0      0  68.188  68.188      1.5939       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      12       0      0  57.326  57.326      3.4036       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       6       0      0  67.878  67.878      3.2683       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       2       0      0   2.786   2.786     0.79535       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     225       0      0   0.734   0.734      2.5736       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      29      11      3  43.207 115.143      37.311       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     116       0      0   0.915   0.915      4.2500       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 6gjb_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    4174.7236\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 13                           : Mainchain Omega dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10747 160T 160T CA  C    1279 1283    0.71 -179.18  179.90   41.81 -179.18  179.90   41.81\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9  -74.13  -68.20   11.31    0.73  -62.50  164.58   27.56\n",
      "    1          2A   2A N   CA      9   10  154.94  145.30                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14  -75.32  -68.20   11.83    1.15  -62.50  177.22   28.44\n",
      "    2          3A   3A N   CA     14   15  135.86  145.30                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19  -81.73  -68.20   13.60    1.10  -62.50  173.46   29.41\n",
      "    3          4A   4A N   CA     19   20  146.71  145.30                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24  -87.00  -68.20   19.01    1.51  -62.50  172.74   29.57\n",
      "    4          5A   5A N   CA     24   25  148.11  145.30                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29  -82.51  -68.20   15.17    1.43  -62.50  179.95   30.51\n",
      "    5          6A   6A N   CA     29   30  140.27  145.30                  -40.90\n",
      "    6  10235   6A   7A C   N      32   34  -81.27  -68.20   13.23    1.05  -62.50  172.77   29.27\n",
      "    6          7A   7A N   CA     34   35  147.36  145.30                  -40.90\n",
      "    7  10559 330K 331F C   N    2657 2659  -67.66  -63.20    5.63    0.68 -124.20 -175.30    9.83\n",
      "    7        331F 331F N   CA   2659 2660  -40.87  -44.30                  143.30\n",
      "    8  10560 331F 332D C   N    2668 2670   98.58   54.50  130.99    6.72  -63.30  167.35   24.18\n",
      "    8        332D 332D N   CA   2670 2671  -82.46   40.90                  -40.00\n",
      "    9  10561 332D 333M C   N    2676 2678  -96.38  -73.00   23.47    1.75  -63.40  177.56   29.14\n",
      "    9        333M 333M N   CA   2678 2679  145.03  143.00                  -40.50\n",
      "   10  10586 357G 358Y C   N    2884 2886 -125.33 -124.30    2.76    0.11  -63.50 -170.96   33.11\n",
      "   10        358Y 358Y N   CA   2886 2887  137.96  135.40                  -43.40\n",
      "   11  10587 358Y 359R C   N    2896 2898 -121.80 -125.20    4.09    0.12  -63.00 -171.19   22.20\n",
      "   11        359R 359R N   CA   2898 2899  138.33  140.60                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    0    1   15   42  126  139  182  247  345  435  504\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2779 atoms and 1 coordinate set(s) were parsed in 0.03s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "----------------------------------------\n",
      "6gjb_MODLL.B99990001.pdb       159.19994\n",
      "\n",
      "Modelando proteína 6gjd\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    37843    35197\n",
      "222 (of 2915 total) atoms selected for optimization\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     222\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    35197    1817\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      415\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         231.2439\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     230       0      0   0.004   0.004     0.92799       1.000\n",
      " 2 Bond angle potential               :     317       0      1   2.205   2.205      27.398       1.000\n",
      " 3 Stereochemical cosine torsion poten:     147       0      6  50.925  50.925      53.609       1.000\n",
      " 4 Stereochemical improper torsion pot:      92       0      0   1.208   1.208      3.4618       1.000\n",
      " 5 Soft-sphere overlap restraints     :     415       0      0   0.004   0.004     0.86437       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     291       0      1   0.256   0.256      14.568       1.000\n",
      "10 Distance restraints 2 (N-O)        :     281       2      6   0.530   0.530      45.557       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      33       0      0   3.985   3.985      5.5270       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      21       0      0  83.936  83.936      6.2106       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      17       0      0  53.777  53.777      6.0460       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       8       0      0  83.388  83.388      3.9619       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       2       0      0 102.621 102.621      2.2056       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     207       0      0   0.545   0.545      1.1214       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      35      19      7  37.250 121.626      57.586       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     136       0      0   0.443   0.443      2.1993       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 6gjd_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    5328.1079\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9  -65.15  -68.20    3.08    0.25  -62.50  174.25   28.69\n",
      "    1          2A   2A N   CA      9   10  144.87  145.30                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14 -101.31  -68.20   36.49    3.51  -62.50  175.21   27.08\n",
      "    2          3A   3A N   CA     14   15  129.96  145.30                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19 -151.80 -134.00   24.53    0.72  -62.50  179.09   34.11\n",
      "    3          4A   4A N   CA     19   20  163.86  147.00                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24  -74.78  -68.20   15.37    1.02  -62.50  160.38   26.90\n",
      "    4          5A   5A N   CA     24   25  159.19  145.30                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29  -76.76  -68.20   12.00    0.76  -62.50  166.00   27.93\n",
      "    5          6A   6A N   CA     29   30  153.72  145.30                  -40.90\n",
      "    6  10235   6A   7A C   N      32   34 -121.24 -134.00   14.02    0.69  -62.50  176.36   32.04\n",
      "    6          7A   7A N   CA     34   35  152.82  147.00                  -40.90\n",
      "    7  10236   7A   8G C   N      37   39  118.71   78.70   40.86    1.83   82.20 -179.17   10.57\n",
      "    7          8G   8G N   CA     39   40 -174.39 -166.10                    8.50\n",
      "    8  10237   8G   9A C   N      41   43 -157.12 -134.00   34.56    1.11  -62.50  174.32   33.59\n",
      "    8          9A   9A N   CA     43   44  172.69  147.00                  -40.90\n",
      "    9  10238   9A  10G C   N      46   48  164.48 -167.20   48.66    1.30  -80.20  122.08    6.58\n",
      "    9         10G  10G N   CA     48   49 -145.84  174.60                  174.10\n",
      "   10  10389 160T 161C C   N    1283 1285  113.83   57.40   57.01    7.19 -117.90  160.84    6.97\n",
      "   10        161C 161C N   CA   1285 1286   44.07   36.00                  141.10\n",
      "   11  10558 329F 330K C   N    2648 2650 -137.33  -62.90   81.02   14.14  -62.90   81.02   14.14\n",
      "   11        330K 330K N   CA   2650 2651  -72.80  -40.80                  -40.80\n",
      "   12  10560 331F 332D C   N    2668 2670  -72.14  -96.50   28.70    1.19  -63.30  139.30   16.68\n",
      "   12        332D 332D N   CA   2670 2671   99.02  114.20                  -40.00\n",
      "   13  10561 332D 333M C   N    2676 2678  -75.71  -73.00   13.13    0.85  -63.40  164.12   25.47\n",
      "   13        333M 333M N   CA   2678 2679  155.84  143.00                  -40.50\n",
      "   14  10562 333M 334E C   N    2684 2686  -76.87  -69.30   11.10    0.62  -63.60  169.60   23.35\n",
      "   14        334E 334E N   CA   2686 2687  150.62  142.50                  -40.30\n",
      "   15  10563 334E 335L C   N    2693 2695  -63.18  -70.70   14.75    0.91  -63.50  170.11   23.64\n",
      "   15        335L 335L N   CA   2695 2696  128.91  141.60                  -41.20\n",
      "   16  10566 337D 338L C   N    2717 2719   42.49  -70.70  113.21   10.37 -108.50  151.43    7.35\n",
      "   16        338L 338L N   CA   2719 2720  144.09  141.60                  132.50\n",
      "   17  10584 355Q 356P C   N    2873 2875  -56.03  -58.70    7.30    0.82  -64.50  171.11   12.33\n",
      "   17        356P 356P N   CA   2875 2876  -23.70  -30.50                  147.20\n",
      "   18  10585 356P 357G C   N    2880 2882  140.83 -167.20   57.71    0.88  -80.20  141.31    8.53\n",
      "   18        357G 357G N   CA   2882 2883 -160.31  174.60                  174.10\n",
      "   19  10587 358Y 359R C   N    2896 2898  -73.21  -72.10    5.03    0.36  -63.00  172.40   23.98\n",
      "   19        359R 359R N   CA   2898 2899  146.80  141.90                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    1    3   14   46  131  169  182  251  355  447  479\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "6gjd_MODLL.B99990001.pdb       231.24388\n",
      "\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2790 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 6nbs\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    37872    35226\n",
      "187 (of 2915 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2858 atoms and 1 coordinate set(s) were parsed in 0.03s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:       92.0534\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     187\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    35226    1350\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      259\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         145.8035\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     194       0      0   0.003   0.003     0.36089       1.000\n",
      " 2 Bond angle potential               :     265       0      0   1.743   1.743      15.427       1.000\n",
      " 3 Stereochemical cosine torsion poten:     125       0      2  50.729  50.729      41.935       1.000\n",
      " 4 Stereochemical improper torsion pot:      78       0      0   0.955   0.955      2.0983       1.000\n",
      " 5 Soft-sphere overlap restraints     :     259       0      0   0.001   0.001     0.94639E-02   1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     188       0      0   0.159   0.159      2.9039       1.000\n",
      "10 Distance restraints 2 (N-O)        :     176       0      0   0.376   0.376      12.790       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      27       0      1   4.498   4.498      6.4417       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      16       0      0  81.485  81.485      4.7288       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      15       0      0  40.138  40.138      4.0175       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       9       0      0  91.506  91.506      4.9736       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       2       0      0 126.048 126.048     0.73341       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     137       0      0   0.544   0.544     0.99041       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      28      16      5  31.254 130.681      46.767       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      90       0      0   0.355   0.355      1.6272       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 6nbs_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    4833.5576\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9 -119.45 -134.00   14.60    0.53  -62.50 -179.91   32.55\n",
      "    1          2A   2A N   CA      9   10  148.26  147.00                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14 -108.08 -134.00   26.87    0.66  -62.50 -175.08   32.71\n",
      "    2          3A   3A N   CA     14   15  139.89  147.00                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19 -148.95 -134.00   28.26    1.10  -62.50  171.51   32.72\n",
      "    3          4A   4A N   CA     19   20  170.98  147.00                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24  -70.08  -68.20   14.85    1.13  -62.50  159.25   26.48\n",
      "    4          5A   5A N   CA     24   25  160.03  145.30                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29  -64.21  -68.20    4.61    0.31  -62.50  176.13   28.95\n",
      "    5          6A   6A N   CA     29   30  142.98  145.30                  -40.90\n",
      "    6  10235   6A   7A C   N      32   34  -83.44  -68.20   25.58    1.62  -62.50  154.68   26.43\n",
      "    6          7A   7A N   CA     34   35  165.85  145.30                  -40.90\n",
      "    7  10237   8G   9A C   N      41   43  -63.38  -68.20    8.88    0.57  -62.50  178.75   29.25\n",
      "    7          9A   9A N   CA     43   44  137.85  145.30                  -40.90\n",
      "    8  10558 329F 330K C   N    2648 2650  -67.25  -70.20    2.96    0.22  -62.90  178.67   23.44\n",
      "    8        330K 330K N   CA   2650 2651  140.58  140.40                  -40.80\n",
      "    9  10559 330K 331F C   N    2657 2659 -105.74 -124.20   32.42    0.98  -63.20  166.48   20.47\n",
      "    9        331F 331F N   CA   2659 2660  116.66  143.30                  -44.30\n",
      "   10  10560 331F 332D C   N    2668 2670 -129.49  -70.90   66.97    3.59  -63.30  152.40   23.08\n",
      "   10        332D 332D N   CA   2670 2671 -177.27  150.30                  -40.00\n",
      "   11  10561 332D 333M C   N    2676 2678  -61.36  -73.00   18.28    1.15  -63.40  169.42   25.45\n",
      "   11        333M 333M N   CA   2678 2679  128.90  143.00                  -40.50\n",
      "   12  10562 333M 334E C   N    2684 2686  -66.28  -69.30    4.01    0.37  -63.60  174.58   23.42\n",
      "   12        334E 334E N   CA   2686 2687  145.14  142.50                  -40.30\n",
      "   13  10563 334E 335L C   N    2693 2695  -92.07 -108.50   71.69    3.68   60.20  155.83   15.44\n",
      "   13        335L 335L N   CA   2695 2696   62.72  132.50                   29.60\n",
      "   14  10565 336D 337D C   N    2709 2711  -91.84  -63.30   61.46    6.52  -63.30   61.46    6.52\n",
      "   14        337D 337D N   CA   2711 2712   14.43  -40.00                  -40.00\n",
      "   15  10586 357G 358Y C   N    2884 2886  -65.52  -98.40   32.91    1.30  -63.50  170.44   26.94\n",
      "   15        358Y 358Y N   CA   2886 2887  127.03  128.40                  -43.40\n",
      "   16  10587 358Y 359R C   N    2896 2898  -78.58  -72.10    7.09    0.48  -63.00  174.82   24.68\n",
      "   16        359R 359R N   CA   2898 2899  144.77  141.90                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:   113  398   2.288\n",
      "  serious non-bonded atom clash:   239  245   2.290\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    2    2    2    4   17   36  115  183  187  264  380  437  488\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "6nbs_MODLL.B99990001.pdb       145.80353\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 6opg\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    38294    35648\n",
      "146 (of 2915 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2854 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     146\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    35648    1190\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      231\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         138.4243\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     151       0      0   0.007   0.007      2.0562       1.000\n",
      " 2 Bond angle potential               :     208       0      1   2.068   2.068      17.827       1.000\n",
      " 3 Stereochemical cosine torsion poten:     100       0      2  47.776  47.776      33.227       1.000\n",
      " 4 Stereochemical improper torsion pot:      68       0      0   1.257   1.257      3.2423       1.000\n",
      " 5 Soft-sphere overlap restraints     :     231       0      0   0.002   0.002     0.71130E-01   1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     205       0      3   0.483   0.483      25.129       1.000\n",
      "10 Distance restraints 2 (N-O)        :     194       0      0   0.531   0.531      19.761       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      22       0      0   3.910   3.910      3.1189       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      11       0      0  52.015  52.015    -0.46231       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       8       0      0  84.911  84.911      1.6858       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       4       0      0  98.523  98.523      3.2761       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       1       0      0   6.412   6.412     0.61541       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :      98       0      0   0.807   0.807      2.3154       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      23      12      3  35.786 126.722      23.974       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      97       0      0   0.509   0.509      2.5878       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 6opg_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    3670.9980\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9  -70.16  -68.20    9.89    0.73  -62.50  164.29   27.31\n",
      "    1          2A   2A N   CA      9   10  155.00  145.30                  -40.90\n",
      "    2  10232   3A   4A C   N      17   19  -65.26  -68.20    2.97    0.24  -62.50  174.25   28.69\n",
      "    2          4A   4A N   CA     19   20  144.87  145.30                  -40.90\n",
      "    3  10233   4A   5A C   N      22   24  -75.57  -68.20   10.69    0.67  -62.50  166.58   27.96\n",
      "    3          5A   5A N   CA     24   25  153.04  145.30                  -40.90\n",
      "    4  10234   5A   6A C   N      27   29 -119.77 -134.00   14.23    0.47  -62.50 -178.70   32.77\n",
      "    4          6A   6A N   CA     29   30  147.09  147.00                  -40.90\n",
      "    5  10235   6A   7A C   N      32   34 -119.75 -134.00   14.39    0.55  -62.50  179.48   32.47\n",
      "    5          7A   7A N   CA     34   35  149.00  147.00                  -40.90\n",
      "    6  10236   7A   8G C   N      37   39 -173.18 -167.20   14.03    0.57   82.20 -174.32   13.54\n",
      "    6          8G   8G N   CA     39   40  161.91  174.60                    8.50\n",
      "    7  10237   8G   9A C   N      41   43  -85.48  -68.20   28.71    1.81  -62.50  152.62   26.20\n",
      "    7          9A   9A N   CA     43   44  168.23  145.30                  -40.90\n",
      "    8  10238   9A  10G C   N      46   48  170.69 -167.20   23.35    0.64  -80.20  109.33    7.49\n",
      "    8         10G  10G N   CA     48   49  167.07  174.60                  174.10\n",
      "    9  10414 185T 186E C   N    1475 1477  -66.01  -69.30    8.60    0.75  -63.60  169.27   22.70\n",
      "    9        186E 186E N   CA   1477 1478  150.45  142.50                  -40.30\n",
      "   10  10417 188V 189A C   N    1503 1505   80.46   55.40  132.61   10.06  -62.50 -152.29   31.94\n",
      "   10        189A 189A N   CA   1505 1506  168.42   38.20                  -40.90\n",
      "   11  10585 356P 357G C   N    2880 2882  102.44   82.20   22.20    1.72 -167.20 -178.87    7.33\n",
      "   11        357G 357G N   CA   2882 2883   17.61    8.50                  174.60\n",
      "   12  10587 358Y 359R C   N    2896 2898 -129.96 -125.20    8.28    0.28  -63.00 -175.87   29.48\n",
      "   12        359R 359R N   CA   2898 2899  147.38  140.60                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    4    5   14   38  111  159  198  288  339  396  483\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "6opg_MODLL.B99990001.pdb       138.42433\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 6oph\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    38081    35435\n",
      "155 (of 2915 total) atoms selected for optimization\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     155\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    35435    1258\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      274\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         181.8082\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     160       0      0   0.009   0.009      4.0959       1.000\n",
      " 2 Bond angle potential               :     220       0      1   2.342   2.342      22.974       1.000\n",
      " 3 Stereochemical cosine torsion poten:     104       0      5  51.646  51.646      40.944       1.000\n",
      " 4 Stereochemical improper torsion pot:      71       0      0   1.313   1.313      2.4519       1.000\n",
      " 5 Soft-sphere overlap restraints     :     274       0      0   0.002   0.002     0.76329E-01   1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     214       0      4   0.498   0.498      29.740       1.000\n",
      "10 Distance restraints 2 (N-O)        :     214       0      1   0.499   0.499      24.353       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      23       0      0   4.868   4.868      5.1545       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      12       0      0  73.845  73.845      3.8444       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:       9       0      0  56.531  56.531      3.4138       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       5       0      0  74.276  74.276      3.0758       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       1       0      0  73.032  73.032     0.82583       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     110       0      0   0.612   0.612     0.91930       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      24      13      3  45.999 141.719      32.691       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :      91       0      0   0.902   0.902      7.2484       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 6oph_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    4180.2676\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9  -76.51  -68.20    8.33    0.69  -62.50  173.79   29.19\n",
      "    1          2A   2A N   CA      9   10  145.88  145.30                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14  -77.63  -68.20   14.78    0.93  -62.50  163.12   27.50\n",
      "    2          3A   3A N   CA     14   15  156.68  145.30                  -40.90\n",
      "    3  10233   4A   5A C   N      22   24 -102.64 -134.00   33.52    0.77  -62.50 -179.42   27.91\n",
      "    3          5A   5A N   CA     24   25  135.17  147.00                  -40.90\n",
      "    4  10234   5A   6A C   N      27   29  -76.61  -68.20    9.59    0.93  -62.50  178.95   30.04\n",
      "    4          6A   6A N   CA     29   30  140.71  145.30                  -40.90\n",
      "    5  10235   6A   7A C   N      32   34 -101.29  -68.20   41.20    4.06  -62.50  166.25   25.63\n",
      "    5          7A   7A N   CA     34   35  120.77  145.30                  -40.90\n",
      "    6  10236   7A   8G C   N      37   39   74.94   78.70    6.78    0.33   82.20  179.90    8.37\n",
      "    6          8G   8G N   CA     39   40 -171.75 -166.10                    8.50\n",
      "    7  10237   8G   9A C   N      41   43  -73.13  -68.20    6.88    0.43  -62.50  169.33   28.28\n",
      "    7          9A   9A N   CA     43   44  150.10  145.30                  -40.90\n",
      "    8  10238   9A  10G C   N      46   48  -89.91  -80.20   27.34    0.67   82.20 -118.90   18.52\n",
      "    8         10G  10G N   CA     48   49 -160.34  174.10                    8.50\n",
      "    9  10239  10G  11P C   N      50   52  -61.02  -58.70    5.74    0.34  -64.50  172.49   12.69\n",
      "    9         11P  11P N   CA     52   53  -25.25  -30.50                  147.20\n",
      "   10  10240  11P  12E C   N      57   59   94.50   54.60   96.60   10.53  -63.60 -127.35   39.22\n",
      "   10         12E  12E N   CA     59   60  130.37   42.40                  -40.30\n",
      "   11  10414 185T 186E C   N    1475 1477  -73.67  -69.30    6.90    0.63  -63.60  177.74   23.16\n",
      "   11        186E 186E N   CA   1477 1478  137.16  142.50                  -40.30\n",
      "   12  10417 188V 189A C   N    1503 1505   71.67   55.40  116.35    8.36  -62.50 -146.79   32.35\n",
      "   12        189A 189A N   CA   1505 1506  153.41   38.20                  -40.90\n",
      "   13  10587 358Y 359R C   N    2896 2898  -81.09  -72.10   16.23    1.40  -63.00  170.45   21.86\n",
      "   13        359R 359R N   CA   2898 2899  128.39  141.90                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    4    8   12   48  101  163  201  251  346  402  472\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "6oph_MODLL.B99990001.pdb       181.80815\n",
      "\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2662 atoms and 1 coordinate set(s) were parsed in 0.04s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 6opi\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    36277    33631\n",
      "379 (of 2915 total) atoms selected for optimization\n",
      "conmin__610W> Precision problem; starting from scratch.\n",
      "conmin__610W> Precision problem; starting from scratch.\n",
      "conmin__610W> Precision problem; starting from scratch.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     379\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    33631    2935\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      624\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         257.2748\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     394       0      0   0.003   0.003      1.1349       1.000\n",
      " 2 Bond angle potential               :     540       0      0   1.781   1.781      31.785       1.000\n",
      " 3 Stereochemical cosine torsion poten:     263       0      9  48.482  48.482      87.250       1.000\n",
      " 4 Stereochemical improper torsion pot:     172       0      0   0.949   0.949      3.9118       1.000\n",
      " 5 Soft-sphere overlap restraints     :     624       0      0   0.002   0.002     0.25934       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     448       0      0   0.156   0.156      5.2570       1.000\n",
      "10 Distance restraints 2 (N-O)        :     432       0      0   0.238   0.238      7.9644       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      53       0      0   3.185   3.185      5.6584       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      37       0      0  76.042  76.042      8.7885       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      29       0      0  66.804  66.804      9.1912       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:      14       0      0  93.619  93.619      8.1662       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       5       0      0  65.133  65.133      2.2728       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     295       0      0   0.523   0.523      2.2100       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      56      28      9  33.058 121.154      74.058       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     197       0      0   0.885   0.885      9.3677       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 6opi_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    8315.2246\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9  -74.44  -68.20   14.97    1.00  -62.50  160.63   26.93\n",
      "    1          2A   2A N   CA      9   10  158.91  145.30                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14  -68.41  -68.20    4.19    0.33  -62.50  169.72   28.11\n",
      "    2          3A   3A N   CA     14   15  149.49  145.30                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19  -78.85  -68.20   21.50    1.39  -62.50  155.98   26.40\n",
      "    3          4A   4A N   CA     19   20  163.98  145.30                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24  -91.01  -68.20   27.66    2.72  -62.50  172.93   27.08\n",
      "    4          5A   5A N   CA     24   25  129.66  145.30                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29 -163.85 -134.00   30.17    0.84  -62.50 -164.02   37.52\n",
      "    5          6A   6A N   CA     29   30  151.36  147.00                  -40.90\n",
      "    6  10235   6A   7A C   N      32   34 -146.22 -134.00   12.33    0.35  -62.50 -170.06   35.61\n",
      "    6          7A   7A N   CA     34   35  148.61  147.00                  -40.90\n",
      "    7  10236   7A   8G C   N      37   39  -78.85  -80.20   38.18    1.44   82.20 -154.63    7.38\n",
      "    7          8G   8G N   CA     39   40  135.95  174.10                    8.50\n",
      "    8  10237   8G   9A C   N      41   43  -82.27  -68.20   14.65    1.36  -62.50  178.99   30.34\n",
      "    8          9A   9A N   CA     43   44  141.21  145.30                  -40.90\n",
      "    9  10406 177D 178H C   N    1410 1412   54.78   56.30    3.45    0.45  -63.20  142.55   24.43\n",
      "    9        178H 178H N   CA   1412 1413   37.71   40.80                  -42.30\n",
      "   10  10408 179D 180H C   N    1428 1430  -71.15  -67.60   10.79    0.72  -63.20  167.70   21.47\n",
      "   10        180H 180H N   CA   1430 1431  150.19  140.00                  -42.30\n",
      "   11  10409 180H 181T C   N    1438 1440 -107.42 -124.80   18.22    1.07  -63.20  174.64   25.34\n",
      "   11        181T 181T N   CA   1440 1441  148.95  143.50                  -42.10\n",
      "   12  10410 181T 182G C   N    1445 1447  -64.56  -80.20   48.27    1.22   82.20 -170.47    6.68\n",
      "   12        182G 182G N   CA   1447 1448  128.43  174.10                    8.50\n",
      "   13  10412 183F 184L C   N    1460 1462 -119.82 -108.50   19.12    0.89  -63.50  179.93   28.46\n",
      "   13        184L 184L N   CA   1462 1463  147.91  132.50                  -41.20\n",
      "   14  10413 184L 185T C   N    1468 1470  -77.33  -78.10    1.71    0.10  -63.20  167.17   22.37\n",
      "   14        185T 185T N   CA   1470 1471  151.33  149.80                  -42.10\n",
      "   15  10414 185T 186E C   N    1475 1477  -94.19  -69.30   35.42    1.98  -63.60  155.04   22.40\n",
      "   15        186E 186E N   CA   1477 1478  167.70  142.50                  -40.30\n",
      "   16  10415 186E 187Y C   N    1484 1486  -65.49  -98.40   36.23    1.93  -63.50  173.07   27.56\n",
      "   16        187Y 187Y N   CA   1486 1487  143.54  128.40                  -43.40\n",
      "   17  10417 188V 189A C   N    1503 1505  -86.42  -68.20   35.40    2.28  -62.50  145.43   25.07\n",
      "   17        189A 189A N   CA   1505 1506  175.65  145.30                  -40.90\n",
      "   18  10430 201N 202S C   N    1616 1618  -74.04  -72.40   25.06    1.53  -64.10  162.70   11.23\n",
      "   18        202S 202S N   CA   1618 1619  127.39  152.40                  -35.00\n",
      "   19  10431 202S 203K C   N    1622 1624 -157.20 -118.00   49.10    1.66  -62.90  177.64   28.67\n",
      "   19        203K 203K N   CA   1624 1625  168.66  139.10                  -40.80\n",
      "   20  10432 203K 204G C   N    1631 1633   85.70   78.70    7.05    0.33   82.20  175.50    8.34\n",
      "   20        204G 204G N   CA   1633 1634 -166.96 -166.10                    8.50\n",
      "   21  10433 204G 205Y C   N    1635 1637  -69.95  -98.40   29.53    1.46  -63.50  164.02   25.70\n",
      "   21        205Y 205Y N   CA   1637 1638  120.49  128.40                  -43.40\n",
      "   22  10558 329F 330K C   N    2648 2650 -111.63 -118.00   53.72    2.52  -62.90  135.62   15.70\n",
      "   22        330K 330K N   CA   2650 2651   85.76  139.10                  -40.80\n",
      "   23  10559 330K 331F C   N    2657 2659 -163.99 -124.20   55.75    1.49  -63.20  167.16   29.29\n",
      "   23        331F 331F N   CA   2659 2660 -177.65  143.30                  -44.30\n",
      "   24  10560 331F 332D C   N    2668 2670  -72.69  -70.90   78.36    3.77  -63.30   91.85   11.92\n",
      "   24        332D 332D N   CA   2670 2671 -131.36  150.30                  -40.00\n",
      "   25  10582 353R 354F C   N    2853 2855 -111.60  -63.20   60.50    7.32  -63.20   60.50    7.32\n",
      "   25        354F 354F N   CA   2855 2856   -8.01  -44.30                  -44.30\n",
      "   26  10585 356P 357G C   N    2880 2882 -178.36 -167.20   12.56    0.20   82.20 -161.45   14.07\n",
      "   26        357G 357G N   CA   2882 2883 -179.65  174.60                    8.50\n",
      "   27  10586 357G 358Y C   N    2884 2886 -116.97 -124.30   11.83    0.49  -63.50  177.75   25.71\n",
      "   27        358Y 358Y N   CA   2886 2887  126.12  135.40                  -43.40\n",
      "   28  10587 358Y 359R C   N    2896 2898  -83.18  -72.10   13.10    1.12  -63.00  177.16   22.64\n",
      "   28        359R 359R N   CA   2898 2899  134.91  141.90                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:  1119 1141   2.196\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2839 atoms and 1 coordinate set(s) were parsed in 0.03s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  serious non-bonded atom clash:  2050 2052   1.959\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     1    1    0    1    6   12   28   58  111  176  210  275  350  397  467\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "6opi_MODLL.B99990001.pdb       257.27481\n",
      "\n",
      "Modelando proteína 6opk\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    37761    35115\n",
      "171 (of 2915 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 5663 atoms and 1 coordinate set(s) were parsed in 0.04s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     171\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    35115    1374\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      277\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         218.9017\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     177       0      0   0.008   0.008      3.6417       1.000\n",
      " 2 Bond angle potential               :     244       0      3   2.508   2.508      29.500       1.000\n",
      " 3 Stereochemical cosine torsion poten:     117       0      3  49.453  49.453      42.751       1.000\n",
      " 4 Stereochemical improper torsion pot:      77       0      1   1.608   1.608      5.8290       1.000\n",
      " 5 Soft-sphere overlap restraints     :     277       0      0   0.003   0.003     0.28381       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     218       0      4   0.503   0.503      27.428       1.000\n",
      "10 Distance restraints 2 (N-O)        :     213       0      2   0.526   0.526      23.780       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      25       1      1  34.959  34.959      8.3789       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      14       0      0  59.860  59.860      1.8779       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      11       0      0  48.506  48.506      3.1728       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       7       0      0  81.122  81.122      4.5620       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       1       0      0  57.094  57.094     0.99459       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     125       0      0   0.725   0.725      1.6047       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      26      14      6  48.302 130.860      58.355       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     119       0      0   0.681   0.681      6.7416       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 6opk_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    4381.2197\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 13                           : Mainchain Omega dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10771 184L 184L CA  C    1463 1468   -6.43 -179.18  172.75   40.15 -179.18  172.75   40.15\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9 -123.71 -134.00   12.97    0.34  -62.50 -169.88   34.42\n",
      "    1          2A   2A N   CA      9   10  139.11  147.00                  -40.90\n",
      "    2  10233   4A   5A C   N      22   24  -63.78  -68.20    7.41    0.47  -62.50  179.76   29.52\n",
      "    2          5A   5A N   CA     24   25  139.35  145.30                  -40.90\n",
      "    3  10234   5A   6A C   N      27   29  -85.38  -68.20   23.18    1.47  -62.50  159.89   27.38\n",
      "    3          6A   6A N   CA     29   30  160.86  145.30                  -40.90\n",
      "    4  10235   6A   7A C   N      32   34 -129.04 -134.00    8.09    0.49  -62.50  178.57   32.83\n",
      "    4          7A   7A N   CA     34   35  153.39  147.00                  -40.90\n",
      "    5  10236   7A   8G C   N      37   39 -163.26 -167.20    7.62    0.22   82.20 -163.58   14.45\n",
      "    5          8G   8G N   CA     39   40  168.07  174.60                    8.50\n",
      "    6  10237   8G   9A C   N      41   43 -128.97 -134.00    5.10    0.20  -62.50 -176.33   33.66\n",
      "    6          9A   9A N   CA     43   44  147.88  147.00                  -40.90\n",
      "    7  10238   9A  10G C   N      46   48  156.70 -167.20   39.13    1.15   82.20  168.38   11.57\n",
      "    7         10G  10G N   CA     48   49  159.50  174.60                    8.50\n",
      "    8  10413 184L 185T C   N    1468 1470  172.38 -124.80  117.85    3.66  -63.20  145.12   25.01\n",
      "    8        185T 185T N   CA   1470 1471 -116.79  143.50                  -42.10\n",
      "    9  10414 185T 186E C   N    1475 1477 -105.45 -117.80   28.65    1.57  -63.60  162.52   24.04\n",
      "    9        186E 186E N   CA   1477 1478  162.65  136.80                  -40.30\n",
      "   10  10417 188V 189A C   N    1503 1505   65.78   55.40  132.06    8.81  -62.50 -163.20   30.00\n",
      "   10        189A 189A N   CA   1505 1506  169.85   38.20                  -40.90\n",
      "   11  10582 353R 354F C   N    2853 2855 -121.50  -63.20   70.00    8.67  -63.20   70.00    8.67\n",
      "   11        354F 354F N   CA   2855 2856   -5.57  -44.30                  -44.30\n",
      "   12  10585 356P 357G C   N    2880 2882  -72.06  -80.20   29.55    0.79   82.20 -153.56    6.91\n",
      "   12        357G 357G N   CA   2882 2883  145.70  174.10                    8.50\n",
      "   13  10586 357G 358Y C   N    2884 2886 -120.15 -124.30    6.28    0.27  -63.50 -176.92   26.42\n",
      "   13        358Y 358Y N   CA   2886 2887  130.69  135.40                  -43.40\n",
      "   14  10587 358Y 359R C   N    2896 2898  -71.98  -72.10    0.71    0.06  -63.00  176.53   24.46\n",
      "   14        359R 359R N   CA   2898 2899  142.60  141.90                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    2    2    5   19   51   93  168  183  283  352  375  458\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "6opk_MODLL.B99990001.pdb       218.90170\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 6ot6\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    38183    35537\n",
      "154 (of 2915 total) atoms selected for optimization\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     154\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    35537    1248\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      221\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :          87.0722\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     158       0      0   0.003   0.003     0.45259       1.000\n",
      " 2 Bond angle potential               :     214       0      0   1.497   1.497      8.9439       1.000\n",
      " 3 Stereochemical cosine torsion poten:      90       0      0  49.263  49.263      34.140       1.000\n",
      " 4 Stereochemical improper torsion pot:      62       0      0   1.108   1.108      1.8713       1.000\n",
      " 5 Soft-sphere overlap restraints     :     221       0      0   0.000   0.000     0.42884E-02   1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     192       0      0   0.085   0.085      1.0377       1.000\n",
      "10 Distance restraints 2 (N-O)        :     165       0      0   0.320   0.320      5.9285       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      23       0      0   3.300   3.300      2.2234       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      12       0      0  76.999  76.999      3.5362       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      11       0      0  53.990  53.990      2.1996       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       7       0      0  97.600  97.600      5.6422       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       2       0      0  23.491  23.491      1.4585       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     162       0      0   0.702   0.702      2.2662       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      24      11      1  18.226 125.578      13.774       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     126       0      0   0.949   0.949      3.5940       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 6ot6_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    3600.8193\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9  -66.10  -68.20   16.87    1.29  -62.50  169.49   27.60\n",
      "    1          2A   2A N   CA      9   10  128.56  145.30                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14 -111.88 -134.00   24.25    1.19  -62.50  169.51   30.41\n",
      "    2          3A   3A N   CA     14   15  156.95  147.00                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19  -76.46  -68.20   15.75    1.01  -62.50  160.99   27.09\n",
      "    3          4A   4A N   CA     19   20  158.72  145.30                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24  -63.82  -68.20    4.57    0.43  -62.50  172.49   28.33\n",
      "    4          5A   5A N   CA     24   25  146.61  145.30                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29  -67.09  -68.20    7.00    0.52  -62.50  179.35   29.17\n",
      "    5          6A   6A N   CA     29   30  138.39  145.30                  -40.90\n",
      "    6  10235   6A   7A C   N      32   34  -65.92  -68.20   11.19    0.82  -62.50  175.28   28.56\n",
      "    6          7A   7A N   CA     34   35  134.35  145.30                  -40.90\n",
      "    7  10236   7A   8G C   N      37   39  -95.66  -80.20   16.09    0.91   82.20 -113.94    7.89\n",
      "    7          8G   8G N   CA     39   40  178.53  174.10                    8.50\n",
      "    8  10237   8G   9A C   N      41   43 -130.49 -134.00    7.74    0.33  -62.50 -168.52   35.02\n",
      "    8          9A   9A N   CA     43   44  140.10  147.00                  -40.90\n",
      "    9  10238   9A  10G C   N      46   48 -166.70 -167.20   12.65    0.49   82.20 -170.56   13.95\n",
      "    9         10G  10G N   CA     48   49  161.95  174.60                    8.50\n",
      "   10  10561 332D 333M C   N    2676 2678 -145.54 -125.60   29.01    0.90  -63.40  178.01   32.99\n",
      "   10        333M 333M N   CA   2678 2679  161.57  140.50                  -40.50\n",
      "   11  10587 358Y 359R C   N    2896 2898  -74.68  -72.10    8.09    0.57  -63.00  169.73   23.72\n",
      "   11        359R 359R N   CA   2898 2899  149.57  141.90                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    1    2    3   14   46  124  184  193  262  352  416  495\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "6ot6_MODLL.B99990001.pdb        87.07216\n",
      "\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 5250 atoms and 1 coordinate set(s) were parsed in 0.04s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 6ots\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    36148    33502\n",
      "441 (of 2915 total) atoms selected for optimization\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     441\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    33502    3491\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      846\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         360.2909\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     457       0      0   0.006   0.006      5.1353       1.000\n",
      " 2 Bond angle potential               :     633       0      1   1.874   1.874      41.191       1.000\n",
      " 3 Stereochemical cosine torsion poten:     297       0     10  49.747  49.747      112.80       1.000\n",
      " 4 Stereochemical improper torsion pot:     196       0      1   1.260   1.260      8.6546       1.000\n",
      " 5 Soft-sphere overlap restraints     :     846       0      0   0.006   0.006      3.3733       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     577       0      0   0.308   0.308      19.893       1.000\n",
      "10 Distance restraints 2 (N-O)        :     544       0      0   0.626   0.626      32.866       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      65       1      0  22.408  22.408      5.1940       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      45       0      1  70.583  70.583      9.5737       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      35       0      2  72.234  72.234      17.105       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:      13       0      0  95.799  95.799      7.8603       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       3       0      0  78.307  78.307      1.4765       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     368       0      0   0.496   0.496      2.2402       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      70      32     12  39.028 123.332      86.726       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     188       0      0   0.811   0.811      6.2016       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 6ots_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:   10552.7090\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 13                           : Mainchain Omega dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10841 254C 254C CA  C    2024 2027    0.15 -179.18  179.33   41.68 -179.18  179.33   41.68\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9 -129.63 -134.00   10.11    0.62  -62.50  176.27   32.48\n",
      "    1          2A   2A N   CA      9   10  156.12  147.00                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14  -69.50  -68.20    6.52    0.48  -62.50  167.56   27.81\n",
      "    2          3A   3A N   CA     14   15  151.69  145.30                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19 -125.74 -134.00   10.91    0.62  -62.50  176.68   32.34\n",
      "    3          4A   4A N   CA     19   20  154.13  147.00                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24  -85.18  -68.20   20.83    2.05  -62.50  175.61   27.75\n",
      "    4          5A   5A N   CA     24   25  133.24  145.30                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29  -79.60  -68.20   12.26    0.88  -62.50  170.15   28.75\n",
      "    5          6A   6A N   CA     29   30  149.82  145.30                  -40.90\n",
      "    6  10235   6A   7A C   N      32   34 -150.85 -134.00   19.13    0.44  -62.50 -174.54   35.12\n",
      "    6          7A   7A N   CA     34   35  156.04  147.00                  -40.90\n",
      "    7  10236   7A   8G C   N      37   39  -97.83  -80.20   36.60    0.86   82.20 -117.64    8.04\n",
      "    7          8G   8G N   CA     39   40 -153.82  174.10                    8.50\n",
      "    8  10237   8G   9A C   N      41   43  -69.16  -68.20    7.68    0.58  -62.50  166.32   27.59\n",
      "    8          9A   9A N   CA     43   44  152.92  145.30                  -40.90\n",
      "    9  10347 118T 119Q C   N     933  935 -158.10 -121.10   50.77    1.63  -63.80  173.16   30.23\n",
      "    9        119Q 119Q N   CA    935  936  174.47  139.70                  -40.30\n",
      "   10  10348 119Q 120H C   N     942  944  -70.12  -67.60    8.80    0.60  -63.20  169.40   21.62\n",
      "   10        120H 120H N   CA    944  945  148.44  140.00                  -42.30\n",
      "   11  10403 174A 175D C   N    1387 1389 -116.08  -96.50   24.44    1.02  -63.30  176.87   19.28\n",
      "   11        175D 175D N   CA   1389 1390  128.81  114.20                  -40.00\n",
      "   12  10405 176P 177D C   N    1402 1404   56.82   54.50    2.59    0.22  -63.30  144.17   24.55\n",
      "   12        177D 177D N   CA   1404 1405   39.73   40.90                  -40.00\n",
      "   13  10406 177D 178H C   N    1410 1412  -62.79  -67.60    5.58    0.34  -63.20  179.46   22.46\n",
      "   13        178H 178H N   CA   1412 1413  137.16  140.00                  -42.30\n",
      "   14  10407 178H 179D C   N    1420 1422  -75.59  -70.90   11.96    0.41  -63.30  159.18   20.41\n",
      "   14        179D 179D N   CA   1422 1423  161.29  150.30                  -40.00\n",
      "   15  10409 180H 181T C   N    1438 1440  -86.60  -78.10   27.10    0.93  -63.20  144.28   20.06\n",
      "   15        181T 181T N   CA   1440 1441  175.53  149.80                  -42.10\n",
      "   16  10411 182G 183F C   N    1449 1451   75.44  -71.40  148.75    9.33  -63.20 -147.38   37.66\n",
      "   16        183F 183F N   CA   1451 1452  116.91  140.70                  -44.30\n",
      "   17  10412 183F 184L C   N    1460 1462 -119.28 -108.50   31.98    1.60  -63.50  165.85   26.48\n",
      "   17        184L 184L N   CA   1462 1463  162.61  132.50                  -41.20\n",
      "   18  10413 184L 185T C   N    1468 1470 -125.96 -124.80    1.75    0.10   55.90 -154.38   25.35\n",
      "   18        185T 185T N   CA   1470 1471  142.19  143.50                   39.50\n",
      "   19  10429 200L 201N C   N    1608 1610   75.99   55.90   99.28    8.46  -63.20 -134.18   35.73\n",
      "   19        201N 201N N   CA   1610 1611  136.72   39.50                  -41.10\n",
      "   20  10430 201N 202S C   N    1616 1618  140.90 -136.60   99.34    3.03  -64.10 -164.92   25.03\n",
      "   20        202S 202S N   CA   1618 1619 -153.46  151.20                  -35.00\n",
      "   21  10431 202S 203K C   N    1622 1624 -109.90 -118.00    9.99    0.48  -62.90 -179.52   26.22\n",
      "   21        203K 203K N   CA   1624 1625  144.95  139.10                  -40.80\n",
      "   22  10432 203K 204G C   N    1631 1633  114.68   78.70   56.85    1.07   82.20  145.05    8.62\n",
      "   22        204G 204G N   CA   1633 1634  149.87 -166.10                    8.50\n",
      "   23  10481 252L 253N C   N    2013 2015  -81.64 -119.90   75.44    2.47  -63.20  114.57   13.39\n",
      "   23        253N 253N N   CA   2015 2016   71.98  137.00                  -41.10\n",
      "   24  10485 256I 257N C   N    2043 2045  -55.87  -71.20   15.76    1.22  -63.20  172.58   21.18\n",
      "   24        257N 257N N   CA   2045 2046  146.48  142.80                  -41.10\n",
      "   25  10548 319P 320S C   N    2573 2575  -70.30  -72.40    4.78    0.35  -64.10  168.42   12.58\n",
      "   25        320S 320S N   CA   2575 2576  156.70  152.40                  -35.00\n",
      "   26  10549 320S 321D C   N    2579 2581  -80.15  -70.90   29.14    1.04  -63.30  143.06   18.72\n",
      "   26        321D 321D N   CA   2581 2582  177.94  150.30                  -40.00\n",
      "   27  10550 321D 322E C   N    2587 2589  -59.94  -69.30   12.40    0.69  -63.60  174.71   23.50\n",
      "   27        322E 322E N   CA   2589 2590  134.37  142.50                  -40.30\n",
      "   28  10553 324I 325A C   N    2611 2613  -58.03  -68.20   10.58    0.79  -62.50  176.76   28.75\n",
      "   28        325A 325A N   CA   2613 2614  142.40  145.30                  -40.90\n",
      "   29  10554 325A 326E C   N    2616 2618 -135.77 -117.80   21.49    0.63  -63.60 -174.29   28.82\n",
      "   29        326E 326E N   CA   2618 2619  148.58  136.80                  -40.30\n",
      "   30  10555 326E 327A C   N    2625 2627 -125.76 -134.00   18.96    0.81  -62.50 -177.84   27.54\n",
      "   30        327A 327A N   CA   2627 2628  129.93  147.00                  -40.90\n",
      "   31  10558 329F 330K C   N    2648 2650   55.60  -62.90  145.03   18.83 -118.00 -161.40    9.30\n",
      "   31        330K 330K N   CA   2650 2651 -124.43  -40.80                  139.10\n",
      "   32  10587 358Y 359R C   N    2896 2898 -119.67 -125.20   18.74    1.00  -63.00  170.12   26.89\n",
      "   32        359R 359R N   CA   2898 2899  158.50  140.60                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2798 atoms and 1 coordinate set(s) were parsed in 0.06s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  serious non-bonded atom clash:  1551 1758   2.261\n",
      "  serious non-bonded atom clash:  2663 2691   2.095\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     1    0    1    3    6   18   19   84  138  181  216  319  353  414  420\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "6ots_MODLL.B99990001.pdb       360.29092\n",
      "\n",
      "Modelando proteína 6q7k\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    37959    35313\n",
      "188 (of 2915 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2785 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     188\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    35313    1592\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      311\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         146.3229\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     193       0      0   0.005   0.005      1.2518       1.000\n",
      " 2 Bond angle potential               :     268       0      1   2.262   2.262      25.320       1.000\n",
      " 3 Stereochemical cosine torsion poten:     113       0      3  51.325  51.325      45.545       1.000\n",
      " 4 Stereochemical improper torsion pot:      74       0      0   1.363   1.363      3.0573       1.000\n",
      " 5 Soft-sphere overlap restraints     :     311       0      0   0.005   0.005      1.0432       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     276       0      0   0.180   0.180      7.3577       1.000\n",
      "10 Distance restraints 2 (N-O)        :     254       0      0   0.399   0.399      10.791       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      29       1      0  33.510  33.510      5.4203       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      18       0      0  37.961  37.961     0.63390       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      14       0      0  50.617  50.617      6.0582       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       6       0      0  73.448  73.448      2.3615       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       1       0      0 178.392 178.392     0.56610       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     190       0      0   0.601   0.601      1.1911       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      31      14      4  27.205 111.717      21.942       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     125       0      1   1.294   1.294      13.783       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 6q7k_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    4363.0244\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9  -74.08  -68.20   12.86    0.84  -62.50  162.78   27.26\n",
      "    1          2A   2A N   CA      9   10  156.74  145.30                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14 -113.99 -134.00   22.04    1.09  -62.50  170.81   30.73\n",
      "    2          3A   3A N   CA     14   15  156.24  147.00                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19 -104.46 -134.00   29.59    1.05  -62.50  175.41   30.96\n",
      "    3          4A   4A N   CA     19   20  148.79  147.00                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24  -95.64  -68.20   28.26    2.60  -62.50 -177.52   28.46\n",
      "    4          5A   5A N   CA     24   25  138.54  145.30                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29  -72.08  -68.20    4.11    0.39  -62.50  175.43   29.23\n",
      "    5          6A   6A N   CA     29   30  143.93  145.30                  -40.90\n",
      "    6  10235   6A   7A C   N      32   34  -77.14  -68.20   13.01    0.82  -62.50  165.01   27.78\n",
      "    6          7A   7A N   CA     34   35  154.75  145.30                  -40.90\n",
      "    7  10562 333M 334E C   N    2684 2686  -56.58  -69.30   20.76    1.18  -63.60  166.54   22.59\n",
      "    7        334E 334E N   CA   2686 2687  126.09  142.50                  -40.30\n",
      "    8  10563 334E 335L C   N    2693 2695 -146.12 -108.50   79.56    3.82  -63.50  142.57   24.81\n",
      "    8        335L 335L N   CA   2695 2696 -157.39  132.50                  -41.20\n",
      "    9  10564 335L 336D C   N    2701 2703  -75.93  -70.90   48.48    2.14  -63.30  122.14   15.87\n",
      "    9        336D 336D N   CA   2703 2704 -161.48  150.30                  -40.00\n",
      "   10  10565 336D 337D C   N    2709 2711   57.19   54.50    4.98    0.57  -63.30  147.51   25.07\n",
      "   10        337D 337D N   CA   2711 2712   45.09   40.90                  -40.00\n",
      "   11  10566 337D 338L C   N    2717 2719  -67.47  -63.50    9.39    1.15 -108.50  170.22    8.71\n",
      "   11        338L 338L N   CA   2719 2720  -32.70  -41.20                  132.50\n",
      "   12  10585 356P 357G C   N    2880 2882  161.14 -167.20   32.45    0.49  -80.20  118.90    7.69\n",
      "   12        357G 357G N   CA   2882 2883 -178.27  174.60                  174.10\n",
      "   13  10586 357G 358Y C   N    2884 2886 -110.26  -98.40   12.56    0.67  -63.50 -177.97   26.66\n",
      "   13        358Y 358Y N   CA   2886 2887  132.52  128.40                  -43.40\n",
      "   14  10587 358Y 359R C   N    2896 2898  -76.11  -72.10   13.79    0.97  -63.00  164.33   23.09\n",
      "   14        359R 359R N   CA   2898 2899  155.09  141.90                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:  1365 1373   2.300\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    1    0    3    8   16   54  131  165  200  287  360  414  472\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "6q7k_MODLL.B99990001.pdb       146.32288\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 6q7s\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    37769    35123\n",
      "224 (of 2915 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2771 atoms and 1 coordinate set(s) were parsed in 0.03s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     224\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    35123    1962\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      410\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         157.3587\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     232       0      0   0.003   0.003     0.78629       1.000\n",
      " 2 Bond angle potential               :     319       0      0   1.871   1.871      20.833       1.000\n",
      " 3 Stereochemical cosine torsion poten:     147       0      7  52.971  52.971      59.623       1.000\n",
      " 4 Stereochemical improper torsion pot:      93       0      0   0.858   0.858      1.7585       1.000\n",
      " 5 Soft-sphere overlap restraints     :     410       0      0   0.002   0.002     0.16096       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     330       0      0   0.143   0.143      4.1767       1.000\n",
      "10 Distance restraints 2 (N-O)        :     308       0      0   0.167   0.167      4.4429       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      33       0      0   3.509   3.509      3.9592       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      21       0      0  73.725  73.725      5.3528       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      17       0      0  56.054  56.054      4.8376       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:      10       0      0  79.595  79.595      6.2563       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       2       0      0   5.291   5.291     0.81801       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     227       0      0   0.560   0.560      1.5062       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      35      15      6  25.835 117.763      34.317       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     188       0      0   0.681   0.681      8.5297       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 6q7s_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    4970.0957\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9  -77.22  -68.20   13.65    0.86  -62.50  164.22   27.66\n",
      "    1          2A   2A N   CA      9   10  155.54  145.30                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14 -105.45 -134.00   31.62    0.72  -62.50  179.52   27.65\n",
      "    2          3A   3A N   CA     14   15  133.41  147.00                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19  -81.68  -68.20   23.11    1.46  -62.50  156.22   26.58\n",
      "    3          4A   4A N   CA     19   20  164.07  145.30                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24  -59.08  -68.20    9.50    0.71  -62.50  176.52   28.76\n",
      "    4          5A   5A N   CA     24   25  142.61  145.30                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29  -78.50  -68.20   12.46    0.82  -62.50  167.56   28.27\n",
      "    5          6A   6A N   CA     29   30  152.31  145.30                  -40.90\n",
      "    6  10235   6A   7A C   N      32   34 -160.59 -134.00   31.57    0.76  -62.50 -176.51   35.28\n",
      "    6          7A   7A N   CA     34   35  164.03  147.00                  -40.90\n",
      "    7  10236   7A   8G C   N      37   39 -159.91 -167.20   18.56    0.60   82.20 -169.98   14.18\n",
      "    7          8G   8G N   CA     39   40  157.53  174.60                    8.50\n",
      "    8  10237   8G   9A C   N      41   43 -144.11 -134.00   19.72    0.78  -62.50  175.32   33.10\n",
      "    8          9A   9A N   CA     43   44  163.93  147.00                  -40.90\n",
      "    9  10389 160T 161C C   N    1283 1285   74.49   57.40   22.49    1.45 -117.90 -154.02    8.87\n",
      "    9        161C 161C N   CA   1285 1286   21.38   36.00                  141.10\n",
      "   10  10558 329F 330K C   N    2648 2650  -63.24  -70.20   16.70    1.35  -62.90  163.62   21.27\n",
      "   10        330K 330K N   CA   2650 2651  155.58  140.40                  -40.80\n",
      "   11  10559 330K 331F C   N    2657 2659  -89.23  -71.40   29.09    1.76  -63.20  154.23   23.17\n",
      "   11        331F 331F N   CA   2659 2660  163.68  140.70                  -44.30\n",
      "   12  10560 331F 332D C   N    2668 2670   51.22   54.50   24.59    1.55  -63.30  155.55   26.05\n",
      "   12        332D 332D N   CA   2670 2671   65.27   40.90                  -40.00\n",
      "   13  10585 356P 357G C   N    2880 2882   64.02   78.70   54.41    0.99   82.20  123.56    6.90\n",
      "   13        357G 357G N   CA   2882 2883 -113.71 -166.10                    8.50\n",
      "   14  10586 357G 358Y C   N    2884 2886  -43.44  -98.40   55.34    2.15  -63.50  179.38   29.50\n",
      "   14        358Y 358Y N   CA   2886 2887  134.86  128.40                  -43.40\n",
      "   15  10587 358Y 359R C   N    2896 2898 -132.72 -125.20   25.82    1.06  -63.00  168.69   27.55\n",
      "   15        359R 359R N   CA   2898 2899  165.30  140.60                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:  2715 2718   2.167\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    1    0    0    1    6   15   61  125  172  210  268  355  416  488\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "6q7s_MODLL.B99990001.pdb       157.35875\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 6q7t\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    37880    35234\n",
      "240 (of 2915 total) atoms selected for optimization\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     240\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    35234    2019\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      429\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         168.8231\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     248       0      0   0.003   0.003     0.68027       1.000\n",
      " 2 Bond angle potential               :     341       0      0   1.835   1.835      22.281       1.000\n",
      " 3 Stereochemical cosine torsion poten:     155       0      3  52.640  52.640      63.063       1.000\n",
      " 4 Stereochemical improper torsion pot:      99       0      0   1.000   1.000      2.7211       1.000\n",
      " 5 Soft-sphere overlap restraints     :     429       0      0   0.004   0.004     0.73788       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     314       0      0   0.109   0.109      2.8090       1.000\n",
      "10 Distance restraints 2 (N-O)        :     300       0      0   0.293   0.293      9.0194       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      35       1      0  30.387  30.387      5.4171       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      23       0      0  75.439  75.439      5.4566       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      19       0      0  44.981  44.981      5.4419       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:      10       0      0  88.169  88.169      4.0006       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       2       0      0 123.130 123.130     0.91738       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     250       0      0   0.470   0.470     0.99720       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      37      19      6  25.703 126.491      43.084       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     186       0      0   0.438   0.438      2.1967       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 6q7t_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    5800.9521\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 13                           : Mainchain Omega dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10747 160T 160T CA  C    1279 1283   -0.57 -179.18  178.61   41.51 -179.18  178.61   41.51\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9  -72.37  -68.20    7.02    0.44  -62.50  168.45   28.10\n",
      "    1          2A   2A N   CA      9   10  150.95  145.30                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14 -140.93 -134.00    7.15    0.31  -62.50 -169.26   35.46\n",
      "    2          3A   3A N   CA     14   15  145.23  147.00                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19  -71.21  -68.20    3.03    0.27  -62.50  174.35   29.01\n",
      "    3          4A   4A N   CA     19   20  144.97  145.30                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24 -145.04 -134.00   17.08    0.57  -62.50  179.21   33.79\n",
      "    4          5A   5A N   CA     24   25  160.03  147.00                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29 -117.15 -134.00   17.92    0.42  -62.50 -173.61   33.45\n",
      "    5          6A   6A N   CA     29   30  140.91  147.00                  -40.90\n",
      "    6  10235   6A   7A C   N      32   34  -70.51  -68.20    5.85    0.39  -62.50  168.62   28.03\n",
      "    6          7A   7A N   CA     34   35  150.67  145.30                  -40.90\n",
      "    7  10236   7A   8G C   N      37   39   77.76   78.70    4.52    0.17   82.20  179.08    8.92\n",
      "    7          8G   8G N   CA     39   40 -170.52 -166.10                    8.50\n",
      "    8  10237   8G   9A C   N      41   43 -145.93 -134.00   17.42    0.54  -62.50  179.92   33.95\n",
      "    8          9A   9A N   CA     43   44  159.69  147.00                  -40.90\n",
      "    9  10238   9A  10G C   N      46   48 -103.59  -80.20   43.94    2.83   82.20 -143.58   16.89\n",
      "    9         10G  10G N   CA     48   49  136.90  174.10                    8.50\n",
      "   10  10558 329F 330K C   N    2648 2650 -118.83 -118.00   35.64    1.74  -62.90  154.73   17.91\n",
      "   10        330K 330K N   CA   2650 2651  103.47  139.10                  -40.80\n",
      "   11  10559 330K 331F C   N    2657 2659 -138.55 -124.20   14.35    0.57  -63.20 -171.66   31.06\n",
      "   11        331F 331F N   CA   2659 2660  143.10  143.30                  -44.30\n",
      "   12  10560 331F 332D C   N    2668 2670  -83.75  -96.50   22.40    0.94  -63.30  137.30   15.81\n",
      "   12        332D 332D N   CA   2670 2671   95.77  114.20                  -40.00\n",
      "   13  10561 332D 333M C   N    2676 2678  -73.97  -73.00    7.58    0.50  -63.40  169.31   26.11\n",
      "   13        333M 333M N   CA   2678 2679  150.52  143.00                  -40.50\n",
      "   14  10562 333M 334E C   N    2684 2686 -149.85 -117.80   56.10    2.02  -63.60  161.77   26.29\n",
      "   14        334E 334E N   CA   2686 2687 -177.16  136.80                  -40.30\n",
      "   15  10563 334E 335L C   N    2693 2695  -67.38  -70.70   22.37    1.82  -63.50  155.13   21.76\n",
      "   15        335L 335L N   CA   2695 2696  163.72  141.60                  -41.20\n",
      "   16  10564 335L 336D C   N    2701 2703  -55.39  -63.30   20.40    2.18  -70.90  151.70    8.25\n",
      "   16        336D 336D N   CA   2703 2704  -58.80  -40.00                  150.30\n",
      "   17  10585 356P 357G C   N    2880 2882   75.15   78.70   21.54    0.76   82.20  164.30    7.62\n",
      "   17        357G 357G N   CA   2882 2883  172.65 -166.10                    8.50\n",
      "   18  10586 357G 358Y C   N    2884 2886 -128.92 -124.30    8.83    0.35  -63.50 -174.41   32.73\n",
      "   18        358Y 358Y N   CA   2886 2887  142.92  135.40                  -43.40\n",
      "   19  10587 358Y 359R C   N    2896 2898  -71.60  -72.10    7.31    0.55  -63.00  175.92   23.20\n",
      "   19        359R 359R N   CA   2898 2899  134.61  141.90                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:   458 2726   2.289\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    1    0    0    7   15   48  131  187  199  240  355  446  481\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2788 atoms and 1 coordinate set(s) were parsed in 0.03s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "6q7t_MODLL.B99990001.pdb       168.82312\n",
      "\n",
      "Modelando proteína 6qa1\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    37708    35062\n",
      "225 (of 2915 total) atoms selected for optimization\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     225\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    35062    1827\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      436\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         151.3441\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     231       0      0   0.004   0.004      1.0065       1.000\n",
      " 2 Bond angle potential               :     317       0      1   1.909   1.909      21.100       1.000\n",
      " 3 Stereochemical cosine torsion poten:     139       0      3  49.828  49.828      53.767       1.000\n",
      " 4 Stereochemical improper torsion pot:      94       0      0   1.155   1.155      2.9208       1.000\n",
      " 5 Soft-sphere overlap restraints     :     436       0      0   0.002   0.002     0.21991       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     275       0      0   0.164   0.164      4.5277       1.000\n",
      "10 Distance restraints 2 (N-O)        :     281       0      0   0.317   0.317      8.1057       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      33       1      0  31.457  31.457      6.9508       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      21       0      0  52.165  52.165      3.7583       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      17       0      0  41.902  41.902      6.5370       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       7       0      0 106.010 106.010      5.5127       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       2       0      0 115.285 115.285      1.4483       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     216       0      0   0.609   0.609      1.4694       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      35      15      3  27.749 118.580      27.997       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     159       0      0   0.627   0.627      6.0231       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 6qa1_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    4907.3105\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 13                           : Mainchain Omega dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10747 160T 160T CA  C    1279 1283    0.00 -179.18  179.18   41.64 -179.18  179.18   41.64\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9  -67.32  -68.20   11.59    0.90  -62.50  174.71   28.40\n",
      "    1          2A   2A N   CA      9   10  133.74  145.30                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14  -71.26  -68.20    7.67    0.52  -62.50  167.00   27.81\n",
      "    2          3A   3A N   CA     14   15  152.33  145.30                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19  -96.14  -68.20   30.32    2.90  -62.50  177.65   27.66\n",
      "    3          4A   4A N   CA     19   20  133.53  145.30                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24 -130.27 -134.00    6.17    0.37  -62.50 -179.60   33.19\n",
      "    4          5A   5A N   CA     24   25  151.91  147.00                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29 -112.01 -134.00   24.77    0.57  -62.50 -176.69   28.06\n",
      "    5          6A   6A N   CA     29   30  135.60  147.00                  -40.90\n",
      "    6  10235   6A   7A C   N      32   34  -74.01  -68.20    5.96    0.55  -62.50  175.52   29.34\n",
      "    6          7A   7A N   CA     34   35  143.96  145.30                  -40.90\n",
      "    7  10236   7A   8G C   N      37   39  138.13 -167.20   54.95    0.93   82.20 -179.50   11.43\n",
      "    7          8G   8G N   CA     39   40 -179.88  174.60                    8.50\n",
      "    8  10237   8G   9A C   N      41   43 -122.13 -134.00   12.58    0.29  -62.50 -173.93   33.67\n",
      "    8          9A   9A N   CA     43   44  142.84  147.00                  -40.90\n",
      "    9  10238   9A  10G C   N      46   48  -89.78  -80.20   10.45    0.53   82.20 -118.34    7.61\n",
      "    9         10G  10G N   CA     48   49  178.28  174.10                    8.50\n",
      "   10  10559 330K 331F C   N    2657 2659  -74.53  -71.40   19.99    1.69  -63.20  165.64   22.25\n",
      "   10        331F 331F N   CA   2659 2660  120.95  140.70                  -44.30\n",
      "   11  10561 332D 333M C   N    2676 2678 -132.71 -125.60   11.67    0.38  -63.40 -176.65   32.86\n",
      "   11        333M 333M N   CA   2678 2679  149.76  140.50                  -40.50\n",
      "   12  10562 333M 334E C   N    2684 2686  -74.14  -69.30   16.74    1.10  -63.60  161.52   22.12\n",
      "   12        334E 334E N   CA   2686 2687  158.53  142.50                  -40.30\n",
      "   13  10563 334E 335L C   N    2693 2695  -59.90  -70.70   35.76    2.34  -63.50  148.76   20.86\n",
      "   13        335L 335L N   CA   2695 2696  107.51  141.60                  -41.20\n",
      "   14  10565 336D 337D C   N    2709 2711  -85.66  -63.30   65.53    7.06   54.50  141.48   17.48\n",
      "   14        337D 337D N   CA   2711 2712   21.60  -40.00                   40.90\n",
      "   15  10587 358Y 359R C   N    2896 2898 -118.00 -125.20    9.73    0.53  -63.00 -179.65   28.16\n",
      "   15        359R 359R N   CA   2898 2899  147.14  140.60                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    0    1   13   46  139  171  172  268  358  429  493\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "6qa1_MODLL.B99990001.pdb       151.34410\n",
      "\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2772 atoms and 1 coordinate set(s) were parsed in 0.03s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 6qa3\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    37700    35054\n",
      "226 (of 2915 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2743 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     226\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    35054    1816\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      381\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         142.4666\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     234       0      0   0.003   0.003     0.65746       1.000\n",
      " 2 Bond angle potential               :     323       0      0   1.891   1.891      21.960       1.000\n",
      " 3 Stereochemical cosine torsion poten:     150       0      1  47.710  47.710      52.594       1.000\n",
      " 4 Stereochemical improper torsion pot:      94       0      0   1.108   1.108      2.4706       1.000\n",
      " 5 Soft-sphere overlap restraints     :     381       0      0   0.002   0.002     0.23514       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     272       0      0   0.124   0.124      2.9605       1.000\n",
      "10 Distance restraints 2 (N-O)        :     276       0      0   0.302   0.302      8.9328       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      33       1      0  31.464  31.464      4.0054       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      22       0      0  78.586  78.586      5.3163       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      18       0      0  67.570  67.570      6.7421       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       8       0      0  91.420  91.420      3.6350       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       2       0      0  45.400  45.400      1.1275       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     199       0      0   0.477   0.477     0.80348       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      35      16      4  24.431 117.207      24.407       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     150       0      0   0.724   0.724      6.6190       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 6qa3_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    5306.2739\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 13                           : Mainchain Omega dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10747 160T 160T CA  C    1279 1283    0.74 -179.18  179.92   41.82 -179.18  179.92   41.82\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9  -67.07  -68.20    6.46    0.56  -62.50  167.50   27.68\n",
      "    1          2A   2A N   CA      9   10  151.67  145.30                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14 -106.87 -134.00   27.83    1.16  -62.50  171.73   30.49\n",
      "    2          3A   3A N   CA     14   15  153.21  147.00                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19  -68.57  -68.20   14.64    1.17  -62.50  159.28   26.41\n",
      "    3          4A   4A N   CA     19   20  159.93  145.30                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24  -62.51  -68.20    5.84    0.54  -62.50  172.46   28.26\n",
      "    4          5A   5A N   CA     24   25  146.64  145.30                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29 -127.96 -134.00   12.38    0.76  -62.50  174.08   32.03\n",
      "    5          6A   6A N   CA     29   30  157.80  147.00                  -40.90\n",
      "    6  10235   6A   7A C   N      32   34  -61.54  -68.20    7.88    0.77  -62.50  169.58   27.74\n",
      "    6          7A   7A N   CA     34   35  149.52  145.30                  -40.90\n",
      "    7  10558 329F 330K C   N    2648 2650 -140.91 -118.00   29.36    1.44  -62.90  179.40   20.62\n",
      "    7        330K 330K N   CA   2650 2651  120.75  139.10                  -40.80\n",
      "    8  10559 330K 331F C   N    2657 2659 -127.05 -124.20   11.14    0.58  -63.20 -171.99   22.32\n",
      "    8        331F 331F N   CA   2659 2660  132.54  143.30                  -44.30\n",
      "    9  10560 331F 332D C   N    2668 2670  -91.95  -70.90   33.85    1.23  -63.30  146.04   19.85\n",
      "    9        332D 332D N   CA   2670 2671  176.80  150.30                  -40.00\n",
      "   10  10561 332D 333M C   N    2676 2678  -95.43  -73.00   24.80    1.98  -63.40  175.88   23.95\n",
      "   10        333M 333M N   CA   2678 2679  132.43  143.00                  -40.50\n",
      "   11  10562 333M 334E C   N    2684 2686  -73.19  -69.30   10.09    0.63  -63.60  168.17   22.95\n",
      "   11        334E 334E N   CA   2686 2687  151.80  142.50                  -40.30\n",
      "   12  10564 335L 336D C   N    2701 2703  -79.64  -70.90    9.36    0.78  -63.30  173.80   22.47\n",
      "   12        336D 336D N   CA   2703 2704  146.98  150.30                  -40.00\n",
      "   13  10565 336D 337D C   N    2709 2711  -88.22  -70.90   23.71    0.99  -63.30  155.53   20.78\n",
      "   13        337D 337D N   CA   2711 2712  166.48  150.30                  -40.00\n",
      "   14  10585 356P 357G C   N    2880 2882  152.59 -167.20   43.71    1.29  -80.20  128.29    8.99\n",
      "   14        357G 357G N   CA   2882 2883  157.46  174.60                  174.10\n",
      "   15  10586 357G 358Y C   N    2884 2886 -113.03 -124.30   11.43    0.91  -63.50 -174.01   32.04\n",
      "   15        358Y 358Y N   CA   2886 2887  137.33  135.40                  -43.40\n",
      "   16  10587 358Y 359R C   N    2896 2898  -60.68  -72.10   11.43    0.87  -63.00  177.28   23.79\n",
      "   16        359R 359R N   CA   2898 2899  141.64  141.90                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    0    1   14   62  107  161  172  242  379  416  459\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "6qa3_MODLL.B99990001.pdb       142.46660\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 6qa4\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    37934    35288\n",
      "290 (of 2915 total) atoms selected for optimization\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     290\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    35288    2481\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      584\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         210.0217\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     300       0      0   0.004   0.004      1.1398       1.000\n",
      " 2 Bond angle potential               :     411       0      0   1.909   1.909      28.232       1.000\n",
      " 3 Stereochemical cosine torsion poten:     183       0      6  52.473  52.473      71.128       1.000\n",
      " 4 Stereochemical improper torsion pot:     120       0      0   0.978   0.978      2.4048       1.000\n",
      " 5 Soft-sphere overlap restraints     :     584       0      0   0.003   0.003     0.84003       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     419       0      1   0.280   0.280      18.465       1.000\n",
      "10 Distance restraints 2 (N-O)        :     423       0      1   0.378   0.378      21.684       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      43       0      0   4.651   4.651      10.290       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      27       0      0  77.041  77.041      6.1985       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      23       0      0  50.873  50.873      7.3734       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:      12       0      0  78.649  78.649      4.6846       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       3       0      0 112.294 112.294      2.2751       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     272       0      0   0.401   0.401     0.93580       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      46      18      6  25.052 114.351      27.131       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     199       0      0   0.738   0.738      7.2397       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 6qa4_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    5785.5869\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9  -67.27  -68.20   17.53    1.38  -62.50  168.77   27.43\n",
      "    1          2A   2A N   CA      9   10  127.80  145.30                  -40.90\n",
      "    2  10232   3A   4A C   N      17   19 -104.54 -134.00   31.32    0.73  -62.50 -177.85   28.10\n",
      "    2          4A   4A N   CA     19   20  136.34  147.00                  -40.90\n",
      "    3  10233   4A   5A C   N      22   24  -74.70  -68.20    8.90    0.56  -62.50  168.17   28.17\n",
      "    3          5A   5A N   CA     24   25  151.37  145.30                  -40.90\n",
      "    4  10234   5A   6A C   N      27   29  -66.77  -68.20    8.29    0.62  -62.50  178.09   28.98\n",
      "    4          6A   6A N   CA     29   30  137.14  145.30                  -40.90\n",
      "    5  10237   8G   9A C   N      41   43 -111.42 -134.00   25.76    0.59  -62.50 -177.81   27.90\n",
      "    5          9A   9A N   CA     43   44  134.60  147.00                  -40.90\n",
      "    6  10238   9A  10G C   N      46   48  -82.84  -80.20    2.81    0.21   82.20 -126.90    7.30\n",
      "    6         10G  10G N   CA     48   49  173.12  174.10                    8.50\n",
      "    7  10263  34G  35A C   N     238  240  -66.25  -68.20   31.90    2.64  -62.50  142.01   23.46\n",
      "    7         35A  35A N   CA    240  241  177.14  145.30                  -40.90\n",
      "    8  10389 160T 161C C   N    1283 1285   67.27   57.40    9.89    1.17 -117.90 -155.74    8.65\n",
      "    8        161C 161C N   CA   1285 1286   35.48   36.00                  141.10\n",
      "    9  10559 330K 331F C   N    2657 2659  169.05 -124.20   67.95    3.02  -63.20 -143.43   24.44\n",
      "    9        331F 331F N   CA   2659 2660  130.58  143.30                  -44.30\n",
      "   10  10560 331F 332D C   N    2668 2670  -98.24  -96.50   15.49    0.65  -63.30  143.14   15.88\n",
      "   10        332D 332D N   CA   2670 2671   98.81  114.20                  -40.00\n",
      "   11  10561 332D 333M C   N    2676 2678  -74.17  -73.00    5.94    0.38  -63.40  171.02   26.38\n",
      "   11        333M 333M N   CA   2678 2679  148.82  143.00                  -40.50\n",
      "   12  10562 333M 334E C   N    2684 2686 -107.52 -117.80   11.13    0.33  -63.60  178.33   21.67\n",
      "   12        334E 334E N   CA   2686 2687  132.53  136.80                  -40.30\n",
      "   13  10563 334E 335L C   N    2693 2695 -122.47 -108.50   16.30    0.88  -63.50  175.50   21.81\n",
      "   13        335L 335L N   CA   2695 2696  124.10  132.50                  -41.20\n",
      "   14  10566 337D 338L C   N    2717 2719  -58.61  -70.70   18.61    1.17  -63.50  168.73   23.71\n",
      "   14        338L 338L N   CA   2719 2720  127.46  141.60                  -41.20\n",
      "   15  10584 355Q 356P C   N    2873 2875  -48.93  -58.70   12.86    0.98  -64.50  174.63   13.87\n",
      "   15        356P 356P N   CA   2875 2876  -38.86  -30.50                  147.20\n",
      "   16  10585 356P 357G C   N    2880 2882   91.36   78.70   18.23    0.40  -80.20  171.70   11.64\n",
      "   16        357G 357G N   CA   2882 2883 -179.22 -166.10                  174.10\n",
      "   17  10586 357G 358Y C   N    2884 2886  -81.93  -98.40   20.39    1.36  -63.50  177.14   29.06\n",
      "   17        358Y 358Y N   CA   2886 2887  140.41  128.40                  -43.40\n",
      "   18  10587 358Y 359R C   N    2896 2898  -58.53  -72.10   18.14    1.18  -63.00  171.02   23.40\n",
      "   18        359R 359R N   CA   2898 2899  129.86  141.90                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    0    2   18   86  141  157  192  308  369  477  508\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "6qa4_MODLL.B99990001.pdb       210.02165\n",
      "\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2768 atoms and 1 coordinate set(s) were parsed in 0.03s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 6qag\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    37619    34973\n",
      "239 (of 2915 total) atoms selected for optimization\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     239\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    34973    1951\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      391\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         153.4898\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     247       0      0   0.004   0.004      1.3878       1.000\n",
      " 2 Bond angle potential               :     340       0      0   1.952   1.952      23.978       1.000\n",
      " 3 Stereochemical cosine torsion poten:     156       0      6  51.797  51.797      60.368       1.000\n",
      " 4 Stereochemical improper torsion pot:      98       0      0   1.087   1.087      2.2963       1.000\n",
      " 5 Soft-sphere overlap restraints     :     391       0      0   0.001   0.001     0.53128E-01   1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     294       0      0   0.131   0.131      3.7943       1.000\n",
      "10 Distance restraints 2 (N-O)        :     296       0      0   0.239   0.239      5.9674       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      35       0      0   3.658   3.658      5.1939       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      23       0      0  58.510  58.510     0.46387       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      19       0      0  57.595  57.595      5.9298       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       9       0      0  52.050  52.050      4.7293       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       2       0      0 138.265 138.265      1.9732       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     220       0      0   0.567   0.567      2.0723       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      37      19      2  22.198 128.522      22.314       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     175       0      0   0.845   0.845      12.969       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 6qag_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    5467.1353\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9 -176.29 -134.00   43.88    1.08  -62.50 -163.32   38.23\n",
      "    1          2A   2A N   CA      9   10  158.68  147.00                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14  -85.65  -68.20   19.54    1.35  -62.50  166.62   28.50\n",
      "    2          3A   3A N   CA     14   15  154.10  145.30                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19 -107.49 -134.00   28.49    0.65  -62.50 -176.93   28.16\n",
      "    3          4A   4A N   CA     19   20  136.56  147.00                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24  -57.99  -68.20   27.10    1.84  -62.50  161.16   26.63\n",
      "    4          5A   5A N   CA     24   25  120.20  145.30                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29  -76.71  -68.20    8.88    0.66  -62.50  171.85   28.88\n",
      "    5          6A   6A N   CA     29   30  147.84  145.30                  -40.90\n",
      "    6  10235   6A   7A C   N      32   34  -67.03  -68.20    4.97    0.44  -62.50  169.03   27.92\n",
      "    6          7A   7A N   CA     34   35  150.13  145.30                  -40.90\n",
      "    7  10236   7A   8G C   N      37   39  -85.46  -80.20   21.08    0.59   82.20 -118.37   18.47\n",
      "    7          8G   8G N   CA     39   40 -165.49  174.10                    8.50\n",
      "    8  10237   8G   9A C   N      41   43  -61.20  -68.20   13.50    1.30  -62.50  162.26   26.53\n",
      "    8          9A   9A N   CA     43   44  156.85  145.30                  -40.90\n",
      "    9  10238   9A  10G C   N      46   48 -138.85 -167.20   31.05    0.46   82.20 -153.01   15.71\n",
      "    9         10G  10G N   CA     48   49  161.93  174.60                    8.50\n",
      "   10  10389 160T 161C C   N    1283 1285   72.57   57.40   18.28    1.39 -117.90 -154.98    8.79\n",
      "   10        161C 161C N   CA   1285 1286   25.81   36.00                  141.10\n",
      "   11  10559 330K 331F C   N    2657 2659 -108.79 -124.20   21.93    1.21  -63.20  163.29   25.72\n",
      "   11        331F 331F N   CA   2659 2660  158.90  143.30                  -44.30\n",
      "   12  10560 331F 332D C   N    2668 2670  -81.67  -70.90   16.50    0.62  -63.30  158.27   20.69\n",
      "   12        332D 332D N   CA   2670 2671  162.80  150.30                  -40.00\n",
      "   13  10562 333M 334E C   N    2684 2686 -150.47 -117.80   41.48    1.25  -63.60  179.74   28.78\n",
      "   13        334E 334E N   CA   2686 2687  162.35  136.80                  -40.30\n",
      "   14  10563 334E 335L C   N    2693 2695  -85.89  -70.70   24.03    2.35  -63.50  165.71   21.82\n",
      "   14        335L 335L N   CA   2695 2696  122.99  141.60                  -41.20\n",
      "   15  10564 335L 336D C   N    2701 2703  -88.32  -70.90   21.06    1.03  -63.30  159.84   21.31\n",
      "   15        336D 336D N   CA   2703 2704  162.13  150.30                  -40.00\n",
      "   16  10565 336D 337D C   N    2709 2711  -93.96  -96.50   39.62    1.67  -63.30  118.69   13.10\n",
      "   16        337D 337D N   CA   2711 2712   74.66  114.20                  -40.00\n",
      "   17  10566 337D 338L C   N    2717 2719  -84.63  -70.70   26.22    2.50   60.20  170.41   12.41\n",
      "   17        338L 338L N   CA   2719 2720  119.39  141.60                   29.60\n",
      "   18  10586 357G 358Y C   N    2884 2886 -115.69 -124.30   15.02    0.60  -63.50  174.48   25.25\n",
      "   18        358Y 358Y N   CA   2886 2887  123.09  135.40                  -43.40\n",
      "   19  10587 358Y 359R C   N    2896 2898  -79.06  -72.10    7.16    0.51  -63.00  176.09   24.88\n",
      "   19        359R 359R N   CA   2898 2899  143.55  141.90                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    0    1   11   57   93  153  176  238  355  462  473\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "6qag_MODLL.B99990001.pdb       153.48984\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2761 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Modelando proteína 6qah\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    37580    34934\n",
      "261 (of 2915 total) atoms selected for optimization\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     261\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    34934    2242\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      484\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         282.1466\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     270       0      0   0.005   0.005      1.7434       1.000\n",
      " 2 Bond angle potential               :     372       0      1   2.247   2.247      35.792       1.000\n",
      " 3 Stereochemical cosine torsion poten:     163       0      7  51.417  51.417      65.267       1.000\n",
      " 4 Stereochemical improper torsion pot:     107       0      1   1.574   1.574      6.7615       1.000\n",
      " 5 Soft-sphere overlap restraints     :     484       0      0   0.002   0.002     0.22126       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     409       0      0   0.224   0.224      13.589       1.000\n",
      "10 Distance restraints 2 (N-O)        :     386       0      0   0.338   0.338      11.922       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      40       0      0   4.687   4.687      9.6610       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      24       0      0  90.106  90.106      8.7369       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      20       0      0  54.561  54.561      5.7853       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:      10       0      0  88.947  88.947      5.8741       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       2       0      0 121.602 121.602     0.85412       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     250       0      0   0.462   0.462      1.2556       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      43      19     10  35.365 110.281      95.287       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     146       0      1   1.388   1.388      19.397       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 6qah_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    5857.6797\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9  -58.24  -68.20   11.14    0.77  -62.50  178.83   29.10\n",
      "    1          2A   2A N   CA      9   10  140.32  145.30                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14 -102.71 -134.00   38.21    0.95  -62.50  170.76   26.32\n",
      "    2          3A   3A N   CA     14   15  125.06  147.00                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19  -86.01  -68.20   21.73    1.42  -62.50  163.05   27.93\n",
      "    3          4A   4A N   CA     19   20  157.75  145.30                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24 -101.82 -134.00   32.43    0.93  -62.50 -179.50   31.65\n",
      "    4          5A   5A N   CA     24   25  142.94  147.00                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29  -64.45  -68.20    9.81    0.66  -62.50  177.14   28.93\n",
      "    5          6A   6A N   CA     29   30  136.23  145.30                  -40.90\n",
      "    6  10236   7A   8G C   N      37   39   82.96   78.70    8.69    0.13   82.20  177.83    8.67\n",
      "    6          8G   8G N   CA     39   40 -173.67 -166.10                    8.50\n",
      "    7  10237   8G   9A C   N      41   43  -74.67  -68.20   18.46    1.27  -62.50  156.98   26.34\n",
      "    7          9A   9A N   CA     43   44  162.60  145.30                  -40.90\n",
      "    8  10238   9A  10G C   N      46   48  114.30   78.70   35.72    1.73   82.20 -179.58    7.14\n",
      "    8         10G  10G N   CA     48   49 -169.03 -166.10                    8.50\n",
      "    9  10261  32G  33E C   N     225  227 -137.53  -63.60  108.36   12.82  -63.60  108.36   12.82\n",
      "    9         33E  33E N   CA    227  228   38.93  -40.30                  -40.30\n",
      "   10  10264  35A  36Y C   N     243  245 -171.05  -63.50  110.08   19.97  -63.50  110.08   19.97\n",
      "   10         36Y  36Y N   CA    245  246  -66.86  -43.40                  -43.40\n",
      "   11  10389 160T 161C C   N    1283 1285   70.67   57.40   15.55    1.25 -117.90 -154.57    8.78\n",
      "   11        161C 161C N   CA   1285 1286   27.90   36.00                  141.10\n",
      "   12  10561 332D 333M C   N    2676 2678 -160.24 -125.60   59.15    1.96  -63.40  162.95   31.61\n",
      "   12        333M 333M N   CA   2678 2679 -171.55  140.50                  -40.50\n",
      "   13  10562 333M 334E C   N    2684 2686  -66.68  -69.30    7.00    0.61  -63.60  170.73   22.93\n",
      "   13        334E 334E N   CA   2686 2687  149.00  142.50                  -40.30\n",
      "   14  10564 335L 336D C   N    2701 2703 -116.08  -96.50   23.13    0.94  -63.30  151.40   16.29\n",
      "   14        336D 336D N   CA   2703 2704  101.91  114.20                  -40.00\n",
      "   15  10565 336D 337D C   N    2709 2711 -139.72  -96.50   94.83    3.93  -63.30  103.50   11.91\n",
      "   15        337D 337D N   CA   2711 2712   29.80  114.20                  -40.00\n",
      "   16  10566 337D 338L C   N    2717 2719  -65.92  -63.50   12.34    1.86   60.20  150.93   18.11\n",
      "   16        338L 338L N   CA   2719 2720  -53.31  -41.20                   29.60\n",
      "   17  10585 356P 357G C   N    2880 2882  118.53   78.70   39.86    2.05  -80.20  162.67   10.13\n",
      "   17        357G 357G N   CA   2882 2883 -164.54 -166.10                  174.10\n",
      "   18  10586 357G 358Y C   N    2884 2886 -114.75 -124.30   11.98    0.99  -63.50 -178.65   31.38\n",
      "   18        358Y 358Y N   CA   2886 2887  142.64  135.40                  -43.40\n",
      "   19  10587 358Y 359R C   N    2896 2898  -59.98  -72.10   12.15    0.95  -63.00  176.19   23.60\n",
      "   19        359R 359R N   CA   2898 2899  142.73  141.90                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    0    1    8   47  141  147  196  259  344  449  503\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2739 atoms and 1 coordinate set(s) were parsed in 0.03s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "6qah_MODLL.B99990001.pdb       282.14661\n",
      "\n",
      "Modelando proteína 6qal\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    37320    34674\n",
      "294 (of 2915 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2801 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     294\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    34674    2500\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      522\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         208.5802\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     304       0      0   0.003   0.003     0.82971       1.000\n",
      " 2 Bond angle potential               :     416       0      0   1.822   1.822      25.341       1.000\n",
      " 3 Stereochemical cosine torsion poten:     183       0      2  48.272  48.272      63.496       1.000\n",
      " 4 Stereochemical improper torsion pot:     121       0      0   0.966   0.966      2.5603       1.000\n",
      " 5 Soft-sphere overlap restraints     :     522       0      0   0.003   0.003     0.55213       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     409       0      0   0.155   0.155      6.5871       1.000\n",
      "10 Distance restraints 2 (N-O)        :     404       0      0   0.276   0.276      10.929       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      44       1      0  27.277  27.277      4.6197       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      27       0      0  79.314  79.314      9.1423       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      23       0      0  69.561  69.561      7.7253       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:      12       0      0  74.547  74.547      6.1136       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       3       0      0  54.610  54.610      2.1116       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     308       0      0   0.747   0.747      3.7730       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      47      23      5  32.620 115.815      49.728       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     199       0      0   1.014   1.014      15.073       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 6qal_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    6656.6514\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 13                           : Mainchain Omega dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       4.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10747 160T 160T CA  C    1279 1283    0.44 -179.18  179.63   41.75 -179.18  179.63   41.75\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9  -77.32  -68.20   15.03    0.95  -62.50  162.53   27.39\n",
      "    1          2A   2A N   CA      9   10  157.25  145.30                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14  -71.53  -68.20    5.31    0.33  -62.50  169.90   28.29\n",
      "    2          3A   3A N   CA     14   15  149.44  145.30                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19 -107.06 -134.00   27.41    1.10  -62.50  172.92   30.70\n",
      "    3          4A   4A N   CA     19   20  152.02  147.00                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24  -64.06  -68.20    6.57    0.41  -62.50  178.91   29.40\n",
      "    4          5A   5A N   CA     24   25  140.20  145.30                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29  -72.85  -68.20   11.56    1.08  -62.50  175.92   28.34\n",
      "    5          6A   6A N   CA     29   30  134.71  145.30                  -40.90\n",
      "    6  10236   7A   8G C   N      37   39  -67.48  -62.40    7.08    1.00   82.20  156.24   11.77\n",
      "    6          8G   8G N   CA     39   40  -36.27  -41.20                    8.50\n",
      "    7  10237   8G   9A C   N      41   43  -51.75  -68.20   16.74    1.52  -62.50  171.01   27.52\n",
      "    7          9A   9A N   CA     43   44  148.43  145.30                  -40.90\n",
      "    8  10238   9A  10G C   N      46   48  117.81   78.70   39.43    2.09   82.20  173.29    6.62\n",
      "    8         10G  10G N   CA     48   49 -161.09 -166.10                    8.50\n",
      "    9  10259  30Y  31I C   N     213  215 -102.34  -63.40   39.36    7.20  -63.40   39.36    7.20\n",
      "    9         31I  31I N   CA    215  216  -49.28  -43.60                  -43.60\n",
      "   10  10261  32G  33E C   N     225  227 -102.30  -69.30  156.97   10.79 -117.80  159.91    8.29\n",
      "   10         33E  33E N   CA    227  228  -64.04  142.50                  136.80\n",
      "   11  10262  33E  34G C   N     234  236   73.42   78.70   29.80    0.64   82.20  145.54    7.51\n",
      "   11         34G  34G N   CA    236  237 -136.77 -166.10                    8.50\n",
      "   12  10263  34G  35A C   N     238  240  -61.77  -68.20   15.64    1.46  -62.50  159.55   26.11\n",
      "   12         35A  35A N   CA    240  241  159.56  145.30                  -40.90\n",
      "   13  10264  35A  36Y C   N     243  245  -46.16  -98.40   53.78    2.29  -63.50  176.30   27.10\n",
      "   13         36Y  36Y N   CA    245  246  141.15  128.40                  -43.40\n",
      "   14  10559 330K 331F C   N    2657 2659 -108.50 -124.20   21.93    1.20  -63.20  163.49   25.73\n",
      "   14        331F 331F N   CA   2659 2660  158.61  143.30                  -44.30\n",
      "   15  10560 331F 332D C   N    2668 2670  -58.95  -70.90   13.46    0.74  -63.30  175.95   21.46\n",
      "   15        332D 332D N   CA   2670 2671  144.11  150.30                  -40.00\n",
      "   16  10562 333M 334E C   N    2684 2686 -119.71 -117.80   18.29    0.88  -63.60  174.00   26.37\n",
      "   16        334E 334E N   CA   2686 2687  154.99  136.80                  -40.30\n",
      "   17  10563 334E 335L C   N    2693 2695  -85.94 -108.50   25.18    1.11  -63.50  164.07   21.60\n",
      "   17        335L 335L N   CA   2695 2696  121.33  132.50                  -41.20\n",
      "   18  10564 335L 336D C   N    2701 2703 -122.21  -96.50   28.01    1.16  -63.30  175.50   18.94\n",
      "   18        336D 336D N   CA   2703 2704  125.32  114.20                  -40.00\n",
      "   19  10565 336D 337D C   N    2709 2711  -65.12  -70.90   24.15    0.93  -63.30  166.85   20.49\n",
      "   19        337D 337D N   CA   2711 2712  126.85  150.30                  -40.00\n",
      "   20  10566 337D 338L C   N    2717 2719 -106.84 -108.50    3.00    0.17  -63.50 -178.54   23.09\n",
      "   20        338L 338L N   CA   2719 2720  135.01  132.50                  -41.20\n",
      "   21  10585 356P 357G C   N    2880 2882   64.18   78.70   21.18    1.12  -80.20  144.45    9.76\n",
      "   21        357G 357G N   CA   2882 2883  178.48 -166.10                  174.10\n",
      "   22  10586 357G 358Y C   N    2884 2886 -148.92 -124.30   44.59    1.77  -63.50  167.45   30.64\n",
      "   22        358Y 358Y N   CA   2886 2887  172.57  135.40                  -43.40\n",
      "   23  10587 358Y 359R C   N    2896 2898  -94.53  -72.10   35.91    3.11  -63.00  158.13   19.43\n",
      "   23        359R 359R N   CA   2898 2899  113.85  141.90                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    1    2   13   51  114  169  174  255  336  434  487\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "6qal_MODLL.B99990001.pdb       208.58017\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Modelando proteína 6qaq\n",
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    37928    35282\n",
      "208 (of 2915 total) atoms selected for optimization\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "@> 2738 atoms and 1 coordinate set(s) were parsed in 0.02s.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_484W> Dihedral still outside +-90:      -90.8202\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     208\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    35282    1821\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      323\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         149.7602\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     215       0      0   0.003   0.003     0.73112       1.000\n",
      " 2 Bond angle potential               :     295       0      0   1.862   1.862      18.846       1.000\n",
      " 3 Stereochemical cosine torsion poten:     134       0      8  56.861  56.861      59.468       1.000\n",
      " 4 Stereochemical improper torsion pot:      87       0      0   0.882   0.882      1.3431       1.000\n",
      " 5 Soft-sphere overlap restraints     :     323       0      0   0.002   0.002     0.16726       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     302       0      0   0.107   0.107      2.3697       1.000\n",
      "10 Distance restraints 2 (N-O)        :     298       0      0   0.339   0.339      7.4592       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      31       0      1   4.621   4.621      6.9774       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      19       0      0  68.301  68.301      2.4895       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      15       0      0  78.356  78.356      2.6004       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:       8       0      0  83.819  83.819      4.9049       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       2       0      0   3.273   3.273     0.82364       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     205       0      0   0.321   0.321     0.49386       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      33      14      5  29.428 110.474      28.726       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     177       0      0   0.773   0.773      12.360       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 6qaq_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    4523.2930\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9  -61.95  -68.20    7.03    0.68  -62.50  170.58   27.93\n",
      "    1          2A   2A N   CA      9   10  148.53  145.30                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14  -60.97  -68.20   12.49    0.79  -62.50  176.02   28.92\n",
      "    2          3A   3A N   CA     14   15  135.11  145.30                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19  -84.77  -68.20   25.19    1.58  -62.50  156.43   26.78\n",
      "    3          4A   4A N   CA     19   20  164.27  145.30                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24  -67.68  -68.20    0.92    0.06  -62.50  174.64   28.88\n",
      "    4          5A   5A N   CA     24   25  144.54  145.30                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29 -105.87 -134.00   34.11    0.84  -62.50  174.09   26.76\n",
      "    5          6A   6A N   CA     29   30  127.70  147.00                  -40.90\n",
      "    6  10237   8G   9A C   N      41   43 -124.37 -134.00   22.60    1.39  -62.50  163.80   30.16\n",
      "    6          9A   9A N   CA     43   44  167.44  147.00                  -40.90\n",
      "    7  10238   9A  10G C   N      46   48   94.27   78.70   16.23    0.90   82.20  170.45    7.66\n",
      "    7         10G  10G N   CA     48   49 -161.52 -166.10                    8.50\n",
      "    8  10389 160T 161C C   N    1283 1285   70.74   57.40   14.74    1.33 -117.90 -155.62    8.72\n",
      "    8        161C 161C N   CA   1285 1286   29.72   36.00                  141.10\n",
      "    9  10559 330K 331F C   N    2657 2659 -170.34 -124.20   74.36    2.15  -63.20  156.51   27.89\n",
      "    9        331F 331F N   CA   2659 2660 -158.39  143.30                  -44.30\n",
      "   10  10560 331F 332D C   N    2668 2670   58.70   54.50    4.34    0.44  -63.30  145.80   24.84\n",
      "   10        332D 332D N   CA   2670 2671   39.83   40.90                  -40.00\n",
      "   11  10561 332D 333M C   N    2676 2678 -103.92 -125.60   67.07    2.69  -63.40  124.32   15.93\n",
      "   11        333M 333M N   CA   2678 2679   77.03  140.50                  -40.50\n",
      "   12  10562 333M 334E C   N    2684 2686 -105.31 -117.80   68.82    3.20   54.60  162.13   17.48\n",
      "   12        334E 334E N   CA   2686 2687   69.12  136.80                   42.40\n",
      "   13  10586 357G 358Y C   N    2884 2886 -123.68 -124.30   13.81    0.76  -63.50  177.89   31.25\n",
      "   13        358Y 358Y N   CA   2886 2887  149.19  135.40                  -43.40\n",
      "   14  10587 358Y 359R C   N    2896 2898  -81.91  -72.10   19.49    1.30  -63.00  161.28   23.08\n",
      "   14        359R 359R N   CA   2898 2899  158.74  141.90                  -41.10\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:     0    0    0    0    0    2   12   43  131  149  185  259  368  420  472\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "6qaq_MODLL.B99990001.pdb       149.76018\n",
      "\n",
      "Modelando proteína 6qaw\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "check_ali___> Checking the sequence-structure alignment. \n",
      "\n",
      "Implied intrachain target CA(i)-CA(i+1) distances longer than  8.0 angstroms:\n",
      "\n",
      "ALN_POS  TMPL  RID1  RID2  NAM1  NAM2     DIST\n",
      "----------------------------------------------\n",
      "END OF TABLE\n",
      "read_to_681_> topology.submodel read from topology file:        3\n",
      "mdtrsr__446W> A potential that relies on one protein is used, yet you have at\n",
      "              least one known structure available. MDT, not library, potential is used.\n",
      "0 atoms in HETATM/BLK residues constrained\n",
      "to protein atoms within 2.30 angstroms\n",
      "and protein CA atoms within 10.00 angstroms\n",
      "0 atoms in residues without defined topology\n",
      "constrained to be rigid bodies\n",
      "condens_443_> Restraints marked for deletion were removed.\n",
      "              Total number of restraints before, now:    37308    34662\n",
      "302 (of 2915 total) atoms selected for optimization\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "iupac_m_397W> Atoms were not swapped because of the uncertainty of how to handle the H atom.\n",
      "\n",
      "\n",
      ">> ENERGY; Differences between the model's features and restraints:\n",
      "Number of all residues in MODEL                   :      360\n",
      "Number of all, selected real atoms                :     2915     302\n",
      "Number of all, selected pseudo atoms              :        0       0\n",
      "Number of all static, selected restraints         :    34662    2745\n",
      "COVALENT_CYS                                      :        F\n",
      "NONBONDED_SEL_ATOMS                               :        1\n",
      "Number of non-bonded pairs (excluding 1-2,1-3,1-4):      658\n",
      "Dynamic pairs routine                             : 5, NATM_SEL x NATM double loop\n",
      "Atomic shift for contacts update (UPDATE_DYNAMIC) :    0.390\n",
      "LENNARD_JONES_SWITCH                              :    6.500   7.500\n",
      "COULOMB_JONES_SWITCH                              :    6.500   7.500\n",
      "RESIDUE_SPAN_RANGE                                :        0   99999\n",
      "NLOGN_USE                                         :       15\n",
      "CONTACT_SHELL                                     :    4.000\n",
      "DYNAMIC_PAIRS,_SPHERE,_COULOMB,_LENNARD,_MODELLER :        T       T       F       F       F\n",
      "SPHERE_STDV                                       :    0.050\n",
      "RADII_FACTOR                                      :    0.820\n",
      "Current energy                                    :         174.4504\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Summary of the restraint violations: \n",
      "\n",
      "   NUM     ... number of restraints.\n",
      "   NUMVI   ... number of restraints with RVIOL > VIOL_REPORT_CUT[i].\n",
      "   RVIOL   ... relative difference from the best value.\n",
      "   NUMVP   ... number of restraints with -Ln(pdf) > VIOL_REPORT_CUT2[i].\n",
      "   RMS_1   ... RMS(feature, minimally_violated_basis_restraint, NUMB).\n",
      "   RMS_2   ... RMS(feature, best_value, NUMB).\n",
      "   MOL.PDF ... scaled contribution to -Ln(Molecular pdf).\n",
      "\n",
      " #                     RESTRAINT_GROUP      NUM   NUMVI  NUMVP   RMS_1   RMS_2         MOL.PDF     S_i\n",
      "------------------------------------------------------------------------------------------------------\n",
      " 1 Bond length potential              :     315       0      0   0.004   0.004      1.1291       1.000\n",
      " 2 Bond angle potential               :     434       0      0   1.924   1.924      29.899       1.000\n",
      " 3 Stereochemical cosine torsion poten:     215       0      3  45.170  45.170      60.374       1.000\n",
      " 4 Stereochemical improper torsion pot:     134       0      0   0.992   0.992      2.8786       1.000\n",
      " 5 Soft-sphere overlap restraints     :     658       0      0   0.001   0.001     0.17432       1.000\n",
      " 6 Lennard-Jones 6-12 potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 7 Coulomb point-point electrostatic p:       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 8 H-bonding potential                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      " 9 Distance restraints 1 (CA-CA)      :     476       0      0   0.118   0.118      3.3380       1.000\n",
      "10 Distance restraints 2 (N-O)        :     440       0      0   0.225   0.225      9.1142       1.000\n",
      "11 Mainchain Phi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "12 Mainchain Psi dihedral restraints  :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "13 Mainchain Omega dihedral restraints:      43       0      0   3.540   3.540      4.9424       1.000\n",
      "14 Sidechain Chi_1 dihedral restraints:      30       0      1  59.243  59.243      8.1785       1.000\n",
      "15 Sidechain Chi_2 dihedral restraints:      25       0      1  59.751  59.751      10.283       1.000\n",
      "16 Sidechain Chi_3 dihedral restraints:      11       0      0  71.234  71.234      7.8694       1.000\n",
      "17 Sidechain Chi_4 dihedral restraints:       3       0      0  93.154  93.154      2.9443       1.000\n",
      "18 Disulfide distance restraints      :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "19 Disulfide angle restraints         :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "20 Disulfide dihedral angle restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "21 Lower bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "22 Upper bound distance restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "23 Distance restraints 3 (SDCH-MNCH)  :     286       0      0   0.524   0.524      2.1872       1.000\n",
      "24 Sidechain Chi_5 dihedral restraints:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "25 Phi/Psi pair of dihedral restraints:      46      21      5  23.096 115.944      18.690       1.000\n",
      "26 Distance restraints 4 (SDCH-SDCH)  :     287       0      0   0.676   0.676      12.449       1.000\n",
      "27 Distance restraints 5 (X-Y)        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "28 NMR distance restraints 6 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "29 NMR distance restraints 7 (X-Y)    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "30 Minimal distance restraints        :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "31 Non-bonded restraints              :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "32 Atomic accessibility restraints    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "33 Atomic density restraints          :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "34 Absolute position restraints       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "35 Dihedral angle difference restraint:       0       0      0   0.000   0.000      0.0000       1.000\n",
      "36 GBSA implicit solvent potential    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "37 EM density fitting potential       :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "38 SAXS restraints                    :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "39 Symmetry restraints                :       0       0      0   0.000   0.000      0.0000       1.000\n",
      "\n",
      "\n",
      "\n",
      "# Heavy relative violation of each residue is written to: 6qaw_MODLL.V99990001\n",
      "# The profile is NOT normalized by the number of restraints.\n",
      "# The profiles are smoothed over a window of residues:    1\n",
      "# The sum of all numbers in the file:    6248.2485\n",
      "\n",
      "\n",
      "\n",
      "List of the violated restraints:\n",
      "   A restraint is violated when the relative difference\n",
      "   from the best value (RVIOL) is larger than CUTOFF.\n",
      "\n",
      "   ICSR   ... index of a restraint in the current set.\n",
      "   RESNO  ... residue numbers of the first two atoms.\n",
      "   ATM    ... IUPAC atom names of the first two atoms.\n",
      "   FEAT   ... the value of the feature in the model.\n",
      "   restr  ... the mean of the basis restraint with the smallest\n",
      "              difference from the model (local minimum).\n",
      "   viol   ... difference from the local minimum.\n",
      "   rviol  ... relative difference from the local minimum.\n",
      "   RESTR  ... the best value (global minimum).\n",
      "   VIOL   ... difference from the best value.\n",
      "   RVIOL  ... relative difference from the best value.\n",
      "\n",
      "\n",
      "-------------------------------------------------------------------------------------------------\n",
      "\n",
      "Feature 25                           : Phi/Psi pair of dihedral restraints     \n",
      "List of the RVIOL violations larger than   :       6.5000\n",
      "\n",
      "    #   ICSR  RESNO1/2 ATM1/2   INDATM1/2    FEAT   restr    viol   rviol   RESTR    VIOL   RVIOL\n",
      "    1  10230   1M   2A C   N       7    9  -82.12  -68.20   18.58    1.18  -62.50  162.69   27.67\n",
      "    1          2A   2A N   CA      9   10  157.60  145.30                  -40.90\n",
      "    2  10231   2A   3A C   N      12   14  -70.12  -68.20   17.52    1.34  -62.50  156.57   26.04\n",
      "    2          3A   3A N   CA     14   15  162.72  145.30                  -40.90\n",
      "    3  10232   3A   4A C   N      17   19  -63.57  -68.20   16.96    1.52  -62.50  157.49   25.86\n",
      "    3          4A   4A N   CA     19   20  161.62  145.30                  -40.90\n",
      "    4  10233   4A   5A C   N      22   24  -68.62  -68.20   16.44    1.31  -62.50  157.49   26.11\n",
      "    4          5A   5A N   CA     24   25  161.73  145.30                  -40.90\n",
      "    5  10234   5A   6A C   N      27   29  -66.28  -68.20   13.15    1.13  -62.50  160.84   26.55\n",
      "    5          6A   6A N   CA     29   30  158.31  145.30                  -40.90\n",
      "    6  10236   7A   8G C   N      37   39  -64.53  -62.40    2.39    0.37   82.20  154.57   11.73\n",
      "    6          8G   8G N   CA     39   40  -40.11  -41.20                    8.50\n",
      "    7  10237   8G   9A C   N      41   43  -62.66  -68.20   10.46    1.01  -62.50  164.93   27.04\n",
      "    7          9A   9A N   CA     43   44  154.17  145.30                  -40.90\n",
      "    8  10238   9A  10G C   N      46   48 -127.42 -167.20   45.32    1.44   82.20 -131.39    6.69\n",
      "    8         10G  10G N   CA     48   49 -163.69  174.60                    8.50\n",
      "    9  10355 126I 127C C   N    1000 1002  -62.18  -63.00    2.31    0.24 -117.90 -175.73    7.94\n",
      "    9        127C 127C N   CA   1002 1003  -43.26  -41.10                  141.10\n",
      "   10  10356 127C 128Y C   N    1006 1008  -62.31  -63.50    2.53    0.35   55.90  145.67   17.79\n",
      "   10        128Y 128Y N   CA   1008 1009  -45.63  -43.40                   39.50\n",
      "   11  10558 329F 330K C   N    2648 2650  -78.14  -70.20   25.59    1.67  -62.90  155.22   21.02\n",
      "   11        330K 330K N   CA   2650 2651  164.72  140.40                  -40.80\n",
      "   12  10559 330K 331F C   N    2657 2659 -110.76 -124.20   39.45    2.12  -63.20  143.42   23.06\n",
      "   12        331F 331F N   CA   2659 2660 -179.60  143.30                  -44.30\n",
      "   13  10560 331F 332D C   N    2668 2670 -106.36  -96.50   13.34    0.56  -63.30  168.77   18.65\n",
      "   13        332D 332D N   CA   2670 2671  123.18  114.20                  -40.00\n",
      "   14  10561 332D 333M C   N    2676 2678 -130.68 -125.60   40.15    1.81  -63.40  154.58   28.36\n",
      "   14        333M 333M N   CA   2678 2679 -179.67  140.50                  -40.50\n",
      "   15  10562 333M 334E C   N    2684 2686  -66.18  -69.30   21.28    1.51  -63.60  161.77   21.43\n",
      "   15        334E 334E N   CA   2686 2687  121.45  142.50                  -40.30\n",
      "   16  10563 334E 335L C   N    2693 2695 -134.82 -108.50   48.64    2.29  -63.50  161.94   26.90\n",
      "   16        335L 335L N   CA   2695 2696  173.41  132.50                  -41.20\n",
      "   17  10564 335L 336D C   N    2701 2703  -71.54  -70.90    2.04    0.07  -63.30  167.98   21.24\n",
      "   17        336D 336D N   CA   2703 2704  152.23  150.30                  -40.00\n",
      "   18  10565 336D 337D C   N    2709 2711  -89.40  -70.90   19.03    1.55  -63.30  176.09   23.38\n",
      "   18        337D 337D N   CA   2711 2712  145.86  150.30                  -40.00\n",
      "   19  10566 337D 338L C   N    2717 2719 -115.56 -108.50   13.39    0.63  -63.50 -177.50   28.55\n",
      "   19        338L 338L N   CA   2719 2720  143.88  132.50                  -41.20\n",
      "   20  10585 356P 357G C   N    2880 2882   94.68   78.70   48.40    0.79  -80.20  176.79   10.91\n",
      "   20        357G 357G N   CA   2882 2883  148.21 -166.10                  174.10\n",
      "   21  10586 357G 358Y C   N    2884 2886  -81.75  -98.40   23.97    1.87  -63.50  171.93   28.22\n",
      "   21        358Y 358Y N   CA   2886 2887  145.64  128.40                  -43.40\n",
      "\n",
      "\n",
      "report______> Distribution of short non-bonded contacts:\n",
      "\n",
      "  serious non-bonded atom clash:   249  414   2.260\n",
      "  serious non-bonded atom clash:   250  409   1.875\n",
      "  serious non-bonded atom clash:   251  413   1.575\n",
      "  serious non-bonded atom clash:   251  414   0.945\n",
      "  serious non-bonded atom clash:   251  415   2.178\n",
      "  serious non-bonded atom clash:   252  407   2.143\n",
      "  serious non-bonded atom clash:   252  409   0.984\n",
      "  serious non-bonded atom clash:   252  410   1.490\n",
      "  serious non-bonded atom clash:   252  413   1.694\n",
      "  serious non-bonded atom clash:   252  415   2.281\n",
      "  serious non-bonded atom clash:   253  409   2.177\n",
      "  serious non-bonded atom clash:   253  410   1.519\n",
      "  serious non-bonded atom clash:   253  413   0.426\n",
      "  serious non-bonded atom clash:   253  414   1.192\n",
      "  serious non-bonded atom clash:   253  415   1.561\n",
      "  serious non-bonded atom clash:   254  410   1.536\n",
      "  serious non-bonded atom clash:   254  413   1.038\n",
      "  serious non-bonded atom clash:   254  414   1.751\n",
      "  serious non-bonded atom clash:   254  415   1.817\n",
      "\n",
      "DISTANCE1:  0.00 2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40\n",
      "DISTANCE2:  2.10 2.20 2.30 2.40 2.50 2.60 2.70 2.80 2.90 3.00 3.10 3.20 3.30 3.40 3.50\n",
      "FREQUENCY:    14    3    2    3    2    2   15   54  124  166  188  250  336  435  482\n",
      "\n",
      "\n",
      "<< end of ENERGY.\n",
      "\n",
      ">> Summary of successfully produced models:\n",
      "Filename                          molpdf\n",
      "----------------------------------------\n",
      "6qaw_MODLL.B99990001.pdb       174.45042\n",
      "\n",
      "6400.763684511185\n"
     ]
    }
   ],
   "source": [
    "import os\n",
    "import time\n",
    "start = time.time()\n",
    "\n",
    "for pdb_code in pdbids_list:\n",
    "    ###########################\n",
    "    if os.path.isfile(pdbs_model_dir + pdb_code + tail_model + \".pdb\"):\n",
    "        print(\"El modelo ya existe en el directorio\")\n",
    "        continue\n",
    "    ###########################\n",
    "    try:\n",
    "        # Se lee el archivo pdb. La selección omite residuos negativos.\n",
    "        # Además omite los residuos no estandar, con modificaciones postTrad como el TPO (Tirosina fosfatada)\n",
    "        stc_prot = parsePDB(struct_dir \n",
    "                            + pdb_code + tail_pdb).select('not nonstdaa and resid > 0') \n",
    "        chaid = stc_prot.getChids()[0] # Obtiene el id de la cadena\n",
    "        ref_hv = stc_prot.getHierView()[chaid] # Se obtiene sólo la cadena\n",
    "        seq_cry = ref_hv.getSequence() # Se obtiene la secuencia de la estructura cristalográfica   \n",
    "    except:\n",
    "        print(\"Error al abrir y modelar:\", pdb_code)\n",
    "        continue\n",
    "    ########################### \n",
    "    # Pregunta si la longitud y la secuencias de Uniprot y de la estructura son iguales\n",
    "    same_seq = len(seq_cry) == len(seq_prot) and seq_cry == seq_prot\n",
    "    # si same_seq es verdadero, se omite hacer el modelado pues la estructura está completa\n",
    "    # y se pasa a la siguiente secuencia\n",
    "    if same_seq:\n",
    "        print(\"La proteína \" + pdb_code + \" ya está completa\")\n",
    "        continue  \n",
    "        ''' SI HACEN FALTA ÁTOMOS A UN RESIDUO, SE AGREGARÁN CON pdb4amber en un fase posterior al modelado.'''\n",
    "    print(\"Modelando proteína \" + pdb_code)\n",
    "    ###########################\n",
    "    # Obtener la secuecnia de la estructura a modelar y guardarla en un archivo\n",
    "    # Alineamiento global, se penalizan con -10 los gaps abiertos, se obtiene el mejor\n",
    "    alignment = pairwise2.align.globalxs(seq_cry, seq_prot, -10, -0.5, gap_char='-')[0]\n",
    "    # Secuencias del alineamiento\n",
    "    algn1_struc = alignment[0] # Secuencia alineada de la estructura cristalográfica\n",
    "    algn2_seq = seq_prot # Secuencia completa de UniProt\n",
    "\n",
    "    # Nombre de los cabezales de las secuencia deben coincidir con el de los archivos de entrada y salida\n",
    "    crys_file_name = pdb_code + tail_pdb\n",
    "    model_file_name = pdb_code + tail_model\n",
    "\n",
    "    ''' NECESARIO: There should be 10 fields separated by colons \":\".\n",
    "    Please check the file to make sure your sequences end with the '*' character.\n",
    "    Nomenclaturas de los campos del header: https://salilab.org/modeller/8v2/manual/node176.html'''\n",
    "\n",
    "    # HEADERS (la secuencia del cristal va primero, luego la secuencia completa)\n",
    "    struc_header = \"structureX:\" + crys_file_name + \":.:\" + chaid + \":.:\" + chaid + \":.:.:.:\"\n",
    "    seq_header = \"sequence:\" + model_file_name + \":.:.:.:.:.:.:.:\"\n",
    "    \n",
    "    # Crea el archivo de alineamiento con la estructura requerida por modeller\n",
    "    alg_filename = pdb_code + \".alg\"\n",
    "    with open(alg_filename, \"w\") as handle:\n",
    "        handle.write(\"\\n>P1;%s\\n%s\\n%s*\\n>P1;%s\\n%s\\n%s*\\n\" % (crys_file_name, struc_header, algn1_struc, \n",
    "                                                               model_file_name, seq_header, algn2_seq))\n",
    "\n",
    "    ##########################\n",
    "    # Ejecuta find_gaps para obtener los posibles gaps de la secuencia\n",
    "    gaps = find_gaps(algn1_struc, r = num_res_window)\n",
    "    num_gaps = gaps[\"num_gaps\"]\n",
    "    # Comprueba si hay gaps\n",
    "    # if num_gaps != 0:\n",
    "     #   print(\"Se omite hacer el modelado de \" + pdb_code + \", la proteína ya está completa o tiene más residuos que la secuencia\")\n",
    "     #   print(\"Longitud de secuencia: \" + str(len(algn2_seq)))\n",
    "     #   print(\"Longitud de la estructura: \" + str(len(algn1_struc)))\n",
    "     #   continue\n",
    "\n",
    "    gap_i = gaps[\"gap_window\"]\n",
    "    # Se define el string que necesita Modeller para definir al primer gap\n",
    "    s = \"self.residue_range('\" + str(gap_i[0][0]) + \"', '\" + str(gap_i[0][1]) + \"')\"\n",
    "\n",
    "    # A pesar de que sólo haya un gap, se ejecuta el ciclo, \n",
    "    # si hay más de uno, se extiende el String para incluir los demás\n",
    "    for i in range(1, num_gaps):\n",
    "        s = s + \", \" + \"self.residue_range('\" + str(gap_i[i][0]) + \"', '\" + str(gap_i[i][1]) + \"')\" #nótese la coma\n",
    "    ##########################\n",
    "    ''' SE EJECUTA MODELLER'''\n",
    "    ##########################\n",
    "    env = environ()\n",
    "    env.io.atom_files_directory = ['.', struct_dir]\n",
    "    # Se modifica la clase MyModel de Modeller para fijar los residuos ya existentes en el cristal\n",
    "    # Estos residuos no se modelarán ni sus átomos cambiarán de posición\n",
    "    # NECESARIO para poder capturar los valores obtenidos por el rango de gaps\n",
    "    MyModel_code = \"\"\"\n",
    "class MyModel(automodel):\n",
    "    def select_atoms(self):\n",
    "        return selection(\"\"\" + s + \"\"\")\n",
    "\"\"\" # Al estar dentro del loop la identación extra es importante\n",
    "    exec(MyModel_code) # Se lleva a cabo la modificación a MyModel para agregar la región fijada\n",
    "    # Se intancia el objeto MyModel\n",
    "    a = MyModel(env, alnfile = alg_filename, # Lee el archivo fasta creado y guardado en el directorio actual\n",
    "                      knowns = crys_file_name, # Archivo pdb crys, que coincide con el id en el archivo fasta\n",
    "                      sequence = model_file_name) # Nombre del modelo\n",
    "    a.starting_model= 1\n",
    "    a.ending_model  = 1\n",
    "    #a.library_schedule = autosched.slow # Originalmente comentado\n",
    "    a.max_var_iterations = 500 #*500\n",
    "    a.md_level = refine.slow # Nivel del rifinamiento\n",
    "    #a.repeat_optimization = 2 ##Originalmente comentado\n",
    "    \n",
    "    # Más info sobre el refinamiento: https://salilab.org/modeller/9.21/manual/node19.html\n",
    "    a.make()\n",
    "\n",
    "    ###########################\n",
    "    # Renombra el archivo pdb a pdb_code + _full.pdb\n",
    "    # DEBE ser el único archivo pdb en el directorio '.'\n",
    "\n",
    "    model_file = glob.glob('./' + pdb_code + '*.pdb')[0] # Nombre del archivo pdb\n",
    "    os.rename(model_file, pdbs_model_dir + pdb_code + tail_model + \".pdb\") # Mueve el pdb a la carpeta model_pdbs\n",
    "    # Elimina los archivos extra generados - Todos empiezan con el codigo del PDB ID\n",
    "    for f in glob.glob(pdb_code + \"*\"):\n",
    "        os.remove(f)\n",
    "\n",
    "end = time. time()\n",
    "print(end - start)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [],
   "source": [
    "16\n",
    "34-36\n",
    "38\n",
    "39*\n",
    "54\n",
    "61*\n",
    "64\n",
    "86\n",
    "90-92\n",
    "153"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Como resultado se modelaron 402 estructuras.\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "\n"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.6.7"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
